<!doctype html>
<meta charset="utf-8">
<title>biosvm references</title>
<body>

<h1>biosvm references</h1>  [Mukherjee1998Support]  S.&nbsp;Mukherjee, 
P.&nbsp;Tamayo, J.&nbsp;P. Mesirov, D.&nbsp;Slonim, A.&nbsp;Verri, and 
T.&nbsp;Poggio. Support vector machine classification of microarray data. 
Technical Report 182, C.B.L.C., 1998. A.I. Memo 1677. [&nbsp;bib&nbsp;| .html
&nbsp;|.pdf&nbsp;] 
<blockquote>  Keywords: biosvm microarray </blockquote> <br>
 [
Jaakkola1999Exploiting]  T.&nbsp;S. Jaakkola and D.&nbsp;Haussler. Exploiting 
generative models in discriminative classifiers. In<em>Proc. of Tenth 
Conference on Advances in Neural Information Processing Systems</em>, 1999. 
[&nbsp;bib&nbsp;| .ps&nbsp;| .pdf&nbsp;] 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Jaakkola1999Using]  
T.&nbsp;S. Jaakkola, M.&nbsp;Diekhans, and D.&nbsp;Haussler. Using the Fisher 
Kernel Method to Detect Remote Protein Homologies. In<em>Proceedings of the 
Seventh International Conference on Intelligent Systems for Molecular Biology
</em>, pages 149-158. AAAI Press, 1999. [&nbsp;bib&nbsp;] 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Haussler1999Convolution]  
D.&nbsp;Haussler. Convolution Kernels on Discrete Structures. Technical Report 
UCSC-CRL-99-10, UC Santa Cruz, 1999. [&nbsp;bib&nbsp;| .ps&nbsp;| .pdf&nbsp;] 
<blockquote>  We introduce a new method of constructing kernels on sets whose 
elements are discrete structures like strings, trees and graphs. The method can 
be applied iteratively to build a kernel on a infinite set from kernels 
involving generators of the set. The family of kernels generated generalizes 
the family of radial basis kernels. It can also be used to define kernels in 
the form of joint Gibbs probability distributions. Kernels can be built from 
hidden Markov random fields, generalized regular expressions, pair-HMMs, or 
ANOVA decompositions. Uses of the method lead to open problems involving the 
theory of infinitely divisible positive definite functions. Fundamentals of 
this theory and the theory of reproducing kernel Hilbert spaces are reviewed 
and applied in establishing the validity of the method. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Zien2000Engineering]  
A.&nbsp;Zien, G.&nbsp;R&auml;tsch, S.&nbsp;Mika, B.&nbsp;Sch&ouml;lkopf, 
T.&nbsp;Lengauer, and K.-R. M&uuml;ller. Engineering support vector machine 
kernels that recognize translation initiation sites.<em>Bioinformatics</em>, 
16(9):799-807, 2000. [&nbsp;bib&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  Motivation: In order to extract protein sequences from 
nucleotide sequences, it is an important step to recognize points at which 
regions start that code for proteins. These points are called translation 
initiation sites (TIS). Results: The task of finding TIS can be modeled as a 
classification problem. We demonstrate the applicability of support vector 
machines for this task, and show how to incorporate prior biological knowledge 
by engineering an appropriate kernel function. With the described techniques 
the recognition performance can be improved by 26 We provide evidence that 
existing related methods (e.g. ESTScan) could profit from advanced TIS 
recognition. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Watkins2000Dynamic]  
C.&nbsp;Watkins. Dynamic alignment kernels. In A.J. Smola, P.L. Bartlett, 
B.&nbsp;Sch&ouml;lkopf, and D.&nbsp;Schuurmans, editors,<em>Advances in Large 
Margin Classifiers</em>, pages 39-50. MIT Press, Cambridge, MA, 2000. [&nbsp;bib
&nbsp;|.ps.gz&nbsp;| .pdf&nbsp;] 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Moler2000Analysis]  
E.&nbsp;J. Moler, M.&nbsp;L. Chow, and I.&nbsp;S. Mian. Analysis of molecular 
profile data using generative and discriminative methods.<em>Physiol. Genomics
</em>, 4(2):109-126, Dec 2000. [&nbsp;bib&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  A modular framework is proposed for modeling and understanding 
the relationships between molecular profile data and other domain knowledge 
using a combination of generative (here, graphical models) and discriminative 
[Support Vector Machines (SVMs)] methods. As illustration, naive Bayes models, 
simple graphical models, and SVMs were applied to published transcription 
profile data for 1,988 genes in 62 colon adenocarcinoma tissue specimens 
labeled as tumor or nontumor. These unsupervised and supervised learning 
methods identified three classes or subtypes of specimens, assigned tumor or 
nontumor labels to new specimens and detected six potentially mislabeled 
specimens. The probability parameters of the three classes were utilized to 
develop a novel gene relevance, ranking, and selection method. SVMs trained to 
discriminate nontumor from tumor specimens using only the 50-200 top-ranked 
genes had the same or better generalization performance than the full 
repertoire of 1,988 genes. Approximately 90 marker genes were pinpointed for 
use in understanding the basic biology of colon adenocarcinoma, defining 
targets for therapeutic intervention and developing diagnostic tools. These 
potential markers highlight the importance of tissue biology in the etiology of 
cancer. Comparative analysis of molecular profile data is proposed as a 
mechanism for predicting the physiological function of genes in instances when 
comparative sequence analysis proves uninformative, such as with human and 
yeast translationally controlled tumour protein. Graphical models and SVMs hold 
promise as the foundations for developing decision support systems for 
diagnosis, prognosis, and monitoring as well as inferring biological networks. 
</blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Lodhi2000Text]  
H.&nbsp;Lodhi, J.&nbsp;Shawe-Taylor, N.&nbsp;Cristianini, and C.&nbsp;J. 
C.&nbsp;H. Watkins. Text Classification using String Kernels. In<em>Adv. Neural 
Inform. Process. Syst.</em>, pages 563-569, 2000. [&nbsp;bib&nbsp;| .ps.gz
&nbsp;|.pdf&nbsp;] 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Jaakkola2000Discriminative]
 T.&nbsp;Jaakkola, M.&nbsp;Diekhans, and D.&nbsp;Haussler. A Discriminative 
Framework for Detecting Remote Protein Homologies.<em>J. Comput. Biol.</em>, 
7(1,2):95-114, 2000. [&nbsp;bib&nbsp;| .ps&nbsp;| .pdf&nbsp;] 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Furey2000Support]  
T.&nbsp;S. Furey, N.&nbsp;Cristianini, N.&nbsp;Duffy, D.&nbsp;W. Bednarski, 
M.&nbsp;Schummer, and D.&nbsp;Haussler. Support vector machine classification 
and validation of cancer tissue samples using microarray expression data.<em>
Bioinformatics</em>, 16(10):906-914, Oct 2000. [&nbsp;bib&nbsp;| http&nbsp;| 
.pdf&nbsp;] 
<blockquote>  Motivation: DNA microarray experiments generating thousands of 
gene expression measurements, are being used to gather information from tissue 
and cell samples regarding gene expression differences that will be useful in 
diagnosing disease. We have developed a new method to analyse this kind of data 
using support vector machines (SVMs). This analysis consists of both 
classification of the tissue samples, and an exploration of the data for 
mis-labeled or questionable tissue results. Results: We demonstrate the method 
in detail on samples consisting of ovarian cancer tissues, normal ovarian 
tissues, and other normal tissues. The dataset consists of expression 
experiment results for 97802 cDNAs for each tissue. As a result of 
computational analysis, a tissue sample is discovered and confirmed to be 
wrongly labeled. Upon correction of this mistake and the removal of an outlier, 
perfect classification of tissues is achieved, but not with high confidence. We 
identify and analyse a subset of genes from the ovarian dataset whose 
expression is highly differentiated between the types of tissues. To show 
robustness of the SVM method, two previously published datasets from other 
types of tissues or cells are analysed. The results are comparable to those 
previously obtained. We show that other machine learning methods also perform 
comparably to the SVM on many of those datasets. Availability: The SVM software 
is available at http://www.cs.columbia.edu/&nbsp;bgrundy/svm. Contact: 
booch@cse.ucsc.edu </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Cai2000Support]  Y.D. Cai, 
X.J. Liu, X.B. Xu, and K.C. Chou. Support vector machines for prediction of 
protein subcellular location.<em>Mol. Cell Biol. Res. Commun.</em>, 
4(4):230-234, 2000. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| www:&nbsp;] 
<blockquote>  Support Vector Machine (SVM), which is one kind of learning 
machines, was applied to predict the subcellular location of proteins from 
their amino acid composition. In this research, the proteins are classified 
into the following 12 groups: (1) chloroplast, (2) cytoplasm, (3) cytoskeleton, 
(4) endoplasmic reticulum, (5) extracall, (6) Golgi apparatus, (7) lysosome, 
(8) mitochondria, (9) nucleus, (10) peroxisome, (11) plasma membrane, and (12) 
vacuole, which have covered almost all the organelles and subcellular 
compartments in an animal or plant cell. The examination for the 
self-consistency and the jackknife test of the SVMs method was tested for the 
three sets: 2022 proteins, 2161 proteins, and 2319 proteins. As a result, the 
correct rate of self-consistency and jackknife test reaches 91 and 82 73 rate 
was tested by the three independent testing datasets containing 2240 proteins, 
2513 proteins, and 2591 proteins. The correct prediction rates reach 82, 75, 
and 73 2591 proteins, respectively. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Brown2000Knowledge-based]  
M.&nbsp;P. Brown, W.&nbsp;N. Grundy, D.&nbsp;Lin, N.&nbsp;Cristianini, 
C.&nbsp;W. Sugnet, T.&nbsp;S. Furey, M.&nbsp;Ares, and D.&nbsp;Haussler. 
Knowledge-based analysis of microarray gene expression data by using support 
vector machines.<em>Proc. Natl. Acad. Sci. USA</em>, 97(1):262-7, Jan 2000. 
[&nbsp;bib&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  We introduce a method of functionally classifying genes by using 
gene expression data from DNA microarray hybridization experiments. The method 
is based on the theory of support vector machines (SVMs). SVMs are considered a 
supervised computer learning method because they exploit prior knowledge of 
gene function to identify unknown genes of similar function from expression 
data. SVMs avoid several problems associated with unsupervised clustering 
methods, such as hierarchical clustering and self-organizing maps. SVMs have 
many mathematical features that make them attractive for gene expression 
analysis, including their flexibility in choosing a similarity function, 
sparseness of solution when dealing with large data sets, the ability to handle 
large feature spaces, and the ability to identify outliers. We test several 
SVMs that use different similarity metrics, as well as some other supervised 
learning methods, and find that the SVMs best identify sets of genes with a 
common function using expression data. Finally, we use SVMs to predict 
functional roles for uncharacterized yeast ORFs based on their expression data. 
</blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm microarray </blockquote> <br>
 [
Ben-Dor2000Tissue]  A.&nbsp;Ben-Dor, L.&nbsp;Bruhn, N.&nbsp;Friedman, 
I.&nbsp;Nachman, M.&nbsp;Schummer, and Z.&nbsp;Yakhini. Tissue classification 
with gene expression profiles.<em>J. Comput. Biol.</em>, 7(3-4):559-583, 2000. 
[&nbsp;bib&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  Constantly improving gene expression profiling technologies are 
expected to provide understanding and insight into cancer-related cellular 
processes. Gene expression data is also expected to significantly aid in the 
development of efficient cancer diagnosis and classification platforms. In this 
work we examine three sets of gene expression data measured across sets of 
tumor(s) and normal clinical samples: The first set consists of 2,000 genes, 
measured in 62 epithelial colon samples (Alon et al., 1999). The second 
consists of approximately equal to 100,000 clones, measured in 32 ovarian 
samples (unpublished extension of data set described in Schummer et al. 
(1999)). The third set consists of approximately equal to 7,100 genes, measured 
in 72 bone marrow and peripheral blood samples (Golub et al, 1999). We examine 
the use of scoring methods, measuring separation of tissue type (e.g., tumors 
from normals) using individual gene expression levels. These are then coupled 
with high-dimensional classification methods to assess the classification power 
of complete expression profiles. We present results of performing leave-one-out 
cross validation (LOOCV) experiments on the three data sets, employing nearest 
neighbor classifier, SVM (Cortes and Vapnik, 1995), AdaBoost (Freund and 
Schapire, 1997) and a novel clustering-based classification technique. As tumor 
samples can differ from normal samples in their cell-type composition, we also 
perform LOOCV experiments using appropriately modified sets of genes, 
attempting to eliminate the resulting bias. We demonstrate success rate of at 
least 90 in tumor versus normal classification, using sets of selected genes, 
with, as well as without, cellular-contamination-related members. These results 
are insensitive to the exact selection mechanism, over a certain range. 
</blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm microarray </blockquote> <br>
 [
Yeang2001Molecular]  C.H. Yeang, S.&nbsp;Ramaswamy, P.&nbsp;Tamayo, 
S.&nbsp;Mukherjee, R.M. Rifkin, M.&nbsp;Angelo, M.&nbsp;Reich, E.&nbsp;Lander, 
J.&nbsp;Mesirov, and T.&nbsp;Golub. Molecular classification of multiple tumor 
types.<em>Bioinformatics</em>, 17(Suppl. 1):S316-S322, 2001. [&nbsp;bib&nbsp;| 
http&nbsp;| .pdf&nbsp;] 
<blockquote>  Using gene expression data to classify tumor types is a very 
promising tool in cancer diagnosis. Previous works show several pairs of tumor 
types can be successfully distinguished by their gene expression patterns 
(Golub et al. 1999, Ben-Dor et al. 2000, Alizadeh et al. 2000). However, the 
simultaneous classification across a heterogeneous set of tumor types has not 
been well studied yet. We obtained 190 samples from 14 tumor classes and 
generated a combined expression dataset containing 16063 genes for each of 
those samples. We performed multi-class classification by combining the outputs 
of binary classifiers. Three binary classifiers (k-nearest neighbors, weighted 
voting, and support vector machines) were applied in conjunction with three 
combination scenarios (one-vs-all, all-pairs, hierarchical partitioning). We 
achieved the best cross validation error rate of 18.75 support vector machine 
algorithm. The results demonstrate the feasibility of performing clinically 
useful classification from samples of multiple tumor types. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Xiong2001Biomarker]  
M.&nbsp;Xiong, X.&nbsp;Fang, and J.&nbsp;Zhao. Biomarker Identification by 
Feature Wrappers.<em>Genome Res.</em>, 11(11):1878-1887, 2001. [&nbsp;bib&nbsp;|
http&nbsp;| .pdf&nbsp;] 
<blockquote>  Gene expression studies bridge the gap between DNA information 
and trait information by dissecting biochemical pathways into intermediate 
components between genotype and phenotype. These studies open new avenues for 
identifying complex disease genes and biomarkers for disease diagnosis and for 
assessing drug efficacy and toxicity. However, the majority of analytical 
methods applied to gene expression data are not efficient for biomarker 
identification and disease diagnosis. In this paper, we propose a general 
framework to incorporate feature (gene) selection into pattern recognition in 
the process to identify biomarkers. Using this framework, we develop three 
feature wrappers that search through the space of feature subsets using the 
classification error as measure of goodness for a particular feature subset 
being &quot;wrapped around&quot;: linear discriminant analysis, logistic 
regression, and support vector machines. To effectively carry out this 
computationally intensive search process, we employ sequential forward search 
and sequential forward floating search algorithms. To evaluate the performance 
of feature selection for biomarker identification we have applied the proposed 
methods to three data sets. The preliminary results demonstrate that very high 
classification accuracy can be attained by identified composite classifiers 
with several biomarkers. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Su2001Molecular]  
A.&nbsp;I. Su, J.&nbsp;B. Welsh, L.&nbsp;M. Sapinoso, S.&nbsp;G. Kern, 
P.&nbsp;Dimitrov, H.&nbsp;Lapp, P.&nbsp;G. Schultz, S.&nbsp;M. Powell, 
C.&nbsp;A. Moskaluk, H.&nbsp;F.Jr. Frierson, and G.&nbsp;M. Hampton. Molecular 
Classification of Human Carcinomas by Use of Gene Expression Signatures.<em>
Cancer Res.</em>, 61(20):7388-7393, 2001. [&nbsp;bib&nbsp;| http&nbsp;| .html
&nbsp;]
<blockquote>  Classification of human tumors according to their primary 
anatomical site of origin is fundamental for the optimal treatment of patients 
with cancer. Here we describe the use of large-scale RNA profiling and 
supervised machine learning algorithms to construct a first-generation 
molecular classification scheme for carcinomas of the prostate, breast, lung, 
ovary, colorectum, kidney, liver, pancreas, bladder/ureter, and 
gastroesophagus, which collectively account for [&nbsp;]70 cancer-related 
deaths in the United States. The classification scheme was based on identifying 
gene subsets whose expression typifies each cancer class, and we quantified the 
extent to which these genes are characteristic of a specific tumor type by 
accurately and confidently predicting the anatomical site of tumor origin for 
90 including 9 of 12 metastatic lesions. The predictor gene subsets include 
those whose expression is typical of specific types of normal epithelial 
differentiation, as well as other genes whose expression is elevated in cancer. 
This study demonstrates the feasibility of predicting the tissue origin of a 
carcinoma in the context of multiple cancer classes. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm, breastcancer </blockquote> <br>
 [
Ramaswamy2001Multiclass]  S.&nbsp;Ramaswamy, P.&nbsp;Tamayo, R.&nbsp;Rifkin, 
S.&nbsp;Mukherjee, C.H. Yeang, M.&nbsp;Angelo, C.&nbsp;Ladd, M.&nbsp;Reich, 
E.&nbsp;Latulippe, J.P. Mesirov, T.&nbsp;Poggio, W.&nbsp;Gerald, M.&nbsp;Loda, 
E.S. Lander, and T.R. Golub. Multiclass cancer diagnosis using tumor gene 
expression signatures.<em>Proc. Natl. Acad. Sci. USA</em>, 98(26):15149-15154, 
Dec 2001. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  The optimal treatment of patients with cancer depends on 
establishing accurate diagnoses by using a complex combination of clinical and 
histopathological data. In some instances, this task is difficult or impossible 
because of atypical clinical presentation or histopathology. To determine 
whether the diagnosis of multiple common adult malignancies could be achieved 
purely by molecular classification, we subjected 218 tumor samples, spanning 14 
common tumor types, and 90 normal tissue samples to oligonucleotide microarray 
gene expression analysis. The expression levels of 16,063 genes and expressed 
sequence tags were used to evaluate the accuracy of a multiclass classifier 
based on a support vector machine algorithm. Overall classification accuracy 
was 78 Poorly differentiated cancers resulted in low-confidence predictions and 
could not be accurately classified according to their tissue of origin, 
indicating that they are molecularly distinct entities with dramatically 
different gene expression patterns compared with their well differentiated 
counterparts. Taken together, these results demonstrate the feasibility of 
accurate, multiclass molecular cancer classification and suggest a strategy for 
future clinical implementation of molecular cancer diagnostics. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm microarray </blockquote> <br>
 [Pavlidis2001Gene
]  P.&nbsp;Pavlidis, J.&nbsp;Weston, J.&nbsp;Cai, and W.N. Grundy. Gene 
functional classification from heterogeneous data. In<em>Proceedings of the 
Fifth Annual International Conference on Computational Biology</em>, pages 
249-255, 2001. [&nbsp;bib&nbsp;| .pdf&nbsp;| .pdf&nbsp;] 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Pavlidis2001Promoter]  
P.&nbsp;Pavlidis, T.&nbsp;S. Furey, M.&nbsp;Liberto, D.&nbsp;Haussler, and 
W.&nbsp;N. Grundy. Promoter Region-Based Classification of Genes. In<em>Pacific 
Symposium on Biocomputing</em>, pages 139-150, 2001. [&nbsp;bib&nbsp;| .pdf
&nbsp;|.pdf&nbsp;] 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Model2001Feature]  
F.&nbsp;Model, P.&nbsp;Adorjan, A.&nbsp;Olek, and C.&nbsp;Piepenbrock. Feature 
selection for DNA methylation based cancer classification.<em>Bioinformatics
</em>, 17(Supp. 1):S157-S164, 2001. [&nbsp;bib&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  Molecular portraits, such as mRNA expression or DNA methylation 
patterns, have been shown to be strongly correlated with phenotypical 
parameters. These molecular patterns can be revealed routinely on a genomic 
scale. However, class prediction based on these patterns is an under-determined 
problem, due to the extreme high dimensionality of the data compared to the 
usually small number of available samples. This makes a reduction of the data 
dimensionality necessary. Here we demonstrate how phenotypic classes can be 
predicted by combining feature selection and discriminant analysis. By 
comparing several feature selection methods we show that the right dimension 
reduction strategy is of crucial importance for the classification performance. 
The techniques are demonstrated by methylation pattern based discrimination 
between acute lymphoblastic leukemia and acute myeloid leukemia. Contact: 
Fabian.Model@epigenomics.com </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Hua2001Support]  
S.&nbsp;Hua and Z.&nbsp;Sun. Support vector machine approach for protein 
subcellular localization prediction.<em>Bioinformatics</em>, 17(8):721-728, 
2001. [&nbsp;bib&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  Motivation: Subcellular localization is a key functional 
characteristic of proteins. A fully automatic and reliable prediction system 
for protein subcellular localization is needed, especially for the analysis of 
large-scale genome sequences. Results: In this paper, Support Vector Machine 
has been introduced to predict the subcellular localization of proteins from 
their amino acid compositions. The total prediction accuracies reach 91.4 in 
prokaryotic organisms and 79.4 organisms. Predictions by our approach are 
robust to errors in the protein N-terminal sequences. This new approach 
provides superior prediction performance compared with existing algorithms 
based on amino acid composition and can be a complementary method to other 
existing methods based on sorting signals. Availability: A web server 
implementing the prediction method is available at 
http://www.bioinfo.tsinghua.edu.cn/SubLoc/. Contact: 
sunzhr@mail.tsinghua.edu.cn; huasj00@mails.tsinghua.edu.cn Supplementary 
information: Supplementary material is available at 
http://www.bioinfo.tsinghua.edu.cn/SubLoc </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Ding2001Multi-class]  
C.H.Q. Ding and I.&nbsp;Dubchak. Multi-class protein fold recognition using 
support vector machines and neural networks.<em>Bioinformatics</em>, 
17:349-358, 2001. [&nbsp;bib&nbsp;| .pdf&nbsp;| .pdf&nbsp;] 
<blockquote>  Motivation: Protein fold recognition is an important approach to 
structure discovery without relying on sequence similarity. We study this 
approach with new multi-class classification methods and examined many issues 
important for a practical recognition system. Results: Most current 
discriminative methods for protein fold prediction use the one-against-others 
method, which has the well-known ?False Positives? problem. We investigated two 
new methods: the unique one-against-others and the all-against-all methods. 
Both improve prediction accuracy by 14?110 SCOP folds. We used the Support 
Vector Machine (SVM) and the Neural Network (NN) learning methods as base 
classifiers. SVMs converges fast and leads to high accuracy. When scores of 
multiple parameter datasets are combined, majority voting reduces noise and 
increases recognition accuracy. We examined many issues involved with large 
number of classes, including dependencies of prediction accuracy on the number 
of folds and on the number of representatives in a fold. Overall, recognition 
systems achieve 56 accuracy on a protein test dataset, where most of the 
proteins have below 25 information: The protein parameter datasets used in this 
paper are available online (http://www.nersc.gov/&nbsp;cding/protein). 
</blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Chow2001Identifying]  
M.&nbsp;L. Chow, E.&nbsp;J. Moler, and I.&nbsp;S. Mian. Identifying marker 
genes in transcription profiling data using a mixture of feature relevance 
experts.<em>Physiol. Genomics</em>, 5(2):99-111, Mar 2001. [&nbsp;bib&nbsp;| 
http&nbsp;| .pdf&nbsp;] 
<blockquote>  Transcription profiling experiments permit the expression levels 
of many genes to be measured simultaneously. Given profiling data from two 
types of samples, genes that most distinguish the samples (marker genes) are 
good candidates for subsequent in-depth experimental studies and developing 
decision support systems for diagnosis, prognosis, and monitoring. This work 
proposes a mixture of feature relevance experts as a method for identifying 
marker genes and illustrates the idea using published data from samples labeled 
as acute lymphoblastic and myeloid leukemia (ALL, AML). A feature relevance 
expert implements an algorithm that calculates how well a gene distinguishes 
samples, reorders genes according to this relevance measure, and uses a 
supervised learning method [here, support vector machines (SVMs)] to determine 
the generalization performances of different nested gene subsets. The mixture 
of three feature relevance experts examined implement two existing and one 
novel feature relevance measures. For each expert, a gene subset consisting of 
the top 50 genes distinguished ALL from AML samples as completely as all 7,070 
genes. The 125 genes at the union of the top 50s are plausible markers for a 
prototype decision support system. Chromosomal aberration and other data 
support the prediction that the three genes at the intersection of the top 50s, 
cystatin C, azurocidin, and adipsin, are good targets for investigating the 
basic biology of ALL/AML. The same data were employed to identify markers that 
distinguish samples based on their labels of T cell/B cell, peripheral 
blood/bone marrow, and male/female. Selenoprotein W may discriminate T cells 
from B cells. Results from analysis of transcription profiling data from 
tumor/nontumor colon adenocarcinoma samples support the general utility of the 
aforementioned approach. Theoretical issues such as choosing SVM kernels and 
their parameters, training and evaluating feature relevance experts, and the 
impact of potentially mislabeled samples on marker identification (feature 
selection) are discussed. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Carter2001computational]  
R.&nbsp;J. Carter, I.&nbsp;Dubchak, and S.&nbsp;R. Holbrook. A computational 
approach to identify genes for functional RNAs in genomic sequences.<em>Nucl. 
Acids Res.</em>, 29(19):3928-3938, 2001. [&nbsp;bib&nbsp;| http&nbsp;| .pdf
&nbsp;]
<blockquote>  Currently there is no successful computational approach for 
identification of genes encoding novel functional RNAs (fRNAs) in genomic 
sequences. We have developed a machine learning approach using neural networks 
and support vector machines to extract common features among known RNAs for 
prediction of new RNA genes in the unannotated regions of prokaryotic and 
archaeal genomes. The Escherichia coli genome was used for development, but we 
have applied this method to several other bacterial and archaeal genomes. 
Networks based on nucleotide composition were 80-90 for bacteria and 90-99 
achieved a significant improvement in accuracy by combining these predictions 
with those obtained using a second set of parameters consisting of known RNA 
sequence motifs and the calculated free energy of folding. Several known fRNAs 
not included in the training datasets were identified as well as several 
hundred predicted novel RNAs. These studies indicate that there are many 
unidentified RNAs in simple genomes that can be predicted computationally as a 
precursor to experimental study. Public access to our RNA gene predictions and 
an interface for user predictions is available via the web. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Cai2001Support]  Y.-D. 
Cai, X.-J. Liu, X.-B. Xu, and G.-P. Zhou. Support Vector Machines for 
predicting protein structural class.<em>BMC Bioinformatics</em>, 2(3):3, 2001. 
[&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  Background We apply a new machine learning method, the so-called 
Support Vector Machine method, to predict the protein structural class. Support 
Vector Machine method is performed based on the database derived from SCOP, in 
which protein domains are classified based on known structures and the 
evolutionary relationships and the principles that govern their 3-D structure. 
Results High rates of both self-consistency and jackknife tests are obtained. 
The good results indicate that the structural class of a protein is 
considerably correlated with its amino acid composition. Conclusions It is 
expected that the Support Vector Machine method and the elegant 
component-coupled method, also named as the covariant discrimination algorithm, 
if complemented with each other, can provide a powerful computational tool for 
predicting the structural classes of proteins. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Bock2001Predicting]  
J.&nbsp;R. Bock and D.&nbsp;A. Gough. Predicting protein-protein interactions 
from primary structure.<em>Bioinformatics</em>, 17(5):455-460, 2001. [&nbsp;bib
&nbsp;|.pdf&nbsp;| .pdf&nbsp;] 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Beerenwinkel2001Geno2pheno]
 N.&nbsp;Beerenwinkel, B.&nbsp;Schmidt, H.&nbsp;Walter, R.&nbsp;Kaiser, 
T.&nbsp;Lengauer, D.&nbsp;Hoffman, K.&nbsp;Korn, and J.&nbsp;Selbig. 
Geno2pheno: Interpreting Genotypic HIV Drug Resistance Tests.<em>IEEE 
Intelligent Systems</em>, 6(6):35-41, 2001. [&nbsp;bib&nbsp;| DOI&nbsp;| http
&nbsp;|.pdf&nbsp;] 
<blockquote>  Rapid accumulation of resistance mutations in the genome of the 
human immunodeficiency virus (HIV) plays a central role in drug treatment 
failure in infected patients. The authors have developed geno2pheno, an 
intelligent system that uses the information encoded in the viral genomic 
sequence to predict resistance or susceptibility of the virus to 13 
antiretroviral agents. To predict phenotypic drug resistance from genotype, 
they applied two machine learning techniques: decision trees and linear support 
vector machines. These techniques performed learning on more than 400 
genotype-phenotype pairs for each drug. The authors compared the generalization 
performance of the two families of models in leave-one-out experiments. Except 
for three drugs, all error estimates ranged between 7.25 and 15.5 percent. 
Support vector machines performed slightly better for most drugs, but knowledge 
extraction was easier for decision trees. Geno2pheno is freely available at 
http://cartan.gmd.de/geno2pheno.html. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Bazzani2001SVM]  
A.&nbsp;Bazzani, A.&nbsp;Bevilacqua, D.&nbsp;Bollini, R.&nbsp;Brancaccio, 
R.&nbsp;Campanini, N.&nbsp;Lanconelli, A.&nbsp;Riccardi, and D.&nbsp;Romani. An 
SVM classifier to separate false signals from microcalcifications in digital 
mammograms.<em>Phys Med Biol</em>, 46(6):1651-63, Jun 2001. [&nbsp;bib&nbsp;| 
DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  In this paper we investigate the feasibility of using an SVM 
(support vector machine) classifier in our automatic system for the detection 
of clustered microcalcifications in digital mammograms. SVM is a technique for 
pattern recognition which relies on the statistical learning theory. It 
minimizes a function of two terms: the number of misclassified vectors of the 
training set and a term regarding the generalization classifier capability. We 
compare the SVM classifier with an MLP (multi-layer perceptron) in the 
false-positive reduction phase of our detection scheme: a detected signal is 
considered either microcalcification or false signal, according to the value of 
a set of its features. The SVM classifier gets slightly better results than the 
MLP one (Az value of 0.963 against 0.958) in the presence of a high number of 
training data; the improvement becomes much more evident (Az value of 0.952 
against 0.918) in training sets of reduced size. Finally, the setting of the 
SVM classifier is much easier than the MLP one. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm image </blockquote> <br>
 [Hua2001Novel]  
S.&nbsp;Hua and Z.&nbsp;Sun. A Novel Method of Protein Secondary Structure 
Prediction with High Segment Overlap Measure: Support Vector Machine Approach.
<em>J. Mol. Biol.</em>, 308(2):397-407, April 2001. [&nbsp;bib&nbsp;| DOI&nbsp;|
.pdf&nbsp;] 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Logan2001Study]  
B.&nbsp;Logan, P.&nbsp;Moreno, B.&nbsp;Suzek, Z.&nbsp;Weng, and S.&nbsp;Kasif. 
A Study of Remote Homology Detection. Technical Report CRL 2001/05, Compaq 
Cambridge Research laboratory, June 2001. [&nbsp;bib&nbsp;| .pdf&nbsp;] 
<blockquote>  Functional annotation of newly sequenced genomes is an important 
challenge for computational biology systems. While much progress has been made 
towards scalingup experimental methods for functional assignment to putative 
genes, most current genomic annotation systems rely on computational solutions 
for homology modeling via sequence or structural similarity. We present a new 
method for remote homology detection that relies on combining probabilistic 
modeling and supervised learning in high-dimensional features spaces. Our 
system uses a transformation that converts protein domains to fixed-dimension 
representative feature vectors, where each feature records the sensitivity of 
each protein domain to a previously learned set of ?protein motifs? or 
?blocks?. Subsequently, the system utilizes Support Vector Machine (SVM) 
classifiers to learn the boundaries between structural protein classes. Our 
experiments suggest that this technique performs well relative to several other 
remote homology methods for the majority of protein domains in SCOP 1.37 PDB90. 
</blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Burbidge2001Drug]  
R.&nbsp;Burbidge, M.&nbsp;Trotter, B.&nbsp;Buxton, and S.&nbsp;Holden. Drug 
design by machine learning: support vector machines for pharmaceutical data 
analysis.<em>Comput. Chem.</em>, 26(1):4-15, December 2001. [&nbsp;bib&nbsp;| 
.pdf&nbsp;| .pdf&nbsp;] 
<blockquote>  Keywords: biosvm chemoinformatics </blockquote> <br>
 [
Zavaljevski2002Support]  N.&nbsp;Zavaljevski, F.J. Stevens, and 
J.&nbsp;Reifman. Support vector machines with selective kernel scaling for 
protein classification and identification of key amino acid positions.<em>
Bioinformatics</em>, 18(5):689-696, 2002. [&nbsp;bib&nbsp;| http&nbsp;| .pdf
&nbsp;]
<blockquote>  Motivation: Data that characterize primary and tertiary 
structures of proteins are now accumulating at a rapid and accelerating rate 
and require automated computational tools to extract critical information 
relating amino acid changes with the spectrum of functionally attributes 
exhibited by a protein. We propose that immunoglobulin-type beta-domains, which 
are found in approximate 400 functionally distinct forms in humans alone, 
provide the immense genetic variation within limited conformational changes 
that might facilitate the development of new computational tools. As an initial 
step, we describe here an approach based on Support Vector Machine (SVM) 
technology to identify amino acid variations that contribute to the functional 
attribute of pathological self-assembly by some human antibody light chains 
produced during plasma cell diseases. Results: We demonstrate that SVMs with 
selective kernel scaling are an effective tool in discriminating between benign 
and pathologic human immunoglobulin light chains. Initial results compare 
favorably against manual classification performed by experts and indicate the 
capability of SVMs to capture the underlying structure of the data. The data 
set consists of 70 proteins of human antibody 1 light chains, each represented 
by aligned sequences of 120 amino acids. We perform feature selection based on 
a first-order adaptive scaling algorithm, which confirms the importance of 
changes in certain amino acid positions and identifies other positions that are 
key in the characterization of protein function. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Yuan2002Prediction]  
Z.&nbsp;Yuan, K.&nbsp;Burrage, and J.S. Mattick. Prediction of protein solvent 
accessibility using support vector machines.<em>Proteins</em>, 48(3):566-570, 
2002. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  A Support Vector Machine learning system has been trained to 
predict protein solvent accessibility from the primary structure. Different 
kernel functions and sliding window sizes have been explored to find how they 
affect the prediction performance. Using a cut-off threshold of 15 of exposed 
and buried residues), this method was able to achieve a prediction accuracy of 
70.1 for multiple alignment sequence input, respectively. The prediction of 
three and more states of solvent accessibility was also studied and compared 
with other methods. The prediction accuracies are better than, or comparable 
to, those obtained by other methods such as neural networks, Bayesian 
classification, multiple linear regression, and information theory. In 
addition, our results further suggest that this system may be combined with 
other prediction methods to achieve more reliable results, and that the Support 
Vector Machine method is a very useful tool for biological sequence analysis. 
</blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Warmuth2002Active]  
M.&nbsp;K. Warmuth, G.&nbsp;R&auml;tsch, M.&nbsp;Mathieson, L.&nbsp;Liao, and 
C.&nbsp;Lemmen. Active learning in the drug discovery process. In T.G. 
Dietterich, S.&nbsp;Becker, and Z.&nbsp;Ghahramani, editors,<em>Adv. Neural 
Inform. Process. Syst.</em>, volume&nbsp;14, pages 1449-1456. MIT Press, 2002. 
[&nbsp;bib&nbsp;] 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Vert2002Graph-driven]  
J.-P. Vert and M.&nbsp;Kanehisa. Graph-driven features extraction from 
microarray data. Technical Report 0206055, Arxiv physics, 2002. [&nbsp;bib
&nbsp;]
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Vert2002tree]  J.-P. Vert. 
A tree kernel to analyze phylogenetic profiles.<em>Bioinformatics</em>, 
18:S276-S284, 2002. [&nbsp;bib&nbsp;| .html&nbsp;| .pdf&nbsp;] 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Vert2002Support]  J.-P. 
Vert. Support vector machine prediction of signal peptide cleavage site using a 
new class of kernels for strings. In R.&nbsp;B. Altman, A.&nbsp;K. Dunker, 
L.&nbsp;Hunter, K.&nbsp;Lauerdale, and T.&nbsp;E. Klein, editors,<em>
Proceedings of the Pacific Symposium on Biocomputing 2002</em>, pages 649-660. 
World Scientific, 2002. [&nbsp;bib&nbsp;| .pdf&nbsp;| .pdf&nbsp;] 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Valentini2002Gene]  
G.&nbsp;Valentini. Gene expression data analysis of human lymphoma using 
support vector machines and output coding ensembles.<em>Artif. Intell. Med.</em>
, 26(3):281-304, Nov 2002. [&nbsp;bib&nbsp;| DOI&nbsp;| .pdf&nbsp;] 
<blockquote>  The large amount of data generated by DNA microarrays was 
originally analysed using unsupervised methods, such as clustering or 
self-organizing maps. Recently supervised methods such as decision trees, 
dot-product support vector machines (SVM) and multi-layer perceptrons (MLP) 
have been applied in order to classify normal and tumoural tissues. We propose 
methods based on non-linear SVM with polynomial and Gaussian kernels, and 
output coding (OC) ensembles of learning machines to separate normal from 
malignant tissues, to classify different types of lymphoma and to analyse the 
role of sets of coordinately expressed genes in carcinogenic processes of 
lymphoid tissues. Using gene expression data from &quot;Lymphochip&quot;, a 
specialised DNA microarray developed at Stanford University School of Medicine, 
we show that SVM can correctly separate normal from tumoural tissues, and OC 
ensembles can be successfully used to classify different types of lymphoma. 
Moreover, we identify a group of coordinately expressed genes related to the 
separation of two distinct subgroups inside diffuse large B-cell lymphoma 
(DLBCL), validating a previous Alizadeh's hypothesis about the existence of two 
distinct diseases inside DLBCL. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Tsuda2002Marginalized]  
K.&nbsp;Tsuda, T.&nbsp;Kin, and K.&nbsp;Asai. Marginalized Kernels for 
Biological Sequences.<em>Bioinformatics</em>, 18:S268-S275, 2002. [&nbsp;bib
&nbsp;|.pdf&nbsp;] 
<blockquote>  Motivation: Kernel methods such as support vector machines 
require a kernel function between objects to be defined a priori. Several works 
have been done to derive kernels from probability distributions, e.g., the 
Fisher kernel. However, a general methodology to design a kernel is not fully 
developed. Results: We propose a reasonable way of designing a kernel when 
objects are generated from latent variable models (e.g., HMM). First of all, a 
joint kernel is designed for complete data which include both visible and 
hidden variables. Then a marginalized kernel for visible data is obtained by 
taking the expectation with respect to hidden variables. We will show that the 
Fisher kernel is a special case of marginalized kernels, which gives another 
viewpoint to the Fisher kernel theory. Although our approach can be applied to 
any object, we particularly derive several marginalized kernels useful for 
biological sequences (e.g., DNA and proteins). The effectiveness of 
marginalized kernels is illustrated in the task of classifying bacterial gyrase 
subunit B (gyrB) amino acid sequences. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Tsuda2002new]  
K.&nbsp;Tsuda, M.&nbsp;Kawanabe, G.&nbsp;R&auml;tsch, S.&nbsp;Sonnenburg, and 
K.-R. M&uuml;ller. A new discriminative kernel from probabilistic models.<em>
Neural Computation</em>, 14(10):2397-2414, 2002. [&nbsp;bib&nbsp;| DOI&nbsp;| 
http&nbsp;| .pdf&nbsp;] 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Stapley2002Predicting]  
B.J. Stapley, L.A. Kelley, and M.J. Sternberg. Predicting the sub-cellular 
location of proteins from text using support vector machines. In Russ&nbsp;B. 
Altman, A.&nbsp;Keith Dunker, Lawrence Hunter, Kevin Lauerdale, and 
Teri&nbsp;E. Klein, editors,<em>Proceedings of the Pacific Symposium on 
Biocomputing 2002</em>, pages 374-385. World Scientific, 2002. [&nbsp;bib&nbsp;|
.pdf&nbsp;| .pdf&nbsp;] 
<blockquote>  We present an automatic method to classify the sub-cellular 
location of proteins based on the text of relevant medline abstracts. For each 
protein, a vector of terms is generated from medline abstracts in which the 
protein/gene's name or synonym occurs. A Support Vector Machine (SVM) is used 
to automatically partition the term space and to thus discriminate the textual 
features that define sub-cellular location. The method is benchmarked on a set 
of proteins of known sub-cellular location from S. cerevisiae. No prior 
knowledge of the problem domain nor any natural language processing is used at 
any stage. The method out-performs support vector machines trained on amino 
acid composition and has comparable performance to rule-based text classifiers. 
Combining text with protein amino-acid composition improves recall for some 
sub-cellular locations. We discuss the generality of the method and its 
potential application to a variety of biological classification problems. 
</blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Sonnenburg2002New]  
S.&nbsp;Sonnenburg, G.&nbsp;R&auml;tsch, A.&nbsp;Jagota, and K.-R. M&uuml;ller. 
New methods for splice-site recognition. In JR. Dorronsoro, editor,<em>Proc. 
International conference on artificial Neural Networks ? ICANN?02</em>, number 
2415 in LNCS, pages 329-336. Springer Berlin, 2002. [&nbsp;bib&nbsp;| .pdf
&nbsp;]
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Shipp2002Diffuse]  
M.&nbsp;A. Shipp, K.&nbsp;N. Ross, P.&nbsp;Tamayo, A.&nbsp;P. Weng, J.&nbsp;L. 
Kutok, R.&nbsp;C.&nbsp;T. Aguiar, M.&nbsp;Gaasenbeek, M.&nbsp;Angelo, 
M.&nbsp;Reich, G.&nbsp;A. Pinkus, T.&nbsp;S. Ray, M.&nbsp;A. Koval, K.&nbsp;W. 
Last, A.&nbsp;Norton, T.&nbsp;A. Lister, J.&nbsp;Mesirov, D.&nbsp;S. Neuberg, 
E.&nbsp;S. Lander, J.&nbsp;C. Aster, and T.&nbsp;R. Golub. Diffuse large B-cell 
lymphoma outcome prediction by gene-expression profiling and supervised machine 
learning.<em>Nat. Med.</em>, 8(1):68-74, 2002. [&nbsp;bib&nbsp;| DOI&nbsp;| .pdf
&nbsp;]
<blockquote>  Diffuse large B-cell lymphoma (DLBCL), the most common lymphoid 
malignancy in adults, is curable in less than 50 models based on pre-treatment 
characteristics, such as the International Prognostic Index (IPI), are 
currently used to predict outcome in DLBCL. However, clinical outcome models 
identify neither the molecular basis of clinical heterogeneity, nor specific 
therapeutic targets. We analyzed the expression of 6,817 genes in diagnostic 
tumor specimens from DLBCL patients who received cyclophosphamide, adriamycin, 
vincristine and prednisone (CHOP)-based chemotherapy, and applied a supervised 
learning prediction method to identify cured versus fatal or refractory 
disease. The algorithm classified two categories of patients with very 
different five-year overall survival rates (70 within specific IPI risk 
categories who were likely to be cured or to die of their disease. Genes 
implicated in DLBCL outcome included some that regulate responses to 
B-cell?receptor signaling, critical serine/threonine phosphorylation pathways 
and apoptosis. Our data indicate that supervised learning classification 
techniques can predict outcome in DLBCL and identify rational targets for 
intervention. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Seeger2002Covariance]  
M.&nbsp;Seeger. Covariance Kernels from Bayesian Generative Models. In<em>Adv. 
Neural Inform. Process. Syst.</em>, volume&nbsp;14, pages 905-912, 2002. [&nbsp;
bib&nbsp;| www:&nbsp;] 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Pavlidis2002Learning]  
P.&nbsp;Pavlidis, J.&nbsp;Weston, J.&nbsp;Cai, and W.S. Noble. Learning gene 
functional classifications from multiple data types.<em>J. Comput. Biol.</em>, 
9(2):401-411, 2002. [&nbsp;bib&nbsp;| DOI&nbsp;| .pdf&nbsp;] 
<blockquote>  In our attempts to understand cellular function at the molecular 
level, we must be able to synthesize information from disparate types of 
genomic data. We consider the problem of inferring gene functional 
classifications from a heterogeneous data set consisting of DNA microarray 
expression measurements and phylogenetic profiles from whole-genome sequence 
comparisons. We demonstrate the application of the support vector machine (SVM) 
learning algorithm to this functional inference task. Our results suggest the 
importance of exploiting prior information about the heterogeneity of the data. 
In particular, we propose an SVM kernel function that is explicitly 
heterogeneous. In addition, we describe feature scaling methods for further 
exploiting prior knowledge of heterogeneity by giving each data type different 
weights. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Patterson2002Pre-mRNA]  
D.J. Patterson, K.&nbsp;Yasuhara, and W.L. Ruzzo. Pre-mRNA secondary structure 
prediction aids splice site prediction. In Russ&nbsp;B. Altman, A.&nbsp;Keith 
Dunker, Lawrence Hunter, Kevin Lauerdale, and Teri&nbsp;E. Klein, editors,<em>
Proceedings of the Pacific Symposium on Biocomputing 2002</em>, pages 223-234. 
World Scientific, 2002. [&nbsp;bib&nbsp;| .pdf&nbsp;| .pdf&nbsp;] 
<blockquote>  Accurate splice site prediction is a critical component of any 
computational approach to gene prediction in higher organisms. Existing 
approaches generally use sequence-based models that capture local dependencies 
among nucleotides in a small window around the splice site. We present evidence 
that computationally predicted secondary structure of moderate length pre-mRNA 
subsequencies contains information that can be exploited to improve acceptor 
splice site prediction beyond that possible with conventional sequence-based 
approaches. Both decision tree and support vector machine classifiers, using 
folding energy and structure metrics characterizing helix formation near the 
splice site, achieve a 5-10 a human data set. Based on our data, we hypothesize 
that acceptors preferentially exhibit short helices at the splice site. 
</blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Myasnikova2002Support]  
E.&nbsp;Myasnikova, A.&nbsp;Samsonova, M.&nbsp;Samsonova, and J.&nbsp;Reinitz. 
Support vector regression applied to the determination of the developmental age 
of a Drosophila embryo from its segmentation gene expression patterns.<em>
Bioinformatics</em>, 18(Suppl. 1):S87-S95, 2002. [&nbsp;bib&nbsp;| http&nbsp;| 
.pdf&nbsp;] 
<blockquote>  Motivation: In this paper we address the problem of the 
determination of developmental age of an embryo from its segmentation gene 
expression patterns in Drosophila. Results: By applying support vector 
regression we have developed a fast method for automated staging of an embryo 
on the basis of its gene expression pattern. Support vector regression is a 
statistical method for creating regression functions of arbitrary type from a 
set of training data. The training set is composed of embryos for which the 
precise developmental age was determined by measuring the degree of membrane 
invagination. Testing the quality of regression on the training set showed good 
prediction accuracy. The optimal regression function was then used for the 
prediction of the gene expression based age of embryos in which the precise age 
has not been measured by membrane morphology. Moreover, we show that the same 
accuracy of prediction can be achieved when the dimensionality of the feature 
vector was reduced by applying factor analysis. The data reduction allowed us 
to avoid over-fitting and to increase the efficiency of the algorithm. 
Availability: This software may be obtained from the authors. Contact: 
samson@fn.csa.ru Keywords: gene expression patterns; development; embryo 
staging; support vector regression; segmentation genes; Drosophila. 
</blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Lodhi2002Text]  
H.&nbsp;Lodhi, C.&nbsp;Saunders, J.&nbsp;Shawe-Taylor, N.&nbsp;Cristianini, and 
C.je n'ai pas vraiment d'&eacute;l&eacute;ments de&nbsp;r&eacute;ponse. 
Watkins. Text classification using string kernels.<em>J. Mach. Learn. Res.</em>
, 2:419-444, 2002. [&nbsp;bib&nbsp;| .html&nbsp;| .pdf&nbsp;] 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Lin2002Conserved]  
K.&nbsp;Lin, Y.&nbsp;Kuang, J.&nbsp;S. Joseph, and P.&nbsp;R. Kolatkar. 
Conserved codon composition of ribosomal protein coding genes in Escherichia 
coli, Mycobacterium tuberculosis and Saccharomyces cerevisiae: lessons from 
supervised machine learning in functional genomics.<em>Nucl. Acids Res.</em>, 
30(11):2599-2607, 2002. [&nbsp;bib&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  Genomics projects have resulted in a flood of sequence data. 
Functional annotation currently relies almost exclusively on inter-species 
sequence comparison and is restricted in cases of limited data from related 
species and widely divergent sequences with no known homologs. Here, we 
demonstrate that codon composition, a fusion of codon usage bias and amino acid 
composition signals, can accurately discriminate, in the absence of sequence 
homology information, cytoplasmic ribosomal protein genes from all other genes 
of known function in Saccharomyces cerevisiae, Escherichia coli and 
Mycobacterium tuberculosis using an implementation of support vector machines, 
SVMlight. Analysis of these codon composition signals is instructive in 
determining features that confer individuality to ribosomal protein genes. Each 
of the sets of positively charged, negatively charged and small hydrophobic 
residues, as well as codon bias, contribute to their distinctive codon 
composition profile. The representation of all these signals is sensitively 
detected, combined and augmented by the SVMs to perform an accurate 
classification. Of special mention is an obvious outlier, yeast gene RPL22B, 
highly homologous to RPL22A but employing very different codon usage, perhaps 
indicating a non-ribosomal function. Finally, we propose that codon composition 
be used in combination with other attributes in gene/protein classification by 
supervised machine learning algorithms. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Liao2002Combining]  
L.&nbsp;Liao and W.&nbsp;S. Noble. Combining pairwise sequence similarity and 
support vector machines for remote protein homology detection. In<em>
Proceedings of the Sixth International Conference on Computational Molecular 
Biology</em>, 2002. [&nbsp;bib&nbsp;| .html&nbsp;| .pdf&nbsp;] 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Leslie2002spectrum]  
C.&nbsp;Leslie, E.&nbsp;Eskin, and W.S. Noble. The spectrum kernel: a string 
kernel for SVM protein classification. In Russ&nbsp;B. Altman, A.&nbsp;Keith 
Dunker, Lawrence Hunter, Kevin Lauerdale, and Teri&nbsp;E. Klein, editors,<em>
Proceedings of the Pacific Symposium on Biocomputing 2002</em>, pages 564-575, 
Singapore, 2002. World Scientific. [&nbsp;bib&nbsp;| .pdf&nbsp;] 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Kramer2002Fragment]  
S.&nbsp;Kramer, E.&nbsp;Frank, and C.&nbsp;Helma. Fragment generation and 
support vector machines for inducing SARs.<em>SAR QSAR Environ Res</em>, 
13(5):509-23, Jul 2002. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;] 
<blockquote>  We present a new approach to the induction of SARs based on the 
generation of structural fragments and support vector machines (SVMs). It is 
tailored for bio-chemical databases, where the examples are two-dimensional 
descriptions of chemical compounds. The fragment generator finds all fragments 
(i.e. linearly connected atoms) that satisfy user-specified constraints 
regarding their frequency and generality. In this paper, we are querying for 
fragments within a minimum and a maximum frequency in the dataset. After 
fragment generation, we propose to apply SVMs to the problem of inducing SARs 
from these fragments. We conjecture that the SVMs are particularly useful in 
this context, as they can deal with a large number of features. Experiments in 
the domains of carcinogenicity and mutagenicity prediction show that the 
minimum and the maximum frequency queries for fragments can be answered within 
a reasonable time, and that the predictive accuracy obtained using these 
fragments is satisfactory. However, further experiments will have to confirm 
that this is a viable approach to inducing SARs. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Kondor2002Diffusion]  
R.&nbsp;I. Kondor and J.&nbsp;Lafferty. Diffusion Kernels on Graphs and Other 
Discrete Input. In<em>ICML 2002</em>, 2002. [&nbsp;bib&nbsp;| .ps.gz&nbsp;| 
.ps.gz&nbsp;] 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Kin2002Marginalized]  
T.&nbsp;Kin, K.&nbsp;Tsuda, and K.&nbsp;Asai. Marginalized kernels for RNA 
sequence data analysis. In R.H. Lathtop, K.&nbsp;Nakai, S.&nbsp;Miyano, 
T.&nbsp;Takagi, and M.&nbsp;Kanehisa, editors,<em>Genome Informatics 2002</em>, 
pages 112-122. Universal Academic Press, 2002. [&nbsp;bib&nbsp;| .html&nbsp;| 
.pdf&nbsp;] 
<blockquote>  We present novel kernels that measure similarity of two RNA 
sequences, taking account of their secondary structures. Two types of kernels 
are presented. One is for RNA sequences with known secondary structures, the 
other for those without known secondary structures. The latter employs 
stochastic context-free grammar (SCFG) for estimating the secondary structure. 
We call the latter the<i> marginalized count kernel</i> (MCK). We show 
computational experiments for MCK using 74 sets of human tRNA sequence data: 
(i) kernel principal component analysis (PCA) for visualizing tRNA 
similarities, (ii) supervised classification with support vector machines 
(SVMs). Both types of experiment show promising results for MCKs. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Karchin2002Classifying]  
R.&nbsp;Karchin, K.&nbsp;Karplus, and D.&nbsp;Haussler. Classifying G-protein 
coupled receptors with support vector machines.<em>Bioinformatics</em>, 
18:147-159, 2002. [&nbsp;bib&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  Motivation: The enormous amount of protein sequence data 
uncovered by genome research has increased the demand for computer software 
that can automate the recognition of new proteins. We discuss the relative 
merits of various automated methods for recognizing G-Protein Coupled Receptors 
(GPCRs), a superfamily of cell membrane proteins. GPCRs are found in a wide 
range of organisms and are central to a cellular signalling network that 
regulates many basic physiological processes. They are the focus of a 
significant amount of current pharmaceutical research because they play a key 
role in many diseases. However, their tertiary structures remain largely 
unsolved. The methods described in this paper use only primary sequence 
information to make their predictions. We compare a simple nearest neighbor 
approach (BLAST), methods based on multiple alignments generated by a 
statistical profile Hidden Markov Model (HMM), and methods, including Support 
Vector Machines (SVMs), that transform protein sequences into fixed-length 
feature vectors. Results: The last is the most computationally expensive 
method, but our experiments show that, for those interested in 
annotation-quality classification, the results are worth the effort. In 
two-fold cross-validation experiments testing recognition of GPCR subfamilies 
that bind a specific ligand (such as a histamine molecule), the errors per 
sequence at the Minimum Error Point (MEP) were 13.7 SVMs, 17.1 SVM 
classification, 25.5 and 49 Kernel Nearest Neighbor (kernNN). The percentage of 
true positives recognized before the first false positive was 65 both SVM 
methods, 13 for kernNN. Availability: We have set up a web server for GPCR 
subfamily classification based on hierarchical multi-class SVMs at 
http://www.soe.ucsc.edu/research/compbio/gpcr-subclass. By scanning predicted 
peptides found in the human genome with the SVMtree server, we have identified 
a large number of genes that encode GPCRs. A list of our predictions for human 
GPCRs is available at 
http://www.soe.ucsc.edu/research/compbio/gpcr&Acirc;&middot;hg/class&Acirc;&middot;results. 
We also provide suggested subfamily classification for 18 sequences previously 
identified as unclassified Class A (rhodopsin-like) GPCRs in GPCRDB (Horn et 
al. , Nucleic Acids Res. , 26, 277?281, 1998), available at 
http://www.soe.ucsc.edu/research/compbio/gpcr/classA&Acirc;&middot;unclassified/
</blockquote> 
<p></p> 
<blockquote>  Keywords: fisher-kernel sequence-classification biosvm 
</blockquote> <br>
 [Guyon2002Gene]  I.&nbsp;Guyon, J.&nbsp;Weston, 
S.&nbsp;Barnhill, and V.&nbsp;Vapnik. Gene selection for cancer classification 
using support vector machines.<em>Mach. Learn.</em>, 46(1/3):389-422, Jan 2002. 
[&nbsp;bib&nbsp;| .pdf&nbsp;| .pdf&nbsp;] 
<blockquote>  DNA micro-arrays now permit scientists to screen thousands of 
genes simultaneously and determine whether those genes are active, hyperactive 
or silent in normal or cancerous tissue. Because these new micro-array devices 
generate bewildering amounts of raw data, new analytical methods must be 
developed to sort out whether cancer tissues have distinctive signatures of 
gene expression over normal tissues or other types of cancer tissues. In this 
paper, we address the problem of selection of a small subset of genes from 
broad patterns of gene expression data, recorded on DNA micro-arrays. Using 
available training examples from cancer and normal patients, we build a 
classifier suitable for genetic diagnosis, as well as drug discovery. Previous 
attempts to address this problem select genes with correlation techniques. We 
propose a new method of gene selection utilizing Support Vector Machine methods 
based on Recursive Feature Elimination (RFE). We demonstrate experimentally 
that the genes selected by our techniques yield better classification 
performance and are biologically relevant to cancer. In contrast with the 
baseline method, our method eliminates gene redundancy automatically and yields 
better and more compact gene subsets. In patients with leukemia our method 
discovered 2 genes that yield zero leave-one-out error, while 64 genes are 
necessary for the baseline method to get the best result (one leave-one-out 
error). In the colon cancer database, using only 4 genes our method is 98% 
accurate, while the baseline method is only 86% accurate. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Guermeur2002Combining]  
Y.&nbsp;Guermeur. Combining Discriminant Models with New Multi-Class SVMs.<em>
Pattern Anal. Appl.</em>, 5(2):168-179, 2002. [&nbsp;bib&nbsp;| DOI&nbsp;| http
&nbsp;|.pdf&nbsp;] 
<blockquote>  The idea of performing model combination, instead of model 
selection, has a long theoretical background in statistics. However, making use 
of theoretical results is ordinarily subject to the satisfaction of strong 
hypotheses (weak error correlation, availability of large training sets, 
possibility to rerun the training procedure an arbitrary number of times, 
etc.). In contrast, the practitioner is frequently faced with the problem of 
combining a given set of pre-trained classifiers, with highly correlated 
errors, using only a small training sample. Overfitting is then the main risk, 
which cannot be overcome but with a strict complexity control of the combiner 
selected. This suggests that SVMs should be well suited for these difficult 
situations. Investigating this idea, we introduce a family of multi-class SVMs 
and assess them as ensemble methods on a real-world problem. This task, protein 
secondary structure prediction, is an open problem in biocomputing for which 
model combination appears to be an issue of central importance. Experimental 
evidence highlights the gain in quality resulting from combining some of the 
most widely used prediction methods with our SVMs rather than with the ensemble 
methods traditionally used in the field. The gain increases when the outputs of 
the combiners are post-processed with a DP algorithm. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Fritz2002Microarray-based] 
 B.&nbsp;Fritz, F.&nbsp;Schubert, G.&nbsp;Wrobel, C.&nbsp;Schwaenen, 
S.&nbsp;Wessendorf, M.&nbsp;Nessling, C.&nbsp;Korz, R.&nbsp;J. Rieker, 
K.&nbsp;Montgomery, R.&nbsp;Kucherlapati, G.&nbsp;Mechtersheimer, R.&nbsp;Eils, 
S.&nbsp;Joos, and P.&nbsp;Lichter. Microarray-based Copy Number and Expression 
Profiling in Dedifferentiated and Pleomorphic Liposarcoma.<em>Cancer Res.</em>, 
62(11):2993-2998, 2002. [&nbsp;bib&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  Sixteen dedifferentiated and pleomorphic liposarcomas were 
analyzed by comparative genomic hybridization (CGH) to genomic microarrays 
(matrix-CGH), cDNA-derived microarrays for expression profiling, and by 
quantitative PCR. Matrix-CGH revealed copy number gains of numerous oncogenes, 
i.e., CCND1, MDM2, GLI, CDK4, MYB, ESR1, and AIB1, several of which correlate 
with a high level of transcripts from the respective gene. In addition, a 
number of genes were found differentially expressed in dedifferentiated and 
pleomorphic liposarcomas. Application of dedicated clustering algorithms 
revealed that both tumor subtypes are clearly separated by the genomic profiles 
but only with a lesser power by the expression profiles. Using a support vector 
machine, a subset of five clones was identified as &quot;class 
discriminators.&quot; Thus, for the distinction of these types of liposarcomas, 
genomic profiling appears to be more advantageous than RNA expression analysis. 
</blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm, cgh </blockquote> <br>
 [Donnes2002Prediction] 
 P.&nbsp;D&ouml;nnes and A.&nbsp;Elofsson. Prediction of MHC class I binding 
peptides, using SVMHC.<em>BMC Bioinformatics</em>, 3(1):25, Sep 2002. [&nbsp;bib
&nbsp;|DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  Background T-cells are key players in regulating a specific 
immune response. Activation of cytotoxic T-cells requires recognition of 
specific peptides bound to Major Histocompatibility Complex (MHC) class I 
molecules. MHC-peptide complexes are potential tools for diagnosis and 
treatment of pathogens and cancer, as well as for the development of peptide 
vaccines. Only one in 100 to 200 potential binders actually binds to a certain 
MHC molecule, therefore a good prediction method for MHC class I binding 
peptides can reduce the number of candidate binders that need to be synthesized 
and tested. Results Here, we present a novel approach, SVMHC, based on support 
vector machines to predict the binding of peptides to MHC class I molecules. 
This method seems to perform slightly better than two profile based methods, 
SYFPEITHI and HLA_BIND. The implementation of SVMHC is quite simple and does 
not involve any manual steps, therefore as more data become available it is 
trivial to provide prediction for more MHC types. SVMHC currently contains 
prediction for 26 MHC class I types from the MHCPEP database or alternatively 6 
MHC class I types from the higher quality SYFPEITHI database. The prediction 
models for these MHC types are implemented in a public web service available at 
http://www.sbc.su.se/svmhc/. Conclusions Prediction of MHC class I binding 
peptides using Support Vector Machines, shows high performance and is easy to 
apply to a large number of MHC class I types. As more peptide data are put into 
MHC databases, SVMHC can easily be updated to give prediction for additional 
MHC class I types. We suggest that the number of binding peptides needed for 
SVM training is at least 20 sequences. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm immunoinformatics </blockquote> <br>
 [
Doniger2002Predicting]  S.&nbsp;Doniger, T.&nbsp;Hofmann, and J.&nbsp;Yeh. 
Predicting CNS permeability of drug molecules: comparison of neural network and 
support vector machine algorithms.<em>J. Comput. Biol.</em>, 9(6):849-864, 
2002. [&nbsp;bib&nbsp;| DOI&nbsp;| .pdf&nbsp;] 
<blockquote>  Two different machine-learning algorithms have been used to 
predict the blood-brain barrier permeability of different classes of molecules, 
to develop a method to predict the ability of drug compounds to penetrate the 
CNS. The first algorithm is based on a multilayer perceptron neural network and 
the second algorithm uses a support vector machine. Both algorithms are trained 
on an identical data set consisting of 179 CNS active molecules and 145 CNS 
inactive molecules. The training parameters include molecular weight, 
lipophilicity, hydrogen bonding, and other variables that govern the ability of 
a molecule to diffuse through a membrane. The results show that the support 
vector machine outperforms the neural network. Based on over 30 different 
validation sets, the SVM can predict up to 96 molecules correctly, averaging 
81.5 of equal numbers of CNS positive and negative molecules. This is quite 
favorable when compared with the neural network's average performance of 75.7 
the SVM algorithm are very encouraging and suggest that a classification tool 
like this one will prove to be a valuable prediction approach. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Deshpande2002Evaluation]  
M.&nbsp;Deshpande and G.&nbsp;Karypis. Evaluation of Techniques for Classifying 
Biological Sequences. In<em>PAKDD '02: Proceedings of the 6th Pacific-Asia 
Conference on Advances in Knowledge Discovery and Data Mining</em>, pages 
417-431. Springer Verlag, 2002. [&nbsp;bib&nbsp;| .pdf&nbsp;] 
<blockquote>  In recent years we have witnessed an exponential increase in the 
amount of biological information, either DNA or protein sequences, that has 
become available in public databases. This has been followed by an increased 
interest in developing computational techniques to automatically classify these 
large volumes of sequence data into various categories corresponding to either 
their role in the chromosomes, their structure, and/or their function. In this 
paper we evaluate some of the widely-used sequence classification algorithms 
and develop a framework for modeling sequences in a fashion so that traditional 
machine learning algorithms, such as support vector machines, can be applied 
easily. Our detailed experimental evaluation shows that the SVM-based 
approaches are able to achieve higher classification accuracy compared to the 
more traditional sequence classification algorithms such as Markov model based 
techniques and K-nearest neighbor based approaches. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Degroeve2002Feature]  
S.&nbsp;Degroeve, B.&nbsp;De&nbsp;Baets, Y.&nbsp;Van&nbsp;de Peer, and 
P.&nbsp;Rouze. Feature subset selection for splice site prediction.<em>
Bioinformatics</em>, 18(Suppl. 1):S75-S83, 2002. [&nbsp;bib&nbsp;| http&nbsp;| 
.pdf&nbsp;] 
<blockquote>  Motivation: The large amount of available annotated Arabidopsis 
thaliana sequences allows the induction of splice site prediction models with 
supervised learning algorithms (see Haussler (1998) for a review and 
references). These algorithms need information sources or features from which 
the models can be computed. For splice site prediction, the features we 
consider in this study are the presence or absence of certain nucleotides in 
close proximity to the splice site. Since it is not known how many and which 
nucleotides are relevant for splice site prediction, the set of features is 
chosen large enough such that the probability that all relevant information 
sources are in the set is very high. Using only those features that are 
relevant for constructing a splice site prediction system might improve the 
system and might also provide us with useful biological knowledge. Using fewer 
features will of course also improve the prediction speed of the system. 
Results: A wrapper-based feature subset selection algorithm using a support 
vector machine or a naive Bayes prediction method was evaluated against the 
traditional method for selecting features relevant for splice site prediction. 
Our results show that this wrapper approach selects features that improve the 
performance against the use of all features and against the use of the features 
selected by the traditional method. Availability: The data and additional 
interactive graphs on the selected feature subsets are available at 
http://www.psb.rug.ac.be/gps Contact: svgro@gengenp.rug.ac.be 
yvdp@gengenp.rug.ac.be </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Chou2002Using]  K.-C. Chou 
and Y.-D. Cai. Using Functional Domain Composition and Support Vector Machines 
for Prediction of Protein Subcellular Location.<em>J. Biol. Chem.</em>, 
277(48):45765-45769, 2002. [&nbsp;bib&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  Proteins are generally classified into the following 12 
subcellular locations: 1) chloroplast, 2) cytoplasm, 3) cytoskeleton, 4) 
endoplasmic reticulum, 5) extracellular, 6) Golgi apparatus, 7) lysosome, 8) 
mitochondria, 9) nucleus, 10) peroxisome, 11) plasma membrane, and 12) vacuole. 
Because the function of a protein is closely correlated with its subcellular 
location, with the rapid increase in new protein sequences entering into 
databanks, it is vitally important for both basic research and pharmaceutical 
industry to establish a high throughput tool for predicting protein subcellular 
location. In this paper, a new concept, the so-called &quot;functional domain 
composition&quot; is introduced. Based on the novel concept, the representation 
for a protein can be defined as a vector in a high-dimensional space, where 
each of the clustered functional domains derived from the protein universe 
serves as a vector base. With such a novel representation for a protein, the 
support vector machine (SVM) algorithm is introduced for predicting protein 
subcellular location. High success rates are obtained by the self-consistency 
test, jackknife test, and independent dataset test, respectively. The current 
approach not only can play an important complementary role to the powerful 
covariant discriminant algorithm based on the pseudo amino acid composition 
representation (Chou, K. C. (2001) Proteins Struct. Funct. Genet. 43, 246-255; 
Correction (2001) Proteins Struct. Funct. Genet. 44, 60), but also may greatly 
stimulate the development of this area. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Cai2002Prediction]  Y.-D. 
Cai, X.-J. Liu, X.-B. Xu, and G.-P. Zhou. Prediction of protein structural 
classes by support vector machines.<em>Comput. Chem.</em>, 26(3):293-296, 2002. 
[&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  In this paper, we apply a new machine learning method which is 
called support vector machine to approach the prediction of protein structural 
class. The support vector machine method is performed based on the database 
derived from SCOP which is based upon domains of known structure and the 
evolutionary relationships and the principles that govern their 3D structure. 
As a result, high rates of both self-consistency and jackknife test are 
obtained. This indicates that the structural class of a protein inconsiderably 
correlated with its amino and composition, and the support vector machine can 
be referred as a powerful computational tool for predicting the structural 
classes of proteins. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Cai2002Support]  Y.-D. 
Cai, X.-J. Liu, X.-B. Xu, and K.-C. Chou. Support vector machines for 
prediction of protein subcellular location by incorporating 
quasi-sequence-order effect.<em>J. Cell. Biochem.</em>, 84(2):343-348, 2002. 
[&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  Support Vector Machine (SVM), which is one class of learning 
machines, was applied to predict the subcellular location of proteins by 
incorporating the quasi-sequence-order effect (Chou [2000] Biochem. Biophys. 
Res. Commun. 278:477-483). In this study, the proteins are classified into the 
following 12 groups: (1) chloroplast, (2) cytoplasm, (3) cytoskeleton, (4) 
endoplasmic reticulum, (5) extracellular, (6) Golgi apparatus, (7) lysosome, 
(8) mitochondria, (9) nucleus, (10) peroxisome, (11) plasma membrane, and (12) 
vacuole, which account for most organelles and subcellular compartments in an 
animal or plant cell. Examinations for self-consistency and jackknife testing 
of the SVMs method were conducted for three sets consisting of 1,911, 2,044, 
and 2,191 proteins. The correct rates for self-consistency and the jackknife 
test values achieved with these protein sets were 94 and 83 89 and 75 for 
correct prediction rates were undertaken with three independent testing 
datasets containing 2,148 proteins, 2,417 proteins, and 2,494 proteins 
producing values of 84, 77, and 74%, respectively. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Cai2002Supportc]  Y.D. 
Cai, X.J. Liu, X.B. Xu, and K.C. Chou. Support vector machines for the 
classification and prediction of beta-turn types.<em>J. Pept. Sci.</em>, 
8(7):297-301, 2002. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| www:&nbsp;] 
<blockquote>  The support vector machines (SVMs) method is proposed because it 
can reflect the sequence-coupling effect for a tetrapeptide in not only a 
beta-turn or non-beta-turn, but also in different types of beta-turn. The 
results of the model for 6022 tetrapeptides indicate that the rates of 
self-consistency for beta-turn types I, I', II, II', VI and VIII and 
non-beta-turns are 99.92 98.02 training data, the rate of correct prediction by 
the SVMs for a given protein: rubredoxin (54 residues. 51 tetrapeptides) which 
includes 12 beta-turn type I tetrapeptides, 1 beta-turn type II tetrapeptide 
and 38 non-beta-turns reached 82.4 of the SVMs implies that the formation of 
different beta-turn types or non-beta-turns is considerably correlated with the 
sequence of a tetrapeptide. The SVMs can save CPU time and avoid the 
overfitting problem compared with the neural network method. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Cai2002Supportb]  Y.D. 
Cai, X.J. Liu, X.B. Xu, and K.C. Chou. Support vector machines for predicting 
the specificity of GalNAc-transferase.<em>Peptides</em>, 23:205-208, 2002. 
[&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  Support Vector Machines (SVMs) which is one kind of learning 
machines, was applied to predict the specificity of GalNAc-transferase. The 
examination for the self-consistency and the jackknife test of the SVMs method 
were tested for the training dataset (305 oligopeptides), the correct rate of 
self-consistency and jackknife test reaches 100 and 84.9 testing dataset (30 
oligopeptides) was tested, the rate reaches 76.67%. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Cai2002Supporta]  Y.D. 
Cai, X.J. Liu, X.B. Xu, and K.C. Chou. Support Vector Machines for predicting 
HIV protease cleavage sites in protein.<em>J. Comput. Chem.</em>, 
23(2):267-274, 2002. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| www:&nbsp;] 
<blockquote>  Knowledge of the polyprotein cleavage sites by HIV protease will 
refine our understanding of its specificity, and the information thus acquired 
is useful for designing specific and efficient HIV protease inhibitors. The 
pace in searching for the proper inhibitors of HIV protease will be greatly 
expedited if one can find an accurate, robust, and rapid method for predicting 
the cleavage sites in proteins by HIV protease. In this article, a Support 
Vector Machine is applied to predict the cleavability of oligopeptides by 
proteases with multiple and extended specificity subsites. We selected HIV-1 
protease as the subject of the study. Two hundred ninety-nine oligopeptides 
were chosen for the training set, while the other 63 oligopeptides were taken 
as a test set. Because of its high rate of self-consistency (299/299 = 100 and 
correct prediction rate (55/63 = 87 Support Vector Machine method can be 
referred to as a useful assistant technique for finding effective inhibitors of 
HIV protease, which is one of the targets in designing potential drugs against 
AIDS. The principle of the Support Vector Machine method can also be applied to 
analyzing the specificity of other multisubsite enzymes. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Bock2002New]  J.&nbsp;R. 
Bock and D.&nbsp;A. Gough. A New Method to Estimate Ligand-Receptor Energetics.
<em>Mol Cell Proteomics</em>, 1(11):904-910, 2002. [&nbsp;bib&nbsp;| http&nbsp;|
.pdf&nbsp;] 
<blockquote>  In the discovery of new drugs, lead identification and 
optimization have assumed critical importance given the number of drug targets 
generated from genetic, genomics, and proteomic technologies. High-throughput 
experimental screening assays have been complemented recently by &quot;virtual 
screening&quot; approaches to identify and filter potential ligands when the 
characteristics of a target receptor structure of interest are known. Virtual 
screening mandates a reliable procedure for automatic ranking of structurally 
distinct ligands in compound library databases. Computing a rank score requires 
the accurate prediction of binding affinities between these ligands and the 
target. Many current scoring strategies require information about the target 
three-dimensional structure. In this study, a new method to estimate the free 
binding energy between a ligand and receptor is proposed. We extend a central 
idea previously reported (Bock, J. R., and Gough, D. A. (2001) Predicting 
protein-protein interactions from primary structure. Bioinformatics 17, 
455-460; Bock, J. R., and Gough, D. A. (2002) Whole-proteome interaction 
mining. Bioinformatics, in press) that uses simple descriptors to represent 
biomolecules as input examples to train a support vector machine (Smola, A. J., 
and Scholkopf, B. (1998) A Tutorial on Support Vector Regression, NeuroCOLT 
Technical Report NC-TR-98-030, Royal Holloway College, University of London, 
UK) and the application of the trained system to previously unseen pairs, 
estimating their propensity for interaction. Here we seek to learn the function 
that maps features of a receptor-ligand pair onto their equilibrium free 
binding energy. These features do not comprise any direct information about the 
three-dimensional structures of ligand or target. In cross-validation 
experiments, it is demonstrated that objective measurements of prediction error 
rate and rank-ordering statistics are competitive with those of several other 
investigations, most of which depend on three-dimensional structural data. The 
size of the sample (n = 2,671) indicates that this approach is robust and may 
have widespread applicability beyond restricted families of receptor types. It 
is concluded that newly sequenced proteins, or those for which 
three-dimensional crystal structures are not easily obtained, can be rapidly 
analyzed for their binding potential against a library of ligands using this 
methodology. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Bao2002Identifying]  
L.&nbsp;Bao and Z.&nbsp;Sun. Identifying genes related to drug anticancer 
mechanisms using support vector machine.<em>FEBS Lett.</em>, 521:109-114, 2002. 
[&nbsp;bib&nbsp;| .html&nbsp;| .pdf&nbsp;] 
<blockquote>  In an effort to identify genes related to the cell line 
chemosensitivity and to evaluate the functional relationships between genes and 
anticancer drugs acting by the same mechanism, a supervised machine learning 
approach called support vector machine was used to label genes into any of the 
five predefined anticancer drug mechanistic categories. Among dozens of 
unequivocally categorized genes, many were known to be causally related to the 
drug mechanisms. For example, a few genes were found to be involved in the 
biological process triggered by the drugs (e.g. DNA polymerase epsilon was the 
direct target for the drugs from DNA antimetabolites category). DNA 
repair-related genes were found to be enriched for about eight-fold in the 
resulting gene set relative to the entire gene set. Some uncharacterized 
transcripts might be of interest in future studies. This method of correlating 
the drugs and genes provides a strategy for finding novel biologically 
significant relationships for molecular pharmacology. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm microarray </blockquote> <br>
 [
Ambroise2002Selection]  C.&nbsp;Ambroise and G.J. McLachlan. Selection bias in 
gene extraction on the basis of microarray gene-expression data.<em>Proc. Natl. 
Acad. Sci. USA</em>, 99(10):6562-6566, 2002. [&nbsp;bib&nbsp;| http&nbsp;| .pdf
&nbsp;]
<blockquote>  In the context of cancer diagnosis and treatment, we consider 
the problem of constructing an accurate prediction rule on the basis of a 
relatively small number of tumor tissue samples of known type containing the 
expression data on very many (possibly thousands) genes. Recently, results have 
been presented in the literature suggesting that it is possible to construct a 
prediction rule from only a few genes such that it has a negligible prediction 
error rate. However, in these results the test error or the leave-one-out 
cross-validated error is calculated without allowance for the selection bias. 
There is no allowance because the rule is either tested on tissue samples that 
were used in the first instance to select the genes being used in the rule or 
because the cross-validation of the rule is not external to the selection 
process; that is, gene selection is not performed in training the rule at each 
stage of the cross-validation process. We describe how in practice the 
selection bias can be assessed and corrected for by either performing a 
cross-validation or applying the bootstrap external to the selection process. 
We recommend using 10-fold rather than leave-one-out cross-validation, and 
concerning the bootstrap, we suggest using the so-called .632+ bootstrap error 
estimate designed to handle overfitted prediction rules. Using two published 
data sets, we demonstrate that when correction is made for the selection bias, 
the cross-validated error is no longer zero for a subset of only a few genes. 
</blockquote> 
<p></p> 
<blockquote>  Keywords: featureselection biosvm </blockquote> <br>
 [
Aliferis2002Machine]  C.F. Aliferis, D.P. Hardin, and P.&nbsp;Massion. Machine 
Learning Models For Lung Cancer Classification Using Array Comparative Genomic 
Hybridization. In<em>Proceedings of the 2002 American Medical Informatics 
Association (AMIA) Annual Symposium</em>, pages 7-11, 2002. [&nbsp;bib&nbsp;| 
.pdf&nbsp;] 
<blockquote>  Array CGH is a recently introduced technology that measures 
changes in the gene copy number of hundreds of genes in a single experiment. 
The primary goal of this study was to develop machine learning models that 
classify non-small Lung Cancers according to histopathology types and to 
compare several machine learning methods in this learning task. DNA from tumors 
of 37 patients (21 squamous carcinomas, and 16 adenocarcinomas) were extracted 
and hybridized onto a 452 BAC clone array. The following algorithms were used: 
KNN, Decision Tree Induction, Support Vector Machines and Feed-Forward Neural 
Networks. Performance was measured via leave-one-out classification accuracy. 
The best multi-gene model found had a leave-one-out accuracy of 89.2%. Decision 
Trees performed poorer than the other methods in this learning task and 
dataset. We conclude that gene copy numbers as measured by array CGH are, 
collectively, an excellent indicator of histological subtype. Several 
interesting research directions are discussed. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm microarray, cgh </blockquote> <br>
 [
Zhao2003Application]  Y.&nbsp;Zhao, C.&nbsp;Pinilla, D.&nbsp;Valmori, 
R.&nbsp;Martin, and R.&nbsp;Simon. Application of support vector machines for 
T-cell epitopes prediction.<em>Bioinformatics</em>, 19(15):1978-1984, 2003. 
[&nbsp;bib&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  Motivation: The T-cell receptor, a major histocompatibility 
complex (MHC) molecule, and a bound antigenic peptide, play major roles in the 
process of antigen-specific T-cell activation. T-cell recognition was long 
considered exquisitely specific. Recent data also indicate that it is highly 
flexible, and one receptor may recognize thousands of different peptides. 
Deciphering the patterns of peptides that elicit a MHC restricted T-cell 
response is critical for vaccine development. Results: For the first time we 
develop a support vector machine (SVM) for T-cell epitope prediction with an 
MHC type I restricted T-cell clone. Using cross-validation, we demonstrate that 
SVMs can be trained on relatively small data sets to provide prediction more 
accurate than those based on previously published methods or on MHC binding. 
Supplementary information: Data for 203 synthesized peptides is available at 
http://linus.nci.nih.gov/Data/LAU203_Peptide.pdf </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm immunoinformatics </blockquote> <br>
 [
Zhang2003Sequence]  X.&nbsp;H-F. Zhang, K.&nbsp;A. Heller, I.&nbsp;Hefter, 
C.&nbsp;S. Leslie, and L.&nbsp;A. Chasin. Sequence Information for the Splicing 
of Human Pre-mRNA Identified by Support Vector Machine Classification.<em>
Genome Res.</em>, 13(12):2637-2650, 2003. [&nbsp;bib&nbsp;| DOI&nbsp;| http
&nbsp;|.pdf&nbsp;] 
<blockquote>  Vertebrate pre-mRNA transcripts contain many sequences that 
resemble splice sites on the basis of agreement to the consensus, yet these 
more numerous false splice sites are usually completely ignored by the cellular 
splicing machinery. Even at the level of exon definition, pseudo exons defined 
by such false splices sites outnumber real exons by an order of magnitude. We 
used a support vector machine to discover sequence information that could be 
used to distinguish real exons from pseudo exons. This machine learning tool 
led to the definition of potential branch points, an extended polypyrimidine 
tract, and C-rich and TG-rich motifs in a region limited to 50 nt upstream of 
constitutively spliced exons. C-rich sequences were also found in a region 
extending to 80 nt downstream of exons, along with G-triplet motifs. In 
addition, it was shown that combinations of three bases within the splice donor 
consensus sequence were more effective than consensus values in distinguishing 
real from pseudo splice sites; two-way base combinations were optimal for 
distinguishing 3' splice sites. These data also suggest that interactions 
between two or more of these elements may contribute to exon recognition, and 
provide candidate sequences for assessment as intronic splicing enhancers. 
</blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Zhang2003Classification]  
S.-W. Zhang, Q.&nbsp;Pan, H.-C. Zhang, Y-L. Zhang, and H.-Y. Wang. 
Classification of protein quaternary structure with support vector machine.<em>
Bioinformatics</em>, 19(18):2390-2396, 2003. [&nbsp;bib&nbsp;| http&nbsp;| .pdf
&nbsp;]
<blockquote>  Motivation: Since the gap between sharply increasing known 
sequences and slow accumulation of known structures is becoming large, an 
automatic classification process based on the primary sequences and known 
three-dimensional structure becomes indispensable. The classification of 
protein quaternary structure based on the primary sequences can provide some 
useful information for the biologists. So a fully automatic and reliable 
classification system is needed. This work tries to look for the effective 
methods of extracting attribute and the algorithm for classifying the 
quaternary structure from the primary sequences. Results: Both of the support 
vector machine (SVM) and the covariant discriminant algorithms have been first 
introduced to predict quaternary structure properties from the protein primary 
sequences. The amino acid composition and the auto-correlation functions based 
on the amino acid index profile of the primary sequence have been taken into 
account in the algorithms. We have analyzed 472 amino acid indices and selected 
the four amino acid indices as the examples, which have the best performance. 
Thus the five attribute parameter data sets (COMP, FASG, NISK, WOLS and KYTJ) 
were established from the protein primary sequences. The COMP attribute data 
set is composed of amino acid composition, and the FASG, NISK, WOLS and KYTJ 
attribute data sets are composed of the amino acid composition and the 
auto-correlation functions of the corresponding amino acid residue index. The 
overall accuracies of SVM are 78.5, 87.5, 83.2, 81.7 and 81.9 and KYTJ data 
sets in jackknife test, which are 19.6, 7.8, 15.5, 13.1 and 15.8 algorithm in 
the same test. The results show that SVM may be applied to discriminate between 
the primary sequences of homodimers and non-homodimers and the two protein 
sequence descriptors can reflect the quaternary structure information. Compared 
with previous Robert Garian's investigation, the performance of SVM is almost 
equal to that of the Decision tree models, and the methods of extracting 
feature vector from the primary sequences are superior to Robert's binning 
function method. Availability: Programs are available on request from the 
authors. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Zernov2003Drug]  
V.&nbsp;V. Zernov, K.&nbsp;V. Balakin, A.&nbsp;A. Ivaschenko, N.&nbsp;P. 
Savchuk, and I.&nbsp;V. Pletnev. Drug discovery using support vector machines. 
The case studies of drug-likeness, agrochemical-likeness, and enzyme inhibition 
predictions.<em>J Chem Inf Comput Sci</em>, 43(6):2048-56, 2003. [&nbsp;bib
&nbsp;|DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  Support Vector Machines (SVM) is a powerful classification and 
regression tool that is becoming increasingly popular in various machine 
learning applications. We tested the ability of SVM, in comparison with 
well-known neural network techniques, to predict drug-likeness and 
agrochemical-likeness for large compound collections. For both kinds of data, 
SVM outperforms various neural networks using the same set of descriptors. We 
also used SVM for estimating the activity of Carbonic Anhydrase II (CA II) 
enzyme inhibitors and found that the prediction quality of our SVM model is 
better than that reported earlier for conventional QSAR. Model characteristics 
and data set features were studied in detail. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm chemoinformatics </blockquote> <br>
 [
Yu2003Fine-grained]  C.S. Yu, J.Y. Wang, J.M. Yang, P.C. Lyu, C.J. Lin, and 
J.K. Hwang. Fine-grained protein fold assignment by support vector machines 
using generalized npeptide coding schemes and jury voting from 
multiple-parameter sets.<em>Proteins</em>, 50(4):531, 6 2003. [&nbsp;bib&nbsp;| 
DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  In the coarse-grained fold assignment of major protein classes, 
such as all-alpha, all-beta, alpha + beta, alpha/beta proteins, one can easily 
achieve high prediction accuracy from primary amino acid sequences. However, 
the fine-grained assignment of folds, such as those defined in the Structural 
Classification of Proteins (SCOP) database, presents a challenge due to the 
larger amount of folds available. Recent study yielded reasonable prediction 
accuracy of 56.0 an independent set of 27 most populated folds. In this 
communication, we apply the support vector machine (SVM) method, using a 
combination of protein descriptors based on the properties derived from the 
composition of n-peptide and jury voting, to the fine-grained fold prediction, 
and are able to achieve an overall prediction accuracy of 69.6 the same 
independent set-significantly higher than the previous results. On 10-fold 
cross-validation, we obtained a prediction accuracy of 65.3 sequence-coding 
schemes can significantly improve the fine-grained fold prediction. Our 
approach should be useful in structure prediction and modeling. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Yoon2003Analysis]  
Y.&nbsp;Yoon, J.&nbsp;Song, S.H. Hong, and J.Q. Kim. Analysis of multiple 
single nucleotide polymorphisms of candidate genes related to coronary heart 
disease susceptibility by using support vector machines.<em>Clin. Chem. Lab. 
Med.</em>, 41(4):529-534, 2003. [&nbsp;bib&nbsp;| .html&nbsp;| .pdf&nbsp;] 
<blockquote>  Coronary heart disease (CHD) is a complex genetic disease 
involving gene-environment interaction. Many association studies between single 
nucleotide polymorphisms (SNPs) of candidate genes and CHD have been reported. 
We have applied a new method to analyze such relationships using support vector 
machines (SVMs), which is one of the methods for artificial neuronal network. 
We assumed that common haplotype implicit in genotypes will differ between 
cases and controls, and that this will allow SVM-derived patterns to be 
classifiable according to subject genotypes. Fourteen SNPs of ten candidate 
genes in 86 CHD patients and 119 controls were investigated. Genotypes were 
transformed to a numerical vector by giving scores based on difference between 
the genotypes of each subject and the reference genotypes, which represent the 
healthy normal population. Overall classification accuracy by SVMs was 64.4 By 
conventional analysis using the chi2 test, the association between CHD and the 
SNP of the scavenger receptor B1 gene was most significant in terms of allele 
frequencies in cases vs. controls (p = 0.0001). In conclusion, we suggest that 
the application of SVMs for association studies of SNPs in candidate genes 
shows considerable promise and that further work could be usefully performed 
upon the estimation of CHD susceptibility in individuals of high risk. 
</blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Yamanishi2003Extraction]  
Y.&nbsp;Yamanishi, J.-P. Vert, A.&nbsp;Nakaya, and M.&nbsp;Kanehisa. Extraction 
of correlated gene clusters from multiple genomic data by generalized kernel 
canonical correlation analysis.<em>Bioinformatics</em>, 19(Suppl. 1):i323-i330, 
2003. [&nbsp;bib&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  Motivation: A major issue in computational biology is the 
reconstruction of pathways from several genomic datasets, such as expression 
data, protein interaction data and phylogenetic profiles. As a first step 
toward this goal, it is important to investigate the amount of correlation 
which exists between these data. Results: These methods are successfully tested 
on their ability to recognize operons in the Escherichia coli genome, from the 
comparison of three datasets corresponding to functional relationships between 
genes in metabolic pathways, geometrical relationships along the chromosome, 
and co-expression relationships as observed by gene expression data. Contact: 
yoshi@kuicr.kyoto-u.ac.jp </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Wu2003Comparison]  
B.&nbsp;Wu, T.&nbsp;Abbott, D.&nbsp;Fishman, W.&nbsp;McMurray, G.&nbsp;Mor, 
K.&nbsp;Stone, D.&nbsp;Ward, K.&nbsp;Williams, and H.&nbsp;Zhao. Comparison of 
statistical methods for classification of ovarian cancer using mass 
spectrometry data.<em>Bioinformatics</em>, 19(13):1636-1643, 2003. [&nbsp;bib
&nbsp;|http&nbsp;| .pdf&nbsp;] 
<blockquote>  Motivation: Novel methods, both molecular and statistical, are 
urgently needed to take advantage of recent advances in biotechnology and the 
human genome project for disease diagnosis and prognosis. Mass spectrometry 
(MS) holds great promise for biomarker identification and genome-wide protein 
profiling. It has been demonstrated in the literature that biomarkers can be 
identified to distinguish normal individuals from cancer patients using MS 
data. Such progress is especially exciting for the detection of early-stage 
ovarian cancer patients. Although various statistical methods have been 
utilized to identify biomarkers from MS data, there has been no systematic 
comparison among these approaches in their relative ability to analyze MS data. 
Results: We compare the performance of several classes of statistical methods 
for the classification of cancer based on MS spectra. These methods include: 
linear discriminant analysis, quadratic discriminant analysis, k-nearest 
neighbor classifier, bagging and boosting classification trees, support vector 
machine, and random forest (RF). The methods are applied to ovarian cancer and 
control serum samples from the National Ovarian Cancer Early Detection Program 
clinic at Northwestern University Hospital. We found that RF outperforms other 
methods in the analysis of MS data. Supplementary information: 
http://bioinformatics.med.yale.edu/proteomics/BioSupp1.html </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Winters-Hilt2003Highly]  
S.&nbsp;Winters-Hilt, W.&nbsp;Vercoutere, V.S. DeGuzman, D.&nbsp;Deamer, 
M.&nbsp;Akeson, and D.&nbsp;Haussler. Highly accurate classification of 
Watson-Crick basepairs on termini of single DNA molecules.<em>Biophys. J.</em>, 
84(2):967-976, 2003. [&nbsp;bib&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  We introduce a computational method for classification of 
individual DNA molecules measured by analpha -hemolysin channel detector. We 
show classification with better than 99 hairpin molecules that differ only in 
their terminal Watson-Crick basepairs. Signal classification was done in silico 
to establish performance metrics (i.e., where train and test data were of known 
type, via single-species data files). It was then performed in solution to 
assay real mixtures of DNA hairpins. Hidden Markov Models (HMMs) were used with 
Expectation/Maximization for denoising and for associating a feature vector 
with the ionic current blockade of the DNA molecule. Support Vector Machines 
(SVMs) were used as discriminators, and were the focus of off-line training. A 
multiclass SVM architecture was designed to place less discriminatory load on 
weaker discriminators, and novel SVM kernels were used to boost discrimination 
strength. The tuning on HMMs and SVMs enabled biophysical analysis of the 
captured molecule states and state transitions; structure revealed in the 
biophysical analysis was used for better feature selection. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Wilton2003Comparison]  
D.&nbsp;Wilton, P.&nbsp;Willett, K.&nbsp;Lawson, and G.&nbsp;Mullier. 
Comparison of ranking methods for virtual screening in lead-discovery programs.
<em>J Chem Inf Comput Sci</em>, 43(2):469-74, 2003. [&nbsp;bib&nbsp;| DOI&nbsp;|
http&nbsp;| .pdf&nbsp;] 
<blockquote>  This paper discusses the use of several rank-based virtual 
screening methods for prioritizing compounds in lead-discovery programs, given 
a training set for which both structural and bioactivity data are available. 
Structures from the NCI AIDS data set and from the Syngenta corporate database 
were represented by two types of fragment bit-string and by sets of high-level 
molecular features. These representations were processed using binary kernel 
discrimination, similarity searching, substructural analysis, support vector 
machine, and trend vector analysis, with the effectiveness of the methods being 
judged by the extent to which active test set molecules were clustered toward 
the top of the resultant rankings. The binary kernel discrimination approach 
yielded consistently superior rankings and would appear to have considerable 
potential for chemical screening applications. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Weston2003Feature]  
J.&nbsp;Weston, F.&nbsp;P&eacute;rez-Cruz, O.&nbsp;Bousquet, O.&nbsp;Chapelle, 
A.&nbsp;Elisseeff, and B.&nbsp;Sch&ouml;lkopf. Feature selection and 
transduction for prediction of molecular bioactivity for drug design.<em>
Bioinformatics</em>, 19(6):764-771, 2003. [&nbsp;bib&nbsp;| http&nbsp;| .pdf
&nbsp;]
<blockquote>  Motivation: In drug discovery a key task is to identify 
characteristics that separate active (binding) compounds from inactive 
(non-binding) ones. An automated prediction system can help reduce resources 
necessary to carry out this task. Results: Two methods for prediction of 
molecular bioactivity for drug design are introduced and shown to perform well 
in a data set previously studied as part of the KDD (Knowledge Discovery and 
Data Mining) Cup 2001. The data is characterized by very few positive examples, 
a very large number of features (describing three-dimensional properties of the 
molecules) and rather different distributions between training and test data. 
Two techniques are introduced specifically to tackle these problems: a feature 
selection method for unbalanced data and a classifier which adapts to the 
distribution of the the unlabeled test data (a so-called transductive method). 
We show both techniques improve identification performance and in conjunction 
provide an improvement over using only one of the techniques. Our results 
suggest the importance of taking into account the characteristics in this data 
which may also be relevant in other problems of a similar type. Availability: 
Matlab source code is available at 
http://www.kyb.tuebingen.mpg.de/bs/people/weston/kdd/kdd.html Contact: 
jason.weston@tuebingen.mpg.de Supplementary information: Supplementary material 
is available at http://www.kyb.tuebingen.mpg.de/bs/people/weston/kdd/kdd.html. 
</blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Warmuth2003Active]  
M.&nbsp;K. Warmuth, J.&nbsp;Liao, G.&nbsp;R&auml;tsch, M.&nbsp;Mathieson, 
S.&nbsp;Putta, and C.&nbsp;Lemmen. Active learning with support vector machines 
in the drug discovery process.<em>J Chem Inf Comput Sci</em>, 43(2):667-673, 
2003. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  We investigate the following data mining problem from 
computer-aided drug design: From a large collection of compounds, find those 
that bind to a target molecule in as few iterations of biochemical testing as 
possible. In each iteration a comparatively small batch of compounds is 
screened for binding activity toward this target. We employed the so-called 
&quot;active learning paradigm&quot; from Machine Learning for selecting the 
successive batches. Our main selection strategy is based on the maximum margin 
hyperplane-generated by &quot;Support Vector Machines&quot;. This hyperplane 
separates the current set of active from the inactive compounds and has the 
largest possible distance from any labeled compound. We perform a thorough 
comparative study of various other selection strategies on data sets provided 
by DuPont Pharmaceuticals and show that the strategies based on the maximum 
margin hyperplane clearly outperform the simpler ones. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Ward2003Secondary]  
J.&nbsp;J. Ward, L.&nbsp;J. McGuffin, B.&nbsp;F. Buxton, and D.&nbsp;T. Jones. 
Secondary structure prediction with support vector machines.<em>Bioinformatics
</em>, 19(13):1650-1655, 2003. [&nbsp;bib&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  Motivation: A new method that uses support vector machines 
(SVMs) to predict protein secondary structure is described and evaluated. The 
study is designed to develop a reliable prediction method using an alternative 
technique and to investigate the applicability of SVMs to this type of 
bioinformatics problem. Methods: Binary SVMs are trained to discriminate 
between two structural classes. The binary classifiers are combined in several 
ways to predict multi-class secondary structure. Results: The average 
three-state prediction accuracy per protein (Q3) is estimated by 
cross-validation to be 77.07 +/- 0.26 +/- 0.39 PSIPRED prediction method on a 
non-homologous test set of 121 proteins despite being trained on substantially 
fewer examples. A simple consensus of the SVM, PSIPRED and PROFsec achieves 
significantly higher prediction accuracy than the individual methods. 
Availability: The SVM classifier is available from the authors. Work is in 
progress to make the method available on-line and to integrate the SVM 
predictions into the PSIPRED server. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Wagner2003Protocols]  
M.&nbsp;Wagner, D.&nbsp;Naik, and A.&nbsp;Pothen. Protocols for disease 
classification from mass spectrometry data.<em>Proteomics</em>, 3(9):1692-1698, 
2003. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  We report our results in classifying protein matrix-assisted 
laser desorption/ionization-time of flight mass spectra obtained from serum 
samples into diseased and healthy groups. We discuss in detail five of the 
steps in preprocessing the mass spectral data for biomarker discovery, as well 
as our criterion for choosing a small set of peaks for classifying the samples. 
Cross-validation studies with four selected proteins yielded misclassification 
rates in the 10-15 for all the classification methods. Three of these proteins 
or protein fragments are down-regulated and one up-regulated in lung cancer, 
the disease under consideration in this data set. When cross-validation studies 
are performed, care must be taken to ensure that the test set does not 
influence the choice of the peaks used in the classification. Misclassification 
rates are lower when both the training and test sets are used to select the 
peaks used in classification versus when only the training set is used. This 
expectation was validated for various statistical discrimination methods when 
thirteen peaks were used in cross-validation studies. One particular 
classification method, a linear support vector machine, exhibited especially 
robust performance when the number of peaks was varied from four to thirteen, 
and when the peaks were selected from the training set alone. Experiments with 
the samples randomly assigned to the two classes confirmed that 
misclassification rates were significantly higher in such cases than those 
observed with the true data. This indicates that our findings are indeed 
significant. We found closely matching masses in a database for protein 
expression in lung cancer for three of the four proteins we used to classify 
lung cancer. Data from additional samples, increased experience with the 
performance of various preprocessing techniques, and affirmation of the 
biological roles of the proteins that help in classification, will strengthen 
our conclusions in the future. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Vert2003Extracting]  J.-P. 
Vert and M.&nbsp;Kanehisa. Extracting active pathways from gene expression data.
<em>Bioinformatics</em>, 19:238ii-234ii, 2003. [&nbsp;bib&nbsp;| http&nbsp;| 
.pdf&nbsp;] 
<blockquote>  Motivation: A promising way to make sense out of gene expression 
profiles is to relate them to the activity of metabolic and signalling 
pathways. Each pathway usually involves many genes, such as enzymes, which can 
themselves participate in many pathways. The set of all known pathways can 
therefore be represented by a complex network of genes. Searching for 
regularities in the set of gene expression profiles with respect to the 
topology of this gene network is a way to automatically extract active pathways 
and their associated patterns of activity. Method: We present a method to 
perform this task, which consists in encoding both the gene network and the set 
of profiles into two kernel functions, and performing a regularized form of 
canonical correlation analysis between the two kernels. Results: When applied 
to publicly available expression data the method is able to extract 
biologically relevant expression patterns, as well as pathways with related 
activity. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Vert2003Graph-driven]  
J.-P. Vert and M.&nbsp;Kanehisa. Graph-driven features extraction from 
microarray data using diffusion kernels and kernel CCA. In S.&nbsp;Becker, 
S.&nbsp;Thrun, and K.&nbsp;Obermayer, editors,<em>Adv. Neural Inform. Process. 
Syst.</em>, pages 1449-1456. MIT Press, 2003. [&nbsp;bib&nbsp;| .pdf&nbsp;] 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Tsuda2003em]  
K.&nbsp;Tsuda, S.&nbsp;Akaho, and K.&nbsp;Asai. The em Algorithm for Kernel 
Matrix Completion with Auxiliary Data.<em>J. Mach. Learn. Res.</em>, 4:67-81, 
2003. [&nbsp;bib&nbsp;| .html&nbsp;| .pdf&nbsp;] 
<blockquote>  In biological data, it is often the case that observed data are 
available only for a subset of samples. When a kernel matrix is derived from 
such data, we have to leave the entries for unavailable samples as missing. In 
this paper, the missing entries are completed by exploiting an auxiliary kernel 
matrix derived from another information source. The parametric model of kernel 
matrices is created as a set of spectral variants of the auxiliary kernel 
matrix, and the missing entries are estimated by fitting this model to the 
existing entries. For model fitting, we adopt the em algorithm (distinguished 
from the EM algorithm of Dempster et al., 1977) based on the information 
geometry of positive definite matrices. We will report promising results on 
bacteria clustering experiments using two marker sequences: 16S and gyrB. 
</blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Takaoka2003Development]  
Y.&nbsp;Takaoka, Y.&nbsp;Endo, S.&nbsp;Yamanobe, H.&nbsp;Kakinuma, 
T.&nbsp;Okubo, Y.&nbsp;Shimazaki, T.&nbsp;Ota, S.&nbsp;Sumiya, and 
K.&nbsp;Yoshikawa. Development of a method for evaluating drug-likeness and 
ease of synthesis using a data set in which compounds are assigned scores based 
on chemists' intuition.<em>J Chem Inf Comput Sci</em>, 43(4):1269-75, 2003. 
[&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  The concept of drug-likeness, an important characteristic for 
any compound in a screening library, is nevertheless difficult to pin down. 
Based on our belief that this concept is implicit within the collective 
experience of working chemists, we devised a data set to capture an intuitive 
human understanding of both this characteristic and ease of synthesis, a second 
key characteristic. Five chemists assigned a pair of scores to each of 3980 
diverse compounds, with the component scores of each pair corresponding to 
drug-likeness and ease of synthesis, respectively. Using this data set, we 
devised binary classifiers with an artificial neural network and a support 
vector machine. These models were found to efficiently eliminate compounds that 
are not drug-like and/or hard-to-synthesize derivatives, demonstrating the 
suitability of these models for use as compound acquisition filters. 
</blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Sun2003Identifying]  Y.F. 
Sun, X.D. Fan, and Y.D. Li. Identifying splicing sites in eukaryotic RNA: 
support vector machine approach.<em>Comput. Biol. Med.</em>, 33(1):17-29, 2003. 
[&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  We introduce a new method for splicing sites prediction based on 
the theory of support vector machines (SVM). The SVM represents a new approach 
to supervised pattern classification and has been successfully applied to a 
wide range of pattern recognition problems. In the process of splicing sites 
prediction, the statistical information of RNA secondary structure in the 
vicinity of splice sites, e.g. donor and acceptor sites, is introduced in order 
to compare recognition ratio of true positive and true negative. From the 
results of comparison, addition of structural information has brought no 
significant benefit for the recognition of splice sites and had even lowered 
the rate of recognition. Our results suggest that, through three cross 
validation, the SVM method can achieve a good performance for splice sites 
identification. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Su2003RankGene]  Yang Su, 
T.M. Murali, Vladimir Pavlovic, Michael Schaffer, and Simon Kasif. RankGene: 
identification of diagnostic genes based on expression data.<em>Bioinformatics
</em>, 19(12):1578-1579, 2003. [&nbsp;bib&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  Summary: RankGene is a program for analyzing gene expression 
data and computing diagnostic genes based on their predictive power in 
distinguishing between different types of samples. The program integrates into 
one system a variety of popular ranking criteria, ranging from the traditional 
t-statistic to one-dimensional support vector machines. This flexibility makes 
RankGene a useful tool in gene expression analysis and feature selection. 
Availability: http://genomics10.bu.edu/yangsu/rankgene Contact: murali@bu.edu 
</blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Sorich2003Comparison]  
M.&nbsp;J. Sorich, J.&nbsp;O. Miners, R.&nbsp;A. McKinnon, D.&nbsp;A. Winkler, 
F.&nbsp;R. Burden, and P.&nbsp;A. Smith. Comparison of linear and nonlinear 
classification algorithms for the prediction of drug and chemical metabolism by 
human UDP-glucuronosyltransferase isoforms.<em>J Chem Inf Comput Sci</em>, 
43(6):2019-24, 2003. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  Partial least squares discriminant analysis (PLSDA), Bayesian 
regularized artificial neural network (BRANN), and support vector machine (SVM) 
methodologies were compared by their ability to classify substrates and 
nonsubstrates of 12 isoforms of human UDP-glucuronosyltransferase (UGT), an 
enzyme &quot;superfamily&quot; involved in the metabolism of drugs, nondrug 
xenobiotics, and endogenous compounds. Simple two-dimensional descriptors were 
used to capture chemical information. For each data set, 70% of the data were 
used for training, and the remainder were used to assess the generalization 
performance. In general, the SVM methodology was able to produce models with 
the best predictive performance, followed by BRANN and then PLSDA. However, a 
small number of data sets showed either equivalent or better predictability 
using PLSDA, which may indicate relatively linear relationships in these data 
sets. All SVM models showed predictive ability (&gt;60% of test set predicted 
correctly) and five out of the 12 test sets showed excellent prediction 
(&gt;80% prediction accuracy). These models represent the first use of pattern 
recognition methods to discriminate between substrates and nonsubstrates of 
human drug metabolizing enzymes and the first thorough assessment of three 
classification algorithms using multiple metabolic data sets. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Siepen2003Beta]  
J.&nbsp;A. Siepen, S.&nbsp;E. Radford, and D.&nbsp;R. Westhead. Beta Edge 
strands in protein structure prediction and aggregation.<em>Protein Sci.</em>, 
12(10):2348-2359, 2003. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  It is well established that recognition between exposed edges of 
beta-sheets is an important mode of protein-protein interaction and can have 
pathological consequences; for instance, it has been linked to the aggregation 
of proteins into a fibrillar structure, which is associated with a number of 
predominantly neurodegenerative disorders. A number of protective mechanisms 
have evolved in the edge strands of beta-sheets, preventing the aggregation and 
insolubility of most natural beta-sheet proteins. Such mechanisms are 
unfavorable in the interior of a beta-sheet. The problem of distinguishing edge 
strands from central strands based on sequence information alone is important 
in predicting residues and mutations likely to be involved in aggregation, and 
is also a first step in predicting folding topology. Here we report support 
vector machine (SVM) and decision tree methods developed to classify edge 
strands from central strands in a representative set of protein domains. 
Interestingly, rules generated by the decision tree method are in close 
agreement with our knowledge of protein structure and are potentially useful in 
a number of different biological applications. When trained on strands from 
proteins of known structure, using structure-based (Dictionary of Secondary 
Structure in Proteins) strand assignments, both methods achieved mean 
cross-validated, prediction accuracies of &nbsp;78 strand assignments from 
secondary structure prediction were used. Further investigation of this effect 
revealed that it could be explained by a significant reduction in the accuracy 
of standard secondary structure prediction methods for edge strands, in 
comparison with central strands. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [
She2003Frequent-subsequence-based]  R.&nbsp;She, F.&nbsp;Chen, K.&nbsp;Wang, 
M.&nbsp;Ester, J.L. Gardy, and F.S.L. Brinkman. Frequent-subsequence-based 
prediction of outer membrane proteins. In<em>KDD '03: Proceedings of the ninth 
ACM SIGKDD international conference on Knowledge discovery and data mining</em>
, pages 436-445. ACM Press, 2003. [&nbsp;bib&nbsp;| DOI&nbsp;| .pdf&nbsp;] 
<blockquote>  A number of medically important disease-causing bacteria 
(collectively called Gram-negative bacteria) are noted for the extra 
&quot;outer&quot; membrane that surrounds their cell. Proteins resident in this 
membrane (outer membrane proteins, or OMPs) are of primary research interest 
for antibiotic and vaccine drug design as they are on the surface of the 
bacteria and so are the most accessible targets to develop new drugs against. 
With the development of genome sequencing technology and bioinformatics, 
biologists can now deduce all the proteins that are likely produced in a given 
bacteria and have attempted to classify where proteins are located in a 
bacterial cell. However such protein localization programs are currently least 
accurate when predicting OMPs, and so there is a current need for the 
development of a better OMP classifier. Data mining research suggests that the 
use of frequent patterns has good performance in aiding the development of 
accurate and efficient classification algorithms. In this paper, we present two 
methods to identify OMPs based on frequent subsequences and test them on all 
Gram-negative bacterial proteins whose localizations have been determined by 
biological experiments. One classifier follows an association rule approach, 
while the other is based on support vector machines (SVMs). We compare the 
proposed methods with the state-of-the-art methods in the biological domain. 
The results demonstrate that our methods are better both in terms of accurately 
identifying OMPs and providing biological insights that increase our 
understanding of the structures and functions of these important proteins. 
</blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Serra2003Development]  
J.R. Serra, E.D. Thompson, and P.C. Jurs. Development of binary classification 
of structural chromosome aberrations for a diverse set of organic compounds 
from molecular structure.<em>Chem. Res. Toxicol.</em>, 16(2):153-163, 2003. 
[&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  Classification models are generated to predict in vitro 
cytogenetic results for a diverse set of 383 organic compounds. Both k-nearest 
neighbor and support vector machine models are developed. They are based on 
calculated molecular structure descriptors. Endpoints used are the labels 
clastogenic or nonclastogenic according to an in vitro chromosomal aberration 
assay with Chinese hamster lung cells. Compounds that were tested with both a 
24 and 48 h exposure are included. Each compound is represented by calculated 
molecular structure descriptors encoding the topological, electronic, 
geometrical, or polar surface area aspects of the structure. Subsets of 
informative descriptors are identified with genetic algorithm feature selection 
coupled to the appropriate classification algorithm. The overall classification 
success rate for a k-nearest neighbor classifier built with just six 
topological descriptors is 81.2 and 86.5 success rate for a three-descriptor 
support vector machine model is 99.7 and 83.8% for an external prediction set. 
</blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Segal2003Classificationa]  
N.&nbsp;H. Segal, P.&nbsp;Pavlidis, C.&nbsp;R. Antonescu, R.&nbsp;G. Maki, 
W.&nbsp;S. Noble, D.&nbsp;DeSantis, J.&nbsp;M. Woodruff, J.&nbsp;J. Lewis, 
M.&nbsp;F. Brennan, A.&nbsp;N. Houghton, and C.&nbsp;Cordon-Cardo. 
Classification and Subtype Prediction of Adult Soft Tissue Sarcoma by 
Functional Genomics.<em>Am. J. Pathol.</em>, 163(2):691-700, Aug 2003. [&nbsp;
bib&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  Adult soft tissue sarcomas are a heterogeneous group of tumors, 
including well-described subtypes by histological and genotypic criteria, and 
pleomorphic tumors typically characterized by non-recurrent genetic aberrations 
and karyotypic heterogeneity. The latter pose a diagnostic challenge, even to 
experienced pathologists. We proposed that gene expression profiling in soft 
tissue sarcoma would identify a genomic-based classification scheme that is 
useful in diagnosis. RNA samples from 51 pathologically confirmed cases, 
representing nine different histological subtypes of adult soft tissue sarcoma, 
were examined using the Affymetrix U95A GeneChip. Statistical tests were 
performed on experimental groups identified by cluster analysis, to find 
discriminating genes that could subsequently be applied in a support vector 
machine algorithm. Synovial sarcomas, round-cell/myxoid liposarcomas, 
clear-cell sarcomas and gastrointestinal stromal tumors displayed remarkably 
distinct and homogenous gene expression profiles. Pleomorphic tumors were 
heterogeneous. Notably, a subset of malignant fibrous histiocytomas, a 
controversialhistological subtype, was identified as a distinct genomic group. 
The support vector machine algorithm supported a genomic basis for diagnosis, 
with both high sensitivity and specificity. In conclusion, we showed gene 
expression profiling to be useful in classification and diagnosis, providing 
insights into pathogenesis and pointing to potential new therapeutic targets of 
soft tissue sarcoma. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Segal2003Regression]  
M.&nbsp;R. Segal, K.&nbsp;D. Dahlquist, and B.&nbsp;R. Conklin. Regression 
approaches for microarray data analysis.<em>J. Comput. Biol.</em>, 
10(6):961-980, 2003. [&nbsp;bib&nbsp;| DOI&nbsp;| .pdf&nbsp;] 
<blockquote>  A variety of new procedures have been devised to handle the 
two-sample comparison (e.g., tumor versus normal tissue) of gene expression 
values as measured with microarrays. Such new methods are required in part 
because of some defining characteristics of microarray-based studies: (i) the 
very large number of genes contributing expression measures which far exceeds 
the number of samples (observations) available and (ii) the fact that by virtue 
of pathway/network relationships, the gene expression measures tend to be 
highly correlated. These concerns are exacerbated in the regression setting, 
where the objective is to relate gene expression, simultaneously for multiple 
genes, to some external outcome or phenotype. Correspondingly, several methods 
have been recently proposed for addressing these issues. We briefly critique 
some of these methods prior to a detailed evaluation of gene harvesting. This 
reveals that gene harvesting, without additional constraints, can yield 
artifactual solutions. Results obtained employing such constraints motivate the 
use of regularized regression procedures such as the lasso, least angle 
regression, and support vector machines. Model selection and solution 
multiplicity issues are also discussed. The methods are evaluated using a 
microarray-based study of cardiomyopathy in transgenic mice. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Sanchez-Carbayo2003Gene]  
Marta Sanchez-Carbayo, Nicholas&nbsp;D Socci, Juan&nbsp;Jose Lozano, Wentian 
Li, Elizabeth Charytonowicz, Thomas&nbsp;J Belbin, Michael&nbsp;B Prystowsky, 
Angel&nbsp;R Ortiz, Geoffrey Childs, and Carlos Cordon-Cardo. Gene discovery in 
bladder cancer progression using cDNA microarrays.<em>Am. J. Pathol.</em>, 
163(2):505-16, Aug 2003. [&nbsp;bib&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  To identify gene expression changes along progression of bladder 
cancer, we compared the expression profiles of early-stage and advanced bladder 
tumors using cDNA microarrays containing 17,842 known genes and expressed 
sequence tags. The application of bootstrapping techniques to hierarchical 
clustering segregated early-stage and invasive transitional carcinomas into two 
main clusters. Multidimensional analysis confirmed these clusters and more 
importantly, it separated carcinoma in situ from papillary superficial lesions 
and subgroups within early-stage and invasive tumors displaying different 
overall survival. Additionally, it recognized early-stage tumors showing gene 
profiles similar to invasive disease. Different techniques including standard 
t-test, single-gene logistic regression, and support vector machine algorithms 
were applied to identify relevant genes involved in bladder cancer progression. 
Cytokeratin 20, neuropilin-2, p21, and p33ING1 were selected among the top 
ranked molecular targets differentially expressed and validated by 
immunohistochemistry using tissue microarrays (n = 173). Their expression 
patterns were significantly associated with pathological stage, tumor grade, 
and altered retinoblastoma (RB) expression. Moreover, p33ING1 expression levels 
were significantly associated with overall survival. Analysis of the annotation 
of the most significant genes revealed the relevance of critical genes and 
pathways during bladder cancer progression, including the overexpression of 
oncogenic genes such as DEK in superficial tumors or immune response genes such 
as Cd86 antigen in invasive disease. Gene profiling successfully classified 
bladder tumors based on their progression and clinical outcome. The present 
study has identified molecular biomarkers of potential clinical significance 
and critical molecular targets associated with bladder cancer progression. 
</blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Saeys2003Fast]  
Y.&nbsp;Saeys, S.&nbsp;Degroeve, D.&nbsp;Aeyels, Y.&nbsp;Van&nbsp;de Peer, and 
P.&nbsp;Rouze. Fast feature selection using a simple estimation of distribution 
algorithm: a case study on splice site prediction.<em>Bioinformatics</em>, 
19(Suppl. 1):ii179-ii188, 2003. [&nbsp;bib&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  Motivation: Feature subset selection is an important 
preprocessing step for classification. In biology, where structures or 
processes are described by a large number of features, the elimination of 
irrelevant and redundant information in a reasonable amount of time has a 
number of advantages. It enables the classification system to achieve good or 
even better solutions with a restricted subset of features, allows for a faster 
classification, and it helps the human expert focus on a relevant subset of 
features, hence providing useful biological knowledge. Results: We present a 
heuristic method based on Estimation of Distribution Algorithms to select 
relevant subsets of features for splice site prediction in Arabidopsis 
thaliana. We show that this method performs a fast detection of relevant 
feature subsets using the technique of constrained feature subsets. Compared to 
the traditional greedy methods the gain in speed can be up to one order of 
magnitude, with results being comparable or even better than the greedy 
methods. This makes it a very practical solution for classification tasks that 
can be solved using a relatively small amount of discriminative features (or 
feature dependencies), but where the initial set of potential discriminative 
features is rather large. Keywords: Machine Learning, Feature Subset Selection, 
Estimation of Distribution Algorithms, Splice Site Prediction. Contact: 
yvsae@gengenp.rug.ac.be </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Qin2003Kernel]  
J.&nbsp;Qin, D.&nbsp;P. Lewis, and W.&nbsp;S. Noble. Kernel hierarchical gene 
clustering from microarray expression data.<em>Bioinformatics</em>, 
19(16):2097-2104, 2003. [&nbsp;bib&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  Motivation: Unsupervised analysis of microarray gene expression 
data attempts to find biologically significant patterns within a given 
collection of expression measurements. For example, hierarchical clustering can 
be applied to expression profiles of genes across multiple experiments, 
identifying groups of genes that share similiar expression profiles. Previous 
work using the support vector machine supervised learning algorithm with 
microarray data suggests that higher-order features, such as pairwise and 
tertiary correlations across multiple experiments, may provide significant 
benefit in learning to recognize classes of co-expressed genes. Results: We 
describe a generalization of the hierarchical clustering algorithm that 
efficiently incorporates these higher-order features by using a kernel function 
to map the data into a high-dimensional feature space. We then evaluate the 
utility of the kernel hierarchical clustering algorithm using both internal and 
external validation. The experiments demonstrate that the kernel representation 
itself is insufficient to provide improved clustering performance. We conclude 
that mapping gene expression data into a high-dimensional feature space is only 
a good idea when combined with a learning algorithm, such as the support vector 
machine that does not suffer from the curse of dimensionality. Availability: 
Supplementary data at www.cs.columbia.edu/compbio/hiclust. Software source code 
available by request. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Qian2003Prediction]  
J.&nbsp;Qian, J.&nbsp;Lin, N.&nbsp;M. Luscombe, H.&nbsp;Yu, and 
M.&nbsp;Gerstein. Prediction of regulatory networks: genome-wide identification 
of transcription factor targets from gene expression data.<em>Bioinformatics
</em>, 19(15):1917-1926, 2003. [&nbsp;bib&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  Motivation: Defining regulatory networks, linking transcription 
factors (TFs) to their targets, is a central problem in post-genomic biology. 
One might imagine one could readily determine these networks through inspection 
of gene expression data. However, the relationship between the expression 
timecourse of a transcription factor and its target is not obvious (e.g. simple 
correlation over the timecourse), and current analysis methods, such as 
hierarchical clustering, have not been very successful in deciphering them. 
Results: Here we introduce an approach based on support vector machines (SVMs) 
to predict the targets of a transcription factor by identifying subtle 
relationships between their expression profiles. In particular, we used SVMs to 
predict the regulatory targets for 36 transcription factors in the 
Saccharomyces cerevisiae genome based on the microarray expression data from 
many different physiological conditions. We trained and tested our SVM on a 
data set constructed to include a significant number of both positive and 
negative examples, directly addressing data imbalance issues. This was 
non-trivial given that most of the known experimental information is only for 
positives. Overall, we found that 63 confirmed through cross-validation. We 
further assessed the performance of our regulatory network identifications by 
comparing them with the results from two recent genome-wide ChIP-chip 
experiments. Overall, we find the agreement between our results and these 
experiments is comparable to the agreement (albeit low) between the two 
experiments. We find that this network has a delocalized structure with respect 
to chromosomal positioning, with a given transcription factor having targets 
spread fairly uniformly across the genome. Availability: The overall network of 
the relationships is available on the web at 
http://bioinfo.mbb.yale.edu/expression/echipchip </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Pham2003Prediction]  
Tho&nbsp;Hoan Pham, Kenji Satou, and Tu&nbsp;Bao Ho. Prediction and analysis of 
beta-turns in proteins by support vector machine.<em>Genome Inform Ser Workshop 
Genome Inform</em>, 14:196-205, 2003. [&nbsp;bib&nbsp;] 
<blockquote>  Tight turn has long been recognized as one of the three 
important features of proteins after the alpha-helix and beta-sheet. Tight 
turns play an important role in globular proteins from both the structural and 
functional points of view. More than 90% tight turns are beta-turns. Analysis 
and prediction of beta-turns in particular and tight turns in general are very 
useful for the design of new molecules such as drugs, pesticides, and antigens. 
In this paper, we introduce a support vector machine (SVM) approach to 
prediction and analysis of beta-turns. We have investigated two aspects of 
applying SVM to the prediction and analysis of beta-turns. First, we developed 
a new SVM method, called BTSVM, which predicts beta-turns of a protein from its 
sequence. The prediction results on the dataset of 426 non-homologous protein 
chains by sevenfold cross-validation technique showed that our method is 
superior to the other previous methods. Second, we analyzed how amino acid 
positions support (or prevent) the formation of beta-turns based on the 
&quot;multivariable&quot; classification model of a linear SVM. This model is 
more general than the other ones of previous statistical methods. Our analysis 
results are more comprehensive and easier to use than previously published 
analysis results. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Peng2003Molecular]  
S.&nbsp;Peng, Q.&nbsp;Xu, X.B. Ling, X.&nbsp;Peng, W.&nbsp;Du, and 
L.&nbsp;Chen. Molecular classification of cancer types from microarray data 
using the combination of genetic algorithms and support vector machines.<em>
FEBS Lett.</em>, 555(2):358-362, 2003. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| 
.pdf&nbsp;] 
<blockquote>  Simultaneous multiclass classification of tumor types is 
essential for future clinical implementations of microarray-based cancer 
diagnosis. In this study, we have combined genetic algorithms (GAs) and all 
paired support vector machines (SVMs) for multiclass cancer identification. The 
predictive features have been selected through iterative SVMs/GAs, and 
recursive feature elimination post-processing steps, leading to a very compact 
cancer-related predictive gene set. Leave-one-out cross-validations yielded 
accuracies of 87.93 the eight-class and 85.19 outperforming the results derived 
from previously published methods. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm microarray </blockquote> <br>
 [
Park2003Prediction]  K.-J. Park and M.&nbsp;Kanehisa. Prediction of protein 
subcellular locations by support vector machines using compositions of amino 
acids and amino acid pairs.<em>Bioinformatics</em>, 19(13):1656-1663, 2003. 
[&nbsp;bib&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  Motivation: The subcellular location of a protein is closely 
correlated to its function. Thus, computational prediction of subcellular 
locations from the amino acid sequence information would help annotation and 
functional prediction of protein coding genes in complete genomes. We have 
developed a method based on support vector machines (SVMs). Results: We 
considered 12 subcellular locations in eukaryotic cells: chloroplast, 
cytoplasm, cytoskeleton, endoplasmic reticulum, extracellular medium, Golgi 
apparatus, lysosome, mitochondrion, nucleus, peroxisome, plasma membrane, and 
vacuole. We constructed a data set of proteins with known locations from the 
SWISS-PROT database. A set of SVMs was trained to predict the subcellular 
location of a given protein based on its amino acid, amino acid pair, and 
gapped amino acid pair compositions. The predictors based on these different 
compositions were then combined using a voting scheme. Results obtained through 
5-fold cross-validation tests showed an improvement in prediction accuracy over 
the algorithm based on the amino acid composition only. This prediction method 
is available via the Internet. Availability: 
http://www.genome.ad.jp/SIT/ploc.html Supplementary information: 
http://web.kuicr.kyoto-u.ac.jp/&nbsp;park/Seqdata/ </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Nguyen2003Multi-class]  
Minh&nbsp;N Nguyen and Jagath&nbsp;C Rajapakse. Multi-class support vector 
machines for protein secondary structure prediction.<em>Genome Inform Ser 
Workshop Genome Inform</em>, 14:218-27, 2003. [&nbsp;bib&nbsp;] 
<blockquote>  The solution of binary classification problems using the Support 
Vector Machine (SVM) method has been well developed. Though multi-class 
classification is typically solved by combining several binary classifiers, 
recently, several multi-class methods that consider all classes at once have 
been proposed. However, these methods require resolving a much larger 
optimization problem and are applicable to small datasets. Three methods based 
on binary classifications: one-against-all (OAA), one-against-one (OAO), and 
directed acyclic graph (DAG), and two approaches for multi-class problem by 
solving one single optimization problem, are implemented to predict protein 
secondary structure. Our experiments indicate that multi-class SVM methods are 
more suitable for protein secondary structure (PSS) prediction than the other 
methods, including binary SVMs, because their capacity to solve an optimization 
problem in one step. Furthermore, in this paper, we argue that it is feasible 
to extend the prediction accuracy by adding a second-stage multi-class SVM to 
capture the contextual information among secondary structural elements and 
thereby further improving the accuracies. We demonstrate that two-stage SVMs 
perform better than single-stage SVM techniques for PSS prediction using two 
datasets and report a maximum accuracy of 79.5%. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Meireles2003Differentially]
 S.I. Meireles, A.F. Carvalho, R.&nbsp;Hirata, A.L. Montagnini, W.K. Martins, 
F.B. Runza, B.S. Stolf, L.&nbsp;Termini, C.E. Neto, R.L. Silva, F.A. Soares, 
E.J. Neves, and L.F. Reis. Differentially expressed genes in gastric tumors 
identified by cDNA array.<em>Cancer Lett.</em>, 190(2):199-211, Feb 2003. 
[&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  Using cDNA fragments from the FAPESP/lICR Cancer Genome Project, 
we constructed a cDNA array having 4512 elements and determined gene expression 
in six normal and six tumor gastric tissues. Using t-statistics, we identified 
80 cDNAs whose expression in normal and tumor samples differed more than 3.5 
sample standard deviations. Using Self-Organizing Map, the expression profile 
of these cDNAs allowed perfect separation of malignant and non-malignant 
samples. Using the supervised learning procedure Support Vector Machine, we 
identified trios of cDNAs that could be used to classify samples as normal or 
tumor, based on single-array analysis. Finally, we identified genes with 
altered linear correlation when their expression in normal and tumor samples 
were compared. Further investigation concerning the function of these genes 
could contribute to the understanding of gastric carcinogenesis and may prove 
useful in molecular diagnostics. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm microarray </blockquote> <br>
 [
McKnight2003Categorization]  Larry McKnight and Padmini Srinivasan. 
Categorization of sentence types in medical abstracts.<em>AMIA Annu Symp Proc
</em>, pages 440-4, 2003. [&nbsp;bib&nbsp;] 
<blockquote>  This study evaluated the use of machine learning techniques in 
the classification of sentence type. 7253 structured abstracts and 204 
unstructured abstracts of Randomized Controlled Trials from MedLINE were parsed 
into sentences and each sentence was labeled as one of four types 
(Introduction, Method, Result, or Conclusion). Support Vector Machine (SVM) and 
Linear Classifier models were generated and evaluated on cross-validated data. 
Treating sentences as a simple &quot;bag of words&quot;, the SVM model had an 
average ROC area of 0.92. Adding a feature of relative sentence location 
improved performance markedly for some models and overall increasing the 
average ROC to 0.95. Linear classifier performance was significantly worse than 
the SVM in all datasets. Using the SVM model trained on structured abstracts to 
predict unstructured abstracts yielded performance similar to that of models 
trained with unstructured abstracts in 3 of the 4 types. We conclude that 
classification of sentence type seems feasible within the domain of RCT's. 
Identification of sentence types may be helpful for providing context to end 
users or other text summarization techniques. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Mattfeldt2003Classification
]  T.&nbsp;Mattfeldt, H.W. Gottfried, H.&nbsp;Wolter, V.&nbsp;Schmidt, H.A. 
Kestler, and J.&nbsp;Mayer. Classification of prostatic carcinoma with 
artificial neural networks using comparative genomic hybridization and 
quantitative stereological data.<em>Pathol. Res. Pract.</em>, 199(12):773-784, 
2003. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;] 
<blockquote>  Staging of prostate cancer is a mainstay of treatment decisions 
and prognostication. In the present study, 50 pT2N0 and 28 pT3N0 prostatic 
adenocarcinomas were characterized by Gleason grading, comparative genomic 
hybridization (CGH), and histological texture analysis based on principles of 
stereology and stochastic geometry. The cases were classified by learning 
vector quantization and support vector machines. The quality of classification 
was tested by cross-validation. Correct prediction of stage from primary tumor 
data was possible with an accuracy of 74-80 of prediction was similar when the 
Gleason score was used as input variable, when stereological data were used, or 
when a combination of CGH data and stereological data was used. The results of 
classification by learning vector quantization were slightly better than those 
by support vector machines. A method is briefly sketched by which training of 
neural networks can be adapted to unequal sample sizes per class. Progression 
from pT2 to pT3 prostate cancer is correlated with complex changes of the 
epithelial cells in terms of volume fraction, of surface area, and of 
second-order stereological properties. Genetically, this progression is 
accompanied by a significant global increase in losses and gains of DNA, and 
specifically by increased numerical aberrations on chromosome arms 1q, 7p, and 
8p. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm, cgh </blockquote> <br>
 [Markowetz2003Support] 
 F.&nbsp;Markowetz, L.&nbsp;Edler, and M.&nbsp;Vingron. Support Vector Machines 
for Protein Fold Class Prediction.<em>Biometrical Journal</em>, 45(3):377-389, 
2003. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  Knowledge of the three-dimensional structure of a protein is 
essential for describing and understanding its function. Today, a large number 
of known protein sequences faces a small number of identified structures. Thus, 
the need arises to predict structure from sequence without using time-consuming 
experimental identification. In this paper the performance of Support Vector 
Machines (SVMs) is compared to Neural Networks and to standard statistical 
classification methods as Discriminant Analysis and Nearest Neighbor 
Classification. We show that SVMs can beat the competing methods on a dataset 
of 268 protein sequences to be classified into a set of 42 fold classes. We 
discuss misclassification with respect to biological function and similarity. 
In a second step we examine the performance of SVMs if the embedding is varied 
from frequencies of single amino acids to frequencies of tripletts of amino 
acids. This work shows that SVM provide a promising alternative to standard 
statistical classification and prediction methods in functional genomics. 
</blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Lu2003Expression]  Y.J. 
Lu, D.&nbsp;Williamson, R.&nbsp;Wang, B.&nbsp;Summersgill, S.&nbsp;Rodriguez, 
S.&nbsp;Rogers, K.&nbsp;Pritchard-Jones, C.&nbsp;Campbell, and J.&nbsp;Shipley. 
Expression profiling targeting chromosomes for tumor classification and 
prediction of clinical behavior.<em>Genes Chromosomes Cancer</em>, 
38(3):207-214, 2003. [&nbsp;bib&nbsp;| DOI&nbsp;| .pdf&nbsp;] 
<blockquote>  Tumors are associated with altered or deregulated gene products 
that affect critical cellular functions. Here we assess the use of a global 
expression profiling technique that identifies chromosome regions corresponding 
to differential gene expression, termed comparative expressed sequence 
hybridization (CESH). CESH analysis was performed on a total of 104 tumors with 
a diagnosis of rhabdomyosarcoma, leiomyosarcoma, prostate cancer, and 
favorable-histology Wilms tumors. Through the use of the chromosome regions 
identified as variables, support vector machine analysis was applied to assess 
classification potential, and feature selection (recursive feature elimination) 
was used to identify the best discriminatory regions. We demonstrate that the 
CESH profiles have characteristic patterns in tumor groups and were also able 
to distinguish subgroups of rhabdomyosarcoma. The overall CESH profiles in 
favorable-histology Wilms tumors were found to correlate with subsequent 
clinical behavior. Classification by use of CESH profiles was shown to be 
similar in performance to previous microarray expression studies and 
highlighted regions for further investigation. We conclude that analysis of 
chromosomal expression profiles can group, subgroup, and even predict clinical 
behavior of tumors to a level of performance similar to that of microarray 
analysis. CESH is independent of selecting sequences for interrogation and is a 
simple, rapid, and widely accessible approach to identify clinically useful 
differential expression. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Liu2003QSAR]  H.&nbsp;X. 
Liu, R.&nbsp;S. Zhang, X.&nbsp;J. Yao, M.&nbsp;C. Liu, Z.&nbsp;D. Hu, and 
B.&nbsp;T. Fan. QSAR study of ethyl 
2-[(3-methyl-2,5-dioxo(3-pyrrolinyl))amino]-4-(trifluoromethyl) 
pyrimidine-5-carboxylate: an inhibitor of AP-1 and NF-kappa B mediated gene 
expression based on support vector machines.<em>J Chem Inf Comput Sci</em>, 
43(4):1288-96, 2003. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  The support vector machine, as a novel type of learning machine, 
for the first time, was used to develop a QSAR model of 57 analogues of ethyl 
2-[(3-methyl-2,5-dioxo(3-pyrrolinyl))amino]-4-(trifluoromethyl)pyrimidine-5-carboxylate 
(EPC), an inhibitor of AP-1 and NF-kappa B mediated gene expression, based on 
calculated quantum chemical parameters. The quantum chemical parameters 
involved in the model are Kier and Hall index (order3) (KHI3), Information 
content (order 0) (IC0), YZ Shadow (YZS) and Max partial charge for an N atom 
(MaxPCN), Min partial charge for an N atom (MinPCN). The mean relative error of 
the training set, the validation set, and the testing set is 1.35%, 1.52%, and 
2.23%, respectively, and the maximum relative error is less than 5.00%. 
</blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Liu2003in-silico]  Huiqing 
Liu, Hao Han, Jinyan Li, and Limsoon Wong. An in-silico method for prediction 
of polyadenylation signals in human sequences.<em>Genome Inform Ser Workshop 
Genome Inform</em>, 14:84-93, 2003. [&nbsp;bib&nbsp;] 
<blockquote>  This paper presents a machine learning method to predict 
polyadenylation signals (PASes) in human DNA and mRNA sequences by analysing 
features around them. This method consists of three sequential steps of feature 
manipulation: generation, selection and integration of features. In the first 
step, new features are generated using k-gram nucleotide acid or amino acid 
patterns. In the second step, a number of important features are selected by an 
entropy-based algorithm. In the third step, support vector machines are 
employed to recognize true PASes from a large number of candidates. Our study 
shows that true PASes in DNA and mRNA sequences can be characterized by 
different features, and also shows that both upstream and downstream sequence 
elements are important for recognizing PASes from DNA sequences. We tested our 
method on several public data sets as well as our own extracted data sets. In 
most cases, we achieved better validation results than those reported 
previously on the same data sets. The important motifs observed are highly 
consistent with those reported in literature. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Lind2003Support]  
P.&nbsp;Lind and T.&nbsp;Maltseva. Support vector machines for the estimation 
of aqueous solubility.<em>J Chem Inf Comput Sci</em>, 43(6):1855-9, 2003. 
[&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  Support Vector Machines (SVMs) are used to estimate aqueous 
solubility of organic compounds. A SVM equipped with a Tanimoto similarity 
kernel estimates solubility with accuracy comparable to results from other 
reported methods where the same data sets have been studied. Complete 
cross-validation on a diverse data set resulted in a root-mean-squared error = 
0.62 and R(2) = 0.88. The data input to the machine is in the form of molecular 
fingerprints. No physical parameters are explicitly involved in calculations. 
</blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm chemoinformatics </blockquote> <br>
 [
Liao2003Combining]  L.&nbsp;Liao and W.S. Noble. Combining Pairwise Sequence 
Similarity and Support Vector Machines for Detecting Remote Protein 
Evolutionary and Structural Relationships.<em>J. Comput. Biol.</em>, 
10(6):857-868, 2003. [&nbsp;bib&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  One key element in understanding the molecular machinery of the 
cell is to understand the structure and function of each protein encoded in the 
genome. A very successful means of inferring the structure or function of a 
previously unannotated protein is via sequence similarity with one or more 
proteins whose structure or function is already known. Toward this end, we 
propose a means of representing proteins using pairwise sequence similarity 
scores. This representation, combined with a discriminative classification 
algorithm known as the support vector machine (SVM), provides a powerful means 
of detecting subtle structural and evolutionary relationships among proteins. 
The algorithm, called SVM-pairwise, when tested on its ability to recognize 
previously unseen families from the SCOP database, yields significantly better 
performance than SVM-Fisher, profile HMMs, and PSI-BLAST. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Leslie2003Mismatch]  
C.&nbsp;Leslie, E.&nbsp;Eskin, J.&nbsp;Weston, and W.S. Noble. Mismatch String 
Kernels for SVM Protein Classification. In Suzanna Becker, Sebastian Thrun, and 
Klaus Obermayer, editors,<em>Advances in Neural Information Processing Systems 
15</em>. MIT Press, 2003. [&nbsp;bib&nbsp;| .pdf&nbsp;| .pdf&nbsp;] 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Lee2003Classification]  
Y.&nbsp;Lee and C.-K. Lee. Classification of multiple cancer types by 
multicategory support vector machines using gene expression data.<em>
Bioinformatics</em>, 19(9):1132-1139, 2003. [&nbsp;bib&nbsp;| http&nbsp;| .pdf
&nbsp;]
<blockquote>  Motivation: High-density DNA microarray measures the activities 
of several thousand genes simultaneously and the gene expression profiles have 
been used for the cancer classification recently. This new approach promises to 
give better therapeutic measurements to cancer patients by diagnosing cancer 
types with improved accuracy. The Support Vector Machine (SVM) is one of the 
classification methods successfully applied to the cancer diagnosis problems. 
However, its optimal extension to more than two classes was not obvious, which 
might impose limitations in its application to multiple tumor types. We briefly 
introduce the Multicategory SVM, which is a recently proposed extension of the 
binary SVM, and apply it to multiclass cancer diagnosis problems Results: Its 
applicability is demonstrated on the leukemia data (Golub et al., 1999) and the 
small round blue cell tumors of childhood data (Khan et al., 2001). Comparable 
classification accuracy shown in the applications and its flexibility render 
the MSVM a viable alternative to other classification methods Supplementary 
Information: http://www.stat.ohio-state.edu/&nbsp;yklee/msvm.html Contact: 
yklee@stat.ohio-state.edu </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Lee2003Discovery]  
Dongkwon Lee, Sang&nbsp;Wook Choi, Myengsoo Kim, Jin&nbsp;Hyun Park, Moonkyu 
Kim, Jungchul Kim, and In-Beum Lee. Discovery of differentially expressed genes 
related to histological subtype of hepatocellular carcinoma.<em>Biotechnol Prog.
</em>, 19(3):1011-5, 2003. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  Hepatocellular carcinoma (HCC) is one of the most common human 
malignancies in the world. To identify the histological subtype-specific genes 
of HCC, we analyzed the gene expression profile of 10 HCC patients by means of 
cDNA microarray. We proposed a systematic approach for determining the 
discriminatory genes and revealing the biological phenomena of HCC with cDNA 
microarray data. First, normalization of cDNA microarray data was performed to 
reduce or minimize systematic variations. On the basis of the suitably 
normalized data, we identified specific genes involved in histological subtype 
of HCC. Two classification methods, Fisher's discriminant analysis (FDA) and 
support vector machine (SVM), were used to evaluate the reliability of the 
selected genes and discriminate the histological subtypes of HCC. This study 
may provide a clue for the needs of different chemotherapy and the reason for 
heterogeneity of the clinical responses according to histological subtypes. 
</blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Krishnan2003comparative]  
V.&nbsp;G. Krishnan and D.&nbsp;R. Westhead. A comparative study of 
machine-learning methods to predict the effects of single nucleotide 
polymorphisms on protein function.<em>Bioinformatics</em>, 19(17):2199-2209, 
2003. [&nbsp;bib&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  Motivation: The large volume of single nucleotide polymorphism 
data now available motivates the development of methods for distinguishing 
neutral changes from those which have real biological effects. Here, two 
different machine-learning methods, decision trees and support vector machines 
(SVMs), are applied for the first time to this problem. In common with most 
other methods, only non-synonymous changes in protein coding regions of the 
genome are considered. Results: In detailed cross-validation analysis, both 
learning methods are shown to compete well with existing methods, and to 
out-perform them in some key tests. SVMs show better generalization 
performance, but decision trees have the advantage of generating interpretable 
rules with robust estimates of prediction confidence. It is shown that the 
inclusion of protein structure information produces more accurate methods, in 
agreement with other recent studies, and the effect of using predicted rather 
than actual structure is evaluated. Availability: Software is available on 
request from the authors. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Kim2003Protein]  
H.&nbsp;Kim and H.&nbsp;Park. Protein secondary structure prediction based on 
an improved support vector machines approach.<em>Protein Eng.</em>, 
16(8):553-560, Aug 2003. [&nbsp;bib&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  The prediction of protein secondary structure is an important 
step in the prediction of protein tertiary structure. A new protein secondary 
structure prediction method, SVMpsi, was developed to improve the current level 
of prediction by incorporating new tertiary classifiers and their jury decision 
system, and the PSI-BLAST PSSM profiles. Additionally, efficient methods to 
handle unbalanced data and a new optimization strategy for maximizing the Q3 
measure were developed. The SVMpsi produces the highest published Q3 and SOV94 
scores on both the RS126 and CB513 data sets to date. For a new KP480 set, the 
prediction accuracy of SVMpsi was Q3 = 78.5 for 136 non-redundant protein 
sequences which do not contain homologues of training data sets were Q3 = 77.2 
SVMpsi results in CASP5 illustrate that it is another competitive method to 
predict protein secondary structure. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Kashima2003Marginalized]  
H.&nbsp;Kashima, K.&nbsp;Tsuda, and A.&nbsp;Inokuchi. Marginalized Kernels 
between Labeled Graphs. In T.&nbsp;Faucett and N.&nbsp;Mishra, editors,<em>
Proceedings of the Twentieth International Conference on Machine Learning</em>, 
pages 321-328, New York, NY, USA, 2003. AAAI Press. [&nbsp;bib&nbsp;| .pdf
&nbsp;]
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Hou2003Efficient]  
Y.&nbsp;Hou, W.&nbsp;Hsu, M.&nbsp;L. Lee, and C.&nbsp;Bystroff. Efficient 
remote homology detection using local structure.<em>Bioinformatics</em>, 
19(17):2294-2301, 2003. [&nbsp;bib&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  Motivation: The function of an unknown biological sequence can 
often be accurately inferred if we are able to map this unknown sequence to its 
corresponding homologous family. At present, discriminative methods such as 
SVM-Fisher and SVM-pairwise, which combine support vector machine (SVM) and 
sequence similarity, are recognized as the most accurate methods, with 
SVM-pairwise being the most accurate. However, these methods typically encode 
sequence information into their feature vectors and ignore the structure 
information. They are also computationally inefficient. Based on these 
observations, we present an alternative method for SVM-based protein 
classification. Our proposed method, SVM-I-sites, utilizes structure similarity 
for remote homology detection. Result: We run experiments on the Structural 
Classification of Proteins 1.53 data set. The results show that SVM-I-sites is 
more efficient than SVM-pairwise. Further, we find that SVM-I-sites outperforms 
sequence-based methods such as PSI-BLAST, SAM, and SVM-Fisher while achieving a 
comparable performance with SVM-pairwise. Availability: I-sites server is 
accessible through the web at http://www.bioinfo.rpi.edu. Programs are 
available upon request for academics. Licensing agreements are available for 
commercial interests. The framework of encoding local structure into feature 
vector is available upon request. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Gururaja2003Multiple]  
T.&nbsp;Gururaja, W.&nbsp;Li, W.S. Noble, D.G. Payan, and D.C. Anderson. 
Multiple functional categories of proteins identified in an in vitro cellular 
ubiquitin affinity extract using shotgun peptide sequencing.<em>J Proteome Res
</em>, 2(394-404):394-404, 2003. [&nbsp;bib&nbsp;| .pdf&nbsp;] 
<blockquote>  Using endogenous human cellular ubiquitin system enzymes and 
added his-tagged ubiquitin, ATP, and an ATP-regenerating system, we labelled 
cellular proteins with hexahistidine tagged ubiquitin in vitro. Labeling was 
dependent on ATP and the ATP recycling system, on the proteasome inhibitor 
MG132 and the ubiquitin protease inhibitor ubiquitin aldehyde, and was 
inhibited by iodoacetamide. Labeled proteins were affinity extracted in 
quadruplicate and tryptic peptides identifed by 2D capillary LC/MS/MS comb9ined 
with SEQUEST and MEDUSA analyses. Support vector machine analyais of the mass 
spectrometry data allowed prediction of correct matches between mass 
spectrometry data and peptide sequences. Overall, 144 proteins were identified 
by peptides predicted to be correctly sequenced, and 113 were identified by at 
least three peptides or one or two peptides with at least an 80 Identified 
proteins included 22 proteasome subunits or associated proteins, 18 E1, E2 or 
E3 ubiquitin system enzymes or related proteins, and four ubiquitin domain 
proteins. Seventeen directly ubiquitinated proteins or proteins associated with 
the ubiquitin system were identified. Functional clusters of other proteins 
included redox enzymes, proteins associated with endocytosis, cytoskeletal 
proteins, DNA damage or repair related proteins, calcium binding proteins, and 
splicing factor and related proteins, suggesting that in vitro ubiquitination 
is not random, and that these functions may be regulated by the ubiquitin 
system. This map of cellular ubiquitinated proteins and their interacting 
proteins will be useful for further studies of ubiquitin system function. 
</blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Gordon2003Sequence]  
L.&nbsp;Gordon, A.&nbsp;Y. Chervonenkis, A.&nbsp;J. Gammerman, I.&nbsp;A. 
Shahmuradov, and V.&nbsp;V. Solovyev. Sequence alignment kernel for recognition 
of promoter regions.<em>Bioinformatics</em>, 19(15):1964-1971, 2003. [&nbsp;bib
&nbsp;|http&nbsp;| .pdf&nbsp;] 
<blockquote>  In this paper we propose a new method for recognition of 
prokaryotic promoter regions with startpoints of transcription. The method is 
based on Sequence Alignment Kernel, a function reflecting the quantitative 
measure of match between two sequences. This kernel function is further used in 
Dual SVM, which performs the recognition. Several recognition methods have been 
trained and tested on positive data set, consisting of 669 sigma70-promoter 
regions with known transcription startpoints of Escherichia coli and two 
negative data sets of 709 examples each, taken from coding and non-coding 
regions of the same genome. The results show that our method performs well and 
achieves 16.5 data and 18.6 data. Availability:The demo version of our method 
is accessible from our website http://mendel.cs.rhul.ac.uk/ </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Gomez2003Learning]  
S.&nbsp;M. Gomez, W.&nbsp;S. Noble, and A.&nbsp;Rzhetsky. Learning to predict 
protein-protein interactions from protein sequences.<em>Bioinformatics</em>, 
19(15):1875-1881, 2003. [&nbsp;bib&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  In order to understand the molecular machinery of the cell, we 
need to know about the multitude of protein-protein interactions that allow the 
cell to function. High-throughput technologies provide some data about these 
interactions, but so far that data is fairly noisy. Therefore, computational 
techniques for predicting protein-protein interactions could be of significant 
value. One approach to predicting interactions in silico is to produce from 
first principles a detailed model of a candidate interaction. We take an 
alternative approach, employing a relatively simple model that learns 
dynamically from a large collection of data. In this work, we describe an 
attraction-repulsion model, in which the interaction between a pair of proteins 
is represented as the sum of attractive and repulsive forces associated with 
small, domain- or motif-sized features along the length of each protein. The 
model is discriminative, learning simultaneously from known interactions and 
from pairs of proteins that are known (or suspected) not to interact. The model 
is efficient to compute and scales well to very large collections of data. In a 
cross-validated comparison using known yeast interactions, the 
attraction-repulsion method performs better than several competing techniques. 
</blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Furlanello2003Entropy-based
]  C.&nbsp;Furlanello, M.&nbsp;Serafini, S.&nbsp;Merler, and G.&nbsp;Jurman. 
Entropy-based gene ranking without selection bias for the predictive 
classification of microarray data.<em>BMC Bioinformatics</em>, 4(54), 2003. 
[&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  Background We describe the E-RFE method for gene ranking, which 
is useful for the identification of markers in the predictive classification of 
array data. The method supports a practical modeling scheme designed to avoid 
the construction of classification rules based on the selection of too small 
gene subsets (an effect known as the selection bias, in which the estimated 
predictive errors are too optimistic due to testing on samples already 
considered in the feature selection process). Results With E-RFE, we speed up 
the recursive feature elimination (RFE) with SVM classifiers by eliminating 
chunks of uninteresting genes using an entropy measure of the SVM weights 
distribution. An optimal subset of genes is selected according to a two-strata 
model evaluation procedure: modeling is replicated by an external 
stratified-partition resampling scheme, and, within each run, an internal 
K-fold cross-validation is used for E-RFE ranking. Also, the optimal number of 
genes can be estimated according to the saturation of Zipf's law profiles. 
Conclusions Without a decrease of classification accuracy, E-RFE allows a 
speed-up factor of 100 with respect to standard RFE, while improving on 
alternative parametric RFE reduction strategies. Thus, a process for gene 
selection and error estimation is made practical, ensuring control of the 
selection bias, and providing additional diagnostic indicators of gene 
importance. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Donaldson2003PreBIND]  
I.&nbsp;Donaldson, J.&nbsp;Martin, B.&nbsp;de&nbsp;Bruijn, C.&nbsp;Wolting, 
V.&nbsp;Lay, B.&nbsp;Tuekam, S.&nbsp;Zhang, B.&nbsp;Baskin, G.D. Bader, 
K.&nbsp;Michalickova, T.&nbsp;Pawson, and C.W.V. Hogue. PreBIND and Textomy - 
mining the biomedical literature for protein-protein interactions using a 
support vector machine.<em>BMC Bioinformatics</em>, 4(1):11, Mar 2003. [&nbsp;
bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  Background The majority of experimentally verified molecular 
interaction and biological pathway data are present in the unstructured text of 
biomedical journal articles where they are inaccessible to computational 
methods. The Biomolecular interaction network database (BIND) seeks to capture 
these data in a machine-readable format. We hypothesized that the formidable 
task-size of backfilling the database could be reduced by using Support Vector 
Machine technology to first locate interaction information in the literature. 
We present an information extraction system that was designed to locate 
protein-protein interaction data in the literature and present these data to 
curators and the public for review and entry into BIND. Results 
Cross-validation estimated the support vector machine's test-set precision, 
accuracy and recall for classifying abstracts describing interaction 
information was 92 would be able to recall up to 60 present in another 
yeast-protein interaction database. Finally, this system was applied to a 
real-world curation problem and its use was found to reduce the task duration 
by 70 days. Conclusions Machine learning methods are useful as tools to direct 
interaction and pathway database back-filling; however, this potential can only 
be realized if these techniques are coupled with human review and entry into a 
factual database such as BIND. The PreBIND system described here is available 
to the public at http://bind.ca. Current capabilities allow searching for 
human, mouse and yeast protein-interaction information. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Dobson2003Distinguishing]  
P.D. Dobson and A.J. Doig. Distinguishing enzyme structures from non-enzymes 
without alignments.<em>J. Mol. Biol.</em>, 330(4):771-783, 2003. [&nbsp;bib
&nbsp;|DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  The ability to predict protein function from structure is 
becoming increasingly important as the number of structures resolved is growing 
more rapidly than our capacity to study function. Current methods for 
predicting protein function are mostly reliant on identifying a similar protein 
of known function. For proteins that are highly dissimilar or are only similar 
to proteins also lacking functional annotations, these methods fail. Here, we 
show that protein function can be predicted as enzymatic or not without 
resorting to alignments. We describe 1178 high-resolution proteins in a 
structurally non-redundant subset of the Protein Data Bank using simple 
features such as secondary-structure content, amino acid propensities, surface 
properties and ligands. The subset is split into two functional groupings, 
enzymes and non-enzymes. We use the support vector machine-learning algorithm 
to develop models that are capable of assigning the protein class. Validation 
of the method shows that the function can be predicted to an accuracy of 77 
protein. An adaptive search of possible subsets of features produces a 
simplified model based on 36 features that predicts at an accuracy of 80 avoid 
calculating alignments and predict a recently released set of unrelated 
proteins. The most useful features for distinguishing enzymes from non-enzymes 
are secondary-structure content, amino acid frequencies, number of disulphide 
bonds and size of the largest cleft. This method is applicable to any structure 
as it does not require the identification of sequence or structural similarity 
to a protein of known function. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Cai2003Support]  Y.-D. 
Cai, G.-P. Zhou, and K.-C. Chou. Support Vector Machines for Predicting 
Membrane Protein Types by Using Functional Domain Composition.<em>Biophys. J.
</em>, 84(5):3257-3263, 2003. [&nbsp;bib&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  Membrane proteins are generally classified into the following 
five types: 1), type I membrane protein; 2), type II membrane protein; 3), 
multipass transmembrane proteins; 4), lipid chain-anchored membrane proteins; 
and 5), GPI-anchored membrane proteins. In this article, based on the concept 
of using the functional domain composition to define a protein, the Support 
Vector Machine algorithm is developed for predicting the membrane protein type. 
High success rates are obtained by both the self-consistency and jackknife 
tests. The current approach, complemented with the powerful covariant 
discriminant algorithm based on the pseudo-amino acid composition that has 
incorporated quasi-sequence-order effect as recently proposed by K. C. Chou 
(2001), may become a very useful high-throughput tool in the area of 
bioinformatics and proteomics. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Cai2003Supportb]  Y.D. 
Cai, X.J. Liu, X.B. Xu, and K.C. Chou. Support vector machines for prediction 
of protein domain structural class.<em>J. Theor. Biol.</em>, 221(1):115-120, 
2003. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  The support vector machines (SVMs) method was introduced for 
predicting the structural class of protein domains. The results obtained 
through the self-consistency test, jack-knife test, and independent dataset 
test have indicated that the current method and the elegant component-coupled 
algorithm developed by Chou and co-workers, if effectively complemented with 
each other, may become a powerful tool for predicting the structural class of 
protein domains. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Cai2003Prediction]  Y.D. 
Cai, X.J. Liu, Y.X. Li, X.B. Xu, and K.C. Chou. Prediction of beta-turns with 
learning machines.<em>Peptides</em>, 24(5):665-669, 2003. [&nbsp;bib&nbsp;| DOI
&nbsp;|http&nbsp;| .pdf&nbsp;] 
<blockquote>  The support vector machine approach was introduced to predict 
the beta-turns in proteins. The overall self-consistency rate by the 
re-substitution test for the training or learning dataset reached 100 were 
taken from Chou [J. Pept. Res. 49 (1997) 120]. The success prediction rates by 
the jackknife test for the beta-turn subset of 455 tetrapeptides and 
non-beta-turn subset of 3807 tetrapeptides in the training dataset were 58.1 
and 98.4 success rates with the independent dataset test for the beta-turn 
subset of 110 tetrapeptides and non-beta-turn subset of 30,231 tetrapeptides 
were 69.1 and 97.3 study support the conclusion that the residue-coupled effect 
along a tetrapeptide is important for the formation of a beta-turn. 
</blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Cai2003Supporta]  Y.D. 
Cai, S.L. Lin, and K.C. Chou. Support vector machines for prediction of protein 
signal sequences and their cleavage sites.<em>Peptides</em>, 24(1):159-161, 
2003. [&nbsp;bib&nbsp;| DOI&nbsp;| .pdf&nbsp;] 
<blockquote>  Given a nascent protein sequence, how can one predict its signal 
peptide or &quot;Zipcode&quot; sequence? This is an important problem for 
scientists to use signal peptides as a vehicle to find new drugs or to 
reprogram cells for gene therapy (see, e.g. [7] K.C. Chou, Current Protein and 
Peptide Science 2002;3:615?22). In this paper, support vector machines (SVMs), 
a new machine learning method, is applied to approach this problem. The overall 
rate of correct prediction for 1939 secretary proteins and 1440 nonsecretary 
proteins was over 91 may also serve as a useful tool for further investigating 
many unclear details regarding the molecular mechanism of the ZIP code 
protein-sorting system in cells. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Cai2003Supportd]  Y.D. Cai 
and S.L. Lin. Support vector machines for predicting rRNA-, RNA-, and 
DNA-binding proteins from amino acid sequence.<em>Biochim. Biophys. Acta</em>, 
1648(1-2):127-133, 2003. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  Classification of gene function remains one of the most 
important and demanding tasks in the post-genome era. Most of the current 
predictive computer methods rely on comparing features that are essentially 
linear to the protein sequence. However, features of a protein nonlinear to the 
sequence may also be predictive to its function. Machine learning methods, for 
instance the Support Vector Machines (SVMs), are particularly suitable for 
exploiting such features. In this work we introduce SVM and the pseudo-amino 
acid composition, a collection of nonlinear features extractable from protein 
sequence, to the field of protein function prediction. We have developed 
prototype SVMs for binary classification of rRNA-, RNA-, and DNA-binding 
proteins. Using a protein's amino acid composition and limited range 
correlation of hydrophobicity and solvent accessible surface area as input, 
each of the SVMs predicts whether the protein belongs to one of the three 
classes. In self-consistency and cross-validation tests, which measures the 
success of learning and prediction, respectively, the rRNA-binding SVM has 
consistently achieved &gt;95 The RNA- and DNA-binding SVMs demonstrate more 
diverse accuracy, ranging from approximately 76 the test results suggests the 
directions of improving the SVMs. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Cai2003Supportc]  Y.D. 
Cai, K.Y. Feng, Y.X. Li, and K.C. Chou. Support vector machine for predicting 
alpha-turn types.<em>Peptides</em>, 24(4):629-630, 2003. [&nbsp;bib&nbsp;| DOI
&nbsp;|http&nbsp;| .pdf&nbsp;] 
<blockquote>  Tight turns play an important role in globular proteins from 
both the structural and functional points of view. Of tight turns, beta-turns 
and gamma-turns have been extensively studied, but alpha-turns were little 
investigated. Recently, a systematic search for alpha-turns classified 
alpha-turns into nine different types according to their backbone trajectory 
features. In this paper, Support Vector Machines (SVMs), a new machine learning 
method, is proposed for predicting the alpha-turn types in proteins. The high 
rates of correct prediction imply that that the formation of different 
alpha-turn types is evidently correlated with the sequence of a pentapeptide, 
and hence can be approximately predicted based on the sequence information of 
the pentapeptide alone, although the incorporation of its interaction with the 
other part of a protein, the so-called &quot;long distance interaction&quot;, 
will further improve the prediction quality. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Cai2003SVM-Prot]  
C.&nbsp;Z. Cai, L.&nbsp;Y. Han, Z.&nbsp;L. Ji, X.&nbsp;Chen, and Y.&nbsp;Z. 
Chen. SVM-Prot: Web-based support vector machine software for functional 
classification of a protein from its primary sequence.<em>Nucleic Acids Res</em>
, 31(13):3692-7, Jul 2003. [&nbsp;bib&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  Prediction of protein function is of significance in studying 
biological processes. One approach for function prediction is to classify a 
protein into functional family. Support vector machine (SVM) is a useful method 
for such classification, which may involve proteins with diverse sequence 
distribution. We have developed a web-based software, SVMProt, for SVM 
classification of a protein into functional family from its primary sequence. 
SVMProt classification system is trained from representative proteins of a 
number of functional families and seed proteins of Pfam curated protein 
families. It currently covers 54 functional families and additional families 
will be added in the near future. The computed accuracy for protein family 
classification is found to be in the range of 69.1-99.6%. SVMProt shows a 
certain degree of capability for the classification of distantly related 
proteins and homologous proteins of different function and thus may be used as 
a protein function prediction tool that complements sequence alignment methods. 
SVMProt can be accessed at http://jing.cz3.nus.edu.sg/cgi-bin/svmprot.cgi. 
</blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Cai2003Protein]  C.Z. Cai, 
W.L. Wang, L.Z. Sun, and Y.Z. Chen. Protein function classification via support 
vector machine approach.<em>Math. Biosci.</em>, 185(2):111-122, 2003. [&nbsp;bib
&nbsp;|DOI&nbsp;| .pdf&nbsp;] 
<blockquote>  Support vector machine (SVM) is introduced as a method for the 
classification of proteins into functionally distinguished classes. Studies are 
conducted on a number of protein classes including RNA-binding proteins; 
protein homodimers, proteins responsible for drug absorption, proteins involved 
in drug distribution and excretion, and drug metabolizing enzymes. Testing 
accuracy for the classification of these protein classes is found to be in the 
range of 84-96 usefulness of SVM in the classification of protein functional 
classes and its potential application in protein function prediction. 
</blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Byvatov2003Support]  
E.&nbsp;Byvatov and G.&nbsp;Schneider. Support vector machine applications in 
bioinformatics.<em>Appl Bioinformatics</em>, 2(2):67-77, 2003. [&nbsp;bib&nbsp;]
<blockquote>  The support vector machine (SVM) approach represents a 
data-driven method for solving classification tasks. It has been shown to 
produce lower prediction error compared to classifiers based on other methods 
like artificial neural networks, especially when large numbers of features are 
considered for sample description. In this review, the theory and main 
principles of the SVM approach are outlined, and successful applications in 
traditional areas of bioinformatics research are described. Current 
developments in techniques related to the SVM approach are reviewed which might 
become relevant for future functional genomics and chemogenomics projects. In a 
comparative study, we developed neural network and SVM models to identify small 
organic molecules that potentially modulate the function of G-protein coupled 
receptors. The SVM system was able to correctly classify approximately 90% of 
the compounds in a cross-validation study yielding a Matthews correlation 
coefficient of 0.78. This classifier can be used for fast filtering of compound 
libraries in virtual screening applications. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Byvatov2003Comparison]  
E.&nbsp;Byvatov, U.&nbsp;Fechner, J.&nbsp;Sadowski, and G.&nbsp;Schneider. 
Comparison of support vector machine and artificial neural network systems for 
drug/nondrug classification.<em>J Chem Inf Comput Sci</em>, 43(6):1882-9, 2003. 
[&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  Support vector machine (SVM) and artificial neural network (ANN) 
systems were applied to a drug/nondrug classification problem as an example of 
binary decision problems in early-phase virtual compound filtering and 
screening. The results indicate that solutions obtained by SVM training seem to 
be more robust with a smaller standard error compared to ANN training. 
Generally, the SVM classifier yielded slightly higher prediction accuracy than 
ANN, irrespective of the type of descriptors used for molecule encoding, the 
size of the training data sets, and the algorithm employed for neural network 
training. The performance was compared using various different descriptor sets 
and descriptor combinations based on the 120 standard Ghose-Crippen fragment 
descriptors, a wide range of 180 different properties and physicochemical 
descriptors from the Molecular Operating Environment (MOE) package, and 225 
topological pharmacophore (CATS) descriptors. For the complete set of 525 
descriptors cross-validated classification by SVM yielded 82% correct 
predictions (Matthews cc = 0.63), whereas ANN reached 80% correct predictions 
(Matthews cc = 0.58). Although SVM outperformed the ANN classifiers with regard 
to overall prediction accuracy, both methods were shown to complement each 
other, as the sets of true positives, false positives (overprediction), true 
negatives, and false negatives (underprediction) produced by the two 
classifiers were not identical. The theory of SVM and ANN training is briefly 
reviewed. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm chemoinformatics </blockquote> <br>
 [
Bock2003Whole-proteome]  J.&nbsp;R. Bock and D.&nbsp;A. Gough. Whole-proteome 
interaction mining.<em>Bioinformatics</em>, 19(1):125-134, 2003. [&nbsp;bib
&nbsp;|http&nbsp;| .pdf&nbsp;] 
<blockquote>  Motivation: A major post-genomic scientific and technological 
pursuit is to describe the functions performed by the proteins encoded by the 
genome. One strategy is to first identify the protein-protein interactions in a 
proteome, then determine pathways and overall structure relating these 
interactions, and finally to statistically infer functional roles of individual 
proteins. Although huge amounts of genomic data are at hand, current 
experimental protein interaction assays must overcome technical problems to 
scale-up for high-throughput analysis. In the meantime, bioinformatics 
approaches may help bridge the information gap required for inference of 
protein function. In this paper, a previously described data mining approach to 
prediction of protein-protein interactions (Bock and Gough, 2001, 
Bioinformatics, 17, 455-460) is extended to interaction mining on a 
proteome-wide scale. An algorithm (the phylogenetic bootstrap) is introduced, 
which suggests traversal of a phenogram, interleaving rounds of computation and 
experiment, to develop a knowledge base of protein interactions in 
genetically-similar organisms. Results: The interaction mining approach was 
demonstrated by building a learning system based on 1,039 experimentally 
validated protein-protein interactions in the human gastric bacterium 
Helicobacter pylori. An estimate of the generalization performance of the 
classifier was derived from 10-fold cross-validation, which indicated expected 
upper bounds on precision of 80 One such organism is the enteric pathogen 
Campylobacter jejuni, in which comprehensive machine learning prediction of all 
possible pairwise protein-protein interactions was performed. The resulting 
network of interactions shares an average protein connectivity characteristic 
in common with previous investigations reported in the literature, offering 
strong evidence supporting the biological feasibility of the hypothesized map. 
For inferences about complete proteomes in which the number of pairwise 
non-interactions is expected to be much larger than the number of actual 
interactions, we anticipate that the sensitivity will remain the same but 
precision may decrease. We present specific biological examples of two 
subnetworks of protein-protein interactions in C. jejuni resulting from the 
application of this approach, including elements of a two-component signal 
transduction systems for thermoregulation, and a ferritin uptake network. 
Contact: dgough@bioeng.ucsd.edu </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Ben-Hur2003Remote]  
A.&nbsp;Ben-Hur and D.&nbsp;Brutlag. Remote homology detection: a motif based 
approach.<em>Bioinformatics</em>, 19(Suppl. 1):i26-i33, 2003. [&nbsp;bib&nbsp;| 
http&nbsp;| .pdf&nbsp;] 
<blockquote>  Motivation: Remote homology detection is the problem of 
detecting homology in cases of low sequence similarity. It is a hard 
computational problem with no approach that works well in all cases. Results: 
We present a method for detecting remote homology that is based on the presence 
of discrete sequence motifs. The motif content of a pair of sequences is used 
to define a similarity that is used as a kernel for a Support Vector Machine 
(SVM) classifier. We test the method on two remote homology detection tasks: 
prediction of a previously unseen SCOP family and prediction of an enzyme class 
given other enzymes that have a similar function on other substrates. We find 
that it performs significantly better than an SVM method that uses BLAST or 
Smith-Waterman similarity scores as features. Availability: The software is 
available from the authors upon request. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Beerenwinkel2003Methods]  
N.&nbsp;Beerenwinkel, T.&nbsp;Lengauer, M.&nbsp;Daumer, R.&nbsp;Kaiser, 
H.&nbsp;Walter, K.&nbsp;Korn, D.&nbsp;Hoffmann, and J.&nbsp;Selbig. Methods for 
optimizing antiviral combination therapies.<em>Bioinformatics</em>, 19(Suppl. 
1):i16-i25, 2003. [&nbsp;bib&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  Motivation: Despite some progress with antiretroviral 
combination therapies, therapeutic success in the management of HIV-infected 
patients is limited. The evolution of drug-resistant genetic variants in 
response to therapy plays a key role in treatment failure and finding a new 
potent drug combination after therapy failure is considered challenging. 
Results: To estimate the activity of a drug combination against a particular 
viral strain, we develop a scoring function whose independent variables 
describe a set of antiviral agents and viral DNA sequences coding for the 
molecular targets of the respective drugs. The construction of this activity 
score involves (1) predicting phenotypic drug resistance from genotypes for 
each drug individually, (2) probabilistic modeling of predicted resistance 
values and integration into a score for drug combinations, and (3) searching 
through the mutational neighborhood of the considered strain in order to 
estimate activity on nearby mutants. For a clinical data set, we determine the 
optimal search depth and show that the scoring scheme is predictive of 
therapeutic outcome. Properties of the activity score and applications are 
discussed. Contact: beerenwinkel@mpi-sb.mpg.de Keywords: HIV, antiretroviral 
therapy, drug resistance, SVM regression, therapy optimization, sequence space 
search. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Anderson2003new]  D.C. 
Anderson, W.&nbsp;Li, D.G. Payan, and W.S. Noble. A new algorithm for the 
evaluation of shotgun peptide sequencing in proteomics: support vector machine 
classification of peptide MS/MS spectra and SEQUEST scores.<em>J Proteome Res
</em>, 2(2):137-146, 2003. [&nbsp;bib&nbsp;| .pdf&nbsp;] 
<blockquote>  Shotgun tandem mass spectrometry-based peptide sequencing using 
programs such as SEQUEST allows high-throughput identification of peptides, 
which in turn allows the identification of corresponding proteins. We have 
applied a machine learning algorithm, called the support vector machine, to 
discriminate between correctly and incorrectly identified peptides using 
SEQUEST output. Each peptide was characterized by SEQUEST-calculated features 
such as delta Cn and Xcorr, measurements such as precursor ion current and 
mass, and additional calculated parameters such as the fraction of matched 
MS/MS peaks. The trained SVM classifier performed significantly better than 
previous cutoff-based methods at separating positive from negative peptides. 
Positive and negative peptides were more readily distinguished in training set 
data acquired on a QTOF, compared to an ion trap mass spectrometer. The use of 
13 features, including four new parameters, significantly improved the 
separation between positive and negative peptides. Use of the support vector 
machine and these additional parameters resulted in a more accurate 
interpretation of peptide MS/MS spectra and is an important step toward 
automated interpretation of peptide tandem mass spectrometry data in proteomics.
</blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm proteomics </blockquote> <br>
 [
Segal2003Classification]  N.&nbsp;H. Segal, P.&nbsp;Pavlidis, W.&nbsp;S. Noble, 
C.&nbsp;R. Antonescu, A.&nbsp;Viale, U.&nbsp;V. Wesley, K.&nbsp;Busam, 
H.&nbsp;Gallardo, D.&nbsp;DeSantis, M.&nbsp;F. Brennan, C.&nbsp;Cordon-Cardo, 
J.&nbsp;D. Wolchok, and A.&nbsp;N. Houghton. Classification of Clear-Cell 
Sarcoma as a Subtype of Melanoma by Genomic Profiling.<em>J. Clin. Oncol.</em>, 
21(9):1775-1781, May 2003. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  Purpose: To develop a genome-based classification scheme for 
clear-cell sarcoma (CCS), also known as melanoma of soft parts (MSP), which 
would have implications for diagnosis and treatment. This tumor displays 
characteristic features of soft tissue sarcoma (STS), including deep soft 
tissue primary location and a characteristic translocation, t(12;22)(q13;q12), 
involving EWS and ATF1 genes. CCS/MSP also has typical melanoma features, 
including immunoreactivity for S100 and HMB45, pigmentation, MITF-M expression, 
and a propensity for regional lymph node metastases. Materials and Methods: RNA 
samples from 21 cell lines and 60 pathologically confirmed cases of STS, 
melanoma, and CCS/MSP were examined using the U95A GeneChip (Affymetrix, Santa 
Clara, CA). Hierarchical cluster analysis, principal component analysis, and 
support vector machine (SVM) analysis exploited genomic correlations within the 
data to classify CCS/MSP. Results: Unsupervised analyses demonstrated a clear 
distinction between STS and melanoma and, furthermore, showed that CCS/MSP 
cluster with the melanomas as a distinct group. A supervised SVM learning 
approach further validated this finding and provided a user-independent 
approach to diagnosis. Genes of interest that discriminate CCS/MSP included 
those encoding melanocyte differentiation antigens, MITF, SOX10, ERBB3, and 
FGFR1. Conclusion: Gene expression profiles support the classification of 
CCS/MSP as a distinct genomic subtype of melanoma. Analysis of these gene 
profiles using the SVM may be an important diagnostic tool. Genomic analysis 
identified potential targets for the development of therapeutic strategies in 
the treatment of this disease. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Yuan2004SVMtm]  
Z.&nbsp;Yuan, J.S. Mattick, and R.D. Teasdale. SVMtm: support vector machines 
to predict transmembrane segments.<em>J. Comput. Chem.</em>, 25(5):632, 6 2004. 
[&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  A new method has been developed for prediction of transmembrane 
helices using support vector machines. Different coding schemes of protein 
sequences were explored, and their performances were assessed by 
crossvalidation tests. The best performance method can predict the 
transmembrane helices with sensitivity of 93.4 of 92.0 given to show the 
strength of transmembrane signal and the prediction reliability. In particular, 
this method can distinguish transmembrane proteins from soluble proteins with 
an accuracy of approximately 99 helix prediction methods and can be used for 
consensus analysis of entire proteomes. The predictor is located at 
http://genet.imb.uq.edu.au/predictors/SVMtm. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Yu2004integrated]  J.K. 
Yu, Y.D. Chen, and S.&nbsp;Zheng. An integrated approach to the detection of 
colorectal cancer utilizing proteomics and bioinformatics.<em>World J. 
Gastroenterol.</em>, 10(21):3127-3131, 2004. [&nbsp;bib&nbsp;| .pdf&nbsp;] 
<blockquote>  AIM: To find new potential biomarkers and to establish patterns 
for early detection of colorectal cancer. METHODS: One hundred and eighty-two 
serum samples including 55 from colorectal cancer (CRC) patients, 35 from 
colorectal adenoma (CRA) patients and 92 from healthy persons (HP) were 
detected by surface-enhanced laser desorption/ionization mass spectrometry 
(SELDI-MS). The data of spectra were analyzed by bioinformatics tools like 
artificial neural network (ANN) and support vector machine (SVM). RESULTS: The 
diagnostic pattern combined with 7 potential biomarkers could differentiate CRC 
patients from CRA patients with a specificity of 83 The diagnostic pattern 
combined with 4 potential biomarkers could differentiate CRC patients from HP 
with a specificity of 92 sensitivity of 89 The combination of SELDI with 
bioinformatics tools could help find new biomarkers and establish patterns with 
high sensitivity and specificity for the detection of CRC. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Yu2004Predicting]  C.-S. 
Yu, C.-J. Lin, and J.-K. Hwang. Predicting subcellular localization of proteins 
for Gram-negative bacteria by support vector machines based on n-peptide 
compositions.<em>Protein Sci.</em>, 13(5):1402-1406, 2004. [&nbsp;bib&nbsp;| DOI
&nbsp;|http&nbsp;| .pdf&nbsp;] 
<blockquote>  Gram-negative bacteria have five major subcellular localization 
sites: the cytoplasm, the periplasm, the inner membrane, the outer membrane, 
and the extracellular space. The subcellular location of a protein can provide 
valuable information about its function. With the rapid increase of sequenced 
genomic data, the need for an automated and accurate tool to predict 
subcellular localization becomes increasingly important. We present an approach 
to predict subcellular localization for Gram-negative bacteria. This method 
uses the support vector machines trained by multiple feature vectors based on 
n-peptide compositions. For a standard data set comprising 1443 proteins, the 
overall prediction accuracy reaches 89 highest prediction rate ever reported. 
Our prediction is 14 than that of the recently developed multimodular PSORT-B. 
Because of its simplicity, this approach can be easily extended to other 
organisms and should be a useful tool for the high-throughput and large-scale 
analysis of proteomic and genomic data. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Yao2004Comparative]  
X.&nbsp;J. Yao, A.&nbsp;Panaye, J.&nbsp;P. Doucet, R.&nbsp;S. Zhang, H.&nbsp;F. 
Chen, M.&nbsp;C. Liu, Z.&nbsp;D. Hu, and B.&nbsp;T. Fan. Comparative study of 
QSAR/QSPR correlations using support vector machines, radial basis function 
neural networks, and multiple linear regression.<em>J Chem Inf Comput Sci</em>, 
44(4):1257-66, 2004. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  Support vector machines (SVMs) were used to develop QSAR models 
that correlate molecular structures to their toxicity and bioactivities. The 
performance and predictive ability of SVM are investigated and compared with 
other methods such as multiple linear regression and radial basis function 
neural network methods. In the present study, two different data sets were 
evaluated. The first one involves an application of SVM to the development of a 
QSAR model for the prediction of toxicities of 153 phenols, and the second 
investigation deals with the QSAR model between the structures and the 
activities of a set of 85 cyclooxygenase 2 (COX-2) inhibitors. For each 
application, the molecular structures were described using either the 
physicochemical parameters or molecular descriptors. In both studied cases, the 
predictive ability of the SVM model is comparable or superior to those obtained 
by MLR and RBFNN. The results indicate that SVM can be used as an alternative 
powerful modeling tool for QSAR studies. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm chemoinformatics </blockquote> <br>
 [
Yang2004Bio-support]  Z.&nbsp;R. Yang and K.-C. Chou. Bio-support vector 
machines for computational proteomics.<em>Bioinformatics</em>, 20(5):735-741, 
2004. [&nbsp;bib&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  Motivation: One of the most important issues in computational 
proteomics is to produce a prediction model for the classification or 
annotation of biological function of novel protein sequences. In order to 
improve the prediction accuracy, much attention has been paid to the 
improvement of the performance of the algorithms used, few is for solving the 
fundamental issue, namely, amino acid encoding as most existing pattern 
recognition algorithms are unable to recognize amino acids in protein 
sequences. Importantly, the most commonly used amino acid encoding method has 
the flaw that leads to large computational cost and recognition bias. Results: 
By replacing kernel functions of support vector machines (SVMs) with amino acid 
similarity measurement matrices, we have modified SVMs, a new type of pattern 
recognition algorithm for analysing protein sequences, particularly for 
proteolytic cleavage site prediction. We refer to the modified SVMs as 
bio-support vector machine. When applied to the prediction of HIV protease 
cleavage sites, the new method has shown a remarkable advantage in reducing the 
model complexity and enhancing the model robustness. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Yan2004Identification]  
C.&nbsp;Yan, V.&nbsp;Honavar, and D.&nbsp;Dobbs. Identification of interface 
residues in protease-inhibitor and antigen-antibody complexes: a support vector 
machine.<em>Neural Comput. &amp; Applic.</em>, 13:123-129, 2004. [&nbsp;bib
&nbsp;|DOI&nbsp;| .pdf&nbsp;] 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Yan2004two-stage]  
C.&nbsp;Yan, D.&nbsp;Dobbs, and V.&nbsp;Honavar. A two-stage classifier for 
identification of protein-protein interface residues.<em>Bioinformatics</em>, 
20(Suppl. 1):i371-i378, 2004. [&nbsp;bib&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  Motivation: The ability to identify protein-protein interaction 
sites and to detect specific amino acid residues that contribute to the 
specificity and affinity of protein interactions has important implications for 
problems ranging from rational drug design to analysis of metabolic and signal 
transduction networks. Results: We have developed a two-stage method consisting 
of a support vector machine (SVM) and a Bayesian classifier for predicting 
surface residues of a protein that participate in protein-protein interactions. 
This approach exploits the fact that interface residues tend to form clusters 
in the primary amino acid sequence. Our results show that the proposed 
two-stage classifier outperforms previously published sequence-based methods 
for predicting interface residues. We also present results obtained using the 
two-stage classifier on an independent test set of seven CAPRI (Critical 
Assessment of PRedicted Interactions) targets. The success of the predictions 
is validated by examining the predictions in the context of the 
three-dimensional structures of protein complexes. Supplementary information: 
http://www.public.iastate.edu/&nbsp;chhyan/ISMB2004/list.html </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Yamanishi2004Protein]  
Y.&nbsp;Yamanishi, J.-P. Vert, and M.&nbsp;Kanehisa. Protein network inference 
from multiple genomic data: a supervised approach.<em>Bioinformatics</em>, 
20:i363-i370, 2004. [&nbsp;bib&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  Motivation: An increasing number of observations support the 
hypothesis that most biological functions involve the interactions between many 
proteins, and that the complexity of living systems arises as a result of such 
interactions. In this context, the problem of inferring a global protein 
network for a given organism, using all available genomic data about the 
organism, is quickly becoming one of the main challenges in current 
computational biology. Results: This paper presents a new method to infer 
protein networks from multiple types of genomic data. Based on a variant of 
kernel canonical correlation analysis, its originality is in the formalization 
of the protein network inference problem as a supervised learning problem, and 
in the integration of heterogeneous genomic data within this framework. We 
present promising results on the prediction of the protein network for the 
yeast Saccharomyces cerevisiae from four types of widely available data: gene 
expressions, protein interactions measured by yeast two-hybrid systems, protein 
localizations in the cell and protein phylogenetic profiles. The method is 
shown to outperform other unsupervised protein network inference methods. We 
finally conduct a comprehensive prediction of the protein network for all 
proteins of the yeast, which enables us to propose protein candidates for 
missing enzymes in a biosynthesis pathway. Availability: Softwares are 
available upon request. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Yamanishi2004Heterogeneous]
 Y.&nbsp;Yamanishi, J.-P. Vert, and M.&nbsp;Kanehisa. Heterogeneous data 
comparison and gene selection with kernel canonical correlation analysis. In 
B.&nbsp;Sch&ouml;lkopf, K.&nbsp;Tsuda, and J.P. Vert, editors,<em>Kernel 
Methods in Computational Biology</em>, pages 209-230. MIT Press, 2004. [&nbsp;
bib&nbsp;| www:&nbsp;] 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Xue2004Prediction]  
Y.&nbsp;Xue, C.&nbsp;W. Yap, L.&nbsp;Z. Sun, Z.&nbsp;W. Cao, J.&nbsp;F. Wang, 
and Y.&nbsp;Z. Chen. Prediction of P-glycoprotein substrates by a support 
vector machine approach.<em>J Chem Inf Comput Sci</em>, 44(4):1497-505, 2004. 
[&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  P-glycoproteins (P-gp) actively transport a wide variety of 
chemicals out of cells and function as drug efflux pumps that mediate multidrug 
resistance and limit the efficacy of many drugs. Methods for facilitating early 
elimination of potential P-gp substrates are useful for facilitating new drug 
discovery. A computational ensemble pharmacophore model has recently been used 
for the prediction of P-gp substrates with a promising accuracy of 63%. It is 
desirable to extend the prediction range beyond compounds covered by the known 
pharmacophore models. For such a purpose, a machine learning method, support 
vector machine (SVM), was explored for the prediction of P-gp substrates. A set 
of 201 chemical compounds, including 116 substrates and 85 nonsubstrates of 
P-gp, was used to train and test a SVM classification system. This SVM system 
gave a prediction accuracy of at least 81.2% for P-gp substrates based on two 
different evaluation methods, which is substantially improved against that 
obtained from the multiple-pharmacophore model. The prediction accuracy for 
nonsubstrates of P-gp is 79.2% using 5-fold cross-validation. These accuracies 
are slightly better than those obtained from other statistical classification 
methods, including k-nearest neighbor (k-NN), probabilistic neural networks 
(PNN), and C4.5 decision tree, that use the same sets of data and molecular 
descriptors. Our study indicates the potential of SVM in facilitating the 
prediction of P-gp substrates. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Xue2004Effect]  
Y.&nbsp;Xue, Z.&nbsp;R. Li, C.&nbsp;W. Yap, L.&nbsp;Z. Sun, X.&nbsp;Chen, and 
Y.&nbsp;Z. Chen. Effect of molecular descriptor feature selection in support 
vector machine classification of pharmacokinetic and toxicological properties 
of chemical agents.<em>J Chem Inf Comput Sci</em>, 44(5):1630-8, 2004. [&nbsp;
bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  Statistical-learning methods have been developed for 
facilitating the prediction of pharmacokinetic and toxicological properties of 
chemical agents. These methods employ a variety of molecular descriptors to 
characterize structural and physicochemical properties of molecules. Some of 
these descriptors are specifically designed for the study of a particular type 
of properties or agents, and their use for other properties or agents might 
generate noise and affect the prediction accuracy of a statistical learning 
system. This work examines to what extent the reduction of this noise can 
improve the prediction accuracy of a statistical learning system. A feature 
selection method, recursive feature elimination (RFE), is used to automatically 
select molecular descriptors for support vector machines (SVM) prediction of 
P-glycoprotein substrates (P-gp), human intestinal absorption of molecules 
(HIA), and agents that cause torsades de pointes (TdP), a rare but serious side 
effect. RFE significantly reduces the number of descriptors for each of these 
properties thereby increasing the computational speed for their classification. 
The SVM prediction accuracies of P-gp and HIA are substantially increased and 
that of TdP remains unchanged by RFE. These prediction accuracies are 
comparable to those of earlier studies derived from a selective set of 
descriptors. Our study suggests that molecular feature selection is useful for 
improving the speed and, in some cases, the accuracy of statistical learning 
methods for the prediction of pharmacokinetic and toxicological properties of 
chemical agents. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Xue2004Study]  C.&nbsp;X. 
Xue, R.&nbsp;S. Zhang, M.&nbsp;C. Liu, Z.&nbsp;D. Hu, and B.&nbsp;T. Fan. Study 
of the quantitative structure-mobility relationship of carboxylic acids in 
capillary electrophoresis based on support vector machines.<em>J Chem Inf 
Comput Sci</em>, 44(3):950-7, 2004. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| 
.pdf&nbsp;] 
<blockquote>  The support vector machines (SVM), as a novel type of learning 
machine, were used to develop a quantitative structure-mobility relationship 
(QSMR) model of 58 aliphatic and aromatic carboxylic acids based on molecular 
descriptors calculated from the structure alone. Multiple linear regression 
(MLR) and radial basis function neural networks (RBFNNs) were also utilized to 
construct the linear and the nonlinear model to compare with the results 
obtained by SVM. The root-mean-square errors in absolute mobility predictions 
for the whole data set given by MLR, RBFNNs, and SVM were 1.530, 1.373, and 
0.888 mobility units (10(-5) cm(2) S(-1) V(-1)), respectively, which indicated 
that the prediction result agrees well with the experimental values of these 
compounds and also revealed the superiority of SVM over MLR and RBFNNs models 
for the prediction of the absolute mobility of carboxylic acids. Moreover, the 
models we proposed could also provide some insight into what structural 
features are related to the absolute mobility of aliphatic and aromatic 
carboxylic acids. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Xue2004QSAR]  C.&nbsp;X. 
Xue, R.&nbsp;S. Zhang, H.&nbsp;X. Liu, X.&nbsp;J. Yao, M.&nbsp;C. Liu, 
Z.&nbsp;D. Hu, and B.&nbsp;T. Fan. QSAR models for the prediction of binding 
affinities to human serum albumin using the heuristic method and a support 
vector machine.<em>J Chem Inf Comput Sci</em>, 44(5):1693-700, 2004. [&nbsp;bib
&nbsp;|DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  The binding affinities to human serum albumin for 94 diverse 
drugs and drug-like compounds were modeled with the descriptors calculated from 
the molecular structure alone using a quantitative structure-activity 
relationship (QSAR) technique. The heuristic method (HM) and support vector 
machine (SVM) were utilized to construct the linear and nonlinear prediction 
models, leading to a good correlation coefficient (R2) of 0.86 and 0.94 and 
root-mean-square errors (rms) of 0.212 and 0.134 albumin drug binding affinity 
units, respectively. Furthermore, the models were evaluated by a 10 compound 
external test set, yielding R2 of 0.71 and 0.89 and rms error of 0.430 and 
0.222. The specific information described by the heuristic linear model could 
give some insights into the factors that are likely to govern the binding 
affinity of the compounds and be used as an aid to the drug design process; 
however, the prediction results of the nonlinear SVM model seem to be better 
than that of the HM. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Xue2004accurate]  
C.&nbsp;X. Xue, R.&nbsp;S. Zhang, H.&nbsp;X. Liu, X.&nbsp;J. Yao, M.&nbsp;C. 
Liu, Z.&nbsp;D. Hu, and B.&nbsp;T. Fan. An accurate QSPR study of O-H bond 
dissociation energy in substituted phenols based on support vector machines.<em>
J Chem Inf Comput Sci</em>, 44(2):669-77, 2004. [&nbsp;bib&nbsp;| DOI&nbsp;| 
http&nbsp;| .pdf&nbsp;] 
<blockquote>  The support vector machine (SVM), as a novel type of learning 
machine, was used to develop a Quantitative Structure-Property Relationship 
(QSPR) model of the O-H bond dissociation energy (BDE) of 78 substituted 
phenols. The six descriptors calculated solely from the molecular structures of 
compounds selected by forward stepwise regression were used as inputs for the 
SVM model. The root-mean-square (rms) errors in BDE predictions for the 
training, test, and overall data sets were 3.808, 3.320, and 3.713 BDE units 
(kJ mol(-1)), respectively. The results obtained by Gaussian-kernel SVM were 
much better than those obtained by multiple linear regression, radial basis 
function neural networks, linear-kernel SVM, and other QSPR approaches. 
</blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Xue2004Support]  
C.&nbsp;X. Xue, R.&nbsp;S. Zhang, H.&nbsp;X. Liu, M.&nbsp;C. Liu, Z.&nbsp;D. 
Hu, and B.&nbsp;T. Fan. Support vector machines-based quantitative 
structure-property relationship for the prediction of heat capacity.<em>J Chem 
Inf Comput Sci</em>, 44(4):1267-74, 2004. [&nbsp;bib&nbsp;| DOI&nbsp;| http
&nbsp;|.pdf&nbsp;] 
<blockquote>  The support vector machine (SVM), as a novel type of learning 
machine, for the first time, was used to develop a Quantitative 
Structure-Property Relationship (QSPR) model of the heat capacity of a diverse 
set of 182 compounds based on the molecular descriptors calculated from the 
structure alone. Multiple linear regression (MLR) and radial basis function 
networks (RBFNNs) were also utilized to construct quantitative linear and 
nonlinear models to compare with the results obtained by SVM. The 
root-mean-square (rms) errors in heat capacity predictions for the whole data 
set given by MLR, RBFNNs, and SVM were 4.648, 4.337, and 2.931 heat capacity 
units, respectively. The prediction results are in good agreement with the 
experimental value of heat capacity; also, the results reveal the superiority 
of the SVM over MLR and RBFNNs models. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Xu2004Molecular]  Xiu-Qin 
Xu, Chon&nbsp;K Leow, Xin Lu, Xuegong Zhang, Jun&nbsp;S Liu, Wing-Hung Wong, 
Arndt Asperger, S&ouml;ren Deininger, and Hon-Chiu&nbsp;Eastwood Leung. 
Molecular classification of liver cirrhosis in a rat model by proteomics and 
bioinformatics.<em>Proteomics</em>, 4(10):3235-45, Oct 2004. [&nbsp;bib&nbsp;| 
DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  Liver cirrhosis is a worldwide health problem. Reliable, 
noninvasive methods for early detection of liver cirrhosis are not available. 
Using a three-step approach, we classified sera from rats with liver cirrhosis 
following different treatment insults. The approach consisted of: (i) protein 
profiling using surface-enhanced laser desorption/ionization (SELDI) 
technology; (ii) selection of a statistically significant serum biomarker set 
using machine learning algorithms; and (iii) identification of selected serum 
biomarkers by peptide sequencing. We generated serum protein profiles from 
three groups of rats: (i) normal (n=8), (ii) thioacetamide-induced liver 
cirrhosis (n=22), and (iii) bile duct ligation-induced liver fibrosis (n=5) 
using a weak cation exchanger surface. Profiling data were further analyzed by 
a recursive support vector machine algorithm to select a panel of statistically 
significant biomarkers for class prediction. Sensitivity and specificity of 
classification using the selected protein marker set were higher than 92%. A 
consistently down-regulated 3495 Da protein in cirrhosis samples was one of the 
selected significant biomarkers. This 3495 Da protein was purified on-chip and 
trypsin digested. Further structural characterization of this biomarkers 
candidate was done by using cross-platform matrix-assisted laser 
desorption/ionization mass spectrometry (MALDI-MS) peptide mass fingerprinting 
(PMF) and matrix-assisted laser desorption/ionization time of flight/time of 
flight (MALDI-TOF/TOF) tandem mass spectrometry (MS/MS). Combined data from PMF 
and MS/MS spectra of two tryptic peptides suggested that this 3495 Da protein 
shared homology to a histidine-rich glycoprotein. These results demonstrated a 
novel approach to discovery of new biomarkers for early detection of liver 
cirrhosis and classification of liver diseases. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Williams2004Prognostic]  
R.D. Williams, S.N. Hing, B.T. Greer, C.C. Whiteford, J.S. Wei, 
R.&nbsp;Natrajan, A.&nbsp;Kelsey, S.&nbsp;Rogers, C.&nbsp;Campbell, 
K.&nbsp;Pritchard-Jones, and J.&nbsp;Khan. Prognostic classification of 
relapsing favorable histology Wilms tumor using cDNA microarray expression 
profiling and support vector machines.<em>Genes Chromosomes Cancer</em>, 
41(1):65-79, Sep 2004. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  Treatment of Wilms tumor has a high success rate, with some 85 
of patients achieving long-term survival. However, late effects of treatment 
and management of relapse remain significant clinical problems. If accurate 
prognostic methods were available, effective risk-adapted therapies could be 
tailored to individual patients at diagnosis. Few molecular prognostic markers 
for Wilms tumor are currently defined, though previous studies have linked 
allele loss on 1p or 16q, genomic gain of 1q, and overexpression from 1q with 
an increased risk of relapse. To identify specific patterns of gene expression 
that are predictive of relapse, we used high-density (30 k) cDNA microarrays to 
analyze RNA samples from 27 favorable histology Wilms tumors taken from primary 
nephrectomies at the time of initial diagnosis. Thirteen of these tumors 
relapsed within 2 years. Genes differentially expressed between the relapsing 
and nonrelapsing tumor classes were identified by statistical scoring (t test). 
These genes encode proteins with diverse molecular functions, including 
transcription factors, developmental regulators, apoptotic factors, and 
signaling molecules. Use of a support vector machine classifier, feature 
selection, and test evaluation using cross-validation led to identification of 
a generalizable expression signature, a small subset of genes whose expression 
potentially can be used to predict tumor outcome in new samples. Similar 
methods were used to identify genes that are differentially expressed between 
tumors with and without genomic 1q gain. This set of discriminators was highly 
enriched in genes on 1q, indicating close agreement between data obtained from 
expression profiling with data from genomic copy number analyses. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Weathers2004Reduced]  
E.&nbsp;A. Weathers, M.&nbsp;E. Paulaitis, T.&nbsp;B. Woolf, and J.&nbsp;H. 
Hoh. Reduced amino acid alphabet is sufficient to accurately recognize 
intrinsically disordered protein.<em>FEBS Lett.</em>, 576(3):348-352, 2004. 
[&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  Intrinsically disordered proteins are an important class of 
proteins with unique functions and properties. Here, we have applied a support 
vector machine (SVM) trained on naturally occurring disordered and ordered 
proteins to examine the contribution of various parameters (vectors) to 
recognizing proteins that contain disordered regions. We find that a SVM that 
incorporates only amino acid composition has a recognition accuracy of 87+/-2 
composition alone is sufficient to accurately recognize disorder. 
Interestingly, SVMs using reduced sets of amino acids based on chemical 
similarity preserve high recognition accuracy. A set as small as four retains 
an accuracy of 84+/-2 general physicochemical properties rather than specific 
amino acids are important factors contributing to protein disorder. 
</blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Waring2004Interlaboratory] 
 Jeffrey&nbsp;F Waring, Roger&nbsp;G Ulrich, Nick Flint, David Morfitt, Arno 
Kalkuhl, Frank Staedtler, Michael Lawton, Johanna&nbsp;M Beekman, and Laura 
Suter. Interlaboratory evaluation of rat hepatic gene expression changes 
induced by methapyrilene.<em>Environ Health Perspect</em>, 112(4):439-48, Mar 
2004. [&nbsp;bib&nbsp;] 
<blockquote>  Several studies using microarrays have shown that changes in 
gene expression provide information about the mechanism of toxicity induced by 
xenobiotic agents. Nevertheless, the issue of whether gene expression profiles 
are reproducible across different laboratories remains to be determined. To 
address this question, several members of the Hepatotoxicity Working Group of 
the International Life Sciences Institute Health and Environmental Sciences 
Institute evaluated the liver gene expression profiles of rats treated with 
methapyrilene (MP). Animals were treated at one facility, and RNA was 
distributed to five different sites for gene expression analysis. A preliminary 
evaluation of the number of modulated genes uncovered striking differences 
between the five different sites. However, additional data analysis 
demonstrated that these differences had an effect on the absolute gene 
expression results but not on the outcome of the study. For all users, 
unsupervised algorithms showed that gene expression allows the distinction of 
the high dose of MP from controls and low dose. In addition, the use of a 
supervised analysis method (support vector machines) made it possible to 
correctly classify samples. In conclusion, the results show that, despite some 
variability, robust gene expression changes were consistent between sites. In 
addition, key expression changes related to the mechanism of MP-induced 
hepatotoxicity were identified. These results provide critical information 
regarding the consistency of microarray results across different laboratories 
and shed light on the strengths and limitations of expression profiling in drug 
safety analysis. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Wang2004Support]  M-L. 
Wang, W-J. Li, M-L. Wang, and W-B. Xu. Support vector machines for prediction 
of peptidyl prolyl cis/trans isomerization.<em>J Pept Res</em>, 63(1):23-8, Jan 
2004. [&nbsp;bib&nbsp;] 
<blockquote>  A new method for peptidyl prolyl cis/trans isomerization 
prediction based on the theory of support vector machines (SVM) was introduced. 
The SVM represents a new approach to supervised pattern classification and has 
been successfully applied to a wide range of pattern recognition problems. In 
this study, six training datasets consisting of different length local sequence 
respectively were used. The polynomial kernel functions with different 
parameter d were chosen. The test for the independent testing dataset and the 
jackknife test were both carried out. When the local sequence length was 
20-residue and the parameter d = 8, the SVM method archived the best 
performance with the correct rate for the cis and trans forms reaching 70.4 and 
69.7% for the independent testing dataset, 76.7 and 76.6% for the jackknife 
test, respectively. Matthew's correlation coefficients for the jackknife test 
could reach about 0.5. The results obtained through this study indicated that 
the SVM method would become a powerful tool for predicting peptidyl prolyl 
cis/trans isomerization. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Wang2004Weighted-support]  
M.&nbsp;Wang, J.&nbsp;Yang, G.-P. Liu, Z.-J. Xu, and K.-C. Chou. 
Weighted-support vector machines for predicting membrane protein types based on 
pseudo-amino acid composition.<em>Protein Eng. Des. Sel.</em>, 17(6):509-516, 
2004. [&nbsp;bib&nbsp;| DOI&nbsp;| arXiv&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  Membrane proteins are generally classified into the following 
five types: (1) type I membrane proteins, (2) type II membrane proteins, (3) 
multipass transmembrane proteins, (4) lipid chain-anchored membrane proteins 
and (5) GPI-anchored membrane proteins. Prediction of membrane protein types 
has become one of the growing hot topics in bioinformatics. Currently, we are 
facing two critical challenges in this area: first, how to take into account 
the extremely complicated sequence-order effects, and second, how to deal with 
the highly uneven sizes of the subsets in a training dataset. In this paper, 
stimulated by the concept of using the pseudo-amino acid composition to 
incorporate the sequence-order effects, the spectral analysis technique is 
introduced to represent the statistical sample of a protein. Based on such a 
framework, the weighted support vector machine (SVM) algorithm is applied. The 
new approach has remarkable power in dealing with the bias caused by the 
situation when one subset in the training dataset contains many more samples 
than the other. The new method is particularly useful when our focus is aimed 
at proteins belonging to small subsets. The results obtained by the 
self-consistency test, jackknife test and independent dataset test are 
encouraging, indicating that the current approach may serve as a powerful 
complementary tool to other existing methods for predicting the types of 
membrane proteins. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Wang2004Predicting]  
Long-Hui Wang, Juan Liu, Yan-Fu Li, and Huai-Bei Zhou. Predicting protein 
secondary structure by a support vector machine based on a new coding scheme.
<em>Genome Inform Ser Workshop Genome Inform</em>, 15(2):181-90, 2004. [&nbsp;
bib&nbsp;| .html&nbsp;] 
<blockquote>  Protein structure prediction is one of the most important 
problems in modern computational biology. Protein secondary structure 
prediction is a key step in prediction of protein tertiary structure. There 
have emerged many methods based on machine learning techniques, such as neural 
networks (NN) and support vector machine (SVM) etc., to focus on the prediction 
of the secondary structures. In this paper, a new method was proposed based on 
SVM. Different from the existing methods, this method takes into account of the 
physical-chemical properties and structure properties of amino acids. When 
tested on the most popular dataset CB513, it achieved a Q(3) accuracy of 
0.7844, which illustrates that it is one of the top range methods for protein 
of secondary structure prediction. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Wagner2004Computational]  
M.&nbsp;Wagner, D.N. Naik, A.&nbsp;Pothen, S.&nbsp;Kasukurti, R.R. Devineni, 
B.L. Adam, O.J. Semmes, and G.L. Wright&nbsp;Jr. Computational protein 
biomarker prediction: a case study for prostate cancer.<em>BMC Bioinformatics
</em>, 5(26), 2004. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  Background Recent technological advances in mass spectrometry 
pose challenges in computational mathematics and statistics to process the mass 
spectral data into predictive models with clinical and biological significance. 
We discuss several classification-based approaches to finding protein biomarker 
candidates using protein profiles obtained via mass spectrometry, and we assess 
their statistical significance. Our overall goal is to implicate peaks that 
have a high likelihood of being biologically linked to a given disease state, 
and thus to narrow the search for biomarker candidates. Results Thorough 
cross-validation studies and randomization tests are performed on a prostate 
cancer dataset with over 300 patients, obtained at the Eastern Virginia Medical 
School using SELDI-TOF mass spectrometry. We obtain average classification 
accuracies of 87 on a four-group classification problem using a two-stage 
linear SVM-based procedure and just 13 peaks, with other methods performing 
comparably. Conclusions Modern feature selection and classification methods are 
powerful techniques for both the identification of biomarker candidates and the 
related problem of building predictive models from protein mass spectrometric 
profiles. Cross-validation and randomization are essential tools that must be 
performed carefully in order not to bias the results unfairly. However, only a 
biological validation and identification of the underlying proteins will 
ultimately confirm the actual value and power of any computational predictions. 
</blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Vinayagam2004Applying]  
A.&nbsp;Vinayagam, R.&nbsp;K&ouml;nig, J.&nbsp;Moormann, F.&nbsp;Schubert, 
R.&nbsp;Eils, K.-H. Glatting, and S.&nbsp;Suhai. Applying Support Vector 
Machines for Gene Ontology based gene function prediction.<em>BMC Bioinformatics
</em>, 5(1):116, Aug 2004. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  BACKGROUND: The current progress in sequencing projects calls 
for rapid, reliable and accurate function assignments of gene products. A 
variety of methods has been designed to annotate sequences on a large scale. 
However, these methods can either only be applied for specific subsets, or 
their results are not formalised, or they do not provide precise confidence 
estimates for their predictions. RESULTS: We have developed a large-scale 
annotation system that tackles all of these shortcomings. In our approach, 
annotation was provided through Gene Ontology terms by applying multiple 
Support Vector Machines (SVM) for the classification of correct and false 
predictions. The general performance of the system was benchmarked with a large 
dataset. An organism-wise cross-validation was performed to define confidence 
estimates, resulting in an average precision of 80% for 74% of all test 
sequences. The validation results show that the prediction performance was 
organism-independent and could reproduce the annotation of other automated 
systems as well as high-quality manual annotations. We applied our trained 
classification system to Xenopus laevis sequences, yielding functional 
annotation for more than half of the known expressed genome. Compared to the 
currently available annotation, we provided more than twice the number of 
contigs with good quality annotation, and additionally we assigned a confidence 
value to each predicted GO term. CONCLUSIONS: We present a complete automated 
annotation system that overcomes many of the usual problems by applying a 
controlled vocabulary of Gene Ontology and an established classification method 
on large and well-described sequence data sets. In a case study, the function 
for Xenopus laevis contig sequences was predicted and the results are publicly 
available at 
ftp://genome.dkfz-heidelberg.de/pub/agd/gene_association.agd_Xenopus. 
</blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Vert2004primer]  J.-P. 
Vert, K.&nbsp;Tsuda, and B.&nbsp;Sch&ouml;lkopf. A primer on kernel methods. In 
B.&nbsp;Sch&ouml;lkopf, K.&nbsp;Tsuda, and J.P. Vert, editors,<em>Kernel 
Methods in Computational Biology</em>, pages 35-70. MIT Press, 2004. [&nbsp;bib
&nbsp;]
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Vert2004Local]  J.-P. 
Vert, H.&nbsp;Saigo, and T.&nbsp;Akutsu. Local alignment kernels for biological 
sequences. In B.&nbsp;Sch&ouml;lkopf, K.&nbsp;Tsuda, and J.P. Vert, editors,<em>
Kernel Methods in Computational Biology</em>, pages 131-154. MIT Press, The MIT 
Press, Cambridge, Massachussetts, 2004. [&nbsp;bib&nbsp;| www:&nbsp;] 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Tsuda2004Learning]  
K.&nbsp;Tsuda and W.S. Noble. Learning kernels from biological networks by 
maximizing entropy.<em>Bioinformatics</em>, 20:i326-i333, 2004. [&nbsp;bib
&nbsp;|DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  Motivation: The diffusion kernel is a general method for 
computing pairwise distances among all nodes in a graph, based on the sum of 
weighted paths between each pair of nodes. This technique has been used 
successfully, in conjunction with kernel-based learning methods, to draw 
inferences from several types of biological networks. Results: We show that 
computing the diffusion kernel is equivalent to maximizing the von Neumann 
entropy, subject to a global constraint on the sum of the Euclidean distances 
between nodes. This global constraint allows for high variance in the pairwise 
distances. Accordingly, we propose an alternative, locally constrained 
diffusion kernel, and we demonstrate that the resulting kernel allows for more 
accurate support vector machine prediction of protein functional 
classifications from metabolic and protein?protein interaction networks. 
Availability: Supplementary results and data are available at 
noble.gs.washington.edu/proj/maxent </blockquote> 
<p></p> 
<blockquote>  Keywords: learning-kernel graph-kernel biosvm </blockquote> <br>

 [Tsai2004Gene]  C.A. Tsai, C.H. Chen, T.C. Lee, I.C. Ho, U.C. Yang, and J.J. 
Chen. Gene selection for sample classifications in microarray experiments.<em>
DNA Cell Biol.</em>, 23(10):607-614, 2004. [&nbsp;bib&nbsp;| DOI&nbsp;| http
&nbsp;|.pdf&nbsp;] 
<blockquote>  DNA microarray technology provides useful tools for profiling 
global gene expression patterns in different cell/tissue samples. One major 
challenge is the large number of genes relative to the number of samples. The 
use of all genes can suppress or reduce the performance of a classification 
rule due to the noise of nondiscriminatory genes. Selection of an optimal 
subset from the original gene set becomes an important prestep in sample 
classification. In this study, we propose a family-wise error (FWE) rate 
approach to selection of discriminatory genes for two-sample or multiple-sample 
classification. The FWE approach controls the probability of the number of one 
or more false positives at a prespecified level. A public colon cancer data set 
is used to evaluate the performance of the proposed approach for the two 
classification methods: k nearest neighbors (k-NN) and support vector machine 
(SVM). The selected gene sets from the proposed procedure appears to perform 
better than or comparable to several results reported in the literature using 
the univariate analysis without performing multivariate search. In addition, we 
apply the FWE approach to a toxicogenomic data set with nine treatments (a 
control and eight metals, As, Cd, Ni, Cr, Sb, Pb, Cu, and AsV) for a total of 
55 samples for a multisample classification. Two gene sets are considered: the 
gene set omegaF formed by the ANOVA F-test, and a gene set omegaT formed by the 
union of one-versus-all t-tests. The predicted accuracies are evaluated using 
the internal and external crossvalidation. Using the SVM classification, the 
overall accuracies to predict 55 samples into one of the nine treatments are 
above 80 crossvalidation. OmegaF has slightly higher accuracy rates than 
omegaT. The overall predicted accuracies are above 70 external crossvalidation; 
the two gene sets omegaT and omegaF performed equally well. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm microarray </blockquote> <br>
 [
Steiner2004Discriminating]  Guido Steiner, Laura Suter, Franziska Boess, 
Rodolfo Gasser, Maria&nbsp;Cristina de&nbsp;Vera, Silvio Albertini, and Stefan 
Ruepp. Discriminating different classes of toxicants by transcript profiling.
<em>Environ. Health Perspect.</em>, 112(12):1236-48, Aug 2004. [&nbsp;bib&nbsp;|
.html&nbsp;| .pdf&nbsp;] 
<blockquote>  Male rats were treated with various model compounds or the 
appropriate vehicle controls. Most substances were either well-known 
hepatotoxicants or showed hepatotoxicity during preclinical testing. The aim of 
the present study was to determine if biological samples from rats treated with 
various compounds can be classified based on gene expression profiles. In 
addition to gene expression analysis using microarrays, a complete serum 
chemistry profile and liver and kidney histopathology were performed. We 
analyzed hepatic gene expression profiles using a supervised learning method 
(support vector machines; SVMs) to generate classification rules and combined 
this with recursive feature elimination to improve classification performance 
and to identify a compact subset of probe sets with potential use as 
biomarkers. Two different SVM algorithms were tested, and the models obtained 
were validated with a compound-based external cross-validation approach. Our 
predictive models were able to discriminate between hepatotoxic and 
nonhepatotoxic compounds. Furthermore, they predicted the correct class of 
hepatotoxicant in most cases. We provide an example showing that a predictive 
model built on transcript profiles from one rat strain can successfully 
classify profiles from another rat strain. In addition, we demonstrate that the 
predictive models identify nonresponders and are able to discriminate between 
gene changes related to pharmacology and toxicity. This work confirms the 
hypothesis that compound classification based on gene expression data is 
feasible. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Statnikov2004Methods]  
Alexander Statnikov, Constantin&nbsp;F Aliferis, and Ioannis Tsamardinos. 
Methods for multi-category cancer diagnosis from gene expression data: a 
comprehensive evaluation to inform decision support system development.<em>
Medinfo</em>, 11(Pt 2):813-7, 2004. [&nbsp;bib&nbsp;] 
<blockquote>  Cancer diagnosis is a major clinical applications area of gene 
expression microarray technology. We are seeking to develop a system for cancer 
diagnostic model creation based on microarray data. In order to equip the 
system with the optimal combination of data modeling methods, we performed a 
comprehensive evaluation of several major classification algorithms, gene 
selection methods, and cross-validation designs using 11 datasets spanning 74 
diagnostic categories (41 cancer types and 12 normal tissue types). The 
Multi-Category Support Vector Machine techniques by Crammer and Singer, Weston 
and Watkins, and one-versus-rest were found to be the best methods and they 
outperform other learning algorithms such as K-Nearest Neighbors and Neural 
Networks often to a remarkable degree. Gene selection techniques are shown to 
significantly improve classification performance. These results guided the 
development of a software system that fully automates cancer diagnostic model 
construction with quality on par with or better than previously published 
results derived by expert human analysts. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Sorich2004Rapid]  
Michael&nbsp;J Sorich, Ross&nbsp;A McKinnon, John&nbsp;O Miners, David&nbsp;A 
Winkler, and Paul&nbsp;A Smith. Rapid prediction of chemical metabolism by 
human UDP-glucuronosyltransferase isoforms using quantum chemical descriptors 
derived with the electronegativity equalization method.<em>J Med Chem</em>, 
47(21):5311-7, Oct 2004. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  This study aimed to evaluate in silico models based on quantum 
chemical (QC) descriptors derived using the electronegativity equalization 
method (EEM) and to assess the use of QC properties to predict chemical 
metabolism by human UDP-glucuronosyltransferase (UGT) isoforms. Various 
EEM-derived QC molecular descriptors were calculated for known UGT substrates 
and nonsubstrates. Classification models were developed using support vector 
machine and partial least squares discriminant analysis. In general, the most 
predictive models were generated with the support vector machine. Combining QC 
and 2D descriptors (from previous work) using a consensus approach resulted in 
a statistically significant improvement in predictivity (to 84%) over both the 
QC and 2D models and the other methods of combining the descriptors. 
EEM-derived QC descriptors were shown to be both highly predictive and 
computationally efficient. It is likely that EEM-derived QC properties will be 
generally useful for predicting ADMET and physicochemical properties during 
drug discovery. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Sen2004Predicting]  T.Z. 
Sen, A.&nbsp;Kloczkowski, R.L. Jernigan, C.&nbsp;Yan, V.&nbsp;Honavar, K.M. Ho, 
C.Z. Wang, Y.&nbsp;Ihm, H.&nbsp;Cao, X.&nbsp;Gu, and D.&nbsp;Dobbs. Predicting 
binding sites of hydrolase-inhibitor complexes by combining several methods.<em>
BMC Bioinformatics</em>, 5(205), 2004. [&nbsp;bib&nbsp;| DOI&nbsp;| .pdf&nbsp;] 
<blockquote>  Background Protein-protein interactions play a critical role in 
protein function. Completion of many genomes is being followed rapidly by major 
efforts to identify interacting protein pairs experimentally in order to 
decipher the networks of interacting, coordinated-in-action proteins. 
Identification of protein-protein interaction sites and detection of specific 
amino acids that contribute to the specificity and the strength of protein 
interactions is an important problem with broad applications ranging from 
rational drug design to the analysis of metabolic and signal transduction 
networks. Results In order to increase the power of predictive methods for 
protein-protein interaction sites, we have developed a consensus methodology 
for combining four different methods. These approaches include: data mining 
using Support Vector Machines, threading through protein structures, prediction 
of conserved residues on the protein surface by analysis of phylogenetic trees, 
and the Conservatism of Conservatism method of Mirny and Shakhnovich. Results 
obtained on a dataset of hydrolase-inhibitor complexes demonstrate that the 
combination of all four methods yield improved predictions over the individual 
methods. Conclusions We developed a consensus method for predicting 
protein-protein interface residues by combining sequence and structure-based 
methods. The success of our consensus approach suggests that similar 
methodologies can be developed to improve prediction accuracies for other 
bioinformatic problems. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Schoelkopf2004Kernel]  
B.&nbsp;Sch&ouml;lkopf, K.&nbsp;Tsuda, and J.-P. Vert.<em>Kernel Methods in 
Computational Biology</em>. MIT Press, The MIT Press, Cambridge, 
Massachussetts, 2004. [&nbsp;bib&nbsp;] 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Schwender2004pilot]  
Holger Schwender, Manuela Zucknick, Katja Ickstadt, Hermann&nbsp;M Bolt, and 
G.&nbsp;E. N. I. C.&nbsp;A. network. A pilot study on the application of 
statistical classification procedures to molecular epidemiological data.<em>
Toxicol Lett</em>, 151(1):291-9, Jun 2004. [&nbsp;bib&nbsp;] 
<blockquote>  The development of new statistical methods for use in molecular 
epidemiology comprises the building and application of appropriate 
classification rules. The aim of this study was to assess various 
classification methods that can potentially handle genetic interactions. A data 
set comprising genotypes at 25 single nucleotide polymorphic (SNP) loci from 
518 breast cancer cases and 586 age-matched population-based controls from the 
GENICA study was used to built a classification rule with the discrimination 
methods SVM (support vector machine), CART (classification and regression 
tree), Bagging, Random Forest, LogitBoost and k nearest neighbours (kNN). A 
blind pilot analysis of the genotypic data set was a first approach to obtain 
an impression of the statistical structure of the data. Furthermore, this 
analysis was performed to explore classification methods that may be applied to 
molecular-epidemiological evaluation. The results showed that all blindly 
applied classification methods had a slightly smaller misclassification rate 
than a random classification. The findings, nevertheless, suggest that SNP data 
might be useful for the classification of individuals into categories of high 
or low risk of diseases. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Saigo2004Protein]  
H.&nbsp;Saigo, J.-P. Vert, N.&nbsp;Ueda, and T.&nbsp;Akutsu. Protein homology 
detection using string alignment kernels.<em>Bioinformatics</em>, 
20(11):1682-1689, 2004. [&nbsp;bib&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  Motivation: Remote homology detection between protein sequences 
is a central problem in computational biology. Discriminative methods involving 
support vector machines (SVMs) are currently the most effective methods for the 
problem of superfamily recognition in the Structural Classification Of Proteins 
(SCOP) database. The performance of SVMs depends critically on the kernel 
function used to quantify the similarity between sequences. Results: We propose 
new kernels for strings adapted to biological sequences, which we call local 
alignment kernels. These kernels measure the similarity between two sequences 
by summing up scores obtained from local alignments with gaps of the sequences. 
When tested in combination with SVM on their ability to recognize SCOP 
superfamilies on a benchmark dataset, the new kernels outperform 
state-of-the-art methods for remote homology detection. Availability: Software 
and data available upon request. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Saeys2004Feature]  
Y.&nbsp;Saeys, S.&nbsp;Degroeve, D.&nbsp;Aeyels, P.&nbsp;Rouz&eacute;, and 
Y.&nbsp;Van&nbsp;de Peer. Feature selection for splice site prediction: A new 
method using EDA-based feature ranking.<em>BMC Bioinformatics</em>, 5(64), 
2004. [&nbsp;bib&nbsp;| DOI&nbsp;| .pdf&nbsp;] 
<blockquote>  Background The identification of relevant biological features in 
large and complex datasets is an important step towards gaining insight in the 
processes underlying the data. Other advantages of feature selection include 
the ability of the classification system to attain good or even better 
solutions using a restricted subset of features, and a faster classification. 
Thus, robust methods for fast feature selection are of key importance in 
extracting knowledge from complex biological data. Results In this paper we 
present a novel method for feature subset selection applied to splice site 
prediction, based on estimation of distribution algorithms, a more general 
framework of genetic algorithms. From the estimated distribution of the 
algorithm, a feature ranking is derived. Afterwards this ranking is used to 
iteratively discard features. We apply this technique to the problem of splice 
site prediction, and show how it can be used to gain insight into the 
underlying biological process of splicing. Conclusion We show that this 
technique proves to be more robust than the traditional use of estimation of 
distribution algorithms for feature selection: instead of returning a single 
best subset of features (as they normally do) this method provides a dynamical 
view of the feature selection process, like the traditional sequential wrapper 
methods. However, the method is faster than the traditional techniques, and 
scales better to datasets described by a large number of features. </blockquote>
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Saetrom2004Predicting]  
P.&nbsp;Saetrom. Predicting the efficacy of short oligonucleotides in antisense 
and RNAi experiments with boosted genetic programming.<em>Bioinformatics</em>, 
20(17):3055-3063, 2004. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  Motivation: Both small interfering RNAs (siRNAs) and antisense 
oligonucleotides can selectively block gene expression. Although the two 
methods rely on different cellular mechanisms, these methods share the common 
property that not all oligonucleotides (oligos) are equally effective. That is, 
if mRNA target sites are picked at random, many of the antisense or siRNA 
oligos will not be effective. Algorithms that can reliably predict the efficacy 
of candidate oligos can greatly reduce the cost of knockdown experiments, but 
previous attempts to predict the efficacy of antisense oligos have had limited 
success. Machine learning has not previously been used to predict siRNA 
efficacy. Results: We develop a genetic programming based prediction system 
that shows promising results on both antisense and siRNA efficacy prediction. 
We train and evaluate our system on a previously published database of 
antisense efficacies and our own database of siRNA efficacies collected from 
the literature. The best models gave an overall correlation between predicted 
and observed efficacy of 0.46 on both antisense and siRNA data. As a 
comparison, the best correlations of support vector machine classifiers trained 
on the same data were 0.40 and 0.30, respectively. Availability: The prediction 
system uses proprietary hardware and is available for both commercial and 
strategic academic collaborations. The siRNA database is available upon request.
</blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Roegnvaldsson2004Why]  
Thorsteinn R&ouml;gnvaldsson and Liwen You. Why neural networks should not be 
used for HIV-1 protease cleavage site prediction.<em>Bioinformatics</em>, 
20(11):1702-9, Jul 2004. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  SUMMARY: Several papers have been published where nonlinear 
machine learning algorithms, e.g. artificial neural networks, support vector 
machines and decision trees, have been used to model the specificity of the 
HIV-1 protease and extract specificity rules. We show that the dataset used in 
these studies is linearly separable and that it is a misuse of nonlinear 
classifiers to apply them to this problem. The best solution on this dataset is 
achieved using a linear classifier like the simple perceptron or the linear 
support vector machine, and it is straightforward to extract rules from these 
linear models. We identify key residues in peptides that are efficiently 
cleaved by the HIV-1 protease and list the most prominent rules, relating them 
to experimental results for the HIV-1 protease. MOTIVATION: Understanding HIV-1 
protease specificity is important when designing HIV inhibitors and several 
different machine learning algorithms have been applied to the problem. 
However, little progress has been made in understanding the specificity because 
nonlinear and overly complex models have been used. RESULTS: We show that the 
problem is much easier than what has previously been reported and that linear 
classifiers like the simple perceptron or linear support vector machines are at 
least as good predictors as nonlinear algorithms. We also show how sets of 
specificity rules can be generated from the resulting linear classifiers. 
AVAILABILITY: The datasets used are available at http://www.hh.se/staff/bioinf/ 
</blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Ratsch2004Accurate]  
G.&nbsp;R&auml;tsch and S.&nbsp;Sonnenburg. Accurate splice site detection for 
Caenorhabditis elegans. In B.&nbsp;Sch&ouml;lkopf, K.&nbsp;Tsuda, and J.P. 
Vert, editors,<em>Kernel Methods in Computational Biology</em>, pages 277-298. 
MIT Press, 2004. [&nbsp;bib&nbsp;] 
<blockquote>  During the past three years, the support vector machine learning 
algorithm has been extensively applied within the field of computational 
biology. The algorithm has been used to detect patterns within and among 
biological sequences, to classify genes and patients based upon gene expression 
profiles, and has recently been applied to several new biological problems. 
This chapter reviews the state of the art with respect to SVM applications in 
computational biology. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Riedesel2004Peptide]  
Henning Riedesel, Bj&ouml;rn Kolbeck, Oliver Schmetzer, and Ernst-Walter Knapp. 
Peptide binding at class I major histocompatibility complex scored with linear 
functions and support vector machines.<em>Genome Inform Ser Workshop Genome 
Inform</em>, 15(1):198-212, 2004. [&nbsp;bib&nbsp;| .html&nbsp;| .pdf&nbsp;] 
<blockquote>  We explore two different methods to predict the binding ability 
of nonapeptides at the class I major histocompatibility complex using a general 
linear scoring function that defines a separating hyperplane in the feature 
space of sequences. In absence of suitable data on non-binding nonapeptides we 
generated sequences randomly from a selected set of proteins from the protein 
data bank. The parameters of the scoring function were determined by a 
generalized least square optimization (LSM) and alternatively by the support 
vector machine (SVM). With the generalized LSM impaired data for learning with 
a small set of binding peptides and a large set of non-binding peptides can be 
treated in a balanced way rendering LSM more successful than SVM, while for 
symmetric data sets SVM has a slight advantage compared to LSM. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Prados2004Mining]  
J.&nbsp;Prados, A.&nbsp;Kalousis, J.C. Sanchez, L.&nbsp;Allard, 
O.&nbsp;Carrette, and M.&nbsp;Hilario. Mining mass spectra for diagnosis and 
biomarker discovery of cerebral accidents.<em>Proteomics</em>, 4(8):2320-2332, 
2004. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  In this paper we try to identify potential biomarkers for early 
stroke diagnosis using surface-enhanced laser desorption/ionization mass 
spectrometry coupled with analysis tools from machine learning and data mining. 
Data consist of 42 specimen samples, i.e., mass spectra divided in two big 
categories, stroke and control specimens. Among the stroke specimens two 
further categories exist that correspond to ischemic and hemorrhagic stroke; in 
this paper we limit our data analysis to discriminating between control and 
stroke specimens. We performed two suites of experiments. In the first one we 
simply applied a number of different machine learning algorithms; in the second 
one we have chosen the best performing algorithm as it was determined from the 
first phase and coupled it with a number of different feature selection 
methods. The reason for this was 2-fold, first to establish whether feature 
selection can indeed improve performance, which in our case it did not seem to 
confirm, but more importantly to acquire a small list of potentially 
interesting biomarkers. Of the different methods explored the most promising 
one was support vector machines which gave us high levels of sensitivity and 
specificity. Finally, by analyzing the models constructed by support vector 
machines we produced a small set of 13 features that could be used as potential 
biomarkers, and which exhibited good performance both in terms of sensitivity, 
specificity and model stability. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm proteomics </blockquote> <br>
 [
Pochet2004Systematic]  N.&nbsp;Pochet, F.&nbsp;De&nbsp;Smet, J.&nbsp;A.&nbsp;K. 
Suykens, and B.&nbsp;L.&nbsp;R. De&nbsp;Moor. Systematic benchmarking of 
microarray data classification: assessing the role of non-linearity and 
dimensionality reduction.<em>Bioinformatics</em>, 20(17):3185-3195, Nov 2004. 
[&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  Motivation: Microarrays are capable of determining the 
expression levels of thousands of genes simultaneously. In combination with 
classification methods, this technology can be useful to support clinical 
management decisions for individual patients, e.g. in oncology. The aim of this 
paper is to systematically benchmark the role of non-linear versus linear 
techniques and dimensionality reduction methods. Results: A systematic 
benchmarking study is performed by comparing linear versions of standard 
classification and dimensionality reduction techniques with their non-linear 
versions based on non-linear kernel functions with a radial basis function 
(RBF) kernel. A total of 9 binary cancer classification problems, derived from 
7 publicly available microarray datasets, and 20 randomizations of each problem 
are examined. Conclusions: Three main conclusions can be formulated based on 
the performances on independent test sets. (1) When performing classification 
with least squares support vector machines (LS-SVMs) (without dimensionality 
reduction), RBF kernels can be used without risking too much overfitting. The 
results obtained with well-tuned RBF kernels are never worse and sometimes even 
statistically significantly better compared to results obtained with a linear 
kernel in terms of test set receiver operating characteristic and test set 
accuracy performances. (2) Even for classification with linear classifiers like 
LS-SVM with linear kernel, using regularization is very important. (3) When 
performing kernel principal component analysis (kernel PCA) before 
classification, using an RBF kernel for kernel PCA tends to result in 
overfitting, especially when using supervised feature selection. It has been 
observed that an optimal selection of a large number of features is often an 
indication for overfitting. Kernel PCA with linear kernel gives better results. 
Availability: Matlab scripts are available on request. Supplementary 
information: http://www.esat.kuleuven.ac.be/&nbsp;npochet/Bioinformatics/ 
</blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm microarray </blockquote> <br>
 [
Passerini2004Learning]  A.&nbsp;Passerini and P.&nbsp;Frasconi. Learning to 
discriminate between ligand-bound and disulfide-bound cysteines.<em>Protein 
Eng. Des. Sel.</em>, 17(4):367-373, 2004. [&nbsp;bib&nbsp;| DOI&nbsp;| http
&nbsp;|.pdf&nbsp;] 
<blockquote>  We present a machine learning method to discriminate between 
cysteines involved in ligand binding and cysteines forming disulfide bridges. 
Our method uses a window of multiple alignment profiles to represent each 
instance and support vector machines with a polynomial kernel as the learning 
algorithm. We also report results obtained with two new kernel functions based 
on similarity matrices. Experimental results indicate that binding type can be 
predicted at significantly higher accuracy than using PROSITE patterns. 
</blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Pan2004Comprehensive]  Fei 
Pan, Baoying Wang, Xin Hu, and William Perrizo. Comprehensive vertical 
sample-based KNN/LSVM classification for gene expression analysis.<em>J Biomed 
Inform</em>, 37(4):240-8, Aug 2004. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| 
.pdf&nbsp;] 
<blockquote>  Classification analysis of microarray gene expression data has 
been widely used to uncover biological features and to distinguish closely 
related cell types that often appear in the diagnosis of cancer. However, the 
number of dimensions of gene expression data is often very high, e.g., in the 
hundreds or thousands. Accurate and efficient classification of such 
high-dimensional data remains a contemporary challenge. In this paper, we 
propose a comprehensive vertical sample-based KNN/LSVM classification approach 
with weights optimized by genetic algorithms for high-dimensional data. 
Experiments on common gene expression datasets demonstrated that our approach 
can achieve high accuracy and efficiency at the same time. The improvement of 
speed is mainly related to the vertical data representation, P-tree,Patents are 
pending on the P-tree technology. This work is partially supported by GSA Grant 
ACT#:K96130308. and its optimized logical algebra. The high accuracy is due to 
the combination of a KNN majority voting approach and a local support vector 
machine approach that makes optimal decisions at the local level. As a result, 
our approach could be a powerful tool for high-dimensional gene expression data 
analysis. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Noble2004Support]  
W.&nbsp;S. Noble. Support vector machine applications in computational biology. 
In B.&nbsp;Sch&ouml;lkopf, K.&nbsp;Tsuda, and J.P. Vert, editors,<em>Kernel 
Methods in Computational Biology</em>, pages 71-92. MIT Press, 2004. [&nbsp;bib
&nbsp;|.pdf&nbsp;] 
<blockquote>  During the past three years, the support vector machine learning 
algorithm has been extensively applied within the field of computational 
biology. The algorithm has been used to detect patterns within and among 
biological sequences, to classify genes and patients based upon gene expression 
profiles, and has recently been applied to several new biological problems. 
This chapter reviews the state of the art with respect to SVM applications in 
computational biology. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Natt2004Prediction]  N.K. 
Natt, H.&nbsp;Kaur, and G.P. Raghava. Prediction of transmembrane regions of 
beta-barrel proteins using ANN- and SVM-based methods.<em>Proteins</em>, 
56(1):11-18, 2004. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  This article describes a method developed for predicting 
transmembrane beta-barrel regions in membrane proteins using machine learning 
techniques: artificial neural network (ANN) and support vector machine (SVM). 
The ANN used in this study is a feed-forward neural network with a standard 
back-propagation training algorithm. The accuracy of the ANN-based method 
improved significantly, from 70.4 when evolutionary information was added to a 
single sequence as a multiple sequence alignment obtained from PSI-BLAST. We 
have also developed an SVM-based method using a primary sequence as input and 
achieved an accuracy of 77.4 by adding 36 physicochemical parameters to the 
amino acid sequence information. Finally, ANN- and SVM-based methods were 
combined to utilize the full potential of both techniques. The accuracy and 
Matthews correlation coefficient (MCC) value of SVM, ANN, and combined method 
are 78.5 and 0.64, respectively. These methods were trained and tested on a 
nonredundant data set of 16 proteins, and performance was evaluated using 
&quot;leave one out cross-validation&quot; (LOOCV). Based on this study, we 
have developed a Web server, TBBPred, for predicting transmembrane beta-barrel 
regions in proteins (available at http://www.imtech.res.in/raghava/tbbpred). 
</blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Mika2004Protein]  Sven 
Mika and Burkhard Rost. Protein names precisely peeled off free text.<em>
Bioinformatics</em>, 20(Suppl. 1):i241-i247, 2004. [&nbsp;bib&nbsp;| http&nbsp;|
.pdf&nbsp;] 
<blockquote>  Motivation: Automatically identifying protein names from the 
scientific literature is a pre-requisite for the increasing demand in 
data-mining this wealth of information. Existing approaches are based on 
dictionaries, rules and machine-learning. Here, we introduced a novel system 
that combines a pre-processing dictionary- and rule-based filtering step with 
several separately trained support vector machines (SVMs) to identify protein 
names in the MEDLINE abstracts. Results: Our new tagging-system NLProt is 
capable of extracting protein names with a precision (accuracy) of 75 76 and 
contains 200 annotated abstracts. For our estimate of sustained performance, we 
considered partially identified names as false positives. One important issue 
frequently ignored in the literature is the redundancy in evaluation sets. We 
suggested some guidelines for removing overly inadequate overlaps between 
training and testing sets. Applying these new guidelines, our program appeared 
to significantly out-perform other methods tagging protein names. NLProt was so 
successful due to the SVM-building blocks that succeeded in utilizing the local 
context of protein names in the scientific literature. We challenge that our 
system may constitute the most general and precise method for tagging protein 
names. Availability: http://cubic.bioc.columbia.edu/services/nlprot/ 
</blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm nlp </blockquote> <br>
 [Mika2004NLProt]  Sven 
Mika and Burkhard Rost. NLProt: extracting protein names and sequences from 
papers.<em>Nucleic Acids Res</em>, 32(Web Server issue):W634-7, Jul 2004. 
[&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;] 
<blockquote>  Automatically extracting protein names from the literature and 
linking these names to the associated entries in sequence databases is becoming 
increasingly important for annotating biological databases. NLProt is a novel 
system that combines dictionary- and rule-based filtering with several support 
vector machines (SVMs) to tag protein names in PubMed abstracts. When 
considering partially tagged names as errors, NLProt still reached a precision 
of 75% at a recall of 76%. By many criteria our system outperformed other 
tagging methods significantly; in particular, it proved very reliable even for 
novel names. Names encountered particularly frequently in Drosophila, such as 
white, wing and bizarre, constitute an obvious limitation of NLProt. Our method 
is available both as an Internet server and as a program for download 
(http://cubic.bioc.columbia.edu/services/NLProt/). Input can be PubMed/MEDLINE 
identifiers, authors, titles and journals, as well as collections of abstracts, 
or entire papers. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm nlp </blockquote> <br>
 [
Middendorf2004Discriminative]  M.&nbsp;Middendorf, E.&nbsp;Ziv, C.&nbsp;Adams, 
J.&nbsp;Hom, R.&nbsp;Koytcheff, C.&nbsp;Levovitz, G.&nbsp;Woods, L.&nbsp;Chen, 
and C.&nbsp;Wiggins. Discriminative topological features reveal biological 
network mechanisms.<em>BMC Bioinformatics</em>, 5(181), 2004. [&nbsp;bib&nbsp;| 
DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  BACKGROUND: Recent genomic and bioinformatic advances have 
motivated the development of numerous network models intending to describe 
graphs of biological, technological, and sociological origin. In most cases the 
success of a model has been evaluated by how well it reproduces a few key 
features of the real-world data, such as degree distributions, mean geodesic 
lengths, and clustering coefficients. Often pairs of models can reproduce these 
features with indistinguishable fidelity despite being generated by vastly 
different mechanisms. In such cases, these few target features are insufficient 
to distinguish which of the different models best describes real world networks 
of interest; moreover, it is not clear a priori that any of the 
presently-existing algorithms for network generation offers a predictive 
description of the networks inspiring them. RESULTS: We present a method to 
assess systematically which of a set of proposed network generation algorithms 
gives the most accurate description of a given biological network. To derive 
discriminative classifiers, we construct a mapping from the set of all graphs 
to a high-dimensional (in principle infinite-dimensional) &quot;word 
space&quot;. This map defines an input space for classification schemes which 
allow us to state unambiguously which models are most descriptive of a given 
network of interest. Our training sets include networks generated from 17 
models either drawn from the literature or introduced in this work. We show 
that different duplication-mutation schemes best describe the E. coli genetic 
network, the S. cerevisiae protein interaction network, and the C. elegans 
neuronal network, out of a set of network models including a linear 
preferential attachment model and a small-world model. CONCLUSIONS: Our method 
is a first step towards systematizing network models and assessing their 
predictability, and we anticipate its usefulness for a number of communities. 
</blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Meinicke2004Oligo]  
P.&nbsp;Meinicke, M.&nbsp;Tech, B.&nbsp;Morgenstern, and R.&nbsp;Merkl. Oligo 
kernels for datamining on biological sequences: a case study on prokaryotic 
translation initiation sites.<em>BMC Bioinformatics</em>, 5(169), 2004. [&nbsp;
bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  Background Kernel-based learning algorithms are among the most 
advanced machine learning methods and have been successfully applied to a 
variety of sequence classification tasks within the field of bioinformatics. 
Conventional kernels utilized so far do not provide an easy interpretation of 
the learnt representations in terms of positional and compositional variability 
of the underlying biological signals. Results We propose a kernel-based 
approach to datamining on biological sequences. With our method it is possible 
to model and analyze positional variability of oligomers of any length in a 
natural way. On one hand this is achieved by mapping the sequences to an 
intuitive but high-dimensional feature space, well-suited for interpretation of 
the learnt models. On the other hand, by means of the kernel trick we can 
provide a general learning algorithm for that high-dimensional representation 
because all required statistics can be computed without performing an explicit 
feature space mapping of the sequences. By introducing a kernel parameter that 
controls the degree of position-dependency, our feature space representation 
can be tailored to the characteristics of the biological problem at hand. A 
regularized learning scheme enables application even to biological problems for 
which only small sets of example sequences are available. Our approach includes 
a visualization method for transparent representation of characteristic 
sequence features. Thereby importance of features can be measured in terms of 
discriminative strength with respect to classification of the underlying 
sequences. To demonstrate and validate our concept on a biochemically 
well-defined case, we analyze E. coli translation initiation sites in order to 
show that we can find biologically relevant signals. For that case, our results 
clearly show that the Shine-Dalgarno sequence is the most important signal 
upstream a start codon. The variability in position and composition we found 
for that signal is in accordance with previous biological knowledge. We also 
find evidence for signals downstream of the start codon, previously introduced 
as transcriptional enhancers. These signals are mainly characterized by 
occurrences of adenine in a region of about 4 nucleotides next to the start 
codon. Conclusions We showed that the oligo kernel can provide a valuable tool 
for the analysis of relevant signals in biological sequences. In the case of 
translation initiation sites we could clearly deduce the most discriminative 
motifs and their positional variation from example sequences. Attractive 
features of our approach are its flexibility with respect to oligomer length 
and position conservation. By means of these two parameters oligo kernels can 
easily be adapted to different biological problems. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Man2004Evaluating]  M.Z. 
Man, G.&nbsp;Dyson, K.&nbsp;Johnson, and B.&nbsp;Liao. Evaluating methods for 
classifying expression data.<em>J. Biopharm. Stat.</em>, 14(4):1065-1084, 2004. 
[&nbsp;bib&nbsp;| DOI&nbsp;| .pdf&nbsp;] 
<blockquote>  An attractive application of expression technologies is to 
predict drug efficacy or safety using expression data of biomarkers. To 
evaluate the performance of various classification methods for building 
predictive models, we applied these methods on six expression datasets. These 
datasets were from studies using microarray technologies and had either two or 
more classes. From each of the original datasets, two subsets were generated to 
simulate two scenarios in biomarker applications. First, a 50-gene subset was 
used to simulate a candidate gene approach when it might not be practical to 
measure a large number of genes/biomarkers. Next, a 2000-gene subset was used 
to simulate a whole genome approach. We evaluated the relative performance of 
several classification methods by using leave-one-out cross-validation and 
bootstrap cross-validation. Although all methods perform well in both subsets 
for a relative easy dataset with two classes, differences in performance do 
exist among methods for other datasets. Overall, partial least squares 
discriminant analysis (PLS-DA) and support vector machines (SVM) outperform all 
other methods. We suggest a practical approach to take advantage of multiple 
methods in biomarker applications. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Mahe2004Extensions]  
P.&nbsp;Mah&eacute;, N.&nbsp;Ueda, T.&nbsp;Akutsu, J.-L. Perret, and J.-P. 
Vert. Extensions of marginalized graph kernels. In R.&nbsp;Greiner and 
D.&nbsp;Schuurmans, editors,<em>Proceedings of the Twenty-First International 
Conference on Machine Learning (ICML 2004)</em>, pages 552-559. ACM Press, 
2004. [&nbsp;bib&nbsp;| www:&nbsp;] 
<blockquote>  Positive definite kernels between labeled graphs have recently 
been proposed.They enable the application of kernel methods, such as support 
vectormachines, to the analysis and classification of graphs, for example, 
chemicalcompounds. These graph kernels are obtained by marginalizing a kernel 
betweenpaths with respect to a random walk model on the graph vertices along 
theedges. We propose two extensions of these graph kernels, with the double 
goal toreduce their computation time and increase their relevance as measure 
ofsimilarity between graphs. First, we propose to modify the label of 
eachvertex by automatically adding information about its environment with the 
useof the Morgan algorithm. Second, we suggest a modification of the random 
walkmodel to prevent the walk from coming back to a vertex that was just 
visited.These extensions are then tested on benchmark experiments of 
chemicalcompounds classification, with promising results. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm chemoinformatics </blockquote> <br>
 [
Liu2004comparative]  Y.&nbsp;Liu. A comparative study on feature selection 
methods for drug discovery.<em>J Chem Inf Comput Sci</em>, 44(5):1823-8, 2004. 
[&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  Feature selection is frequently used as a preprocessing step to 
machine learning. The removal of irrelevant and redundant information often 
improves the performance of learning algorithms. This paper is a comparative 
study of feature selection in drug discovery. The focus is on aggressive 
dimensionality reduction. Five methods were evaluated, including information 
gain, mutual information, a chi2-test, odds ratio, and GSS coefficient. Two 
well-known classification algorithms, Na&iuml;ve Bayesian and Support Vector 
Machine (SVM), were used to classify the chemical compounds. The results showed 
that Na&iuml;ve Bayesian benefited significantly from the feature selection, 
while SVM performed better when all features were used. In this experiment, 
information gain and chi2-test were most effective feature selection methods. 
Using information gain with a Na&iuml;ve Bayesian classifier, removal of up to 
96% of the features yielded an improved classification accuracy measured by 
sensitivity. When information gain was used to select the features, SVM was 
much less sensitive to the reduction of feature space. The feature set size was 
reduced by 99%, while losing only a few percent in terms of sensitivity (from 
58.7% to 52.5%) and specificity (from 98.4% to 97.2%). In contrast to 
information gain and chi2-test, mutual information had relatively poor 
performance due to its bias toward favoring rare features and its sensitivity 
to probability estimation errors. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Liu2004Active]  
Y.&nbsp;Liu. Active learning with support vector machine applied to gene 
expression data for cancer classification.<em>J. Chem. Inf. Comput. Sci.</em>, 
44(6):1936-1941, 2004. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  There is growing interest in the application of machine learning 
techniques in bioinformatics. The supervised machine learning approach has been 
widely applied to bioinformatics and gained a lot of success in this research 
area. With this learning approach researchers first develop a large training 
set, which is a time-consuming and costly process. Moreover, the proportion of 
the positive examples and negative examples in the training set may not 
represent the real-world data distribution, which causes concept drift. Active 
learning avoids these problems. Unlike most conventional learning methods where 
the training set used to derive the model remains static, the classifier can 
actively choose the training data and the size of training set increases. We 
introduced an algorithm for performing active learning with support vector 
machine and applied the algorithm to gene expression profiles of colon cancer, 
lung cancer, and prostate cancer samples. We compared the classification 
performance of active learning with that of passive learning. The results 
showed that employing the active learning method can achieve high accuracy and 
significantly reduce the need for labeled training instances. For lung cancer 
classification, to achieve 96 only 31 labeled examples were needed in active 
learning whereas in passive learning 174 labeled examples were required. That 
meant over 82 the areas under the receiver operating characteristic (ROC) 
curves were over 0.81, while in passive learning the areas under the ROC curves 
were below 0.50. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Liu2004QSAR]  H.&nbsp;X. 
Liu, R.&nbsp;S. Zhang, X.&nbsp;J. Yao, M.&nbsp;C. Liu, Z.&nbsp;D. Hu, and 
B.&nbsp;T. Fan. QSAR and classification models of a novel series of COX-2 
selective inhibitors: 1,5-diarylimidazoles based on support vector machines.<em>
J Comput Aided Mol Des</em>, 18(6):389-99, Jun 2004. [&nbsp;bib&nbsp;] 
<blockquote>  The support vector machine, which is a novel algorithm from the 
machine learning community, was used to develop quantitation and classification 
models which can be used as a potential screening mechanism for a novel series 
of COX-2 selective inhibitors. Each compound was represented by calculated 
structural descriptors that encode constitutional, topological, geometrical, 
electrostatic, and quantum-chemical features. The heuristic method was then 
used to search the descriptor space and select the descriptors responsible for 
activity. Quantitative modelling results in a nonlinear, seven-descriptor model 
based on SVMs with root mean-square errors of 0.107 and 0.136 for training and 
prediction sets, respectively. The best classification results are found using 
SVMs: the accuracy for training and test sets is 91.2% and 88.2%, respectively. 
This paper proposes a new and effective method for drug design and screening. 
</blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm chemoinformatics </blockquote> <br>
 [
Liu2004Prediction]  H.&nbsp;X. Liu, R.&nbsp;S. Zhang, X.&nbsp;J. Yao, 
M.&nbsp;C. Liu, Z.&nbsp;D. Hu, and B.&nbsp;T. Fan. Prediction of the 
isoelectric point of an amino acid based on GA-PLS and SVMs.<em>J Chem Inf 
Comput Sci</em>, 44(1):161-7, 2004. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| 
.pdf&nbsp;] 
<blockquote>  The support vector machine (SVM), as a novel type of a learning 
machine, for the first time, was used to develop a QSPR model that relates the 
structures of 35 amino acids to their isoelectric point. Molecular descriptors 
calculated from the structure alone were used to represent molecular 
structures. The seven descriptors selected using GA-PLS, which is a 
sophisticated hybrid approach that combines GA as a powerful optimization 
method with PLS as a robust statistical method for variable selection, were 
used as inputs of RBFNNs and SVM to predict the isoelectric point of an amino 
acid. The optimal QSPR model developed was based on support vector machines, 
which showed the following results: the root-mean-square error of 0.2383 and 
the prediction correlation coefficient R=0.9702 were obtained for the whole 
data set. Satisfactory results indicated that the GA-PLS approach is a very 
effective method for variable selection, and the support vector machine is a 
very promising tool for the nonlinear approximation. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Liu2004Quantitative]  
H.&nbsp;X. Liu, C.&nbsp;X. Xue, R.&nbsp;S. Zhang, X.&nbsp;J. Yao, M.&nbsp;C. 
Liu, Z.&nbsp;D. Hu, and B.&nbsp;T. Fan. Quantitative prediction of logk of 
peptides in high-performance liquid chromatography based on molecular 
descriptors by using the heuristic method and support vector machine.<em>J Chem 
Inf Comput Sci</em>, 44(6):1979-86, 2004. [&nbsp;bib&nbsp;| DOI&nbsp;| http
&nbsp;|.pdf&nbsp;] 
<blockquote>  A new method support vector machine (SVM) and the heuristic 
method (HM) were used to develop the nonlinear and linear models between the 
capacity factor (logk) and seven molecular descriptors of 75 peptides for the 
first time. The molecular descriptors representing the structural features of 
the compounds only included the constitutional and topological descriptors, 
which can be obtained easily without optimizing the structure of the molecule. 
The seven molecular descriptors selected by the heuristic method in CODESSA 
were used as inputs for SVM. The results obtained by SVM were compared with 
those obtained by the heuristic method. The prediction result of the SVM model 
is better than that of heuristic method. For the test set, a predictive 
correlation coefficient R = 0.9801 and root-mean-square error of 0.1523 were 
obtained. The prediction results are in very good agreement with the 
experimental values. But the linear model of the heuristic method is easier to 
understand and ready to use for a chemist. This paper provided a new and 
effective method for predicting the chromatography retention of peptides and 
some insight into the structural features which are related to the capacity 
factor of peptides. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Liu2004Using]  Huiqing 
Liu, Hao Han, Jinyan Li, and Limsoon Wong. Using amino acid patterns to 
accurately predict translation initiation sites.<em>In Silico Biol.</em>, 
4(3):255-69, 2004. [&nbsp;bib&nbsp;| http&nbsp;] 
<blockquote>  The translation initiation site (TIS) prediction problem is 
about how to correctly identify TIS in mRNA, cDNA, or other types of genomic 
sequences. High prediction accuracy can be helpful in a better understanding of 
protein coding from nucleotide sequences. This is an important step in genomic 
analysis to determine protein coding from nucleotide sequences. In this paper, 
we present an in silico method to predict translation initiation sites in 
vertebrate cDNA or mRNA sequences. This method consists of three sequential 
steps as follows. In the first step, candidate features are generated using 
k-gram amino acid patterns. In the second step, a small number of top-ranked 
features are selected by an entropy-based algorithm. In the third step, a 
classification model is built to recognize true TISs by applying support vector 
machines or ensembles of decision trees to the selected features. We have 
tested our method on several independent data sets, including two public ones 
and our own extracted sequences. The experimental results achieved are better 
than those reported previously using the same data sets. Our high accuracy not 
only demonstrates the feasibility of our method, but also indicates that there 
might be &quot;amino acid&quot; patterns around TIS in cDNA and mRNA sequences. 
</blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Listgarten2004Predictive]  
J.&nbsp;Listgarten, S.&nbsp;Damaraju, B.&nbsp;Poulin, L.&nbsp;Cook, 
J.&nbsp;Dufour, A.&nbsp;Driga, J.&nbsp;Mackey, D.&nbsp;Wishart, 
R.&nbsp;Greiner, and B.&nbsp;Zanke. Predictive Models for Breast Cancer 
Susceptibility from Multiple Single Nucleotide Polymorphisms.<em>Clin. Cancer 
Res.</em>, 10(8):2725-2737, 2004. [&nbsp;bib&nbsp;| arXiv&nbsp;| http&nbsp;| 
.pdf&nbsp;] 
<blockquote>  Hereditary predisposition and causative environmental exposures 
have long been recognized in human malignancies. In most instances, cancer 
cases occur sporadically, suggesting that environmental influences are critical 
in determining cancer risk. To test the influence of genetic polymorphisms on 
breast cancer risk, we have measured 98 single nucleotide polymorphisms (SNPs) 
distributed over 45 genes of potential relevance to breast cancer etiology in 
174 patients and have compared these with matched normal controls. Using 
machine learning techniques such as support vector machines (SVMs), decision 
trees, and naive Bayes, we identified a subset of three SNPs as key 
discriminators between breast cancer and controls. The SVMs performed maximally 
among predictive models, achieving 69 power in distinguishing between the two 
groups, compared with a 50 baseline predictive power obtained from the data 
after repeated random permutation of class labels (individuals with cancer or 
controls). However, the simpler naive Bayes model as well as the decision tree 
model performed quite similarly to the SVM. The three SNP sites most useful in 
this model were (a) the +4536T/C site of the aldosterone synthase gene CYP11B2 
at amino acid residue 386 Val/Ala (T/C) (rs4541); (b) the +4328C/G site of the 
aryl hydrocarbon hydroxylase CYP1B1 at amino acid residue 293 Leu/Val (C/G) 
(rs5292); and (c) the +4449C/T site of the transcription factor BCL6 at amino 
acid 387 Asp/Asp (rs1056932). No single SNP site on its own could achieve more 
than 60 predictive accuracy. We have shown that multiple SNP sites from 
different genes over distant parts of the genome are better at identifying 
breast cancer patients than any one SNP alone. As high-throughput technology 
for SNPs improves and as more SNPs are identified, it is likely that much 
higher predictive accuracy will be achieved and a useful clinical tool 
developed. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm, breastcancer </blockquote> <br>
 [
Li2004comparative]  T.&nbsp;Li, C.&nbsp;Zhang, and M.&nbsp;Ogihara. A 
comparative study of feature selection and multiclass classification methods 
for tissue classification based on gene expression.<em>Bioinformatics</em>, 
20(15):2429-2437, 2004. [&nbsp;bib&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  Summary: This paper studies the problem of building multiclass 
classifiers for tissue classification based on gene expression. The recent 
development of microarray technologies has enabled biologists to quantify gene 
expression of tens of thousands of genes in a single experiment. Biologists 
have begun collecting gene expression for a large number of samples. One of the 
urgent issues in the use of microarray data is to develop methods for 
characterizing samples based on their gene expression. The most basic step in 
the research direction is binary sample classification, which has been studied 
extensively over the past few years. This paper investigates the next 
step-multiclass classification of samples based on gene expression. The 
characteristics of expression data (e.g. large number of genes with small 
sample size) makes the classification problem more challenging. The process of 
building multiclass classifiers is divided into two components: (i) selection 
of the features (i.e. genes) to be used for training and testing and (ii) 
selection of the classification method. This paper compares various feature 
selection methods as well as various state-of-the-art classification methods on 
various multiclass gene expression datasets. Our study indicates that 
multiclass classification problem is much more difficult than the binary one 
for the gene expression datasets. The difficulty lies in the fact that the data 
are of high dimensionality and that the sample size is small. The 
classification accuracy appears to degrade very rapidly as the number of 
classes increases. In particular, the accuracy was very low regardless of the 
choices of the methods for large-class datasets (e.g. NCI60 and GCM). While 
increasing the number of samples is a plausible solution to the problem of 
accuracy degradation, it is important to develop algorithms that are able to 
analyze effectively multiple-class expression data for these special datasets. 
</blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Li2004Data]  L.&nbsp;Li, 
H.&nbsp;Tang, Z.&nbsp;Wu, J.&nbsp;Gong, M.&nbsp;Gruidl, J.&nbsp;Zou, 
M.&nbsp;Tockman, and R.A. Clark. Data mining techniques for cancer detection 
using serum proteomic profiling.<em>Artif. Intell. Med.</em>, 32(2):71-83, 
2004. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  OBJECTIVE: Pathological changes in an organ or tissue may be 
reflected in proteomic patterns in serum. It is possible that unique serum 
proteomic patterns could be used to discriminate cancer samples from non-cancer 
ones. Due to the complexity of proteomic profiling, a higher order analysis 
such as data mining is needed to uncover the differences in complex proteomic 
patterns. The objectives of this paper are (1) to briefly review the 
application of data mining techniques in proteomics for cancer 
detection/diagnosis; (2) to explore a novel analytic method with different 
feature selection methods; (3) to compare the results obtained on different 
datasets and that reported by Petricoin et al. in terms of detection 
performance and selected proteomic patterns. METHODS AND MATERIAL: Three serum 
SELDI MS data sets were used in this research to identify serum proteomic 
patterns that distinguish the serum of ovarian cancer cases from non-cancer 
controls. A support vector machine-based method is applied in this study, in 
which statistical testing and genetic algorithm-based methods are used for 
feature selection respectively. Leave-one-out cross validation with receiver 
operating characteristic (ROC) curve is used for evaluation and comparison of 
cancer detection performance. RESULTS AND CONCLUSIONS: The results showed that 
(1) data mining techniques can be successfully applied to ovarian cancer 
detection with a reasonably high performance; (2) the classification using 
features selected by the genetic algorithm consistently outperformed those 
selected by statistical testing in terms of accuracy and robustness; (3) the 
discriminatory features (proteomic patterns) can be very different from one 
selection method to another. In other words, the pattern selection and its 
classification efficiency are highly classifier dependent. Therefore, when 
using data mining techniques, the discrimination of cancer from normal does not 
depend solely upon the identity and origination of cancer-related proteins. 
</blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Lett2004Interaction]  
D.&nbsp;Lett, M.&nbsp;Hsing, and F.&nbsp;Pio. Interaction profile-based protein 
classification of death domain.<em>BMC Bioinformatics</em>, 5(75), 2004. [&nbsp;
bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  Background The increasing number of protein sequences and 3D 
structure obtained from genomic initiatives is leading many of us to focus on 
proteomics, and to dedicate our experimental and computational efforts on the 
creation and analysis of information derived from 3D structure. In particular, 
the high-throughput generation of protein-protein interaction data from a few 
organisms makes such an approach very important towards understanding the 
molecular recognition that make-up the entire protein-protein interaction 
network. Since the generation of sequences, and experimental protein-protein 
interactions increases faster than the 3D structure determination of protein 
complexes, there is tremendous interest in developing in silico methods that 
generate such structure for prediction and classification purposes. In this 
study we focused on classifying protein family members based on their 
protein-protein interaction distinctiveness. Structure-based classification of 
protein-protein interfaces has been described initially by Ponstingl et al. [1] 
and more recently by Valdar et al. [2] and Mintseris et al. [3], from complex 
structures that have been solved experimentally. However, little has been done 
on protein classification based on the prediction of protein-protein complexes 
obtained from homology modeling and docking simulation. Results We have 
developed an in silico classification system entitled HODOCO (Homology 
modeling, Docking and Classification Oracle), in which protein Residue 
Potential Interaction Profiles (RPIPS) are used to summarize protein-protein 
interaction characteristics. This system applied to a dataset of 64 proteins of 
the death domain superfamily was used to classify each member into its proper 
subfamily. Two classification methods were attempted, heuristic and support 
vector machine learning. Both methods were tested with a 5-fold 
cross-validation. The heuristic approach yielded a 61 accuracy, while the 
machine learning approach yielded an 89 accuracy. Conclusion We have confirmed 
the reliability and potential value of classifying proteins via their predicted 
interactions. Our results are in the same range of accuracy as other studies 
that classify protein-protein interactions from 3D complex structure obtained 
experimentally. While our classification scheme does not take directly into 
account sequence information our results are in agreement with functional and 
sequence based classification of death domain family members. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Leslie2004Mismatch]  
C.&nbsp;S. Leslie, E.&nbsp;Eskin, A.&nbsp;Cohen, J.&nbsp;Weston, and W.&nbsp;S. 
Noble. Mismatch string kernels for discriminative protein classification.<em>
Bioinformatics</em>, 20(4):467-476, 2004. [&nbsp;bib&nbsp;| http&nbsp;| .pdf
&nbsp;]
<blockquote>  Motivation: Classification of proteins sequences into functional 
and structural families based on sequence homology is a central problem in 
computational biology. Discriminative supervised machine learning approaches 
provide good performance, but simplicity and computational efficiency of 
training and prediction are also important concerns. Results: We introduce a 
class of string kernels, called mismatch kernels, for use with support vector 
machines (SVMs) in a discriminative approach to the problem of protein 
classification and remote homology detection. These kernels measure sequence 
similarity based on shared occurrences of fixed-length patterns in the data, 
allowing for mutations between patterns. Thus, the kernels provide a 
biologically well-motivated way to compare protein sequences without relying on 
family-based generative models such as hidden Markov models. We compute the 
kernels efficiently using a mismatch tree data structure, allowing us to 
calculate the contributions of all patterns occurring in the data in one pass 
while traversing the tree. When used with an SVM, the kernels enable fast 
prediction on test sequences. We report experiments on two benchmark SCOP 
datasets, where we show that the mismatch kernel used with an SVM classifier 
performs competitively with state-of-the-art methods for homology detection, 
particularly when very few training examples are available. Examination of the 
highest-weighted patterns learned by the SVM classifier recovers biologically 
important motifs in protein families and superfamilies. Availability: SVM 
software is publicly available at http://microarray.cpmc.columbia.edu/gist. 
Mismatch kernel software is available upon request. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Lanckriet2004statistical]  
G.&nbsp;R.&nbsp;G. Lanckriet, T.&nbsp;De&nbsp;Bie, N.&nbsp;Cristianini, 
M.&nbsp;I. Jordan, and W.&nbsp;S. Noble. A statistical framework for genomic 
data fusion.<em>Bioinformatics</em>, 20(16):2626-2635, 2004. [&nbsp;bib&nbsp;| 
DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  Motivation: During the past decade, the new focus on genomics 
has highlighted a particular challenge: to integrate the different views of the 
genome that are provided by various types of experimental data. Results: This 
paper describes a computational framework for integrating and drawing 
inferences from a collection of genome-wide measurements. Each dataset is 
represented via a kernel function, which defines generalized similarity 
relationships between pairs of entities, such as genes or proteins. The kernel 
representation is both flexible and efficient, and can be applied to many 
different types of data. Furthermore, kernel functions derived from different 
types of data can be combined in a straightforward fashion. Recent advances in 
the theory of kernel methods have provided efficient algorithms to perform such 
combinations in a way that minimizes a statistical loss function. These methods 
exploit semidefinite programming techniques to reduce the problem of finding 
optimizing kernel combinations to a convex optimization problem. Computational 
experiments performed using yeast genome-wide datasets, including amino acid 
sequences, hydropathy profiles, gene expression data and known protein-protein 
interactions, demonstrate the utility of this approach. A statistical learning 
algorithm trained from all of these data to recognize particular classes of 
proteins-membrane proteins and ribosomal proteins-performs significantly better 
than the same algorithm trained on any single type of data. Availability: 
Supplementary data at http://noble.gs.washington.edu/proj/sdp-svm </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Lanckriet2004Kernel-baseda]
 G.R. Lanckriet, M.&nbsp;Deng, N.&nbsp;Cristianini, M.I. Jordan, and W.S. 
Noble. Kernel-based data fusion and its application to protein function 
prediction in yeast. In<em>Proceedings of the Pacific Symposium on Biocomputing
</em>, pages 300-311, 2004. [&nbsp;bib&nbsp;| .pdf&nbsp;] 
<blockquote>  Kernel methods provide a principled framework in which to 
represent many types of data, including vectors, strings, trees and graphs. As 
such, these methods are useful for drawing inferences about biological 
phenomena. We describe a method for combining multiple kernel representations 
in an optimal fashion, by formulating the problem as a convex optimization 
problem that can be solved using semidefinite programming techniques. The 
method is applied to the problem of predicting yeast protein functional 
classifications using a support vector machine (SVM) trained on five types of 
data. For this problem, the new method performs better than a 
previously-described Markov random field method, and better than the SVM 
trained on any single type of data. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Lanckriet2004Kernel-based] 
 G.R.G. Lanckriet, N.&nbsp;Cristianini, M.I. Jordan, and W.S. Noble. 
Kernel-based integration of genomic data using semidefinite programming. In 
B.&nbsp;Sch&ouml;lkopf, K.&nbsp;Tsuda, and J.P. Vert, editors,<em>Kernel 
Methods in Computational Biology</em>, pages 231-259. MIT Press, 2004. [&nbsp;
bib&nbsp;] 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Kuang2004Protein]  
R.&nbsp;Kuang, C.&nbsp;S. Leslie, and A.-S. Yang. Protein backbone angle 
prediction with machine learning approaches.<em>Bioinformatics</em>, 
20(10):1612-1621, 2004. [&nbsp;bib&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  Motivation: Protein backbone torsion angle prediction provides 
useful local structural information that goes beyond conventional three-state 
(alpha, beta and coil) secondary structure predictions. Accurate prediction of 
protein backbone torsion angles will substantially improve modeling procedures 
for local structures of protein sequence segments, especially in modeling loop 
conformations that do not form regular structures as in alpha-helices or 
beta-strands. Results: We have devised two novel automated methods in protein 
backbone conformational state prediction: one method is based on support vector 
machines (SVMs); the other method combines a standard feed-forward 
back-propagation artificial neural network (NN) with a local structure-based 
sequence profile database (LSBSP1). Extensive benchmark experiments demonstrate 
that both methods have improved the prediction accuracy rate over the 
previously published methods for conformation state prediction when using an 
alphabet of three or four states. Availability: LSBSP1 and the NN algorithm 
have been implemented in PrISM.1, which is available from 
www.columbia.edu/&nbsp;ay1/. Supplementary information: Supplementary data for 
the SVM method can be downloaded from the Website 
www.cs.columbia.edu/compbio/backbone. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Kuang2004Profile-based]  
R.&nbsp;Kuang, E.&nbsp;Ie, K.&nbsp;Wang, K.&nbsp;Wang, M.&nbsp;Siddiqi, 
Y.&nbsp;Freund, and C.&nbsp;Leslie. Profile-based string kernels for remote 
homology detection and motif extraction.<em>Proc IEEE Comput Syst Bioinform Conf
</em>, pages 152-160, 2004. [&nbsp;bib&nbsp;] 
<blockquote>  We introduce novel profile-based string kernels for use with 
support vector machines (SVMs) for the problems of protein classification and 
remote homology detection. These kernels use probabilistic profiles, such as 
those produced by the PSI-BLAST algorithm, to define position-dependent 
mutation neighborhoods along protein sequences for inexact matching of k-length 
subsequences (&quot;k-mers&quot;) in the data. By use of an efficient data 
structure, the kernels are fast to compute once the profiles have been 
obtained. For example, the time needed to run PSI-BLAST in order to build the 
pro- files is significantly longer than both the kernel computation time and 
the SVM training time. We present remote homology detection experiments based 
on the SCOP database where we show that profile-based string kernels used with 
SVM classifiers strongly outperform all recently presented supervised SVM 
methods. We also show how we can use the learned SVM classifier to extract 
&quot;discriminative sequence motifs&quot; - short regions of the original 
profile that contribute almost all the weight of the SVM classification score - 
and show that these discriminative motifs correspond to meaningful structural 
features in the protein data. The use of PSI-BLAST profiles can be seen as a 
semi-supervised learning technique, since PSI-BLAST leverages unlabeled data 
from a large sequence database to build more informative profiles. Recently 
presented &quot;cluster kernels&quot; give general semi-supervised methods for 
improving SVM protein classification performance. We show that our profile 
kernel results are comparable to cluster kernels while providing much better 
scalability to large datasets. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Krishnapuram2004bayesian]  
Balaji Krishnapuram, Alexander&nbsp;J Hartemink, Lawrence Carin, and 
M&Atilde;&Acirc;&iexcl;rio A&nbsp;T Figueiredo. A bayesian approach to joint 
feature selection and classifier design.<em>IEEE Trans Pattern Anal Mach Intell
</em>, 26(9):1105-11, Sep 2004. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf
&nbsp;]
<blockquote>  This paper adopts a Bayesian approach to simultaneously learn 
both an optimal nonlinear classifier and a subset of predictor variables (or 
features) that are most relevant to the classification task. The approach uses 
heavy-tailed priors to promote sparsity in the utilization of both basis 
functions and features; these priors act as regularizers for the likelihood 
function that rewards good classification on the training data. We derive an 
expectation-maximization (EM) algorithm to efficiently compute a maximum a 
posteriori (MAP) point estimate of the various parameters. The algorithm is an 
extension of recent state-of-the-art sparse Bayesian classifiers, which in turn 
can be seen as Bayesian counterparts of support vector machines. Experimental 
comparisons using kernel classifiers demonstrate both parsimonious feature 
selection and excellent classification accuracy on a range of synthetic and 
benchmark data sets. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Krishnapuram2004Joint]  
B.&nbsp;Krishnapuram, L.&nbsp;Carin, and A.&nbsp;Hartemink. Joint Classifier 
and Feature Optimization for Comprehensive Cancer Diagnosis Using Gene 
Expression Data.<em>J. Comput. Biol.</em>, 11(2-3):227-242, 2004. [&nbsp;bib
&nbsp;|DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  ecent research has demonstrated quite convincingly that accurate 
cancer diagnosis can be achieved by constructing classifiers that are designed 
to compare the gene expression profile of a tissue of unknown cancer status to 
a database of stored expression profiles from tissues of known cancer status. 
This paper introduces the JCFO, a novel algorithm that uses a sparse Bayesian 
approach to jointly identify both the optimal nonlinear classifier for 
diagnosis and the optimal set of genes on which to base that diagnosis. We show 
that the diagnostic classification accuracy of the proposed algorithm is 
superior to a number of current state-of-the-art methods in a full 
leave-one-out cross-validation study of five widely used benchmark datasets. In 
addition to its superior classification accuracy, the algorithm is designed to 
automatically identify a small subset of genes (typically around twenty in our 
experiments) that are capable of providing complete discriminatory information 
for diagnosis. Focusing attention on a small subset of genes is useful not only 
because it produces a classifier with good generalization capacity, but also 
because this set of genes may provide insights into the mechanisms responsible 
for the disease itself. A number of the genes identified by the JCFO in our 
experiments are already in use as clinical markers for cancer diagnosis; some 
of the remaining genes may be excellent candidates for further clinical 
investigation. If it is possible to identify a small set of genes that is 
indeed capable of providing complete discrimination, inexpensive diagnostic 
assays might be widely deployable in clinical settings. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Krishnapuram2004Gene]  
B.&nbsp;Krishnapuram, L.&nbsp;Carin, and A.&nbsp;Hartemink. Gene expression 
analysis: joint feature selection and classifier design. In 
B.&nbsp;Sch&ouml;lkopf, K.&nbsp;Tsuda, and J.P. Vert, editors,<em>Kernel 
Methods in Computational Biology</em>, pages 299-317. MIT Press, 2004. [&nbsp;
bib&nbsp;| www:&nbsp;] 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Kote-Jarai2004Gene]  
Zsofia Kote-Jarai, Richard&nbsp;D Williams, Nicola Cattini, Maria Copeland, Ian 
Giddings, Richard Wooster, Robert&nbsp;H tePoele, Paul Workman, Barry 
Gusterson, John Peacock, Gerald Gui, Colin Campbell, and Ros Eeles. Gene 
expression profiling after radiation-induced DNA damage is strongly predictive 
of BRCA1 mutation carrier status.<em>Clin. Cancer Res.</em>, 10(3):958-63, Feb 
2004. [&nbsp;bib&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  PURPOSE: The impact of the presence of a germ-line BRCA1 
mutation on gene expression in normal breast fibroblasts after 
radiation-induced DNA damage has been investigated. EXPERIMENTAL DESIGN: 
High-density cDNA microarray technology was used to identify differential 
responses to DNA damage in fibroblasts from nine heterozygous BRCA1 mutation 
carriers compared with five control samples without personal or family history 
of any cancer. Fibroblast cultures were irradiated, and their expression 
profile was compared using intensity ratios of the cDNA microarrays 
representing 5603 IMAGE clones. RESULTS: Class comparison and class prediction 
analysis has shown that BRCA1 mutation carriers can be distinguished from 
controls with high probability (approximately 85%). Significance analysis of 
microarrays and the support vector machine classifier identified gene sets that 
discriminate the samples according to their mutation status. These include 
genes already known to interact with BRCA1 such as CDKN1B, ATR, and RAD51. 
CONCLUSIONS: The results of this initial study suggest that normal cells from 
heterozygous BRCA1 mutation carriers display a different gene expression 
profile from controls in response to DNA damage. Adaptations of this pilot 
result to other cell types could result in the development of a functional 
assay for BRCA1 mutation status. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm , breastcancer </blockquote> <br>
 [
Kondor2004Diffusion]  R.&nbsp;Kondor and J.-P. Vert. Diffusion kernels. In 
B.&nbsp;Sch&ouml;lkopf, K.&nbsp;Tsuda, and J.P. Vert, editors,<em>Kernel 
Methods in Computational Biology</em>, pages 171-192. MIT Press, 2004. [&nbsp;
bib&nbsp;| www:&nbsp;] 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Koike2004Prediction]  
A.&nbsp;Koike and T.&nbsp;Takagi. Prediction of protein-protein interaction 
sites using support vector machines.<em>Protein Eng. Des. Sel.</em>, 
17(2):165-173, Feb 2004. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  The identification of protein-protein interaction sites is 
essential for the mutant design and prediction of protein-protein networks. The 
interaction sites of residue units were predicted using support vector machines 
(SVM) and the profiles of sequentially/spatially neighboring residues, plus 
additional information. When only sequence information was used, prediction 
performance was highest using the feature vectors, sequentially neighboring 
profiles and predicted interaction site ratios, which were calculated by SVM 
regression using amino acid compositions. When structural information was also 
used, prediction performance was highest using the feature vectors, spatially 
neighboring residue profiles, accessible surface areas, and the with/without 
protein interaction sites ratios predicted by SVM regression and amino acid 
compositions. In the latter case, the precision at recall = 50 test set and 
&gt;20 30 closest sequentially/spatially neighboring on the interaction site 
residues. The predicted residues covered 86-87 (96-97 appeared to be slightly 
higher than a previously reported study. Comparing the prediction accuracy of 
each molecule, it seems to be easier to predict interaction sites for stable 
complexes. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Kohlmann2004Pediatric]  
A.&nbsp;Kohlmann, C.&nbsp;Schoch, S.&nbsp;Schnittger, M.&nbsp;Dugas, 
W.&nbsp;Hiddemann, W.&nbsp;Kern, and T.&nbsp;Haferlach. Pediatric acute 
lymphoblastic leukemia (ALL) gene expression signatures classify an independent 
cohort of adult ALL patients.<em>Leukemia</em>, 18(1):63-71, 2004. [&nbsp;bib
&nbsp;|DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  Recent reports support a possible future application of gene 
expression profiling for the diagnosis of leukemias. However, the robustness of 
subtype-specific gene expression signatures has to be proven on independent 
patient samples. Here, we present gene expression data of 34 adult acute 
lymphoblastic leukemia (ALL) patients (Affymetrix U133A microarrays). Support 
Vector Machines (SVMs) were applied to stratify our samples based on given gene 
lists reported to predict MLL, BCR-ABL, and T-ALL, as well as MLL and non-MLL 
gene rearrangement positive pediatric ALL. In addition, seven other B-precursor 
ALL cases not bearing t(9;22) or t(11q23)/MLL chromosomal aberrations were 
analyzed. Using top differentially expressed genes, hierarchical cluster and 
principal component analyses demonstrate that the genetically more 
heterogeneous B-precursor ALL samples intercalate with BCR-ABL-positive cases, 
but were clearly distinct from T-ALL and MLL profiles. Similar expression 
signatures were observed for both heterogeneous B-precursor ALL and for 
BCR-ABL-positive cases. As an unrelated laboratory, we demonstrate that gene 
signatures defined for childhood ALL were also capable of stratifying distinct 
subtypes in our cohort of adult ALL patients. As such, previously reported gene 
expression patterns identified by microarray technology are validated and 
confirmed on truly independent leukemia patient samples. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Kim2004Prediction]  
J.&nbsp;H. Kim, J.&nbsp;Lee, B.&nbsp;Oh, K.&nbsp;Kimm, and I.&nbsp;Koh. 
Prediction of phosphorylation sites using SVMs.<em>Bioinformatics</em>, 
20(17):3179-3184, 2004. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  Motivation: Phosphorylation is involved in diverse signal 
transduction pathways. By predicting phosphorylation sites and their kinases 
from primary protein sequences, we can obtain much valuable information that 
can form the basis for further research. Using support vector machines, we 
attempted to predict phosphorylation sites and the type of kinase that acts at 
each site. Results: Our prediction system was limited to phosphorylation sites 
catalyzed by four protein kinase families and four protein kinase groups. The 
accuracy of the predictions ranged from 83 to 95 kinase group level. The 
prediction system used-PredPhospho-can be applied to the functional study of 
proteins, and can help predict the changes in phosphorylation sites caused by 
amino acid variations at intra- and interspecies levels. Availability: 
PredPhospho is available at http://www.ngri.re.kr/proteo/PredPhospho.htm. 
Supplementary information: http://www.ngri.re.kr/proteo/supplementary.doc 
</blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Kim2004Predictiona]  
H.&nbsp;Kim and H.&nbsp;Park. Prediction of protein relative solvent 
accessibility with support vector machines and long-range interaction 3D local 
descriptor.<em>Proteins</em>, 54(3):557-562, Feb 2004. [&nbsp;bib&nbsp;| DOI
&nbsp;|http&nbsp;| .pdf&nbsp;] 
<blockquote>  The prediction of protein relative solvent accessibility gives 
us helpful information for the prediction of tertiary structure of a protein. 
The SVMpsi method, which uses support vector machines (SVMs), and the 
position-specific scoring matrix (PSSM) generated from PSI-BLAST have been 
applied to achieve better prediction accuracy of the relative solvent 
accessibility. We have introduced a three-dimensional local descriptor that 
contains information about the expected remote contacts by both the long-range 
interaction matrix and neighbor sequences. Moreover, we applied feature weights 
to kernels in SVMs in order to consider the degree of significance that depends 
on the distance from the specific amino acid. Relative solvent accessibility 
based on a two state-model, for 25 and 0 accuracy, respectively. Three-state 
prediction results provide a 64.5 approach has successfully been applied for 
solvent accessibility prediction by considering long-range interaction and 
handling unbalanced data. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Kashima2004Kernels]  
H.&nbsp;Kashima, K.&nbsp;Tsuda, and A.&nbsp;Inokuchi. Kernels for graphs. In 
B.&nbsp;Sch&ouml;lkopf, K.&nbsp;Tsuda, and J.P. Vert, editors,<em>Kernel 
Methods in Computational Biology</em>, pages 155-170. MIT Press, The MIT Press, 
Cambridge, Massachussetts, 2004. [&nbsp;bib&nbsp;] 
<blockquote>  Keywords: biosvm chemoinformatics </blockquote> <br>
 [
Jiang-Ning2004Cooperativity]  S.&nbsp;Jiang-Ning, L.&nbsp;Wei-Jiang, and 
X.&nbsp;Wen-Bo. Cooperativity of the oxidization of cysteines in globular 
proteins.<em>J. Theor. Biol.</em>, 231(1):85-95, 2004. [&nbsp;bib&nbsp;| DOI
&nbsp;|http&nbsp;| .pdf&nbsp;] 
<blockquote>  Based on the 639 non-homologous proteins with 2910 
cysteine-containing segments of well-resolved three-dimensional structures, a 
novel approach has been proposed to predict the disulfide-bonding state of 
cysteines in proteins by constructing a two-stage classifier combining a first 
global linear discriminator based on their amino acid composition and a second 
local support vector machine classifier. The overall prediction accuracy of 
this hybrid classifier for the disulfide-bonding state of cysteines in proteins 
has scored 84.1 on cysteine and protein basis using the rigorous jack-knife 
procedure, respectively. It shows that whether cysteines should form disulfide 
bonds depends not only on the global structural features of proteins but also 
on the local sequence environment of proteins. The result demonstrates the 
applicability of this novel method and provides comparable prediction 
performance compared with existing methods for the prediction of the oxidation 
states of cysteines in proteins. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Hutter2004Prediction]  
B.&nbsp;Hutter, C.&nbsp;Schaab, S.&nbsp;Albrecht, M.&nbsp;Borgmann, N.&nbsp;A. 
Brunner, C.&nbsp;Freiberg, K.&nbsp;Ziegelbauer, C.&nbsp;O. Rock, 
I.&nbsp;Ivanov, and H.&nbsp;Loferer. Prediction of Mechanisms of Action of 
Antibacterial Compounds by Gene Expression Profiling.<em>Antimicrob. Agents 
Chemother.</em>, 48(8):2838-2844, Aug 2004. [&nbsp;bib&nbsp;| DOI&nbsp;| arXiv
&nbsp;|http&nbsp;| .pdf&nbsp;] 
<blockquote>  We have generated a database of expression profiles carrying the 
transcriptional responses of the model organism Bacillus subtilis following 
treatment with 37 well-characterized antibacterial compounds of different 
classes. The database was used to build a predictor for the assignment of the 
mechanisms of action (MoAs) of antibacterial compounds by the use of support 
vector machines. This predictor was able to correctly classify the MoA class 
for most compounds tested. Furthermore, we provide evidence that the in vivo 
MoA of hexachlorophene does not match the MoA predicted from in vitro data, a 
situation frequently faced in drug discovery. A database of this kind may 
facilitate the prioritization of novel antibacterial entities in drug discovery 
programs. Potential applications and limitations are discussed. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Huan2004Accurate]  
J.&nbsp;Huan, W.&nbsp;Wang, A.&nbsp;Washington, J.&nbsp;Prins, R.&nbsp;Shah, 
and A.&nbsp;Tropsha. Accurate classification of protein structural families 
using coherent subgraph analysis. In<em>Proceedings of the Pacific Symposium on 
Biocomputing 2002</em>, pages 411-422, 2004. [&nbsp;bib&nbsp;| .pdf&nbsp;] 
<blockquote>  Protein structural annotation and classification is an important 
problem in bioinformatics. We report on the development of an efficient 
subgraph mining technique and its application to finding characteristic 
substructural patterns within protein structural families. In our method, 
protein structures are represented by graphs where the nodes are residues and 
the edges connect residues found within certain distance from each other. 
Application of subgraph mining to proteins is challenging for a number reasons: 
(1) protein graphs are large and complex, (2) current protein databases are 
large and continue to grow rapidly, and (3) only a small fraction of the 
frequent subgraphs among the huge pool of all possible subgraphs could be 
significant in the context of protein classification. To address these 
challenges, we have developed an information theoretic model called coherent 
subgraph mining. From information theory, the entropy of a random variable X 
measures the information content carried by X and the Mutual Information (MI) 
between two random variables X and Y measures the correlation between X and Y. 
We define a subgraph X as coherent if it is strongly correlated with every 
sufficiently large sub-subgraph Y embedded in it. Based on the MI metric, we 
have designed a search scheme that only reports coherent subgraphs. To 
determine the significance of coherent protein subgraphs, we have conducted an 
experimental study in which all coherent subgraphs were identified in several 
protein structural families annotated in the SCOP database (Murzin et al, 
1995). The Support Vector Machine algorithm was used to classify proteins from 
different families under the binary classification scheme. We find that this 
approach identifies spatial motifs unique to individual SCOP families and 
affords excellent discrimination between families. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Hu2004Improved]  H.J. Hu, 
Y.&nbsp;Pan, R.&nbsp;Harrison, and P.C. Tai. Improved protein secondary 
structure prediction using support vector machine with a new encoding scheme 
and an advanced tertiary classifier.<em>IEEE Trans. Nanobioscience</em>, 
3(4):265-271, 2004. [&nbsp;bib&nbsp;| .pdf&nbsp;] 
<blockquote>  Prediction of protein secondary structures is an important 
problem in bioinformatics and has many applications. The recent trend of 
secondary structure prediction studies is mostly based on the neural network or 
the support vector machine (SVM). The SVM method is a comparatively new 
learning system which has mostly been used in pattern recognition problems. In 
this study, SVM is used as a machine learning tool for the prediction of 
secondary structure and several encoding schemes, including orthogonal matrix, 
hydrophobicity matrix, BLOSUM62 substitution matrix, and combined matrix of 
these, are applied and optimized to improve the prediction accuracy. Also, the 
optimal window length for six SVM binary classifiers is established by testing 
different window sizes and our new encoding scheme is tested based on this 
optimal window size via sevenfold cross validation tests. The results show 2 
classifiers when compared with the instances in which the classical orthogonal 
matrix is used. Finally, to combine the results of the six SVM binary 
classifiers, a new tertiary classifier which combines the results of 
one-versus-one binary classifiers is introduced and the performance is compared 
with those of existing tertiary classifiers. According to the results, the Q3 
prediction accuracy of new tertiary classifier reaches 78.8 reported in the 
literature. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Hu2004Developing]  
C.&nbsp;Hu, X.&nbsp;Li, and J.&nbsp;Liang. Developing optimal non-linear 
scoring function for protein design.<em>Bioinformatics</em>, 20(17):3080-3098, 
2004. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| www:&nbsp;] 
<blockquote>  Motivation. Protein design aims to identify sequences compatible 
with a given protein fold but incompatible to any alternative folds. To select 
the correct sequences and to guide the search process, a design scoring 
function is critically important. Such a scoring function should be able to 
characterize the global fitness landscape of many proteins simultaneously. 
Results: To find optimal design scoring functions, we introduce two geometric 
views and propose a formulation using a mixture of non-linear Gaussian kernel 
functions. We aim to solve a simplified protein sequence design problem. Our 
goal is to distinguish each native sequence for a major portion of 
representative protein structures from a large number of alternative decoy 
sequences, each a fragment from proteins of different folds. Our scoring 
function discriminates perfectly a set of 440 native proteins from 14 million 
sequence decoys. We show that no linear scoring function can succeed in this 
task. In a blind test of unrelated proteins, our scoring function misclassfies 
only 13 native proteins out of 194. This compares favorably with about 
three-four times more misclassifications when optimal linear functions reported 
in the literature are used. We also discuss how to develop protein folding 
scoring function. Availability: Available on request from the authors. 
</blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Hou2004Remote]  
Y.&nbsp;Hou, W.&nbsp;Hsu, M.&nbsp;L. Lee, and C.&nbsp;Bystroff. Remote homolog 
detection using local sequence-structure correlations.<em>Proteins</em>, 
57(3):518-530, 2004. [&nbsp;bib&nbsp;| DOI&nbsp;| .pdf&nbsp;] 
<blockquote>  Remote homology detection refers to the detection of structural 
homology in proteins when there is little or no sequence similarity. In this 
article, we present a remote homolog detection method called SVM-HMMSTR that 
overcomes the reliance on detectable sequence similarity by transforming the 
sequences into strings of hidden Markov states that represent local folding 
motif patterns. These state strings are transformed into fixed-dimension 
feature vectors for input to a support vector machine. Two sets of features are 
defined: an order-independent feature set that captures the amino acid and 
local structure composition; and an order-dependent feature set that captures 
the sequential ordering of the local structures. Tests using the Structural 
Classification of Proteins (SCOP) 1.53 data set show that the SVM-HMMSTR gives 
a significant improvement over several current methods. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Hochreiter2004Gene]  
S.&nbsp;Hochreiter and K.&nbsp;Obermayer. Gene selection for microarray data. 
In B.&nbsp;Sch&ouml;lkopf, K.&nbsp;Tsuda, and J.P. Vert, editors,<em>Kernel 
Methods in Computational Biology</em>, pages 319-355. MIT Press, 2004. [&nbsp;
bib&nbsp;| www:&nbsp;] 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Helma2004Data]  
C.&nbsp;Helma, T.&nbsp;Cramer, S.&nbsp;Kramer, and L.&nbsp;De&nbsp;Raedt. Data 
mining and machine learning techniques for the identification of mutagenicity 
inducing substructures and structure activity relationships of noncongeneric 
compounds.<em>J. Chem. Inf. Comput. Sci.</em>, 44(4):1402-11, 2004. [&nbsp;bib
&nbsp;|DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  This paper explores the utility of data mining and machine 
learning algorithms for the induction of mutagenicity structure-activity 
relationships (SARs) from noncongeneric data sets. We compare (i) a newly 
developed algorithm (MOLFEA) for the generation of descriptors (molecular 
fragments) for noncongeneric compounds with traditional SAR approaches 
(molecular properties) and (ii) different machine learning algorithms for the 
induction of SARs from these descriptors. In addition we investigate the 
optimal parameter settings for these programs and give an exemplary 
interpretation of the derived models. The predictive accuracies of models using 
MOLFEA derived descriptors is approximately 10-15%age points higher than those 
using molecular properties alone. Using both types of descriptors together does 
not improve the derived models. From the applied machine learning techniques 
the rule learner PART and support vector machines gave the best results, 
although the differences between the learning algorithms are only marginal. We 
were able to achieve predictive accuracies up to 78% for 10-fold 
cross-validation. The resulting models are relatively easy to interpret and 
usable for predictive as well as for explanatory purposes. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm chemoinformatics </blockquote> <br>
 [
Han2004Prediction]  L.Y. Han, C.Z. Cai, S.L. Lo, M.C. Chung, and Y.Z. Chen. 
Prediction of RNA-binding proteins from primary sequence by a support vector 
machine approach.<em>RNA</em>, 10(3):355-368, 2004. [&nbsp;bib&nbsp;| http
&nbsp;|.pdf&nbsp;] 
<blockquote>  Elucidation of the interaction of proteins with different 
molecules is of significance in the understanding of cellular processes. 
Computational methods have been developed for the prediction of protein-protein 
interactions. But insufficient attention has been paid to the prediction of 
protein-RNA interactions, which play central roles in regulating gene 
expression and certain RNA-mediated enzymatic processes. This work explored the 
use of a machine learning method, support vector machines (SVM), for the 
prediction of RNA-binding proteins directly from their primary sequence. Based 
on the knowledge of known RNA-binding and non-RNA-binding proteins, an SVM 
system was trained to recognize RNA-binding proteins. A total of 4011 
RNA-binding and 9781 non-RNA-binding proteins was used to train and test the 
SVM classification system, and an independent set of 447 RNA-binding and 4881 
non-RNA-binding proteins was used to evaluate the classification accuracy. 
Testing results using this independent evaluation set show a prediction 
accuracy of 94.1 proteins, and 98.7 and non-tRNA-binding proteins, 
respectively. The SVM classification system was further tested on a small class 
of snRNA-binding proteins with only 60 available sequences. The prediction 
accuracy is 40.0 and 99.9 a need for a sufficient number of proteins to train 
SVM. The SVM classification systems trained in this work were added to our 
Web-based protein functional classification software SVMProt, at 
http://jing.cz3.nus.edu.sg/cgi-bin/svmprot.cgi. Our study suggests the 
potential of SVM as a useful tool for facilitating the prediction of 
protein-RNA interactions. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Han2004Predicting]  L.Y. 
Han, C.Z. Cai, Z.L. Ji, Z.W. Cao, J.&nbsp;Cui, and Y.Z. Chen. Predicting 
functional family of novel enzymes irrespective of sequence similarity: a 
statistical learning approach.<em>Nucl. Acids Res.</em>, 32(21):6437-6444, 
2004. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  The function of a protein that has no sequence homolog of known 
function is difficult to assign on the basis of sequence similarity. The same 
problem may arise for homologous proteins of different functions if one is 
newly discovered and the other is the only known protein of similar sequence. 
It is desirable to explore methods that are not based on sequence similarity. 
One approach is to assign functional family of a protein to provide useful hint 
about its function. Several groups have employed a statistical learning method, 
support vector machines (SVMs), for predicting protein functional family 
directly from sequence irrespective of sequence similarity. These studies 
showed that SVM prediction accuracy is at a level useful for functional family 
assignment. But its capability for assignment of distantly related proteins and 
homologous proteins of different functions has not been critically and 
adequately assessed. Here SVM is tested for functional family assignment of two 
groups of enzymes. One consists of 50 enzymes that have no homolog of known 
function from PSI-BLAST search of protein databases. The other contains eight 
pairs of homologous enzymes of different families. SVM correctly assigns 72 
pairs in the second group, suggesting that it is potentially useful for 
facilitating functional study of novel proteins. A web version of our software, 
SVMProt, is accessible at http://jing.cz3.nus.edu.sg/cgi-bin/svmprot.cgi. 
</blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Hakenberg2004Finding]  
J.&nbsp;Hakenberg, S.&nbsp;Schmeier, A.&nbsp;Kowald, E.&nbsp;Klipp, and 
U.&nbsp;Leser. Finding kinetic parameters using text mining.<em>OMICS</em>, 
8(2):131-152, 2004. [&nbsp;bib&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  The mathematical modeling and description of complex biological 
processes has become more and more important over the last years. Systems 
biology aims at the computational simulation of complex systems, up to whole 
cell simulations. An essential part focuses on solving a large number of 
parameterized differential equations. However, measuring those parameters is an 
expensive task, and finding them in the literature is very laborious. We 
developed a text mining system that supports researchers in their search for 
experimentally obtained parameters for kinetic models. Our system classifies 
full text documents regarding the question whether or not they contain 
appropriate data using a support vector machine. We evaluated our approach on a 
manually tagged corpus of 800 documents and found that it outperforms keyword 
searches in abstracts by a factor of five in terms of precision. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Gartner2004Kernels]  
T.&nbsp;G&auml;rtner, J.W. Lloyd, and P.A. Flach. Kernels and distances for 
structured data.<em>Mach. Learn.</em>, 57(3):205-232, 2004. [&nbsp;bib&nbsp;| 
DOI&nbsp;| http&nbsp;] 
<blockquote>  This paper brings together two strands of machine learning of 
increasing importance: kernel methods and highly structured data. We propose a 
general method for constructing a kernel following the syntactic structure of 
the data, as defined by its type signature in a higher-order logic. Our main 
theoretical result is the positive definiteness of any kernel thus defined. We 
report encouraging experimental results on a range of real-world data sets. By 
converting our kernel to a distance pseudo-metric for 1-nearest neighbour, we 
were able to improve the best accuracy from the literature on the Diterpene 
data set by more than 10%. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Guo2004novel]  
J.&nbsp;Guo, H.&nbsp;Chen, Z.&nbsp;Sun, and Y.&nbsp;Lin. A novel method for 
protein secondary structure prediction using dual-layer SVM and profiles.<em>
Proteins</em>, 54(4):738-743, 2004. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| 
.pdf&nbsp;] 
<blockquote>  A high-performance method was developed for protein secondary 
structure prediction based on the dual-layer support vector machine (SVM) and 
position-specific scoring matrices (PSSMs). SVM is a new machine learning 
technology that has been successfully applied in solving problems in the field 
of bioinformatics. The SVM's performance is usually better than that of 
traditional machine learning approaches. The performance was further improved 
by combining PSSM profiles with the SVM analysis. The PSSMs were generated from 
PSI-BLAST profiles, which contain important evolution information. The final 
prediction results were generated from the second SVM layer output. On the 
CB513 data set, the three-state overall per-residue accuracy, Q3, reached 75.2 
to 80.0 74.0 has been constructed and is available at 
http://www.bioinfo.tsinghua.edu.cn/pmsvm. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Guermeur2004Combining]  
Y.&nbsp;Guermeur, G.&nbsp;Pollastri, A.&nbsp;Elisseeff, D.&nbsp;Zelus, 
H.&nbsp;Paugam-Moisy, and P.&nbsp;Baldi. Combining protein secondary structure 
prediction models with ensemble methods of optimal complexity.<em>Neurocomputing
</em>, 56:305-327, 2004. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  Many sophisticated methods are currently available to perform 
protein secondary structure prediction. Since they are frequently based on 
different principles, and different knowledge sources, significant benefits can 
be expected from combining them. However, the choice of an appropriate combiner 
appears to be an issue in its own right. The first difficulty to overcome when 
combining prediction methods is overfitting. This is the reason why we 
investigate the implementation of Support Vector Machines to perform the task. 
A family of multi-class SVMs is introduced. Two of these machines are used to 
combine some of the current best protein secondary structure prediction 
methods. Their performance is consistently superior to the performance of the 
ensemble methods traditionally used in the field. They also outperform the 
decomposition approaches based on bi-class SVMs. Furthermore, initial 
experimental evidence suggests that their outputs could be processed by the 
biologist to perform higher-level treatments. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Guermeur2004kernel]  
Y.&nbsp;Guermeur, A.&nbsp;Lifschitz, and R.&nbsp;Vert. A kernel for protein 
secondary structure prediction. In B.&nbsp;Sch&ouml;lkopf, K.&nbsp;Tsuda, and 
J.P. Vert, editors,<em>Kernel Methods in Computational Biology</em>, pages 
193-206. MIT Press, 2004. [&nbsp;bib&nbsp;] 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Fong2004Predicting]  
J.&nbsp;H. Fong, A.&nbsp;E. Keating, and M.&nbsp;Singh. Predicting specificity 
in bZIP coiled-coil protein interactions.<em>Genome Biol.</em>, 5(R11), 2004. 
[&nbsp;bib&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  We present a method for predicting protein-protein interactions 
mediated by the coiled-coil motif. When tested on interactions between nearly 
all human and yeast bZIP proteins, our method identifies 70 strong interactions 
while maintaining that 92 correct. Furthermore, cross-validation testing shows 
that including the bZIP experimental data significantly improves performance. 
Our method can be used to predict bZIP interactions in other genomes and is a 
promising approach for predicting coiled-coil interactions more generally. 
</blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Cuturi2004mutual]  
M.&nbsp;Cuturi and J.-P. Vert. A mutual information kernel for strings. In<em>
Proceedings of IJCNN 2004</em>, pages 1904-1910, 2004. [&nbsp;bib&nbsp;| www:
&nbsp;]
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Cui2004Esub8]  
Q.&nbsp;Cui, T.&nbsp;Jiang, B.&nbsp;Liu, and S.&nbsp;Ma. Esub8: A novel tool to 
predict protein subcellular localizations in eukaryotic organisms.<em>BMC 
Bioinformatics</em>, 5(66):66, 2004. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| 
.pdf&nbsp;] 
<blockquote>  Background Subcellular localization of a new protein sequence is 
very important and fruitful for understanding its function. As the number of 
new genomes has dramatically increased over recent years, a reliable and 
efficient system to predict protein subcellular location is urgently needed. 
Results Esub8 was developed to predict protein subcellular localizations for 
eukaryotic proteins based on amino acid composition. In this research, the 
proteins are classified into the following eight groups: chloroplast, 
cytoplasm, extracellular, Golgi apparatus, lysosome, mitochondria, nucleus and 
peroxisome. We know subcellular localization is a typical classification 
problem; consequently, a one-against-one (1-v-1) multi-class support vector 
machine was introduced to construct the classifier. Unlike previous methods, 
ours considers the order information of protein sequences by a different 
method. Our method is tested in three subcellular localization predictions for 
prokaryotic proteins and four subcellular localization predictions for 
eukaryotic proteins on Reinhardt's dataset. The results are then compared to 
several other methods. The total prediction accuracies of two tests are both 
100 self-consistency test, and are 92.9 test, respectively. Esub8 also provides 
excellent results: the total prediction accuracies are 100 87 a different 
approach for predicting protein subcellular localization and achieved a 
satisfactory result; furthermore, we believe Esub8 will be a useful tool for 
predicting protein subcellular localizations in eukaryotic organisms. 
</blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Collier2004Comparison]  
Nigel Collier and Koichi Takeuchi. Comparison of character-level and part of 
speech features for name recognition in biomedical texts.<em>J Biomed Inform
</em>, 37(6):423-35, Dec 2004. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf
&nbsp;]
<blockquote>  The immense volume of data which is now available from 
experiments in molecular biology has led to an explosion in reported results 
most of which are available only in unstructured text format. For this reason 
there has been great interest in the task of text mining to aid in fact 
extraction, document screening, citation analysis, and linkage with large gene 
and gene-product databases. In particular there has been an intensive 
investigation into the named entity (NE) task as a core technology in all of 
these tasks which has been driven by the availability of high volume training 
sets such as the GENIA v3.02 corpus. Despite such large training sets accuracy 
for biology NE has proven to be consistently far below the high levels of 
performance in the news domain where F scores above 90 are commonly reported 
which can be considered near to human performance. We argue that it is crucial 
that more rigorous analysis of the factors that contribute to the model's 
performance be applied to discover where the underlying limitations are and 
what our future research direction should be. Our investigation in this paper 
reports on variations of two widely used feature types, part of speech (POS) 
tags and character-level orthographic features, and makes a comparison of how 
these variations influence performance. We base our experiments on a proven 
state-of-the-art model, support vector machines using a high quality subset of 
100 annotated MEDLINE abstracts. Experiments reveal that the best performing 
features are orthographic features with F score of 72.6. Although the Brill 
tagger trained in-domain on the GENIA v3.02p POS corpus gives the best overall 
performance of any POS tagger, at an F score of 68.6, this is still 
significantly below the orthographic features. In combination these two 
features types appear to interfere with each other and degrade performance 
slightly to an F score of 72.3. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm nlp </blockquote> <br>
 [Chen2004Prediction]  
Y.C. Chen, Y.S. Lin, C.J. Lin, and J.K. Hwang. Prediction of the bonding states 
of cysteines using the support vector machines based on multiple feature 
vectors and cysteine state sequences.<em>Proteins</em>, 55(4):1036-1042, 2004. 
[&nbsp;bib&nbsp;| DOI&nbsp;| .pdf&nbsp;| .pdf&nbsp;] 
<blockquote>  The support vector machine (SVM) method is used to predict the 
bonding states of cysteines. Besides using local descriptors such as the local 
sequences, we include global information, such as amino acid compositions and 
the patterns of the states of cysteines (bonded or nonbonded), or cysteine 
state sequences, of the proteins. We found that SVM based on local sequences or 
global amino acid compositions yielded similar prediction accuracies for the 
data set comprising 4136 cysteine-containing segments extracted from 969 
nonhomologous proteins. However, the SVM method based on multiple feature 
vectors (combining local sequences and global amino acid compositions) 
significantly improves the prediction accuracy, from 80 cysteine state 
sequences, SVM based on multiple feature vectors yields 90 coefficient, around 
10 obtained by SVM based on local sequence information. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Camps-Valls2004Profiled]  
G.&nbsp;Camps-Valls, A.M. Chalk, A.J. Serrano-Lopez, J.D. Martin-Guerrero, and 
E.L. Sonnhammer. Profiled support vector machines for antisense oligonucleotide 
efficacy prediction.<em>BMC Bioinformatics</em>, 5(135):135, 2004. [&nbsp;bib
&nbsp;|DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  Background This paper presents the use of Support Vector 
Machines (SVMs) for prediction and analysis of antisense oligonucleotide (AO) 
efficacy. The collected database comprises 315 AO molecules including 68 
features each, inducing a problem well-suited to SVMs. The task of feature 
selection is crucial given the presence of noisy or redundant features, and the 
well-known problem of the curse of dimensionality. We propose a two-stage 
strategy to develop an optimal model: (1) feature selection using correlation 
analysis, mutual information, and SVM-based recursive feature elimination 
(SVM-RFE), and (2) AO prediction using standard and profiled SVM formulations. 
A profiled SVM gives different weights to different parts of the training data 
to focus the training on the most important regions. Results In the first 
stage, the SVM-RFE technique was most efficient and robust in the presence of 
low number of samples and high input space dimension. This method yielded an 
optimal subset of 14 representative features, which were all related to energy 
and sequence motifs. The second stage evaluated the performance of the 
predictors (overall correlation coefficient between observed and predicted 
efficacy, r; mean error, ME; and root-mean-square-error, RMSE) using 8-fold and 
minus-one-RNA cross-validation methods. The profiled SVM produced the best 
results (r = 0.44, ME = 0.022, and RMSE= 0.278) and predicted high (&gt;75 gene 
expression) and low efficacy (&lt;25 of 83.3 approaches. A web server for AO 
prediction is available online at http://aosvm.cgb.ki.se/. Conclusions The SVM 
approach is well suited to the AO prediction problem, and yields a prediction 
accuracy superior to previous methods. The profiled SVM was found to perform 
better than the standard SVM, suggesting that it could lead to improvements in 
other prediction problems as well. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Cai2004Identify]  Y.D. 
Cai, G.P. Zhou, C.H. Jen, S.L. Lin, and K.C. Chou. Identify catalytic triads of 
serine hydrolases by support vector machines.<em>J. Theor. Biol.</em>, 
228(4):551-557, 2004. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  The core of an enzyme molecule is its active site from the 
viewpoints of both academic research and industrial application. To reveal the 
structural and functional mechanism of an enzyme, one needs to know its active 
site; to conduct structure-based drug design by regulating the function of an 
enzyme, one needs to know the active site and its microenvironment as well. 
Given the atomic coordinates of an enzyme molecule, how can we predict its 
active site? To tackle such a problem, a distance group approach was proposed 
and the support vector machine algorithm applied to predict the catalytic triad 
of serine hydrolase family. The success rate by jackknife test for the 139 
serine hydrolases was 85 promising and may become a useful tool in structural 
bioinformatics. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Cai2004Application]  Y.D. 
Cai, P.W. Ricardo, C.H. Jen, and K.C. Chou. Application of SVM to predict 
membrane protein types.<em>J. Theor. Biol.</em>, 226(4):373-376, 2004. [&nbsp;
bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  As a continuous effort to develop automated methods for 
predicting membrane protein types that was initiated by Chou and Elrod 
(PROTEINS: Structure, Function, and Genetics, 1999, 34, 137-153), the support 
vector machine (SVM) is introduced. Results obtained through re-substitution, 
jackknife, and independent data set tests, respectively, have indicated that 
the SVM approach is quite a promising one, suggesting that the covariant 
discriminant algorithm (Chou and Elrod, Protein Eng. 12 (1999) 107) and SVM, if 
effectively complemented with each other, will become a powerful tool for 
predicting membrane protein types and the other protein attributes as well. 
</blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Cai2004Enzyme]  C.Z. Cai, 
L.Y. Han, Z.L. Ji, and Y.Z. Chen. Enzyme family classification by support 
vector machines.<em>Proteins</em>, 55(1):66-76, 2004. [&nbsp;bib&nbsp;| DOI
&nbsp;|http&nbsp;| .pdf&nbsp;] 
<blockquote>  One approach for facilitating protein function prediction is to 
classify proteins into functional families. Recent studies on the 
classification of G-protein coupled receptors and other proteins suggest that a 
statistical learning method, Support vector machines (SVM), may be potentially 
useful for protein classification into functional families. In this work, SVM 
is applied and tested on the classification of enzymes into functional families 
defined by the Enzyme Nomenclature Committee of IUBMB. SVM classification 
system for each family is trained from representative enzymes of that family 
and seed proteins of Pfam curated protein families. The classification accuracy 
for enzymes from 46 families and for non-enzymes is in the range of 50.0 
Matthews correlation coefficient is in the range of 54.1 Moreover, 80.3 
classified into a specific enzyme family by using a scoring function, 
indicating that SVM may have certain level of unique prediction capability. 
Testing results also suggest that SVM in some cases is capable of 
classification of distantly related enzymes and homologous enzymes of different 
functions. Effort is being made to use a more comprehensive set of enzymes as 
training sets and to incorporate multi-class SVM classification systems to 
further enhance the unique prediction accuracy. Our results suggest the 
potential of SVM for enzyme family classification and for facilitating protein 
function prediction. Our software is accessible at 
http://jing.cz3.nus.edu.sg/cgi-bin/svmprot.cgi. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Byvatov2004SVM-based]  
Evgeny Byvatov and Gisbert Schneider. SVM-based feature selection for 
characterization of focused compound collections.<em>J Chem Inf Comput Sci</em>
, 44(3):993-9, 2004. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  Artificial neural networks, the support vector machine (SVM), 
and other machine learning methods for the classification of molecules are 
often considered as a &quot;black box&quot;, since the molecular features that 
are most relevant for a given classifier are usually not presented in a 
human-interpretable form. We report on an SVM-based algorithm for the selection 
of relevant molecular features from a trained classifier that might be 
important for an understanding of ligand-receptor interactions. The original 
SVM approach was extended to allow for feature selection. The method was 
applied to characterize focused libraries of enzyme inhibitors. A comparison 
with classical Kolmogorov-Smirnov (KS)-based feature selection was performed. 
In most of the applications the SVM method showed sustained classification 
accuracy, thereby relying on a smaller number of molecular features than 
KS-based classifiers. In one case both methods produced comparable results. 
Limiting the calculation of descriptors to only the most relevant ones for a 
certain biological activity can also be used to speed up high-throughput 
virtual screening. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm chemoinformatics featureselection </blockquote> 
<br>  [Busuttil2004Support]  S.&nbsp;Busuttil, J.&nbsp;Abela, and G.&nbsp;J. 
Pace. Support vector machines with profile-based kernels for remote protein 
homology detection.<em>Genome Inform Ser Workshop Genome Inform</em>, 
15(2):191-200, 2004. [&nbsp;bib&nbsp;| .html&nbsp;| .pdf&nbsp;] 
<blockquote>  Two new techniques for remote protein homology detection 
particulary suited for sparse data are introduced. These methods are based on 
position specific scoring matrices or profiles and use a support vector machine 
(SVM) for discrimination. The performance on standard benchmarks outperforms 
previous non-discriminative techniques and is comparable to that of other 
SVM-based methods while giving distinct advantages. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Bhasin2004SVM]  
M.&nbsp;Bhasin and G.&nbsp;P.&nbsp;S. Raghava. SVM based method for predicting 
HLA-DRB1*0401 binding peptides in an antigen sequence.<em>Bioinformatics</em>, 
20(3):421-423, 2004. [&nbsp;bib&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  Summary: Prediction of peptides binding with MHC class II allele 
HLA-DRB1*0401 can effectively reduce the number of experiments required for 
identifying helper T cell epitopes. This paper describes support vector machine 
(SVM) based method developed for identifying HLA-DRB1*0401 binding peptides in 
an antigenic sequence. SVM was trained and tested on large and clean data set 
consisting of 567 binders and equal number of non-binders. The accuracy of the 
method was 86 Available: A web server HLA-DR4Pred based on above approach is 
available at http://www.imtech.res.in/raghava/hladr4pred/ and 
http://bioinformatics.uams.edu/mirror/hladr4pred/ (Mirror Site). Supplementary 
information: http://www.imtech.res.in/raghava/hladr4pred/info.html </blockquote>
<p></p> 
<blockquote>  Keywords: biosvm immunoinformatics </blockquote> <br>
 [
Bhasin2004Prediction]  M.&nbsp;Bhasin and G.&nbsp;P.&nbsp;S. Raghava. 
Prediction of CTL epitopes using QM, SVM and ANN techniques.<em>Vaccine</em>, 
22(23-24):3195-3204, 2004. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  Cytotoxic T lymphocyte (CTL) epitopes are potential candidates 
for subunit vaccine design for various diseases. Most of the existing T cell 
epitope prediction methods are indirect methods that predict MHC class I 
binders instead of CTL epitopes. In this study, a systematic attempt has been 
made to develop a direct method for predicting CTL epitopes from an antigenic 
sequence. This method is based on quantitative matrix (QM) and machine learning 
techniques such as Support Vector Machine (SVM) and Artificial Neural Network 
(ANN). This method has been trained and tested on non-redundant dataset of T 
cell epitopes and non-epitopes that includes 1137 experimentally proven MHC 
class I restricted T cell epitopes. The accuracy of QM-, ANN- and SVM-based 
methods was 70.0, 72.2 and 75.2 has been evaluated through Leave One Out 
Cross-Validation (LOOCV) at a cutoff score where sensitivity and specificity 
was nearly equal. Finally, both machine-learning methods were used for 
consensus and combined prediction of CTL epitopes. The performances of these 
methods were evaluated on blind dataset where machine learning-based methods 
perform better than QM-based method. We also demonstrated through subgroup 
analysis that our methods can discriminate between T-cell epitopes and MHC 
binders (non-epitopes). In brief this method allows prediction of CTL epitopes 
using QM, SVM, ANN approaches. The method also facilitates prediction of MHC 
restriction in predicted T cell epitopes. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm immunoinformatics </blockquote> <br>
 [
Bhasin2004GPCRpred]  M.&nbsp;Bhasin and G.&nbsp;P.&nbsp;S. Raghava. GPCRpred: 
an SVM-based method for prediction of families and subfamilies of G-protein 
coupled receptors.<em>Nucl. Acids Res.</em>, 32(Supp.2):W383-389, 2004. [&nbsp;
bib&nbsp;| DOI&nbsp;| arXiv&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  G-protein coupled receptors (GPCRs) belong to one of the largest 
superfamilies of membrane proteins and are important targets for drug design. 
In this study, a support vector machine (SVM)-based method, GPCRpred, has been 
developed for predicting families and subfamilies of GPCRs from the dipeptide 
composition of proteins. The dataset used in this study for training and 
testing was obtained from http://www.soe.ucsc.edu/research/compbio/gpcr/. The 
method classified GPCRs and non-GPCRs with an accuracy of 99.5 evaluated using 
5-fold cross-validation. The method is further able to predict five major 
classes or families of GPCRs with an overall Matthew's correlation coefficient 
(MCC) and accuracy of 0.81 and 97.5 of the rhodopsin-like family, the method 
achieved an average MCC and accuracy of 0.97 and 97.3 overall accuracy of 91.3 
respectively when evaluated on an independent/blind dataset of 650 GPCRs. A 
server for recognition and classification of GPCRs based on multiclass SVMs has 
been set up at http://www.imtech.res.in/raghava/gpcrpred/. We have also 
suggested subfamilies for 42 sequences which were previously identified as 
unclassified ClassA GPCRs. The supplementary information is available at 
http://www.imtech.res.in/raghava/gpcrpred/info.html. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Bhasin2004ESLpred]  
M.&nbsp;Bhasin and G.&nbsp;P.&nbsp;S. Raghava. ESLpred: SVM-based method for 
subcellular localization of eukaryotic proteins using dipeptide composition and 
PSI-BLAST.<em>Nucl. Acids Res.</em>, 32(Suppl. 2):W414-419, 2004. [&nbsp;bib
&nbsp;|DOI&nbsp;| arXiv&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  Automated prediction of subcellular localization of proteins is 
an important step in the functional annotation of genomes. The existing 
subcellular localization prediction methods are based on either amino acid 
composition or N-terminal characteristics of the proteins. In this paper, 
support vector machine (SVM) has been used to predict the subcellular location 
of eukaryotic proteins from their different features such as amino acid 
composition, dipeptide composition and physico-chemical properties. The SVM 
module based on dipeptide composition performed better than the SVM modules 
based on amino acid composition or physico-chemical properties. In addition, 
PSI-BLAST was also used to search the query sequence against the dataset of 
proteins (experimentally annotated proteins) to predict its subcellular 
location. In order to improve the prediction accuracy, we developed a hybrid 
module using all features of a protein, which consisted of an input vector of 
458 dimensions (400 dipeptide compositions, 33 properties, 20 amino acid 
compositions of the protein and 5 from PSI-BLAST output). Using this hybrid 
approach, the prediction accuracies of nuclear, cytoplasmic, mitochondrial and 
extracellular proteins reached 95.3, 85.2, 68.2 and 88.9 overall prediction 
accuracy of SVM modules based on amino acid composition, physico-chemical 
properties, dipeptide composition and the hybrid approach was 78.1, 77.8, 82.9 
and 88.0 The accuracy of all the modules was evaluated using a 5-fold 
cross-validation technique. Assigning a reliability index (reliability index 
&gt; or =3), 73.5 on the above approach, an online web server ESLpred was 
developed, which is available at http://www.imtech.res.in/raghava/eslpred/. 
</blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Bhasin2004Classification]  
M.&nbsp;Bhasin and G.&nbsp;P.&nbsp;S. Raghava. Classification of Nuclear 
Receptors Based on Amino Acid Composition and Dipeptide Composition.<em>J. 
Biol. Chem.</em>, 279(22):23262-23266, 2004. [&nbsp;bib&nbsp;| DOI&nbsp;| arXiv
&nbsp;|http&nbsp;| .pdf&nbsp;] 
<blockquote>  Nuclear receptors are key transcription factors that regulate 
crucial gene networks responsible for cell growth, differentiation, and 
homeostasis. Nuclear receptors form a superfamily of phylogenetically related 
proteins and control functions associated with major diseases (e.g. diabetes, 
osteoporosis, and cancer). In this study, a novel method has been developed for 
classifying the subfamilies of nuclear receptors. The classification was 
achieved on the basis of amino acid and dipeptide composition from a sequence 
of receptors using support vector machines. The training and testing was done 
on a non-redundant data set of 282 proteins obtained from the NucleaRDB data 
base (1). The performance of all classifiers was evaluated using a 5-fold cross 
validation test. In the 5-fold cross-validation, the data set was randomly 
partitioned into five equal sets and evaluated five times on each distinct set 
while keeping the remaining four sets for training. It was found that different 
subfamilies of nuclear receptors were quite closely correlated in terms of 
amino acid composition as well as dipeptide composition. The overall accuracy 
of amino acid composition-based and dipeptide compositionbased classifiers were 
82.6 and 97.5 that different subfamilies of nuclear receptors are predictable 
with considerable accuracy using amino acid or dipeptide composition. 
Furthermore, based on above approach, an online web service, NRpred, was 
developed, which is available at www.imtech.res.in/raghava/nrpred. </blockquote>
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Bhasin2004Analysis]  
M.&nbsp;Bhasin and G.&nbsp;P.&nbsp;S. Raghava. Analysis and prediction of 
affinity of TAP binding peptides using cascade SVM.<em>Protein Sci.</em>, 
13(3):596-607, Mar 2004. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  The generation of cytotoxic T lymphocyte (CTL) epitopes from an 
antigenic sequence involves number of intracellular processes, including 
production of peptide fragments by proteasome and transport of peptides to 
endoplasmic reticulum through transporter associated with antigen processing 
(TAP). In this study, 409 peptides that bind to human TAP transporter with 
varying affinity were analyzed to explore the selectivity and specificity of 
TAP transporter. The abundance of each amino acid from P1 to P9 positions in 
high-, intermediate-, and low-affinity TAP binders were examined. The rules for 
predicting TAP binding regions in an antigenic sequence were derived from the 
above analysis. The quantitative matrix was generated on the basis of 
contribution of each position and residue in binding affinity. The correlation 
of r = 0.65 was obtained between experimentally determined and predicted 
binding affinity by using a quantitative matrix. Further a support vector 
machine (SVM)-based method has been developed to model the TAP binding affinity 
of peptides. The correlation (r = 0.80) was obtained between the predicted and 
experimental measured values by using sequence-based SVM. The reliability of 
prediction was further improved by cascade SVM that uses features of amino 
acids along with sequence. An extremely good correlation (r = 0.88) was 
obtained between measured and predicted values, when the cascade SVM-based 
method was evaluated through jackknife testing. A Web service, TAPPred 
(http://www.imtech.res.in/raghava/tappred/ or 
http://bioinformatics.uams.edu/mirror/tappred/), has been developed based on 
this approach. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Bern2004Automatic]  
M.&nbsp;Bern, D.&nbsp;Goldberg, W.&nbsp;H. McDonald, and III Yates, J.&nbsp;R. 
Automatic Quality Assessment of Peptide Tandem Mass Spectra.<em>Bioinformatics
</em>, 20(Suppl. 1):i49-i54, 2004. [&nbsp;bib&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  Motivation: A powerful proteomics methodology couples 
high-performance liquid chromatography (HPLC) with tandem mass spectrometry and 
database-search software, such as SEQUEST. Such a set-up, however, produces a 
large number of spectra, many of which are of too poor quality to be useful. 
Hence a filter that eliminates poor spectra before the database search can 
significantly improve throughput and robustness. Moreover, spectra judged to be 
of high quality, but that cannot be identified by database search, are prime 
candidates for still more computationally intensive methods, such as de novo 
sequencing or wider database searches including post-translational 
modifications. Results: We report on two different approaches to assessing 
spectral quality prior to identification: binary classification, which predicts 
whether or not SEQUEST will be able to make an identification, and statistical 
regression, which predicts a more universal quality metric involving the number 
of b- and y-ion peaks. The best of our binary classifiers can eliminate over 75 
spectra while losing only 10 regression can pick out spectra of modified 
peptides that can be identified by a de novo program but not by SEQUEST. In a 
section of independent interest, we discuss intensity normalization of mass 
spectra. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm proteomics </blockquote> <br>
 [
Baumgartner2004Supervised]  C.&nbsp;Baumgartner, C.&nbsp;Bohm, 
D.&nbsp;Baumgartner, G.&nbsp;Marini, K.&nbsp;Weinberger, B.&nbsp;Olgemoller, 
B.&nbsp;Liebl, and A.&nbsp;A. Roscher. Supervised machine learning techniques 
for the classification of metabolic disorders in newborns.<em>Bioinformatics
</em>, 20(17):2985-2996, 2004. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf
&nbsp;]
<blockquote>  Motivation: During the Bavarian newborn screening programme all 
newborns have been tested for about 20 inherited metabolic disorders. Owing to 
the amount and complexity of the generated experimental data, machine learning 
techniques provide a promising approach to investigate novel patterns in 
high-dimensional metabolic data which form the source for constructing 
classification rules with high discriminatory power. Results: Six machine 
learning techniques have been investigated for their classification accuracy 
focusing on two metabolic disorders, phenylketo nuria (PKU) and medium-chain 
acyl-CoA dehydrogenase deficiency (MCADD). Logistic regression analysis led to 
superior classification rules (sensitivity &gt;96.8 to all investigated 
algorithms. Including novel constellations of metabolites into the models, the 
positive predictive value could be strongly increased (PKU 71.9 54.6 clearly 
prove that the mined data confirm the known and indicate some novel metabolic 
patterns which may contribute to a better understanding of newborn metabolism. 
Availability: WEKA machine learning package: www.cs.waikato.ac.nz/&nbsp;ml/weka 
and statistical software package ADE-4: http://pbil.univ-lyon1.fr/ADE-4 
</blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm proteomics </blockquote> <br>
 [
Yap2004Prediction]  C.&nbsp;W. Yap, C.&nbsp;Z. Cai, Y.&nbsp;Xue, and Y.&nbsp;Z. 
Chen. Prediction of torsade-causing potential of drugs by support vector 
machine approach.<em>Toxicol Sci</em>, 79(1):170-7, May 2004. [&nbsp;bib&nbsp;| 
DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  In an effort to facilitate drug discovery, computational methods 
for facilitating the prediction of various adverse drug reactions (ADRs) have 
been developed. So far, attention has not been sufficiently paid to the 
development of methods for the prediction of serious ADRs that occur less 
frequently. Some of these ADRs, such as torsade de pointes (TdP), are important 
issues in the approval of drugs for certain diseases. Thus there is a need to 
develop tools for facilitating the prediction of these ADRs. This work explores 
the use of a statistical learning method, support vector machine (SVM), for TdP 
prediction. TdP involves multiple mechanisms and SVM is a method suitable for 
such a problem. Our SVM classification system used a set of linear solvation 
energy relationship (LSER) descriptors and was optimized by leave-one-out cross 
validation procedure. Its prediction accuracy was evaluated by using an 
independent set of agents and by comparison with results obtained from other 
commonly used classification methods using the same dataset and optimization 
procedure. The accuracies for the SVM prediction of TdP-causing agents and 
non-TdP-causing agents are 97.4 and 84.6% respectively; one is substantially 
improved against and the other is comparable to the results obtained by other 
classification methods useful for multiple-mechanism prediction problems. This 
indicates the potential of SVM in facilitating the prediction of TdP-causing 
risk of small molecules and perhaps other ADRs that involve multiple mechanisms.
</blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm chemoinformatics </blockquote> <br>
 [
Pavey2004Microarray]  S.&nbsp;Pavey, P.&nbsp;Johansson, L.&nbsp;Packer, 
J.&nbsp;Taylor, M.&nbsp;Stark, P.M. Pollock, G.J. Walker, G.M. Boyle, 
U.&nbsp;Harper, S.J. Cozzi, K.&nbsp;Hansen, L.&nbsp;Yudt, C.&nbsp;Schmidt, 
P.&nbsp;Hersey, K.A. Ellem, M.G. O'Rourke, P.G. Parsons, P.&nbsp;Meltzer, 
M.&nbsp;Ringner, and N.K. Hayward. Microarray expression profiling in melanoma 
reveals a BRAF mutation signature.<em>Oncogene</em>, 23(23):4060-4067, May 
2004. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  We have used microarray gene expression profiling and machine 
learning to predict the presence of BRAF mutations in a panel of 61 melanoma 
cell lines. The BRAF gene was found to be mutated in 42 samples (69 seven 
samples (11 Using support vector machines, we have built a classifier that 
differentiates between melanoma cell lines based on BRAF mutation status. As 
few as 83 genes are able to discriminate between BRAF mutant and BRAF wild-type 
samples with clear separation observed using hierarchical clustering. 
Multidimensional scaling was used to visualize the relationship between a BRAF 
mutation signature and that of a generalized mitogen-activated protein kinase 
(MAPK) activation (either BRAF or NRAS mutation) in the context of the 
discriminating gene list. We observed that samples carrying NRAS mutations lie 
somewhere between those with or without BRAF mutations. These observations 
suggest that there are gene-specific mutation signals in addition to a common 
MAPK activation that result from the pleiotropic effects of either BRAF or NRAS 
on other signaling pathways, leading to measurably different transcriptional 
changes. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm microarray </blockquote> <br>
 [
Cai2004Prediction]  Yu-Dong Cai and Andrew&nbsp;J Doig. Prediction of 
Saccharomyces cerevisiae protein functional class from functional domain 
composition.<em>Bioinformatics</em>, 20(8):1292-300, May 2004. [&nbsp;bib&nbsp;|
DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  MOTIVATION: A key goal of genomics is to assign function to 
genes, especially for orphan sequences. RESULTS: We compared the clustered 
functional domains in the SBASE database to each protein sequence using BLASTP. 
This representation for a protein is a vector, where each of the non-zero 
entries in the vector indicates a significant match between the sequence of 
interest and the SBASE domain. The machine learning methods nearest neighbour 
algorithm (NNA) and support vector machines are used for predicting protein 
functional classes from this information. We find that the best results are 
found using the SBASE-A database and the NNA, namely 72% accuracy for 79% 
coverage. We tested an assigning function based on searching for InterPro 
sequence motifs and by taking the most significant BLAST match within the 
dataset. We applied the functional domain composition method to predict the 
functional class of 2018 currently unclassified yeast open reading frames. 
AVAILABILITY: A program for the prediction method, that uses NNA called 
Functional Class Prediction based on Functional Domains (FCPFD) is available 
and can be obtained by contacting Y.D.Cai at y.cai@umist.ac.uk </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Zhou2005LS]  Xin Zhou and 
K.&nbsp;Z. Mao. LS Bound based gene selection for DNA microarray data.<em>
Bioinformatics</em>, 21(8):1559-64, Apr 2005. [&nbsp;bib&nbsp;| DOI&nbsp;| http
&nbsp;|.pdf&nbsp;] 
<blockquote>  MOTIVATION: One problem with discriminant analysis of DNA 
microarray data is that each sample is represented by quite a large number of 
genes, and many of them are irrelevant, insignificant or redundant to the 
discriminant problem at hand. Methods for selecting important genes are, 
therefore, of much significance in microarray data analysis. In the present 
study, a new criterion, called LS Bound measure, is proposed to address the 
gene selection problem. The LS Bound measure is derived from leave-one-out 
procedure of LS-SVMs (least squares support vector machines), and as the upper 
bound for leave-one-out classification results it reflects to some extent the 
generalization performance of gene subsets. RESULTS: We applied this LS Bound 
measure for gene selection on two benchmark microarray datasets: colon cancer 
and leukemia. We also compared the LS Bound measure with other evaluation 
criteria, including the well-known Fisher's ratio and Mahalanobis class 
separability measure, and other published gene selection algorithms, including 
Weighting factor and SVM Recursive Feature Elimination. The strength of the LS 
Bound measure is that it provides gene subsets leading to more accurate 
classification results than the filter method while its computational 
complexity is at the level of the filter method. AVAILABILITY: A companion 
website can be accessed at http://www.ntu.edu.sg/home5/pg02776030/lsbound/. The 
website contains: (1) the source code of the gene selection algorithm; (2) the 
complete set of tables and figures regarding the experimental study; (3) proof 
of the inequality (9). CONTACT: ekzmao@ntu.edu.sg. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm featureselection microarray </blockquote> <br>
 
[Zhou2005Recognition]  GuoDong Zhou, Dan Shen, Jie Zhang, Jian Su, and SoonHeng 
Tan. Recognition of protein/gene names from text using an ensemble of 
classifiers.<em>BMC Bioinformatics</em>, 6 Suppl 1:S7, 2005. [&nbsp;bib&nbsp;| 
DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  This paper proposes an ensemble of classifiers for biomedical 
name recognition in which three classifiers, one Support Vector Machine and two 
discriminative Hidden Markov Models, are combined effectively using a simple 
majority voting strategy. In addition, we incorporate three post-processing 
modules, including an abbreviation resolution module, a protein/gene name 
refinement module and a simple dictionary matching module, into the system to 
further improve the performance. Evaluation shows that our system achieves the 
best performance from among 10 systems with a balanced F-measure of 82.58 on 
the closed evaluation of the BioCreative protein/gene name recognition task 
(Task 1A). </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm nlp </blockquote> <br>
 [
Zhang2005Descriptor-based]  Z.&nbsp;Zhang, S.&nbsp;Kochhar, and M.&nbsp;G. 
Grigorov. Descriptor-based protein remote homology identification.<em>Protein 
Sci.</em>, 42(2):431-444, 2005. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf
&nbsp;]
<blockquote>  Here, we report a novel protein sequence descriptor-based remote 
homology identification method, able to infer fold relationships without the 
explicit knowledge of structure. In a first phase, we have individually 
benchmarked 13 different descriptor types in fold identification experiments in 
a highly diverse set of protein sequences. The relevant descriptors were 
related to the fold class membership by using simple similarity measures in the 
descriptor spaces, such as the cosine angle. Our results revealed that the 
three best-performing sets of descriptors were the sequence-alignment-based 
descriptor using PSI-BLAST e-values, the descriptors based on the alignment of 
secondary structural elements (SSEA), and the descriptors based on the 
occurrence of PROSITE functional motifs. In a second phase, the three 
top-performing descriptors were combined to obtain a final method with improved 
performance, which we named DescFold. Class membership was predicted by Support 
Vector Machine (SVM) learning. In comparison with the individual 
PSI-BLAST-based descriptor, the rate of remote homology identification 
increased from 33.7 was able to identify the true remote homolog for nearly 
every sixth sequence at the 95 PSI-BLAST search. We have benchmarked the 
DescFold method against several other state-of-the-art fold recognition 
algorithms for the 172 LiveBench-8 targets, and we concluded that it was able 
to add value to the existing techniques by providing a confident hit for at 
least 10 known methods. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Zaki2005Application]  
N.&nbsp;M. Zaki, S.&nbsp;Deris, and R.&nbsp;Illias. Application of string 
kernels in protein sequence classification.<em>Appl. Bioinformatics</em>, 
4(1):45-52, 2005. [&nbsp;bib&nbsp;] 
<blockquote>  INTRODUCTION: The production of biological information has 
become much greater than its consumption. The key issue now is how to organise 
and manage the huge amount of novel information to facilitate access to this 
useful and important biological information. One core problem in classifying 
biological information is the annotation of new protein sequences with 
structural and functional features. METHOD: This article introduces the 
application of string kernels in classifying protein sequences into homogeneous 
families. A string kernel approach used in conjunction with support vector 
machines has been shown to achieve good performance in text categorisation 
tasks. We evaluated and analysed the performance of this approach, and we 
present experimental results on three selected families from the SCOP 
(Structural Classification of Proteins) database. We then compared the overall 
performance of this method with the existing protein classification methods on 
benchmark SCOP datasets. RESULTS: According to the F1 performance measure and 
the rate of false positive (RFP) measure, the string kernel method performs 
well in classifying protein sequences. The method outperformed all the 
generative-based methods and is comparable with the SVM-Fisher method. 
DISCUSSION: Although the string kernel approach makes no use of prior 
biological knowledge, it still captures sufficient biological information to 
enable it to outperform some of the state-of-the-art methods. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Yu2005integrated]  J.-k. 
Yu, S.&nbsp;Zheng, Y.&nbsp;Tang, and L.&nbsp;Li. An integrated approach 
utilizing proteomics and bioinformatics to detect ovarian cancer.<em>J Zhejiang 
Univ Sci B</em>, 6(4):227-31, Apr 2005. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;|
.pdf&nbsp;] 
<blockquote>  OBJECTIVE: To find new potential biomarkers and establish the 
patterns for the detection of ovarian cancer. METHODS: Sixty one serum samples 
including 32 ovarian cancer patients and 29 healthy people were detected by 
surface-enhanced laser desorption/ionization mass spectrometry (SELDI-MS). The 
protein fingerprint data were analyzed by bioinformatics tools. Ten folds 
cross-validation support vector machine (SVM) was used to establish the 
diagnostic pattern. RESULTS: Five potential biomarkers were found (2085 Da, 
5881 Da, 7564 Da, 9422 Da, 6044 Da), combined with which the diagnostic pattern 
separated the ovarian cancer from the healthy samples with a sensitivity of 
96.7%, a specificity of 96.7% and a positive predictive value of 96.7%. 
CONCLUSIONS: The combination of SELDI with bioinformatics tools could find new 
biomarkers and establish patterns with high sensitivity and specificity for the 
detection of ovarian cancer. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Yu2005Classifying]  
C.&nbsp;Yu, N.&nbsp;Zavaljevski, F.&nbsp;J. Stevens, K.&nbsp;Yackovich, and 
J.&nbsp;Reifman. Classifying noisy protein sequence data: a case study of 
immunoglobulin light chains.<em>Bioinformatics</em>, 21(Supp 1):i495-i501, Jun 
2005. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  SUMMARY: The classification of protein sequences obtained from 
patients with various immunoglobulin-related conformational diseases may 
provide insight into structural correlates of pathogenicity. However, clinical 
data are very sparse and, in the case of antibody-related proteins, the 
collected sequences have large variability with only a small subset of 
variations relevant to the protein pathogenicity (function). On this basis, 
these sequences represent a model system for development of strategies to 
recognize the small subset of function-determining variations among the much 
larger number of primary structure diversifications introduced during 
evolution. Under such conditions, most protein classification algorithms have 
limited accuracy. To address this problem, we propose a support vector machine 
(SVM)-based classifier that combines sequence and 3D structural averaging 
information. Each amino acid in the sequence is represented by a set of six 
physicochemical properties: hydrophobicity, hydrophilicity, volume, surface 
area, bulkiness and refractivity. Each position in the sequence is described by 
the properties of the amino acid at that position and the properties of its 
neighbors in 3D space or in the sequence. A structure template is selected to 
determine neighbors in 3D space and a window size is used to determine the 
neighbors in the sequence. The test data consist of 209 proteins of human 
antibody immunoglobulin light chains, each represented by aligned sequences of 
120 amino acids. The methodology is applied to the classification of protein 
sequences collected from patients with and without amyloidosis, and indicates 
that the proposed modified classifiers are more robust to sequence variability 
than standard SVM classifiers, improving classification error between 5 and 25% 
and sensitivity between 9 and 17%. The classification results might also 
suggest possible mechanisms for the propensity of immunoglobulin light chains 
to amyloid formation. CONTACT: cyu@bioanalysis.org. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Yiu2005Filtering]  
S.&nbsp;M. Yiu, Prudence W.&nbsp;H. Wong, T.W. Lam, Y.C. Mui, H.&nbsp;F. Kung, 
Marie Lin, and Y.&nbsp;T. Cheung. Filtering of Ineffective siRNAs and Improved 
siRNA Design Tool.<em>Bioinformatics</em>, 21(2):144-151, Jan 2005. To appear. 
[&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  Motivation: Short interfering RNAs (siRNAs) can be used to 
suppress gene expression and possess many potential applications in therapy, 
but how to design an effective siRNA is still not clear. Based on the MPI 
(Max-Planck-Institute) basic principles, a number of siRNA design tools have 
been developed recently. The set of candidates reported by these tools is 
usually large and often contains ineffective siRNAs. In view of this, we 
initiate the study of filtering ineffective siRNAs. Results: The contribution 
of this paper is 2-fold. First, we propose a fair scheme to compare existing 
design tools based on real data in the literature. Second, we attempt to 
improve the MPI principles and existing tools by an algorithm that can filter 
ineffective siRNAs. The algorithm is based on some new observations on the 
secondary structure, which we have verified by AI techniques (decision trees 
and support vector machines). We have tested our algorithm together with the 
MPI principles and the existing tools. The results show that our filtering 
algorithm is effective. Availability: The siRNA design software tool can be 
found in the website http://www.cs.hku.hk/&nbsp;sirna/ Contact: smyiu@cs.hku.hk 
</blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Yap2005Prediction]  
C.&nbsp;W. Yap and Y.&nbsp;Z. Chen. Prediction of Cytochrome P450 3A4, 2D6, and 
2C9 Inhibitors and Substrates by Using Support Vector Machines.<em>J Chem Inf 
Model</em>, 45(4):982-92, 2005. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf
&nbsp;]
<blockquote>  Statistical learning methods have been used in developing 
filters for predicting inhibitors of two P450 isoenzymes, CYP3A4 and CYP2D6. 
This work explores the use of different statistical learning methods for 
predicting inhibitors of these enzymes and an additional P450 enzyme, CYP2C9, 
and the substrates of the three P450 isoenzymes. Two consensus support vector 
machine (CSVM) methods, &quot;positive majority&quot; (PM-CSVM) and 
&quot;positive probability&quot; (PP-CSVM), were used in this work. These 
methods were first tested for the prediction of inhibitors of CYP3A4 and CYP2D6 
by using a significantly higher number of inhibitors and noninhibitors than 
that used in earlier studies. They were then applied to the prediction of 
inhibitors of CYP2C9 and substrates of the three enzymes. Both methods predict 
inhibitors of CYP3A4 and CYP2D6 at a similar level of accuracy as those of 
earlier studies. For classification of inhibitors of CYP2C9, the best CSVM 
method gives an accuracy of 88.9% for inhibitors and 96.3% for noninhibitors. 
The accuracies for classification of substrates and nonsubstrates of CYP3A4, 
CYP2D6, and CYP2C9 are 98.2 and 90.9%, 96.6 and 94.4%, and 85.7 and 98.8%, 
respectively. Both CSVM methods are potentially useful as filters for 
predicting inhibitors and substrates of P450 isoenzymes. These methods 
generally give better accuracies than single SVM classification systems, and 
the performance of the PP-CSVM method is slightly better than that of the 
PM-CSVM method. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm chemoinformatics </blockquote> <br>
 [
Yamanishi2005Supervised]  Y.&nbsp;Yamanishi, J.-P. Vert, and M.&nbsp;Kanehisa. 
Supervised enzyme network inference from the integration of genomic data and 
chemical information.<em>Bioinformatics</em>, 21:i468-i477, 2005. [&nbsp;bib
&nbsp;|DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  Motivation: The metabolic network is an important biological 
network which relates enzyme proteins and chemical compounds. A large number of 
metabolic pathways remain unknown nowadays, and many enzymes are missing even 
in known metabolic pathways. There is, therefore, an incentive to develop 
methods to reconstruct the unknown parts of the metabolic network and to 
identify genes coding for missing enzymes. Results: This paper presents new 
methods to infer enzyme networks from the integration of multiple genomic data 
and chemical information, in the framework of supervised graph inference. The 
originality of the methods is the introduction of chemical compatibility as a 
constraint for refining the network predicted by the network inference engine. 
The chemical compatibility between two enzymes is obtained automatically from 
the information encoded by their Enzyme Commission (EC) numbers. The proposed 
methods are tested and compared on their ability to infer the enzyme network of 
the yeast Saccharomyces cerevisiae from four datasets for enzymes with assigned 
EC numbers: gene expression data, protein localization data, phylogenetic 
profiles and chemical compatibility information. It is shown that the 
prediction accuracy of the network reconstruction consistently improves owing 
to the introduction of chemical constraints, the use of a supervised approach 
and the weighted integration of multiple datasets. Finally, we conduct a 
comprehensive prediction of a global enzyme network consisting of all enzyme 
candidate proteins of the yeast to obtain new biological findings. </blockquote>
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Yabuki2005GRIFFIN]  
Y.&nbsp;Yabuki, T.&nbsp;Muramatsu, T.&nbsp;Hirokawa, H.&nbsp;Mukai, and 
M.&nbsp;Suwa. GRIFFIN: a system for predicting GPCR-G-protein coupling 
selectivity using a support vector machine and a hidden Markov model.<em>
Nucleic Acids Res.</em>, 33(Web Server issue):W148-53, Jul 2005. [&nbsp;bib
&nbsp;|DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  We describe a novel system, GRIFFIN (G-protein and Receptor 
Interaction Feature Finding INstrument), that predicts G-protein coupled 
receptor (GPCR) and G-protein coupling selectivity based on a support vector 
machine (SVM) and a hidden Markov model (HMM) with high sensitivity and 
specificity. Based on our assumption that whole structural segments of ligands, 
GPCRs and G-proteins are essential to determine GPCR and G-protein coupling, 
various quantitative features were selected for ligands, GPCRs and G-protein 
complex structures, and those parameters that are the most effective in 
selecting G-protein type were used as feature vectors in the SVM. The main part 
of GRIFFIN includes a hierarchical SVM classifier using the feature vectors, 
which is useful for Class A GPCRs, the major family. For the opsins and 
olfactory subfamilies of Class A and other minor families (Classes B, C, 
frizzled and smoothened), the binding G-protein is predicted with high accuracy 
using the HMM. Applying this system to known GPCR sequences, each binding 
G-protein is predicted with high sensitivity and specificity (&gt;85% on 
average). GRIFFIN (http://griffin.cbrc.jp/) is freely available and allows 
users to easily execute this reliable prediction of G-proteins. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Xie2005LOCSVMPSI]  Dan 
Xie, Ao&nbsp;Li, Minghui Wang, Zhewen Fan, and Huanqing Feng. LOCSVMPSI: a web 
server for subcellular localization of eukaryotic proteins using SVM and 
profile of PSI-BLAST.<em>Nucleic Acids Res.</em>, 33(Web Server issue):W105-10, 
Jul 2005. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  Subcellular location of a protein is one of the key functional 
characters as proteins must be localized correctly at the subcellular level to 
have normal biological function. In this paper, a novel method named LOCSVMPSI 
has been introduced, which is based on the support vector machine (SVM) and the 
position-specific scoring matrix generated from profiles of PSI-BLAST. With a 
jackknife test on the RH2427 data set, LOCSVMPSI achieved a high overall 
prediction accuracy of 90.2%, which is higher than the prediction results by 
SubLoc and ESLpred on this data set. In addition, prediction performance of 
LOCSVMPSI was evaluated with 5-fold cross validation test on the PK7579 data 
set and the prediction results were consistently better than the previous 
method based on several SVMs using composition of both amino acids and amino 
acid pairs. Further test on the SWISSPROT new-unique data set showed that 
LOCSVMPSI also performed better than some widely used prediction methods, such 
as PSORTII, TargetP and LOCnet. All these results indicate that LOCSVMPSI is a 
powerful tool for the prediction of eukaryotic protein subcellular 
localization. An online web server (current version is 1.3) based on this 
method has been developed and is freely available to both academic and 
commercial users, which can be accessed by at 
http://Bioinformatics.ustc.edu.cn/LOCSVMPSI/LOCSVMPSI.php. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Wang2005Gene]  
Yu&nbsp;Wang, Igor&nbsp;V Tetko, Mark&nbsp;A Hall, Eibe Frank, Axel Facius, 
Klaus F&nbsp;X Mayer, and Hans&nbsp;W Mewes. Gene selection from microarray 
data for cancer classification-a machine learning approach.<em>Comput. Biol. 
Chem.</em>, 29(1):37-46, Feb 2005. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf
&nbsp;]
<blockquote>  A DNA microarray can track the expression levels of thousands of 
genes simultaneously. Previous research has demonstrated that this technology 
can be useful in the classification of cancers. Cancer microarray data normally 
contains a small number of samples which have a large number of gene expression 
levels as features. To select relevant genes involved in different types of 
cancer remains a challenge. In order to extract useful gene information from 
cancer microarray data and reduce dimensionality, feature selection algorithms 
were systematically investigated in this study. Using a correlation-based 
feature selector combined with machine learning algorithms such as decision 
trees, na&iuml;ve Bayes and support vector machines, we show that 
classification performance at least as good as published results can be 
obtained on acute leukemia and diffuse large B-cell lymphoma microarray data 
sets. We also demonstrate that a combined use of different classification and 
feature selection approaches makes it possible to select relevant genes with 
high confidence. This is also the first paper which discusses both 
computational and biological evidence for the involvement of zyxin in 
leukaemogenesis. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm microarray </blockquote> <br>
 [
Wang2005Prediction]  Ming-Lei Wang, Hui Yao, and Wen-Bo Xu. Prediction by 
support vector machines and analysis by Z-score of poly-L-proline type II 
conformation based on local sequence.<em>Comput. Biol. Chem.</em>, 
29(2):95-100, Apr 2005. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  In recent years, the poly-L-proline type II (PPII) conformation 
has gained more and more importance. This structure plays vital roles in many 
biological processes. But few studies have been made to predict PPII secondary 
structures computationally. The support vector machine (SVM) represents a new 
approach to supervised pattern classification and has been successfully applied 
to a wide range of pattern recognition problems. In this paper, we present a 
SVM prediction method of PPII conformation based on local sequence. The overall 
accuracy for both the independent testing set and estimate of jackknife testing 
reached approximately 70%. Matthew's correlation coefficient (MCC) could reach 
0.4. By comparing the results of training and testing datasets with different 
sequence identities, we suggest that the performance of this method correlates 
with the sequence identity of dataset. The parameter of SVM kernel function was 
an important factor to the performance of this method. The propensities of 
residues located at different positions were also analyzed. By computing 
Z-scores, we found that P and G were the two most important residues to PPII 
structure conformation. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Wang2005Using]  
M.&nbsp;Wang, J.&nbsp;Yang, and K-C. Chou. Using string kernel to predict 
signal peptide cleavage site based on subsite coupling model.<em>Amino Acids
</em>, 28(4):395-402, Jun 2005. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf
&nbsp;]
<blockquote>  Owing to the importance of signal peptides for studying the 
molecular mechanisms of genetic diseases, reprogramming cells for gene therapy, 
and finding new drugs for healing a specific defect, it is in great demand to 
develop a fast and accurate method to identify the signal peptides. 
Introduction of the so-called -3,-1, +1 coupling model (Chou, K. C.: Protein 
Engineering, 2001, 14-2, 75-79) has made it possible to take into account the 
coupling effect among some key subsites and hence can significantly enhance the 
prediction quality of peptide cleavage site. Based on the subsite coupling 
model, a kind of string kernels for protein sequence is introduced. Integrating 
the biologically relevant prior knowledge, the constructed string kernels can 
thus be used by any kernel-based method. A Support vector machines (SVM) is 
thus built to predict the cleavage site of signal peptides from the protein 
sequences. The current approach is compared with the classical weight matrix 
method. At small false positive ratios, our method outperforms the classical 
weight matrix method, indicating the current approach may at least serve as a 
powerful complemental tool to other existing methods for predicting the signal 
peptide cleavage site.The software that generated the results reported in this 
paper is available upon requirement, and will appear at 
http://www.pami.sjtu.edu.cn/wm. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Wang2005Protein]  
J.&nbsp;Wang, W.-K. Sung, A.&nbsp;Krishnan, and K.-B. Li. Protein subcellular 
localization prediction for Gram-negative bacteria using amino acid 
subalphabets and a combination of multiple support vector machines.<em>BMC 
Bioinformatics</em>, 6(1):174, Jul 2005. [&nbsp;bib&nbsp;| DOI&nbsp;| http
&nbsp;|.pdf&nbsp;] 
<blockquote>  BACKGROUND: Predicting the subcellular localization of proteins 
is important for determining the function of proteins. Previous works focused 
on predicting protein localization in Gram-negative bacteria obtained good 
results. However, these methods had relatively low accuracies for the 
localization of extracellular proteins. This paper studies ways to improve the 
accuracy for predicting extracellular localization in Gram-negative bacteria. 
RESULTS: We have developed a system for predicting the subcellular localization 
of proteins for Gram-negative bacteria based on amino acid subalphabets and a 
combination of multiple support vector machines. The recall of the 
extracellular site and overall recall of our predictor reach 86.0% and 89.8%, 
respectively, in 5-fold cross-validation. To the best of our knowledge, these 
are the most accurate results for predicting subcellular localization in 
Gram-negative bacteria. CONCLUSIONS: Clustering 20 amino acids into a few 
groups by the proposed greedy algorithm provides a new way to extract features 
from protein sequences to cover more adjacent amino acids and hence reduce the 
dimensionality of the input vector of protein features. It was observed that a 
good amino acid grouping leads to an increase in prediction performance. 
Furthermore, a proper choice of a subset of complementary support vector 
machines constructed by different features of proteins maximizes the prediction 
accuracy. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Vlahovicek2005SBASE]  
Kristian Vlahovicek, L&Atilde;&Acirc;&iexcl;szl&oacute; 
Kaj&Atilde;&Acirc;&iexcl;n, Vilmos Agoston, and S&Atilde;&Acirc;&iexcl;ndor 
Pongor. The SBASE domain sequence resource, release 12: prediction of protein 
domain-architecture using support vector machines.<em>Nucleic Acids Res</em>, 
33(Database issue):D223-5, Jan 2005. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| 
.pdf&nbsp;] 
<blockquote>  SBASE (http://www.icgeb.trieste.it/sbase) is an online resource 
designed to facilitate the detection of domain homologies based on sequence 
database search. The present release of the SBASE A library of protein domain 
sequences contains 972,397 protein sequence segments annotated by structure, 
function, ligand-binding or cellular topology, clustered into 8547 domain 
groups. SBASE B contains 169,916 domain sequences clustered into 2526 less 
well-characterized groups. Domain prediction is based on an evaluation of 
database search results in comparison with a 'similarity network' of 
inter-sequence similarity scores, using support vector machines trained on 
similarity search results of known domains. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Vert2005Supervised]  J.-P. 
Vert and Y.&nbsp;Yamanishi. Supervised graph inference. In L.&nbsp;K. Saul, 
Y.&nbsp;Weiss, and L.&nbsp;Bottou, editors,<em>Adv. Neural Inform. Process. 
Syst.</em>, volume&nbsp;17, pages 1433-1440. MIT Press, Cambridge, MA, 2005. 
[&nbsp;bib&nbsp;| www:&nbsp;] 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Vert2005Kernel]  J.-P. 
Vert. Kernel methods in computational biology. Technical Report ccsd-00012124, 
CNRS-HAL, Oct 2005. [&nbsp;bib&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  Support vector machines and kernel methods are increasingly 
popular in genomics and computational biology, due to their good performance in 
real-world applications and strong modularity that makes them suitable to a 
wide range of problems, from the classification of tumors to the automatic 
annotation of proteins. Their ability to work in high dimension, to process 
non-vectorial data, and the natural framework they provide to integrate 
heterogeneous data are particularly relevant to various problems arising in 
computational biology. In this chapter we survey some of the most prominent 
applications published so far, highlighting the particular developments in 
kernel methods triggered by problems in biology, and mention a few promising 
research directions likely to expand in the future. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Tsirigos2005sensitive]  
A.&nbsp;Tsirigos and I.&nbsp;Rigoutsos. A sensitive, support-vector-machine 
method for the detection of horizontal gene transfers in viral, archaeal and 
bacterial genomes.<em>Nucleic Acids Res.</em>, 33(12):3699-707, 2005. [&nbsp;bib
&nbsp;|DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  In earlier work, we introduced and discussed a generalized 
computational framework for identifying horizontal transfers. This framework 
relied on a gene's nucleotide composition, obviated the need for knowledge of 
codon boundaries and database searches, and was shown to perform very well 
across a wide range of archaeal and bacterial genomes when compared with 
previously published approaches, such as Codon Adaptation Index and C + G 
content. Nonetheless, two considerations remained outstanding: we wanted to 
further increase the sensitivity of detecting horizontal transfers and also to 
be able to apply the method to increasingly smaller genomes. In the discussion 
that follows, we present such a method, Wn-SVM, and show that it exhibits a 
very significant improvement in sensitivity compared with earlier approaches. 
Wn-SVM uses a one-class support-vector machine and can learn using rather small 
training sets. This property makes Wn-SVM particularly suitable for studying 
small-size genomes, similar to those of viruses, as well as the typically 
larger archaeal and bacterial genomes. We show experimentally that the new 
method results in a superior performance across a wide range of organisms and 
that it improves even upon our own earlier method by an average of 10% across 
all examined genomes. As a small-genome case study, we analyze the genome of 
the human cytomegalovirus and demonstrate that Wn-SVM correctly identifies 
regions that are known to be conserved and prototypical of all 
beta-herpesvirinae, regions that are known to have been acquired horizontally 
from the human host and, finally, regions that had not up to now been suspected 
to be horizontally transferred. Atypical region predictions for many eukaryotic 
viruses, including the alpha-, beta- and gamma-herpesvirinae, and 123 archaeal 
and bacterial genomes, have been made available online at 
http://cbcsrv.watson.ibm.com/HGT_SVM/. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Tobita2005discriminant]  
M.&nbsp;Tobita, T.&nbsp;Nishikawa, and R.&nbsp;Nagashima. A discriminant model 
constructed by the support vector machine method for HERG potassium channel 
inhibitors.<em>Bioorg. Med. Chem. Lett.</em>, 15(11):2886-90, Jun 2005. [&nbsp;
bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  HERG attracts attention as a risk factor for arrhythmia, which 
might trigger torsade de pointes. A highly accurate classifier of chemical 
compounds for inhibition of the HERG potassium channel is constructed using 
support vector machine. For two test sets, our discriminant models achieved 90% 
and 95% accuracy, respectively. The classifier is even applied for the 
prediction of cardio vascular adverse effects to achieve about 70% accuracy. 
While modest inhibitors are partly characterized by properties linked to global 
structure of a molecule including hydrophobicity and diameter, strong 
inhibitors are exclusively characterized by properties linked to substructures 
of a molecule. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm chemoinformatics herg </blockquote> <br>
 [
Tang2005Discovering]  Thomas Tang, Jinbo Xu, and Ming Li. Discovering 
sequence-structure motifs from protein segments and two applications.<em>Pac 
Symp Biocomput</em>, pages 370-81, 2005. [&nbsp;bib&nbsp;] 
<blockquote>  We present a novel method for clustering short protein segments 
having strong sequence-structure correlations, and demonstrate that these 
clusters contain useful structural information via two applications. When 
applied to local tertiary structure prediction, we achieve approximately 60% 
accuracy with a novel dynamic programming algorithm. When applied to secondary 
structure prediction based on Support Vector Machines, we obtain a 
approximately 2% gain in Q3 performance by incorporating cluster-derived data 
into training and classification. These encouraging results illustrate the 
great potential of using conserved local motifs to tackle protein structure 
predictions and possibly other important problems in biology. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Takeuchi2005Bio-medical]  
Koichi Takeuchi and Nigel Collier. Bio-medical entity extraction using support 
vector machines.<em>Artif. Intell. Med.</em>, 33(2):125-37, Feb 2005. [&nbsp;bib
&nbsp;|DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  OBJECTIVE: Support vector machines (SVMs) have achieved 
state-of-the-art performance in several classification tasks. In this article 
we apply them to the identification and semantic annotation of scientific and 
technical terminology in the domain of molecular biology. This illustrates the 
extensibility of the traditional named entity task to special domains with 
large-scale terminologies such as those in medicine and related disciplines. 
METHODS AND MATERIALS: The foundation for the model is a sample of text 
annotated by a domain expert according to an ontology of concepts, properties 
and relations. The model then learns to annotate unseen terms in new texts and 
contexts. The results can be used for a variety of intelligent language 
processing applications. We illustrate SVMs capabilities using a sample of 100 
journal abstracts texts taken from the human, blood cell, transcription factor 
domain of MEDLINE. RESULTS: Approximately 3400 terms are annotated and the 
model performs at about 74% F-score on cross-validation tests. A detailed 
analysis based on empirical evidence shows the contribution of various feature 
sets to performance. CONCLUSION: Our experiments indicate a relationship 
between feature window size and the amount of training data and that a 
combination of surface words, orthographic features and head noun features 
achieve the best performance among the feature sets tested. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Swamidass2005Kernels]  
S.&nbsp;J. Swamidass, J.&nbsp;Chen, J.&nbsp;Bruand, P.&nbsp;Phung, 
L.&nbsp;Ralaivola, and P.&nbsp;Baldi. Kernels for small molecules and the 
prediction of mutagenicity, toxicity and anti-cancer activity.<em>Bioinformatics
</em>, 21(Suppl. 1):i359-i368, Jun 2005. [&nbsp;bib&nbsp;| DOI&nbsp;| http
&nbsp;|.pdf&nbsp;] 
<blockquote>  MOTIVATION: Small molecules play a fundamental role in organic 
chemistry and biology. They can be used to probe biological systems and to 
discover new drugs and other useful compounds. As increasing numbers of large 
datasets of small molecules become available, it is necessary to develop 
computational methods that can deal with molecules of variable size and 
structure and predict their physical, chemical and biological properties. 
RESULTS: Here we develop several new classes of kernels for small molecules 
using their 1D, 2D and 3D representations. In 1D, we consider string kernels 
based on SMILES strings. In 2D, we introduce several similarity kernels based 
on conventional or generalized fingerprints. Generalized fingerprints are 
derived by counting in different ways subpaths contained in the graph of bonds, 
using depth-first searches. In 3D, we consider similarity measures between 
histograms of pairwise distances between atom classes. These kernels can be 
computed efficiently and are applied to problems of classification and 
prediction of mutagenicity, toxicity and anti-cancer activity on three publicly 
available datasets. The results derived using cross-validation methods are 
state-of-the-art. Tradeoffs between various kernels are briefly discussed. 
AVAILABILITY: Datasets available from 
http://www.igb.uci.edu/servers/servers.html CONTACT: pfbaldi@ics.uci.edu. 
</blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Statnikov2005comprehensive]
 A.&nbsp;Statnikov, C.&nbsp;F. Aliferis, I.&nbsp;Tsamardinos, D.&nbsp;Hardin, 
and S.&nbsp;Levy. A comprehensive evaluation of multicategory classification 
methods for microarray gene expression cancer diagnosis.<em>Bioinformatics</em>
, 2005. To appear. [&nbsp;bib&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  Motivation: Cancer diagnosis is one of the most important 
emerging clinical applications of gene expression microarray technology. We are 
seeking to develop a computer system for powerful and reliable cancer 
diagnostic model creation based on microarray data. To keep a realistic 
perspective on clinical applications we focus on multicategory diagnosis. In 
order to equip the system with the optimum combination of classifier, gene 
selection and cross-validation methods, we performed a systematic and 
comprehensive evaluation of several major algorithms for multicategory 
classification, several gene selection methods, multiple ensemble classifier 
methods, and two cross validation designs using 11 datasets spanning 74 
diagnostic categories and 41 cancer types and 12 normal tissue types.Results: 
Multicategory Support Vector Machines (MC-SVMs) are the most effective 
classifiers in performing accurate cancer diagnosis from gene expression data. 
The MC-SVM techniques by Crammer and Singer, Weston and Watkins, and 
one-versus-rest were found to be the best methods in this domain. MC-SVMs 
outperform other popular machine learning algorithms such as K-Nearest 
Neighbors, Backpropagation and Probabilistic Neural Networks, often to a 
remarkable degree. Gene selection techniques can significantly improve 
classification performance of both MC-SVMs and other non-SVM learning 
algorithms. Ensemble classifiers do not generally improve performance of the 
best non-ensemble models. These results guided the construction of a software 
system GEMS (Gene Expression Model Selector) that automates high-quality model 
construction and enforces sound optimization and performance estimation 
procedures. This is the first such system to be informed by a rigorous 
comparative analysis of the available algorithms and datasets.Availability: The 
software system GEMS is available for download from http://www.gems-system.org 
for non-commercial use. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm microarray </blockquote> <br>
 [Shi2005Building]
 Lei Shi and Fabien Campagne. Building a protein name dictionary from full 
text: a machine learning term extraction approach.<em>BMC Bioinformatics</em>, 
6(1):88, Apr 2005. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  BACKGROUND: The majority of information in the biological 
literature resides in full text articles, instead of abstracts. Yet, abstracts 
remain the focus of many publicly available literature data mining tools. Most 
literature mining tools rely on pre-existing lexicons of biological names, 
often extracted from curated gene or protein databases. This is a limitation, 
because such databases have low coverage of the many name variants which are 
used to refer to biological entities in the literature. RESULTS: We present an 
approach to recognize named entities in full text. The approach collects high 
frequency terms in an article, and uses support vector machines (SVM) to 
identify biological entity names. It is also computationally efficient and 
robust to noise commonly found in full text material. We use the method to 
create a protein name dictionary from a set of 80,528 full text articles. Only 
8.3% of the names in this dictionary match SwissProt description lines. We 
assess the quality of the dictionary by studying its protein name recognition 
performance in full text. CONCLUSION: This dictionary term lookup method 
compares favourably to other published methods, supporting the significance of 
our direct extraction approach. The method is strong in recognizing name 
variants not found in SwissProt. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Sharan2005motif-based]  
Roded Sharan and Eugene&nbsp;W Myers. A motif-based framework for recognizing 
sequence families.<em>Bioinformatics</em>, 21 Suppl 1:i387-i393, Jun 2005. 
[&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  MOTIVATION: Many signals in biological sequences are based on 
the presence or absence of base signals and their spatial combinations. One of 
the best known examples of this is the signal identifying a core promoter-the 
site at which the basal transcription machinery starts the transcription of a 
gene. Our goal is a fully automatic pattern recognition system for a family of 
sequences, which simultaneously discovers the base signals, their spatial 
relationships and a classifier based upon them. RESULTS: In this paper we 
present a general method for characterizing a set of sequences by their 
recurrent motifs. Our approach relies on novel probabilistic models for DNA 
binding sites and modules of binding sites, on algorithms to study them from 
the data and on a support vector machine that uses the models studied to 
classify a set of sequences. We demonstrate the applicability of our approach 
to diverse instances, ranging from families of promoter sequences to a dataset 
of intronic sequences flanking alternatively spliced exons. On a core promoter 
dataset our results are comparable with the state-of-the-art McPromoter. On a 
dataset of alternatively spliced exons we outperform a previous approach. We 
also achieve high success rates in recognizing cell cycle regulated genes. 
These results demonstrate that a fully automatic pattern recognition algorithm 
can meet or exceed the performance of hand-crafted approaches. AVAILABILITY: 
The software and datasets are available from the authors upon request. CONTACT: 
roded@tau.ac.il. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Senawongse2005Predicting]  
Pasak Senawongse, Andrew&nbsp;R Dalby, and Zheng&nbsp;Rong Yang. Predicting the 
phosphorylation sites using hidden markov models and machine learning methods.
<em>J Chem Inf Model</em>, 45(4):1147-52, 2005. [&nbsp;bib&nbsp;| DOI&nbsp;| 
http&nbsp;| .pdf&nbsp;] 
<blockquote>  Accurately predicting phosphorylation sites in proteins is an 
important issue in postgenomics, for which how to efficiently extract the most 
predictive features from amino acid sequences for modeling is still 
challenging. Although both the distributed encoding method and the bio-basis 
function method work well, they still have some limits in use. The distributed 
encoding method is unable to code the biological content in sequences 
efficiently, whereas the bio-basis function method is a nonparametric method, 
which is often computationally expensive. As hidden Markov models (HMMs) can be 
used to generate one model for one cluster of aligned protein sequences, the 
aim in this study is to use HMMs to extract features from amino acid sequences, 
where sequence clusters are determined using available biological knowledge. In 
this novel method, HMMs are first constructed using functional sequences only. 
Both functional and nonfunctional training sequences are then inputted into the 
trained HMMs to generate functional and nonfunctional feature vectors. From 
this, a machine learning algorithm is used to construct a classifier based on 
these feature vectors. It is found in this work that (1) this method provides 
much better prediction accuracy than the use of HMMs only for prediction, and 
(2) the support vector machines (SVMs) algorithm outperforms decision trees and 
neural network algorithms when they are constructed on the features extracted 
using the trained HMMs. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Seike2005Proteomic]  
M.&nbsp;Seike, T.&nbsp;Kondo, K.&nbsp;Fujii, T.&nbsp;Okano, T.&nbsp;Yamada, 
Y.&nbsp;Matsuno, A.&nbsp;Gemma, S.&nbsp;Kudoh, and S.&nbsp;Hirohashi. Proteomic 
signatures for histological types of lung cancer.<em>Proteomics</em>, Jul 2005. 
[&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  We performed proteomic studies on lung cancer cells to elucidate 
the mechanisms that determine histological phenotype. Thirty lung cancer cell 
lines with three different histological backgrounds (squamous cell carcinoma, 
small cell lung carcinoma and adenocarcinoma) were subjected to two-dimensional 
difference gel electrophoresis (2-D DIGE) and grouped by multivariate analyses 
on the basis of their protein expression profiles. 2-D DIGE achieves more 
accurate quantification of protein expression by using highly sensitive 
fluorescence dyes to label the cysteine residues of proteins prior to 
two-dimensional polyacrylamide gel electrophoresis. We found that hierarchical 
clustering analysis and principal component analysis divided the cell lines 
according to their original histology. Spot ranking analysis using a support 
vector machine algorithm and unsupervised classification methods identified 32 
protein spots essential for the classification. The proteins corresponding to 
the spots were identified by mass spectrometry. Next, lung cancer cells 
isolated from tumor tissue by laser microdissection were classified on the 
basis of the expression pattern of these 32 protein spots. Based on the 
expression profile of the 32 spots, the isolated cancer cells were categorized 
into three histological groups: the squamous cell carcinoma group, the 
adenocarcinoma group, and a group of carcinomas with other histological types. 
In conclusion, our results demonstrate the utility of quantitative proteomic 
analysis for molecular diagnosis and classification of lung cancer cells. 
</blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm proteomics </blockquote> <br>
 [Sarda2005pSLIP] 
 Deepak Sarda, Gek&nbsp;Huey Chua, Kuo-Bin Li, and Arun Krishnan. pSLIP: SVM 
based protein subcellular localization prediction using multiple 
physicochemical properties.<em>BMC Bioinformatics</em>, 6(1):152, Jun 2005. 
[&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  BACKGROUND: Protein subcellular localization is an important 
determinant of protein function and hence, reliable methods for prediction of 
localization are needed. A number of prediction algorithms have been developed 
based on amino acid compositions or on the N-terminal characteristics (signal 
peptides) of proteins. However, such approaches lead to a loss of contextual 
information. Moreover, where information about the physicochemical properties 
of amino acids has been used, the methods employed to exploit that information 
are less than optimal and could use the information more effectively. RESULTS: 
In this paper, we propose a new algorithm called pSLIP which uses Support 
Vector Machines (SVMs) in conjunction with multiple physicochemical properties 
of amino acids to predict protein subcellular localization in eukaryotes across 
six different locations, namely, chloroplast, cytoplasmic, extracellular, 
mitochondrial, nuclear and plasma membrane. The algorithm was applied to the 
dataset provided by Park and Kanehisa and we obtained prediction accuracies for 
the different classes ranging from 87.7%-97.0% with an overall accuracy of 
93.1%. CONCLUSIONS: This study presents a physicochemical property based 
protein localization prediction algorithm. Unlike other algorithms, contextual 
information is preserved by dividing the protein sequences into clusters. The 
prediction accuracy shows an improvement over other algorithms based on various 
types of amino acid composition (single, pair and gapped pair). We have also 
implemented a web server to predict protein localization across the six classes 
(available at http://pslip.bii.a-star.edu.sg). </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Saeh2005Lead]  
J.&nbsp;Saeh, P.&nbsp;Lyne, B.&nbsp;Takasaki, and D.&nbsp;Cosgrove. Lead 
hopping using SVM and 3D pharmacophore fingerprints.<em>J Chem Inf Model</em>, 
45(4):1122-1133, Jul 2005. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  The combination of 3D pharmacophore fingerprints and the support 
vector machine classification algorithm has been used to generate robust models 
that are able to classify compounds as active or inactive in a number of 
G-protein-coupled receptor assays. The models have been tested against 
progressively more challenging validation sets where steps are taken to ensure 
that compounds in the validation set are chemically and structurally distinct 
from the training set. In the most challenging example, we simulate a 
lead-hopping experiment by excluding an entire class of compounds (defined by a 
core substructure) from the training set. The left-out active compounds 
comprised approximately 40% of the actives. The model trained on the remaining 
compounds is able to recall 75% of the actives from the &quot;new&quot; lead 
series while correctly classifying &gt;99% of the 5000 inactives included in 
the validation set. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm chemoinformatics </blockquote> <br>
 [
Raetsch2005RASE]  G.&nbsp;R&auml;tsch, S.&nbsp;Sonnenburg, and 
B.&nbsp;Sch&ouml;lkopf. RASE: recognition of alternatively spliced exons in 
C.elegans.<em>Bioinformatics</em>, 21(Suppl. 1):i369-i377, Jun 2005. [&nbsp;bib
&nbsp;|DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  MOTIVATION: Eukaryotic pre-mRNAs are spliced to form mature 
mRNA. Pre-mRNA alternative splicing greatly increases the complexity of gene 
expression. Estimates show that more than half of the human genes and at least 
one-third of the genes of less complex organisms, such as nematodes or flies, 
are alternatively spliced. In this work, we consider one major form of 
alternative splicing, namely the exclusion of exons from the transcript. It has 
been shown that alternatively spliced exons have certain properties that 
distinguish them from constitutively spliced exons. Although most recent 
computational studies on alternative splicing apply only to exons which are 
conserved among two species, our method only uses information that is available 
to the splicing machinery, i.e. the DNA sequence itself. We employ advanced 
machine learning techniques in order to answer the following two questions: (1) 
Is a certain exon alternatively spliced? (2) How can we identify yet 
unidentified exons within known introns? RESULTS: We designed a support vector 
machine (SVM) kernel well suited for the task of classifying sequences with 
motifs having positional preferences. In order to solve the task (1), we 
combine the kernel with additional local sequence information, such as lengths 
of the exon and the flanking introns. The resulting SVM-based classifier 
achieves a true positive rate of 48.5% at a false positive rate of 1%. By 
scanning over single EST confirmed exons we identified 215 potential 
alternatively spliced exons. For 10 randomly selected such exons we 
successfully performed biological verification experiments and confirmed three 
novel alternatively spliced exons. To answer question (2), we additionally used 
SVM-based predictions to recognize acceptor and donor splice sites. Combined 
with the above mentioned features we were able to identify 85.2% of skipped 
exons within known introns at a false positive rate of 1%. AVAILABILITY: 
Datasets, model selection results, our predictions and additional experimental 
results are available at http://www.fml.tuebingen.mpg.de/&nbsp;raetsch/RASE 
CONTACT: Gunnar.Raetsch@tuebingen.mpg.de SUPPLEMENTARY INFORMATION: 
http://www.fml.tuebingen.mpg.de/raetsch/RASE. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Ruepp2005Assessment]  
S.&nbsp;Ruepp, F.&nbsp;Boess, L.&nbsp;Suter, M.&nbsp;C. de&nbsp;Vera, 
G.&nbsp;Steiner, T.&nbsp;Steele, T.&nbsp;Weiser, and S.&nbsp;Albertini. 
Assessment of hepatotoxic liabilities by transcript profiling.<em>Toxicol Appl 
Pharmacol</em>, Jun 2005. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  Male Wistar rats were treated with various model compounds or 
the appropriate vehicle controls in order to create a reference database for 
toxicogenomics assessment of novel compounds. Hepatotoxic compounds in the 
database were either known hepatotoxicants or showed hepatotoxicity during 
preclinical testing. Histopathology and clinical chemistry data were used to 
anchor the transcript profiles to an established endpoint (steatosis, 
cholestasis, direct acting, peroxisomal proliferation or nontoxic/control). 
These reference data were analyzed using a supervised learning method (support 
vector machines, SVM) to generate classification rules. This predictive model 
was subsequently used to assess compounds with regard to a potential 
hepatotoxic liability. A steatotic and a non-hepatotoxic 5HT(6) receptor 
antagonist compound from the same series were successfully discriminated by 
this toxicogenomics model. Additionally, an example is shown where a 
hepatotoxic liability was correctly recognized in the absence of pathological 
findings. In vitro experiments and a dog study confirmed the correctness of the 
toxicogenomics alert. Another interesting observation was that transcript 
profiles indicate toxicologically relevant changes at an earlier timepoint than 
routinely used methods. Together, these results support the useful application 
of toxicogenomics in raising alerts for adverse effects and generating 
mechanistic hypotheses that can be followed up by confirmatory experiments. 
</blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Rudd2005Eclair]  
S.&nbsp;Rudd and I.&nbsp;V. Tetko. Eclair-a web service for unravelling species 
origin of sequences sampled from mixed host interfaces.<em>Nucleic Acids Res
</em>, 33(Web Server issue):W724-7, Jul 2005. [&nbsp;bib&nbsp;| DOI&nbsp;| http
&nbsp;|.pdf&nbsp;] 
<blockquote>  The identification of the genes that participate at the 
biological interface of two species remains critical to our understanding of 
the mechanisms of disease resistance, disease susceptibility and symbiosis. The 
sequencing of complementary DNA (cDNA) libraries prepared from the biological 
interface between two organisms provides an inexpensive way to identify the 
novel genes that may be expressed as a cause or consequence of compatible or 
incompatible interactions. Sequence classification and annotation of species 
origin typically use an orthology-based approach and require access to large 
portions of either genome, or a close relative. Novel species- or 
clade-specific sequences may have no counterpart within existing databases and 
remain ambiguous features. Here we present a web-service, Eclair, which 
utilizes support vector machines for the classification of the origin of 
expressed sequence tags stemming from mixed host cDNA libraries. In addition to 
providing an interface for the classification of sequences, users are presented 
with the opportunity to train a model to suit their preferred species pair. 
Eclair is freely available at http://eclair.btk.fi. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Rose2005Correlation]  
J.&nbsp;R. Rose, Jr. Turkett, W.&nbsp;H., I.&nbsp;C. Oroian, W.&nbsp;W. 
Laegreid, and J.&nbsp;Keele. Correlation of amino acid preference and mammalian 
viral genome type.<em>Bioinformatics</em>, 2005. [&nbsp;bib&nbsp;| DOI&nbsp;| 
http&nbsp;| .pdf&nbsp;] 
<blockquote>  Motivation: In the event of an outbreak of a disease caused by 
an initially unknown pathogen, the ability to characterize anonymous sequences 
prior to isolation and culturing of the pathogen will be helpful. We show that 
it is possible to classify viral sequences by genome type (dsDNA, ssDNA, ssRNA 
positive strand, ssRNA negative strand, retroid) using amino acid 
distribution.Results: In this paper we describe the results of analysis of 
amino acid preference in mammalian viruses. The study was carried out at the 
genome level as well as two shorter sequence levels: short (300 amino acids) 
and medium length (660 amino acids). The analysis indicates a correlation 
between the viral genome types dsDNA, ssDNA, ssRNA positive strand, ssRNA 
negative strand, and retroid and amino acid preference. We investigated three 
different models of amino acid preference. The simplest amino acid preference 
model, 1-AAP, is a normalized description of the frequency of amino acids in 
genomes of a viral genome type. A slightly more complex model is the ordered 
pair amino acid preference model (2-AAP), which characterizes genomes of 
different viral genome types by the frequency of ordered pairs of amino acids. 
The most complex and accurate model is the ordered triple amino acid preference 
model (3-AAP), which is based on ordered triples of amino acids. The results 
demonstrate that mammalian viral genome types differ in their amino acid 
preference.Availability: The tools used to format and analyze data and 
supplementary material are available at 
http://www.cse.sc.edu/&nbsp;rose/aminoPreference/index.html. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Rice2005Mining]  
Simon&nbsp;B Rice, Goran Nenadic, and Benjamin&nbsp;J Stapley. Mining protein 
function from text using term-based support vector machines.<em>BMC 
Bioinformatics</em>, 6 Suppl 1:S22, 2005. [&nbsp;bib&nbsp;| DOI&nbsp;| http
&nbsp;|.pdf&nbsp;] 
<blockquote>  BACKGROUND: Text mining has spurred huge interest in the domain 
of biology. The goal of the BioCreAtIvE exercise was to evaluate the 
performance of current text mining systems. We participated in Task 2, which 
addressed assigning Gene Ontology terms to human proteins and selecting 
relevant evidence from full-text documents. We approached it as a modified form 
of the document classification task. We used a supervised machine-learning 
approach (based on support vector machines) to assign protein function and 
select passages that support the assignments. As classification features, we 
used a protein's co-occurring terms that were automatically extracted from 
documents. RESULTS: The results evaluated by curators were modest, and quite 
variable for different problems: in many cases we have relatively good 
assignment of GO terms to proteins, but the selected supporting text was 
typically non-relevant (precision spanning from 3% to 50%). The method appears 
to work best when a substantial set of relevant documents is obtained, while it 
works poorly on single documents and/or short passages. The initial results 
suggest that our approach can also mine annotations from text even when an 
explicit statement relating a protein to a GO term is absent. CONCLUSION: A 
machine learning approach to mining protein function predictions from text can 
yield good performance only if sufficient training data is available, and 
significant amount of supporting data is used for prediction. The most 
promising results are for combined document retrieval and GO term assignment, 
which calls for the integration of methods developed in BioCreAtIvE Task 1 and 
Task 2. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Rensing2005Protein]  
Stefan&nbsp;A Rensing, Dana Fritzowsky, Daniel Lang, and Ralf Reski. Protein 
encoding genes in an ancient plant: analysis of codon usage, retained genes and 
splice sites in a moss, Physcomitrella patens.<em>BMC Genomics</em>, 6(1):43, 
Mar 2005. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  BACKGROUND: The moss Physcomitrella patens is an emerging plant 
model system due to its high rate of homologous recombination, haploidy, simple 
body plan, physiological properties as well as phylogenetic position. Available 
EST data was clustered and assembled, and provided the basis for a genome-wide 
analysis of protein encoding genes. RESULTS: We have clustered and assembled 
Physcomitrella patens EST and CDS data in order to represent the transcriptome 
of this non-seed plant. Clustering of the publicly available data and 
subsequent prediction resulted in a total of 19,081 non-redundant ORF. Of these 
putative transcripts, approximately 30% have a homolog in both rice and 
Arabidopsis transcriptome. More than 130 transcripts are not present in seed 
plants but can be found in other kingdoms. These potential &quot;retained 
genes&quot; might have been lost during seed plant evolution. Functional 
annotation of these genes reveals unequal distribution among taxonomic groups 
and intriguing putative functions such as cytotoxicity and nucleic acid repair. 
Whereas introns in the moss are larger on average than in the seed plant 
Arabidopsis thaliana, position and amount of introns are approximately the 
same. Contrary to Arabidopsis, where CDS contain on average 44% G/C, in 
Physcomitrella the average G/C content is 50%. Interestingly, moss orthologs of 
Arabidopsis genes show a significant drift of codon fraction usage, towards the 
seed plant. While averaged codon bias is the same in Physcomitrella and 
Arabidopsis, the distribution pattern is different, with 15% of moss genes 
being unbiased. Species-specific, sensitive and selective splice site 
prediction for Physcomitrella has been developed using a dataset of 368 donor 
and acceptor sites, utilizing a support vector machine. The prediction accuracy 
is better than those achieved with tools trained on Arabidopsis data. 
CONCLUSION: Analysis of the moss transcriptome displays differences in gene 
structure, codon and splice site usage in comparison with the seed plant 
Arabidopsis. Putative retained genes exhibit possible functions that might 
explain the peculiar physiological properties of mosses. Both the transcriptome 
representation (including a BLAST and retrieval service) and splice site 
prediction have been made available on http://www.cosmoss.org, setting the 
basis for assembly and annotation of the Physcomitrella genome, of which draft 
shotgun sequences will become available in 2005. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Rangwala2005Profile-based] 
 H.&nbsp;Rangwala and G.&nbsp;Karypis. Profile-based direct kernels for remote 
homology detection and fold recognition.<em>Bioinformatics</em>, 
21(23):4239-4247, Dec 2005. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;] 
<blockquote>  MOTIVATION: Protein remote homology detection is a central 
problem in computational biology. Supervised learning algorithms based on 
support vector machines are currently one of the most effective methods for 
remote homology detection. The performance of these methods depends on how the 
protein sequences are modeled and on the method used to compute the kernel 
function between them. RESULTS: We introduce two classes of kernel functions 
that are constructed by combining sequence profiles with new and existing 
approaches for determining the similarity between pairs of protein sequences. 
These kernels are constructed directly from these explicit protein similarity 
measures and employ effective profile-to-profile scoring schemes for measuring 
the similarity between pairs of proteins. Experiments with remote homology 
detection and fold recognition problems show that these kernels are capable of 
producing results that are substantially better than those produced by all of 
the existing state-of-the-art SVM-based methods. In addition, the experiments 
show that these kernels, even when used in the absence of profiles, produce 
results that are better than those produced by existing non-profile-based 
schemes. AVAILABILITY: The programs for computing the various kernel functions 
are available on request from the authors. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Raghava2005Correlation]  
Gajendra P&nbsp;S Raghava and Joon&nbsp;H Han. Correlation and prediction of 
gene expression level from amino acid and dipeptide composition of its protein.
<em>BMC Bioinformatics</em>, 6(1):59, Mar 2005. [&nbsp;bib&nbsp;| DOI&nbsp;| 
http&nbsp;] 
<blockquote>  BACKGROUND: A large number of papers have been published on 
analysis of microarray data with particular emphasis on normalization of data, 
detection of differentially expressed genes, clustering of genes and regulatory 
network. On other hand there are only few studies on relation between 
expression level and composition of nucleotide/protein sequence, using 
expression data. There is a need to understand why particular genes/proteins 
express more in particular conditions. In this study, we analyze 3468 genes of 
Saccharomyces cerevisiae obtained from Holstege et al., (1998) to understand 
the relationship between expression level and amino acid composition. RESULTS: 
We compute the correlation between expression of a gene and amino acid 
composition of its protein. It was observed that some residues (like Ala, Gly, 
Arg and Val) have significant positive correlation (r &gt; 0.20) and some other 
residues (Like Asp, Leu, Asn and Ser) have negative correlation (r &lt; -0.15) 
with the expression of genes. A significant negative correlation (r = -0.18) 
was also found between length and gene expression. These observations indicate 
the relationship between percent composition and gene expression level. Thus, 
attempts have been made to develop a Support Vector Machine (SVM) based method 
for predicting the expression level of genes from its protein sequence. In this 
method the SVM is trained with proteins whose gene expression data is known in 
a given condition. Then trained SVM is used to predict the gene expression of 
other proteins of the same organism in the same condition. A correlation 
coefficient r = 0.70 was obtained between predicted and experimentally 
determined expression of genes, which improves from r = 0.70 to 0.72 when 
dipeptide composition was used instead of residue composition. The method was 
evaluated using 5-fold cross validation test. We also demonstrate that amino 
acid composition information along with gene expression data can be used for 
improving the function classification of proteins. CONCLUSION: There is a 
correlation between gene expression and amino acid composition that can be used 
to predict the expression level of genes up to a certain extent. A web server 
based on the above strategy has been developed for calculating the correlation 
between amino acid composition and gene expression and prediction of expression 
level http://kiwi.postech.ac.kr/raghava/lgepred/. This server will allow users 
to study the evolution from expression data. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Plewczyski2005support]  
Dariusz Plewczynski, Adrian Tkacz, Adam Godzik, and Leszek Rychlewski. A 
support vector machine approach to the identification of phosphorylation sites.
<em>Cell Mol Biol Lett</em>, 10(1):73-89, 2005. [&nbsp;bib&nbsp;| .pdf&nbsp;] 
<blockquote>  We describe a bioinformatics tool that can be used to predict 
the position of phosphorylation sites in proteins based only on sequence 
information. The method uses the support vector machine (SVM) statistical 
learning theory. The statistical models for phosphorylation by various types of 
kinases are built using a dataset of short (9-amino acid long) sequence 
fragments. The sequence segments are dissected around post-translationally 
modified sites of proteins that are on the current release of the Swiss-Prot 
database, and that were experimentally confirmed to be phosphorylated by any 
kinase. We represent them as vectors in a multidimensional abstract space of 
short sequence fragments. The prediction method is as follows. First, a given 
query protein sequence is dissected into overlapping short segments. All the 
fragments are then projected into the multidimensional space of sequence 
fragments via a collection of different representations. Those points are 
classified with pre-built statistical models (the SVM method with linear, 
polynomial and radial kernel functions) either as phosphorylated or inactive 
ones. The resulting list of plausible sites for phosphorylation by various 
types of kinases in the query protein is returned to the user. The efficiency 
of the method for each type of phosphorylation is estimated using leave-one-out 
tests and presented here. The sensitivities of the models can reach over 70%, 
depending on the type of kinase. The additional information from profile 
representations of short sequence fragments helps in gaining a higher degree of 
accuracy in some phosphorylation types. The further development of an automatic 
phosphorylation site annotation predictor based on our algorithm should yield a 
significant improvement when using statistical algorithms in order to quantify 
the results. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Pham2005Support]  
Tho&nbsp;Hoan Pham, Kenji Satou, and Tu&nbsp;Bao Ho. Support vector machines 
for prediction and analysis of beta and gamma-turns in proteins.<em>J. 
Bioinform. Comput. Biol.</em>, 3(2):343-58, Apr 2005. [&nbsp;bib&nbsp;] 
<blockquote>  Tight turns have long been recognized as one of the three 
important features of proteins, together with alpha-helix and beta-sheet. Tight 
turns play an important role in globular proteins from both the structural and 
functional points of view. More than 90% tight turns are beta-turns and most of 
the rest are gamma-turns. Analysis and prediction of beta-turns and gamma-turns 
is very useful for design of new molecules such as drugs, pesticides, and 
antigens. In this paper we investigated two aspects of applying support vector 
machine (SVM), a promising machine learning method for bioinformatics, to 
prediction and analysis of beta-turns and gamma-turns. First, we developed two 
SVM-based methods, called BTSVM and GTSVM, which predict beta-turns and 
gamma-turns in a protein from its sequence. When compared with other methods, 
BTSVM has a superior performance and GTSVM is competitive. Second, we used SVMs 
with a linear kernel to estimate the support of amino acids for the formation 
of beta-turns and gamma-turns depending on their position in a protein. Our 
analysis results are more comprehensive and easier to use than the previous 
results in designing turns in proteins. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Pahikkala2005Contextual]  
Tapio Pahikkala, Filip Ginter, Jorma Boberg, Jouni Jarvinen, and Tapio 
Salakoski. Contextual weighting for Support Vector Machines in literature 
mining: an application to gene versus protein name disambiguation.<em>BMC 
Bioinformatics</em>, 6(1):157, Jun 2005. [&nbsp;bib&nbsp;| DOI&nbsp;| http
&nbsp;|.pdf&nbsp;] 
<blockquote>  BACKGROUND: The ability to distinguish between genes and 
proteins is essential for understanding biological text. Support Vector 
Machines (SVMs) have been proven to be very efficient in general data mining 
tasks. We explore their capability for the gene versus protein name 
disambiguation task. RESULTS: We incorporated into the conventional SVM a 
weighting scheme based on distances of context words from the word to be 
disambiguated. This weighting scheme increased the performance of SVMs by five 
percentage points giving performance better than 85% as measured by the area 
under ROC curve and outperformed the Weighted Additive Classifier, which also 
incorporates the weighting, and the Naive Bayes classifier. CONCLUSIONS: We 
show that the performance of SVMs can be improved by the proposed weighting 
scheme. Furthermore, our results suggest that in this study the increase of the 
classification performance due to the weighting is greater than that obtained 
by selecting the underlying classifier or the kernel part of the SVM. 
</blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [ODonnell2005Gene]  
Rebekah&nbsp;K O'Donnell, Michael Kupferman, S.&nbsp;Jack Wei, Sunil Singhal, 
Randal Weber, Bert O'Malley, Yi&nbsp;Cheng, Mary Putt, Michael Feldman, Barry 
Ziober, and Ruth&nbsp;J Muschel. Gene expression signature predicts lymphatic 
metastasis in squamous cell carcinoma of the oral cavity.<em>Oncogene</em>, 
24(7):1244-51, Feb 2005. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  Metastasis via the lymphatics is a major risk factor in squamous 
cell carcinoma of the oral cavity (OSCC). We sought to determine whether the 
presence of metastasis in the regional lymph node could be predicted by a gene 
expression signature of the primary tumor. A total of 18 OSCCs were 
characterized for gene expression by hybridizing RNA to Affymetrix U133A gene 
chips. Genes with differential expression were identified using a permutation 
technique and verified by quantitative RT-PCR and immunohistochemistry. A 
predictive rule was built using a support vector machine, and the accuracy of 
the rule was evaluated using crossvalidation on the original data set and 
prediction of an independent set of four patients. Metastatic primary tumors 
could be differentiated from nonmetastatic primary tumors by a signature gene 
set of 116 genes. This signature gene set correctly predicted the four 
independent patients as well as associating five lymph node metastases from the 
original patient set with the metastatic primary tumor group. We concluded that 
lymph node metastasis could be predicted by gene expression profiles of primary 
oral cavity squamous cell carcinomas. The presence of a gene expression 
signature for lymph node metastasis indicates that clinical testing to assess 
risk for lymph node metastasis should be possible. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm microarray </blockquote> <br>
 [
Nguyen2005Two-stage]  M.&nbsp;N. Nguyen and J.&nbsp;C. Rajapakse. Two-stage 
multi-class support vector machines to protein secondary structure prediction.
<em>Pac Symp Biocomput</em>, pages 346-57, 2005. [&nbsp;bib&nbsp;] 
<blockquote>  Bioinformatics techniques to protein secondary structure (PSS) 
prediction are mostly single-stage approaches in the sense that they predict 
secondary structures of proteins by taking into account only the contextual 
information in amino acid sequences. In this paper, we propose two-stage 
Multi-class Support Vector Machine (MSVM) approach where a MSVM predictor is 
introduced to the output of the first stage MSVM to capture the sequential 
relationship among secondary structure elements for the prediction. By using 
position specific scoring matrices, generated by PSI-BLAST, the two-stage MSVM 
approach achieves Q3 accuracies of 78.0% and 76.3% on the RS126 dataset of 126 
nonhomologous globular proteins and the CB396 dataset of 396 nonhomologous 
proteins, respectively, which are better than the highest scores published on 
both datasets to date. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Nguyen2005Prediction]  
Minh&nbsp;N Nguyen and Jagath&nbsp;C Rajapakse. Prediction of protein relative 
solvent accessibility with a two-stage SVM approach.<em>Proteins</em>, 
59(1):30-7, Apr 2005. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  Information on relative solvent accessibility (RSA) of amino 
acid residues in proteins provides valuable clues to the prediction of protein 
structure and function. A two-stage approach with support vector machines 
(SVMs) is proposed, where an SVM predictor is introduced to the output of the 
single-stage SVM approach to take into account the contextual relationships 
among solvent accessibilities for the prediction. By using the 
position-specific scoring matrices (PSSMs) generated by PSI-BLAST, the 
two-stage SVM approach achieves accuracies up to 90.4% and 90.2% on the Manesh 
data set of 215 protein structures and the RS126 data set of 126 nonhomologous 
globular proteins, respectively, which are better than the highest published 
scores on both data sets to date. A Web server for protein RSA prediction using 
a two-stage SVM method has been developed and is available 
(http://birc.ntu.edu.sg/&nbsp;pas0186457/rsa.html). </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Nair2005Mimicking]  Rajesh 
Nair and Burkhard Rost. Mimicking cellular sorting improves prediction of 
subcellular localization.<em>J Mol Biol</em>, 348(1):85-100, Apr 2005. [&nbsp;
bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  Predicting the native subcellular compartment of a protein is an 
important step toward elucidating its function. Here we introduce LOCtree, a 
hierarchical system combining support vector machines (SVMs) and other 
prediction methods. LOCtree predicts the subcellular compartment of a protein 
by mimicking the mechanism of cellular sorting and exploiting a variety of 
sequence and predicted structural features in its input. Currently LOCtree does 
not predict localization for membrane proteins, since the compositional 
properties of membrane proteins significantly differ from those of non-membrane 
proteins. While any information about function can be used by the system, we 
present estimates of performance that are valid when only the amino acid 
sequence of a protein is known. When evaluated on a non-redundant test set, 
LOCtree achieved sustained levels of 74% accuracy for non-plant eukaryotes, 70% 
for plants, and 84% for prokaryotes. We rigorously benchmarked LOCtree in 
comparison to the best alternative methods for localization prediction. LOCtree 
outperformed all other methods in nearly all benchmarks. Localization 
assignments using LOCtree agreed quite well with data from recent large-scale 
experiments. Our preliminary analysis of a few entirely sequenced organisms, 
namely human (Homo sapiens), yeast (Saccharomyces cerevisiae), and weed 
(Arabidopsis thaliana) suggested that over 35% of all non-membrane proteins are 
nuclear, about 20% are retained in the cytosol, and that every fifth protein in 
the weed resides in the chloroplast. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Mueller2005Classifying]  
K.-R. M&uuml;ller, G.&nbsp;R&auml;tsch, S.&nbsp;Sonnenburg, S.&nbsp;Mika, 
M.&nbsp;Grimm, and N.&nbsp;Heinrich. Classifying 'drug-likeness' with 
Kernel-based learning methods.<em>J Chem Inf Model</em>, 45(2):249-53, 2005. 
[&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  In this article we report about a successful application of 
modern machine learning technology, namely Support Vector Machines, to the 
problem of assessing the 'drug-likeness' of a chemical from a given set of 
descriptors of the substance. We were able to drastically improve the recent 
result by Byvatov et al. (2003) on this task and achieved an error rate of 
about 7% on unseen compounds using Support Vector Machines. We see a very high 
potential of such machine learning techniques for a variety of computational 
chemistry problems that occur in the drug discovery and drug design process. 
</blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm chemoinformatics </blockquote> <br>
 [
Mitsumori2005Gene]  Tomohiro Mitsumori, Sevrani Fation, Masaki Murata, Kouichi 
Doi, and Hirohumi Doi. Gene/protein name recognition based on support vector 
machine using dictionary as features.<em>BMC Bioinformatics</em>, 6 Suppl 1:S8, 
2005. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  BACKGROUND: Automated information extraction from biomedical 
literature is important because a vast amount of biomedical literature has been 
published. Recognition of the biomedical named entities is the first step in 
information extraction. We developed an automated recognition system based on 
the SVM algorithm and evaluated it in Task 1.A of BioCreAtIvE, a competition 
for automated gene/protein name recognition. RESULTS: In the work presented 
here, our recognition system uses the feature set of the word, the 
part-of-speech (POS), the orthography, the prefix, the suffix, and the 
preceding class. We call these features &quot;internal resource features&quot;, 
i.e., features that can be found in the training data. Additionally, we 
consider the features of matching against dictionaries to be external resource 
features. We investigated and evaluated the effect of these features as well as 
the effect of tuning the parameters of the SVM algorithm. We found that the 
dictionary matching features contributed slightly to the improvement in the 
performance of the f-score. We attribute this to the possibility that the 
dictionary matching features might overlap with other features in the current 
multiple feature setting. CONCLUSION: During SVM learning, each feature alone 
had a marginally positive effect on system performance. This supports the fact 
that the SVM algorithm is robust on the high dimensionality of the feature 
vector space and means that feature selection is not required. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm nlp </blockquote> <br>
 [Matsuda2005novel]  
A.&nbsp;Matsuda, J.-P. Vert, H.&nbsp;Saigo, N.&nbsp;Ueda, H.&nbsp;Toh, and 
T.&nbsp;Akutsu. A novel representation of protein sequences for prediction of 
subcellular location using support vector machines.<em>Protein Sci.</em>, 
14(11):2804-2813, 2005. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;] 
<blockquote>  As the number of complete genomes rapidly increases, accurate 
methods to automatically predict the subcellular location of proteins are 
increasingly useful to help their functional annotation. In order to improve 
the predictive accuracy of the many prediction methods developed to date, a 
novel representation of protein sequences is proposed. This representation 
involves local compositions of amino acids and twin amino acids, and local 
frequencies of distance between successive (basic, hydrophobic, and other) 
amino acids. For calculating the local features, each sequence is split into 
three parts: N-terminal, middle, and C-terminal. The N-terminal part is further 
divided into four regions to consider ambiguity in the length and position of 
signal sequences. We tested this representation with support vector machines on 
two data sets extracted from the SWISS-PROT database. Through fivefold 
cross-validation tests, overall accuracies of more than 87 proteins, 
respectively. It is concluded that considering the respective features in the 
N-terminal, middle, and C-terminal parts is helpful to predict the subcellular 
location. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Martin2005Predicting]  
S.&nbsp;Martin, D.&nbsp;Roe, and J.-L. Faulon. Predicting protein-protein 
interactions using signature products.<em>Bioinformatics</em>, 21(2):218-226, 
Jan 2005. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  Motivation: Proteome-wide prediction of protein-protein 
interaction is a difficult and important problem in biology. Although there 
have been recent advances in both experimental and computational methods for 
predicting protein-protein interactions, we are only beginning to see a 
confluence of these techniques. In this paper, we describe a very general, 
high-throughput method for predicting protein-protein interactions. Our method 
combines a sequence-based description of proteins with experimental information 
that can be gathered from any type of protein-protein interaction screen. The 
method uses a novel description of interacting proteins by extending the 
signature descriptor, which has demonstrated success in predicting 
peptide/protein binding interactions for individual proteins. This descriptor 
is extended to protein pairs by taking signature products. The signature 
product is implemented within a support vector machine classifier as a kernel 
function. Results: We have applied our method to publicly available yeast, 
Helicobacter pylori, human and mouse datasets. We used the yeast and H.pylori 
datasets to verify the predictive ability of our method, achieving from 70 to 
80 human and mouse datasets to demonstrate that our method is capable of 
cross-species prediction. Finally, we reused the yeast dataset to explore the 
ability of our algorithm to predict domains. Contact: smartin@sandia.gov. 
</blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Mao2005Multiclass]  Yong 
Mao, Xiaobo Zhou, Daoying Pi, Youxian Sun, and Stephen T&nbsp;C Wong. 
Multiclass cancer classification by using fuzzy support vector machine and 
binary decision tree with gene selection.<em>J Biomed Biotechnol</em>, 
2005(2):160-71, 2005. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  We investigate the problems of multiclass cancer classification 
with gene selection from gene expression data. Two different constructed 
multiclass classifiers with gene selection are proposed, which are fuzzy 
support vector machine (FSVM) with gene selection and binary classification 
tree based on SVM with gene selection. Using F test and recursive feature 
elimination based on SVM as gene selection methods, binary classification tree 
based on SVM with F test, binary classification tree based on SVM with 
recursive feature elimination based on SVM, and FSVM with recursive feature 
elimination based on SVM are tested in our experiments. To accelerate 
computation, preselecting the strongest genes is also used. The proposed 
techniques are applied to analyze breast cancer data, small round blue-cell 
tumors, and acute leukemia data. Compared to existing multiclass cancer 
classifiers and binary classification tree based on SVM with F test or binary 
classification tree based on SVM with recursive feature elimination based on 
SVM mentioned in this paper, FSVM based on recursive feature elimination based 
on SVM can find most important genes that affect certain types of cancer with 
high recognition accuracy. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Mahe2005Graph]  
P.&nbsp;Mah&eacute;, N.&nbsp;Ueda, T.&nbsp;Akutsu, J.-L. Perret, and J.-P. 
Vert. Graph kernels for molecular structure-activity relationship analysis with 
support vector machines.<em>J. Chem. Inf. Model.</em>, 45(4):939-51, 2005. 
[&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  The support vector machine algorithm together with graph kernel 
functions has recently been introduced to model structure-activity 
relationships (SAR) of molecules from their 2D structure, without the need for 
explicit molecular descriptor computation. We propose two extensions to this 
approach with the double goal to reduce the computational burden associated 
with the model and to enhance its predictive accuracy: description of the 
molecules by a Morgan index process and definition of a second-order Markov 
model for random walks on 2D structures. Experiments on two mutagenicity data 
sets validate the proposed extensions, making this approach a possible 
complementary alternative to other modeling strategies. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm chemoinformatics </blockquote> <br>
 [
Luan2005Classification]  Feng Luan, Ruisheng Zhang, Chunyan Zhao, Xiaojun Yao, 
Mancang Liu, Zhide Hu, and Botao Fan. Classification of the carcinogenicity of 
N-nitroso compounds based on support vector machines and linear discriminant 
analysis.<em>Chem Res Toxicol</em>, 18(2):198-203, Feb 2005. [&nbsp;bib&nbsp;| 
DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  The support vector machine (SVM), as a novel type of learning 
machine, was used to develop a classification model of carcinogenic properties 
of 148 N-nitroso compounds. The seven descriptors calculated solely from the 
molecular structures of compounds selected by forward stepwise linear 
discriminant analysis (LDA) were used as inputs of the SVM model. The obtained 
results confirmed the discriminative capacity of the calculated descriptors. 
The result of SVM (total accuracy of 95.2%) is better than that of LDA (total 
accuracy of 89.8%). </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Lo2005Effect]  
Siaw&nbsp;Ling Lo, Cong&nbsp;Zhong Cai, Yu&nbsp;Zong Chen, and Maxey C&nbsp;M 
Chung. Effect of training datasets on support vector machine prediction of 
protein-protein interactions.<em>Proteomics</em>, 5(4):876-84, Mar 2005. [&nbsp;
bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  Knowledge of protein-protein interaction is useful for 
elucidating protein function via the concept of 'guilt-by-association'. A 
statistical learning method, Support Vector Machine (SVM), has recently been 
explored for the prediction of protein-protein interactions using artificial 
shuffled sequences as hypothetical noninteracting proteins and it has shown 
promising results (Bock, J. R., Gough, D. A., Bioinformatics 2001, 17, 
455-460). It remains unclear however, how the prediction accuracy is affected 
if real protein sequences are used to represent noninteracting proteins. In 
this work, this effect is assessed by comparison of the results derived from 
the use of real protein sequences with that derived from the use of shuffled 
sequences. The real protein sequences of hypothetical noninteracting proteins 
are generated from an exclusion analysis in combination with subcellular 
localization information of interacting proteins found in the Database of 
Interacting Proteins. Prediction accuracy using real protein sequences is 76.9% 
compared to 94.1% using artificial shuffled sequences. The discrepancy likely 
arises from the expected higher level of difficulty for separating two sets of 
real protein sequences than that for separating a set of real protein sequences 
from a set of artificial sequences. The use of real protein sequences for 
training a SVM classification system is expected to give better prediction 
results in practical cases. This is tested by using both SVM systems for 
predicting putative protein partners of a set of thioredoxin related proteins. 
The prediction results are consistent with observations, suggesting that real 
sequence is more practically useful in development of SVM classification system 
for facilitating protein-protein interaction prediction. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Liu2005Gene]  Zhenqiu Liu, 
Dechang Chen, and Halima Bensmail. Gene expression data classification with 
kernel principal component analysis.<em>J Biomed Biotechnol</em>, 
2005(2):155-9, 2005. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  One important feature of the gene expression data is that the 
number of genes<em>M</em> far exceeds the number of samples <em>N</em> . 
Standard statistical methods do not work well when<em>N</em> &lt; <em>M</em> . 
Development of new methodologies or modification of existing methodologies is 
needed for the analysis of the microarray data. In this paper, we propose a 
novel analysis procedure for classifying the gene expression data. This 
procedure involves dimension reduction using kernel principal component 
analysis (KPCA) and classification with logistic regression (discrimination). 
KPCA is a generalization and nonlinear version of principal component analysis. 
The proposed algorithm was applied to five different gene expression datasets 
involving human tumor samples. Comparison with other popular classification 
methods such as support vector machines and neural networks shows that our 
algorithm is very promising in classifying gene expression data. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Liu2005Multiclass]  
Jane&nbsp;Jijun Liu, Gene Cutler, Wuxiong Li, Zheng Pan, Sihua Peng, Tim Hoey, 
Liangbiao Chen, and Xuefeng&nbsp;Bruce Ling. Multiclass cancer classification 
and biomarker discovery using GA-based algorithms.<em>Bioinformatics</em>, 
21(11):2691-7, Jun 2005. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  MOTIVATION: The development of microarray-based high-throughput 
gene profiling has led to the hope that this technology could provide an 
efficient and accurate means of diagnosing and classifying tumors, as well as 
predicting prognoses and effective treatments. However, the large amount of 
data generated by microarrays requires effective reduction of discriminant gene 
features into reliable sets of tumor biomarkers for such multiclass tumor 
discrimination. The availability of reliable sets of biomarkers, especially 
serum biomarkers, should have a major impact on our understanding and treatment 
of cancer. RESULTS: We have combined genetic algorithm (GA) and all paired (AP) 
support vector machine (SVM) methods for multiclass cancer categorization. 
Predictive features can be automatically determined through iterative GA/SVM, 
leading to very compact sets of non-redundant cancer-relevant genes with the 
best classification performance reported to date. Interestingly, these 
different classifier sets harbor only modest overlapping gene features but have 
similar levels of accuracy in leave-one-out cross-validations (LOOCV). Further 
characterization of these optimal tumor discriminant features, including the 
use of nearest shrunken centroids (NSC), analysis of annotations and literature 
text mining, reveals previously unappreciated tumor subclasses and a series of 
genes that could be used as cancer biomarkers. With this approach, we believe 
that microarray-based multiclass molecular analysis can be an effective tool 
for cancer biomarker discovery and subsequent molecular cancer diagnosis. 
</blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Liu2005Use]  Huiqing Liu, 
Jinyan Li, and Limsoon Wong. Use of extreme patient samples for outcome 
prediction from gene expression data.<em>Bioinformatics</em>, Jun 2005. [&nbsp;
bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  MOTIVATION: Patient outcome prediction using microarray 
technologies is an important application in bioinformatics. Based on patients' 
genotypic microarray data, predictions are made to estimate patients' survival 
time and their risk of tumor metastasis or recurrence. So, accurate prediction 
can potentially help to provide better treatment for patients. RESULTS: We 
present a new computational method for patient outcome prediction. In the 
training phase of this method, we make use of two types of extreme patient 
samples: short-term survivors who got an unfavorable outcome within a short 
period and long-term survivors who were maintaining a favorable outcome after a 
long follow-up time. These extreme training samples yield a clear platform for 
us to identify relevant genes whose expression is closely related to the 
outcome. The selected extreme samples and the relevant genes are then 
integrated by a support vector machine to build a prediction model, by which 
each validation sample is assigned a risk score that falls into one of special 
pre-defined risk groups. We apply this method to several public data sets. In 
most cases, patients in high and low risk groups stratified by our method have 
clearly distinguishable outcome status as seen in their Kaplan-Meier curves. We 
also show that the idea of selecting only extreme patient samples for training 
is effective for improving the prediction accuracy when different gene 
selection methods are used. SUPPLEMENTARY INFORMATION: 
http://research.i2r.a-star.edu.sg/huiqing/supplementaldata/survival/survival.html.
</blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Li2005robust]  L.&nbsp;Li, 
W.&nbsp;Jiang, X.&nbsp;Li, K.L. Moser, Z.&nbsp;Guo, L.&nbsp;Du, Q.&nbsp;Wang, 
E.J. Topol, Q.&nbsp;Wang, and S.&nbsp;Rao. A robust hybrid between genetic 
algorithm and support vector machine for extracting an optimal feature gene 
subset.<em>Genomics</em>, 85(1):16-23, 2005. [&nbsp;bib&nbsp;| DOI&nbsp;| http
&nbsp;|.pdf&nbsp;] 
<blockquote>  Development of a robust and efficient approach for extracting 
useful information from microarray data continues to be a significant and 
challenging task. Microarray data are characterized by a high dimension, high 
signal-to-noise ratio, and high correlations between genes, but with a 
relatively small sample size. Current methods for dimensional reduction can 
further be improved for the scenario of the presence of a single (or a few) 
high influential gene(s) in which its effect in the feature subset would 
prohibit inclusion of other important genes. We have formalized a robust gene 
selection approach based on a hybrid between genetic algorithm and support 
vector machine. The major goal of this hybridization was to exploit fully their 
respective merits (e.g., robustness to the size of solution space and 
capability of handling a very large dimension of feature genes) for 
identification of key feature genes (or molecular signatures) for a complex 
biological phenotype. We have applied the approach to the microarray data of 
diffuse large B cell lymphoma to demonstrate its behaviors and properties for 
mining the high-dimension data of genome-wide gene expression profiles. The 
resulting classifier(s) (the optimal gene subset(s)) has achieved the highest 
accuracy (99 for prediction of independent microarray samples in comparisons 
with marginal filters and a hybrid between genetic algorithm and K nearest 
neighbors. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Li2005Prediction]  
H.&nbsp;Li, C.&nbsp;Ung, C.&nbsp;Yap, Y.&nbsp;Xue, Z.&nbsp;Li, Z.&nbsp;Cao, and 
Y.&nbsp;Chen. Prediction of genotoxicity of chemical compounds by statistical 
learning methods.<em>Chem. Res. Toxicol.</em>, 18(6):1071-1080, Jun 2005. 
[&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  Various toxicological profiles, such as genotoxic potential, 
need to be studied in drug discovery processes and submitted to the drug 
regulatory authorities for drug safety evaluation. As part of the effort for 
developing low cost and efficient adverse drug reaction testing tools, several 
statistical learning methods have been used for developing genotoxicity 
prediction systems with an accuracy of up to 73.8% for genotoxic (GT+) and 
92.8% for nongenotoxic (GT-) agents. These systems have been developed and 
tested by using less than 400 known GT+ and GT- agents, which is significantly 
less in number and diversity than the 860 GT+ and GT- agents known at present. 
There is a need to examine if a similar level of accuracy can be achieved for 
the more diverse set of molecules and to evaluate other statistical learning 
methods not yet applied to genotoxicity prediction. This work is intended for 
testing several statistical learning methods by using 860 GT+ and GT- agents, 
which include support vector machines (SVM), probabilistic neural network 
(PNN), k-nearest neighbor (k-NN), and C4.5 decision tree (DT). A feature 
selection method, recursive feature elimination, is used for selecting 
molecular descriptors relevant to genotoxicity study. The overall accuracies of 
SVM, k-NN, and PNN are comparable to and those of DT lower than the results 
from earlier studies, with SVM giving the highest accuracies of 77.8% for GT+ 
and 92.7% for GT- agents. Our study suggests that statistical learning methods, 
particularly SVM, k-NN, and PNN, are useful for facilitating the prediction of 
genotoxic potential of a diverse set of molecules. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm chemoinformatics </blockquote> <br>
 [
Kumar2005BhairPred]  M.&nbsp;Kumar, M.&nbsp;Bhasin, N.&nbsp;K. Natt, and 
G.&nbsp;P.&nbsp;S. Raghava. BhairPred: prediction of beta-hairpins in a protein 
from multiple alignment information using ANN and SVM techniques.<em>Nucleic 
Acids Res</em>, 33(Web Server issue):W154-9, Jul 2005. [&nbsp;bib&nbsp;| DOI
&nbsp;|http&nbsp;| .pdf&nbsp;] 
<blockquote>  This paper describes a method for predicting a supersecondary 
structural motif, beta-hairpins, in a protein sequence. The method was trained 
and tested on a set of 5102 hairpins and 5131 non-hairpins, obtained from a 
non-redundant dataset of 2880 proteins using the DSSP and PROMOTIF programs. 
Two machine-learning techniques, an artificial neural network (ANN) and a 
support vector machine (SVM), were used to predict beta-hairpins. An accuracy 
of 65.5% was achieved using ANN when an amino acid sequence was used as the 
input. The accuracy improved from 65.5 to 69.1% when evolutionary information 
(PSI-BLAST profile), observed secondary structure and surface accessibility 
were used as the inputs. The accuracy of the method further improved from 69.1 
to 79.2% when the SVM was used for classification instead of the ANN. The 
performances of the methods developed were assessed in a test case, where 
predicted secondary structure and surface accessibility were used instead of 
the observed structure. The highest accuracy achieved by the SVM based method 
in the test case was 77.9%. A maximum accuracy of 71.1% with Matthew's 
correlation coefficient of 0.41 in the test case was obtained on a dataset 
previously used by X. Cruz, E. G. Hutchinson, A. Shephard and J. M. Thornton 
(2002) Proc. Natl Acad. Sci. USA, 99, 11157-11162. The performance of the 
method was also evaluated on proteins used in the '6th community-wide 
experiment on the critical assessment of techniques for protein structure 
prediction (CASP6)'. Based on the algorithm described, a web server, BhairPred 
(http://www.imtech.res.in/raghava/bhairpred/), has been developed, which can be 
used to predict beta-hairpins in a protein using the SVM approach. </blockquote>
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Kuang2005Profile-based]  
R.&nbsp;Kuang, E.&nbsp;Ie, K.&nbsp;Wang, K.&nbsp;Wang, M.&nbsp;Siddiqi, 
Y.&nbsp;Freund, and C.&nbsp;Leslie. Profile-based string kernels for remote 
homology detection and motif extraction.<em>J. Bioinform. Comput. Biol.</em>, 
3(3):527-550, Jun 2005. [&nbsp;bib&nbsp;] 
<blockquote>  We introduce novel profile-based string kernels for use with 
support vector machines (SVMs) for the problems of protein classification and 
remote homology detection. These kernels use probabilistic profiles, such as 
those produced by the PSI-BLAST algorithm, to define position-dependent 
mutation neighborhoods along protein sequences for inexact matching of k-length 
subsequences (&quot;k-mers&quot;) in the data. By use of an efficient data 
structure, the kernels are fast to compute once the profiles have been 
obtained. For example, the time needed to run PSI-BLAST in order to build the 
profiles is significantly longer than both the kernel computation time and the 
SVM training time. We present remote homology detection experiments based on 
the SCOP database where we show that profile-based string kernels used with SVM 
classifiers strongly outperform all recently presented supervised SVM methods. 
We further examine how to incorporate predicted secondary structure information 
into the profile kernel to obtain a small but significant performance 
improvement. We also show how we can use the learned SVM classifier to extract 
&quot;discriminative sequence motifs&quot;-short regions of the original 
profile that contribute almost all the weight of the SVM classification 
score-and show that these discriminative motifs correspond to meaningful 
structural features in the protein data. The use of PSI-BLAST profiles can be 
seen as a semi-supervised learning technique, since PSI-BLAST leverages 
unlabeled data from a large sequence database to build more informative 
profiles. Recently presented &quot;cluster kernels&quot; give general 
semi-supervised methods for improving SVM protein classification performance. 
We show that our profile kernel results also outperform cluster kernels while 
providing much better scalability to large datasets. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Komura2005Multidimensional]
 D.&nbsp;Komura, H.&nbsp;Nakamura, S.&nbsp;Tsutsumi, H.&nbsp;Aburatani, and 
S.&nbsp;Ihara. Multidimensional support vector machines for visualization of 
gene expression data.<em>Bioinformatics</em>, 21(4):439-444, Feb 2005. [&nbsp;
bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  Motivation: Since DNA microarray experiments provide us with 
huge amount of gene expression data, they should be analyzed with statistical 
methods to extract the meanings of experimental results. Some dimensionality 
reduction methods such as Principal Component Analysis (PCA) are used to 
roughly visualize the distribution of high dimensional gene expression data. 
However, in the case of binary classification of gene expression data, PCA does 
not utilize class information when choosing axes. Thus clearly separable data 
in the original space may not be so in the reduced space used in PCA.Results: 
For visualization and class prediction of gene expression data, we have 
developed a new SVM-based method called multidimensional SVMs, that generate 
multiple orthogonal axes. This method projects high dimensional data into lower 
dimensional space to exhibit properties of the data clearly and to visualize a 
distribution of the data roughly. Furthermore, the multiple axes can be used 
for class prediction. The basic properties of conventional SVMs are retained in 
our method: solutions of mathematical programming are sparse, and nonlinear 
classification is implemented implicitly through the use of kernel functions. 
The application of our method to the experimentally obtained gene expression 
datasets for patients' samples indicates that our algorithm is efficient and 
useful for visualization and class prediction. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Karklin2005Classification] 
 Y.&nbsp;Karklin, R.&nbsp;F. Meraz, and S.R. Holbrook. Classification of 
non-coding RNA using graph representations of secondary structure.<em>Pac. 
Symp. Biocomput.</em>, pages 4-15, 2005. [&nbsp;bib&nbsp;| .pdf&nbsp;] 
<blockquote>  Some genes produce transcripts that function directly in 
regulatory, catalytic, or structural roles in the cell. These non-coding RNAs 
are prevalent in all living organisms, and methods that aid the understanding 
of their functional roles are essential. RNA secondary structure, the pattern 
of base-pairing, contains the critical information for determining the three 
dimensional structure and function of the molecule. In this work we examine 
whether the basic geometric and topological properties of secondary structure 
are sufficient to distinguish between RNA families in a learning framework. 
First, we develop a labeled dual graph representation of RNA secondary 
structure by adding biologically meaningful labels to the dual graphs proposed 
by Gan et al [1]. Next, we define a similarity measure directly on the labeled 
dual graphs using the recently developed marginalized kernels [2]. Using this 
similarity measure, we were able to train Support Vector Machine classifiers to 
distinguish RNAs of known families from random RNAs with similar statistics. 
For 22 of the 25 families tested, the classifier achieved better than 70% 
accuracy, with much higher accuracy rates for some families. Training a set of 
classifiers to automatically assign family labels to RNAs using a one vs. all 
multi-class scheme also yielded encouraging results. From these initial 
learning experiments, we suggest that the labeled dual graph representation, 
together with kernel machine methods, has potential for use in automated 
analysis and classification of uncharacterized RNA molecules or efficient 
genome-wide screens for RNA molecules from existing families. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Karchin2005Improving]  
R.&nbsp;Karchin, L.&nbsp;Kelly, and A.&nbsp;Sali. Improving functional 
annotation of non-synonomous SNPs with information theory.<em>Pac Symp Biocomput
</em>, pages 397-408, 2005. [&nbsp;bib&nbsp;] 
<blockquote>  Automated functional annotation of nsSNPs requires that 
amino-acid residue changes are represented by a set of descriptive features, 
such as evolutionary conservation, side-chain volume change, effect on 
ligand-binding, and residue structural rigidity. Identifying the most 
informative combinations of features is critical to the success of a 
computational prediction method. We rank 32 features according to their mutual 
information with functional effects of amino-acid substitutions, as measured by 
in vivo assays. In addition, we use a greedy algorithm to identify a subset of 
highly informative features. The method is simple to implement and provides a 
quantitative measure for selecting the best predictive features given a set of 
features that a human expert believes to be informative. We demonstrate the 
usefulness of the selected highly informative features by cross-validated tests 
of a computational classifier, a support vector machine (SVM). The SVM's 
classification accuracy is highly correlated with the ranking of the input 
features by their mutual information. Two features describing the solvent 
accessibility of &quot;wild-type&quot; and &quot;mutant&quot; amino-acid 
residues and one evolutionary feature based on superfamily-level multiple 
alignments produce comparable overall accuracy and 6% fewer false positives 
than a 32-feature set that considers physiochemical properties of amino acids, 
protein electrostatics, amino-acid residue flexibility, and binding 
interactions. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Jorissen2005Virtual]  
R.&nbsp;N. Jorissen and M.&nbsp;K. Gilson. Virtual screening of molecular 
databases using a support vector machine.<em>J Chem Inf Model</em>, 
45(3):549-61, 2005. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  The Support Vector Machine (SVM) is an algorithm that derives a 
model used for the classification of data into two categories and which has 
good generalization properties. This study applies the SVM algorithm to the 
problem of virtual screening for molecules with a desired activity. In contrast 
to typical applications of the SVM, we emphasize not classification but 
enrichment of actives by using a modified version of the standard SVM function 
to rank molecules. The method employs a simple and novel criterion for picking 
molecular descriptors and uses cross-validation to select SVM parameters. The 
resulting method is more effective at enriching for active compounds with novel 
chemistries than binary fingerprint-based methods such as binary kernel 
discrimination. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Jarzab2005Gene]  Barbara 
Jarzab, Malgorzata Wiench, Krzysztof Fujarewicz, Krzysztof Simek, Michal 
Jarzab, Malgorzata Oczko-Wojciechowska, Jan Wloch, Agnieszka Czarniecka, Ewa 
Chmielik, Dariusz Lange, Agnieszka Pawlaczek, Sylwia Szpak, Elzbieta Gubala, 
and Andrzej Swierniak. Gene expression profile of papillary thyroid cancer: 
sources of variability and diagnostic implications.<em>Cancer Res.</em>, 
65(4):1587-97, Feb 2005. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  The study looked for an optimal set of genes differentiating 
between papillary thyroid cancer (PTC) and normal thyroid tissue and assessed 
the sources of variability in gene expression profiles. The analysis was done 
by oligonucleotide microarrays (GeneChip HG-U133A) in 50 tissue samples taken 
intraoperatively from 33 patients (23 PTC patients and 10 patients with other 
thyroid disease). In the initial group of 16 PTC and 16 normal samples, we 
assessed the sources of variability in the gene expression profile by singular 
value decomposition which specified three major patterns of variability. The 
first and the most distinct mode grouped transcripts differentiating between 
tumor and normal tissues. Two consecutive modes contained a large proportion of 
immunity-related genes. To generate a multigene classifier for tumor-normal 
difference, we used support vector machines-based technique (recursive feature 
replacement). It included the following 19 genes: DPP4, GJB3, ST14, SERPINA1, 
LRP4, MET, EVA1, SPUVE, LGALS3, HBB, MKRN2, MRC2, IGSF1, KIAA0830, RXRG, P4HA2, 
CDH3, IL13RA1, and MTMR4, and correctly discriminated 17 of 18 additional 
PTC/normal thyroid samples and all 16 samples published in a previous 
microarray study. Selected novel genes (LRP4, EVA1, TMPRSS4, QPCT, and SLC34A2) 
were confirmed by Q-PCR.Our results prove that the gene expression signal of 
PTC is easily detectable even when cancer cells do not prevail over tumor 
stroma. We indicate and separate the confounding variability related to the 
immune response. Finally, we propose a potent molecular classifier able to 
discriminate between PTC and nonmalignant thyroid in more than 90% of 
investigated samples. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Huang2005Gene]  T.&nbsp;M. 
Huang and V.&nbsp;Kecman. Gene extraction for cancer diagnosis by support 
vector machines-An improvement.<em>Artif. Intell. Med.</em>, Jul 2005. [&nbsp;
bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  OBJECTIVE:: To improve the performance of gene extraction for 
cancer diagnosis by recursive feature elimination with support vector machines 
(RFE-SVMs): A cancer diagnosis by using the DNA microarray data faces many 
challenges the most serious one being the presence of thousands of genes and 
only several dozens (at the best) of patient's samples. Thus, making any kind 
of classification in high-dimensional spaces from a limited number of data is 
both an extremely difficult and a prone to an error procedure. The improved 
RFE-SVMs is introduced and used here for an elimination of less relevant genes 
and just for a reduction of the overall number of genes used in a medical 
diagnostic. METHODS:: The paper shows why and how the, usually neglected, 
penalty parameter C and some standard data preprocessing techniques 
(normalizing and scaling) influence classification results and the gene 
selection of RFE-SVMs. The gene selected by RFE-SVMs is compared with eight 
other gene selection algorithms implemented in the Rankgene software to 
investigate whether there is any consensus among the algorithms, so the scope 
of finding the right set of genes can be reduced. RESULTS:: The improved 
RFE-SVMs is applied on the two benchmarking colon and lymphoma cancer data sets 
with various C parameters and different standard preprocessing techniques. 
Here, decreasing C leads to the smaller diagnosis error in comparisons to other 
known methods applied to the benchmarking data sets. With an appropriate 
parameter C and with a proper preprocessing procedure, the reduction in a 
diagnosis error is as high as 36%. CONCLUSIONS:: The results suggest that with 
a properly chosen parameter C, the extracted genes and the constructed 
classifier will ensure less overfitting of the training data leading to an 
increased accuracy in selecting relevant genes. Finally, comparison in gene 
ranking obtained by different algorithms shows that there is a significant 
consensus among the various algorithms as to which set of genes is relevant. 
</blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Huang2005Computation]  
Shao-Wei Huang and Jenn-Kang Hwang. Computation of conformational entropy from 
protein sequences using the machine-learning method-application to the study of 
the relationship between structural conservation and local structural stability.
<em>Proteins</em>, 59(4):802-9, Jun 2005. [&nbsp;bib&nbsp;| DOI&nbsp;| http
&nbsp;|.pdf&nbsp;] 
<blockquote>  A complete protein sequence can usually determine a unique 
conformation; however, the situation is different for shorter subsequences-some 
of them are able to adopt unique conformations, independent of context; while 
others assume diverse conformations in different contexts. The conformations of 
subsequences are determined by the interplay between local and nonlocal 
interactions. A quantitative measure of such structural conservation or 
variability will be useful in the understanding of the sequence-structure 
relationship. In this report, we developed an approach using the support vector 
machine method to compute the conformational variability directly from 
sequences, which is referred to as the sequence structural entropy. As a 
practical application, we studied the relationship between sequence structural 
entropy and the hydrogen exchange for a set of well-studied proteins. We found 
that the slowest exchange cores usually comprise amino acids of the lowest 
sequence structural entropy. Our results indicate that structural conservation 
is closely related to the local structural stability. This relationship may 
have interesting implications in the protein folding processes, and may be 
useful in the study of the sequence-structure relationship. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Huang2005CTKPred]  
N.&nbsp;Huang, H.&nbsp;Chen, and Z.&nbsp;Sun. CTKPred: an SVM-based method for 
the prediction and classification of the cytokine superfamily.<em>Protein Eng. 
Des. Sel.</em>, Jun 2005. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  Cell proliferation, differentiation and death are controlled by 
a multitude of cell-cell signals and loss of this control has devastating 
consequences. Prominent among these regulatory signals is the cytokine 
superfamily, which has crucial functions in the development, differentiation 
and regulation of immune cells. In this study, a support vector machine 
(SVM)-based method was developed for predicting families and subfamilies of 
cytokines using dipeptide composition. The taxonomy of the cytokine superfamily 
with which our method complies was described in the Cytokine Family cDNA 
Database (dbCFC) and the dataset used in this study for training and testing 
was obtained from the dbCFC and Structural Classification of Proteins (SCOP). 
The method classified cytokines and non-cytokines with an accuracy of 92.5% by 
7-fold cross-validation. The method is further able to predict seven major 
classes of cytokine with an overall accuracy of 94.7%. A server for recognition 
and classification of cytokines based on multi-class SVMs has been set up at 
http://bioinfo.tsinghua.edu.cn/&nbsp;huangni/CTKPred/. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Huang2005Support]  Jing 
Huang and Feng Shi. Support vector machines for predicting apoptosis proteins 
types.<em>Acta Biotheor.</em>, 53(1):39-47, 2005. [&nbsp;bib&nbsp;| DOI&nbsp;| 
http&nbsp;| .pdf&nbsp;] 
<blockquote>  Apoptosis proteins have a central role in the development and 
homeostasis of an organism. These proteins are very important for understanding 
the mechanism of programmed cell death, and their function is related to their 
types. According to the classification scheme by Zhou and Doctor (2003), the 
apoptosis proteins are categorized into the following four types: (1) 
cytoplasmic protein; (2) plasma membrane-bound protein; (3) mitochondrial inner 
and outer proteins; (4) other proteins. A powerful learning machine, the 
Support Vector Machine, is applied for predicting the type of a given apoptosis 
protein by incorporating the sqrt-amino acid composition effect. High success 
rates were obtained by the re-substitute test (98/98 = 100 %) and the jackknife 
test (89/98 = 90.8%). </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Hua2005Optimal]  
J.&nbsp;Hua, Z.&nbsp;Xiong, J.&nbsp;Lowey, E.&nbsp;Suh, and E.&nbsp;R. 
Dougherty. Optimal number of features as a function of sample size for various 
classification rules.<em>Bioinformatics</em>, 21(8):1509-1515, Apr 2005. To 
appear. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  Motivation: Given the joint feature-label distribution, 
increasing the number of features always results in decreased classification 
error; however, this is not the case when a classifier is designed via a 
classification rule from sample data. Typically (but not always), for fixed 
sample size, the error of a designed classifier decreases and then increases as 
the number of features grows. The potential downside of using too many features 
is most critical for small samples, which are commonplace for 
gene-expression-based classifiers for phenotype discrimination. For fixed 
sample size and feature-label distribution, the issue is to find an optimal 
number of features.Results: Since only in rare cases is there a known 
distribution of the error as a function of the number of features and sample 
size, this study employs simulation for various feature-label distributions and 
classification rules, and across a wide range of sample and feature-set sizes. 
To achieve the desired end, finding the optimal number of features as a 
function of sample size, it employs massively parallel computation. Seven 
classifiers are treated: 3-nearest-neighbor, Gaussian kernel, linear support 
vector machine, polynomial support vector machine, perceptron, regular 
histogram and linear discriminant analysis. Three Gaussian-based models are 
considered: linear, nonlinear and bimodal. In addition, real patient data from 
a large breast-cancer study is considered. To mitigate the combinatorial search 
for finding optimal feature sets, and to model the situation in which subsets 
of genes are co-regulated and correlation is internal to these subsets, we 
assume that the covariance matrix of the features is blocked, with each block 
corresponding to a group of correlated features. Altogether there is a large 
number of error surfaces for the many cases. These are provided in full on a 
companion web-site, which is meant to serve as resource for those working with 
small-sample classification.Availability: For the companion web-site, please 
visit http://public.tgen.org/tamu/ofs/. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Han2005Fold]  Sangjo Han, 
Byung-Chul Lee, Seung&nbsp;Taek Yu, Chan-Seok Jeong, Soyoung Lee, and Dongsup 
Kim. Fold recognition by combining profile-profile alignment and support vector 
machine.<em>Bioinformatics</em>, 21(11):2667-73, Jun 2005. [&nbsp;bib&nbsp;| DOI
&nbsp;|http&nbsp;| .pdf&nbsp;] 
<blockquote>  MOTIVATION: Currently, the most accurate fold-recognition method 
is to perform profile-profile alignments and estimate the statistical 
significances of those alignments by calculating Z-score or E-value. Although 
this scheme is reliable in recognizing relatively close homologs related at the 
family level, it has difficulty in finding the remote homologs that are related 
at the superfamily or fold level. RESULTS: In this paper, we present an 
alternative method to estimate the significance of the alignments. The 
alignment between a query protein and a template of length n in the fold 
library is transformed into a feature vector of length n + 1, which is then 
evaluated by support vector machine (SVM). The output from SVM is converted to 
a posterior probability that a query sequence is related to a template, given 
SVM output. Results show that a new method shows significantly better 
performance than PSI-BLAST and profile-profile alignment with Z-score scheme. 
While PSI-BLAST and Z-score scheme detect 16 and 20% of superfamily-related 
proteins, respectively, at 90% specificity, a new method detects 46% of these 
proteins, resulting in more than 2-fold increase in sensitivity. More 
significantly, at the fold level, a new method can detect 14% of remotely 
related proteins at 90% specificity, a remarkable result considering the fact 
that the other methods can detect almost none at the same level of specificity. 
</blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Han2005Prediction]  L.Y. 
Han, C.Z. Cai, Z.L. Ji, and Y.Z. Chen. Prediction of functional class of novel 
viral proteins by a statistical learning method irrespective of sequence 
similarity.<em>Virology</em>, 331(1):136-143, 2005. [&nbsp;bib&nbsp;| DOI&nbsp;|
http&nbsp;| .pdf&nbsp;] 
<blockquote>  The function of a substantial percentage of the putative 
protein-coding open reading frames (ORFs) in viral genomes is unknown. As their 
sequence is not similar to that of proteins of known function, the function of 
these ORFs cannot be assigned on the basis of sequence similarity. Methods 
complement or in combination with sequence similarity-based approaches are 
being explored. The web-based software SVMProt () to some extent assigns 
protein functional family irrespective of sequence similarity and has been 
found to be useful for studying distantly related proteins [Cai, C.Z., Han, 
L.Y., Ji, Z.L., Chen, X., Chen, Y.Z., 2003. SVM-Prot: web-based support vector 
machine software for functional classification of a protein from its primary 
sequence. Nucleic Acids Res. 31(13): 3692-3697]. Here 25 novel viral proteins 
are selected to test the capability of SVMProt for functional family assignment 
of viral proteins whose function cannot be confidently predicted on by sequence 
similarity methods at present. These proteins are without a sequence homolog in 
the Swissprot database, with its precise function provided in the literature, 
and not included in the training sets of SVMProt. The predicted functional 
classes of 72 the literature-described function, which is compared to the 
overall accuracy of 87 proteins. This suggests that SVMProt to some extent is 
capable of functional class assignment irrespective of sequence similarity and 
it is potentially useful for facilitating functional study of novel viral 
proteins. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Hakenberg2005Systematic]  
J&ouml;rg Hakenberg, Steffen Bickel, Conrad Plake, Ulf Brefeld, Hagen Zahn, 
Lukas Faulstich, Ulf Leser, and Tobias Scheffer. Systematic feature evaluation 
for gene name recognition.<em>BMC Bioinformatics</em>, 6 Suppl 1:S9, 2005. 
[&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  In task 1A of the BioCreAtIvE evaluation, systems had to be 
devised that recognize words and phrases forming gene or protein names in 
natural language sentences. We approach this problem by building a word 
classification system based on a sliding window approach with a Support Vector 
Machine, combined with a pattern-based post-processing for the recognition of 
phrases. The performance of such a system crucially depends on the type of 
features chosen for consideration by the classification method, such as pre- or 
postfixes, character n-grams, patterns of capitalization, or classification of 
preceding or following words. We present a systematic approach to evaluate the 
performance of different feature sets based on recursive feature elimination, 
RFE. Based on a systematic reduction of the number of features used by the 
system, we can quantify the impact of different feature sets on the results of 
the word classification problem. This helps us to identify descriptive 
features, to learn about the structure of the problem, and to design systems 
that are faster and easier to understand. We observe that the SVM is robust to 
redundant features. RFE improves the performance by 0.7%, compared to using the 
complete set of attributes. Moreover, a performance that is only 2.3% below 
this maximum can be obtained using fewer than 5% of the features. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Haferlach2005global]  
Torsten Haferlach, Alexander Kohlmann, Susanne Schnittger, Martin Dugas, 
Wolfgang Hiddemann, Wolfgang Kern, and Claudia Schoch. A global approach to the 
diagnosis of leukemia using gene expression profiling.<em>Blood</em>, 
106(4):1189-1198, Aug 2005. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;]
<blockquote>  Accurate diagnosis and classification of leukemias are the bases 
for the appropriate management of patients. The diagnostic accuracy and 
efficiency of present methods may be improved by the use of microarrays for 
gene expression profiling. We analyzed gene expression profiles in bone marrow 
and peripheral blood samples from 937 patients with all clinically relevant 
leukemia subtypes (n=892) and non-leukemic controls (n=45) by U133A and B 
GeneChips (Affymetrix). For each subgroup differentially expressed genes were 
calculated. Class prediction was performed using support vector machines. 
Prediction accuracies were estimated by 10-fold cross validation and assessed 
for robustness in a 100-fold resampling approach using randomly chosen 
test-sets consisting of 1/3 of the samples. Applying the top 100 genes of each 
subgroup an overall prediction accuracy of 95.1% was achieved which was 
confirmed by resampling (median, 93.8%; 95% confidence interval, 91.4%-95.8%). 
In particular, AML with t(15;17), t(8;21), or inv(16), CLL, and Pro-B-ALL with 
t(11q23) were classified with 100% sensitivity and 100% specificity. 
Accordingly, cluster analysis completely separated all of the 13 subgroups 
analyzed. Gene expression profiling can predict all clinically relevant 
subentities of leukemia with high accuracy. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm microarray </blockquote> <br>
 [Haferlach2005AML
]  Torsten Haferlach, Alexander Kohlmann, Susanne Schnittger, Martin Dugas, 
Wolfgang Hiddemann, Wolfgang Kern, and Claudia Schoch. AML M3 and AML M3 
variant each have a distinct gene expression signature but also share patterns 
different from other genetically defined AML subtypes.<em>Genes Chromosomes 
Cancer</em>, 43(2):113-27, Jun 2005. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| 
.pdf&nbsp;] 
<blockquote>  Acute promyelocytic leukemia (APL) with t(15;17) appears in two 
phenotypes: AML M3, with abnormal promyelocytes showing heavy granulation and 
bundles of Auer rods, and AML M3 variant (M3v), with non- or hypogranular 
cytoplasm and a bilobed nucleus. We investigated the global gene expression 
profiles of 35 APL patients (19 AML M3, 16 AML M3v) by using high-density 
DNA-oligonucleotide microarrays. First, an unsupervised approach clearly 
separated APL samples from other AMLs characterized genetically as t(8;21) (n = 
35), inv(16) (n = 35), or t(11q23)/MLL (n = 35) or as having a normal karyotype 
(n = 50). Second, we found genes with functional relevance for blood 
coagulation that were differentially expressed between APL and other AMLs. 
Furthermore, a supervised pairwise comparison between M3 and M3v revealed 
differential expression of genes that encode for biological functions and 
pathways such as granulation and maturation of hematologic cells, explaining 
morphologic and clinical differences. Discrimination between M3 and M3v based 
on gene signatures showed a median classification accuracy of 90% by use of 
10-fold CV and support vector machines. Additional molecular mutations such as 
FLT3-LM, which were significantly more frequent in M3v than in M3 (P &lt; 
0.0001), may partly contribute to the different phenotypes. However, linear 
regression analysis demonstrated that genes differentially expressed between M3 
and M3v did not correlate with FLT3-LM. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm microarray </blockquote> <br>
 [Guo2005novel]  
Ting Guo, Yanxin Shi, and Zhirong Sun. A novel statistical ligand-binding site 
predictor: application to ATP-binding sites.<em>Protein Eng Des Sel</em>, 
18(2):65-70, Feb 2005. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  Structural genomics initiatives are leading to rapid growth in 
newly determined protein 3D structures, the functional characterization of 
which may still be inadequate. As an attempt to provide insights into the 
possible roles of the emerging proteins whose structures are available and/or 
to complement biochemical research, a variety of computational methods have 
been developed for the screening and prediction of ligand-binding sites in raw 
structural data, including statistical pattern classification techniques. In 
this paper, we report a novel statistical descriptor (the Oriented Shell Model) 
for protein ligand-binding sites, which utilizes the distance and angular 
position distribution of various structural and physicochemical features 
present in immediate proximity to the center of a binding site. Using the 
support vector machine (SVM) as the classifier, our model identified 69% of the 
ATP-binding sites in whole-protein scanning tests and in eukaryotic proteins 
the accuracy is particularly high. We propose that this feature extraction and 
machine learning procedure can screen out ligand-binding-capable protein 
candidates and can yield valuable biochemical information for individual 
proteins. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Gaudan2005Resolving]  
S.&nbsp;Gaudan, H.&nbsp;Kirsch, and D.&nbsp;Rebholz-Schuhmann. Resolving 
abbreviations to their senses in Medline.<em>Bioinformatics</em>, Jul 2005. 
[&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  MOTIVATION: Biological literature contains many abbreviations 
with one particular sense in each document. However, most abbreviations do not 
have a unique sense across the literature. Furthermore, many documents do not 
contain the long-forms of the abbreviations. Resolving an abbreviation in a 
document consists of retrieving its sense in use. Abbreviation resolution 
improves accuracy of document retrieval engines and of information extraction 
systems. RESULTS: We combine an automatic analysis of Medline abstracts and 
linguistic methods to build a dictionary of abbreviation/sense pairs. The 
dictionary is used for the resolution of abbreviations occurring with their 
long-forms. Ambiguous global abbreviations are resolved using Support Vector 
Machines that have been trained on the context of each instance of the 
abbreviation/sense pairs, previously extracted for the dictionary setup. The 
system disambiguates abbreviations with a precision of 98.9% for a recall of 
98.2% (98.5% accuracy). This performance is superior in comparison to 
previously reported research work. AVAILABILITY: The abbreviation resolution 
module is available at http://www.ebi.ac.uk/Rebholz/software.html. </blockquote>
<p></p> 
<blockquote>  Keywords: biosvm nlp </blockquote> <br>
 [Garg2005SVM-based]  
A.&nbsp;Garg, M.&nbsp;Bhasin, and G.P. Raghava. SVM-based method for 
subcellular localization of human proteins using amino acid compositions, their 
order and similarity search.<em>J. Biol. Chem.</em>, 280(15):14427-32, Apr 
2005. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  Here we report a systematic approach for predicting subcellular 
localization (cytoplasm, mitochondrial, nuclear and plasma membrane) of human 
proteins. Firstly, SVM based modules for predicting subcellular localization 
using traditional amino acid and dipeptide (i+1) composition achieved overall 
accuracy of 76.6 when carried out using similarity-based search against 
non-redundant database of experimentally annotated proteins yielded 73.3 To 
gain further insight, hybrid module (hybrid1) was developed based on amino acid 
composition, dipeptide composition, and similarity information and attained 
better accuracy of 84.9 SVM module based on different higher order dipeptide 
i.e. i+2, i+3, and i+4 were also constructed for the prediction of subcellular 
localization of human proteins and overall accuracy of 79.7 and 77.1 module 
hybrid2 was developed using traditional dipeptide (i+1) and higher order 
dipeptide (i+2, i+3, and i+4) compositions, which gave an overall accuracy of 
81.3 based on amino acid composition, traditional and higher order dipeptide 
compositions and PSI-BLAST output and achieved an overall accuracy of 84.4 or 
http://bioinformatics.uams.edu/raghava/hslpred/) has been designed to predict 
subcellular localization of human proteins using the above approaches. 
</blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Gardy2005PSORTb]  
J.&nbsp;L. Gardy, M.&nbsp;R. Laird, F.&nbsp;Chen, S.&nbsp;Rey, C.&nbsp;J. 
Walsh, M.&nbsp;Ester, and F.&nbsp;S.&nbsp;L. Brinkman. PSORTb v.2.0: expanded 
prediction of bacterial protein subcellular localization and insights gained 
from comparative proteome analysis.<em>Bioinformatics</em>, 21(5):617-623, Mar 
2005. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  Motivation: PSORTb v.1.1 is the most precise bacterial 
localization prediction tool available. However the program's predictive 
coverage and recall are low and the method is only applicable to Gram-negative 
bacteria. The goals of the present work were: increase PSORTb's coverage while 
maintaining the existing precision level, expand it to include Gram-positive 
bacteria, and then carry out a comparative analysis of localization.Results: An 
expanded database of proteins of known localization and new modules using 
frequent subsequence-based support vector machines were introduced into PSORTb 
v.2.0. The program attains a precision of 96 bacteria and predictive coverage 
comparable to other tools for whole proteome analysis. We show that the 
proportion of proteins at each localization is remarkably consistent across 
species, even in species with varying proteome size.Availability: Web-based 
version: http://www.psort.org/psortb. Standalone version: Available through the 
website under GNU General Public License.Supplementary Information: 
http://www.psort.org/psortb/supplementaryinfo.html. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Gangal2005Human]  Rajeev 
Gangal and Pankaj Sharma. Human pol II promoter prediction: time series 
descriptors and machine learning.<em>Nucleic Acids Res</em>, 33(4):1332-6, 
2005. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  Although several in silico promoter prediction methods have been 
developed to date, they are still limited in predictive performance. The 
limitations are due to the challenge of selecting appropriate features of 
promoters that distinguish them from non-promoters and the generalization or 
predictive ability of the machine-learning algorithms. In this paper we attempt 
to define a novel approach by using unique descriptors and machine-learning 
methods for the recognition of eukaryotic polymerase II promoters. In this 
study, non-linear time series descriptors along with non-linear 
machine-learning algorithms, such as support vector machine (SVM), are used to 
discriminate between promoter and non-promoter regions. The basic idea here is 
to use descriptors that do not depend on the primary DNA sequence and provide a 
clear distinction between promoter and non-promoter regions. The classification 
model built on a set of 1000 promoter and 1500 non-promoter sequences, showed a 
10-fold cross-validation accuracy of 87% and an independent test set had an 
accuracy &gt;85% in both promoter and non-promoter identification. This 
approach correctly identified all 20 experimentally verified promoters of human 
chromosome 22. The high sensitivity and selectivity indicates that n-mer 
frequencies along with non-linear time series descriptors, such as Lyapunov 
component stability and Tsallis entropy, and supervised machine-learning 
methods, such as SVMs, can be useful in the identification of pol II promoters. 
</blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Friedel2005Support]  
C.&nbsp;C. Friedel, K.&nbsp;H.&nbsp;V. Jahn, S.&nbsp;Sommer, S.&nbsp;Rudd, 
H.&nbsp;W. Mewes, and I.&nbsp;V. Tetko. Support vector machines for separation 
of mixed plant-pathogen EST collections based on codon usage.<em>Bioinformatics
</em>, 21:1383-1388, 2005. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  Motivation: Discovery of host and pathogen genes expressed at 
the plant-pathogen interface often requires the construction of mixed libraries 
that contain sequences from both genomes. Sequence identification requires 
high-throughput and reliable classification of genome origin. When using 
single-pass cDNA sequences difficulties arise from the short sequence length, 
the lack of sufficient taxonomically relevant sequence data in public databases 
and ambiguous sequence homology between plant and pathogen genes.Results: A 
novel method is described, which is independent of the availability of 
homologous genes and relies on subtle differences in codon usage between plant 
and fungal genes. We used support vector machines (SVMs) to identify the 
probable origin of sequences. SVMs were compared to several other machine 
learning techniques and to a probabilistic algorithm (PF-IND, Maor et al., 
2003) for EST classification also based on codon bias differences. Our software 
(ECLAT) has achieved a classification accuracy of 93.1 vulgare and B. graminis, 
which is a significant improvement compared to PF-IND (prediction accuracy of 
81.2 EST sequences with at least 50 nt of coding sequence can be classified by 
ECLAT with high confidence. ECLAT allows training of classifiers for any 
host-pathogen combination for which there are sufficient classified training 
sequences.Availability: ECLAT is freely available on the internet 
(http://mips.gsf.de/proj/est) or on request as a standalone version. 
</blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Donnes2005Integrated]  
P.&nbsp;D&ouml;nnes and O.&nbsp;Kohlbacher. Integrated modeling of the major 
events in the MHC class I antigen processing pathway.<em>Protein Sci.</em>, 
14:2132-2140, Jun 2005. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  Rational design of epitope-driven vaccines is a key goal of 
immunoinformatics. Typically, candidate selection relies on the prediction of 
MHC-peptide binding only, as this is known to be the most selective step in the 
MHC class I antigen processing pathway. However, proteasomal cleavage and 
transport by the transporter associated with antigen processing (TAP) are 
essential steps in antigen processing as well. While prediction methods exist 
for the individual steps, no method has yet offered an integrated prediction of 
all three major processing events. Here we present WAPP, a method combining 
prediction of proteasomal cleavage, TAP transport, and MHC binding into a 
single prediction system. The proteasomal cleavage site prediction employs a 
new matrix-based method that is based on experimentally verified proteasomal 
cleavage sites. Support vector regression is used for predicting peptides 
transported by TAP. MHC binding is the last step in the antigen processing 
pathway and was predicted using a support vector machine method, SVMHC. The 
individual methods are combined in a filtering approach mimicking the natural 
processing pathway. WAPP thus predicts peptides that are cleaved by the 
proteasome at the C terminus, transported by TAP, and show significant affinity 
to MHC class I molecules. This results in a decrease in false positive rates 
compared to MHC binding prediction alone. Compared to prediction of MHC binding 
only, we report an increased overall accuracy and a lower rate of false 
positive predictions for the HLA-A*0201, HLA-B*2705, HLA-A*01, and HLA-A*03 
alleles using WAPP. The method is available online through our prediction 
server at http://www-bs.informatik.uni-tuebingen.de/WAPP. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm immunoinformatics </blockquote> <br>
 [
Dubey2005Support]  Anshul Dubey, Matthew&nbsp;J Realff, Jay&nbsp;H Lee, and 
Andreas&nbsp;S Bommarius. Support vector machines for learning to identify the 
critical positions of a protein.<em>J Theor Biol</em>, 234(3):351-61, Jun 2005. 
[&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  A method for identifying the positions in the amino acid 
sequence, which are critical for the catalytic activity of a protein using 
support vector machines (SVMs) is introduced and analysed. SVMs are supported 
by an efficient learning algorithm and can utilize some prior knowledge about 
the structure of the problem. The amino acid sequences of the variants of a 
protein, created by inducing mutations, along with their fitness are required 
as input data by the method to predict its critical positions. To investigate 
the performance of this algorithm, variants of the beta-lactamase enzyme were 
created in silico using simulations of both mutagenesis and recombination 
protocols. Results from literature on beta-lactamase were used to test the 
accuracy of this method. It was also compared with the results from a simple 
search algorithm. The algorithm was also shown to be able to predict critical 
positions that can tolerate two different amino acids and retain function. 
</blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Dror2005Accurate]  
G.&nbsp;Dror, R.&nbsp;Sorek, and R.&nbsp;Shamir. Accurate identification of 
alternatively spliced exons using support vector machine.<em>Bioinformatics</em>
, 21(7):897-901, Apr 2005. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  Motivation: Alternative splicing is a major component of the 
regulation acting on mammalian transcriptomes. It is estimated that over half 
of all human genes have more than one splice variant. Previous studies have 
shown that alternatively spliced exons possess several features that 
distinguish them from constitutively spliced ones. Recently, we have 
demonstrated that such features can be used to distinguish alternative from 
constitutive exons. In the current study we use advanced machine learning 
methods to generate robust alternative exons classifier.Results: We extracted 
several hundred local sequence features of constitutive as well as alternative 
exons. Using feature selection methods we find seven attributes that are 
dominant for the task of classification. Several less informative features help 
to slightly increase the performance of the classifier. The classifier achieves 
a true positive rate of 50 positive rate of 0.5 alternatively spliced exons in 
exon databases that are believed to be dominated by constitutive 
exons.Availability: Upon request from the authors. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Dobson2005Predicting]  
P.D. Dobson and A.J. Doig. Predicting enzyme class from protein structure 
without alignments.<em>J. Mol. Biol.</em>, 345(1):187-199, Jan 2005. [&nbsp;bib
&nbsp;|DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  Methods for predicting protein function from structure are 
becoming more important as the rate at which structures are solved increases 
more rapidly than experimental knowledge. As a result, protein structures now 
frequently lack functional annotations. The majority of methods for predicting 
protein function are reliant upon identifying a similar protein and 
transferring its annotations to the query protein. This method fails when a 
similar protein cannot be identified, or when any similar proteins identified 
also lack reliable annotations. Here, we describe a method that can assign 
function from structure without the use of algorithms reliant upon alignments. 
Using simple attributes that can be calculated from any crystal structure, such 
as secondary structure content, amino acid propensities, surface properties and 
ligands, we describe each enzyme in a non-redundant set. The set is split 
according to Enzyme Classification (EC) number. We combine the predictions of 
one-class versus one-class support vector machine models to make overall 
assignments of EC number to an accuracy of 35 to 60 the utility of simple 
structural attributes in protein function prediction and shed light on the link 
between structure and function. We apply our methods to predict the function of 
every currently unclassified protein in the Protein Data Bank. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Degroeve2005SpliceMachine] 
 S.&nbsp;Degroeve, Y.&nbsp;Saeys, B.&nbsp;De&nbsp;Baets, P.&nbsp;Rouze, and 
Y.&nbsp;Van&nbsp;de Peer. SpliceMachine: predicting splice sites from 
high-dimensional local context representations.<em>Bioinformatics</em>, 
21:1332-1338, 2005. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  Motivation: In this age of complete genome sequencing, finding 
the location and structure of genes is crucial for further molecular research. 
The accurate prediction of intron boundaries largely facilitates the correct 
prediction of gene structure in nuclear genomes. Many tools for localizing 
these boundaries on DNA sequences have been developed and are available to 
researchers through the internet. Nevertheless, these tools still make many 
false positive predictions.Results: This manuscript presents a novel publicly 
available splice site prediction tool named SpliceMachine that (i) shows 
state-of-the-art prediction performance on Arabidopsis thaliana and human 
sequences, (ii) performs a computationally fast annotation, and (iii) can be 
trained by the user on its own data.Availability: Results, figures and software 
are available at http://bioinformatics.psb.ugent.be/supplementary_data/. 
</blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Cuturi2005context-tree]  
M.&nbsp;Cuturi and J.-P. Vert. The context-tree kernel for strings.<em>Neural 
Network.</em>, 18(4):1111-1123, 2005. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| 
.pdf&nbsp;] 
<blockquote>  We propose a new kernel for strings which borrows ideas and 
techniques from information theory and data compression. This kernel can be 
used in combination with any kernel method, in particular Support Vector 
Machines for string classi- fication, with notable applications in proteomics. 
By using a Bayesian averaging framework with conjugate priors on a class of 
Markovian models known as prob- abilistic suffix trees or context-trees, we 
compute the value of this kernel in linear time and space while only using the 
information contained in the spectrum of the considered strings. This is 
ensured through an adaptation of a compression method known as the context-tree 
weighting algorithm. Encouraging classification results are reported on a 
standard protein homology detection experiment, showing that the context-tree 
kernel performs well with respect to other state-of-the-art methods while using 
no biological prior knowledge. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Chen2005Understanding]  
Y.&nbsp;Chen and D.&nbsp;Xu. Understanding protein dispensability through 
machine-learning analysis of high-throughput data.<em>Bioinformatics</em>, 
21:575-581, Mar 2005. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  Motivation: Protein dispensability is fundamental to 
understanding of gene function and evolution. Recent advances in generating 
high-throughput data such as genomic sequence data, protein-protein interaction 
data, gene-expression data, and growth-rate data of mutants allow us to 
investigate protein dispensability systematically at the genome scale.Results: 
In our studies, protein dispensability is represented as a fitness score that 
is measured by the growth rate of gene-deletion mutants. Through analyses of 
high-throughput data in yeast Saccharomyces cerevisia, we found that a 
protein's dispensability had significant correlations with its evolutionary 
rate and duplication rate, as well as its connectivity in protein-protein 
interaction network and gene-expression correlation network. Neural network and 
support vector machine were applied to predict protein dispensability through 
high-throughput data. Our studies shed some lights on global characteristics of 
protein dispensability and evolution.Availability: The original datasets for 
protein dispensability analysis and prediction, together with related scripts, 
are available at http://digbio.missouri.edu/&nbsp;ychen/ProDispen/. 
</blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Capriotti2005I-Mutant]  
E.&nbsp;Capriotti, P.&nbsp;Fariselli, and R.&nbsp;Casadio. I-Mutant2.0: 
predicting stability changes upon mutation from the protein sequence or 
structure.<em>Nucleic Acids Res.</em>, 33(Web Server issue):W306-10, Jul 2005. 
[&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| www:&nbsp;] 
<blockquote>  I-Mutant2.0 is a support vector machine (SVM)-based tool for the 
automatic prediction of protein stability changes upon single point mutations. 
I-Mutant2.0 predictions are performed starting either from the protein 
structure or, more importantly, from the protein sequence. This latter task, to 
the best of our knowledge, is exploited for the first time. The method was 
trained and tested on a data set derived from ProTherm, which is presently the 
most comprehensive available database of thermodynamic experimental data of 
free energy changes of protein stability upon mutation under different 
conditions. I-Mutant2.0 can be used both as a classifier for predicting the 
sign of the protein stability change upon mutation and as a regression 
estimator for predicting the related DeltaDeltaG values. Acting as a 
classifier, I-Mutant2.0 correctly predicts (with a cross-validation procedure) 
80% or 77% of the data set, depending on the usage of structural or sequence 
information, respectively. When predicting DeltaDeltaG values associated with 
mutations, the correlation of predicted with expected/experimental values is 
0.71 (with a standard error of 1.30 kcal/mol) and 0.62 (with a standard error 
of 1.45 kcal/mol) when structural or sequence information are respectively 
adopted. Our web interface allows the selection of a predictive mode that 
depends on the availability of the protein structure and/or sequence. In this 
latter case, the web server requires only pasting of a protein sequence in a 
raw format. We therefore introduce I-Mutant2.0 as a unique and valuable helper 
for protein design, even when the protein structure is not yet known with 
atomic resolution. Availability: 
http://gpcr.biocomp.unibo.it/cgi/predictors/I-Mutant2.0/I-Mutant2.0.cgi. 
</blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Burckin2005Exploring]  
T.&nbsp;Burckin, R.&nbsp;Nagel, Y.&nbsp;Mandel-Gutfreund, L.&nbsp;Shiue, 
T.&nbsp;A. Clark, J.-L. Chong, T.-H. Chang, S.&nbsp;Squazzo, G.&nbsp;Hartzog, 
and M.&nbsp;Ares. Exploring functional relationships between components of the 
gene expression machinery.<em>Nat. Struct. Mol. Biol.</em>, 12(2):175-82, Feb 
2005. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  Eukaryotic gene expression requires the coordinated activity of 
many macromolecular machines including transcription factors and RNA 
polymerase, the spliceosome, mRNA export factors, the nuclear pore, the 
ribosome and decay machineries. Yeast carrying mutations in genes encoding 
components of these machineries were examined using microarrays to measure 
changes in both pre-mRNA and mRNA levels. We used these measurements as a 
quantitative phenotype to ask how steps in the gene expression pathway are 
functionally connected. A multiclass support vector machine was trained to 
recognize the gene expression phenotypes caused by these mutations. In several 
cases, unexpected phenotype assignments by the computer revealed functional 
roles for specific factors at multiple steps in the gene expression pathway. 
The ability to resolve gene expression pathway phenotypes provides insight into 
how the major machineries of gene expression communicate with each other. 
</blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm microarray </blockquote> <br>
 [
Bunescu2005Comparative]  R.&nbsp;Bunescu, R.&nbsp;Ge, R.&nbsp;J. Kate, 
E.&nbsp;M. Marcotte, R.&nbsp;J. Mooney, A.&nbsp;K. Ramani, and Y.&nbsp;W. Wong. 
Comparative experiments on learning information extractors for proteins and 
their interactions.<em>Artif. Intell. Med.</em>, 33(2):139-55, Feb 2005. [&nbsp;
bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  OBJECTIVE: Automatically extracting information from biomedical 
text holds the promise of easily consolidating large amounts of biological 
knowledge in computer-accessible form. This strategy is particularly attractive 
for extracting data relevant to genes of the human genome from the 11 million 
abstracts in Medline. However, extraction efforts have been frustrated by the 
lack of conventions for describing human genes and proteins. We have developed 
and evaluated a variety of learned information extraction systems for 
identifying human protein names in Medline abstracts and subsequently 
extracting information on interactions between the proteins. METHODS AND 
MATERIAL: We used a variety of machine learning methods to automatically 
develop information extraction systems for extracting information on 
gene/protein name, function and interactions from Medline abstracts. We present 
cross-validated results on identifying human proteins and their interactions by 
training and testing on a set of approximately 1000 manually-annotated Medline 
abstracts that discuss human genes/proteins. RESULTS: We demonstrate that 
machine learning approaches using support vector machines and maximum entropy 
are able to identify human proteins with higher accuracy than several previous 
approaches. We also demonstrate that various rule induction methods are able to 
identify protein interactions with higher precision than manually-developed 
rules. CONCLUSION: Our results show that it is promising to use machine 
learning to automatically build systems for extracting information from 
biomedical text. The results also give a broad picture of the relative 
strengths of a wide variety of methods when tested on a reasonably large 
human-annotated corpus. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Briem2005Classifying]  
Hans Briem and Judith G&uuml;nther. Classifying &quot;kinase 
inhibitor-likeness&quot; by using machine-learning methods.<em>ChemBioChem</em>
, 6(3):558-66, Mar 2005. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  By using an in-house data set of small-molecule structures, 
encoded by Ghose-Crippen parameters, several machine learning techniques were 
applied to distinguish between kinase inhibitors and other molecules with no 
reported activity on any protein kinase. All four approaches 
pursued-support-vector machines (SVM), artificial neural networks (ANN), k 
nearest neighbor classification with GA-optimized feature selection (GA/kNN), 
and recursive partitioning (RP)-proved capable of providing a reasonable 
discrimination. Nevertheless, substantial differences in performance among the 
methods were observed. For all techniques tested, the use of a consensus vote 
of the 13 different models derived improved the quality of the predictions in 
terms of accuracy, precision, recall, and F1 value. Support-vector machines, 
followed by the GA/kNN combination, outperformed the other techniques when 
comparing the average of individual models. By using the respective majority 
votes, the prediction of neural networks yielded the highest F1 value, followed 
by SVMs. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm chemoinformatics </blockquote> <br>
 [
Bradford2005Improved]  James&nbsp;R Bradford and David&nbsp;R Westhead. 
Improved prediction of protein-protein binding sites using a support vector 
machines approach.<em>Bioinformatics</em>, 21(8):1487-94, Apr 2005. [&nbsp;bib
&nbsp;|DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  MOTIVATION: Structural genomics projects are beginning to 
produce protein structures with unknown function, therefore, accurate, 
automated predictors of protein function are required if all these structures 
are to be properly annotated in reasonable time. Identifying the interface 
between two interacting proteins provides important clues to the function of a 
protein and can reduce the search space required by docking algorithms to 
predict the structures of complexes. RESULTS: We have combined a support vector 
machine (SVM) approach with surface patch analysis to predict protein-protein 
binding sites. Using a leave-one-out cross-validation procedure, we were able 
to successfully predict the location of the binding site on 76% of our dataset 
made up of proteins with both transient and obligate interfaces. With 
heterogeneous cross-validation, where we trained the SVM on transient complexes 
to predict on obligate complexes (and vice versa), we still achieved comparable 
success rates to the leave-one-out cross-validation suggesting that sufficient 
properties are shared between transient and obligate interfaces. AVAILABILITY: 
A web application based on the method can be found at 
http://www.bioinformatics.leeds.ac.uk/ppi_pred. The dataset of 180 proteins 
used in this study is also available via the same web site. CONTACT: 
westhead@bmb.leeds.ac.uk SUPPLEMENTARY INFORMATION: 
http://www.bioinformatics.leeds.ac.uk/ppi-pred/supp-material. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Borgwardt2005Protein]  
K.M. Borgwardt, C.S. Ong, S.&nbsp;Sch&ouml;nauer, S.V.N. Vishwanathan, A.J. 
Smola, and H.-P. Kriegel. Protein function prediction via graph kernels.<em>
Bioinformatics</em>, 21(Suppl. 1):i47-i56, Jun 2005. [&nbsp;bib&nbsp;| DOI
&nbsp;|http&nbsp;| .pdf&nbsp;] 
<blockquote>  MOTIVATION: Computational approaches to protein function 
prediction infer protein function by finding proteins with similar sequence, 
structure, surface clefts, chemical properties, amino acid motifs, interaction 
partners or phylogenetic profiles. We present a new approach that combines 
sequential, structural and chemical information into one graph model of 
proteins. We predict functional class membership of enzymes and non-enzymes 
using graph kernels and support vector machine classification on these protein 
graphs. RESULTS: Our graph model, derivable from protein sequence and structure 
only, is competitive with vector models that require additional protein 
information, such as the size of surface pockets. If we include this extra 
information into our graph model, our classifier yields significantly higher 
accuracy levels than the vector models. Hyperkernels allow us to select and to 
optimally combine the most relevant node attributes in our protein graphs. We 
have laid the foundation for a protein function prediction system that 
integrates protein information from various sources efficiently and 
effectively. AVAILABILITY: More information available via 
www.dbs.ifi.lmu.de/Mitarbeiter/borgwardt.html. CONTACT: 
borgwardt@dbs.ifi.lmu.de. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Bordner2005Statistical]  
Andrew&nbsp;J Bordner and Ruben Abagyan. Statistical analysis and prediction of 
protein-protein interfaces.<em>Proteins</em>, 60(3):353-66, Aug 2005. [&nbsp;bib
&nbsp;|DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  Predicting protein-protein interfaces from a three-dimensional 
structure is a key task of computational structural proteomics. In contrast to 
geometrically distinct small molecule binding sites, protein-protein interface 
are notoriously difficult to predict. We generated a large nonredundant data 
set of 1494 true protein-protein interfaces using biological symmetry 
annotation where necessary. The data set was carefully analyzed and a Support 
Vector Machine was trained on a combination of a new robust evolutionary 
conservation signal with the local surface properties to predict 
protein-protein interfaces. Fivefold cross validation verifies the high 
sensitivity and selectivity of the model. As much as 97% of the predicted 
patches had an overlap with the true interface patch while only 22% of the 
surface residues were included in an average predicted patch. The model allowed 
the identification of potential new interfaces and the correction of mislabeled 
oligomeric states. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Bhasin2005Pcleavage]  
M.&nbsp;Bhasin and G.&nbsp;P.&nbsp;S. Raghava. Pcleavage: an SVM based method 
for prediction of constitutive proteasome and immunoproteasome cleavage sites 
in antigenic sequences.<em>Nucleic Acids Res</em>, 33(Web Server issue):W202-7, 
Jul 2005. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  This manuscript describes a support vector machine based method 
for the prediction of constitutive as well as immunoproteasome cleavage sites 
in antigenic sequences. This method achieved Matthew's correlation coefficents 
of 0.54 and 0.43 on in vitro and major histocompatibility complex ligand data, 
respectively. This shows that the performance of our method is comparable to 
that of the NetChop method, which is currently considered to be the best method 
for proteasome cleavage site prediction. Based on the method, a web server, 
Pcleavage, has also been developed. This server accepts protein sequences in 
any standard format and present results in a user-friendly format. The server 
is available for free use by all academic users at the URL 
http://www.imtech.res.in/raghava/pcleavage/ or 
http://bioinformatics.uams.edu/mirror/pcleavage/. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm immunoinformatics </blockquote> <br>
 [
Bhasin2005GPCRsclass]  M.&nbsp;Bhasin and G.&nbsp;P.&nbsp;S. Raghava. 
GPCRsclass: a web tool for the classification of amine type of 
G-protein-coupled receptors.<em>Nucleic Acids Res.</em>, 33(Web Server 
issue):W143-7, Jul 2005. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  The receptors of amine subfamily are specifically major drug 
targets for therapy of nervous disorders and psychiatric diseases. The 
recognition of novel amine type of receptors and their cognate ligands is of 
paramount interest for pharmaceutical companies. In the past, Chou and 
co-workers have shown that different types of amine receptors are correlated 
with their amino acid composition and are predictable on its basis with 
considerable accuracy [Elrod and Chou (2002) Protein Eng., 15, 713-715]. This 
motivated us to develop a better method for the recognition of novel amine 
receptors and for their further classification. The method was developed on the 
basis of amino acid composition and dipeptide composition of proteins using 
support vector machine. The method was trained and tested on 167 proteins of 
amine subfamily of G-protein-coupled receptors (GPCRs). The method 
discriminated amine subfamily of GPCRs from globular proteins with Matthew's 
correlation coefficient of 0.98 and 0.99 using amino acid composition and 
dipeptide composition, respectively. In classifying different types of amine 
receptors using amino acid composition and dipeptide composition, the method 
achieved an accuracy of 89.8 and 96.4%, respectively. The performance of the 
method was evaluated using 5-fold cross-validation. The dipeptide composition 
based method predicted 67.6% of protein sequences with an accuracy of 100% with 
a reliability index &gt; or =5. A web server GPCRsclass has been developed for 
predicting amine-binding receptors from its amino acid sequence 
[http://www.imtech.res.in/raghava/gpcrsclass/ and 
http://bioinformatics.uams.edu/raghava/gpersclass/ (mirror site)]. </blockquote>
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Ben-Hur2005Kernel]  
A.&nbsp;Ben-Hur and W.&nbsp;S. Noble. Kernel methods for predicting 
protein-protein interactions.<em>Bioinformatics</em>, 21(Suppl. 1):i38-i46, Jun 
2005. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  MOTIVATION: Despite advances in high-throughput methods for 
discovering protein-protein interactions, the interaction networks of even 
well-studied model organisms are sketchy at best, highlighting the continued 
need for computational methods to help direct experimentalists in the search 
for novel interactions. RESULTS: We present a kernel method for predicting 
protein-protein interactions using a combination of data sources, including 
protein sequences, Gene Ontology annotations, local properties of the network, 
and homologous interactions in other species. Whereas protein kernels proposed 
in the literature provide a similarity between single proteins, prediction of 
interactions requires a kernel between pairs of proteins. We propose a pairwise 
kernel that converts a kernel between single proteins into a kernel between 
pairs of proteins, and we illustrate the kernel's effectiveness in conjunction 
with a support vector machine classifier. Furthermore, we obtain improved 
performance by combining several sequence-based kernels based on k-mer 
frequency, motif and domain content and by further augmenting the pairwise 
sequence kernel with features that are based on other sources of data.We apply 
our method to predict physical interactions in yeast using data from the BIND 
database. At a false positive rate of 1% the classifier retrieves close to 80% 
of a set of trusted interactions. We thus demonstrate the ability of our method 
to make accurate predictions despite the sizeable fraction of false positives 
that are known to exist in interaction databases. AVAILABILITY: The 
classification experiments were performed using PyML available at 
http://pyml.sourceforge.net. Data are available at: 
http://noble.gs.washington.edu/proj/sppi CONTACT: asa@gs.washington.edu. 
</blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Bao2005Identifying]  Lei 
Bao. Identifying genes related to chemosensitivity using support vector machine.
<em>Methods Mol Med</em>, 111:233-40, 2005. [&nbsp;bib&nbsp;] 
<blockquote>  In an effort to identify genes involved in chemosensitivity and 
to evaluate the functional relationships between genes and anticancer drugs 
acting by the same mechanism, a supervised machine learning approach called 
support vector machine (SVM) is used to associate genes with any of five 
predefined anticancer drug mechanistic categories. The drug activity profiles 
are used as training examples to train the SVM and then the gene expression 
profiles are used as test examples to predict their associated mechanistic 
categories. This method of correlating drugs and genes provides a strategy for 
finding novel biologically significant relationships for molecular pharmacology.
</blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Atalay2005Implicit]  
V.&nbsp;Atalay and R.&nbsp;Cetin-Atalay. Implicit motif distribution based 
hybrid computational kernel for sequence classification.<em>Bioinformatics</em>
, 21(8):1429-1436, Apr 2005. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf
&nbsp;]
<blockquote>  MOTIVATION: We designed a general computational kernel for 
classification problems that require specific motif extraction and search from 
sequences. Instead of searching for explicit motifs, our approach finds the 
distribution of implicit motifs and uses as a feature for classification. 
Implicit motif distribution approach may be used as modus operandi for 
bioinformatics problems that require specific motif extraction and search, 
which is otherwise computationally prohibitive. RESULTS: A system named P2SL 
that infer protein subcellular targeting was developed through this 
computational kernel. Targeting-signal was modeled by the distribution of 
subsequence occurrences (implicit motifs) using self-organizing maps. The 
boundaries among the classes were then determined with a set of support vector 
machines. P2SL hybrid computational system achieved approximately 81% of 
prediction accuracy rate over ER targeted, cytosolic, mitochondrial and nuclear 
protein localization classes. P2SL additionally offers the distribution 
potential of proteins among localization classes, which is particularly 
important for proteins, shuttle between nucleus and cytosol. AVAILABILITY: 
http://staff.vbi.vt.edu/volkan/p2sl and 
http://www.i-cancer.fen.bilkent.edu.tr/p2sl CONTACT: rengul@bilkent.edu.tr. 
</blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Arodz2005Pattern]  Tomasz 
Arod&#377;, Marcin Kurdziel, Erik O&nbsp;D Sevre, and David&nbsp;A Yuen. 
Pattern recognition techniques for automatic detection of suspicious-looking 
anomalies in mammograms.<em>Comput. Methods Programs Biomed.</em>, 
79(2):135-49, Aug 2005. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  We have employed two pattern recognition methods used commonly 
for face recognition in order to analyse digital mammograms. The methods are 
based on novel classification schemes, the AdaBoost and the support vector 
machines (SVM). A number of tests have been carried out to evaluate the 
accuracy of these two algorithms under different circumstances. Results for the 
AdaBoost classifier method are promising, especially for classifying mass-type 
lesions. In the best case the algorithm achieved accuracy of 76% for all lesion 
types and 90% for masses only. The SVM based algorithm did not perform as well. 
In order to achieve a higher accuracy for this method, we should choose image 
features that are better suited for analysing digital mammograms than the 
currently used ones. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm image </blockquote> <br>
 [
Arimoto2005Development]  Rieko Arimoto, Madhu-Ashni Prasad, and Eric&nbsp;M 
Gifford. Development of CYP3A4 inhibition models: comparisons of 
machine-learning techniques and molecular descriptors.<em>J Biomol Screen</em>, 
10(3):197-205, Apr 2005. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;] 
<blockquote>  Computational models of cytochrome P450 3A4 inhibition were 
developed based on high-throughput screening data for 4470 proprietary 
compounds. Multiple models differentiating inhibitors (IC(50) &lt;3 microM) and 
noninhibitors were generated using various machine-learning algorithms 
(recursive partitioning [RP], Bayesian classifier, logistic regression, 
k-nearest-neighbor, and support vector machine [SVM]) with structural 
fingerprints and topological indices. Nineteen models were evaluated by 
internal 10-fold cross-validation and also by an independent test set. Three 
most predictive models, Barnard Chemical Information (BCI)-fingerprint/SVM, 
MDL-keyset/SVM, and topological indices/RP, correctly classified 249, 248, and 
236 compounds of 291 noninhibitors and 135, 137, and 147 compounds of 179 
inhibitors in the validation set. Their overall accuracies were 82%, 82%, and 
81%, respectively. Investigating applicability of the BCI/SVM model found a 
strong correlation between the predictive performance and the structural 
similarity to the training set. Using Tanimoto similarity index as a confidence 
measurement for the predictions, the limitation of the extrapolation was 0.7 in 
the case of the BCI/SVM model. Taking consensus of the 3 best models yielded a 
further improvement in predictive capability, kappa = 0.65 and accuracy = 83%. 
The consensus model could also be tuned to minimize either false positives or 
false negatives depending on the emphasis of the screening. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm chemoinformatics </blockquote> <br>
 [
Aphinyanaphongs2005Text]  Yindalon Aphinyanaphongs, Ioannis Tsamardinos, 
Alexander Statnikov, Douglas Hardin, and Constantin&nbsp;F Aliferis. Text 
categorization models for high-quality article retrieval in internal medicine.
<em>J. Am. Med. Inform. Assoc.</em>, 12(2):207-16, 2005. [&nbsp;bib&nbsp;| DOI
&nbsp;|http&nbsp;| .pdf&nbsp;] 
<blockquote>  OBJECTIVE Finding the best scientific evidence that applies to a 
patient problem is becoming exceedingly difficult due to the exponential growth 
of medical publications. The objective of this study was to apply machine 
learning techniques to automatically identify high-quality, content-specific 
articles for one time period in internal medicine and compare their performance 
with previous Boolean-based PubMed clinical query filters of Haynes et al. 
DESIGN The selection criteria of the ACP Journal Club for articles in internal 
medicine were the basis for identifying high-quality articles in the areas of 
etiology, prognosis, diagnosis, and treatment. Naive Bayes, a specialized 
AdaBoost algorithm, and linear and polynomial support vector machines were 
applied to identify these articles. MEASUREMENTS The machine learning models 
were compared in each category with each other and with the clinical query 
filters using area under the receiver operating characteristic curves, 11-point 
average recall precision, and a sensitivity/specificity match method. RESULTS 
In most categories, the data-induced models have better or comparable 
sensitivity, specificity, and precision than the clinical query filters. The 
polynomial support vector machine models perform the best among all learning 
methods in ranking the articles as evaluated by area under the receiver 
operating curve and 11-point average recall precision. CONCLUSION This research 
shows that, using machine learning methods, it is possible to automatically 
build models for retrieving high-quality, content-specific articles using 
inclusion or citation by the ACP Journal Club as a gold standard in a given 
time period in internal medicine that perform better than the 1994 PubMed 
clinical query filters. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm nlp </blockquote> <br>
 [
Aires-de-Sousa2005Prediction]  J.&nbsp;Aires-de Sousa and J.&nbsp;Gasteiger. 
Prediction of enantiomeric excess in a combinatorial library of catalytic 
enantioselective reactions.<em>J Comb Chem</em>, 7(2):298-301, 2005. [&nbsp;bib
&nbsp;|DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  A quantitative structure-enantioselectivity relationship was 
established for a combinatorial library of enantioselective reactions performed 
by addition of diethyl zinc to benzaldehyde. Chiral catalysts and additives 
were encoded by their chirality codes and presented as input to neural 
networks. The networks were trained to predict the enantiomeric excess. With 
independent test sets, predictions of enantiomeric excess could be made with an 
average error as low as 6% ee. Multilinear regression, perceptrons, and support 
vector machines were also evaluated as modeling tools. The method is of 
interest for the computer-aided design of combinatorial libraries involving 
chiral compounds or enantioselective reactions. This is the first example of a 
quantitative structure-property relationship based on chirality codes. 
</blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm chemoinformatics </blockquote> <br>
 [
Zhang2005Improved]  Qidong Zhang, Sukjoon Yoon, and William&nbsp;J Welsh. 
Improved method for predicting beta-turn using support vector machine.<em>
Bioinformatics</em>, 21(10):2370-4, May 2005. [&nbsp;bib&nbsp;| DOI&nbsp;| http
&nbsp;|.pdf&nbsp;] 
<blockquote>  MOTIVATION: Numerous methods for predicting beta-turns in 
proteins have been developed based on various computational schemes. Here, we 
introduce a new method of beta-turn prediction that uses the support vector 
machine (SVM) algorithm together with predicted secondary structure 
information. Various parameters from the SVM have been adjusted to achieve 
optimal prediction performance. RESULTS: The SVM method achieved excellent 
performance as measured by the Matthews correlation coefficient (MCC = 0.45) 
using a 7-fold cross validation on a database of 426 non-homologous protein 
chains. To our best knowledge, this MCC value is the highest achieved so far 
for predicting beta-turn. The overall prediction accuracy Qtotal was 77.3%, 
which is the best among the existing prediction methods. Among its unique 
attractive features, the present SVM method avoids overtraining and compresses 
information and provides a predicted reliability index. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Yu2005Ovarian]  J.&nbsp;S. 
Yu, S.&nbsp;Ongarello, R.&nbsp;Fiedler, X.&nbsp;W. Chen, G.&nbsp;Toffolo, 
C.&nbsp;Cobelli, and Z.&nbsp;Trajanoski. Ovarian cancer identification based on 
dimensionality reduction for high-throughput mass spectrometry data.<em>
Bioinformatics</em>, 21(10):2200-9, May 2005. [&nbsp;bib&nbsp;| DOI&nbsp;| http
&nbsp;|.pdf&nbsp;] 
<blockquote>  MOTIVATION: High-throughput and high-resolution mass 
spectrometry instruments are increasingly used for disease classification and 
therapeutic guidance. However, the analysis of immense amount of data poses 
considerable challenges. We have therefore developed a novel method for 
dimensionality reduction and tested on a published ovarian high-resolution 
SELDI-TOF dataset. RESULTS: We have developed a four-step strategy for data 
preprocessing based on: (1) binning, (2) Kolmogorov-Smirnov test, (3) 
restriction of coefficient of variation and (4) wavelet analysis. Subsequently, 
support vector machines were used for classification. The developed method 
achieves an average sensitivity of 97.38% (sd = 0.0125) and an average 
specificity of 93.30% (sd = 0.0174) in 1000 independent k-fold 
cross-validations, where k = 2, ..., 10. AVAILABILITY: The software is 
available for academic and non-commercial institutions. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm proteomics </blockquote> <br>
 [
Tothill2005expression-based]  Richard&nbsp;W Tothill, Adam Kowalczyk, Danny 
Rischin, Alex Bousioutas, Izhak Haviv, Ryan&nbsp;K van Laar, Paul&nbsp;M 
Waring, John Zalcberg, Robyn Ward, Andrew&nbsp;V Biankin, Robert&nbsp;L 
Sutherland, Susan&nbsp;M Henshall, Kwun Fong, Jonathan&nbsp;R Pollack, David 
D&nbsp;L Bowtell, and Andrew&nbsp;J Holloway. An expression-based site of 
origin diagnostic method designed for clinical application to cancer of unknown 
origin.<em>Cancer Res.</em>, 65(10):4031-40, May 2005. [&nbsp;bib&nbsp;| DOI
&nbsp;|http&nbsp;| .pdf&nbsp;] 
<blockquote>  Gene expression profiling offers a promising new technique for 
the diagnosis and prognosis of cancer. We have applied this technology to build 
a clinically robust site of origin classifier with the ultimate aim of applying 
it to determine the origin of cancer of unknown primary (CUP). A single cDNA 
microarray platform was used to profile 229 primary and metastatic tumors 
representing 14 tumor types and multiple histologic subtypes. This data set was 
subsequently used for training and validation of a support vector machine (SVM) 
classifier, demonstrating 89% accuracy using a 13-class model. Further, we show 
the translation of a five-class classifier to a quantitative PCR-based 
platform. Selecting 79 optimal gene markers, we generated a quantitative-PCR 
low-density array, allowing the assay of both fresh-frozen and formalin-fixed 
paraffin-embedded (FFPE) tissue. Data generated using both quantitative PCR and 
microarray were subsequently used to train and validate a cross-platform SVM 
model with high prediction accuracy. Finally, we applied our SVM classifiers to 
13 cases of CUP. We show that the microarray SVM classifier was capable of 
making high confidence predictions in 11 of 13 cases. These predictions were 
supported by comprehensive review of the patients' clinical histories. 
</blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm microarray </blockquote> <br>
 [
Teramoto2005Prediction]  Reiji Teramoto, Mikio Aoki, Toru Kimura, and Masaharu 
Kanaoka. Prediction of siRNA functionality using generalized string kernel and 
support vector machine.<em>FEBS Lett.</em>, 579(13):2878-82, May 2005. [&nbsp;
bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  Small interfering RNAs (siRNAs) are becoming widely used for 
sequence-specific gene silencing in mammalian cells, but designing an effective 
siRNA is still a challenging task. In this study, we developed an algorithm for 
predicting siRNA functionality by using generalized string kernel (GSK) 
combined with support vector machine (SVM). With GSK, siRNA sequences were 
represented as vectors in a multi-dimensional feature space according to the 
numbers of subsequences in each siRNA, and subsequently classified with SVM 
into effective or ineffective siRNAs. We applied this algorithm to published 
siRNAs, and could classify effective and ineffective siRNAs with 90.6%, 86.2% 
accuracy, respectively. </blockquote> 
<p></p> 
<blockquote>  Keywords: sirna biosvm </blockquote> <br>
 [Res2005evolution]  
I.&nbsp;Res, I.&nbsp;Mihalek, and O.&nbsp;Lichtarge. An evolution based 
classifier for prediction of protein interfaces without using protein 
structures.<em>Bioinformatics</em>, 21(10):2496-501, May 2005. [&nbsp;bib&nbsp;|
DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  MOTIVATION: The number of available protein structures still 
lags far behind the number of known protein sequences. This makes it important 
to predict which residues participate in protein-protein interactions using 
only sequence information. Few studies have tackled this problem until now. 
RESULTS: We applied support vector machines to sequences in order to generate a 
classification of all protein residues into those that are part of a protein 
interface and those that are not. For the first time evolutionary information 
was used as one of the attributes and this inclusion of evolutionary importance 
rankings improves the classification. Leave-one-out cross-validation 
experiments show that prediction accuracy reaches 64%. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Plewczynski2005AutoMotif]  
Dariusz Plewczynski, Adrian Tkacz, Lucjan&nbsp;Stanislaw Wyrwicz, and Leszek 
Rychlewski. AutoMotif server: prediction of single residue post-translational 
modifications in proteins.<em>Bioinformatics</em>, 21(10):2525-7, May 2005. 
[&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  The AutoMotif Server allows for identification of 
post-translational modification (PTM) sites in proteins based only on local 
sequence information. The local sequence preferences of short segments around 
PTM residues are described here as linear functional motifs (LFMs). Sequence 
models for all types of PTMs are trained by support vector machine on 
short-sequence fragments of proteins in the current release of Swiss-Prot 
database (phosphorylation by various protein kinases, sulfation, acetylation, 
methylation, amidation, etc.). The accuracy of the identification is estimated 
using the standard leave-one-out procedure. The sensitivities for all types of 
short LFMs are in the range of 70%. AVAILABILITY: The AutoMotif Server is 
available free for academic use at http://automotif.bioinfo.pl/ </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [OFlanagan2005Non]  
R.&nbsp;A. O'Flanagan, G.&nbsp;Paillard, R.&nbsp;Lavery, and A.&nbsp;M. 
Sengupta. Non-additivity in protein-DNA binding.<em>Bioinformatics</em>, 
21(10):2254-63, May 2005. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  MOTIVATION: Localizing protein binding sites within genomic DNA 
is of considerable importance, but remains difficult for protein families, such 
as transcription factors, which have loosely defined target sequences. It is 
generally assumed that protein affinity for DNA involves additive contributions 
from successive nucleotide pairs within the target sequence. This is not 
necessarily true, and non-additive effects have already been experimentally 
demonstrated in a small number of cases. The principal origin of non-additivity 
involves the so-called indirect component of protein-DNA recognition which is 
related to the sequence dependence of DNA deformation induced during complex 
formation. Non-additive effects are difficult to study because they require the 
identification of many more binding sequences than are normally necessary for 
describing additive specificity (typically via the construction of weight 
matrices). RESULTS: In the present work we will use theoretically estimated 
binding energies as a basis for overcoming this problem. Our approach enables 
us to study the full combinatorial set of sequences for a variety of 
DNA-binding proteins, make a detailed analysis of non-additive effects and 
exploit this information to improve binding site predictions using either 
weight matrices or support vector machines. The results underline the fact 
that, even in the presence of significant deformation, non-additive effects may 
involve only a limited number of dinucleotide steps. This information helps to 
reduce the number of binding sites which need to be identified for successful 
predictions and to avoid problems of over-fitting. AVAILABILITY: The SVM 
software is available upon request from the authors. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Hofmann2005Concept-based]  
Oliver Hofmann and Dietmar Schomburg. Concept-based annotation of enzyme 
classes.<em>Bioinformatics</em>, 21(9):2059-66, May 2005. [&nbsp;bib&nbsp;| DOI
&nbsp;|http&nbsp;| .pdf&nbsp;] 
<blockquote>  MOTIVATION: Given the explosive growth of biomedical data as 
well as the literature describing results and findings, it is getting 
increasingly difficult to keep up to date with new information. Keeping 
databases synchronized with current knowledge is a time-consuming and expensive 
task-one which can be alleviated by automatically gathering findings from the 
literature using linguistic approaches. We describe a method to automatically 
annotate enzyme classes with disease-related information extracted from the 
biomedical literature for inclusion in such a database. RESULTS: Enzyme names 
for the 3901 enzyme classes in the BRENDA database, a repository for 
quantitative and qualitative enzyme information, were identified in more than 
100,000 abstracts retrieved from the PubMed literature database. Phrases in the 
abstracts were assigned to concepts from the Unified Medical Language System 
(UMLS) utilizing the MetaMap program, allowing for the identification of 
disease-related concepts by their semantic fields in the UMLS ontology. 
Assignments between enzyme classes and diseases were created based on their 
co-occurrence within a single sentence. False positives could be removed by a 
variety of filters including minimum number of co-occurrences, removal of 
sentences containing a negation and the classification of sentences based on 
their semantic fields by a Support Vector Machine. Verification of the 
assignments with a manually annotated set of 1500 sentences yielded favorable 
results of 92% precision at 50% recall, sufficient for inclusion in a 
high-quality database. AVAILABILITY: Source code is available from the author 
upon request. SUPPLEMENTARY INFORMATION: 
ftp.uni-koeln.de/institute/biochemie/pub/brenda/info/diseaseSupp.pdf. 
</blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Bao2005Prediction]  Lei 
Bao and Yan Cui. Prediction of the phenotypic effects of non-synonymous single 
nucleotide polymorphisms using structural and evolutionary information.<em>
Bioinformatics</em>, 21(10):2185-90, May 2005. [&nbsp;bib&nbsp;| DOI&nbsp;| http
&nbsp;|.pdf&nbsp;] 
<blockquote>  MOTIVATION: There has been great expectation that the knowledge 
of an individual's genotype will provide a basis for assessing susceptibility 
to diseases and designing individualized therapy. Non-synonymous single 
nucleotide polymorphisms (nsSNPs) that lead to an amino acid change in the 
protein product are of particular interest because they account for nearly half 
of the known genetic variations related to human inherited diseases. To 
facilitate the identification of disease-associated nsSNPs from a large number 
of neutral nsSNPs, it is important to develop computational tools to predict 
the phenotypic effects of nsSNPs. RESULTS: We prepared a training set based on 
the variant phenotypic annotation of the Swiss-Prot database and focused our 
analysis on nsSNPs having homologous 3D structures. Structural environment 
parameters derived from the 3D homologous structure as well as evolutionary 
information derived from the multiple sequence alignment were used as 
predictors. Two machine learning methods, support vector machine and random 
forest, were trained and evaluated. We compared the performance of our method 
with that of the SIFT algorithm, which is one of the best predictive methods to 
date. An unbiased evaluation study shows that for nsSNPs with sufficient 
evolutionary information (with not &lt;10 homologous sequences), the 
performance of our method is comparable with the SIFT algorithm, while for 
nsSNPs with insufficient evolutionary information (&lt;10 homologous 
sequences), our method outperforms the SIFT algorithm significantly. These 
findings indicate that incorporating structural information is critical to 
achieving good prediction accuracy when sufficient evolutionary information is 
not available. AVAILABILITY: The codes and curated dataset are available at 
http://compbio.utmem.edu/snp/dataset/ </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm </blockquote> <br>
 [Vert2006Kernels]  J.-P. 
Vert, R.&nbsp;Thurman, and W.&nbsp;S. Noble. Kernels for gene regulatory 
regions. In Y.&nbsp;Weiss, B.&nbsp;Sch&ouml;lkopf, and J.&nbsp;Platt, editors,
<em>Adv. Neural. Inform. Process Syst.</em>, volume&nbsp;18, pages 1401-1408, 
Cambridge, MA, 2006. MIT Press. [&nbsp;bib&nbsp;] 
<blockquote>  Keywords: biosvm </blockquote> <br>
<br>

<p><em>This file was generated by bibtex2html 1.94.</em></p> 
</body>