<!doctype html>
<meta charset="utf-8">
<title>Molecular Cancer | Full text | Identification of a panel of sensitive and specific DNA methylation markers for lung adenocarcinoma</title>
<body>
 molecular-cancer.com/article/10.1186/1476/4598/6/70 <br>
 Bottom,Top 
<dl> 
<dt></dt> 
<dd> </dd> </dl> 
<dl> 
<dt></dt> 
<dd> </dd> </dl> 
<ul> 
<li><strong>Welcome Carnegie Mellon University[NERL]</strong></li> 
<li>Log on</li> </ul> 
<ul> 
<li> &nbsp; BioMed Central </li> 
<li>Journals</li> 
<li> Gateways </li> </ul> 
<h1> </h1> 3.78 <br>
 Search this journal BioMed Central  for Go<br>
Advanced 
search <br>

<ul> 
<li>Home</li> 
<li>Articles</li> 
<li>Authors</li> 
<li>Reviewers</li> 
<li>About this journal</li> 
<li>My Molecular Cancer</li> </ul> 
<ul> 
<ul> 
<li> Top </li> 
<li> Abstract </li> 
<li> Background </li> 
<li> Results </li> 
<li> Discussion </li> 
<li> Conclusion </li> 
<li> Methods </li> 
<li> Competing interests </li> 
<li> Authors' contributions </li> 
<li> Acknowledgements </li> 
<li> References </li> </ul> </ul> 
<dl> 
<dl> 
<dt> Advertisement </dt> 
<dd> </dd> </dl> 
<dl> 
<dt> Advertisement </dt> 
<dd> </dd> </dl> </dl> 
<h5><strong>Molecular Cancer</strong></h5> 
<ul> 
<li>Volume 6</li> </ul> 
<h5>Viewing options</h5> 
<ul> 
<li> Abstract </li> 
<li> <strong>Full text</strong> </li> 
<li> PDF (566KB) </li> </ul> 
<h5>Associated material</h5> 
<ul> 
<li> PubMed record </li> 
<li>About this article</li> 
<li> Readers' comments </li> 
<li> </li> </ul> 
<h5>Related literature</h5> 
<ul> 
<li> 
<h6>Cited by</h6></li> 
<h6>Other articles by authors</h6> 
<li> <i>&nbsp;</i>on Google Scholar 
<ul> 
<li> Tsou JA </li> 
<li> Galler JS </li> 
<li> Siegmund KD </li> 
<li> Laird PW </li> 
<li> Turla S </li> 
<li> Cozen W </li> 
<li> Hagen JA </li> 
<li> Koss MN </li> 
<li> Laird-Offringa IA </li> </ul> </li> 
<li> <i>&nbsp;</i>on PubMed 
<ul> 
<li>Tsou JA </li> 
<li>Galler JS </li> 
<li>Siegmund KD </li> 
<li>Laird PW </li> 
<li>Turla S </li> 
<li>Cozen W </li> 
<li>Hagen JA </li> 
<li>Koss MN </li> 
<li>Laird-Offringa IA </li> </ul> </li> </ul> 
<h6>Related articles/pages</h6> 
<ul> 
<li>on Google</li> 
<li>on Google Scholar</li> 
<li>on PubMed</li> </ul> 
<h5>Tools</h5> 
<ul> 
<li> Download references </li> 
<li>Download XML</li> 
<li> Email to a friend </li> 
<li>Order reprints</li> 
<li> Post a comment </li> </ul> 
<h5>Share this article</h5> 
<ul> 
<li> </li> 
<li> Tweet </li> 
<li> </li> 
<li> &nbsp;More options... 
<ul> 
<li> Citeulike </li> 
<li> Connotea </li> 
<li> Del.icio.us </li> 
<li> Email </li> 
<li> Facebook </li> 
<li> <br>
Google+ </li> 
<li> Mendeley </li> 
<li> Twitter </li> </ul> </li> </ul> Research 
<h1>Identification of a panel of sensitive and specific DNA methylation 
markers for lung adenocarcinoma</h1> 
<p> <strong>Jeffrey A Tsou</strong>1&#134;, <strong>Janice S Galler</strong>1
&#134;, <strong>Kimberly D Siegmund</strong>2, <strong>Peter W Laird</strong>3,1
,<strong>Sally Turla</strong>4, <strong>Wendy Cozen</strong>2, <strong>Jeffrey 
A Hagen</strong>3, <strong>Michael N Koss</strong>4 and <strong>Ite A 
Laird-Offringa</strong>3,1* </p> 
<ul> 
<li> 
<p> * Corresponding author: Ite A Laird-Offringa ilaird@usc.edu </p> </li> 
<li>&#134; Equal contributors</li> </ul> 
<p><i></i>Author Affiliations</p> 
<p> 1 Norris Cancer Center and Department of Surgery and ofBiochemistry and 
Molecular Biology, Keck School of Medicine, University of Southern California, 
Los Angeles, CA 90089-9176, USA</p> 
<p> 2 Department of Preventive Medicine, Keck School of Medicine, University 
of Southern California, Los Angeles, CA 90089-9176, USA</p> 
<p> 3 Department of Surgery, Keck School of Medicine, University of Southern 
California, Los Angeles, CA 90089-9176, USA</p> 
<p> 4 Department of Pathology, Keck School of Medicine, University of Southern 
California, Los Angeles, CA 90089-9176, USA</p> 
<p> For all author emails, please log on. </p> 
<p><em>Molecular Cancer</em> 2007, <strong>6</strong>:70&nbsp;
doi:10.1186/1476-4598-6-70</p> <br>

<p>The electronic version of this article is the complete one and can be found 
online at:http://www.molecular-cancer.com/content/6/1/70</p> <br>
Received: 5 
September 2007 <br>
Accepted: 29 October 2007 <br>
Published: 29 October 2007 
<br> <br>
<br>

<p> &copy; 2007 Tsou et al; licensee BioMed Central Ltd. <br>
</p> 
<p> This is an Open Access article distributed under the terms of the Creative 
Commons Attribution License (http://creativecommons.org/licenses/by/2.0), which 
permits unrestricted use, distribution, and reproduction in any medium, 
provided the original work is properly cited.</p> 
<h3>Abstract</h3> 
<h4>Background</h4> 
<p>Lung cancer is the number one cancer killer of both men and women in the 
United States. Three quarters of lung cancer patients are diagnosed with 
regionally or distantly disseminated disease; their 5-year survival is only 
15%. DNA hypermethylation at promoter CpG islands shows great promise as a 
cancer-specific marker that would complement visual lung cancer screening tools 
such as spiral CT, improving early detection. In lung cancer patients, such 
hypermethylation is detectable in a variety of samples ranging from tumor 
material to blood and sputum. To date the penetrance of DNA methylation at any 
single locus has been too low to provide great clinical sensitivity. We used 
the real-time PCR-based method MethyLight to examine DNA methylation 
quantitatively at twenty-eight loci in 51 primary human lung adenocarcinomas, 
38 adjacent non-tumor lung samples, and 11 lung samples from non-lung cancer 
patients.</p> 
<h4>Results</h4> 
<p>We identified thirteen loci showing significant differential DNA 
methylation levels between tumor and non-tumor lung; eight of these show highly 
significant hypermethylation in adenocarcinoma: CDH13, CDKN2A EX2, CDX2, HOXA1, 
OPCML, RASSF1, SFPR1, and TWIST1 (p-value &lt;&lt; 0.0001). Using the current 
tissue collection and 5-fold cross validation, the four most significant loci 
(CDKN2A EX2, CDX2, HOXA1 and OPCML) individually distinguish lung 
adenocarcinoma from non-cancer lung with a sensitivity of 67&ndash;86% and 
specificity of 74&ndash;82%. DNA methylation of these loci did not differ 
significantly based on gender, race, age or tumor stage, indicating their wide 
applicability as potential lung adenocarcinoma markers. We applied random 
forests to determine a good classifier based on a subset of our loci and 
determined that combined use of the same four top markers allows identification 
of lung cancer tissue from non-lung cancer tissue with 94% sensitivity and 90% 
specificity.</p> 
<h4>Conclusion</h4> 
<p>The identification of eight CpG island loci showing highly significant 
hypermethylation in lung adenocarcinoma provides strong candidates for 
evaluation in patient remote media such as plasma and sputum. The four most 
highly ranked loci, CDKN2A EX2, CDX2, HOXA1 and OPCML, which show significant 
DNA methylation even in stage IA tumor samples, merit further investigation as 
some of the most promising lung adenocarcinoma markers identified to date.</p> 
<h3>Background</h3> 
<p>Lung cancer is expected to cause over 160,000 deaths in 2007 -killing more 
Americans than cancer of the prostate, breast, colon, rectum and pancreas 
combined[1]. Lung cancer is clinically classified into two classes: the 
aggressive subtype small cell lung cancer (SCLC, ~13% of cases) and non-small 
cell lung cancer (NSCLC, the remaining ~87%)[1]. NSCLC is histologically 
subdivided into four major subtypes with distinct pathological and molecular 
characteristics: adenocarcinoma, squamous cell lung cancer, large cell lung 
cancer and &quot;other&quot; (comprising neuroendocrine cancers, carcinoids 
etc.)[2]. Of these, adenocarcinoma has recently surpassed squamous cell lung 
cancer as the most common subtype in the United States, accounting for 
approximately 40% of NSCLC[3]. The incidence of lung adenocarcinoma is on the 
rise in many countries, in particular in women[4,5]. Adenocarcinoma is also the 
most common lung cancer subtype in non- and previous smokers[6]. </p> 
<p>The 5-year survival of lung cancer patients is only 15%, largely due to the 
fact that three quarters of lung cancer patients are diagnosed when their 
disease has spread regionally or distantly[7]. To make an impact on long term 
survival, better strategies are needed for early detection. Prior experience 
with chest X-ray, sputum cytology, and fiberoptic examination have failed to 
decrease lung cancer patient mortality, although several recent strategies show 
promise. Spiral computed tomography (spiral CT) is one such approach. It allows 
detailed imaging of the lung, and can detect very small lesions. Recent results 
from the Early Lung Cancer Action Project (ELCAP) indicate that this approach 
allows detection of early stage lung cancer[8], but in this and other studies, 
non-cancerous lesions far outnumber malignancies (less than 10% of lesions are 
cancer). In addition, it is unclear whether the early stage lung cancers 
identified by spiral CT represent cancers that would ultimately progress and 
lead to death. A recent analysis suggests spiral CT screening may not reduce 
lung cancer mortality[9]. Molecular analyses of plasma, sputum, and bronchial 
lavage fluids have also shown promise as strategies for early detection, but 
these methods still lack sensitivity[10]. If molecular markers with high 
sensitivity and specificity for cancers that will progress can be identified, 
such markers could be combined with spiral CT to screen high-risk individuals, 
allowing molecular detection and visualization of clinically relevant early 
lesions. This would greatly increase the chances of curative resection of lung 
cancer, while minimizing unnecessary and potentially life-threatening 
procedures in patients with benign lesions.</p> 
<p>Of the many potential molecular markers, DNA hypermethylation &ndash; an 
epigenetic alteration &ndash; shows great promise. DNA hypermethylation occurs 
in all cancers, frequently leading to gene silencing through methylation of 
CpG-rich regions (CpG islands) near the transcriptional start sites of genes[11
]. In lung cancer patients, such hypermethylation is quantitatively detectable 
in a variety of samples ranging from tumor material to blood and sputum[10]. 
However, to date the penetrance of DNA methylation at any single locus has not 
been high enough to provide great clinical sensitivity. Our focus is to 
increase the repertoire of sensitive DNA hypermethylation markers for lung 
cancer, and to compose a small panel of molecular markers that could be used to 
detect lung cancer with high sensitivity and specificity. Given the 
histopathologic, clinical and molecular differences between lung cancer 
subtypes, we believe that markers should be developed individually for the 
major histological subtypes. These markers can later be combined into a lung 
cancer hypermethylation panel that can be used for detection of all lung 
cancers.</p> 
<p>Because of its increasing frequency and its preponderance in non- and 
previous smokers, we focused first on lung adenocarcinoma (AD). Here we 
describe our evaluation of 28 potential DNA methylation markers using primary 
human lung adenocarcinoma samples. To ensure that these markers detect<em>
cancer-specific</em>hypermethylation changes, associated with histologically 
visible lung cancer (allowing surgical resection), we compared the DNA 
methylation profiles of the tumors with histologically normal adjacent lung 
tissue (AdjNTL) from lung cancer patients. We also examined non-tumor lung from 
non-cancer patients (NTL).</p> 
<h3>Results</h3> 
<p>Ideal DNA hypermethylation markers for lung adenocarcinoma should show a 
high frequency of methylation in tumors as well as DNA methylation levels that 
are significantly elevated in tumor compared to non-tumor lung tissue. 
Environmental exposures, such as those arising from tobacco smoke, could lead 
to higher basal levels of methylation in non-tumor lung[12], which might affect 
the background signal when any resulting markers are applied to non-invasive 
molecular analyses of bodily fluids in the future. To ensure the identification 
of markers that are more highly methylated in adenocarcinoma even when compared 
to heavily exposed but histologically cancer-free lung, we used adjacent 
non-tumor lung (AdjNTL) from lung cancer cases as our cancer-free comparison. 
The AdjNTL sections were derived from separate, histologically verified 
cancer-free paraffin blocks. We also examined a number of non-tumor lung (NTL) 
samples from patients operated for non-cancer reasons (emphysema, lung 
collapse, etc.). Quantitative assessment of DNA methylation levels allows a 
more detailed evaluation of candidate DNA methylation markers, and of their 
suitability for correctly identifying a cancer vs. non-cancer sample. For this 
reason, we used the bisulfite conversion based real-time PCR technique, 
MethyLight, to measure DNA methylation in tumor and control tissues[13]. </p> 
<p>Twenty-eight loci were chosen for evaluation (Table 1). The choice of loci 
was based on a prescreening of 114 loci carried out on a collection of human 
lung cancer cell lines, including 11 adenocarcinoma cell lines (unpublished 
data). We also included many loci that appeared promising based on previous 
reports describing their DNA methylation in lung cancer or other cancers, so 
that all markers of interest could be compared on one set of tissues using a 
single technique and platform. Among others, the 28 loci included CpG islands 
in the promoters of tumor suppressor genes and genes with important roles in 
cell cycle regulation, DNA repair, and apoptosis (Table1). </p> 
<p><strong>Table 1.</strong> Gene name and function of the 28 loci studied </p>
<p>The results of the DNA methylation analyses for the 28 loci in 51 AD, 38 
AdjNTL and 11 NTL samples are shown in Fig.1. DNA methylation, expressed as the 
percentage methylated reference (PMR[14]) is visualized by color coding. 
Comparison of AD in Fig.1 panel A with AdjNTL in panel B shows that a number of 
loci are more heavily methylated in AD. The effect appears to be even more 
pronounced when AD is compared to NTL from non-cancer patients. Although the 
exposure history of these NTL samples is unknown, their generally lower DNA 
methylation levels emphasize that these samples may not be the best controls 
when searching for loci that show cancer-specific hypermethylation. 
Interestingly, for one locus, LZTS1 (leftmost locus in Fig.1), the DNA 
methylation pattern appeared to be reversed; NTL showed the highest level of 
DNA methylation, while AD samples were least methylated.</p> 
<p><strong>Figure 1.</strong> Graphic representation of PMR values obtained 
for 28 loci in AD (A), AdjNTL (B) and NTL (C). Samples are indicated at the 
left, loci at thetop. PMR values have been categorized as colored boxes 
denoting no detectable DNA methylation (blue), DNA methylation below the median 
of all positive samples of each locus (yellow), and DNA methylation equal to or 
above the median (red). The black bar at bottom indicates loci showing 
statistically significant differences in DNA methylation levels between tumor 
and non-tumor lung.</p> 
<p>We applied two-dimensional hierarchical clustering to examine the 
relationship between the loci and the tumor and non-tumor lung samples (Fig2; 
VHL was omitted because it showed no DNA methylation in any samples). All but 
one of the tumor samples clustered together in a major branch of the 
dendrogram, while the majority of non-tumor lung samples grouped in a separate 
cluster. Nine loci, CDH13, SFRP1, OPCML, TWIST1, SFRP5, CDKN2A EX2, CDX2, HOXA1 
and RASSF1, clustered together (bottom right), showing heavier DNA methylation 
in the tumor samples.</p> 
<p><strong>Figure 2.</strong> Two-dimensional hierarchical clustering of 
samples and loci based on DNA methylation data. In the center, DNA methylation 
levels are indicated by a color gradient, with the highest DNA methylation 
levels for each locus indicated in red and the lowest in deep blue. The Ward 
hierarchical clustering method was used to categorize between cancer and 
non-tumor samples. Sample IDs are indicated on the left, with AD samples in 
red, AdjNTL samples in black, and NTL samples in blue. The relationship of 
samples is indicated at right in the same color schematic as the labels. At 
bottom, the relationship of the loci is indicated. Note that all eight of the 
most significant loci cluster at bottom right.</p> 
<p>We next analyzed the statistical significance of the differences in DNA 
methylation levels for individual markers and different combinations of tissue 
samples (Table2): AD vs. all NTL samples, AD vs. AdjNTL, and AD vs. <em>paired 
</em>AdjNTL (32 of the 38 AdjNTL samples were derived from the AD patients in 
Fig.1A). The paired AdjNTL form an exquisite control for the cancer-specific 
nature of the observed DNA methylation changes, as each of these samples 
conforms to its tumor sample in patient age, environmental exposure, and 
genetic background. To avoid assigning statistical significance to spurious 
associations, we incorporated a multiple comparisons threshold for those loci 
that at time of analysis lacked any prior data suggesting they might be 
hypermethylated in lung adenocarcinoma (Table2, before-last column, [15] see 
Materials and Methods for details). Thirteen of the analyzed loci showed 
statistically significant differences in DNA methylation when AD samples were 
compared to all NTL samples: OPCML, CDX2, HOXA1, CDKN2A EX2, SFRP1, CDH13, 
TWIST1, LZTS1, RASSF1, SFRP4 and 5, ESR1, and CDH1. All of these except CDH1 
remained significant when AD samples were compared to AdjNTL, while all except 
LZTS1 remained significant in the comparison of AD to paired AdjNTL. Because 
DNA methylation of LZTS1 is reduced in tumors it is not a candidate for a 
positive lung adenocarcinoma marker and it was not studied further at this 
time. APC methylation was found to be statistically significantly different 
only when paired tumor and non-tumor lung samples were compared. This suggests 
that basal DNA methylation is high but variable at this locus; elevated DNA 
methylation in tumors is likely masked by interpatient variability and only 
becomes visible when samples from the same patient are compared. Indeed, Waki 
and coworkers have observed frequent DNA methylation of APC in non-cancer lung 
and other organs[16]. </p> 
<p><strong>Table 2.</strong> Frequency and median PMR values of AD, Adj NTL 
and NTL tissues for 28 loci</p> 
<p>Of the thirteen significant loci, OPCML, CDX2, HOXA1, CDKN2A EX2, SFRP1, 
CDH13, TWIST1 and RASSF1 show considerable promise as cancer-specific DNA 
methylation markers, exhibiting highly significant hypermethylation in tumors 
compared to paired non-tumor tissues (p &le; 1 &times; 10-7, Table 2). All 
eight of these loci grouped together in the hierarchical clustering (Fig.2). 
The ability of the top four candidates, CDKN2A EX2, CDX2, HOXA1 and OPCML (all 
p &lt; 1 &times; 10-9), to individually identify lung cancer samples was next 
evaluated. Fig.3 shows the distribution of PMR values in the examined sample 
collection. Note that for all four markers, the mean value in non-tumor lung 
from<em>non-cancer </em>patients is lower than that of adjacent non-tumor lung 
from<em>lung cancer </em>patients. This emphasizes the importance of using 
histologically normal tissue adjacent to lung cancer for comparison; such 
tissue may show higher basal DNA methylation levels while appearing 
histologically normal, and should be used for comparison with lung cancer 
tissue to ensure identification of<em>cancer-specific </em>markers. While all 
four markers show increased DNA methylation in adenocarcinoma compared to 
adjacent non-tumor tissue, the spread of DNA methylation levels differs, which 
would affect their sensitivity and specificity in future detection strategies. 
The marker potential of quantitative markers is frequently presented in the 
form of a receiver operating characteristic (ROC) curve, in which sensitivity 
vs. 1-specificity at all possible cut-off values is plotted. While these DNA 
methylation markers are ultimately intended for the non-invasive analysis of 
patient bodily fluids, a preliminary indication of their potential to 
sensitively and specifically detect cancer could be obtained by plotting ROC 
curves using the PMR values from the tumor vs. adjacent non-tumor samples. Fig.4
 shows that the area under the curve (AUC, and indicator of marker performance 
that would be 1 for a marker showing 100% specificity and sensitivty) is 
0.87&ndash;0.95 for the four top loci.</p> 
<p><strong>Figure 3.</strong> The distribution of PMR values by group. 
Log-transformed PMR values for AD, AdjNTL and NTL are shown. The mean is shown 
by the wide horizontal line, and the top and bottom of the diamond indicate a 
95% normal confidence interval for the sample mean.</p> 
<p><strong>Figure 4.</strong> Receiver operating characteristic curves for the 
four top markers. All AD and AdjNTL lung samples for which there was complete 
DNA methylation data were used for the analysis.</p> 
<p>Despite the promising AUC values, the sensitivity and specificity of these 
top four markers, used<em>individually </em>and determined using the current 
sample collection in a five-fold cross-validation, was limited: 67&ndash;86% 
and 74&ndash;82% respectively. This supports the notion that DNA 
hypermethylation markers are best used in the form of a panel. Because of the 
costs associated with quantitative molecular analyses, it would be important to 
limit the number of markers included in the panel. To determine which 
combinations of markers would be most effective to correctly identify tumor vs. 
non-tumor samples, we fit a random forest classifier to the data set, using 87 
samples and 28 variables (2 AD samples with missing PMR data were omitted, 
resulting in 49 AD vs. 38 AdjNTL). Using bootstrap samples of the data, we grew 
a forest of 30,000 trees. Splits were determined using a random sample of five 
variables and trees were grown until there was only one observation in each 
leaf. Utilizing all 28 loci, we estimated a sensitivity of 92% and a 
specificity of 95%. Using the Gini index from the random forest classifier 
(last column, Table2) to measure locus importance, we restricted our analysis 
to the most highly ranked variables. Reducing the locus number to 13 did not 
affect sensitivity and specificity, and limiting our markers to the top-ranked 
four (HOXA1, OPCML, CDKN2AEX2 and CDX2, which were also the most significant 
based on our statistical analysis) resulted in a sensitivity of 94% and a 
specificity of 90%. Thus, these four markers appear to be highly promising DNA 
hypermethylation markers for development into non-invasive molecular markers of 
lung adenocarcinoma, through examination of DNA shed into bodily fluids such as 
sputum, bronchioalveolar lavage, or blood.</p> 
<p>For candidate hypermethylation markers of lung adenocarcinoma, two 
important questions arise. First, are these markers hypermethylated in cancer 
samples irrespective of the subject's age, gender and racial/ethnic background? 
And secondly, are these markers hypermethylated even in the earliest stages of 
lung adenocarcinoma? While the population analyzed in the current study is 
small, we reasoned that an indication of the potential of our top four markers 
to broadly identify lung adenocarcinoma might be obtained. To address the first 
question, we assessed correlations to age and determined whether each of the 
four markers showed statistically significant hypermethylation in tumor vs. 
adjacent normal tissues in men, women, and all four racial/ethnic groups. We 
found no correlation of methylation of CDKN2A EX2, CDX2, HOXA1 and OPCML with 
the age. In addition, all four markers remained significantly hypermethylated 
in tumor vs. AdjNTL when subjects were stratified by gender or by ethnic group 
(p &lt; 0.05) (Table3). The only exception was CDKN2A EX2 methylation in Asian 
subjects (p = 0.11), which may be related to the small sample size but will 
need to be further explored.</p> 
<p><strong>Table 3.</strong> Performance of top four markers in samples based 
on gender, race/ethnicity and stage</p> 
<p>To address the second question, we determined whether each of the four 
markers was significantly hypermethylated in early and later stage tumors, 
using paired samples (Table3). We examined stages IA and IB individually, but 
grouped stages IIA, IIB, and IIIA (only one paired sample was available for IIA 
and IIIA, and none for stage IIIB). Importantly, all four markers were 
significantly hypermethylated in stage IA cancers. Only CDKN2A EX2 
hypermethylation was significant in stage IB tumors, but this could be due to 
the small number of paired samples (n = 6). All markers were also significantly 
hypermethylated in later stage lung adenocarcinoma (Stages IIA-IIIA). These 
analyses indicate that the top four markers show high potential for 
identification of lung adenocarcinoma, even in its earliest stages, an 
important characteristic if these markers are to be used for early detection.
</p> 
<p>To determine whether any of the four top markers might have prognostic 
implications, we determined whether there was any relationship between their 
DNA methylation level and survival. We found no significant association between 
DNA methylation and survival for the four loci, or any of the other 24 loci 
studied (data not shown).</p> 
<h3>Discussion</h3> 
<p>Based on the results of our analyses, four loci that are very strong 
candidates for a DNA methylation panel aimed at early lung adenocarcinoma 
detection have been identified: CDKN2A EX2, CDX2, HOXA1 and OPCML. CDNK2A, also 
referred to as<em>p16</em><em>INK4a</em>, encodes an important cell cycle 
regulator that is frequently inactivated in cancer. CDKN2A is one of the first 
tumor suppressor genes found to be methylated in a variety of cancers, 
including lung cancer[17]. It is one of the most widely studied hypermethylated 
loci, and methylation of its promoter CpG island appears to be a very early 
event in the development of non-small cell lung cancer (recently reviewed in[10
]). In fact, methylation of the CDKN2A promoter CpG island has been observed in 
the sputum of subjects at risk for lung cancer 3 years prior to diagnosis[18] 
and in the sputum of asymptomatic heavy smokers[19]. A recent analysis of 
prospectively collected sputum showed CDKN2A methylation in 39% of cases and 
25% of controls; methylation of this gene was associated with an elevated risk 
of lung cancer[20]. It is thought that DNA methylation observed in the sputum 
is indicative of field cancerization of the airways and not necessarily a 
symptom of a present cancer[20]. Our goal was to identify cancer-specific 
markers, not risk markers. We had evaluated DNA methylation of the CDKN2A 
promoter CpG island as a cancer indicator, but found substantial DNA 
methylation in AdjNTL, and no significant difference between AdjNTL and cancer 
(data not shown). Based on the cancer-specific hypermethylation of the CDKN2A 
exon 2 CpG island observed in colorectal and bladder cancers[21,22], we tested 
this downstream island instead. We established that its level of DNA 
methylation is a strong indicator of lung adenocarcinoma. While substantial 
methylation at the exon 2 CpG island is detected in histologically normal 
AdjNTL, by comparison, DNA methylation in adenocarcinoma is highly 
significantly elevated (p &le; 1 &times; 10E-10). The detection of CDKN2A 
methylation in a high fraction of lung cancer patient plasma samples bodes well 
for its application to non-invasive detection[23]. Two groups reported an 
association of DNA methylation of CDKN2A with poor survival in 
adenocarcinoma/NSCLC patients[24,25], while Divine et al (2005), like us, 
reported no such association[12]. The differences between the obtained results 
might be due to the examination of a different CpG island or a different 
population.</p> 
<p>DNA methylation of HOX genes, encoding homeobox transcription factors 
involved in embryogenesis and differentiation, had recently been observed in 
lung adenocarcinoma and squamous cell lung cancer. In an analysis of eight 
adenocarcinomas and matching adjacent lung, substantial DNA methylation of the 
HOXA and D clusters was observed[26]. Five cancer samples showed DNA 
methylation of HOXA1, while only one AdjNTL sample was methylated at this 
locus. In a different study, analysis of a stage I adenocarcinoma and squamous 
cell lung carcinoma showed DNA methylation of the HOX clusters, and examination 
of the HOXA and D clusters in more detail in squamous cell cancers and control 
tissue indicated a DNA methylation frequency of 45&ndash;80% for HOXA7-9, but 
methylation of HOXA1 was limited[27]. Neither of these studies examined a large 
number of adenocarcinomas, nor were quantitative techniques used. Here we 
demonstrate that HOXA1 is a very promising DNA methylation marker for lung 
adenocarcinoma. We have also observed DNA methylation of additional HOX genes 
(unpublished studies), but HOXA1 appears to be particularly informative.</p> 
<p>OPCML, encoding an opioid-binding cell adhesion molecule, has been shown to 
be frequently methylated in ovarian cancer[28]. Given that opioids have 
demonstrated growth inhibitory and pro-apoptotic effects in lung cancer cells[29
-31], it is perhaps not surprising that the OPCML promoter CpG island might be 
a target for DNA methylation in lung cancer. Very recently, high throughput DNA 
methylation profiling of 11 lung adenocarcinomas and control lung identified a 
number of CpG dinucleotides methylated in the cancer samples[32]. One probe 
identified DNA methylation in the area covered by the OPCML probe used here. 
Although the OPCML locus was not studied in detail in the Bibikova study, the 
observed methylation supports the idea that OPCML is a strong candidate marker 
in lung adenocarcinoma.</p> 
<p>CDX2, another homeobox transcription factor, had been described to be 
methylated in squamous esophageal cancer[33] and colorectal carcinoma [34], but 
to our knowledge, its DNA methylation in lung cancer has never been examined. 
We find it to be methylated in 100% of lung adenocarcinomas, showing a 10-fold 
higher median methylation than AdjNTL tissue (Table2). </p> 
<p>Because our primary goal is marker development, here we focused only on 
whether loci showed consistent hypermethylation. Whether or not this 
hypermethylation results in gene inactivation is not relevant for the use of 
these loci as DNA methylation markers, and was not determined. However, the 
biological consequences of the observed hypermethylation would also be worth 
investigating. While each of the four top-ranked loci is of interest as a DNA 
methylation marker, it is as a<em>panel </em>that they promise to be most 
powerful. To our knowledge, we are the first to examine CDKN2A EX2, CDX2, HOXA1 
and OPCML in combination. The fact that this marker set allows identification 
of cancer specimens in the current tissue collection with a substantially 
higher sensitivity and specificity than any previously identified single 
markers underlines the importance of developing suitable marker panels.</p> 
<h3>Conclusion</h3> 
<p>From a starting panel of 28 DNA methylation loci, we have identified 13 
that show statistically significant methylation differences between lung 
adenocarcinoma and non-cancer lung tissue. Of these, 8 show highly significant 
differences. The four most significant markers also ranked as the top four to 
be used in a marker panel, as determined by a random forest approach. Thus, we 
suggest that CDKN2A EX2, CDX2, HOXA1 and OPCML are the top candidates from the 
28 tested, and should be validated as DNA methylation markers for lung 
adenocarcinoma. These validations should consist of examining a sufficiently 
large number of new subjects representing both genders and all four major 
ethnic/racial subgroups in the United States (Whites of non-Hispanic and 
Hispanic descent, Blacks, and Asians), as well as early and late stage cancer. 
Such studies are currently ongoing. Our analyses of the present sample 
collection, which contains modest numbers of representatives from all these 
groups, is very encouraging as they suggest that the markers function 
independently of subject age, gender or ethnic subgroup, and are positive in 
early stage cancer.</p> 
<p>The next step would entail the exploration of different methods to measure 
these markers non-invasively in early stage lung cancer patients. Potential 
&quot;remote&quot; media to be considered are sputum, bronchioalveolar lavage, 
and blood plasma, all of which we are in the process of collecting for 
examination. The ability of our four-marker panel to clinically detect lung 
cancer with high sensitivity and specificity will depend on many factors. A 
loss of sensitivity might be foreseen due to the small amounts of DNA shed into 
the blood of each patient, but at the same time, an increase in specificity 
might be expected if tumor DNA is shed more readily into the bloodstream than 
DNA from adjacent histologically normal tissue.</p> 
<p>To our knowledge, CDKN2A EX2, CDX2, HOXA1 and OPCML constitute the 
strongest lung adenocarcinoma DNA methylation markers identified to date, and 
we are working on further evaluations of their potential with great 
anticipation.</p> 
<h3>Methods</h3> 
<h4>Study subjects</h4> 
<p>Lung adenocarcinoma and when available adjacent non-tumor lung was obtained 
from archival paraffin blocks from 51 subjects who had been treated at three 
Los Angeles hospitals: the Los Angeles County Hospital, the USC University 
Hospital and the Norris Comprehensive Cancer Center. Clinical information was 
missing for 5 patients. Of the rest, 28 were male and 18 were female, 14 were 
White of non-Hispanic descent, 14 White of Hispanic descent, 11 were Black, and 
7 were of various Asian origins. Ages ranged from 37&ndash;82 years old at time 
of surgery (median: 58 years old). For 32 of these cases, a separate paraffin 
block containing histologically verified cancer-free lung was available. These 
adjacent non-tumor lung samples were supplemented with 6 additional cancer-free 
archival samples from lung cancer patients for which the tumor block was 
unavailable, and 11 archival non-tumor lung samples from patients operated for 
non-cancer reasons, such as pneumothorax or emphysema. All studies were 
institutionally approved by the University of Southern California Institutional 
Review Board (IRB# HS-016041, HS-06-00447), and the identities of patients were 
not made available to laboratory investigators.</p> 
<h4>Tissue samples and DNA extraction</h4> 
<p>Hematoxylin and eosin-stained slides were reviewed by an experienced lung 
pathologist (MNK) to support the original classification of the tumor and to 
select optimal tumor and non-tumor areas of the specimens. DNA was extracted 
from microdissected tumor and non-tumor lung samples via proteinase K digestion[
35]. Briefly, cells were lysed in a solution containing 100 mmol/L Tris-HCl (pH 
8.0), 10 mmol/L EDTA (pH 8.0), 1 mg/mL proteinase K, and 0.05 mg/mL tRNA and 
incubated at 50&deg;C overnight. The DNA was bisulfite converted as previously 
described[13]. </p> 
<h4>DNA methylation analysis</h4> 
<p>DNA methylation analysis was done by MethyLight as previously described [14
]. Primer and probe sequences were as described[14,36,37]. In addition to 
primers and probe sets designed specifically for the gene of interest, an 
internal reference primer and probe set designed to analyze Alu repeats (Alu) 
was included in the analysis to normalize for input DNA[38]. The percentage 
methylated reference (PMR) was calculated as the GENE:reference ratio of a 
sample divided by the GENE:reference ratio of<em>in vitro </em>methylated (<em>
Sss</em>I-treated) human white blood cell DNA and multiplying by 100 [14]. 
Occasionally, PMR values over 100 were observed. This can happen when genes are 
very heavily methylated in the cancer sample, while the SssI-treated sample (in 
spite of extensive<em>in vitro </em>DNA methylation) is not fully methylated at 
that locus. This does not affect the significance of the loci identified in 
this study, as the same batch of<em>Sss</em>I-treated DNA was used throughout 
the study.</p> 
<h4>Statistical analyses</h4> 
<p>PMR values of AD were compared to AdjNTL and NTL lung as continuous 
variables by means of the Wilcoxon rank sum test. For the comparison of paired 
AD and AdjNTL samples from the same patients, the Wilcoxon signed rank test was 
used. To control the false discovery rate at 5%, a multiple comparisons 
threshold was set. It was only applied to those 20 loci for which no previous 
information supporting a hypothesis of DNA methylation in lung AD was available[
15]. Receiver operating characteristic (ROC) curves were plotted using the AD 
vs. all AdjNTL lung PMR values and JMP 6.0 software (SAS Institute, Cary, NC). 
The distribution of PMR values by group (AD, AdjNTL and NTL) were shown using 
log-transformed data in JMP 6.0. The two-dimensional hierarchical clustering 
was carried out using JMP 6.0 and log-transformed PMR values. VHL, which was 
negative in all specimens, was omitted from the clustering analysis. 
Associations between age, gender and race of AD cases were tested by 
dichotomizing the subjects either by the presence/absence of DNA methylation, 
or, if the samples were frequently methylated, by the median of all positive 
PMR values. All statistical tests were two-sided.</p> 
<p>To determine which combinations of markers would be most effective to 
correctly identify tumor vs. non-tumor samples, we fit a random forest 
classifier to the data set, using the R programming language (v 2.5[39]) and 87 
samples and 28 variables (2 AD samples with missing PMR data were omitted, 
resulting in 49 AD/38 AdjNTL). Using bootstrap samples of the data, we grew a 
forest of 30,000 trees. Splits were determined using a random sample of five 
variables and trees were grown until there was only one observation in each 
leaf. We determined error rates using the observations that were not used to 
generate the trees. For each observation, its outcome was predicted by having 
the majority vote from the trees that were generated without the original data 
point in their bootstrap sample. These predicted values were compared against 
the true tissue type to estimate prediction error.</p> 
<h3>Competing interests</h3> 
<p>IALO and PWL are shareholders of Epigenomics AG, which has a commercial 
interest in the development of DNA markers for disease detection and diagnosis. 
None of the work performed in the laboratories of the authors is or has been 
supported or directed by Epigenomics.</p> 
<h3>Authors' contributions</h3> 
<p>JAT was involved in marker design, experimental execution and initial 
analysis. JSG was involved in experimental execution and extensive data 
analysis, drafting the manuscript, and generation of figures. KDS oversaw 
statistical analysis and drafted statistical sections of the manuscript. PWL 
provided experimental advice and mentoring for JAT. ST helped locate and 
section most of the tissues and provided the linked and de-identified 
clinicopathological information. WC provided additional samples through the 
Norris Cancer Center's Los Angeles area tissue discard repository. JAH provided 
non-cancer lung samples and statistical discussions. MNK reviewed all slides 
prior to microdissection. IALO designed the study, oversaw all aspects of the 
project, mentored JAT and JSG, and revised manuscript drafts. All authors 
reviewed and commented on the manuscript during its drafting and approved the 
final version.</p> 
<h3>Acknowledgements</h3> 
<p>The authors thank members of the Laird lab for help with MethyLight and 
probe/primer design, and Laird-Offringa lab members for critical comments on 
the manuscript. This project was funded by grant support for IALO: National 
Institutes of Health/National Cancer Institute R21 CA102247 and R01 CA119029, 
Whittier Foundation Translational Research Grant, and a STOP Cancer award. Two 
of the cancer samples used in this study were provided by the Norris 
Comprehensive Cancer Center's NIH-funded Slide Retrieval and Tissue Discard 
Repository. None of the funding agencies played any role in the collection, 
analysis, interpretation of the data, writing of the manuscript, nor the 
decision to publish.</p> 
<h3>References</h3> 
<ol> 
<li> 
<p><strong> American Cancer Society: Cancer Facts &amp; Figures 2007. 
Atlanta:American Cancer Society.</strong> [http:/ / www.cancer.org/ docroot/ 
MED/ content/ MED_1_1_Most-Requested_Graphs_and_F igures_2007.asp] webcite</p>
<p>-. </p> 
<p></p> 
<p></p> </li> 
<li> 
<p> Travis WD, Linder J, Mackay B: <strong> Classification, histology, 
cytology, and electron microscopy.</strong>In <em>Lung cancer: principles and 
practice</em>. Edited by Pass HI, Mitchell JB, Johnson DH, Turrisi AT. 
Philadelphia , Lippincott-Raven Publishers; 1996:361-395.</p> 
<p></p> 
<p></p> </li> 
<li> 
<p> Travis WD, Travis LB, Devesa SS: <strong> Lung Cancer. </strong></p>
<p><em>Cancer</em> 1995, <strong>75</strong><strong>:</strong>191-202. 
PubMed&nbsp;Abstract | Publisher&nbsp;Full&nbsp;Text </p> 
<p></p> 
<p></p> </li> 
<li> 
<p> Chen KY, Chang CH, Yu CJ, Kuo SH, Yang PC: <strong> Distribution according 
to histologic type and outcome by gender and age group in Taiwanese patients 
with lung carcinoma.</strong></p>
<p><em>Cancer</em> 2005, <strong>103</strong><strong>:</strong>2566-2574. 
PubMed&nbsp;Abstract | Publisher&nbsp;Full&nbsp;Text </p> 
<p></p> 
<p></p> </li> 
<li> 
<p> Yoshimi I, Ohshima A, Ajiki W, Tsukuma H, Sobue T: <strong> A comparison 
of trends in the incidence rate of lung cancer by histological type in the 
Osaka Cancer Registry, Japan, and in the Surveillance, Epidemiology and End 
Results Program, USA.</strong></p>
<p><em>Jpn J Clin Oncol</em> 2003, <strong>33</strong><strong>:</strong>98-104.
PubMed&nbsp;Abstract | Publisher&nbsp;Full&nbsp;Text </p> 
<p></p> 
<p></p> </li> 
<li> 
<p> Liu NS, Spitz MR, Kemp BL, Cooksley C, Fossela FV, Lee JS, Hong WK, Khuri 
FR:<strong> Adenocarcinoma of the lung in young patients; the M.D. Anderson 
experience.</strong></p>
<p><em>Cancer</em> 2000, <strong>88</strong><strong>:</strong>1837-1841. 
PubMed&nbsp;Abstract | Publisher&nbsp;Full&nbsp;Text </p> 
<p></p> 
<p></p> </li> 
<li> 
<p> Ries LAG, Harkins D, Krapcho M, Mariotto A, Feuer EJ, Clegg L, Eisner MP, 
Horner MJ, Howlader N, Hayat M, Hankey BF, Edwards BK:<strong> SEER Cancer 
Statistics Review, 1975-2004, National Cancer Institute. Bethesda, MD.</strong>[
http://seer.cancer.gov/csr/1975_2004/] webcite</p>
<p> 2006.</p> 
<p></p> 
<p></p> </li> 
<li> 
<p> Henschke CI, Yankelevitz DF, Libby DM, Pasmantier MW, Smith JP, Miettinen 
OS:<strong> Survival of patients with stage I lung cancer detected on CT 
screening.</strong></p>
<p><em>N Engl J Med</em> 2006, <strong>355</strong><strong>:</strong>1763-1771.
PubMed&nbsp;Abstract | Publisher&nbsp;Full&nbsp;Text </p> 
<p></p> 
<p></p> </li> 
<li> 
<p> Bach PB, Jett JR, Pastorino U, Tockman MS, Swensen SJ, Begg CB: <strong> 
Computed tomography screening and lung cancer outcomes.</strong></p>
<p><em>Jama</em> 2007, <strong>297</strong><strong>:</strong>953-961. 
PubMed&nbsp;Abstract | Publisher&nbsp;Full&nbsp;Text </p> 
<p></p> 
<p></p> </li> 
<li> 
<p> Belinsky SA: <strong> Gene-promoter hypermethylation as a biomarker in 
lung cancer.</strong></p>
<p><em>Nature Reviews Cancer</em> 2004, <strong>4</strong><strong>:</strong>
1-11.Publisher&nbsp;Full&nbsp;Text </p> 
<p></p> 
<p></p> </li> 
<li> 
<p> Laird PW: <strong> The power and the promise of DNA methylation markers. 
</strong></p>
<p><em>Nature Reviews Cancer</em> 2003, <strong>3</strong><strong>:</strong>
253-266.PubMed&nbsp;Abstract | Publisher&nbsp;Full&nbsp;Text </p> 
<p></p> 
<p></p> </li> 
<li> 
<p> Divine KK, Pulling LC, Marron-Terada PG, Liechty KC, Kang T, Schwartz AG, 
Bocklage TJ, Coons TA, Gilliland FD, Belinsky SA:<strong> Multiplicity of 
abnormal promoter methylation in lung adenocarcinomas from smokers and never 
smokers.</strong></p>
<p><em>Int J Cancer</em> 2005, <strong>114</strong><strong>:</strong>400-405. 
PubMed&nbsp;Abstract | Publisher&nbsp;Full&nbsp;Text </p> 
<p></p> 
<p></p> </li> 
<li> 
<p> Eads CA, Danenberg KD, Kawakami K, Saltz LB, Blake C, Shibata D, Danenberg 
PV, Laird PW:<strong> MethyLight: a high-throughput assay to measure DNA 
methylation.</strong></p>
<p><em>Nucleic Acids Res</em> 2000, <strong>28</strong><strong>:</strong>E32. 
PubMed&nbsp;Abstract | Publisher&nbsp;Full&nbsp;Text | 
PubMed&nbsp;Central&nbsp;Full&nbsp;Text </p> 
<p></p> 
<p></p> </li> 
<li> 
<p> Weisenberger DJ, Siegmund KD, Campan M, Young J, Long TI, Faasse MA, Kang 
GH, Widschwendter M, Weener D, Buchanan D, Koh H, Simms L, Barker M, Leggett B, 
Levine J, Kim M, French AJ, Thibodeau SN, Jass J, Haile R, Laird PW:<strong> 
CpG island methylator phenotype underlies sporadic microsatellite instability 
and is tightly associated with BRAF mutation in colorectal cancer.</strong></p>
<p><em>Nat Genet</em> 2006, <strong>38</strong><strong>:</strong>787-793. 
PubMed&nbsp;Abstract | Publisher&nbsp;Full&nbsp;Text </p> 
<p></p> 
<p></p> </li> 
<li> 
<p> Benjamini Y, Hochberg Y: <strong> Controlling the false discovery rate: a 
practical and powerful approach to multiple testing.</strong></p>
<p><em>J R Statist Soc B</em> 1995, <strong>57</strong><strong>:</strong>
289-300.</p> 
<p></p> 
<p></p> </li> 
<li> 
<p> Waki T, Tamura G, Sato M, Motoyama T: <strong> Age-related methylation of 
tumor suppressor and tumor-related genes: an analysis of autopsy samples.
</strong></p>
<p><em>Oncogene</em> 2003, <strong>22</strong><strong>:</strong>4128-4133. 
PubMed&nbsp;Abstract | Publisher&nbsp;Full&nbsp;Text </p> 
<p></p> 
<p></p> </li> 
<li> 
<p> Merlo A, Herman JG, Mao L, Lee DJ, Gabrielson E, Burger PC, Baylin SB, 
Sydranski D:<strong> 5'CpG island methylation is associated with 
transcriptional silencing of the tumor suppressor p16/CDKN2/MTS1 in human 
cancers.</strong></p>
<p><em>Nat Med</em> 1995, <strong>1</strong><strong>:</strong>686-692. 
PubMed&nbsp;Abstract | Publisher&nbsp;Full&nbsp;Text </p> 
<p></p> 
<p></p> </li> 
<li> 
<p> Palmisano WA, Divine KK, Saccomanno G, Gilliland FD, Baylin SB, Herman JG, 
Belinsky SA:<strong> Predicting lung cancer by detecting aberrant promoter 
methylation in sputum.</strong></p>
<p><em>Cancer Res</em> 2000, <strong>60</strong><strong>:</strong>5954-5958. 
PubMed&nbsp;Abstract | Publisher&nbsp;Full&nbsp;Text </p> 
<p></p> 
<p></p> </li> 
<li> 
<p> Destro A, Bianchi P, Alloisio M, Laghi L, Di Gioia S, Malesci A, Cariboni 
U, Gribaudi G, Bulfamante G, Marchetti A, Bosari S, Infante M, Ravasi G, 
Roncalli M:<strong> K-ras and p16(INK4A)alterations in sputum of NSCLC patients 
and in heavy asymptomatic chronic smokers.</strong></p>
<p><em>Lung Cancer</em> 2004, <strong>44</strong><strong>:</strong>23-32. 
PubMed&nbsp;Abstract | Publisher&nbsp;Full&nbsp;Text </p> 
<p></p> 
<p></p> </li> 
<li> 
<p> Belinsky SA, Liechty KC, Gentry FD, Wolf HJ, Rogers J, Vu K, Haney J, 
Kennedy TC, Hirsch FR, Miller Y, Franklin WA, Herman JG, Baylin SB, Bunn PA, 
Byers T:<strong> Promoter hypermethylation of multiple genes in sputum precedes 
lung cancer incidence in a high-risk cohort.</strong></p>
<p><em>Cancer Res</em> 2006, <strong>66</strong><strong>:</strong>3338-3344. 
PubMed&nbsp;Abstract | Publisher&nbsp;Full&nbsp;Text </p> 
<p></p> 
<p></p> </li> 
<li> 
<p> Nguyen C, Liang G, Nguyen TT, Tsao-Wei D, Groshen S, Lubbert M, Zhou JH, 
Benedict WF, Jones PA:<strong> Susceptibility of non-promoter CpG islands to de 
novo methylation in normal and neoplastic cells.</strong></p>
<p><em>J Natl Cancer Inst</em> 2001, <strong>93</strong><strong>:</strong>
1465-1472.PubMed&nbsp;Abstract | Publisher&nbsp;Full&nbsp;Text </p> 
<p></p> 
<p></p> </li> 
<li> 
<p> Salem C, Liang G, Tsai YC, Coulter J, Knowles MA, Feng AC, Groshen S, 
Nichols PW, Jones PA:<strong> Progressive increases in de novo methylation of 
CpG islands in bladder cancer.</strong></p>
<p><em>Cancer Res</em> 2000, <strong>60</strong><strong>:</strong>2473-2476. 
PubMed&nbsp;Abstract | Publisher&nbsp;Full&nbsp;Text </p> 
<p></p> 
<p></p> </li> 
<li> 
<p> Liu Y, An Q, Li L, Zhang D, Huang J, Feng X, Cheng S, Gao Y: <strong> 
Hypermethylation of p16INK4a in Chinese lung cancer patients: biological and 
clinical implications.</strong></p>
<p><em>Carcinogenesis</em> 2003, <strong>24</strong><strong>:</strong>
1897-1901.PubMed&nbsp;Abstract | Publisher&nbsp;Full&nbsp;Text </p> 
<p></p> 
<p></p> </li> 
<li> 
<p> Toyooka S, Suzuki M, Maruyama R, Toyooka KO, Tsukuda K, Fukuyama Y, Lizasa 
T, Aoe M, Date H, Fujisawa T, Shimizu N, Gazdar AF:<strong> The relationship 
between aberrant methylation and survival in non-small cell lung cancers.
</strong></p>
<p><em>Br J Cancer</em> 2004, <strong>91</strong>(4)<strong>:</strong>771-774. 
PubMed&nbsp;Abstract | Publisher&nbsp;Full&nbsp;Text </p> 
<p></p> 
<p></p> </li> 
<li> 
<p> Wang J, Lee JJ, Wang L, Liu DD, Lu C, Fan YH, Hong WK, Mao L: <strong> 
Value of p16ink4a and RASSF1A promoter hypermethylation in prognosis of 
patients with resectable non-small cell lung cancer.</strong></p>
<p><em>Clin Cancer Res</em> 2004, <strong>10</strong><strong>:</strong>
6119-6125.PubMed&nbsp;Abstract | Publisher&nbsp;Full&nbsp;Text </p> 
<p></p> 
<p></p> </li> 
<li> 
<p> Shiraishi M, Sekiguchi A, Oates AJ, Terry MJ, Miyamoto Y: <strong> HOX 
gene clusters are hotspots of de novo methylation in CpG islands of human lung 
adenocarcinomas.</strong></p>
<p><em>Oncogene</em> 2002, <strong>21</strong><strong>:</strong>3659-3662. 
PubMed&nbsp;Abstract | Publisher&nbsp;Full&nbsp;Text </p> 
<p></p> 
<p></p> </li> 
<li> 
<p> Rauch T, Wang Z, Zhang X, Zhong X, Wu X, Lau SK, Kernstine KH, Riggs AD, 
Pfeifer GP:<strong> Homeobox gene methylation in lung cancer studied by 
genome-wide analysis with a microarray-based methylated CpG island recovery 
assay.</strong></p>
<p><em>Proc Natl Acad Sci U S A</em> 2007, <strong>104</strong><strong>:
</strong>5527-5532. PubMed&nbsp;Abstract | Publisher&nbsp;Full&nbsp;Text | 
PubMed&nbsp;Central&nbsp;Full&nbsp;Text </p> 
<p></p> 
<p></p> </li> 
<li> 
<p> Sellar GC, Watt KP, Rabiasz GJ, Stronach EA, Li L, Miller EP, Massie CE, 
Miller J, Contreras-Moreira B, Scott D, Brown I, Williams AR, Bates PA, Smyth 
JF, Gabra H:<strong> OPCML at 11q25 is epigenetically inactivated and has 
tumor-suppressor function in epithelial ovarian cancer.</strong></p>
<p><em>Nature Genet</em> 2003, <strong>34</strong><strong>:</strong>337-343. 
PubMed&nbsp;Abstract | Publisher&nbsp;Full&nbsp;Text </p> 
<p></p> 
<p></p> </li> 
<li> 
<p> Maneckjee R, Minna JD: <strong> Opioid and nicotine receptors affect 
growth regulation of human lung cancer cell lines.</strong></p>
<p><em>Proc Natl Acad Sci U S A</em> 1990, <strong>87</strong><strong>:
</strong>3294-3298. PubMed&nbsp;Abstract | Publisher&nbsp;Full&nbsp;Text | 
PubMed&nbsp;Central&nbsp;Full&nbsp;Text </p> 
<p></p> 
<p></p> </li> 
<li> 
<p> Maneckjee R, Minna JD: <strong> Nonconventional opioid binding sites 
mediate growth inhibitory effects of methadone on human lung cancer cells.
</strong></p>
<p><em>Proc Natl Acad Sci U S A</em> 1992, <strong>89</strong><strong>:
</strong>1169-1173. PubMed&nbsp;Abstract | Publisher&nbsp;Full&nbsp;Text | 
PubMed&nbsp;Central&nbsp;Full&nbsp;Text </p> 
<p></p> 
<p></p> </li> 
<li> 
<p> Maneckjee R, Minna JD: <strong> Opioids induce while nicotine suppresses 
apoptosis in human lung cancer cells.</strong></p>
<p><em>Cell Growth Differ</em> 1994, <strong>5</strong><strong>:</strong>
1033-1040.PubMed&nbsp;Abstract | Publisher&nbsp;Full&nbsp;Text </p> 
<p></p> 
<p></p> </li> 
<li> 
<p> Bibikova M, Lin Z, Zhou L, Chudin E, Garcia EW, Wu B, Doucet D, Thomas NJ, 
Wang Y, Vollmer E, Goldmann T, Seifart C, Jiang W, Barker DL, Chee MS, Floros 
J, Fan JB:<strong> High-throughput DNA methylation profiling using universal 
bead arrays.</strong></p>
<p><em>Genome Res</em> 2006, <strong>16</strong><strong>:</strong>383-393. 
PubMed&nbsp;Abstract | Publisher&nbsp;Full&nbsp;Text | 
PubMed&nbsp;Central&nbsp;Full&nbsp;Text </p> 
<p></p> 
<p></p> </li> 
<li> 
<p> Guo M, House MG, Suzuki H, Ye Y, Brock MV, Lu F, Liu Z, Rustgi AK, Herman 
JG:<strong> Epigenetic silencing of CDX2 is a feature of squamous esophageal 
cancer.</strong></p>
<p><em>Int J Cancer</em> 2007, <strong>121</strong><strong>:</strong>1219-1226.
PubMed&nbsp;Abstract | Publisher&nbsp;Full&nbsp;Text </p> 
<p></p> 
<p></p> </li> 
<li> 
<p> Kawai H, Tomii K, Toyooka S, Yano M, Murakami M, Tsukuda K, Shimizu N: 
<strong> Promoter methylation downregulates CDX2 expression in colorectal 
carcinomas.</strong></p>
<p><em>Oncol Rep</em> 2005, <strong>13</strong><strong>:</strong>547-551. 
PubMed&nbsp;Abstract </p> 
<p></p> 
<p></p> </li> 
<li> 
<p> Laird PW, Zijderveld A, Linders K, Rudnicki MA, Jaenisch R, Berns A: 
<strong> Simplified mammalian DNA isolation procedure. </strong></p>
<p><em>Nucleic Acids Res</em> 1991, <strong>19</strong><strong>:</strong>4293. 
PubMed&nbsp;Abstract | Publisher&nbsp;Full&nbsp;Text | 
PubMed&nbsp;Central&nbsp;Full&nbsp;Text </p> 
<p></p> 
<p></p> </li> 
<li> 
<p> Tsou JA, Galler JS, Wali A, Ye W, Siegmund KD, Groshen S, Laird PW, Turla 
S, Koss MN, Pass HI, Laird-Offringa IA:<strong> DNA methylation profile of 28 
potential marker loci in malignant mesothelioma.</strong></p>
<p><em>Lung Cancer</em> 2007, <strong>58</strong><strong>:</strong>220-30. 
PubMed&nbsp;Abstract | Publisher&nbsp;Full&nbsp;Text </p> 
<p></p> 
<p></p> </li> 
<li> 
<p> Virmani AK, Tsou JA, Siegmund KD, Shen L, Long TI, Laird PW, Gazdar AF, 
Laird-Offringa IA:<strong> Hierarchical clustering of lung cancer cell lines 
using DNA methylation markers.</strong></p>
<p><em>Cancer Epidemiol Biomarkers Prev</em> 2002, <strong>11</strong><strong>:
</strong>291-297. PubMed&nbsp;Abstract | Publisher&nbsp;Full&nbsp;Text </p> 
<p></p> 
<p></p> </li> 
<li> 
<p> Weisenberger DJ, Campan M, Long TI, Kim M, Woods C, Fiala E, Ehrlich M, 
Laird PW:<strong> Analysis of Repetitive Element DNA Methylation by MethyLight. 
</strong></p>
<p><em>Nucleic Acids Res</em> 2005, <strong>33</strong><strong>:</strong>
6823-6836.PubMed&nbsp;Abstract | Publisher&nbsp;Full&nbsp;Text | 
PubMed&nbsp;Central&nbsp;Full&nbsp;Text </p> 
<p></p> 
<p></p> </li> 
<li> 
<p> Ihaka R, Gentleman R: <strong> R: a language for data analysis and 
graphics.</strong></p>
<p><em>J Comput Graph Statist</em> 1996, <strong>5</strong><strong>:</strong>
299-314.Publisher&nbsp;Full&nbsp;Text </p> 
<p></p> 
<p></p> </li> </ol> 
<dl> 
<dt> Advertisement </dt> 
<dd> </dd> </dl> 
<dl> 
<dt> Advertisement </dt> 
<dd> </dd> </dl> <br>

<ul> 
<li>Terms and Conditions</li> 
<li>Privacy statement</li> 
<li>Press</li> 
<li>Information for advertisers</li> 
<li>Jobs at BMC</li> 
<li>Support</li> 
<li>Contact us</li> </ul> 
<p>&copy; 2012 BioMed Central Ltd unless otherwise stated. Part of Springer 
Science+Business Media.</p> <br>
<br>

</body>