<!doctype html>
<meta charset="utf-8">
<title>prot references</title>
<body>

<h1>prot references</h1>  [Bernstein1977Protein]  F.&nbsp;C. Bernstein, 
T.&nbsp;F. Koetzle, G.&nbsp;J. Williams, E.&nbsp;F. Meyer, M.&nbsp;D. Brice, 
J.&nbsp;R. Rodgers, O.&nbsp;Kennard, T.&nbsp;Shimanouchi, and M.&nbsp;Tasumi. 
The protein data bank: a computer-based archival file for macromolecular 
structures.<em>J. Mol. Biol.</em>, 112(3):535-542, May 1977. [&nbsp;bib&nbsp;] 
<blockquote>  Keywords: Computers; Great Britain; Information Systems; Japan; 
Protein Conformation; Proteins; United States </blockquote> <br>
 [
Pepperrell1991Techniques]  C.&nbsp;A. Pepperrell and P.&nbsp;Willett. 
Techniques for the calculation of three-dimensional structural similarity using 
inter-atomic distances.<em>J Comput Aided Mol Des</em>, 5(5):455-474, Oct 1991. 
[&nbsp;bib&nbsp;] 
<blockquote>  This paper reports a comparison of several methods for measuring 
the degree of similarity between pairs of 3-D chemical structures that are 
represented by inter-atomic distance matrices. The methods that have been 
tested use the distance information in very different ways and have very 
different computational requirements. Experiments with 10 small datasets, for 
which both structural and biological activity data are available, suggest that 
the most cost-effective technique is based on a mapping procedure that tries to 
match pairs of atoms, one from each of the molecules that are being compared, 
that have neighbouring atoms at approximately the same distances. </blockquote> 
<p></p> 
<blockquote>  Keywords: Algorithms, Binding Sites, Chemical, Chemistry, 
Comparative Study, Computer Simulation, Databases, Factual, Macromolecular 
Substances, Models, Molecular Conformation, Molecular Structure, Non-U.S. 
Gov't, Physical, Protein Conformation, Protein Structure, Proteins, Research 
Support, Structure-Activity Relationship, Tertiary, 1770381 </blockquote> <br>
 
[Russell1992Multiple]  R.&nbsp;B. Russell and G.&nbsp;J. Barton. Multiple 
protein sequence alignment from tertiary structure comparison: assignment of 
global and residue confidence levels.<em>Proteins</em>, 14(2):309-323, Oct 
1992. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;] 
<blockquote>  An algorithm is presented for the accurate and rapid generation 
of multiple protein sequence alignments from tertiary structure comparisons. A 
preliminary multiple sequence alignment is performed using sequence 
information, which then determines an initial superposition of the structures. 
A structure comparison algorithm is applied to all pairs of proteins in the 
superimposed set and a similarity tree calculated. Multiple sequence alignments 
are then generated by following the tree from the branches to the root. At each 
branchpoint of the tree, a structure-based sequence alignment and coordinate 
transformations are output, with the multiple alignment of all structures 
output at the root. The algorithm encoded in STAMP (STructural Alignment of 
Multiple Proteins) is shown to give alignments in good agreement with published 
structural accounts within the dehydrogenase fold domains, globins, and serine 
proteinases. In order to reduce the need for visual verification, two 
similarity indices are introduced to determine the quality of each generated 
structural alignment. Sc quantifies the global structural similarity between 
pairs or groups of proteins, whereas Pij' provides a normalized measure of the 
confidence in the alignment of each residue. STAMP alignments have the quality 
of each alignment characterized by Sc and Pij' values and thus provide a 
reproducible resource for studies of residue conservation within structural 
motifs. </blockquote> 
<p></p> 
<blockquote>  Keywords: Algorithms; Amino Acid Sequence; Animals; Confidence 
Intervals; Globins; Humans; Molecular Sequence Data; Protein Structure, 
Tertiary; Sequence Alignment; Sequence Homology, Amino Acid; Serine 
Endopeptidases; Software </blockquote> <br>
 [Henikoff1992Amino]  
S.&nbsp;Henikoff and J.&nbsp;G. Henikoff. Amino acid substitution matrices from 
protein blocks.<em>Proc. Natl. Acad. Sci. USA</em>, 89(22):10915-10919, Nov 
1992. [&nbsp;bib&nbsp;] 
<blockquote>  Methods for alignment of protein sequences typically measure 
similarity by using a substitution matrix with scores for all possible 
exchanges of one amino acid with another. The most widely used matrices are 
based on the Dayhoff model of evolutionary rates. Using a different approach, 
we have derived substitution matrices from about 2000 blocks of aligned 
sequence segments characterizing more than 500 groups of related proteins. This 
led to marked improvements in alignments and in searches using queries from 
each of the groups. </blockquote> 
<p></p> 
<blockquote>  Keywords: Algorithms; Amino Acid Sequence; Animals; 
Caenorhabditis elegans; Drosophila; Lod Score; Mathematics; Molecular Sequence 
Data; Probability; Proteins; Sequence Homology, Amino Acid; Software 
</blockquote> <br>
 [Sette1994relationship]  A.&nbsp;Sette, A.&nbsp;Vitiello, 
B.&nbsp;Reherman, P.&nbsp;Fowler, R.&nbsp;Nayersina, W.&nbsp;M. Kast, 
C.&nbsp;J. Melief, C.&nbsp;Oseroff, L.&nbsp;Yuan, J.&nbsp;Ruppert, 
J.&nbsp;Sidney, M.&nbsp;F. del Guercio, S.&nbsp;Southwood, R.&nbsp;T. Kubo, 
R.&nbsp;W. Chesnut, H.&nbsp;M. Grey, and F.&nbsp;V. Chisari. The relationship 
between class i binding affinity and immunogenicity of potential cytotoxic t 
cell epitopes.<em>J. Immunol.</em>, 153(12):5586-5592, Dec 1994. [&nbsp;bib
&nbsp;]
<blockquote>  The relationship between binding affinity for HLA class I 
molecules and immunogenicity of discrete peptide epitopes has been analyzed in 
two different experimental approaches. In the first approach, the 
immunogenicity of potential epitopes ranging in MHC binding affinity over a 
10,000-fold range was analyzed in HLA-A*0201 transgenic mice. In the second 
approach, the antigenicity of approximately 100 different hepatitis B virus 
(HBV)-derived potential epitopes, all carrying A*0201 binding motifs, was 
assessed by using PBL of acute hepatitis patients. In both cases, it was found 
that an affinity threshold of approximately 500 nM (preferably 50 nM or less) 
apparently determines the capacity of a peptide epitope to elicit a CTL 
response. These data correlate well with class I binding affinity measurements 
of either naturally processed peptides or previously described T cell epitopes. 
Taken together, these data have important implications for the selection of 
epitopes for peptide-based vaccines, and also formally demonstrate the crucial 
role of determinant selection in the shaping of T cell responses. Because in 
most (but not all) cases, high affinity peptides seem to be immunogenic, our 
data also suggest that holes in the functional T cell repertoire, if they 
exist, may be relatively rare. </blockquote> 
<p></p> 
<blockquote>  Keywords: Amino Acid Sequence; Animals; Cell Line; Cytotoxicity 
Tests, Immunologic; Epitopes; HLA-A Antigens; Hepatitis B; Hepatitis B 
Antigens; Humans; Mice; Mice, Transgenic; Molecular Sequence Data; Peptides; 
Protein Binding; T-Lymphocytes, Cytotoxic </blockquote> <br>
 [Sidney1995Several
]  J.&nbsp;Sidney, M.&nbsp;F. del Guercio, S.&nbsp;Southwood, V.&nbsp;H. 
Engelhard, E.&nbsp;Appella, H.&nbsp;G. Rammensee, K.&nbsp;Falk, 
O.&nbsp;R&ouml;tzschke, M.&nbsp;Takiguchi, and R.&nbsp;T. Kubo. Several HLA 
alleles share overlapping peptide specificities.<em>J. Immunol.</em>, 
154(1):247-259, Jan 1995. [&nbsp;bib&nbsp;] 
<blockquote>  Herein we describe the establishment of assays to measure 
peptide binding to purified HLA-B*0701, -B*0801, -B*2705, -B*3501-03, -B*5401, 
-Cw*0401, -Cw*0602, and -Cw*0702 molecules. The binding of known peptide 
epitopes or naturally processed peptides correlates well with HLA restriction 
or origin, underscoring the immunologic relevance of these assays. Analysis of 
the sequences of various HLA class I alleles suggested that alleles with 
peptide motifs characterized by proline in position 2 and aromatic or 
hydrophobic residues at their C-terminus shared key consensus residues at 
positions 9, 63, 66, 67, and 70 (B pocket) and residue 116 (F pocket). 
Prediction of the peptide-binding specificity of HLA-B*5401, on the basis of 
this consensus B and F pocket structure, verified this hypothesis and suggested 
that a relatively large family of HLA-B alleles (which we have defined as the 
HLA-B7-like supertype) may significantly overlap in peptide binding 
specificity. Availability of quantitative binding assays allowed verification 
that, indeed, many (25%) of the peptide ligands carrying proline in position 2 
and hydrophobic/aromatic residues at the C-terminus (the B7-like supermotif) 
were capable of binding at least three of five HLA-B7-like supertype alleles. 
Identification of epitopes carrying the B7-like supermotif and binding to a 
family of alleles represented in over 40% of individuals from all major ethnic 
groups may be of considerable use in the design of peptide vaccines. 
</blockquote> 
<p></p> 
<blockquote>  Keywords: Alleles; Amino Acid Sequence; Cell Line, Transformed; 
Consensus Sequence; Epitopes; Genes, MHC Class I; HLA-B Antigens; HLA-C 
Antigens; Humans; Molecular Sequence Data; Peptide Fragments; Protein Binding; 
Protein Structure, Tertiary; Structure-Activity Relationship; Substrate 
Specificity </blockquote> <br>
 [Wilkins1996From]  M.&nbsp;R. Wilkins, 
C.&nbsp;Pasquali, R.&nbsp;D. Appel, K.&nbsp;Ou, O.&nbsp;Golaz, J.&nbsp;C. 
Sanchez, J.&nbsp;X. Yan, A.&nbsp;A. Gooley, G.&nbsp;Hughes, 
I.&nbsp;Humphery-Smith, K.&nbsp;L. Williams, and D.&nbsp;F. Hochstrasser. From 
proteins to proteomes: large scale protein identification by two-dimensional 
electrophoresis and amino acid analysis.<em>Biotechnology (N Y)</em>, 
14(1):61-65, Jan 1996. [&nbsp;bib&nbsp;] 
<blockquote>  Separation and identification of proteins by two-dimensional 
(2-D) electrophoresis can be used for protein-based gene expression analysis. 
In this report single protein spots, from polyvinylidene difluoride blots of 
micropreparative E. coli 2-D gels, were rapidly and economically identified by 
matching their amino acid composition, estimated pI and molecular weight 
against all E. coli entries in the SWISS-PROT database. Thirty proteins from an 
E. coli 2-D map were analyzed and identities assigned. Three of the proteins 
were unknown. By protein sequencing analysis, 20 of the 27 proteins were 
correctly identified. Importantly, correct identifications showed unambiguous 
&quot;correct&quot; score patterns. While incorrect protein identifications 
also showed distinctive score patterns, indicating that protein must be 
identified by other means. These techniques allow large-scale screening of the 
protein complement of simple organisms, or tissues in normal and disease 
states. The computer program described here is accessible via the World Wide 
Web at URL address (http:@expasy.hcuge.ch/). </blockquote> 
<p></p> 
<blockquote>  Keywords: Amino Acids; Bacterial Proteins; Blood Proteins; 
Databases, Factual; Electrophoresis, Gel, Two-Dimensional; Escherichia coli; 
Humans; Microchemistry; Molecular Weight; Multienzyme Complexes; Proteins; 
Reproducibility of Results; Software; Time Factors </blockquote> <br>
 [
Sidney1996Definition]  J.&nbsp;Sidney, H.&nbsp;M. Grey, S.&nbsp;Southwood, 
E.&nbsp;Celis, P.&nbsp;A. Wentworth, M.&nbsp;F. del Guercio, R.&nbsp;T. Kubo, 
R.&nbsp;W. Chesnut, and A.&nbsp;Sette. Definition of an HLA-A3-like supermotif 
demonstrates the overlapping peptide-binding repertoires of common HLA 
molecules.<em>Hum Immunol</em>, 45(2):79-93, Feb 1996. [&nbsp;bib&nbsp;] 
<blockquote>  An HLA-A3-like supertype (minimally comprised of products from 
the HLA class I alleles A3, A11, A31, A*3301, and A*6801) has been defined on 
the basis of (a) structural similarities in the antigen-binding groove, (b) 
shared main anchor peptide-binding motifs, (c) the identification of peptides 
cross-reacting with most or all of these molecules, and (d) the definition of 
an A3-like supermotif that efficiently predicts highly cross-reactive peptides. 
Detailed secondary anchor maps for A3, A11, A31, A*3301, and A*6801 are also 
described. The biologic relevance of the A3-like supertype is indicated by the 
fact that high frequencies of the A3-like supertype alleles are conserved in 
all major ethnic groups. Because A3-like supertype alleles are found in most 
major HLA evolutionary lineages, possibly a reflection of common ancestry, the 
A3-like supermotif might in fact represent a primeval human HLA class I 
peptide-binding specificity. It is also possible that these phenomena might be 
related to optimal exploitation of the peptide specificity by human TAP 
molecules. The grouping of HLA alleles into supertypes on the basis of their 
overlapping peptide-binding repertoires represents an alternative to serologic 
or phylogenetic classification. </blockquote> 
<p></p> 
<blockquote>  Keywords: Alleles; Amino Acid Sequence; Cell Line, Transformed; 
Cross Reactions; HLA Antigens; HLA-A3 Antigen; HLA-B Antigens; Haplotypes; 
Humans; Molecular Sequence Data; Peptide Fragments; Protein Binding; 
Structure-Activity Relationship </blockquote> <br>
 [Rarey1996Placement]  
M.&nbsp;Rarey, S.&nbsp;Wefing, and T.&nbsp;Lengauer. Placement of medium-sized 
molecular fragments into active sites of proteins.<em>J Comput Aided Mol Des
</em>, 10(1):41-54, Feb 1996. [&nbsp;bib&nbsp;] 
<blockquote>  We present an algorithm for placing molecular fragments into the 
active site of a receptor. A molecular fragment is defined as a connected part 
of a molecule containing only complete ring systems. The algorithm is part of a 
docking tool, called FLEXX, which is currently under development at GMD. The 
overall goal is to provide means of automatically computing low-energy 
conformations of the ligand within the active site, with an accuracy 
approaching the limitations of experimental methods for resolving molecular 
structures and within a run time that allows for docking large sets of ligands. 
The methods by which we plan to achieve this goal are the explicit exploitation 
of molecular flexibility of the ligand and the incorporation of physicochemical 
properties of the molecules. The algorithm for fragment placement, which is the 
topic of this paper, is based on pattern recognition techniques and is able to 
predict a small set of possible positions of a molecular fragment with low 
flexibility within seconds on a workstation. In most cases, a placement with 
rms deviation below 1.0 A with respect to the X-ray structure is found among 
the 10 highest ranking solutions, assuming that the receptor is given in the 
bound conformation. </blockquote> 
<p></p> 
<blockquote>  Keywords: Algorithms; Binding Sites; Databases, Factual; 
Ligands; Models, Chemical; Peptide Fragments, chemistry; Proteins, chemistry; 
Software </blockquote> <br>
 [Rarey1996fast]  M.&nbsp;Rarey, B.&nbsp;Kramer, 
T.&nbsp;Lengauer, and G.&nbsp;Klebe. A fast flexible docking method using an 
incremental construction algorithm.<em>J. Mol. Biol.</em>, 261(3):470-489, Aug 
1996. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;] 
<blockquote>  We present an automatic method for docking organic ligands into 
protein binding sites. The method can be used in the design process of specific 
protein ligands. It combines an appropriate model of the physico-chemical 
properties of the docked molecules with efficient methods for sampling the 
conformational space of the ligand. If the ligand is flexible, it can adopt a 
large variety of different conformations. Each such minimum in conformational 
space presents a potential candidate for the conformation of the ligand in the 
complexed state. Our docking method samples the conformation space of the 
ligand on the basis of a discrete model and uses a tree-search technique for 
placing the ligand incrementally into the active site. For placing the first 
fragment of the ligand into the protein, we use hashing techniques adapted from 
computer vision. The incremental construction algorithm is based on a greedy 
strategy combined with efficient methods for overlap detection and for the 
search of new interactions. We present results on 19 complexes of which the 
binding geometry has been crystallographically determined. All considered 
ligands are docked in at most three minutes on a current workstation. The 
experimentally observed binding mode of the ligand is reproduced with 0.5 to 
1.2 A rms deviation. It is almost always found among the highest-ranking 
conformations computed. </blockquote> 
<p></p> 
<blockquote>  Keywords: Aldehyde Reductase, Algorithms, Amiloride, 
Aminoimidazole Carboxamide, Animals, Arabinose, Automation, Binding Sites, 
Carbonic Anhydrases, Computational Biology, Computer Simulation, Concanavalin 
A, Crystallography, Databases, Drug Design, Drug Evaluation, Enzyme Inhibitors, 
Factual, Folic Acid, Folic Acid Antagonists, Fructose-Bisphosphatase, Humans, 
Internet, Ligands, Methotrexate, Models, Molecular, Non-U.S. Gov't, Pancreatic 
Elastase, Pentamidine, Pliability, Point Mutation, Preclinical, Protein 
Binding, Protein Conformation, Proteins, Reproducibility of Results, Research 
Support, Ribonucleosides, Software, Tetrahydrofolate Dehydrogenase, 
Thermolysin, Time Factors, Trypsin, X-Ray, 8780787 </blockquote> <br>
 [
Kristiansen1996database]  K.&nbsp;Kristiansen, S.&nbsp;G. Dahl, and 
O.&nbsp;Edvardsen. A database of mutants and effects of site-directed 
mutagenesis experiments on G protein-coupled receptors.<em>Proteins</em>, 
26(1):81-94, Sep 1996. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;] 
<blockquote>  A database system and computer programs for storage and 
retrieval of information about guanine nucleotide-binding protein (G protein) 
-coupled receptor mutants and associated biological effects have been 
developed. Mutation data on the receptors were collected from the literature 
and a database of mutants and effects of mutations was developed. The G 
protein-coupled receptor, family A, point mutation database (GRAP) provides 
detailed information on ligand-binding and signal transduction properties of 
more than 2130 receptor mutants. The amino acid sequences of receptors for 
which mutation experiments have been reported were aligned, and from this 
alignment mutation data may be retrieved. Alternatively, a search form allowing 
detailed specification of which mutants to retrieve may be used, for example, 
to search for specific amino acid substitutions, substitutions in specific 
protein domains or reported biological effects. Furthermore, ligand and 
bibliographic oriented queries may be performed. GRAP is available on the 
Internet (URL: http://www-grap.fagmed.uit.no/GRAP/+ +homepage.html) using the 
World-Wide Web system. </blockquote> 
<p></p> 
<blockquote>  Keywords: Amino Acid Sequence; Computer Communication Networks; 
Computers; GTP-Binding Proteins; Information Systems; Molecular Sequence Data; 
Mutagenesis, Site-Directed; Mutation; Receptors, Cell Surface; Sequence 
Alignment </blockquote> <br>
 [Humphrey1996VMD]  W.&nbsp;Humphrey, 
A.&nbsp;Dalke, and K.&nbsp;Schulten. VMD: visual molecular dynamics.<em>J. Mol. 
Graph.</em>, 14(1):33-8, 27-8, Feb 1996. [&nbsp;bib&nbsp;] 
<blockquote>  VMD is a molecular graphics program designed for the display and 
analysis of molecular assemblies, in particular biopolymers such as proteins 
and nucleic acids. VMD can simultaneously display any number of structures 
using a wide variety of rendering styles and coloring methods. Molecules are 
displayed as one or more &quot;representations,&quot; in which each 
representation embodies a particular rendering method and coloring scheme for a 
selected subset of atoms. The atoms displayed in each representation are chosen 
using an extensive atom selection syntax, which includes Boolean operators and 
regular expressions. VMD provides a complete graphical user interface for 
program control, as well as a text interface using the Tcl embeddable parser to 
allow for complex scripts with variable substitution, control loops, and 
function calls. Full session logging is supported, which produces a VMD command 
script for later playback. High-resolution raster images of displayed molecules 
may be produced by generating input scripts for use by a number of 
photorealistic image-rendering applications. VMD has also been expressly 
designed with the ability to animate molecular dynamics (MD) simulation 
trajectories, imported either from files or from a direct connection to a 
running MD simulation. VMD is the visualization component of MDScope, a set of 
tools for interactive problem solving in structural biology, which also 
includes the parallel MD program NAMD, and the MDCOMM software used to connect 
the visualization and simulation programs. VMD is written in C++, using an 
object-oriented design; the program, including source code and extensive 
documentation, is freely available via anonymous ftp and through the World Wide 
Web. </blockquote> 
<p></p> 
<blockquote>  Keywords: Computer Graphics; Computer Simulation; Computers; 
Models, Molecular; Nucleic Acids; Proteins; User-Computer Interface 
</blockquote> <br>
 [Nielsen1997Identification]  H.&nbsp;Nielsen, 
J.&nbsp;Engelbrecht, S.&nbsp;Brunak, and G.&nbsp;von Heijne. Identification of 
prokaryotic and eukaryotic signal peptides and prediction of their cleavage 
sites.<em>Protein Eng.</em>, 10(1):1-6, 1997. [&nbsp;bib&nbsp;| http&nbsp;| .pdf
&nbsp;] <br>
 [Jones1997Development]  G.&nbsp;Jones, P.&nbsp;Willett, 
R.&nbsp;C. Glen, A.&nbsp;R. Leach, and R.&nbsp;Taylor. Development and 
validation of a genetic algorithm for flexible docking.<em>J. Mol. Biol.</em>, 
267(3):727-748, Apr 1997. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;] 
<blockquote>  Prediction of small molecule binding modes to macromolecules of 
known three-dimensional structure is a problem of paramount importance in 
rational drug design (the &quot;docking&quot; problem). We report the 
development and validation of the program GOLD (Genetic Optimisation for Ligand 
Docking). GOLD is an automated ligand docking program that uses a genetic 
algorithm to explore the full range of ligand conformational flexibility with 
partial flexibility of the protein, and satisfies the fundamental requirement 
that the ligand must displace loosely bound water on binding. Numerous 
enhancements and modifications have been applied to the original technique 
resulting in a substantial increase in the reliability and the applicability of 
the algorithm. The advanced algorithm has been tested on a dataset of 100 
complexes extracted from the Brookhaven Protein DataBank. When used to dock the 
ligand back into the binding site, GOLD achieved a 71% success rate in 
identifying the experimental binding mode. </blockquote> 
<p></p> 
<blockquote>  Keywords: Algorithms, Binding Sites, Computer Simulation, 
Crystallography, Genetic, Humans, Ligands, Models, Molecular, NADP, Protein 
Binding, Protein Conformation, Proteins, Tetrahydrofolate Dehydrogenase, X-Ray, 
9126849 </blockquote> <br>
 [Gulukota1997Two]  K.&nbsp;Gulukota, 
J.&nbsp;Sidney, A.&nbsp;Sette, and C.&nbsp;DeLisi. Two complementary methods 
for predicting peptides binding major histocompatibility complex molecules.<em>
J. Mol. Biol.</em>, 267(5):1258-1267, Apr 1997. [&nbsp;bib&nbsp;| DOI&nbsp;| 
http&nbsp;] 
<blockquote>  Peptides that bind to major histocompatibility complex products 
(MHC) are known to exhibit certain sequence motifs which, though common, are 
neither necessary nor sufficient for binding: MHCs bind certain peptides that 
do not have the characteristic motifs and only about 30% of the peptides having 
the required motif, bind. In order to develop and test more accurate methods we 
measured the binding affinity of 463 nonamer peptides to HLA-A2.1. We describe 
two methods for predicting whether a given peptide will bind to an MHC and 
apply them to these peptides. One method is based on simulating a neural 
network and another, called the polynomial method, is based on statistical 
parameter estimation assuming independent binding of the side-chains of 
residues. We compare these methods with each other and with standard 
motif-based methods. The two methods are complementary, and both are superior 
to sequence motifs. The neural net is superior to simple motif searches in 
eliminating false positives. Its behavior can be coarsely tuned to the strength 
of binding desired and it is extendable in a straightforward fashion to other 
alleles. The polynomial method, on the other hand, has high sensitivity and is 
a superior method for eliminating false negatives. We discuss the validity of 
the independent binding assumption in such predictions. </blockquote> 
<p></p> 
<blockquote>  Keywords: Artificial Intelligence; Computing Methodologies; 
HLA-A2 Antigen; Neural Networks (Computer); Oligopeptides; Protein Binding; 
Reproducibility of Results </blockquote> <br>
 [Schneider1998Artificial]  
G.&nbsp;Schneider and P.&nbsp;Wrede. Artificial neural networks for 
computer-based molecular design.<em>Prog Biophys Mol Biol</em>, 70(3):175-222, 
1998. [&nbsp;bib&nbsp;] 
<blockquote>  The theory of artificial neural networks is briefly reviewed 
focusing on supervised and unsupervised techniques which have great impact on 
current chemical applications. An introduction to molecular descriptors and 
representation schemes is given. In addition, worked examples of recent 
advances in this field are highlighted and pioneering publications are 
discussed. Applications of several types of artificial neural networks to 
compound classification, modelling of structure-activity relationships, 
biological target identification, and feature extraction from biopolymers are 
presented and compared to other techniques. Advantages and limitations of 
neural networks for computer-aided molecular design and sequence analysis are 
discussed. </blockquote> 
<p></p> 
<blockquote>  Keywords: Algorithms, Amino Acid Sequence, Amino Acids, Animals, 
Artificial Intelligence, Automated, Bacterial, Bacterial Proteins, Bicuculline, 
Binding Sites, Biological, Biological Availability, Blood Proteins, Blood-Brain 
Barrier, Cation Transport Proteins, Cats, Cell Membrane Permeability, Chemical, 
Chemistry, Cluster Analysis, Combinatorial Chemistry Techniques, Comparative 
Study, Computational Biology, Computer Simulation, Computer Systems, 
Computer-Aided Design, Computer-Assisted, Computing Methodologies, DNA-Binding 
Proteins, Databases, Dogs, Drug Design, Electric Stimulation, Electromyography, 
Enzyme Inhibitors, Ether-A-Go-Go Potassium Channels, Excitatory Amino Acid 
Antagonists, Factual, False Positive Reactions, Forecasting, Forelimb, GABA 
Antagonists, Gene Expression Profiling, Genome, Glutamic Acid, Humans, Hydrogen 
Bonding, Image Enhancement, Image Interpretation, Image Processing, Information 
Storage and Retrieval, Iontophoresis, Kynurenic Acid, Least-Squares Analysis, 
Linear Models, Liver, Markov Chains, Metabolic Clearance Rate, 
Metalloendopeptidases, Microelectrodes, Models, Molecular, Molecular 
Conformation, Molecular Sequence Data, Molecular Structure, Motor Cortex, 
Movement, Multivariate Analysis, Nerve Net, Neural Networks (Computer), 
Neuropeptides, Non-U.S. Gov't, Nonlinear Dynamics, Pattern Recognition, 
Pharmaceutical, Pharmaceutical Preparations, Pharmacokinetics, Phylogeny, 
Potassium Channels, Predictive Value of Tests, Protein Interaction Mapping, 
Protein Sorting Signals, Protein Structure, Proteins, Rats, Reproducibility of 
Results, Research Support, Sensitivity and Specificity, Sequence Alignment, 
Sequence Analysis, Shoulder, Signal Processing, Software, Statistical, 
Stereotaxic Techniques, Structure-Activity Relationship, Terminology, Tertiary, 
Trans-Activators, Voltage-Gated, Zinc, 9830312 </blockquote> <br>
 [
Poggio1998Sparse]  Poggio and Girosi. A Sparse Representation for Function 
Approximation.<em>Neural Comput</em>, 10(6):1445-54, Jul 1998. [&nbsp;bib&nbsp;]
<blockquote>  We derive a new general representation for a function as a 
linear combination of local correlation kernels at optimal sparse locations 
(and scales) and characterize its relation to principal component analysis, 
regularization, sparsity principles, and support vector machines. </blockquote> 
<p></p> 
<blockquote>  Keywords: Algorithms, Automated, Biometry, Computers, DNA, 
Databases, Factual, Fungal, Fungal Proteins, GTP-Binding Proteins, Gene 
Expression, Genes, Learning, Markov Chains, Models, Neural Networks (Computer), 
Neurological, Non-P.H.S., Non-U.S. Gov't, Nucleic Acid Hybridization, Open 
Reading Frames, P.H.S., Pattern Recognition, Protein, Protein Structure, 
Proteins, Reproducibility of Results, Research Support, Saccharomyces 
cerevisiae, Sequence Alignment, Sequence Analysis, Software, Statistical, 
Tertiary, U.S. Gov't, 9698352 </blockquote> <br>
 [Girosi1998Equivalence]  
Girosi. An Equivalence Between Sparse Approximation and Support Vector Machines.
<em>Neural Comput</em>, 10(6):1455-80, Jul 1998. [&nbsp;bib&nbsp;] 
<blockquote>  This article shows a relationship between two different 
approximation techniques: the support vector machines (SVM), proposed by V. 
Vapnik (1995) and a sparse approximation scheme that resembles the basis 
pursuit denoising algorithm (Chen, 1995; Chen, Donoho, and Saunders, 1995). SVM 
is a technique that can be derived from the structural risk minimization 
principle (Vapnik, 1982) and can be used to estimate the parameters of several 
different approximation schemes, including radial basis functions, algebraic 
and trigonometric polynomials, B-splines, and some forms of multilayer 
perceptrons. Basis pursuit denoising is a sparse approximation technique in 
which a function is reconstructed by using a small number of basis functions 
chosen from a large set (the dictionary). We show that if the data are 
noiseless, the modified version of basis pursuit denoising proposed in this 
article is equivalent to SVM in the following sense: if applied to the same 
data set, the two techniques give the same solution, which is obtained by 
solving the same quadratic programming problem. In the appendix, we present a 
derivation of the SVM technique in one framework of regularization theory, 
rather than statistical learning theory, establishing a connection between SVM, 
sparse approximation, and regularization theory. </blockquote> 
<p></p> 
<blockquote>  Keywords: Algorithms, Automated, Biometry, Computers, DNA, 
Databases, Factual, Fungal, Fungal Proteins, GTP-Binding Proteins, Gene 
Expression, Genes, Learning, Markov Chains, Models, Neural Networks (Computer), 
Neurological, Non-P.H.S., Non-U.S. Gov't, Nucleic Acid Hybridization, Open 
Reading Frames, P.H.S., Pattern Recognition, Protein, Protein Structure, 
Proteins, Reproducibility of Results, Research Support, Saccharomyces 
cerevisiae, Sequence Alignment, Sequence Analysis, Software, Statistical, 
Tertiary, U.S. Gov't, 9698353 </blockquote> <br>
 [Pontil1998Properties]  
M.&nbsp;Pontil and A.&nbsp;Verri. Properties of support vector machines.<em>
Neural Comput</em>, 10(4):955-74, May 1998. [&nbsp;bib&nbsp;] 
<blockquote>  Support vector machines (SVMs) perform pattern recognition 
between two point classes by finding a decision surface determined by certain 
points of the training set, termed support vectors (SV). This surface, which in 
some feature space of possibly infinite dimension can be regarded as a 
hyperplane, is obtained from the solution of a problem of quadratic programming 
that depends on a regularization parameter. In this article, we study some 
mathematical properties of support vectors and show that the decision surface 
can be written as the sum of two orthogonal terms, the first depending on only 
the margin vectors (which are SVs lying on the margin), the second proportional 
to the regularization parameter. For almost all values of the parameter, this 
enables us to predict how the decision surface varies for small parameter 
changes. In the special but important case of feature space of finite dimension 
m, we also show that m + 1 SVs are usually sufficient to determine the decision 
surface fully. For relatively small m, this latter result leads to a consistent 
reduction of the SV number. </blockquote> 
<p></p> 
<blockquote>  Keywords: Algorithms, Artificial Intelligence, Automated, 
Biometry, Computers, DNA, Databases, Factual, Fungal, Fungal Proteins, 
GTP-Binding Proteins, Gene Expression, Genes, Learning, Linear Models, Markov 
Chains, Mathematics, Models, Neural Networks (Computer), Neurological, 
Non-P.H.S., Non-U.S. Gov't, Nonlinear Dynamics, Nucleic Acid Hybridization, 
Open Reading Frames, P.H.S., Pattern Recognition, Protein, Protein Structure, 
Proteins, Reproducibility of Results, Research Support, Saccharomyces 
cerevisiae, Sequence Alignment, Sequence Analysis, Software, Statistical, 
Tertiary, U.S. Gov't, 9573414 </blockquote> <br>
 [Mason1999New]  J.&nbsp;S. 
Mason, I.&nbsp;Morize, P.&nbsp;R. Menard, D.&nbsp;L. Cheney, C.&nbsp;Hulme, and 
R.&nbsp;F. Labaudiniere. New 4-point pharmacophore method for molecular 
similarity and diversity applications: overview of the method and applications, 
including a novel approach to the design of combinatorial libraries containing 
privileged substructures.<em>J. Med. Chem.</em>, 42(17):3251-3264, Aug 1999. 
[&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;] 
<blockquote>  A new 4-point pharmacophore method for molecular similarity and 
diversity that rapidly calculates all potential pharmacophores/pharmacophoric 
shapes for a molecule or a protein site is described. The method, an extension 
to the ChemDiverse/Chem-X software (Oxford Molecular, Oxford, England), has 
also been customized to enable a new internally referenced measure of 
pharmacophore diversity. The &quot;privileged&quot; substructure concept for 
the design of high-affinity ligands is presented, and an example of this new 
method is described for the design of combinatorial libraries for 
7-transmembrane G-protein-coupled receptor targets, where 
&quot;privileged&quot; substructures are used as special features to internally 
reference the pharmacophoric shapes. Up to 7 features and 15 distance ranges 
are considered, giving up to 350 million potential 4-point 3D 
pharmacophores/molecule. The resultant pharmacophore &quot;key&quot; 
(&quot;fingerprint&quot;) serves as a powerful measure for diversity or 
similarity, calculable for both a ligand and a protein site, and provides a 
consistent frame of reference for comparing molecules, sets of molecules, and 
protein sites. Explicit &quot;on-the-fly&quot; conformational sampling is 
performed for a molecule to enable the calculation of all geometries accessible 
for all combinations of four features (i.e., 4-point pharmacophores) at any 
desired sampling resolution. For a protein site, complementary site points to 
groups displayed in the site are generated and all combinations of four site 
points are considered. In this paper we report (i) the details of our 
customized implementation of the method and its modification to systematically 
measure 4-point pharmacophores relative to a &quot;special&quot; substructure 
of interest present in the molecules under study; (ii) comparisons of 3- and 
4-point pharmacophore methods, highlighting the much increased resolution of 
the 4-point method; (iii) applications of the 4-point potential pharmacophore 
descriptors as a new measure of molecular similarity and diversity and for the 
design of focused/biased combinatorial libraries. </blockquote> 
<p></p> 
<blockquote>  Keywords: Animals, Binding Sites, Chemical, Combinatorial 
Chemistry Techniques, Computational Biology, Computer Graphics, Computer 
Simulation, Databases, Drug, Drug Design, Drug Evaluation, Enzyme Inhibitors, 
Factor Xa, Factual, Humans, Indicators and Reagents, Ligands, Models, 
Molecular, Molecular Conformation, Phosphotransferases (Alcohol Group 
Acceptor), Preclinical, Protein, Protein Binding, Receptor Protein-Tyrosine 
Kinases, Receptors, Saccharomyces cerevisiae Proteins, Serine Endopeptidases, 
Software, Structure-Activity Relationship, User-Computer Interface, 10464012 
</blockquote> <br>
 [Debouck1999DNA]  C.&nbsp;Debouck and P.&nbsp;N. 
Goodfellow. DNA microarrays in drug discovery and development.<em>Nat. Genet.
</em>, 21(1 Suppl):48-50, Jan 1999. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;] 
<blockquote>  DNA microarrays can be used to measure the expression patterns 
of thousands of genes in parallel, generating clues to gene function that can 
help to identify appropriate targets for therapeutic intervention. They can 
also be used to monitor changes in gene expression in response to drug 
treatments. Here, we discuss the different ways in which microarray analysis is 
likely to affect drug discovery. </blockquote> 
<p></p> 
<blockquote>  Keywords: Agricultural, Alleles, Alternaria, Amino Acid, Amino 
Acid Chloromethyl Ketones, Amino Acid Sequence, Animal, Animals, Apoptosis, 
Asthma, Bacteria, Base Sequence, Binding Sites, Biotechnology, Blotting, Bone 
Density, Bone Matrix, Bone and Bones, CCR5, Camptothecin, Caspases, Cathepsins, 
Cell Surface, Central America, Chloroplast, Chondrocytes, Chromosome Mapping, 
Chromosomes, Cloning, Cluster Analysis, Collagen, Comparative Study, Coumarins, 
Crops, Crystallography, DNA, DNA Primers, Dipeptides, Disease, Disease Models, 
Drug Design, Drug Evaluation, Drug Industry, Enzyme Activation, Enzyme 
Inhibitors, Escherichia coli, Evolution, Exons, Expressed Sequence Tags, 
Female, Fetus, Fluorescent Dyes, Food Microbiology, Founder Effect, GTP-Binding 
Proteins, Gene Expression, Gene Frequency, Gene Library, Genes, Genetic, 
Genetic Predisposition to Disease, Genome, Geography, Growth Plate, Haplotypes, 
Hordeum, Human, Humans, Inclusion Bodies, Injections, Intraperitoneal, Introns, 
Isatin, Knockout, Male, Membrane Proteins, Messenger, Mice, Models, Molecular, 
Molecular Sequence Data, Molecular Structure, Mutation, Mycotoxins, 
Neutrophils, Non-U.S. Gov't, Northern, Oligonucleotide Array Sequence Analysis, 
Osteoarthritis, Osteochondrodysplasias, Osteoclasts, Osteopetrosis, Pair 15, 
Phaseolus, Polymorphism, Preclinical, Pregnancy, Promoter Regions (Genetics), 
Protein Precursors, Proteomics, RNA, Receptors, Recombinant Fusion Proteins, 
Recombinant Proteins, Research Support, Restriction Fragment Length, Ribosomal 
Proteins, Sequence Alignment, Sequence Analysis, Sequence Homology, South 
America, Species Specificity, Splenomegaly, Sulfonamides, Synteny, Tissue 
Distribution, Transcription, Trichothecenes, X-Ray, 9915501 </blockquote> <br>
 
[Mathews1999Expandeda]  D.&nbsp;H. Mathews, J.&nbsp;Sabina, M.&nbsp;Zuker, and 
D.&nbsp;H. Turner. Expanded sequence dependence of thermodynamic parameters 
improves prediction of RNA secondary structure.<em>J. Mol. Biol.</em>, 
288(5):911-940, May 1999. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;] 
<blockquote>  An improved dynamic programming algorithm is reported for RNA 
secondary structure prediction by free energy minimization. Thermodynamic 
parameters for the stabilities of secondary structure motifs are revised to 
include expanded sequence dependence as revealed by recent experiments. 
Additional algorithmic improvements include reduced search time and storage for 
multibranch loop free energies and improved imposition of folding constraints. 
An extended database of 151,503 nt in 955 structures? determined by comparative 
sequence analysis was assembled to allow optimization of parameters not based 
on experiments and to test the accuracy of the algorithm. On average, the 
predicted lowest free energy structure contains 73 % of known base-pairs when 
domains of fewer than 700 nt are folded; this compares with 64 % accuracy for 
previous versions of the algorithm and parameters. For a given sequence, a set 
of 750 generated structures contains one structure that, on average, has 86 % 
of known base-pairs. Experimental constraints, derived from enzymatic and 
flavin mononucleotide cleavage, improve the accuracy of structure predictions. 
</blockquote> 
<p></p> 
<blockquote>  Keywords: 16S, 23S, 5S, Affinity, Algorithms, Aluminum 
Silicates, Amino Acid, Amino Acid Sequence, Amyloidosis, Archaeal, Bacillus, 
Bacterial, Bacterial Proteins, Bacteriophage T4, Base Sequence, Chloroplast, 
Chromatography, Circular Dichroism, Comparative Study, Computational Biology, 
Databases, Electrophoresis, Entropy, Enzyme Stability, Escherichia coli, 
Factual, Fibroblast Growth Factor 2, Flavin Mononucleotide, Fluorescence, 
Genetic, Guanidine, Humans, Huntington Disease, Kinetics, Light, Models, 
Molecular Sequence Data, Non-P.H.S., Non-U.S. Gov't, Nucleic Acid Conformation, 
P.H.S., Peptides, Phylogeny, Polyacrylamide Gel, Predictive Value of Tests, 
Protein Binding, Protein Denaturation, Protein Folding, Protein Structure, RNA, 
Radiation, Recombinant Proteins, Research Support, Ribosomal, Scattering, 
Secondary, Sequence Homology, Solutions, Spectrometry, Statistical, 
Temperature, Thermodynamics, Time Factors, Trinucleotide Repeat Expansion, U.S. 
Gov't, alpha-Amylase, 10329189 </blockquote> <br>
 [Wilbur2000Boosting]  
W.&nbsp;J. Wilbur. Boosting na&iuml; ve Bayesian learning on a large subset of 
MEDLINE.<em>Proc AMIA Symp</em>, pages 918-22, 2000. [&nbsp;bib&nbsp;] 
<blockquote>  We are concerned with the rating of new documents that appear in 
a large database (MEDLINE) and are candidates for inclusion in a small 
specialty database (REBASE). The requirement is to rank the new documents as 
nearly in order of decreasing potential to be added to the smaller database as 
possible, so as to improve the coverage of the smaller database without 
increasing the effort of those who manage this specialty database. To perform 
this ranking task we have considered several machine learning approaches based 
on the na&iuml; ve Bayesian algorithm. We find that adaptive boosting 
outperforms na&iuml; ve Bayes, but that a new form of boosting which we term 
staged Bayesian retrieval outperforms adaptive boosting. Staged Bayesian 
retrieval involves two stages of Bayesian retrieval and we further find that if 
the second stage is replaced by a support vector machine we again obtain a 
significant improvement over the strictly Bayesian approach. </blockquote> 
<p></p> 
<blockquote>  Keywords: Acute, Acute Disease, Adenocarcinoma, Algorithms, 
Amino Acid Sequence, Animals, Artificial Intelligence, Automated, 
B-Lymphocytes, Bacterial Proteins, Base Pair Mismatch, Base Sequence, Bayes 
Theorem, Binding Sites, Biological, Bone Marrow Cells, Brachyura, Cell 
Compartmentation, Chemistry, Child, Chromosome Aberrations, Classification, 
Codon, Colonic Neoplasms, Comparative Study, Computational Biology, Computer 
Simulation, Computer-Assisted, DNA, Data Interpretation, Databases, Decision 
Trees, Diabetes Mellitus, Diagnosis, Discriminant Analysis, Discrimination 
Learning, Electric Conductivity, Electrophysiology, Escherichia coli Proteins, 
Factual, Feedback, Female, Fungal, Gastric Emptying, Gene Expression Profiling, 
Gene Expression Regulation, Genes, Genetic, Genetic Markers, Genetic 
Predisposition to Disease, Genomics, Hemolysins, Humans, Indians, Information 
Storage and Retrieval, Initiator, Ion Channels, Kinetics, Leukemia, Likelihood 
Functions, Lipid Bilayers, Logistic Models, Lymphocytic, MEDLINE, Male, Markov 
Chains, Melanoma, Models, Molecular, Myeloid, Neoplasm, Neoplasms, Neoplastic, 
Neural Networks (Computer), Neurological, Nevus, Non-P.H.S., Non-U.S. Gov't, 
Nonlinear Dynamics, Normal Distribution, North American, Nucleic Acid 
Conformation, Oligonucleotide Array Sequence Analysis, Organ Specificity, 
Organelles, Ovarian Neoplasms, Ovary, P.H.S., Pattern Recognition, Physical, 
Pigmented, Predictive Value of Tests, Promoter Regions (Genetics), Protein 
Biosynthesis, Protein Folding, Protein Structure, Proteins, Proteome, RNA, 
Reproducibility of Results, Research Support, Saccharomyces cerevisiae, 
Secondary, Sensitivity and Specificity, Sequence Alignment, Sequence Analysis, 
Sex Characteristics, Skin Diseases, Skin Neoplasms, Skin Pigmentation, 
Software, Sound Spectrography, Statistical, Stomach Diseases, T-Lymphocytes, 
Thermodynamics, Transcription, Transcription Factors, Tumor Markers, Type 2, 
U.S. Gov't, Vertebrates, 11080018 </blockquote> <br>
 [
Risau-Gusman2000Generalization]  Risau-Gusman and Gordon. Generalization 
properties of finite-size polynomial support vector machines.<em>Phys Rev E 
Stat Phys Plasmas Fluids Relat Interdiscip Topics</em>, 62(5 Pt B):7092-9, Nov 
2000. [&nbsp;bib&nbsp;] 
<blockquote>  The learning properties of finite-size polynomial support vector 
machines are analyzed in the case of realizable classification tasks. The 
normalization of the high-order features acts as a squeezing factor, 
introducing a strong anisotropy in the patterns distribution in feature space. 
As a function of the training set size, the corresponding generalization error 
presents a crossover, more or less abrupt depending on the distribution's 
anisotropy and on the task to be learned, between a fast-decreasing and a 
slowly decreasing regime. This behavior corresponds to the stepwise decrease 
found by Dietrich et al. [Phys. Rev. Lett. 82, 2975 (1999)] in the 
thermodynamic limit. The theoretical results are in excellent agreement with 
the numerical simulations. </blockquote> 
<p></p> 
<blockquote>  Keywords: Acute, Acute Disease, Adenocarcinoma, Algorithms, 
Amino Acid Sequence, Animals, Artificial Intelligence, Automated, 
B-Lymphocytes, Bacterial Proteins, Base Pair Mismatch, Base Sequence, Bayes 
Theorem, Binding Sites, Biological, Bone Marrow Cells, Brachyura, Cell 
Compartmentation, Chemistry, Child, Chromosome Aberrations, Classification, 
Codon, Colonic Neoplasms, Comparative Study, Computational Biology, Computer 
Simulation, Computer-Assisted, DNA, Data Interpretation, Databases, Decision 
Trees, Diabetes Mellitus, Diagnosis, Discriminant Analysis, Discrimination 
Learning, Electric Conductivity, Electrophysiology, Escherichia coli Proteins, 
Factual, Feedback, Female, Fungal, Gastric Emptying, Gene Expression Profiling, 
Gene Expression Regulation, Genes, Genetic, Genetic Markers, Genetic 
Predisposition to Disease, Genomics, Hemolysins, Humans, Indians, Initiator, 
Ion Channels, Kinetics, Leukemia, Likelihood Functions, Lipid Bilayers, 
Logistic Models, Lymphocytic, Male, Markov Chains, Melanoma, Models, Molecular, 
Myeloid, Neoplasm, Neoplasms, Neoplastic, Neural Networks (Computer), 
Neurological, Nevus, Non-P.H.S., Non-U.S. Gov't, Nonlinear Dynamics, Normal 
Distribution, North American, Nucleic Acid Conformation, Oligonucleotide Array 
Sequence Analysis, Organ Specificity, Organelles, Ovarian Neoplasms, Ovary, 
P.H.S., Pattern Recognition, Physical, Pigmented, Predictive Value of Tests, 
Promoter Regions (Genetics), Protein Biosynthesis, Protein Folding, Protein 
Structure, Proteins, Proteome, RNA, Reproducibility of Results, Research 
Support, Saccharomyces cerevisiae, Secondary, Sensitivity and Specificity, 
Sequence Alignment, Sequence Analysis, Sex Characteristics, Skin Diseases, Skin 
Neoplasms, Skin Pigmentation, Software, Sound Spectrography, Statistical, 
Stomach Diseases, T-Lymphocytes, Thermodynamics, Transcription, Transcription 
Factors, Tumor Markers, Type 2, U.S. Gov't, Vertebrates, 0011102066 
</blockquote> <br>
 [Pandey2000Proteomics]  A.&nbsp;Pandey and M.&nbsp;Mann. 
Proteomics to study genes and genomes.<em>Nature</em>, 405:837-846, 2000. 
[&nbsp;bib&nbsp;| http&nbsp;| .pdf&nbsp;] <br>
 [Opper2000Gaussian]  
M.&nbsp;Opper and O.&nbsp;Winther. Gaussian processes for classification: 
mean-field algorithms.<em>Neural Comput</em>, 12(11):2655-84, Nov 2000. [&nbsp;
bib&nbsp;] 
<blockquote>  We derive a mean-field algorithm for binary classification with 
gaussian processes that is based on the TAP approach originally proposed in 
statistical physics of disordered systems. The theory also yields an 
approximate leave-one-out estimator for the generalization error, which is 
computed with no extra computational cost. We show that from the TAP approach, 
it is possible to derive both a simpler &quot;naive&quot; mean-field theory and 
support vector machines (SVMs) as limiting cases. For both mean-field 
algorithms and support vector machines, simulation results for three small 
benchmark data sets are presented. They show that one may get state-of-the-art 
performance by using the leave-one-out estimator for model selection and the 
built-in leave-one-out estimators are extremely precise when compared to the 
exact leave-one-out estimate. The second result is taken as strong support for 
the internal consistency of the mean-field approach. </blockquote> 
<p></p> 
<blockquote>  Keywords: Acute, Acute Disease, Adenocarcinoma, Algorithms, 
Amino Acid Sequence, Animals, Artificial Intelligence, Automated, 
B-Lymphocytes, Bacterial Proteins, Base Pair Mismatch, Base Sequence, Bayes 
Theorem, Binding Sites, Biological, Bone Marrow Cells, Brachyura, Cell 
Compartmentation, Chemistry, Child, Chromosome Aberrations, Classification, 
Colonic Neoplasms, Comparative Study, Computational Biology, Computer 
Simulation, Computer-Assisted, DNA, Data Interpretation, Databases, Decision 
Trees, Diabetes Mellitus, Diagnosis, Discriminant Analysis, Discrimination 
Learning, Electric Conductivity, Electrophysiology, Escherichia coli Proteins, 
Factual, Feedback, Female, Fungal, Gastric Emptying, Gene Expression Profiling, 
Gene Expression Regulation, Genes, Genetic, Genetic Markers, Genetic 
Predisposition to Disease, Hemolysins, Humans, Indians, Ion Channels, Kinetics, 
Leukemia, Likelihood Functions, Lipid Bilayers, Logistic Models, Lymphocytic, 
Male, Markov Chains, Melanoma, Models, Molecular, Myeloid, Neoplasm, Neoplasms, 
Neoplastic, Neural Networks (Computer), Neurological, Nevus, Non-P.H.S., 
Non-U.S. Gov't, Nonlinear Dynamics, Normal Distribution, North American, 
Nucleic Acid Conformation, Oligonucleotide Array Sequence Analysis, Organ 
Specificity, Organelles, Ovarian Neoplasms, Ovary, P.H.S., Pattern Recognition, 
Physical, Pigmented, Predictive Value of Tests, Promoter Regions (Genetics), 
Protein Folding, Protein Structure, Proteins, Proteome, RNA, Reproducibility of 
Results, Research Support, Saccharomyces cerevisiae, Secondary, Sensitivity and 
Specificity, Sequence Alignment, Sex Characteristics, Skin Diseases, Skin 
Neoplasms, Skin Pigmentation, Software, Sound Spectrography, Statistical, 
Stomach Diseases, T-Lymphocytes, Thermodynamics, Transcription, Transcription 
Factors, Tumor Markers, Type 2, U.S. Gov't, 11110131 </blockquote> <br>
 [
Lazo2000Combinatorial]  J.&nbsp;S. Lazo and P.&nbsp;Wipf. Combinatorial 
chemistry and contemporary pharmacology.<em>J. Pharmacol. Exp. Ther.</em>, 
293(3):705-709, Jun 2000. [&nbsp;bib&nbsp;] 
<blockquote>  Both solid- and liquid-phase combinatorial chemistry have 
emerged as powerful tools for identifying pharmacologically active compounds 
and optimizing the biological activity of a lead compound. Complementary 
high-throughput in vitro assays are essential for compound evaluation. 
Cell-based assays that use optical endpoints permit investigation of a wide 
variety of functional properties of these compounds including specific 
intracellular biochemical pathways, protein-protein interactions, and the 
subcellular localization of targets. Integration of combinatorial chemistry 
with contemporary pharmacology now represents an important factor in drug 
discovery and development. </blockquote> 
<p></p> 
<blockquote>  Keywords: Alzheimer Disease, Animals, Antineoplastic Agents, 
Biological, Bleomycin, Cell Cycle, Cell Cycle Proteins, Cell Death, Cell Line, 
Cell Nucleus, Cell Shape, Cell Transformation, Combinatorial Chemistry 
Techniques, Cultured, Drug Delivery Systems, Drug Design, Drug Evaluation, 
Enzyme Inhibitors, Formazans, Gene Expression, Humans, Inhibitory Concentration 
50, Kinetics, Magnetic Resonance Spectroscopy, Mass, Mitochondria, Models, 
Molecular, Neoplasms, Neoplastic, Non-P.H.S., Non-U.S. Gov't, P.H.S., 
Paclitaxel, Peptide Library, Pharmaceutical Preparations, Pharmacology, 
Phosphoprotein Phosphatase, Preclinical, Protease Inhibitors, 
Protein-Tyrosine-Phosphatase, Research Support, Sensitivity and Specificity, 
Signal Transduction, Spectrum Analysis, Stereoisomerism, Structure-Activity 
Relationship, Sulfonic Acids, Tetrazolium Salts, Thiazoles, Toxicity Tests, 
Tumor, Tumor Cells, U.S. Gov't, cdc25 Phosphatase, 10869367 </blockquote> <br>
 
[Klebe2000Recent]  G.&nbsp;Klebe. Recent developments in structure-based drug 
design.<em>J Mol Med</em>, 78(5):269-281, 2000. [&nbsp;bib&nbsp;] 
<blockquote>  Structure-based design has emerged as a new tool in medicinal 
chemistry. A prerequisite for this new approach is an understanding of the 
principles of molecular recognition in protein-ligand complexes. If the 
three-dimensional structure of a given protein is known, this information can 
be directly exploited for the retrieval and design of new ligands. 
Structure-based ligand design is an iterative approach. First of all, it 
requires the crystal structure or a model derived from the crystal structure of 
a closely related homolog of the target protein, preferentially complexed with 
a ligand. This complex unravels the binding mode and conformation of a ligand 
under investigation and indicates the essential aspects determining its binding 
affinity. It is then used to generate new ideas about ways of improving an 
existing ligand or of developing new alternative bonding skeletons. 
Computational methods supplemented by molecular graphics are applied to assist 
this step of hypothesis generation. The features of the protein binding pocket 
can be translated into queries used for virtual computer screening of large 
compound libraries or to design novel ligands de novo. These initial proposals 
must be confirmed experimentally. Subsequently they are optimized toward higher 
affinity and better selectivity. The latter aspect is of utmost importance in 
defining and controlling the pharmacological profile of a ligand. A 
prerequisite to tailoring selectivity by rational design is a detailed 
understanding of molecular parameters determining selectivity. Taking examples 
from current drug development programs (HIV proteinase, t-RNA transglycosylase, 
thymidylate synthase, thrombin and, related serine proteinases), we describe 
recent advances in lead discovery via computer screening, iterative design, and 
understanding of selectivity discrimination. </blockquote> 
<p></p> 
<blockquote>  Keywords: Animals, Chemistry, Computer Simulation, Cross-Over 
Studies, Crystallography, Deglutition, Deglutition Disorders, Drug Design, 
Endoscopy, Enzyme Inhibitors, Female, Fluoroscopy, Glossopharyngeal Nerve, HIV 
Protease Inhibitors, Horse Diseases, Horses, Male, Models, Molecular, Nerve 
Block, Non-U.S. Gov't, P.H.S., Pharmaceutical, Proteins, Quantitative 
Structure-Activity Relationship, Random Allocation, Research Support, Thrombin, 
Thymidylate Synthase, U.S. Gov't, X-Ray, 10954196 </blockquote> <br>
 [
Gether2000Uncovering]  U.&nbsp;Gether. Uncovering molecular mechanisms involved 
in activation of g protein-coupled receptors.<em>Endocr Rev</em>, 21(1):90-113, 
Feb 2000. [&nbsp;bib&nbsp;] 
<blockquote>  G protein-coupled, seven-transmembrane segment receptors (GPCRs 
or 7TM receptors), with more than 1000 different members, comprise the largest 
superfamily of proteins in the body. Since the cloning of the first receptors 
more than a decade ago, extensive experimental work has uncovered multiple 
aspects of their function and challenged many traditional paradigms. However, 
it is only recently that we are beginning to gain insight into some of the most 
fundamental questions in the molecular function of this class of receptors. How 
can, for example, so many chemically diverse hormones, neurotransmitters, and 
other signaling molecules activate receptors believed to share a similar 
overall tertiary structure? What is the nature of the physical changes linking 
agonist binding to receptor activation and subsequent transduction of the 
signal to the associated G protein on the cytoplasmic side of the membrane and 
to other putative signaling pathways? The goal of the present review is to 
specifically address these questions as well as to depict the current awareness 
about GPCR structure-function relationships in general. </blockquote> 
<p></p> 
<blockquote>  Keywords: Animals; GTP-Binding Proteins; Humans; Ligands; 
Models, Biological; Molecular Conformation; Receptors, Cell Surface 
</blockquote> <br>
 [Baudat2000Generalized]  G.&nbsp;Baudat and F.&nbsp;Anouar. 
Generalized discriminant analysis using a kernel approach.<em>Neural Comput</em>
, 12(10):2385-404, Oct 2000. [&nbsp;bib&nbsp;] 
<blockquote>  We present a new method that we call generalized discriminant 
analysis (GDA) to deal with nonlinear discriminant analysis using kernel 
function operator. The underlying theory is close to the support vector 
machines (SVM) insofar as the GDA method provides a mapping of the input 
vectors into high-dimensional feature space. In the transformed space, linear 
properties make it easy to extend and generalize the classical linear 
discriminant analysis (LDA) to nonlinear discriminant analysis. The formulation 
is expressed as an eigenvalue problem resolution. Using a different kernel, one 
can cover a wide class of nonlinearities. For both simulated data and alternate 
kernels, we give classification results, as well as the shape of the decision 
function. The results are confirmed using real data to perform seed 
classification. </blockquote> 
<p></p> 
<blockquote>  Keywords: Acute, Acute Disease, Adenocarcinoma, Algorithms, 
Amino Acid Sequence, Animals, Artificial Intelligence, Automated, 
B-Lymphocytes, Bacterial Proteins, Base Pair Mismatch, Base Sequence, Bayes 
Theorem, Binding Sites, Biological, Bone Marrow Cells, Brachyura, Cell 
Compartmentation, Chemistry, Child, Chromosome Aberrations, Classification, 
Codon, Colonic Neoplasms, Comparative Study, Computational Biology, Computer 
Simulation, Computer-Assisted, DNA, Data Interpretation, Databases, Decision 
Trees, Diabetes Mellitus, Diagnosis, Discriminant Analysis, Discrimination 
Learning, Electric Conductivity, Electroencephalography, Electrophysiology, 
Epilepsy, Escherichia coli Proteins, Factual, Feedback, Female, Fungal, Gastric 
Emptying, Gene Expression Profiling, Gene Expression Regulation, Genes, 
Genetic, Genetic Markers, Genetic Predisposition to Disease, Genomics, 
Hemolysins, Humans, Indians, Information Storage and Retrieval, Initiator, Ion 
Channels, Kinetics, Leukemia, Likelihood Functions, Linear Models, Lipid 
Bilayers, Logistic Models, Lymphocytic, MEDLINE, Male, Markov Chains, Melanoma, 
Models, Molecular, Myeloid, Neoplasm, Neoplasms, Neoplastic, Neural Networks 
(Computer), Neurological, Nevus, Non-P.H.S., Non-U.S. Gov't, Nonlinear 
Dynamics, Normal Distribution, North American, Nucleic Acid Conformation, 
Oligonucleotide Array Sequence Analysis, Organ Specificity, Organelles, Ovarian 
Neoplasms, Ovary, P.H.S., Pattern Recognition, Physical, Pigmented, Predictive 
Value of Tests, Promoter Regions (Genetics), Protein Biosynthesis, Protein 
Folding, Protein Structure, Proteins, Proteome, RNA, Reproducibility of 
Results, Research Support, Saccharomyces cerevisiae, Secondary, Seeds, 
Sensitivity and Specificity, Sequence Alignment, Sequence Analysis, Sex 
Characteristics, Skin Diseases, Skin Neoplasms, Skin Pigmentation, Software, 
Sound Spectrography, Statistical, Stochastic Processes, Stomach Diseases, 
T-Lymphocytes, Thermodynamics, Transcription, Transcription Factors, Tumor 
Markers, Type 2, U.S. Gov't, Vertebrates, 11032039 </blockquote> <br>
 [
Zhu2001Global]  H.&nbsp;Zhu, M.&nbsp;Bilgin, R.&nbsp;Bangham, D.&nbsp;Hall, 
A.&nbsp;Casamayor, P.&nbsp;Bertone, N.&nbsp;Lan, R.&nbsp;Jansen, 
S.&nbsp;Bidlingmaier, T.&nbsp;Houfek, T.&nbsp;Mitchell, P.&nbsp;Miller, 
R.&nbsp;A. Dean, M.&nbsp;Gerstein, and M.&nbsp;Snyder. Global analysis of 
protein activities using proteome chips.<em>Science</em>, 293(5537):2101-5, Sep 
2001. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  To facilitate studies of the yeast proteome, we cloned 5800 open 
reading frames and overexpressed and purified their corresponding proteins. The 
proteins were printed onto slides at high spatial density to form a yeast 
proteome microarray and screened for their ability to interact with proteins 
and phospholipids. We identified many new calmodulin- and 
phospholipid-interacting proteins; a common potential binding motif was 
identified for many of the calmodulin-binding proteins. Thus, microarrays of an 
entire eukaryotic proteome can be prepared and screened for diverse biochemical 
activities. The microarrays can also be used to screen protein-drug 
interactions and to detect posttranslational modifications. </blockquote> 
<p></p> 
<blockquote>  Keywords: Amino Acid Motifs, Amino Acid Sequence, Calmodulin, 
Calmodulin-Binding Proteins, Cell Membrane, Cloning, Fungal Proteins, Glucose, 
Liposomes, Membrane Proteins, Molecular, Molecular Sequence Data, Non-U.S. 
Gov't, Open Reading Frames, P.H.S., Peptide Library, Phosphatidylcholines, 
Phosphatidylinositols, Phospholipids, Protein Binding, Proteome, Recombinant 
Fusion Proteins, Research Support, Saccharomyces cerevisiae, Signal 
Transduction, Streptavidin, U.S. Gov't, 11474067 </blockquote> <br>
 [
Xue2001Mini-fingerprints]  L.&nbsp;Xue, F.&nbsp;L. Stahura, J.&nbsp;W. Godden, 
and J.&nbsp;Bajorath. Mini-fingerprints detect similar activity of receptor 
ligands previously recognized only by three-dimensional pharmacophore-based 
methods.<em>J Chem Inf Comput Sci</em>, 41(2):394-401, 2001. [&nbsp;bib&nbsp;] 
<blockquote>  Mini-fingerprints (MFPs) are short binary bit string 
representations of molecular structure and properties, composed of few selected 
two-dimensional (2D) descriptors and a number of structural keys. MFPs were 
specifically designed to recognize compounds with similar activity. Here we 
report that MFPs are capable of detecting similar activities of some druglike 
molecules, including endothelin A antagonists and alpha(1)-adrenergic receptor 
ligands, the recognition of which was previously thought to depend on the use 
of multiple point three-dimensional (3D) pharmacophore methods. Thus, in these 
cases, MFPs and pharmacophore fingerprints produce similar results, although 
they define, in terms of their complexity, opposite ends of the spectrum of 
methods currently used to study molecular similarity or diversity. For each of 
the studied compound classes, comparison of MFP bit settings identified a 
consensus or signature pattern. Scaling factors can be applied to these bits in 
order to increase the probability of finding compounds with similar activity by 
virtual screening. </blockquote> 
<p></p> 
<blockquote>  Keywords: Adrenergic, Angiotensin II, Cell Surface, 
Combinatorial Chemistry Techniques, Databases, Drug Evaluation, Endothelins, 
Environmental Pollutants, Factual, Information Management, Ligands, Molecular 
Structure, Pharmaceutical Preparations, Platelet Glycoprotein GPIIb-IIIa 
Complex, Preclinical, Receptors, Serine Proteinase Inhibitors, 
Structure-Activity Relationship, User-Computer Interface, alpha-1, 11277728 
</blockquote> <br>
 [Vercoutere2001Rapid]  W.&nbsp;Vercoutere, 
S.&nbsp;Winters-Hilt, H.&nbsp;Olsen, D.&nbsp;Deamer, D.&nbsp;Haussler, and 
M.&nbsp;Akeson. Rapid discrimination among individual DNA hairpin molecules at 
single-nucleotide resolution using an ion channel.<em>Nat Biotechnol</em>, 
19(3):248-52, Mar 2001. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  RNA and DNA strands produce ionic current signatures when driven 
through an alpha-hemolysin channel by an applied voltage. Here we combine this 
nanopore detector with a support vector machine (SVM) to analyze DNA hairpin 
molecules on the millisecond time scale. Measurable properties include duplex 
stem length, base pair mismatches, and loop length. This nanopore instrument 
can discriminate between individual DNA hairpins that differ by one base pair 
or by one nucleotide. </blockquote> 
<p></p> 
<blockquote>  Keywords: Acute, Acute Disease, Adenocarcinoma, Algorithms, 
Amino Acid Sequence, Artificial Intelligence, Automated, B-Lymphocytes, 
Bacterial Proteins, Base Pair Mismatch, Base Sequence, Bayes Theorem, Binding 
Sites, Biological, Bone Marrow Cells, Cell Compartmentation, Chemistry, Child, 
Chromosome Aberrations, Comparative Study, Computational Biology, Computer 
Simulation, Computer-Assisted, DNA, Data Interpretation, Databases, Decision 
Trees, Diagnosis, Discriminant Analysis, Electric Conductivity, 
Electrophysiology, Escherichia coli Proteins, Factual, Female, Fungal, Gastric 
Emptying, Gene Expression Profiling, Gene Expression Regulation, Genes, 
Genetic, Genetic Markers, Hemolysins, Humans, Ion Channels, Kinetics, Leukemia, 
Lipid Bilayers, Logistic Models, Lymphocytic, Male, Markov Chains, Melanoma, 
Models, Molecular, Myeloid, Neoplasm, Neoplastic, Neural Networks (Computer), 
Nevus, Non-P.H.S., Non-U.S. Gov't, Nucleic Acid Conformation, Organ 
Specificity, Organelles, P.H.S., Pattern Recognition, Physical, Pigmented, 
Predictive Value of Tests, Promoter Regions (Genetics), Protein Folding, 
Protein Structure, Proteins, Proteome, RNA, Reproducibility of Results, 
Research Support, Saccharomyces cerevisiae, Secondary, Sensitivity and 
Specificity, Sequence Alignment, Sex Characteristics, Skin Diseases, Skin 
Neoplasms, Skin Pigmentation, Software, Statistical, Stomach Diseases, 
T-Lymphocytes, Thermodynamics, Transcription, Transcription Factors, Tumor 
Markers, U.S. Gov't, 11231558 </blockquote> <br>
 [Vazquez2001Modeling]  
A.&nbsp;Vazquez, A.&nbsp;Flammini, A.&nbsp;Maritan, and A.&nbsp;Vespignani. 
Modeling of protein interaction networks. E-print cond-mat/0108043, Aug 2001. 
[&nbsp;bib&nbsp;| http&nbsp;| .pdf&nbsp;] <br>
 [Teixeira2001Recent]  
R.&nbsp;D. Teixeira, A.&nbsp;P. Braga, R.&nbsp;H. Takahashi, and R.&nbsp;R. 
Saldanha. Recent advances in the MOBJ algorithm for training artificial neural 
networks.<em>Int J Neural Syst</em>, 11(3):265-70, Jun 2001. [&nbsp;bib&nbsp;] 
<blockquote>  This paper presents a new scheme for training MLPs which employs 
a relaxation method for multi-objective optimization. The algorithm works by 
obtaining a reduced set of solutions, from which the one with the best 
generalization is selected. This approach allows balancing between the training 
error and norm of network weight vectors, which are the two objective functions 
of the multi-objective optimization problem. The method is applied to 
classification and regression problems and compared with Weight Decay (WD), 
Support Vector Machines (SVMs) and standard Backpropagation (BP). It is shown 
that the systematic procedure for training proposed results on good 
generalization neural models, and outperforms traditional methods. </blockquote>
<p></p> 
<blockquote>  Keywords: 80 and over, Adult, Aged, Algorithms, Amino Acid 
Sequence, Animals, Artificial Intelligence, Automated, Automatic Data 
Processing, Bayes Theorem, Binding Sites, Biological, Brain, Cell Surface, 
Chromosomes, Clinical, Cluster Analysis, Colonic Polyps, Colonography, 
Comparative Study, Computational Biology, Computed Tomographic, 
Computer-Assisted, Databases, Diagnosis, Differential, Drug Design, Factual, 
False Negative Reactions, Female, GTP-Binding Proteins, Gene Expression 
Profiling, Genetic, Genetic Markers, Genetic Predisposition to Disease, 
Genotype, Glaucoma, Human, Humans, Image Cytometry, Image Processing, Imaging, 
Internet, Karyotyping, Learning, Male, Mathematical Computing, Middle Aged, 
Models, Molecular Sequence Data, Molecular Structure, Multigene Family, 
N-Acetylgalactosaminyltransferases, Neoplasms, Neural Networks (Computer), 
Neurons, Non-P.H.S., Non-U.S. Gov't, Nonlinear Dynamics, Normal Distribution, 
Nucleic Acid Hybridization, Oligonucleotide Array Sequence Analysis, Optic 
Nerve, P.H.S., Pattern Recognition, Perimetry, Pharmacology, Phenotype, 
Photography, Pred, Principal Component Analysis, Programming Languages, 
Promoter Regions (Genetics), Protein, Proteins, ROC Curve, Radiographic Image 
Interpretation, Receptors, Reproducibility of Results, Research Support, 
Sensitivity and Specificity, Sequence Alignment, Software, Statistical, 
Structure-Activity Relationship, Subcellular Fractions, Substrate Specificity, 
Theoretical, Thermodynamics, Three-Dimensional, Tomography, Transcription, 
Tumor Markers, U.S. Gov't, Visual, Visual Fields, X-Ray Computed, ictive Value 
of Tests, 11574964 </blockquote> <br>
 [Talukder2001closed-form]  
A.&nbsp;Talukder and D.&nbsp;Casasent. A closed-form neural network for 
discriminatory feature extraction from high-dimensional data.<em>Neural Netw
</em>, 14(9):1201-18, Nov 2001. [&nbsp;bib&nbsp;] 
<blockquote>  We consider a new neural network for data discrimination in 
pattern recognition applications. We refer to this as a maximum discriminating 
feature (MDF) neural network. Its weights are obtained in closed-form, thereby 
overcoming problems associated with other nonlinear neural networks. It uses 
neuron activation functions that are dynamically chosen based on the 
application. It is theoretically shown to provide nonlinear transforms of the 
input data that are more general than those provided by other nonlinear 
multilayer perceptron neural network and support-vector machine techniques for 
cases involving high-dimensional (image) inputs where training data are limited 
and the classes are not linearly separable. We experimentally verify this on 
synthetic examples. </blockquote> 
<p></p> 
<blockquote>  Keywords: 80 and over, Adult, Aged, Algorithms, Amino Acid 
Sequence, Animals, Artificial Intelligence, Automated, Automatic Data 
Processing, Binding Sites, Biological, Brain, Cell Surface, Chromosomes, 
Clinical, Cluster Analysis, Colonic Polyps, Colonography, Comparative Study, 
Computational Biology, Computed Tomographic, Computer-Assisted, Databases, 
Diagnosis, Differential, Drug Design, Factual, False Negative Reactions, 
Female, GTP-Binding Proteins, Gene Expression Profiling, Genetic, Glaucoma, 
Human, Humans, Image Cytometry, Image Processing, Imaging, Internet, 
Karyotyping, Male, Middle Aged, Models, Molecular Sequence Data, Molecular 
Structure, Multigene Family, N-Acetylgalactosaminyltransferases, Neoplasms, 
Neural Networks (Computer), Neurons, Non-P.H.S., Non-U.S. Gov't, Nonlinear 
Dynamics, Nucleic Acid Hybridization, Oligonucleotide Array Sequence Analysis, 
Optic Nerve, P.H.S., Pattern Recognition, Perimetry, Pharmacology, Photography, 
Pred, Principal Component Analysis, Programming Languages, Promoter Regions 
(Genetics), Protein, Proteins, ROC Curve, Radiographic Image Interpretation, 
Receptors, Reproducibility of Results, Research Support, Sensitivity and 
Specificity, Sequence Alignment, Software, Statistical, Structure-Activity 
Relationship, Subcellular Fractions, Substrate Specificity, Theoretical, 
Three-Dimensional, Tomography, Transcription, Tumor Markers, U.S. Gov't, 
Visual, Visual Fields, X-Ray Computed, ictive Value of Tests, 11718421 
</blockquote> <br>
 [Soerlie2001Genea]  T.&nbsp;S&oslash;rlie, C.&nbsp;M. 
Perou, R.&nbsp;Tibshirani, T.&nbsp;Aas, S.&nbsp;Geisler, H.&nbsp;Johnsen, 
T.&nbsp;Hastie, M.&nbsp;B. Eisen, M.&nbsp;van&nbsp;de Rijn, S.&nbsp;S. Jeffrey, 
T.&nbsp;Thorsen, H.&nbsp;Quist, J.&nbsp;C. Matese, P.&nbsp;O. Brown, 
D.&nbsp;Botstein, P.&nbsp;Eystein L&oslash;nning, and A.&nbsp;L. 
B&oslash;rresen-Dale. Gene expression patterns of breast carcinomas distinguish 
tumor subclasses with clinical implications.<em>Proc. Natl. Acad. Sci. U. S. A.
</em>, 98(19):10869-10874, Sep 2001. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;] 
<blockquote>  The purpose of this study was to classify breast carcinomas 
based on variations in gene expression patterns derived from cDNA microarrays 
and to correlate tumor characteristics to clinical outcome. A total of 85 cDNA 
microarray experiments representing 78 cancers, three fibroadenomas, and four 
normal breast tissues were analyzed by hierarchical clustering. As reported 
previously, the cancers could be classified into a basal epithelial-like group, 
an ERBB2-overexpressing group and a normal breast-like group based on 
variations in gene expression. A novel finding was that the previously 
characterized luminal epithelial/estrogen receptor-positive group could be 
divided into at least two subgroups, each with a distinctive expression 
profile. These subtypes proved to be reasonably robust by clustering using two 
different gene sets: first, a set of 456 cDNA clones previously selected to 
reflect intrinsic properties of the tumors and, second, a gene set that highly 
correlated with patient outcome. Survival analyses on a subcohort of patients 
with locally advanced breast cancer uniformly treated in a prospective study 
showed significantly different outcomes for the patients belonging to the 
various groups, including a poor prognosis for the basal-like subtype and a 
significant difference in outcome for the two estrogen receptor-positive groups.
</blockquote> 
<p></p> 
<blockquote>  Keywords: Algorithms; Breast Neoplasms; Carcinoma in Situ; 
Carcinoma, Ductal, Breast; Carcinoma, Lobular; DNA, Neoplasm; Female; 
Fibroadenoma; Gene Expression; Gene Expression Profiling; Humans; 
Oligonucleotide Array Sequence Analysis; Tumor Suppressor Protein p53 
</blockquote> <br>
 [Suykens2001Optimal]  J.&nbsp;A. Suykens, 
J.&nbsp;Vandewalle, and B.&nbsp;De Moor. Optimal control by least squares 
support vector machines.<em>Neural Netw</em>, 14(1):23-35, Jan 2001. [&nbsp;bib
&nbsp;]
<blockquote>  Support vector machines have been very successful in pattern 
recognition and function estimation problems. In this paper we introduce the 
use of least squares support vector machines (LS-SVM's) for the optimal control 
of nonlinear systems. Linear and neural full static state feedback controllers 
are considered. The problem is formulated in such a way that it incorporates 
the N-stage optimal control problem as well as a least squares support vector 
machine approach for mapping the state space into the action space. The 
solution is characterized by a set of nonlinear equations. An alternative 
formulation as a constrained nonlinear optimization problem in less unknowns is 
given, together with a method for imposing local stability in the LS-SVM 
control scheme. The results are discussed for support vector machines with 
radial basis function kernel. Advantages of LS-SVM control are that no number 
of hidden units has to be determined for the controller and that no centers 
have to be specified for the Gaussian kernels when applying Mercer's condition. 
The curse of dimensionality is avoided in comparison with defining a regular 
grid for the centers in classical radial basis function networks. This is at 
the expense of taking the trajectory of state variables as additional unknowns 
in the optimization problem, while classical neural network approaches 
typically lead to parametric optimization problems. In the SVM methodology the 
number of unknowns equals the number of training data, while in the primal 
space the number of unknowns can be infinite dimensional. The method is 
illustrated both on stabilization and tracking problems including examples on 
swinging up an inverted pendulum with local stabilization at the endpoint and a 
tracking problem for a ball and beam system. </blockquote> 
<p></p> 
<blockquote>  Keywords: Acute, Acute Disease, Adenocarcinoma, Algorithms, 
Amino Acid Sequence, Artificial Intelligence, Automated, B-Lymphocytes, 
Bacterial Proteins, Base Pair Mismatch, Base Sequence, Bayes Theorem, Binding 
Sites, Biological, Bone Marrow Cells, Cell Compartmentation, Chemistry, Child, 
Chromosome Aberrations, Comparative Study, Computational Biology, Computer 
Simulation, Computer-Assisted, DNA, Data Interpretation, Databases, Decision 
Trees, Diagnosis, Discriminant Analysis, Electric Conductivity, 
Electrophysiology, Escherichia coli Proteins, Factual, Feedback, Female, 
Fungal, Gastric Emptying, Gene Expression Profiling, Gene Expression 
Regulation, Genes, Genetic, Genetic Markers, Hemolysins, Humans, Ion Channels, 
Kinetics, Leukemia, Lipid Bilayers, Logistic Models, Lymphocytic, Male, Markov 
Chains, Melanoma, Models, Molecular, Myeloid, Neoplasm, Neoplastic, Neural 
Networks (Computer), Nevus, Non-P.H.S., Non-U.S. Gov't, Nonlinear Dynamics, 
Normal Distribution, Nucleic Acid Conformation, Organ Specificity, Organelles, 
P.H.S., Pattern Recognition, Physical, Pigmented, Predictive Value of Tests, 
Promoter Regions (Genetics), Protein Folding, Protein Structure, Proteins, 
Proteome, RNA, Reproducibility of Results, Research Support, Saccharomyces 
cerevisiae, Secondary, Sensitivity and Specificity, Sequence Alignment, Sex 
Characteristics, Skin Diseases, Skin Neoplasms, Skin Pigmentation, Software, 
Statistical, Stomach Diseases, T-Lymphocytes, Thermodynamics, Transcription, 
Transcription Factors, Tumor Markers, U.S. Gov't, 11213211 </blockquote> <br>
 [
Sole2001Model]  R.&nbsp;V. Sol&eacute;, R.&nbsp;Pastor-Satorras, E.&nbsp;D. 
Smith, and T.&nbsp;Kepler. A Model of Large-Scale Proteome Evolution. Technical 
report, Santa Fe Institute, 2001. Working paper 01-08-041. [&nbsp;bib&nbsp;| 
.html&nbsp;| .pdf&nbsp;] <br>
 [Seol2001Skp1]  J.&nbsp;H. Seol, 
A.&nbsp;Shevchenko, A.&nbsp;Shevchenko, and R.&nbsp;J. Deshaies. Skp1 forms 
multiple protein complexes, including RAVE, a regulator of V-ATPase assembly.
<em>Nat Cell Biol</em>, 3(4):384-91, Apr 2001. [&nbsp;bib&nbsp;| DOI&nbsp;| http
&nbsp;|.pdf&nbsp;] 
<blockquote>  SCF ubiquitin ligases are composed of Skp1, Cdc53, Hrt1 and one 
member of a large family of substrate receptors known as F-box proteins (FBPs). 
Here we report the identification, using sequential rounds of epitope tagging, 
affinity purification and mass spectrometry, of 16 Skp1 and Cdc53-associated 
proteins in budding yeast, including all components of SCF, 9 FBPs, Yjr033 
(Rav1) and Ydr202 (Rav2). Rav1, Rav2 and Skp1 form a complex that we have named 
'regulator of the (H+)-ATPase of the vacuolar and endosomal membranes' (RAVE), 
which associates with the V1 domain of the vacuolar membrane (H+)-ATPase 
(V-ATPase). V-ATPases are conserved throughout eukaryotes, and have been 
implicated in tumour metastasis and multidrug resistance, and here we show that 
RAVE promotes glucose-triggered assembly of the V-ATPase holoenzyme. Previous 
systematic genome-wide two-hybrid screens yielded 17 proteins that interact 
with Skp1 and Cdc53, only 3 of which overlap with those reported here. Thus, 
our results provide a distinct view of the interactions that link proteins into 
a comprehensive cellular network. </blockquote> 
<p></p> 
<blockquote>  Keywords: Affinity, Affinity Labels, Amino Acid Sequence, 
Animals, Cell Cycle Proteins, Cells, Chromatography, Cloning, Comparative 
Study, Cullin Proteins, Cultured, Cytoplasm, DNA, DNA Damage, DNA Repair, 
Electrospray Ionization, Fungal, Fungal Proteins, Gene Targeting, Genetic, 
Glucose, Holoenzymes, Humans, Macromolecular Substances, Mass, Matrix-Assisted 
Laser Desorption-Ionization, Mitosis, Molecular, Molecular Sequence Data, 
Non-P.H.S., Non-U.S. Gov't, P.H.S., Phosphoric Monoester Hydrolases, Protein 
Binding, Protein Interaction Mapping, Protein Kinases, Proteome, Proteomics, 
Proton-Translocating ATPases, Recombinant Fusion Proteins, Research Support, 
Ribonucleoproteins, Ribosomes, S-Phase Kinase-Associated Proteins, 
Saccharomyces cerevisiae, Saccharomyces cerevisiae Proteins, Sensitivity and 
Specificity, Sequence Alignment, Signal Transduction, Species Specificity, 
Spectrometry, Spectrum Analysis, Transcription, U.S. Gov't, Vacuolar 
Proton-Translocating ATPases, 11283612 </blockquote> <br>
 [
Risau-Gusman2001Statistical]  S.&nbsp;Risau-Gusman and M.&nbsp;B. Gordon. 
Statistical mechanics of learning with soft margin classifiers.<em>Phys Rev E 
Stat Nonlin Soft Matter Phys</em>, 64(3 Pt 1):031907, Sep 2001. [&nbsp;bib
&nbsp;]
<blockquote>  We study the typical learning properties of the recently 
introduced soft margin classifiers (SMCs), learning realizable and unrealizable 
tasks, with the tools of statistical mechanics. We derive analytically the 
behavior of the learning curves in the regime of very large training sets. We 
obtain exponential and power laws for the decay of the generalization error 
towards the asymptotic value, depending on the task and on general 
characteristics of the distribution of stabilities of the patterns to be 
learned. The optimal learning curves of the SMCs, which give the minimal 
generalization error, are obtained by tuning the coefficient controlling the 
trade-off between the error and the regularization terms in the cost function. 
If the task is realizable by the SMC, the optimal performance is better than 
that of a hard margin support vector machine and is very close to that of a 
Bayesian classifier. </blockquote> 
<p></p> 
<blockquote>  Keywords: 80 and over, Adult, Aged, Algorithms, Amino Acid 
Sequence, Animals, Artificial Intelligence, Automated, Automatic Data 
Processing, Bayes Theorem, Binding Sites, Biological, Brain, Cell Surface, 
Chromosomes, Clinical, Cluster Analysis, Colonic Polyps, Colonography, 
Comparative Study, Computational Biology, Computed Tomographic, 
Computer-Assisted, Databases, Diagnosis, Differential, Drug Design, Factual, 
False Negative Reactions, Female, GTP-Binding Proteins, Gene Expression 
Profiling, Genetic, Genetic Markers, Genetic Predisposition to Disease, 
Genotype, Glaucoma, Human, Humans, Image Cytometry, Image Processing, Imaging, 
Internet, Karyotyping, Learning, Male, Mathematical Computing, Middle Aged, 
Models, Molecular Sequence Data, Molecular Structure, Multigene Family, 
N-Acetylgalactosaminyltransferases, Neoplasms, Neural Networks (Computer), 
Neurons, Non-P.H.S., Non-U.S. Gov't, Nonlinear Dynamics, Normal Distribution, 
Nucleic Acid Hybridization, Oligonucleotide Array Sequence Analysis, Optic 
Nerve, P.H.S., Pattern Recognition, Perimetry, Pharmacology, Phenotype, 
Photography, Pred, Principal Component Analysis, Programming Languages, 
Promoter Regions (Genetics), Protein, Proteins, ROC Curve, Radiographic Image 
Interpretation, Receptors, Reproducibility of Results, Research Support, 
Sensitivity and Specificity, Sequence Alignment, Software, Statistical, 
Structure-Activity Relationship, Subcellular Fractions, Substrate Specificity, 
Theoretical, Thermodynamics, Three-Dimensional, Tomography, Transcription, 
Tumor Markers, U.S. Gov't, Visual, Visual Fields, X-Ray Computed, ictive Value 
of Tests, 11580367 </blockquote> <br>
 [Rain2001protein-protein]  J.-C. Rain, 
L.&nbsp;Selig, H.&nbsp;De&nbsp;Reuse, V.&nbsp;Battaglia, C.&nbsp;Reverdy, 
S.&nbsp;Simon, G.&nbsp;Lenzen, F.&nbsp;Petel, J.&nbsp;Wojcik, 
V.&nbsp;Sch&auml;chter, Y.&nbsp;Chemama, A.&nbsp;Labigne, and P.&nbsp;Legrain. 
The protein-protein interaction map of Helicobacter pylori.<em>Nature</em>, 
409:211-215, 2001. [&nbsp;bib&nbsp;| http&nbsp;| .pdf&nbsp;] <br>
 [
Miwakeichi2001comparison]  F.&nbsp;Miwakeichi, R.&nbsp;Ramirez-Padron, 
P.&nbsp;A. Valdes-Sosa, and T.&nbsp;Ozaki. A comparison of non-linear 
non-parametric models for epilepsy data.<em>Comput. Biol. Med.</em>, 
31(1):41-57, Jan 2001. [&nbsp;bib&nbsp;] 
<blockquote>  EEG spike and wave (SW) activity has been described through a 
non-parametric stochastic model estimated by the Nadaraya-Watson (NW) method. 
In this paper the performance of the NW, the local linear polynomial regression 
and support vector machines (SVM) methods were compared. The noise-free 
realizations obtained by the NW and SVM methods reproduced SW better than as 
reported in previous works. The tuning parameters had to be estimated manually. 
Adding dynamical noise, only the NW method was capable of generating SW similar 
to training data. The standard deviation of the dynamical noise was estimated 
by means of the correlation dimension. </blockquote> 
<p></p> 
<blockquote>  Keywords: Acute, Acute Disease, Adenocarcinoma, Algorithms, 
Amino Acid Sequence, Animals, Artificial Intelligence, Automated, 
B-Lymphocytes, Bacterial Proteins, Base Pair Mismatch, Base Sequence, Bayes 
Theorem, Binding Sites, Biological, Bone Marrow Cells, Brachyura, Cell 
Compartmentation, Chemistry, Child, Chromosome Aberrations, Classification, 
Codon, Colonic Neoplasms, Comparative Study, Computational Biology, Computer 
Simulation, Computer-Assisted, DNA, Data Interpretation, Databases, Decision 
Trees, Diabetes Mellitus, Diagnosis, Discriminant Analysis, Discrimination 
Learning, Electric Conductivity, Electroencephalography, Electrophysiology, 
Epilepsy, Escherichia coli Proteins, Factual, Feedback, Female, Fungal, Gastric 
Emptying, Gene Expression Profiling, Gene Expression Regulation, Genes, 
Genetic, Genetic Markers, Genetic Predisposition to Disease, Genomics, 
Hemolysins, Humans, Indians, Information Storage and Retrieval, Initiator, Ion 
Channels, Kinetics, Leukemia, Likelihood Functions, Linear Models, Lipid 
Bilayers, Logistic Models, Lymphocytic, MEDLINE, Male, Markov Chains, Melanoma, 
Models, Molecular, Myeloid, Neoplasm, Neoplasms, Neoplastic, Neural Networks 
(Computer), Neurological, Nevus, Non-P.H.S., Non-U.S. Gov't, Nonlinear 
Dynamics, Normal Distribution, North American, Nucleic Acid Conformation, 
Oligonucleotide Array Sequence Analysis, Organ Specificity, Organelles, Ovarian 
Neoplasms, Ovary, P.H.S., Pattern Recognition, Physical, Pigmented, Predictive 
Value of Tests, Promoter Regions (Genetics), Protein Biosynthesis, Protein 
Folding, Protein Structure, Proteins, Proteome, RNA, Reproducibility of 
Results, Research Support, Saccharomyces cerevisiae, Secondary, Sensitivity and 
Specificity, Sequence Alignment, Sequence Analysis, Sex Characteristics, Skin 
Diseases, Skin Neoplasms, Skin Pigmentation, Software, Sound Spectrography, 
Statistical, Stochastic Processes, Stomach Diseases, T-Lymphocytes, 
Thermodynamics, Transcription, Transcription Factors, Tumor Markers, Type 2, 
U.S. Gov't, Vertebrates, 11058693 </blockquote> <br>
 [Kim2001Evolving]  
J.&nbsp;Kim, P.L. Krapivsky, B.&nbsp;Kahng, and S.&nbsp;Redner. Evolving 
protein interaction networks. E-print cond-mat/0203167, 2001. [&nbsp;bib&nbsp;| 
http&nbsp;| .pdf&nbsp;] <br>
 [Goektuerk2001statistical]  S.&nbsp;B. 
G&ouml;kt&uuml;rk, C.&nbsp;Tomasi, B.&nbsp;Acar, C.&nbsp;F. Beaulieu, 
D.&nbsp;S. Paik, R.&nbsp;B. Jeffrey, J.&nbsp;Yee, and S.&nbsp;Napel. A 
statistical 3-D pattern processing method for computer-aided detection of 
polyps in CT colonography.<em>IEEE Trans Med Imaging</em>, 20(12):1251-60, Dec 
2001. [&nbsp;bib&nbsp;] 
<blockquote>  Adenomatous polyps in the colon are believed to be the precursor 
to colorectal carcinoma, the second leading cause of cancer deaths in United 
States. In this paper, we propose a new method for computer-aided detection of 
polyps in computed tomography (CT) colonography (virtual colonoscopy), a 
technique in which polyps are imaged along the wall of the air-inflated, 
cleansed colon with X-ray CT. Initial work with computer aided detection has 
shown high sensitivity, but at a cost of too many false positives. We present a 
statistical approach that uses support vector machines to distinguish the 
differentiating characteristics of polyps and healthy tissue, and uses this 
information for the classification of the new cases. One of the main 
contributions of the paper is the new three-dimensional pattern processing 
approach, called random orthogonal shape sections method, which combines the 
information from many random images to generate reliable signatures of shape. 
The input to the proposed system is a collection of volume data from candidate 
polyps obtained by a high-sensitivity, low-specificity system that we developed 
previously. The results of our ten-fold cross-validation experiments show that, 
on the average, the system increases the specificity from 0.19 (0.35) to 0.69 
(0.74) at a sensitivity level of 1.0 (0.95). </blockquote> 
<p></p> 
<blockquote>  Keywords: 80 and over, Adult, Aged, Algorithms, Amino Acid 
Sequence, Artificial Intelligence, Automated, Binding Sites, Cell Surface, 
Cluster Analysis, Colonic Polyps, Colonography, Comparative Study, 
Computational Biology, Computed Tomographic, Computer-Assisted, Databases, 
Diagnosis, Differential, Female, GTP-Binding Proteins, Gene Expression 
Profiling, Genetic, Humans, Imaging, Internet, Male, Middle Aged, Models, 
Molecular Sequence Data, Molecular Structure, 
N-Acetylgalactosaminyltransferases, Neural Networks (Computer), Non-U.S. Gov't, 
Oligonucleotide Array Sequence Analysis, P.H.S., Pattern Recognition, Principal 
Component Analysis, Programming Languages, Promoter Regions (Genetics), 
Protein, Proteins, ROC Curve, Radiographic Image Interpretation, Receptors, 
Reproducibility of Results, Research Support, Sensitivity and Specificity, 
Sequence Alignment, Software, Statistical, Substrate Specificity, Theoretical, 
Three-Dimensional, Tomography, Transcription, U.S. Gov't, X-Ray Computed, 
11811825 </blockquote> <br>
 [Dreiseitl2001comparison]  S.&nbsp;Dreiseitl, 
L.&nbsp;Ohno-Machado, H.&nbsp;Kittler, S.&nbsp;Vinterbo, H.&nbsp;Billhardt, and 
M.&nbsp;Binder. A comparison of machine learning methods for the diagnosis of 
pigmented skin lesions.<em>J Biomed Inform</em>, 34(1):28-36, Feb 2001. [&nbsp;
bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  We analyze the discriminatory power of k-nearest neighbors, 
logistic regression, artificial neural networks (ANNs), decision tress, and 
support vector machines (SVMs) on the task of classifying pigmented skin 
lesions as common nevi, dysplastic nevi, or melanoma. Three different 
classification tasks were used as benchmarks: the dichotomous problem of 
distinguishing common nevi from dysplastic nevi and melanoma, the dichotomous 
problem of distinguishing melanoma from common and dysplastic nevi, and the 
trichotomous problem of correctly distinguishing all three classes. Using ROC 
analysis to measure the discriminatory power of the methods shows that 
excellent results for specific classification problems in the domain of 
pigmented skin lesions can be achieved with machine-learning methods. On both 
dichotomous and trichotomous tasks, logistic regression, ANNs, and SVMs 
performed on about the same level, with k-nearest neighbors and decision trees 
performing worse. </blockquote> 
<p></p> 
<blockquote>  Keywords: Algorithms, Amino Acid Sequence, Artificial 
Intelligence, Biological, Cell Compartmentation, Comparative Study, Computer 
Simulation, Computer-Assisted, Decision Trees, Diagnosis, Discriminant 
Analysis, Humans, Logistic Models, Melanoma, Models, Neural Networks 
(Computer), Nevus, Non-U.S. Gov't, Organelles, P.H.S., Pigmented, Predictive 
Value of Tests, Proteins, Reproducibility of Results, Research Support, Skin 
Diseases, Skin Neoplasms, Skin Pigmentation, U.S. Gov't, 11376540 </blockquote> 
<br>  [Chou2001Using]  K.-C. Chou. Using subsite coupling to predict signal 
peptides.<em>Protein Eng.</em>, 14(2):75-79, 2001. [&nbsp;bib&nbsp;| http&nbsp;|
.pdf&nbsp;] <br>
 [Chou2001Prediction]  K.-C. Chou. Prediction of protein 
signal sequences and their cleavage sites.<em>Protein. Struct. Funct. Genet.
</em>, 42:136-139, 2001. [&nbsp;bib&nbsp;| http&nbsp;| .pdf&nbsp;] <br>
 [
Bostroem2001Reproducing]  J.&nbsp;Bostr&ouml;m. Reproducing the conformations 
of protein-bound ligands: a critical evaluation of several popular 
conformational searching tools.<em>J Comput Aided Mol Des</em>, 
15(12):1137-1152, Dec 2001. [&nbsp;bib&nbsp;] 
<blockquote>  Several programs (Catalyst, Confort, Flo99, MacroModel, and 
Omega) that are commonly used to generate conformational ensembles have been 
tested for their ability to reproduce bioactive conformations. The ligands from 
thirty-two different ligand-protein complexes determined by high-resolution 
(&lt; 2.0 A) X-ray crystallography have been analyzed. The Low-Mode 
Conformational Search method (with AMBER* and the GB/SA hydration model), as 
implemented in MacroModel, was found to perform better than the other 
algorithms. The rule-based method Omega, which is orders of magnitude faster 
than the other methods, also gave reasonable results but were found to be 
dependent on the input structure. The methods supporting diverse sampling 
(Catalyst, Confort) performed least well. For the seven ligands in the set 
having eight or more rotatable bonds, none of the bioactive conformations were 
ever found, save for one exception (Flo99). These ligands do not bind in a 
local minimum conformation according to AMBER*/SA. Taking these last two 
observations together, it is clear that geometrically similar structures should 
be collected in order to increase the probability of finding the bioactive 
conformation among the generated ensembles. Factors influencing bioactive 
conformational retrieval have been identified and are discussed. </blockquote> 
<p></p> 
<blockquote>  Keywords: Algorithms; Crystallography, X-Ray; Ligands; Models, 
Molecular; Molecular Conformation; Protein Binding; Quantum Theory; Software 
</blockquote> <br>
 [Ballesteros2001G]  J.&nbsp;Ballesteros and 
K.&nbsp;Palczewski. G protein-coupled receptor drug discovery: implications 
from the crystal structure of rhodopsin.<em>Curr. Opin. Drug Discov. Devel.</em>
, 4(5):561-574, Sep 2001. [&nbsp;bib&nbsp;] 
<blockquote>  G protein-coupled receptors (GPCRs) are a functionally diverse 
group of membrane proteins that play a critical role in signal transduction. 
Because of the lack of a high-resolution structure, the heptahelical 
transmembrane bundle within the N-terminal extracellular and C-terminal 
intracellular region of these receptors has initially been modeled based on the 
high-resolution structure of bacterial retinal-binding protein, 
bacteriorhodopsin. However, the low-resolution structure of rhodopsin, a 
prototypical GPCR, revealed that there is a minor relationship between GPCRs 
and bacteriorhodopsins. The high-resolution crystal structure of the rhodopsin 
ground state and further refinements of the model provide the first structural 
information about the entire organization of the polypeptide chain and 
post-translational moieties. These studies provide a structural template for 
Family 1 GPCRs that has the potential to significantly improve structure-based 
approaches to GPCR drug discovery. </blockquote> 
<p></p> 
<blockquote>  Keywords: Amino Acid Sequence; Animals; Crystallography, X-Ray; 
Drug Design; GTP-Binding Proteins; Humans; Models, Molecular; Molecular 
Sequence Data; Receptors, Drug; Rhodopsin </blockquote> <br>
 [
Opper2001Universal]  M.&nbsp;Opper and R.&nbsp;Urbanczik. Universal learning 
curves of support vector machines.<em>Phys Rev Lett</em>, 86(19):4410-3, May 
2001. [&nbsp;bib&nbsp;] 
<blockquote>  Using methods of statistical physics, we investigate the role of 
model complexity in learning with support vector machines (SVMs), which are an 
important alternative to neural networks. We show the advantages of using SVMs 
with kernels of infinite complexity on noisy target rules, which, in contrast 
to common theoretical beliefs, are found to achieve optimal generalization 
error although the training error does not converge to the generalization 
error. Moreover, we find a universal asymptotics of the learning curves which 
depend only on the target rule but not on the SVM kernel. </blockquote> 
<p></p> 
<blockquote>  Keywords: Algorithms, Amino Acid Sequence, Artificial 
Intelligence, Biological, Cell Compartmentation, Chemistry, Comparative Study, 
Computational Biology, Computer Simulation, Computer-Assisted, Databases, 
Decision Trees, Diagnosis, Discriminant Analysis, Electrophysiology, Factual, 
Gastric Emptying, Humans, Logistic Models, Melanoma, Models, Neural Networks 
(Computer), Nevus, Non-U.S. Gov't, Organelles, P.H.S., Physical, Pigmented, 
Predictive Value of Tests, Proteins, Proteome, Reproducibility of Results, 
Research Support, Skin Diseases, Skin Neoplasms, Skin Pigmentation, Software, 
Stomach Diseases, U.S. Gov't, 11328187 </blockquote> <br>
 [Liang2001Detection] 
 H.&nbsp;Liang and Z.&nbsp;Lin. Detection of delayed gastric emptying from 
electrogastrograms with support vector machine.<em>IEEE Trans Biomed Eng</em>, 
48(5):601-4, May 2001. [&nbsp;bib&nbsp;] 
<blockquote>  A recent study reported a conventional neural network (NN) 
approach for the noninvasive diagnosis of delayed gastric emptying from the 
cutaneous electrogastrograms. Using support vector machine, we show that this 
relatively new technique can be used for detection of delayed gastric emptying 
and is in fact able to outdo the conventional NN. </blockquote> 
<p></p> 
<blockquote>  Keywords: Algorithms, Amino Acid Sequence, Artificial 
Intelligence, Biological, Cell Compartmentation, Comparative Study, Computer 
Simulation, Computer-Assisted, Decision Trees, Diagnosis, Discriminant 
Analysis, Electrophysiology, Gastric Emptying, Humans, Logistic Models, 
Melanoma, Models, Neural Networks (Computer), Nevus, Non-U.S. Gov't, 
Organelles, P.H.S., Pigmented, Predictive Value of Tests, Proteins, 
Reproducibility of Results, Research Support, Skin Diseases, Skin Neoplasms, 
Skin Pigmentation, Stomach Diseases, U.S. Gov't, 11341535 </blockquote> <br>
 [
Yu2002Methods]  Kun Yu, Nikolai Petrovsky, Christian Sch&ouml;nbach, Judice 
Y&nbsp;L Koh, and Vladimir Brusic. Methods for prediction of peptide binding to 
MHC molecules: a comparative study.<em>Mol Med</em>, 8(3):137-148, Mar 2002. 
[&nbsp;bib&nbsp;] 
<blockquote>  BACKGROUND: A variety of methods for prediction of peptide 
binding to major histocompatibility complex (MHC) have been proposed. These 
methods are based on binding motifs, binding matrices, hidden Markov models 
(HMM), or artificial neural networks (ANN). There has been little prior work on 
the comparative analysis of these methods. MATERIALS AND METHODS: We performed 
a comparison of the performance of six methods applied to the prediction of two 
human MHC class I molecules, including binding matrices and motifs, ANNs, and 
HMMs. RESULTS: The selection of the optimal prediction method depends on the 
amount of available data (the number of peptides of known binding affinity to 
the MHC molecule of interest), the biases in the data set and the intended 
purpose of the prediction (screening of a single protein versus mass 
screening). When little or no peptide data are available, binding motifs are 
the most useful alternative to random guessing or use of a complete overlapping 
set of peptides for selection of candidate binders. As the number of known 
peptide binders increases, binding matrices and HMM become more useful 
predictors. ANN and HMM are the predictive methods of choice for MHC alleles 
with more than 100 known binding peptides. CONCLUSION: The ability of 
bioinformatic methods to reliably predict MHC binding peptides, and thereby 
potential T-cell epitopes, has major implications for clinical immunology, 
particularly in the area of vaccine design. </blockquote> 
<p></p> 
<blockquote>  Keywords: Amino Acid Motifs; Computational Biology; 
Histocompatibility Antigens Class I; Humans; Models, Molecular; Peptides; 
Protein Binding </blockquote> <br>
 [Weber2002Building]  Griffin Weber, Staal 
Vinterbo, and Lucila Ohno-Machado. Building an asynchronous web-based tool for 
machine learning classification.<em>Proc AMIA Symp</em>, pages 869-73, 2002. 
[&nbsp;bib&nbsp;] 
<blockquote>  Various unsupervised and supervised learning methods including 
support vector machines, classification trees, linear discriminant analysis and 
nearest neighbor classifiers have been used to classify high-throughput gene 
expression data. Simpler and more widely accepted statistical tools have not 
yet been used for this purpose, hence proper comparisons between classification 
methods have not been conducted. We developed free software that implements 
logistic regression with stepwise variable selection as a quick and simple 
method for initial exploration of important genetic markers in disease 
classification. To implement the algorithm and allow our collaborators in 
remote locations to evaluate and compare its results against those of other 
methods, we developed a user-friendly asynchronous web-based application with a 
minimal amount of programming using free, downloadable software tools. With 
this program, we show that classification using logistic regression can perform 
as well as other more sophisticated algorithms, and it has the advantages of 
being easy to interpret and reproduce. By making the tool freely and easily 
available, we hope to promote the comparison of classification methods. In 
addition, we believe our web application can be used as a model for other 
bioinformatics laboratories that need to develop web-based analysis tools in a 
short amount of time and on a limited budget. </blockquote> 
<p></p> 
<blockquote>  Keywords: Acute, Algorithms, Animals, Artificial Intelligence, 
Automated, Base Pair Mismatch, Base Pairing, Base Sequence, Biological, 
Biosensing Techniques, Classification, Cluster Analysis, Comparative Study, 
Computational Biology, Computer-Assisted, Cystadenoma, DNA, Drug, Drug Design, 
Eukaryotic Cells, Female, Gene Expression, Gene Expression Profiling, Gene 
Expression Regulation, Genes, Genetic, Genetic Markers, Hemolysins, Humans, 
Internet, Leukemia, Ligands, Likelihood Functions, Logistic Models, 
Lymphocytic, Markov Chains, Mathematics, Messenger, Models, Molecular, 
Molecular Probe Techniques, Molecular Sequence Data, Nanotechnology, Neoplasm, 
Neoplasms, Neoplastic, Neural Networks (Computer), Non-P.H.S., Non-U.S. Gov't, 
Nucleic Acid Conformation, Observer Variation, Oligonucleotide Array Sequence 
Analysis, Ovarian Neoplasms, P.H.S., Pattern Recognition, Probability, Protein 
Binding, Proteins, Quality Control, RNA, RNA Splicing, Receptors, Reference 
Values, Reproducibility of Results, Research Support, Sensitivity and 
Specificity, Sequence Analysis, Signal Processing, Software, Statistical, 
Stomach Neoplasms, Thermodynamics, Transcription, Tumor Markers, U.S. Gov't, 
12463949 </blockquote> <br>
 [Wahba2002Soft]  Grace Wahba. Soft and hard 
classification by reproducing kernel Hilbert space methods.<em>Proc Natl Acad 
Sci U S A</em>, 99(26):16524-30, Dec 2002. [&nbsp;bib&nbsp;| DOI&nbsp;| http
&nbsp;|.pdf&nbsp;] 
<blockquote>  Reproducing kernel Hilbert space (RKHS) methods provide a 
unified context for solving a wide variety of statistical modelling and 
function estimation problems. We consider two such problems: We are given a 
training set [yi, ti, i = 1, em leader, n], where yi is the response for the 
ith subject, and ti is a vector of attributes for this subject. The value of 
y(i) is a label that indicates which category it came from. For the first 
problem, we wish to build a model from the training set that assigns to each t 
in an attribute domain of interest an estimate of the probability pj(t) that a 
(future) subject with attribute vector t is in category j. The second problem 
is in some sense less ambitious; it is to build a model that assigns to each t 
a label, which classifies a future subject with that t into one of the 
categories or possibly &quot;none of the above.&quot; The approach to the first 
of these two problems discussed here is a special case of what is known as 
penalized likelihood estimation. The approach to the second problem is known as 
the support vector machine. We also note some alternate but closely related 
approaches to the second problem. These approaches are all obtained as 
solutions to optimization problems in RKHS. Many other problems, in particular 
the solution of ill-posed inverse problems, can be obtained as solutions to 
optimization problems in RKHS and are mentioned in passing. We caution the 
reader that although a large literature exists in all of these topics, in this 
inaugural article we are selectively highlighting work of the author, former 
students, and other collaborators. </blockquote> 
<p></p> 
<blockquote>  Keywords: Acute, Algorithms, Animals, Automated, Base Pair 
Mismatch, Base Pairing, Base Sequence, Biological, Biosensing Techniques, 
Classification, Cluster Analysis, Comparative Study, Computational Biology, 
Computer-Assisted, Cystadenoma, DNA, Drug, Drug Design, Eukaryotic Cells, 
Female, Gene Expression, Gene Expression Profiling, Gene Expression Regulation, 
Genes, Genetic, Genetic Markers, Hemolysins, Humans, Leukemia, Ligands, 
Likelihood Functions, Lymphocytic, Markov Chains, Mathematics, Messenger, 
Models, Molecular, Molecular Probe Techniques, Molecular Sequence Data, 
Nanotechnology, Neoplasm, Neoplastic, Neural Networks (Computer), Non-P.H.S., 
Non-U.S. Gov't, Nucleic Acid Conformation, Observer Variation, Oligonucleotide 
Array Sequence Analysis, Ovarian Neoplasms, P.H.S., Pattern Recognition, 
Probability, Protein Binding, Proteins, Quality Control, RNA, RNA Splicing, 
Receptors, Reference Values, Reproducibility of Results, Research Support, 
Sensitivity and Specificity, Sequence Analysis, Signal Processing, Statistical, 
Stomach Neoplasms, Thermodynamics, Transcription, Tumor Markers, U.S. Gov't, 
12477931 </blockquote> <br>
 [Tucker2002Gene]  D.&nbsp;L. Tucker, 
N.&nbsp;Tucker, and T.&nbsp;Conway. Gene expression profiling of the ph 
response in escherichia coli.<em>J Bacteriol.</em>, 184(23):6551-6558, Dec 
2002. [&nbsp;bib&nbsp;] 
<blockquote>  Escherichia coli MG1655 acid-inducible genes were identified by 
whole-genome expression profiling. Cultures were grown to the mid-logarithmic 
phase on acidified glucose minimal medium, conditions that induce 
glutamate-dependent acid resistance (AR), while the other AR systems are either 
repressed or not induced. A total of 28 genes were induced in at least two of 
three experiments in which the gene expression profiles of cells grown in acid 
(pH 5.5 or 4.5) were compared to those of cells grown at pH 7.4. As expected, 
the genes encoding glutamate decarboxylase, gadA and gadB, were significantly 
induced. Interestingly, two acid-inducible genes code for small basic proteins 
with pIs of &gt;10.5, and six code for small acidic proteins with pIs ranging 
from 5.7 to 4.0; the roles of these small basic and acidic proteins in acid 
resistance are unknown. The acid-induced genes represented only five functional 
grouping categories, including eight genes involved in metabolism, nine 
associated with cell envelope structures or modifications, two encoding 
chaperones, six regulatory genes, and six unknown genes. It is unlikely that 
all of these genes are involved in the glutamate-dependent AR. However, nine 
acid-inducible genes are clustered in the gadA region, including hdeA, which 
encodes a putative periplasmic chaperone, and four putative regulatory genes. 
One of these putative regulators, yhiE, was shown to significantly increase 
acid resistance when overexpressed in cells that had not been preinduced by 
growth at pH 5.5, and mutation of yhiE decreased acid resistance; yhiE could 
therefore encode an activator of AR genes. Thus, the acid-inducible genes 
clustered in the gadA region appear to be involved in glutatmate-dependent acid 
resistance, although their specific roles remain to be elucidated. </blockquote>
<p></p> 
<blockquote>  Keywords: Culture Media; Escherichia coli; Escherichia coli 
Proteins; Gene Expression Profiling; Gene Expression Regulation, Bacterial; 
Heat-Shock Response; Hydrogen-Ion Concentration; Morpholines; Oligonucleotide 
Array Sequence Analysis </blockquote> <br>
 [Sultan2002Binary]  M.&nbsp;Sultan, 
D.&nbsp;A. Wigle, C.&nbsp;A. Cumbaa, M.&nbsp;Maziarz, J.&nbsp;Glasgow, 
M.&nbsp;S. Tsao, and I.&nbsp;Jurisica. Binary tree-structured vector 
quantization approach to clustering and visualizing microarray data.<em>
Bioinformatics</em>, 18 Suppl 1:S111-9, 2002. [&nbsp;bib&nbsp;] 
<blockquote>  MOTIVATION: With the increasing number of gene expression 
databases, the need for more powerful analysis and visualization tools is 
growing. Many techniques have successfully been applied to unravel latent 
similarities among genes and/or experiments. Most of the current systems for 
microarray data analysis use statistical methods, hierarchical clustering, 
self-organizing maps, support vector machines, or k-means clustering to 
organize genes or experiments into 'meaningful' groups. Without prior explicit 
bias almost all of these clustering methods applied to gene expression data not 
only produce different results, but may also produce clusters with little or no 
biological relevance. Of these methods, agglomerative hierarchical clustering 
has been the most widely applied, although many limitations have been 
identified. RESULTS: Starting with a systematic comparison of the underlying 
theories behind clustering approaches, we have devised a technique that 
combines tree-structured vector quantization and partitive k-means clustering 
(BTSVQ). This hybrid technique has revealed clinically relevant clusters in 
three large publicly available data sets. In contrast to existing systems, our 
approach is less sensitive to data preprocessing and data normalization. In 
addition, the clustering results produced by the technique have strong 
similarities to those of self-organizing maps (SOMs). We discuss the advantages 
and the mathematical reasoning behind our approach. </blockquote> 
<p></p> 
<blockquote>  Keywords: Algorithms, Amino Acid Sequence, Artificial 
Intelligence, Automated, Carcinoma, Chemical, Cluster Analysis, Comparative 
Study, Computer Graphics, DNA Gyrase, Evolution, Gene Expression Profiling, 
Gene Expression Regulation, Genetic, Humans, Lung Neoplasms, Markov Chains, 
Models, Molecular, Molecular Sequence Data, Non-Small-Cell Lung, Non-U.S. 
Gov't, Oligonucleotide Array Sequence Analysis, Pattern Recognition, Phylogeny, 
Proteins, Reproducibility of Results, Research Support, Saccharomyces 
cerevisiae, Saccharomyces cerevisiae Proteins, Sensitivity and Specificity, 
Sequence Alignment, Sequence Analysis, Sequence Homology, Software, 
Statistical, Subcellular Fractions, User-Computer Interface, 12169538 
</blockquote> <br>
 [Sturn2002Genesis:]  Alexander Sturn, John Quackenbush, and 
Zlatko Trajanoski. Genesis: cluster analysis of microarray data.<em>
Bioinformatics</em>, 18(1):207-8, Jan 2002. [&nbsp;bib&nbsp;] 
<blockquote>  A versatile, platform independent and easy to use Java suite for 
large-scale gene expression analysis was developed. Genesis integrates various 
tools for microarray data analysis such as filters, normalization and 
visualization tools, distance measures as well as common clustering algorithms 
including hierarchical clustering, self-organizing maps, k-means, principal 
component analysis, and support vector machines. The results of the clustering 
are transparent across all implemented methods and enable the analysis of the 
outcome of different algorithms and parameters. Additionally, mapping of gene 
expression data onto chromosomal sequences was implemented to enhance promoter 
analysis and investigation of transcriptional control mechanisms. </blockquote> 
<p></p> 
<blockquote>  Keywords: Algorithms, Artificial Intelligence, Cluster Analysis, 
Comparative Study, Computational Biology, Databases, Gene Expression Profiling, 
Genetic, Models, Molecular Structure, Neural Networks (Computer), Non-U.S. 
Gov't, Oligonucleotide Array Sequence Analysis, Principal Component Analysis, 
Programming Languages, Promoter Regions (Genetics), Protein, Proteins, Research 
Support, Software, Statistical, Transcription, 11836235 </blockquote> <br>
 [
Song2002Prediction]  Minghu Song, Curt&nbsp;M Breneman, Jinbo Bi, 
N.&nbsp;Sukumar, Kristin&nbsp;P Bennett, Steven Cramer, and Nihal Tugcu. 
Prediction of protein retention times in anion-exchange chromatography systems 
using support vector regression.<em>J Chem Inf Comput Sci</em>, 42(6):1347-57, 
2002. [&nbsp;bib&nbsp;] 
<blockquote>  Quantitative Structure-Retention Relationship (QSRR) models are 
developed for the prediction of protein retention times in anion-exchange 
chromatography systems. Topological, subdivided surface area, and TAE 
(Transferable Atom Equivalent) electron-density-based descriptors are computed 
directly for a set of proteins using molecular connectivity patterns and 
crystal structure geometries. A novel algorithm based on Support Vector Machine 
(SVM) regression has been employed to obtain predictive QSRR models using a 
two-step computational strategy. In the first step, a sparse linear SVM was 
utilized as a feature selection procedure to remove irrelevant or redundant 
information. Subsequently, the selected features were used to produce an 
ensemble of nonlinear SVM regression models that were combined using bootstrap 
aggregation (bagging) techniques, where various combinations of training and 
validation data sets were selected from the pool of available data. A 
visualization scheme (star plots) was used to display the relative importance 
of each selected descriptor in the final set of &quot;bagged&quot; models. Once 
these predictive models have been validated, they can be used as an automated 
prediction tool for virtual high-throughput screening (VHTS). </blockquote> 
<p></p> 
<blockquote>  Keywords: Acute, Algorithms, Animals, Anion Exchange Resins, 
Artificial Intelligence, Automated, Base Pair Mismatch, Base Pairing, Base 
Sequence, Biological, Biosensing Techniques, Carcinoma, Chemical, 
Chromatography, Classification, Cluster Analysis, Comparative Study, 
Computational Biology, Computer-Assisted, Cystadenoma, DNA, Decision Making, 
Diagnosis, Differential, Drug, Drug Design, Electrostatics, Eukaryotic Cells, 
Feasibility Studies, Female, Gene Expression, Gene Expression Profiling, Gene 
Expression Regulation, Genes, Genetic, Genetic Markers, Hemolysins, Humans, 
Internet, Ion Exchange, Leukemia, Ligands, Likelihood Functions, Logistic 
Models, Lung Neoplasms, Lymphocytic, Lymphoma, Markov Chains, Mathematics, 
Messenger, Models, Molecular, Molecular Probe Techniques, Molecular Sequence 
Data, Nanotechnology, Neoplasm, Neoplasms, Neoplastic, Neural Networks 
(Computer), Non-P.H.S., Non-Small-Cell Lung, Non-U.S. Gov't, Nucleic Acid 
Conformation, Nucleic Acid Hybridization, Observer Variation, Oligonucleotide 
Array Sequence Analysis, Ovarian Neoplasms, P.H.S., Pattern Recognition, 
Probability, Protein Binding, Protein Conformation, Proteins, Quality Control, 
Quantum Theory, RNA, RNA Splicing, Receptors, Reference Values, Regression 
Analysis, Reproducibility of Results, Research Support, Sensitivity and 
Specificity, Sequence Analysis, Signal Processing, Software, Statistical, 
Stomach Neoplasms, Thermodynamics, Transcription, Tumor Markers, U.S. Gov't, 
12444731 </blockquote> <br>
 [Schmitt2002New]  Stefan Schmitt, Daniel Kuhn, and 
Gerhard Klebe. A new method to detect related function among proteins 
independent of sequence and fold homology.<em>J. Mol. Biol.</em>, 
323(2):387-406, Oct 2002. [&nbsp;bib&nbsp;] 
<blockquote>  A new method has been developed to detect functional 
relationships among proteins independent of a given sequence or fold homology. 
It is based on the idea that protein function is intimately related to the 
recognition and subsequent response to the binding of a substrate or an 
endogenous ligand in a well-characterized binding pocket. Thus, recognition of 
similar ligands, supposedly linked to similar function, requires conserved 
recognition features exposed in terms of common physicochemical interaction 
properties via the functional groups of the residues flanking a particular 
binding cavity. Following a technique commonly used in the comparison of small 
molecule ligands, generic pseudocenters coding for possible interaction 
properties were assigned for a large sample set of cavities extracted from the 
entire PDB and stored in the database Cavbase. Using a particular query cavity 
a series of related cavities of decreasing similarity is detected based on a 
clique detection algorithm. The detected similarity is ranked according to 
property-based surface patches shared in common by the different clique 
solutions. The approach either retrieves protein cavities accommodating the 
same (e.g. co-factors) or closely related ligands or it extracts proteins 
exhibiting similar function in terms of a related catalytic mechanism. Finally 
the new method has strong potential to suggest alternative molecular skeletons 
in de novo design. The retrieval of molecular building blocks accommodated in a 
particular sub-pocket that shares similarity with the pocket in a protein 
studied by drug design can inspire the discovery of novel ligands. </blockquote>
<p></p> 
<blockquote>  Keywords: Algorithms; Binding Sites; Databases, Protein; Models, 
Molecular; Molecular Structure; Protein Binding; Protein Folding; Protein 
Structure, Tertiary; Proteins, chemistry/metabolism; Reproducibility of Results 
</blockquote> <br>
 [Sample2002Using]  Pamela&nbsp;A Sample, Michael&nbsp;H 
Goldbaum, Kwokleung Chan, Catherine Boden, Te-Won Lee, Christiana Vasile, 
Andreas&nbsp;G Boehm, Terrence Sejnowski, Chris&nbsp;A Johnson, and 
Robert&nbsp;N Weinreb. Using machine learning classifiers to identify 
glaucomatous change earlier in standard visual fields.<em>Invest Ophthalmol Vis 
Sci</em>, 43(8):2660-5, Aug 2002. [&nbsp;bib&nbsp;] 
<blockquote>  PURPOSE: To compare the ability of several machine learning 
classifiers to predict development of abnormal fields at follow-up in ocular 
hypertensive (OHT) eyes that had normal visual fields in baseline examination. 
METHODS: The visual fields of 114 eyes of 114 patients with OHT with four or 
more visual field tests with standard automated perimetry over three or more 
years and for whom stereophotographs were available were assessed. The mean 
(+/-SD) number of visual field tests was 7.89 +/- 3.04. The mean number of 
years covered (+/-SD) was 5.92 +/- 2.34 (range, 2.81-11.77). Fields were 
classified as normal or abnormal based on Statpac-like methods (Humphrey 
Instruments, Dublin, CA) and by several machine learning classifiers. The 
machine learning classifiers were two types of support vector machine (SVM), a 
mixture of Gaussian (MoG) classifier, a constrained MoG, and a mixture of 
generalized Gaussian (MGG). Specificity was set to 96% for all classifiers, 
using data from 94 normal eyes evaluated longitudinally. Specificity cutoffs 
required confirmation of abnormality. RESULTS: Thirty-two percent (36/114) of 
the eyes converted to abnormal fields during follow-up based on the 
Statpac-like methods. All 36 were identified by at least one machine 
classifier. In nearly all cases, the machine learning classifiers predicted the 
confirmed abnormality, on average, 3.92 +/- 0.55 years earlier than traditional 
Statpac-like methods. CONCLUSIONS: Machine learning classifiers can learn 
complex patterns and trends in data and adapt to create a decision surface 
without the constraints imposed by statistical classifiers. This adaptation 
allowed the machine learning classifiers to identify abnormality in visual 
field converts much earlier than the traditional methods. </blockquote> 
<p></p> 
<blockquote>  Keywords: Algorithms, Amino Acid Sequence, Artificial 
Intelligence, Automated, Carcinoma, Chemical, Cluster Analysis, Comparative 
Study, Computational Biology, Computer Graphics, Computer-Assisted, DNA Gyrase, 
Databases, Diagnosis, Evolution, Factual, Follow-Up Studies, Gene Expression 
Profiling, Gene Expression Regulation, Genetic, Glaucoma, Humans, Intraocular 
Pressure, Lung Neoplasms, Markov Chains, Models, Molecular, Molecular Sequence 
Data, Neural Networks (Computer), Non-Small-Cell Lung, Non-U.S. Gov't, Ocular 
Hypertension, Oligonucleotide Array Sequence Analysis, Optic Disk, Optic Nerve 
Diseases, P.H.S., Pattern Recognition, Perimetry, Photography, Phylogeny, 
Protein Structure, Proteins, Reproducibility of Results, Research Support, 
Saccharomyces cerevisiae, Saccharomyces cerevisiae Proteins, Secondary, 
Sensitivity and Specificity, Sequence Alignment, Sequence Analysis, Sequence 
Homology, Software, Statistical, Subcellular Fractions, U.S. Gov't, 
User-Computer Interface, Vision Disorders, Visual Fields, 12147600 </blockquote>
<br>  [Prakash2002Fetal]  K.&nbsp;N&nbsp;Bhanu Prakash, A.&nbsp;G. 
Ramakrishnan, S.&nbsp;Suresh, and Teresa W&nbsp;P Chow. Fetal lung maturity 
analysis using ultrasound image features.<em>IEEE Trans Inf Technol Biomed</em>
, 6(1):38-45, Mar 2002. [&nbsp;bib&nbsp;] 
<blockquote>  This pilot study was carried out to find the feasibility of 
analyzing the maturity of the fetal lung using ultrasound images. Data were 
collected from normal pregnant women at intervals of two weeks from the 
gestation age of 24 to 38 weeks. Images were acquired at two centers located at 
different geographical locations. The total data acquired consisted of 750 
images of immature and 250 images of mature class. A region of interest of 64 x 
64 pixels was used for extracting the features. Various textural features were 
computed from the fetal lung and liver images. The ratios of fetal lung to 
liver feature values were investigated as possible indexes for classifying the 
images into those from mature (reduced pulmonary risk) and immature (possible 
pulmonary risk) lung. The features used are fractal dimension, lacunarity, and 
features derived from the histogram of the images. The following classifiers 
were used to classify the fetal lung images as belonging to mature or immature 
lung: nearest neighbor, k-nearest neighbor, modified k-nearest neighbor, 
multilayer perceptron, radial basis function network, and support vector 
machines. The classification accuracy obtained for the testing set ranges from 
73% to 96%. </blockquote> 
<p></p> 
<blockquote>  Keywords: Algorithms, Amino Acid Sequence, Antineoplastic 
Agents, Artificial Intelligence, Automated, Bacterial, Base Composition, Base 
Sequence, Bayes Theorem, Bias (Epidemiology), Carcinoma, Cell Line, Chemical, 
Chromosome Aberrations, Cluster Analysis, Codon, Comparative Study, 
Computational Biology, Computer Graphics, Computer Simulation, 
Computer-Assisted, Conserved Sequence, DNA Gyrase, Databases, Diagnosis, 
Electrostatics, Escherichia coli, Evolution, Factual, Female, Follow-Up 
Studies, Fungal, Gene Dosage, Gene Expression, Gene Expression Profiling, Gene 
Expression Regulation, Genes, Genetic, Genetic Vectors, Genomics, Glaucoma, 
Humans, Hydrophobicity, Immunoglobulins, Information Theory, Intraocular 
Pressure, Least-Squares Analysis, Linear Models, Liposarcoma, Lung, Lung 
Neoplasms, Markov Chains, Models, Molecular, Molecular Sequence Data, Multigene 
Family, Mycobacterium tuberculosis, Neural Networks (Computer), Non-P.H.S., 
Non-Small-Cell Lung, Non-U.S. Gov't, Normal Distribution, Nucleic Acid 
Hybridization, Ocular Hypertension, Oligonucleotide Array Sequence Analysis, 
Oncogenes, Optic Disk, Optic Nerve Diseases, P.H.S., Pattern Recognition, 
Perimetry, Photography, Phylogeny, Pilot Projects, Polymerase Chain Reaction, 
Pregnancy, Pregnancy Trimester, Protein, Protein Structure, Proteins, 
Reproducibility of Results, Research Suppor, Research Support, Ribosomal 
Proteins, Saccharomyces cerevisiae, Saccharomyces cerevisiae Proteins, Second, 
Secondary, Sensitivity and Specificity, Sequence Alignment, Sequence Analysis, 
Sequence Homology, Soft Tissue Neoplasms, Software, Solvents, Statistical, 
Subcellular Fractions, Theoretical, Third, Tumor Suppressor, U.S. Gov't, 
User-Computer Interface, Vision Disorders, Visual Fields, kappa-Chain, t, 
11936595 </blockquote> <br>
 [Pastor-Satorras2002Evolving]  
R.&nbsp;Pastor-Satorras, E.&nbsp;D. Smith, and R.&nbsp;V. Sol&eacute;. Evolving 
protein interaction networks through gene duplication. Technical report, Santa 
Fe Institute, 2002. Working paper 02-02-008. [&nbsp;bib&nbsp;| .html&nbsp;| .pdf
&nbsp;] <br>
 [Mateos2002Systematic]  Alvaro Mateos, Joaqu&Atilde;&Acirc;&shy;n 
Dopazo, Ronald Jansen, Yuhai Tu, Mark Gerstein, and Gustavo Stolovitzky. 
Systematic learning of gene functional classes from DNA array expression data 
by using multilayer perceptrons.<em>Genome Res.</em>, 12(11):1703-15, Nov 2002. 
[&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  Recent advances in microarray technology have opened new ways 
for functional annotation of previously uncharacterised genes on a genomic 
scale. This has been demonstrated by unsupervised clustering of co-expressed 
genes and, more importantly, by supervised learning algorithms. Using prior 
knowledge, these algorithms can assign functional annotations based on more 
complex expression signatures found in existing functional classes. Previously, 
support vector machines (SVMs) and other machine-learning methods have been 
applied to a limited number of functional classes for this purpose. Here we 
present, for the first time, the comprehensive application of supervised neural 
networks (SNNs) for functional annotation. Our study is novel in that we report 
systematic results for &nbsp;100 classes in the Munich Information Center for 
Protein Sequences (MIPS) functional catalog. We found that only &nbsp;10% of 
these are learnable (based on the rate of false negatives). A closer analysis 
reveals that false positives (and negatives) in a machine-learning context are 
not necessarily &quot;false&quot; in a biological sense. We show that the high 
degree of interconnections among functional classes confounds the signatures 
that ought to be learned for a unique class. We term this the &quot;Borges 
effect&quot; and introduce two new numerical indices for its quantification. 
Our analysis indicates that classification systems with a lower Borges effect 
are better suitable for machine learning. Furthermore, we introduce a learning 
procedure for combining false positives with the original class. We show that 
in a few iterations this process converges to a gene set that is learnable with 
considerably low rates of false positives and negatives and contains genes that 
are biologically related to the original class, allowing for a coarse 
reconstruction of the interactions between associated biological pathways. We 
exemplify this methodology using the well-studied tricarboxylic acid cycle. 
</blockquote> 
<p></p> 
<blockquote>  Keywords: Acute, Algorithms, Animals, Anion Exchange Resins, 
Artificial Intelligence, Automated, Base Pair Mismatch, Base Pairing, Base 
Sequence, Biological, Biosensing Techniques, Carcinoma, Chemical, 
Chromatography, Citric Acid Cycle, Classification, Cluster Analysis, 
Comparative Study, Computational Biology, Computer-Assisted, Cystadenoma, DNA, 
Databases, Decision Making, Diagnosis, Differential, Drug, Drug Design, 
Electrostatics, Eukaryotic Cells, Factual, Feasibility Studies, Female, Gene 
Expression, Gene Expression Profiling, Gene Expression Regulation, Genes, 
Genetic, Genetic Heterogeneity, Genetic Markers, Hemolysins, Humans, Internet, 
Ion Exchange, Leukemia, Ligands, Likelihood Functions, Logistic Models, Lung 
Neoplasms, Lymphocytic, Lymphoma, Markov Chains, Mathematics, Messenger, 
Models, Molecular, Molecular Probe Techniques, Molecular Sequence Data, 
Nanotechnology, Neoplasm, Neoplasms, Neoplastic, Neural Networks (Computer), 
Non-P.H.S., Non-Small-Cell Lung, Non-U.S. Gov't, Nucleic Acid Conformation, 
Nucleic Acid Hybridization, Observer Variation, Oligonucleotide Array Sequence 
Analysis, Ovarian Neoplasms, P.H.S., Pattern Recognition, Probability, Protein 
Binding, Protein Conformation, Proteins, Quality Control, Quantum Theory, RNA, 
RNA Splicing, Receptors, Reference Values, Regression Analysis, Reproducibility 
of Results, Research Support, Saccharomyces cerevisiae Proteins, Sensitivity 
and Specificity, Sequence Analysis, Signal Processing, Software, Statistical, 
Stomach Neoplasms, Structural, Structure-Activity Relationship, Thermodynamics, 
Transcription, Tumor Markers, U.S. Gov't, 12421757 </blockquote> <br>
 [
Marsland2002self-organising]  Stephen Marsland, Jonathan Shapiro, and Ulrich 
Nehmzow. A self-organising network that grows when required.<em>Neural Netw</em>
, 15(8-9):1041-58, 2002. [&nbsp;bib&nbsp;] 
<blockquote>  The ability to grow extra nodes is a potentially useful facility 
for a self-organising neural network. A network that can add nodes into its map 
space can approximate the input space more accurately, and often more 
parsimoniously, than a network with predefined structure and size, such as the 
Self-Organising Map. In addition, a growing network can deal with dynamic input 
distributions. Most of the growing networks that have been proposed in the 
literature add new nodes to support the node that has accumulated the highest 
error during previous iterations or to support topological structures. This 
usually means that new nodes are added only when the number of iterations is an 
integer multiple of some pre-defined constant, A. This paper suggests a way in 
which the learning algorithm can add nodes whenever the network in its current 
state does not sufficiently match the input. In this way the network grows very 
quickly when new data is presented, but stops growing once the network has 
matched the data. This is particularly important when we consider dynamic data 
sets, where the distribution of inputs can change to a new regime after some 
time. We also demonstrate the preservation of neighbourhood relations in the 
data by the network. The new network is compared to an existing growing 
network, the Growing Neural Gas (GNG), on a artificial dataset, showing how the 
network deals with a change in input distribution after some time. Finally, the 
new network is applied to several novelty detection tasks and is compared with 
both the GNG and an unsupervised form of the Reduced Coulomb Energy network on 
a robotic inspection task and with a Support Vector Machine on two benchmark 
novelty detection tasks. </blockquote> 
<p></p> 
<blockquote>  Keywords: Acute, Algorithms, Animals, Anion Exchange Resins, 
Artificial Intelligence, Automated, Base Pair Mismatch, Base Pairing, Base 
Sequence, Biological, Biosensing Techniques, Carcinoma, Chemical, 
Chromatography, Citric Acid Cycle, Classification, Cluster Analysis, 
Comparative Study, Computational Biology, Computer-Assisted, Cystadenoma, DNA, 
Databases, Decision Making, Diagnosis, Differential, Drug, Drug Design, 
Electrostatics, Eukaryotic Cells, Factual, Feasibility Studies, Female, Gene 
Expression, Gene Expression Profiling, Gene Expression Regulation, Genes, 
Genetic, Genetic Heterogeneity, Genetic Markers, Hemolysins, Humans, Internet, 
Ion Exchange, Leukemia, Ligands, Likelihood Functions, Logistic Models, Lung 
Neoplasms, Lymphocytic, Lymphoma, Markov Chains, Mathematics, Messenger, 
Models, Molecular, Molecular Probe Techniques, Molecular Sequence Data, 
Nanotechnology, Neoplasm, Neoplasms, Neoplastic, Neural Networks (Computer), 
Non-P.H.S., Non-Small-Cell Lung, Non-U.S. Gov't, Nucleic Acid Conformation, 
Nucleic Acid Hybridization, Observer Variation, Oligonucleotide Array Sequence 
Analysis, Ovarian Neoplasms, P.H.S., Pattern Recognition, Probability, 
Probability Learning, Protein Binding, Protein Conformation, Proteins, Quality 
Control, Quantum Theory, RNA, RNA Splicing, Receptors, Reference Values, 
Regression Analysis, Reproducibility of Results, Research Support, Robotics, 
Saccharomyces cerevisiae Proteins, Sensitivity and Specificity, Sequence 
Analysis, Signal Processing, Software, Statistical, Stomach Neoplasms, 
Structural, Structure-Activity Relationship, Thermodynamics, Transcription, 
Tumor Markers, U.S. Gov't, 12416693 </blockquote> <br>
 [Ho2002Systematic]  
Yuen Ho, Albrecht Gruhler, Adrian Heilbut, Gary&nbsp;D Bader, Lynda Moore, 
Sally-Lin Adams, Anna Millar, Paul Taylor, Keiryn Bennett, Kelly Boutilier, 
Lingyun Yang, Cheryl Wolting, Ian Donaldson, S&Atilde;&Acirc;&cedil;ren 
Schandorff, Juanita Shewnarane, Mai Vo, Joanne Taggart, Marilyn Goudreault, 
Brenda Muskat, Cris Alfarano, Danielle Dewar, Zhen Lin, Katerina Michalickova, 
Andrew&nbsp;R Willems, Holly Sassi, Peter&nbsp;A Nielsen, Karina&nbsp;J 
Rasmussen, Jens&nbsp;R Andersen, Lene&nbsp;E Johansen, Lykke&nbsp;H Hansen, 
Hans Jespersen, Alexandre Podtelejnikov, Eva Nielsen, Janne Crawford, Vibeke 
Poulsen, Birgitte&nbsp;D S&Atilde;&Acirc;&cedil;rensen, Jesper Matthiesen, 
Ronald&nbsp;C Hendrickson, Frank Gleeson, Tony Pawson, Michael&nbsp;F Moran, 
Daniel Durocher, Matthias Mann, Christopher W&nbsp;V Hogue, Daniel Figeys, and 
Mike Tyers. Systematic identification of protein complexes in Saccharomyces 
cerevisiae by mass spectrometry.<em>Nature</em>, 415(6868):180-3, Jan 2002. 
[&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  The recent abundance of genome sequence data has brought an 
urgent need for systematic proteomics to decipher the encoded protein networks 
that dictate cellular function. To date, generation of large-scale 
protein-protein interaction maps has relied on the yeast two-hybrid system, 
which detects binary interactions through activation of reporter gene 
expression. With the advent of ultrasensitive mass spectrometric protein 
identification methods, it is feasible to identify directly protein complexes 
on a proteome-wide scale. Here we report, using the budding yeast Saccharomyces 
cerevisiae as a test case, an example of this approach, which we term 
high-throughput mass spectrometric protein complex identification (HMS-PCI). 
Beginning with 10% of predicted yeast proteins as baits, we detected 3,617 
associated proteins covering 25% of the yeast proteome. Numerous protein 
complexes were identified, including many new interactions in various 
signalling pathways and in the DNA damage response. Comparison of the HMS-PCI 
data set with interactions reported in the literature revealed an average 
threefold higher success rate in detection of known complexes compared with 
large-scale two-hybrid studies. Given the high degree of connectivity observed 
in this study, even partial HMS-PCI coverage of complex proteomes, including 
that of humans, should allow comprehensive identification of cellular networks. 
</blockquote> 
<p></p> 
<blockquote>  Keywords: Affinity Labels, Amino Acid Sequence, Animals, Cell 
Cycle Proteins, Cloning, Comparative Study, DNA, DNA Damage, DNA Repair, 
Electrospray Ionization, Fungal, Genetic, Humans, Macromolecular Substances, 
Mass, Mitosis, Molecular, Molecular Sequence Data, Non-P.H.S., Non-U.S. Gov't, 
P.H.S., Phosphoric Monoester Hydrolases, Protein Binding, Protein Interaction 
Mapping, Protein Kinases, Proteome, Proteomics, Research Support, 
Ribonucleoproteins, Ribosomes, Saccharomyces cerevisiae, Saccharomyces 
cerevisiae Proteins, Sequence Alignment, Signal Transduction, Spectrometry, 
Spectrum Analysis, Transcription, U.S. Gov't, 11805813 </blockquote> <br>
 [
Goldbaum2002Comparing]  Michael&nbsp;H Goldbaum, Pamela&nbsp;A Sample, 
Kwokleung Chan, Julia Williams, Te-Won Lee, Eytan Blumenthal, 
Christopher&nbsp;A Girkin, Linda&nbsp;M Zangwill, Christopher Bowd, Terrence 
Sejnowski, and Robert&nbsp;N Weinreb. Comparing machine learning classifiers 
for diagnosing glaucoma from standard automated perimetry.<em>Invest Ophthalmol 
Vis Sci</em>, 43(1):162-9, Jan 2002. [&nbsp;bib&nbsp;] 
<blockquote>  PURPOSE: To determine which machine learning classifier learns 
best to interpret standard automated perimetry (SAP) and to compare the best of 
the machine classifiers with the global indices of STATPAC 2 and with experts 
in glaucoma. METHODS: Multilayer perceptrons (MLP), support vector machines 
(SVM), mixture of Gaussian (MoG), and mixture of generalized Gaussian (MGG) 
classifiers were trained and tested by cross validation on the numerical plot 
of absolute sensitivity plus age of 189 normal eyes and 156 glaucomatous eyes, 
designated as such by the appearance of the optic nerve. The authors compared 
performance of these classifiers with the global indices of STATPAC, using the 
area under the ROC curve. Two human experts were judged against the machine 
classifiers and the global indices by plotting their sensitivity-specificity 
pairs. RESULTS: MoG had the greatest area under the ROC curve of the machine 
classifiers. Pattern SD (PSD) and corrected PSD (CPSD) had the largest areas 
under the curve of the global indices. MoG had significantly greater ROC area 
than PSD and CPSD. Human experts were not better at classifying visual fields 
than the machine classifiers or the global indices. CONCLUSIONS: MoG, using the 
entire visual field and age for input, interpreted SAP better than the global 
indices of STATPAC. Machine classifiers may augment the global indices of 
STATPAC. </blockquote> 
<p></p> 
<blockquote>  Keywords: 80 and over, Adult, Aged, Algorithms, Amino Acid 
Sequence, Artificial Intelligence, Automated, Binding Sites, Cell Surface, 
Chromosomes, Cluster Analysis, Colonic Polyps, Colonography, Comparative Study, 
Computational Biology, Computed Tomographic, Computer-Assisted, Databases, 
Diagnosis, Differential, False Negative Reactions, Female, GTP-Binding 
Proteins, Gene Expression Profiling, Genetic, Glaucoma, Human, Humans, Image 
Cytometry, Image Processing, Imaging, Internet, Karyotyping, Male, Middle Aged, 
Models, Molecular Sequence Data, Molecular Structure, 
N-Acetylgalactosaminyltransferases, Neural Networks (Computer), Non-U.S. Gov't, 
Nucleic Acid Hybridization, Oligonucleotide Array Sequence Analysis, Optic 
Nerve, P.H.S., Pattern Recognition, Perimetry, Photography, Pred, Principal 
Component Analysis, Programming Languages, Promoter Regions (Genetics), 
Protein, Proteins, ROC Curve, Radiographic Image Interpretation, Receptors, 
Reproducibility of Results, Research Support, Sensitivity and Specificity, 
Sequence Alignment, Software, Statistical, Subcellular Fractions, Substrate 
Specificity, Theoretical, Three-Dimensional, Tomography, Transcription, U.S. 
Gov't, Visual Fields, X-Ray Computed, ictive Value of Tests, 11773027 
</blockquote> <br>
 [Gavin2002Functionala]  Anne-Claude Gavin, Markus 
B&ouml;sche, Roland Krause, Paola Grandi, Martina Marzioch, Andreas Bauer, 
J&ouml;rg Schultz, Jens&nbsp;M Rick, Anne-Marie Michon, Cristina-Maria Cruciat, 
Marita Remor, Christian H&ouml;fert, Malgorzata Schelder, Miro Brajenovic, 
Heinz Ruffner, Alejandro Merino, Karin Klein, Manuela Hudak, David Dickson, 
Tatjana Rudi, Volker Gnau, Angela Bauch, Sonja Bastuck, Bettina Huhse, 
Christina Leutwein, Marie-Anne Heurtier, Richard&nbsp;R Copley, Angela 
Edelmann, Erich Querfurth, Vladimir Rybin, Gerard Drewes, Manfred Raida, Tewis 
Bouwmeester, Peer Bork, Bertrand Seraphin, Bernhard Kuster, Gitte Neubauer, and 
Giulio Superti-Furga. Functional organization of the yeast proteome by 
systematic analysis of protein complexes.<em>Nature</em>, 415(6868):141-7, Jan 
2002. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  Most cellular processes are carried out by multiprotein 
complexes. The identification and analysis of their components provides insight 
into how the ensemble of expressed proteins (proteome) is organized into 
functional units. We used tandem-affinity purification (TAP) and mass 
spectrometry in a large-scale approach to characterize multiprotein complexes 
in Saccharomyces cerevisiae. We processed 1,739 genes, including 1,143 human 
orthologues of relevance to human biology, and purified 589 protein assemblies. 
Bioinformatic analysis of these assemblies defined 232 distinct multiprotein 
complexes and proposed new cellular roles for 344 proteins, including 231 
proteins with no previous functional annotation. Comparison of yeast and human 
complexes showed that conservation across species extends from single proteins 
to their molecular environment. Our analysis provides an outline of the 
eukaryotic proteome as a network of protein complexes at a level of 
organization beyond binary interactions. This higher-order map contains 
fundamental biological information and offers the context for a more reasoned 
and informed approach to drug discovery. </blockquote> 
<p></p> 
<blockquote>  Keywords: Affinity, Affinity Labels, Amino Acid Sequence, 
Animals, Cell Cycle Proteins, Cells, Chromatography, Cloning, Comparative 
Study, Cultured, DNA, DNA Damage, DNA Repair, Electrospray Ionization, Fungal, 
Gene Targeting, Genetic, Humans, Macromolecular Substances, Mass, 
Matrix-Assisted Laser Desorption-Ionization, Mitosis, Molecular, Molecular 
Sequence Data, Non-P.H.S., Non-U.S. Gov't, P.H.S., Phosphoric Monoester 
Hydrolases, Protein Binding, Protein Interaction Mapping, Protein Kinases, 
Proteome, Proteomics, Recombinant Fusion Proteins, Research Support, 
Ribonucleoproteins, Ribosomes, Saccharomyces cerevisiae, Saccharomyces 
cerevisiae Proteins, Sensitivity and Specificity, Sequence Alignment, Signal 
Transduction, Species Specificity, Spectrometry, Spectrum Analysis, 
Transcription, U.S. Gov't, 11805813 </blockquote> <br>
 [El-Naqa2002support]  
I.&nbsp;El-Naqa, Y.&nbsp;Yang, M.&nbsp;N. Wernick, N.&nbsp;P. Galatsanos, and 
R.&nbsp;M. Nishikawa. A support vector machine approach for detection of 
microcalcifications.<em>IEEE Trans Med Imaging</em>, 21(12):1552-63, Dec 2002. 
[&nbsp;bib&nbsp;] 
<blockquote>  In this paper, we investigate an approach based on support 
vector machines (SVMs) for detection of microcalcification (MC) clusters in 
digital mammograms, and propose a successive enhancement learning scheme for 
improved performance. SVM is a machine-learning method, based on the principle 
of structural risk minimization, which performs well when applied to data 
outside the training set. We formulate MC detection as a supervised-learning 
problem and apply SVM to develop the detection algorithm. We use the SVM to 
detect at each location in the image whether an MC is present or not. We tested 
the proposed method using a database of 76 clinical mammograms containing 1120 
MCs. We use free-response receiver operating characteristic curves to evaluate 
detection performance, and compare the proposed algorithm with several existing 
methods. In our experiments, the proposed SVM framework outperformed all the 
other methods tested. In particular, a sensitivity as high as 94% was achieved 
by the SVM method at an error rate of one false-positive cluster per image. The 
ability of SVM to out perform several well-known methods developed for the 
widely studied problem of MC detection suggests that SVM is a promising 
technique for object detection in a medical imaging application. </blockquote> 
<p></p> 
<blockquote>  Keywords: Algorithms, Alleles, Amino Acid Sequence, Amino Acids, 
Animals, Antigens, Artificial Intelligence, Automated, Biological, Biological 
Markers, Blood-Brain Barrier, Breast Diseases, Breast Neoplasms, CD36, 
Calcinosis, Carcinoma, Case-Control Studies, Central Nervous System, Chemical, 
Chromosome Aberr, Clear Cell, Cluster Analysis, Comparative Study, 
Computational Biology, Computer-Aided Design, Computer-Assisted, Computing 
Methodologies, Coronary Disease, Cultured, DNA, DNA-Binding Proteins, 
Databases, Diagnosis, Differential, Discriminant Analysis, Disease 
Susceptibility, Drug Design, Expert Systems, Factual, False Positive Reactions, 
Female, Fungal, Gene Expression Profiling, Genome, Genotype, Hamsters, 
Haplotypes, Hepatocellular, Humans, Image Processing, Immunohistochemistry, 
Immunologic, Information Storage and Retrieval, Least-Squares Analysis, 
Lipoprotein, Liver Neoplasms, Local, Lung, Male, Mammary, Mammography, Mass, 
Melanoma, Membrane Proteins, Middle Aged, Models, Molecular Sequence Data, 
Molecular Structure, Neoplasm, Neoplasm Recurrence, Neural Networks (Computer), 
Non-P.H.S., Non-U.S. Gov't, Oligonucleotide Array Sequence Analysis, Organic 
Chemicals, P.H.S., Pattern Recognition, Peptides, Permeability, Pharmaceutical 
Preparations, Polymorphism, Predictive Value of Tests, Protein, Protein 
Conformation, Protein Interaction Mapping, Protein Structure, Proteins, 
Proteomics, PubMed, RNA-Binding Proteins, ROC Curve, Radiographic Image 
Enhancement, Radiographic Image Interpretation, Receptors, Reproducibility of 
Results, Research Support, Saccharomyces cerevisiae, Saccharomyces cerevisiae 
Proteins, Sarcoma, Sensitivity and Specificity, Sequence Analysis, Single 
Nucleotide, Soft Tissue Neoplasms, Spectrum Analysis, Structure-Activity 
Relationship, Tertiary, Tumor Cells, U.S. Gov't, Ultrasonography, ations, 
12588039 </blockquote> <br>
 [Ekins2002Towards]  S.&nbsp;Ekins, 
B.&nbsp;Boulanger, P.&nbsp;W. Swaan, and M.&nbsp;A.&nbsp;Z. Hupcey. Towards a 
new age of virtual ADME/TOX and multidimensional drug discovery.<em>J Comput 
Aided Mol Des</em>, 16(5-6):381-401, 2002. [&nbsp;bib&nbsp;] 
<blockquote>  With the continual pressure to ensure follow-up molecules to 
billion dollar blockbuster drugs, there is a hurdle in profitability and growth 
for pharmaceutical companies in the next decades. With each success and failure 
we increasingly appreciate that a key to the success of synthesized molecules 
through the research and development process is the possession of drug-like 
properties. These properties include an adequate bioactivity as well as 
adequate solubility, an ability to cross critical membranes (intestinal and 
sometimes blood-brain barrier), reasonable metabolic stability and of course 
safety in humans. Dependent on the therapeutic area being investigated it might 
also be desirable to avoid certain enzymes or transporters to circumvent 
potential drug-drug interactions. It may also be important to limit the 
induction of these same proteins that can result in further toxicities. We have 
clearly moved the assessment of in vitro absorption, distribution, metabolism, 
excretion and toxicity (ADME/TOX) parameters much earlier in the discovery 
organization than a decade ago with the inclusion of higher throughput systems. 
We are also now faced with huge amounts of ADME/TOX data for each molecule that 
need interpretation and also provide a valuable resource for generating 
predictive computational models for future drug discovery. The present review 
aims to show what tools exist today for visualizing and modeling ADME/TOX data, 
what tools need to be developed, and how both the present and future tools are 
valuable for virtual filtering using ADME/TOX and bioactivity properties in 
parallel as a viable addition to present practices. </blockquote> 
<p></p> 
<blockquote>  Keywords: ATP-Binding Cassette Transporters, Algorithms, 
Animals, Biological, Biological Availability, Computer Simulation, Drug Design, 
Drug Evaluation, Drug Industry, Gene Expression Profiling, Humans, Models, 
Organic Anion Transporters, P.H.S., Pharmaceutical, Pharmaceutical 
Preparations, Pharmacogenetics, Pharmacokinetics, Preclinical, Proteomics, 
Research Support, Software, Systems Biology, Technology, Toxicity Tests, U.S. 
Gov't, 12489686 </blockquote> <br>
 [Doennes2002Prediction]  Pierre D&ouml;nnes 
and Arne Elofsson. Prediction of MHC class I binding peptides, using SVMHC.<em>
BMC Bioinformatics</em>, 3:25, Sep 2002. [&nbsp;bib&nbsp;] 
<blockquote>  BACKGROUND: T-cells are key players in regulating a specific 
immune response. Activation of cytotoxic T-cells requires recognition of 
specific peptides bound to Major Histocompatibility Complex (MHC) class I 
molecules. MHC-peptide complexes are potential tools for diagnosis and 
treatment of pathogens and cancer, as well as for the development of peptide 
vaccines. Only one in 100 to 200 potential binders actually binds to a certain 
MHC molecule, therefore a good prediction method for MHC class I binding 
peptides can reduce the number of candidate binders that need to be synthesized 
and tested. RESULTS: Here, we present a novel approach, SVMHC, based on support 
vector machines to predict the binding of peptides to MHC class I molecules. 
This method seems to perform slightly better than two profile based methods, 
SYFPEITHI and HLA_BIND. The implementation of SVMHC is quite simple and does 
not involve any manual steps, therefore as more data become available it is 
trivial to provide prediction for more MHC types. SVMHC currently contains 
prediction for 26 MHC class I types from the MHCPEP database or alternatively 6 
MHC class I types from the higher quality SYFPEITHI database. The prediction 
models for these MHC types are implemented in a public web service available at 
http://www.sbc.su.se/svmhc/. CONCLUSIONS: Prediction of MHC class I binding 
peptides using Support Vector Machines, shows high performance and is easy to 
apply to a large number of MHC class I types. As more peptide data are put into 
MHC databases, SVMHC can easily be updated to give prediction for additional 
MHC class I types. We suggest that the number of binding peptides needed for 
SVM training is at least 20 sequences. </blockquote> 
<p></p> 
<blockquote>  Keywords: Animals; Artificial Intelligence; Comparative Study; 
Computational Biology; Databases, Protein; Epitopes, T-Lymphocyte; HLA 
Antigens; Histocompatibility Antigens Class I; Humans; Peptides; Predictive 
Value of Tests; Protein Binding; Research Support, Non-U.S. Gov't; Sensitivity 
and Specificity </blockquote> <br>
 [Chan2002Comparison]  Kwokleung Chan, 
Te-Won Lee, Pamela&nbsp;A Sample, Michael&nbsp;H Goldbaum, Robert&nbsp;N 
Weinreb, and Terrence&nbsp;J Sejnowski. Comparison of machine learning and 
traditional classifiers in glaucoma diagnosis.<em>IEEE Trans Biomed Eng</em>, 
49(9):963-74, Sep 2002. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  Glaucoma is a progressive optic neuropathy with characteristic 
structural changes in the optic nerve head reflected in the visual field. The 
visual-field sensitivity test is commonly used in a clinical setting to 
evaluate glaucoma. Standard automated perimetry (SAP) is a common computerized 
visual-field test whose output is amenable to machine learning. We compared the 
performance of a number of machine learning algorithms with STATPAC indexes 
mean deviation, pattern standard deviation, and corrected pattern standard 
deviation. The machine learning algorithms studied included multilayer 
perceptron (MLP), support vector machine (SVM), and linear (LDA) and quadratic 
discriminant analysis (QDA), Parzen window, mixture of Gaussian (MOG), and 
mixture of generalized Gaussian (MGG). MLP and SVM are classifiers that work 
directly on the decision boundary and fall under the discriminative paradigm. 
Generative classifiers, which first model the data probability density and then 
perform classification via Bayes' rule, usually give deeper insight into the 
structure of the data space. We have applied MOG, MGG, LDA, QDA, and Parzen 
window to the classification of glaucoma from SAP. Performance of the various 
classifiers was compared by the areas under their receiver operating 
characteristic curves and by sensitivities (true-positive rates) at chosen 
specificities (true-negative rates). The machine-learning-type classifiers 
showed improved performance over the best indexes from STATPAC. 
Forward-selection and backward-elimination methodology further improved the 
classification rate and also has the potential to reduce testing time by 
diminishing the number of visual-field location measurements. </blockquote> 
<p></p> 
<blockquote>  Keywords: Acute, Algorithms, Animals, Anion Exchange Resins, 
Artificial Intelligence, Automated, Base Pair Mismatch, Base Pairing, Base 
Sequence, Biological, Biosensing Techniques, Carcinoma, Chemical, 
Chromatography, Citric Acid Cycle, Classification, Cluster Analysis, 
Comparative Study, Computational Biology, Computer-Assisted, Cystadenoma, DNA, 
Databases, Decision Making, Diagnosis, Differential, Discriminant Analysis, 
Drug, Drug Design, Electrostatics, Epitopes, Eukaryotic Cells, Factual, False 
Negative Reactions, False Positive Reactions, Feasibility Studies, Female, Gene 
Expression, Gene Expression Profiling, Gene Expression Regulation, Genes, 
Genetic, Genetic Heterogeneity, Genetic Markers, Glaucoma, HLA Antigens, 
Hemolysins, Histocompatibility Antigens Class I, Humans, Internet, Intraocular 
Pressure, Ion Exchange, Lasers, Leukemia, Ligands, Likelihood Functions, 
Logistic Models, Lung Neoplasms, Lymphocytic, Lymphoma, Markov Chains, 
Mathematics, Messenger, Models, Molecular, Molecular Probe Techniques, 
Molecular Sequence Data, Nanotechnology, Neoplasm, Neoplasms, Neoplastic, 
Neural Networks (Computer), Neurological, Non-P.H.S., Non-Small-Cell Lung, 
Non-U.S. Gov't, Nucleic Acid Conformation, Nucleic Acid Hybridization, Observer 
Variation, Oligonucleotide Array Sequence Analysis, Open-Angle, Ophthalmoscopy, 
Optic Disk, Optic Nerve Diseases, Ovarian Neoplasms, P.H.S., Pattern 
Recognition, Peptides, Perimetry, Predictive Value of Tests, Probability, 
Probability Learning, Protein, Protein Binding, Protein Conformation, Proteins, 
Quality Control, Quantum Theory, RNA, RNA Splicing, ROC Curve, Receptors, 
Reference Values, Regression Analysis, Reproducibility of Results, Research 
Support, Robotics, Saccharomyces cerevisiae Proteins, Sensitivity and 
Specificity, Sequence Analysis, Signal Processing, Software, Statistical, 
Stomach Neoplasms, Structural, Structure-Activity Relationship, T-Lymphocyte, 
Thermodynamics, Transcription, Tumor Markers, U.S. Gov't, 12214886 </blockquote>
<br>  [Brusic2002Prediction]  V.&nbsp;Brusic, N.&nbsp;Petrovsky, G.&nbsp;Zhang, 
and V.&nbsp;B. Bajic. Prediction of promiscuous peptides that bind HLA class I 
molecules.<em>Immunol. Cell Biol.</em>, 80(3):280-285, Jun 2002. [&nbsp;bib
&nbsp;]
<blockquote>  Promiscuous T-cell epitopes make ideal targets for vaccine 
development. We report here a computational system, MULTIPRED, for the 
prediction of peptide binding to the HLA-A2 supertype. It combines a novel 
representation of peptide/MHC interactions with a hidden Markov model as the 
prediction algorithm. MULTIPREDis both sensitive and specific, and demonstrates 
high accuracy of peptide-binding predictions for HLA-A*0201, *0204, and *0205 
alleles, good accuracy for *0206 allele, and marginal accuracy for *0203 
allele. MULTIPREDreplaces earlier requirements for individual prediction models 
for each HLA allelic variant and simplifies computational aspects of 
peptide-binding prediction. Preliminary testing indicates that MULTIPRED can 
predict peptide binding to HLA-A2 supertype molecules with high accuracy, 
including those allelic variants for which no experimental binding data are 
currently available. </blockquote> 
<p></p> 
<blockquote>  Keywords: Algorithms, Amino Acid Motifs, Amino Acid Sequence, 
Antigen-Antibody Complex, Automated, Binding Sites, Computational Biology, Drug 
Delivery Systems, Drug Design, Epitopes, Forecasting, Genes, HLA Antigens, 
HLA-A Antigens, HLA-A2 Antigen, HLA-DR Antigens, Humans, Internet, MHC Class I, 
Markov Chains, Molecular Sequence Data, Neural Networks (Computer), Pattern 
Recognition, Peptide Fragments, Peptides, Protein, Protein Binding, Protein 
Interaction Mapping, Sensitivity and Specificity, Sequence Analysis, Software, 
T-Lymphocyte, User-Computer Interface, Viral Vaccines, 12067415 </blockquote> 
<br>  [Bowd2002Comparing]  Christopher Bowd, Kwokleung Chan, Linda&nbsp;M 
Zangwill, Michael&nbsp;H Goldbaum, Te-Won Lee, Terrence&nbsp;J Sejnowski, and 
Robert&nbsp;N Weinreb. Comparing neural networks and linear discriminant 
functions for glaucoma detection using confocal scanning laser ophthalmoscopy 
of the optic disc.<em>Invest Ophthalmol Vis Sci</em>, 43(11):3444-54, Nov 2002. 
[&nbsp;bib&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  PURPOSE: To determine whether neural network techniques can 
improve differentiation between glaucomatous and nonglaucomatous eyes, using 
the optic disc topography parameters of the Heidelberg Retina Tomograph (HRT; 
Heidelberg Engineering, Heidelberg, Germany). METHODS: With the HRT, one eye 
was imaged from each of 108 patients with glaucoma (defined as having 
repeatable visual field defects with standard automated perimetry) and 189 
subjects without glaucoma (no visual field defects with healthy-appearing optic 
disc and retinal nerve fiber layer on clinical examination) and the optic nerve 
topography was defined by 17 global and 66 regional HRT parameters. With all 
the HRT parameters used as input, receiver operating characteristic (ROC) 
curves were generated for the classification of eyes, by three neural network 
techniques: linear and Gaussian support vector machines (SVM linear and SVM 
Gaussian, respectively) and a multilayer perceptron (MLP), as well as four 
previously proposed linear discriminant functions (LDFs) and one LDF developed 
on the current data with all HRT parameters used as input. RESULTS: The areas 
under the ROC curves for SVM linear and SVM Gaussian were 0.938 and 0.945, 
respectively; for MLP, 0.941; for the current LDF, 0.906; and for the best 
previously proposed LDF, 0.890. With the use of forward selection and backward 
elimination optimization techniques, the areas under the ROC curves for SVM 
Gaussian and the current LDF were increased to approximately 0.96. CONCLUSIONS: 
Trained neural networks, with global and regional HRT parameters used as input, 
improve on previously proposed HRT parameter-based LDFs for discriminating 
between glaucomatous and nonglaucomatous eyes. The performance of both neural 
networks and LDFs can be improved with optimization of the features in the 
input. Neural network analyses show promise for increasing diagnostic accuracy 
of tests for glaucoma. </blockquote> 
<p></p> 
<blockquote>  Keywords: Acute, Algorithms, Animals, Anion Exchange Resins, 
Artificial Intelligence, Automated, Base Pair Mismatch, Base Pairing, Base 
Sequence, Biological, Biosensing Techniques, Carcinoma, Chemical, 
Chromatography, Citric Acid Cycle, Classification, Cluster Analysis, 
Comparative Study, Computational Biology, Computer-Assisted, Cystadenoma, DNA, 
Databases, Decision Making, Diagnosis, Differential, Discriminant Analysis, 
Drug, Drug Design, Electrostatics, Eukaryotic Cells, Factual, Feasibility 
Studies, Female, Gene Expression, Gene Expression Profiling, Gene Expression 
Regulation, Genes, Genetic, Genetic Heterogeneity, Genetic Markers, Glaucoma, 
Hemolysins, Humans, Internet, Intraocular Pressure, Ion Exchange, Lasers, 
Leukemia, Ligands, Likelihood Functions, Logistic Models, Lung Neoplasms, 
Lymphocytic, Lymphoma, Markov Chains, Mathematics, Messenger, Models, 
Molecular, Molecular Probe Techniques, Molecular Sequence Data, Nanotechnology, 
Neoplasm, Neoplasms, Neoplastic, Neural Networks (Computer), Non-P.H.S., 
Non-Small-Cell Lung, Non-U.S. Gov't, Nucleic Acid Conformation, Nucleic Acid 
Hybridization, Observer Variation, Oligonucleotide Array Sequence Analysis, 
Open-Angle, Ophthalmoscopy, Optic Disk, Ovarian Neoplasms, P.H.S., Pattern 
Recognition, Probability, Probability Learning, Protein Binding, Protein 
Conformation, Proteins, Quality Control, Quantum Theory, RNA, RNA Splicing, ROC 
Curve, Receptors, Reference Values, Regression Analysis, Reproducibility of 
Results, Research Support, Robotics, Saccharomyces cerevisiae Proteins, 
Sensitivity and Specificity, Sequence Analysis, Signal Processing, Software, 
Statistical, Stomach Neoplasms, Structural, Structure-Activity Relationship, 
Thermodynamics, Transcription, Tumor Markers, U.S. Gov't, 12407155 </blockquote>
<br>  [Collobert2002parallel]  Ronan Collobert, Samy Bengio, and Yoshua Bengio. 
A parallel mixture of SVMs for very large scale problems.<em>Neural Comput</em>
, 14(5):1105-14, May 2002. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;] 
<blockquote>  Support vector machines (SVMs) are the state-of-the-art models 
for many classification problems, but they suffer from the complexity of their 
training algorithm, which is at least quadratic with respect to the number of 
examples. Hence, it is hopeless to try to solve real-life problems having more 
than a few hundred thousand examples with SVMs. This article proposes a new 
mixture of SVMs that can be easily implemented in parallel and where each SVM 
is trained on a small subset of the whole data set. Experiments on a large 
benchmark data set (Forest) yielded significant time improvement (time 
complexity appears empirically to locally grow linearly with the number of 
examples). In addition, and surprisingly, a significant improvement in 
generalization was observed. </blockquote> 
<p></p> 
<blockquote>  Keywords: Algorithms, Amino Acid Sequence, Antineoplastic 
Agents, Artificial Intelligence, Automated, Bacterial, Base Composition, Base 
Sequence, Bayes Theorem, Bias (Epidemiology), Carcinoma, Cell Line, Chemical, 
Chromosome Aberrations, Cluster Analysis, Codon, Comparative Study, 
Computational Biology, Computer Graphics, Computer Simulation, 
Computer-Assisted, Conserved Sequence, DNA Gyrase, Databases, Diagnosis, 
Electrostatics, Escherichia coli, Evolution, Factual, Follow-Up Studies, 
Fungal, Gene Dosage, Gene Expression, Gene Expression Profiling, Gene 
Expression Regulation, Genes, Genetic, Genetic Vectors, Genomics, Glaucoma, 
Humans, Hydrophobicity, Immunoglobulins, Information Theory, Intraocular 
Pressure, Least-Squares Analysis, Linear Models, Liposarcoma, Lung Neoplasms, 
Markov Chains, Models, Molecular, Molecular Sequence Data, Multigene Family, 
Mycobacterium tuberculosis, Neural Networks (Computer), Non-P.H.S., 
Non-Small-Cell Lung, Non-U.S. Gov't, Normal Distribution, Nucleic Acid 
Hybridization, Ocular Hypertension, Oligonucleotide Array Sequence Analysis, 
Oncogenes, Optic Disk, Optic Nerve Diseases, P.H.S., Pattern Recognition, 
Perimetry, Photography, Phylogeny, Polymerase Chain Reaction, Protein, Protein 
Structure, Proteins, Reproducibility of Results, Research Suppor, Research 
Support, Ribosomal Proteins, Saccharomyces cerevisiae, Saccharomyces cerevisiae 
Proteins, Secondary, Sensitivity and Specificity, Sequence Alignment, Sequence 
Analysis, Sequence Homology, Soft Tissue Neoplasms, Software, Solvents, 
Statistical, Subcellular Fractions, Theoretical, Tumor Suppressor, U.S. Gov't, 
User-Computer Interface, Vision Disorders, Visual Fields, kappa-Chain, t, 
15165393 </blockquote> <br>
 [Zhu2003Introduction]  Lingyun Zhu, Baoming Wu, 
and Changxiu Cao. Introduction to medical data mining.<em>Sheng Wu Yi Xue Gong 
Cheng Xue Za Zhi</em>, 20(3):559-62, Sep 2003. [&nbsp;bib&nbsp;] 
<blockquote>  Modern medicine generates a great deal of information stored in 
the medical database. Extracting useful knowledge and providing scientific 
decision-making for the diagnosis and treatment of disease from the database 
increasingly becomes necessary. Data mining in medicine can deal with this 
problem. It can also improve the management level of hospital information and 
promote the development of telemedicine and community medicine. Because the 
medical information is characteristic of redundancy, multi-attribution, 
incompletion and closely related with time, medical data mining differs from 
other one. In this paper we have discussed the key techniques of medical data 
mining involving pretreatment of medical data, fusion of different pattern and 
resource, fast and robust mining algorithms and reliability of mining results. 
The methods and applications of medical data mining based on computation 
intelligence such as artificial neural network, fuzzy system, evolutionary 
algorithms, rough set, and support vector machine have been introduced. The 
features and problems in data mining are summarized in the last section. 
</blockquote> 
<p></p> 
<blockquote>  Keywords: Algorithms, Anion Exchange Resins, Automatic Data 
Processing, Chemical, Chromatography, Computational Biology, Computer-Assisted, 
Data Interpretation, Databases, Decision Making, Decision Trees, English 
Abstract, Factual, Fuzzy Logic, Humans, Indicators and Reagents, Information 
Storage and Retrieval, Ion Exchange, Models, Neural Networks (Computer), 
Non-P.H.S., Non-U.S. Gov't, Nucleic Acid Conformation, P.H.S., Proteins, 
Quantitative Structure-Activity Relationship, RNA, ROC Curve, Research Support, 
Sequence Analysis, Statistical, Transfer, U.S. Gov't, 14565039 </blockquote> 
<br>  [Zhao2003Applicationa]  Y.&nbsp;Zhao, C.&nbsp;Pinilla, D.&nbsp;Valmori, 
R.&nbsp;Martin, and R.&nbsp;Simon. Application of support vector machines for 
T-cell epitopes prediction.<em>Bioinformatics</em>, 19(15):1978-1984, Oct 2003. 
[&nbsp;bib&nbsp;] 
<blockquote>  MOTIVATION: The T-cell receptor, a major histocompatibility 
complex (MHC) molecule, and a bound antigenic peptide, play major roles in the 
process of antigen-specific T-cell activation. T-cell recognition was long 
considered exquisitely specific. Recent data also indicate that it is highly 
flexible, and one receptor may recognize thousands of different peptides. 
Deciphering the patterns of peptides that elicit a MHC restricted T-cell 
response is critical for vaccine development. RESULTS: For the first time we 
develop a support vector machine (SVM) for T-cell epitope prediction with an 
MHC type I restricted T-cell clone. Using cross-validation, we demonstrate that 
SVMs can be trained on relatively small data sets to provide prediction more 
accurate than those based on previously published methods or on MHC binding. 
SUPPLEMENTARY INFORMATION: Data for 203 synthesized peptides is available at 
http://linus.nci.nih.gov/Data/LAU203_Peptide.pdf </blockquote> 
<p></p> 
<blockquote>  Keywords: Algorithms, Amino Acid Sequence, Antigen, Antigen 
Presentation, Antigen-Antibody Complex, Artificial Intelligence, Autoimmune 
Diseases, Autoimmunity, Bacterial Proteins, CD4-Positive T-Lymphocytes, Cell 
Proliferation, Cells, Clone Cells, Cluster Analysis, Conserved Sequence, Cross 
Reactions, Cultured, Cytokines, Databases, Epitope Mapping, Epitopes, Gene 
Products, Genetic, HIV-1, HLA-DQ Antigens, HLA-DR2 Antigen, Haplotypes, 
Helper-Inducer, Hemagglutination, Histocompatibility Antigens Class I, Humans, 
K562 Cells, Molecular Mimicry, Molecular Sequence Data, Multiple Sclerosis, 
Myelin Proteins, Neural Networks (Computer), Orthomyxoviridae, Peptide Library, 
Peptides, Protein, Protein Binding, Protein Interaction Mapping, ROC Curve, 
Receptors, Relapsing-Remitting, Reproducibility of Results, Reverse 
Transcriptase Polymerase Chain Reaction, Sensitivity and Specificity, Sequence 
Analysis, Structure-Activity Relationship, T-Cell, T-Lymphocyte, T-Lymphocytes, 
Torque teno virus, Viral, Viral Proteins, gag, 14555632 </blockquote> <br>
 [
Waterman2003Transcriptional]  Scott&nbsp;R Waterman and P.&nbsp;L&nbsp;C Small. 
Transcriptional expression of escherichia coli glutamate-dependent acid 
resistance genes gada and gadbc in an hns rpos mutant.<em>J. Bacteriol.</em>, 
185(15):4644-4647, Aug 2003. [&nbsp;bib&nbsp;] 
<blockquote>  Resistance to being killed by acidic environments with pH values 
lower than 3 is an important feature of both pathogenic and nonpathogenic 
Escherichia coli. The most potent E. coli acid resistance system utilizes two 
isoforms of glutamate decarboxylase encoded by gadA and gadB and a putative 
glutamate:gamma-aminobutyric acid antiporter encoded by gadC. The gad system is 
controlled by two repressors (H-NS and CRP), one activator (GadX), one 
repressor-activator (GadW), and two sigma factors (sigma(S) and sigma(70)). In 
contrast to results of previous reports, we demonstrate that gad transcription 
can be detected in an hns rpoS mutant strain of E. coli K-12, indicating that 
gad promoters can be initiated by sigma(70) in the absence of H-NS. 
</blockquote> 
<p></p> 
<blockquote>  Keywords: Bacterial Proteins; DNA-Binding Proteins; Drug 
Resistance, Bacterial; Escherichia coli; Escherichia coli Proteins; Gene 
Expression Regulation, Bacterial; Glutamate Decarboxylase; Glutamates; 
Hydrogen-Ion Concentration; Membrane Proteins; Mutation; Sigma Factor; 
Transcription, Genetic </blockquote> <br>
 [Vickers2003Efficient]  T.&nbsp;A. 
Vickers, S.&nbsp;Koo, C.&nbsp;F. Bennett, S.&nbsp;T. Crooke, N.&nbsp;M. Dean, 
and B.&nbsp;F. Baker. Efficient reduction of target RNAs by small interfering 
RNA and RNase H-dependent antisense agents. A comparative analysis.<em>J. Biol. 
Chem.</em>, 278(9):7108-18, Feb 2003. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;] 
<blockquote>  RNA interference can be considered as an antisense mechanism of 
action that utilizes a double-stranded RNase to promote hydrolysis of the 
target RNA. We have performed a comparative study of optimized antisense 
oligonucleotides designed to work by an RNA interference mechanism to 
oligonucleotides designed to work by an RNase H-dependent mechanism in human 
cells. The potency, maximal effectiveness, duration of action, and sequence 
specificity of optimized RNase H-dependent oligonucleotides and small 
interfering RNA (siRNA) oligonucleotide duplexes were evaluated and found to be 
comparable. Effects of base mismatches on activity were determined to be 
position-dependent for both siRNA oligonucleotides and RNase H-dependent 
oligonucleotides. In addition, we determined that the activity of both siRNA 
oligonucleotides and RNase H-dependent oligonucleotides is affected by the 
secondary structure of the target mRNA. To determine whether positions on 
target RNA identified as being susceptible for RNase H-mediated degradation 
would be coincident with siRNA target sites, we evaluated the effectiveness of 
siRNAs designed to bind the same position on the target mRNA as RNase 
H-dependent oligonucleotides. Examination of 80 siRNA oligonucleotide duplexes 
designed to bind to RNA from four distinct human genes revealed that, in 
general, activity correlated with the activity to RNase H-dependent 
oligonucleotides designed to the same site, although some exceptions were 
noted. The one major difference between the two strategies is that RNase 
H-dependent oligonucleotides were determined to be active when directed against 
targets in the pre-mRNA, whereas siRNAs were not. These results demonstrate 
that siRNA oligonucleotide- and RNase H-dependent antisense strategies are both 
valid strategies for evaluating function of genes in cell-based assays. 
</blockquote> 
<p></p> 
<blockquote>  Keywords: Animals, Antisense, Base Sequence, COS Cells, Calf 
Thymus, Cultured, Dose-Response Relationship, Drug, Flow Cytometry, Humans, 
Intercellular Adhesion Molecule-1, Introns, Luciferases, Messenger, Molecular 
Sequence Data, Nucleic Acid Conformation, Oligonucleotides, PTEN 
Phosphohydrolase, Phosphoric Monoester Hydrolases, Protein Structure, RNA, 
Ribonuclease H, Small Interfering, Tertiary, Time Factors, Tumor Cells, Tumor 
Suppressor Proteins, 12500975 </blockquote> <br>
 [Takahashi2003Proteomic]  
Nobuhiro Takahashi, Mitsuaki Yanagida, Sally Fujiyama, Toshiya Hayano, and 
Toshiaki Isobe. Proteomic snapshot analyses of preribosomal ribonucleoprotein 
complexes formed at various stages of ribosome biogenesis in yeast and 
mammalian cells.<em>Mass Spectrom Rev</em>, 22(5):287-317, 2003. [&nbsp;bib
&nbsp;|DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  Proteomic technologies powered by advancements in mass 
spectrometry and bioinformatics and coupled with accumulated genome sequence 
data allow a comprehensive study of cell function through large-scale and 
systematic protein identifications of protein constituents of the cell and 
tissues, as well as of multi-protein complexes that carry out many cellular 
function in a higher-order network in the cell. One of the most extensively 
analyzed cellular functions by proteomics is the production of ribosome, the 
protein-synthesis machinery, in the nucle(ol)us-the main site of ribosome 
biogenesis. The use of tagged proteins as affinity bait, coupled with mass 
spectrometric identification, enabled us to isolate synthetic intermediates of 
ribosomes that might represent snapshots of nascent ribosomes at particular 
stages of ribosome biogenesis and to identify their constituents-some of which 
showed dynamic changes for association with the intermediates at various stages 
of ribosome biogenesis. In this review, in conjunction with the results from 
yeast cells, our proteomic approach to analyze ribosome biogenesis in mammalian 
cells is described. </blockquote> 
<p></p> 
<blockquote>  Keywords: Affinity Labels, Animals, Comparative Study, 
Electrospray Ionization, Genetic, Macromolecular Substances, Mass, Mitosis, 
Non-P.H.S., Non-U.S. Gov't, P.H.S., Protein Interaction Mapping, Proteome, 
Proteomics, Research Support, Ribonucleoproteins, Ribosomes, Saccharomyces 
cerevisiae, Saccharomyces cerevisiae Proteins, Signal Transduction, 
Spectrometry, Transcription, U.S. Gov't, 12949916 </blockquote> <br>
 [
Sheinerman2003Sequence]  Felix&nbsp;B Sheinerman, Bissan Al-Lazikani, and Barry 
Honig. Sequence, structure and energetic determinants of phosphopeptide 
selectivity of SH2 domains.<em>J. Mol. Biol.</em>, 334(4):823-841, Dec 2003. 
[&nbsp;bib&nbsp;] 
<blockquote>  Here, we present an approach for the prediction of binding 
preferences of members of a large protein family for which structural 
information for a number of family members bound to a substrate is available. 
The approach involves a number of steps. First, an accurate multiple alignment 
of sequences of all members of a protein family is constructed on the basis of 
a multiple structural superposition of family members with known structure. 
Second, the methods of continuum electrostatics are used to characterize the 
energetic contribution of each residue in a protein to the binding of its 
substrate. Residues that make a significant contribution are mapped onto the 
protein sequence and are used to define a &quot;binding site signature&quot; 
for the complex being considered. Third, sequences whose structures have not 
been determined are checked to see if they have binding-site signatures similar 
to one of the known complexes. Predictions of binding affinity to a given 
substrate are based on similarities in binding-site signature. An important 
component of the approach is the introduction of a context-specific 
substitution matrix suitable for comparison of binding-site residues.The 
methods are applied to the prediction of phosphopeptide selectivity of SH2 
domains. To this end, the energetic roles of all protein residues in 17 
different complexes of SH2 domains with their cognate targets are analyzed. The 
total number of residues that make significant contributions to binding is 
found to vary from nine to 19 in different complexes. These energetically 
important residues are found to contribute to binding through a variety of 
mechanisms, involving both electrostatic and hydrophobic interactions. 
Binding-site signatures are found to involve residues in different positions in 
SH2 sequences, some of them as far as 9A away from a bound peptide. 
Surprisingly, similarities in the signatures of different domains do not 
correlate with whole-domain sequence identities unless the latter is greater 
than 50%.An extensive comparison with the optimal binding motifs determined by 
peptide library experiments, as well as other experimental data indicate that 
the similarity in binding preferences of different SH2 domains can be deduced 
on the basis of their binding-site signatures. The analysis provides a 
rationale for the empirically derived classification of SH2 domains described 
by Songyang &amp; Cantley, in that proteins in the same group are found to have 
similar residues at positions important for binding. Confident predictions of 
binding preference can be made for about 85% of SH2 domain sequences found in 
SWISSPROT. The approach described in this work is quite general and can, in 
principle, be used to analyze binding preferences of members of large protein 
families for which structural information for a number of family members is 
available. It also offers a strategy for predicting cross-reactivity of 
compounds designed to bind to a particular target, for example in 
structure-based drug design. </blockquote> 
<p></p> 
<blockquote>  Keywords: Amino Acid Sequence; Binding Sites; Molecular Sequence 
Data; Peptide Library; Phosphopeptides; Protein Binding; Sequence Alignment; 
Substrate Specificity; src Homology Domains </blockquote> <br>
 [
Salim2003Combination]  N.&nbsp;Salim, J.&nbsp;Holliday, and P.&nbsp;Willett. 
Combination of fingerprint-based similarity coefficients using data fusion.<em>
J Chem Inf Comput Sci</em>, 43(2):435-442, 2003. [&nbsp;bib&nbsp;| DOI&nbsp;| 
http&nbsp;] 
<blockquote>  Many different types of similarity coefficients have been 
described in the literature. Since different coefficients take into account 
different characteristics when assessing the degree of similarity between 
molecules, it is reasonable to combine them to further optimize the measures of 
similarity between molecules. This paper describes experiments in which data 
fusion is used to combine several binary similarity coefficients to get an 
overall estimate of similarity for searching databases of bioactive molecules. 
The results show that search performances can be improved by combining 
coefficients with little extra computational cost. However, there is no single 
combination which gives a consistently high performance for all search types. 
</blockquote> 
<p></p> 
<blockquote>  Keywords: 80 and over, Acid-Base Imbalance, Acute, Acute 
Disease, Adolescent, Adult, African Americans, Aged, Anemia, Animals, Anti-HIV 
Agents, Anti-Infective Agents, Antibiotics, Antibodies, Antineoplastic, 
Antineoplastic Agents, Antineoplastic Combined Chemotherapy Protocols, 
Antitubercular Agents, Aorta, Asparaginase, Autoimmune, B-Cell, Bangladesh, 
Bicarbonates, Biological Markers, Blood Glucose, California, Camptothecin, 
Cellulitis, Chorionic Gonadotropin, Chronic Disease, Ciprofloxacin, Clinical 
Protocols, Colorectal Neoplasms, Combination, Comparative Study, Daunorubicin, 
Decision Trees, Dexamethasone, Diabetes Mellitus, Dideoxynucleosides, Directly 
Observed Therapy, Disease Transmission, Drug Administration Schedule, Drug 
Resistance, Drug Therapy, English Abstract, Female, Fluorouracil, Follow-Up 
Studies, Glucose Tolerance Test, Glucosephosphate Dehydrogenase, Glyburide, HIV 
Infections, HIV-1, Health Planning, Health Resources, Helminth, Hemolysis, 
Hemolytic, Hormonal, Hospital Mortality, Human, Humans, Hypoglycemic Agents, 
Immunoglobulin M, In Vitro, Incidence, Indinavir, Insulin, Intensive Care 
Units, Interstitial, Lactates, Leucovorin, Leukemia, Male, Maternal Age, Middle 
Aged, Motor Activity, Multidrug-Resistant, Mutation, Nephritis, Non-U.S. Gov't, 
Organoplatinum Compounds, Pennsylvania, Phytotherapy, Plant Extracts, Plant 
Leaves, Population Dynamics, Potassium Channels, Prednisone, Pregnancy, 
Pregnancy Outcome, Prenatal, Prenatal Care, Progesterone, Prognosis, 
Prospective Studies, Pulmonary, Rabbits, Randomized Controlled Trials, Rats, 
Research Support, Retrospective Studies, Risk Assessment, Scalp Dermatoses, 
Schistosomiasis japonica, Severity of Illness Index, Spondylarthropathies, 
Streptozocin, Survival Rate, Trauma Centers, Trauma Severity Indices, Tubal, 
Tuberculosis, Type 2, Ultrasonography, Vertical, Vincristine, Viral, Viral 
Load, Wistar, Wounds and Injuries, Ziziphus, beta Subunit, 12653506 
</blockquote> <br>
 [Jambon2003New]  Martin Jambon, Anne Imberty, Gilbert 
Del&eacute;age, and Christophe Geourjon. A new bioinformatic approach to detect 
common 3d sites in protein structures.<em>Proteins</em>, 52(2):137-145, Aug 
2003. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;] 
<blockquote>  An innovative bioinformatic method has been designed and 
implemented to detect similar three-dimensional (3D) sites in proteins. This 
approach allows the comparison of protein structures or substructures and 
detects local spatial similarities: this method is completely independent from 
the amino acid sequence and from the backbone structure. In contrast to already 
existing tools, the basis for this method is a representation of the protein 
structure by a set of stereochemical groups that are defined independently from 
the notion of amino acid. An efficient heuristic for finding similarities that 
uses graphs of triangles of chemical groups to represent the protein structures 
has been developed. The implementation of this heuristic constitutes a software 
named SuMo (Surfing the Molecules), which allows the dynamic definition of 
chemical groups, the selection of sites in the proteins, and the management and 
screening of databases. To show the relevance of this approach, we focused on 
two extreme examples illustrating convergent and divergent evolution. In two 
unrelated serine proteases, SuMo detects one common site, which corresponds to 
the catalytic triad. In the legume lectins family composed of &gt;100 
structures that share similar sequences and folds but may have lost their 
ability to bind a carbohydrate molecule, SuMo discriminates between functional 
and non-functional lectins with a selectivity of 96%. The time needed for 
searching a given site in a protein structure is typically 0.1 s on a PIII 
800MHz/Linux computer; thus, in further studies, SuMo will be used to screen 
the PDB. </blockquote> 
<p></p> 
<blockquote>  Keywords: Algorithms; Catalytic Domain; Chymotrypsin, 
chemistry/genetics; Computational Biology, methods; Evolution, Molecular; 
Fabaceae, chemistry; Models, Molecular; Plant Lectins, chemistry/genetics; 
Protein Conformation; Proteins, chemistry; Reproducibility of Results; 
Subtilisin, chemistry/genetics </blockquote> <br>
 [Bild2003]  E.&nbsp;Huang, 
S.&nbsp;Ishida, J.&nbsp;Pittman, H.&nbsp;Dressman, A.&nbsp;Bild, M.&nbsp;Kloos, 
M.&nbsp;D'Amico, R.&nbsp;G. Pestell, M.&nbsp;West, and J.&nbsp;R. Nevins. Gene 
expression phenotypic models that predict the activity of oncogenic pathways.
<em>Nat Genet</em>, 34(2):226-30, 2003. [&nbsp;bib&nbsp;] 
<blockquote>  High-density DNA microarrays measure expression of large numbers 
of genes in one assay. The ability to find underlying structure in complex gene 
expression data sets and rigorously test association of that structure with 
biological conditions is essential to developing multi-faceted views of the 
gene activity that defines cellular phenotype. We sought to connect features of 
gene expression data with biological hypotheses by integrating 'metagene' 
patterns from DNA microarray experiments in the characterization and prediction 
of oncogenic phenotypes. We applied these techniques to the analysis of 
regulatory pathways controlled by the genes HRAS (Harvey rat sarcoma viral 
oncogene homolog), MYC (myelocytomatosis viral oncogene homolog) and E2F1, E2F2 
and E2F3 (encoding E2F transcription factors 1, 2 and 3, respectively). The 
phenotypic models accurately predict the activity of these pathways in the 
context of normal cell proliferation. Moreover, the metagene models trained 
with gene expression patterns evoked by ectopic production of Myc or Ras 
proteins in primary tissue culture cells properly predict the activity of in 
vivo tumor models that result from deregulation of the MYC or HRAS pathways. We 
conclude that these gene expression phenotypes have the potential to 
characterize the complex genetic alterations that typify the neoplastic state, 
whether in vitro or in vivo, in a way that truly reflects the complexity of the 
regulatory pathways that are affected. </blockquote> 
<p></p> 
<blockquote>  Keywords: Animals *Cell Cycle Proteins *DNA-Binding Proteins E2F 
Transcription Factors E2F1 Transcription Factor E2F2 Transcription Factor E2F3 
Transcription Factor Female *Gene Expression Gene Expression Profiling Gene 
Expression Regulation, Neoplastic Genes, myc Genes, ras Mammary Neoplasms, 
Experimental/genetics Mice Mice, Transgenic *Models, Genetic Oligonucleotide 
Array Sequence Analysis *Oncogenes Phenotype Transcription Factors/genetics 
</blockquote> <br>
 [Harborth2003Sequence]  J.&nbsp;Harborth, S.&nbsp;M. 
Elbashir, K.&nbsp;Vandenburgh, H.&nbsp;Manninga, S.&nbsp;A. Scaringe, 
K.&nbsp;Weber, and T.&nbsp;Tuschl. Sequence, chemical, and structural variation 
of small interfering RNAs and short hairpin RNAs and the effect on mammalian 
gene silencing.<em>Antisense Nucleic Acid. Drug. Dev.</em>, 13(2):83-105, Apr 
2003. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;] 
<blockquote>  Small interfering RNAs (siRNAs) induce sequence-specific gene 
silencing in mammalian cells and guide mRNA degradation in the process of RNA 
interference (RNAi). By targeting endogenous lamin A/C mRNA in human HeLa or 
mouse SW3T3 cells, we investigated the positional variation of siRNA-mediated 
gene silencing. We find cell-type-dependent global effects and 
cell-type-independent positional effects. HeLa cells were about 2-fold more 
responsive to siRNAs than SW3T3 cells but displayed a very similar pattern of 
positional variation of lamin A/C silencing. In HeLa cells, 26 of 44 tested 
standard 21-nucleotide (nt) siRNA duplexes reduced the protein expression by at 
least 90%, and only 2 duplexes reduced the lamin A/C proteins to &lt;50%. 
Fluorescent chromophores did not perturb gene silencing when conjugated to the 
5'-end or 3'-end of the sense siRNA strand and the 5'-end of the antisense 
siRNA strand, but conjugation to the 3'-end of the antisense siRNA abolished 
gene silencing. RNase-protecting phosphorothioate and 2'-fluoropyrimidine RNA 
backbone modifications of siRNAs did not significantly affect silencing 
efficiency, although cytotoxic effects were observed when every second 
phosphate of an siRNA duplex was replaced by phosphorothioate. Synthetic RNA 
hairpin loops were subsequently evaluated for lamin A/C silencing as a function 
of stem length and loop composition. As long as the 5'-end of the guide strand 
coincided with the 5'-end of the hairpin RNA, 19-29 base pair (bp) hairpins 
effectively silenced lamin A/C, but when the hairpin started with the 5'-end of 
the sense strand, only 21-29 bp hairpins were highly active. </blockquote> 
<p></p> 
<blockquote>  Keywords: Adaptor Protein Complex alpha Subunits, Animal, 
Animals, Antisense, Apolipoproteins B, Base Sequence, Biological Transport, 
Blotting, Catalytic, Cell Line, Cell Membrane, Cell Survival, Chemical, 
Cholesterol, Clathrin, Clathrin Heavy Chains, Disease Models, Endocytosis, 
Epidermal Growth Factor, Fluorescence, Gene Expression Profiling, Gene 
Silencing, Gene Therapy, Hela Cells, Humans, Injections, Intravenous, Jejunum, 
Kinetics, Lamin Type A, Liver, Messenger, Metabolic Syndrome X, Mice, 
Microscopy, Models, Molecular Sequence Data, NIH 3T3 Cells, Non-U.S. Gov't, 
Nucleic Acid, Oligonucleotides, Open Reading Frames, Post-Transcriptional, 
Protein Isoforms, Pyrimidines, RNA, RNA Interference, RNA Processing, RNA 
Stability, Research Support, Reverse Transcriptase Polymerase Chain Reaction, 
Sensitivity and Specificity, Sequence Homology, Small Interfering, Subcellular 
Fractions, Swiss 3T3 Cells, Thionucleotides, Time Factors, Transfection, 
Transferrin, Transgenic, Tumor, Western, 12804036 </blockquote> <br>
 [
Ge2003Reducing]  Xijin Ge, Shuichi Tsutsumi, Hiroyuki Aburatani, and Shuichi 
Iwata. Reducing false positives in molecular pattern recognition.<em>Genome 
Inform Ser Workshop Genome Inform</em>, 14:34-43, 2003. [&nbsp;bib&nbsp;] 
<blockquote>  In the search for new cancer subtypes by gene expression 
profiling, it is essential to avoid misclassifying samples of unknown subtypes 
as known ones. In this paper, we evaluated the false positive error rates of 
several classification algorithms through a 'null test' by presenting 
classifiers a large collection of independent samples that do not belong to any 
of the tumor types in the training dataset. The benchmark dataset is available 
at www2.genome.rcast.u-tokyo.ac.jp/pm/. We found that k-nearest neighbor (KNN) 
and support vector machine (SVM) have very high false positive error rates when 
fewer genes (&lt;100) are used in prediction. The error rate can be partially 
reduced by including more genes. On the other hand, prototype matching (PM) 
method has a much lower false positive error rate. Such robustness can be 
achieved without loss of sensitivity by introducing suitable measures of 
prediction confidence. We also proposed a cluster-and-select technique to 
select genes for classification. The nonparametric Kruskal-Wallis H test is 
employed to select genes differentially expressed in multiple tumor types. To 
reduce the redundancy, we then divided these genes into clusters with similar 
expression patterns and selected a given number of genes from each cluster. The 
reliability of the new algorithm is tested on three public datasets. 
</blockquote> 
<p></p> 
<blockquote>  Keywords: Amino Acid Sequence, Amino Acids, Animals, Automated, 
Base Sequence, Bayes Theorem, Biological, Carbohydrate Conformation, 
Carbohydrate Sequence, Cattle, Computational Biology, Computer Simulation, 
Crystallography, DNA, Databases, Factual, False Positive Reactions, Gene 
Expression Profiling, Genes, Genetic, Genetic Techniques, Genome, 
Histocompatibility Antigens Class I, Human, Humans, Introns, Least-Squares 
Analysis, MHC Class I, Major Histocompatibility Complex, Markov Chains, 
Messenger, Mice, Models, Monosaccharides, Neoplasms, Non-U.S. Gov't, 
Nonparametric, Pattern Recognition, Peptides, Phylogeny, Plants, Poly A, 
Polysaccharides, Predictive Value of Tests, Protein, Protein Structure, 
Proteins, RNA, Rats, Reproducibility of Results, Research Support, 
Saccharomyces cerevisiae, Secondary, Sequence Alignment, Software, Species 
Specificity, Statistics, Theoretical, X-Ray, 15706518 </blockquote> <br>
 [
Djordjevic2003biophysical]  Marko Djordjevic, Anirvan&nbsp;M Sengupta, and 
Boris&nbsp;I Shraiman. A biophysical approach to transcription factor binding 
site discovery.<em>Genome Res.</em>, 13(11):2381-90, Nov 2003. [&nbsp;bib&nbsp;|
DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  Identification of transcription factor binding sites within 
regulatory segments of genomic DNA is an important step toward understanding of 
the regulatory circuits that control expression of genes. Here, we describe a 
novel bioinformatics method that bases classification of potential binding 
sites explicitly on the estimate of sequence-specific binding energy of a given 
transcription factor. The method also estimates the chemical potential of the 
factor that defines the threshold of binding. In contrast with the widely used 
information-theoretic weight matrix method, the new approach correctly 
describes saturation in the transcription factor/DNA binding probability. This 
results in a significant improvement in the number of expected false positives, 
particularly in the ubiquitous case of low-specificity factors. In the strong 
binding limit, the algorithm is related to the &quot;support vector 
machine&quot; approach to pattern recognition. The new method is used to 
identify likely genomic binding sites for the E. coli transcription factors 
collected in the DPInteract database. In addition, for CRP (a global regulatory 
factor), the likely regulatory modality (i.e., repressor or activator) of 
predicted binding sites is determined. </blockquote> 
<p></p> 
<blockquote>  Keywords: Acute, Adult, Agrochemicals, Air Pollutants, Aircraft, 
Algorithms, Amino Acid, Amino Acid Sequence, Animals, Artificial Intelligence, 
Automated, Bacterial, Base Composition, Base Sequence, Bayes Theorem, Binding 
Sites, Biophysics, Bone Marrow Cells, Brain, Breast Neoplasms, Caenorhabditis 
elegans Proteins, Carbonic Anhydrase Inhibitors, Chromosome Aberrations, 
Cluster Analysis, Cohort Studies, Colonic Neoplasms, Comparative Study, 
Computational Biology, Computer Simulation, Computer Systems, 
Computer-Assisted, Computing Methodologies, Confidence Intervals, Cytosine, 
DNA, Data Interpretation, Databases, Diabetes Mellitus, Diabetic Neuropathies, 
Diagnosis, Drug Design, Emission-Computed, Enhancer Elements (Genetics), 
Environmental Monitoring, Enzyme Inhibitors, Escherichia coli, Ethanol, Exons, 
Female, Fluorescence, Forecasting, Fourier Transform Infrared, Fusion Proteins, 
Gene Expression Profiling, Gene Expression Regulation, Gene Rearrangement, 
Genetic, Genetic Screening, Glucuronosyltransferase, Greece, Guanine, Health 
Services Research, Humans, Image Interpretation, Internet, Isoenzymes, 
Least-Squares Analysis, Leukemia, Leukemic, Linear Models, Lymphocytic, 
Lymphoma, Male, Middle Aged, Models, Molecular, Molecular Conformation, 
Molecular Sequence Data, Natural Disasters, Neoplasm Proteins, Neoplasms, 
Neoplastic, Neural Networks (Computer), Non-P.H.S., Non-U.S. Gov't, Nonlinear 
Dynamics, Oligonucleotide Array Sequence Analysis, Online Systems, P.H.S., 
Pattern Recognition, Pharmaceutical Preparations, Phenotype, Photography, 
Physician's Practice Patterns, Polymorphism, Predictive Value of Tests, 
Principal Component Analysis, Probability, Prognosis, Protein, Protein 
Conformation, Proteins, Pyrimidines, Quantitative Structure-Activity 
Relationship, RNA Precursors, RNA Splice Sites, RNA Splicing, Radiation, 
Radiographic Image Interpretation, Reproducibility of Results, Research 
Support, Sensitivity and Specificity, Sequence Alignment, Sequence Analysis, 
Sequence Homology, Signal Processing, Single Nucleotide, Single-Photon, 
Software, Species Specificity, Spectrometry, Spectrophotometry, Spectroscopy, 
Statistical, Structure-Activity Relationship, Subtraction Technique, 
Terminology, Thermodynamics, Time Factors, Tomography, Transcription Factors, 
Translocation, Type 2, U.S. Gov't, Ultraviolet, Untranslated Regions, Video 
Recording, Walking, bcr-abl, 14597652 </blockquote> <br>
 [Diekman2003Hybrid]  
Casey Diekman, Wei He, Nagabhushana Prabhu, and Harvey Cramer. Hybrid methods 
for automated diagnosis of breast tumors.<em>Anal Quant Cytol Histol</em>, 
25(4):183-90, Aug 2003. [&nbsp;bib&nbsp;] 
<blockquote>  OBJECTIVE: To design and analyze a new family of hybrid methods 
for the diagnosis of breast tumors using fine needle aspirates. STUDY DESIGN: 
We present a radically new approach to the design of diagnosis systems. In the 
new approach, a nonlinear classifier with high sensitivity but low specificity 
is hybridized with a linear classifier having low sensitivity but high 
specificity. Data from the Wisconsin Breast Cancer Database are used to 
evaluate, computationally, the performance of the hybrid classifiers. RESULTS: 
The diagnosis scheme obtained by hybridizing the nonlinear classifier 
ellipsoidal multisurface method (EMSM) with the linear classifier proximal 
support vector machine (PSVM) was found to have a mean sensitivity of 97.36% 
and a mean specificity of 95.14% and was found to yield a 2.44% improvement in 
the reliability of positive diagnosis over that of EMSM at the expense of 0.4% 
degradation in the reliability of negative diagnosis, again compared to EMSM. 
At the 95% confidence level we can trust the hybrid method to be 96.19-98.53% 
correct in its malignant diagnosis of new tumors and 93.57-96.71% correct in 
its benign diagnosis. CONCLUSION: Hybrid diagnosis schemes represent a 
significant paradigm shift and provide a promising new technique to improve the 
specificity of nonlinear classifiers without seriously affecting the high 
sensitivity of nonlinear classifiers. </blockquote> 
<p></p> 
<blockquote>  Keywords: Algorithms, Amino Acid Sequence, Amino Acids, Anion 
Exchange Resins, Antigen-Antibody Complex, Artificial Intelligence, Automated, 
Automatic Data Processing, Benchmarking, Biological, Biological Markers, 
Biopsy, Blood Cells, Blood Proteins, Breast Neoplasms, Cell Line, Cellular 
Structures, Chemical, Chromatography, Chromosome Aberrations, Cluster Analysis, 
Colonic Neoplasms, Comparative Study, Computational Biology, Computer 
Simulation, Computer-Assisted, Computing Methodologies, DNA, Data 
Interpretation, Databases, Decision Making, Decision Trees, Diagnosis, 
Diffusion Magnetic Resonance Imaging, Disease, English Abstract, Epitopes, 
Expert Systems, Factual, Female, Fine-Needle, Fusion, Fuzzy Logic, Gene 
Expression Profiling, Gene Expression Regulation, Gene Targeting, Genetic, 
Genome, Histocompatibility Antigens Class I, Humans, Hydrogen Bonding, 
Hydrophobicity, Image Interpretation, Image Processing, In Vitro, Indicators 
and Reagents, Information Storage and Retrieval, Ion Exchange, Least-Squares 
Analysis, Leiomyosarcoma, Liver Cirrhosis, Lung Neoplasms, Magnetic Resonance 
Imaging, Male, Mass, Mathematical Computing, Matrix-Assisted Laser 
Desorption-Ionization, Models, Molecular, Molecular Sequence Data, Neoplasm 
Proteins, Neoplasms, Neoplastic, Nephroblastoma, Neural Networks (Computer), 
Non-P.H.S., Non-U.S. Gov't, Nonl, Nucleic Acid Conformation, Nucleic Acid 
Hybridization, Oligonucleotide Array Sequence Analysis, Oncogene Proteins, 
Ovarian Neoplasms, P.H.S., Pattern Recognition, Predictive Value of Tests, Pro, 
Prostatic Neoplasms, Protein, Protein Binding, Protein Interaction Mapping, 
Protein Structure, Proteins, Quantitative Structure-Activity Relationship, RNA, 
ROC Curve, Reproducibility of Results, Research Support, Rhabdomyosarcoma, 
Secondary, Sensitivity and Specificity, Sequence Alignment, Sequence Analysis, 
Severity of Illness Index, Software, Solubility, Spectrometry, Statistical, 
Structure-Activity Relationship, Subcellular Fractions, Subtraction Technique, 
T-Lymphocyte, Tissue Distribution, Transcription Factors, Transfer, Treatment 
Outcome, Tumor, Tumor Markers, U.S. Gov't, User-Computer Interface, inear 
Dynamics, teome, 12961824 </blockquote> <br>
 [Chung2003Radius]  Kai-Min Chung, 
Wei-Chun Kao, Chia-Liang Sun, Li-Lun Wang, and Chih-Jen Lin. Radius margin 
bounds for support vector machines with the RBF kernel.<em>Neural Comput</em>, 
15(11):2643-81, Nov 2003. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;] 
<blockquote>  An important approach for efficient support vector machine (SVM) 
model selection is to use differentiable bounds of the leave-one-out (loo) 
error. Past efforts focused on finding tight bounds of loo (e.g., radius margin 
bounds, span bounds). However, their practical viability is still not very 
satisfactory. Duan, Keerthi, and Poo (2003) showed that radius margin bound 
gives good prediction for L2-SVM, one of the cases we look at. In this letter, 
through analyses about why this bound performs well for L2-SVM, we show that 
finding a bound whose minima are in a region with small loo values may be more 
important than its tightness. Based on this principle, we propose modified 
radius margin bounds for L1-SVM (the other case) where the original bound is 
applicable only to the hard-margin case. Our modification for L1-SVM achieves 
comparable performance to L2-SVM. To study whether L1- or L2-SVM should be 
used, we analyze other properties, such as their differentiability, number of 
support vectors, and number of free support vectors. In this aspect, L1-SVM 
possesses the advantage of having fewer support vectors. Their implementations 
are also different, so we discuss related issues in detail. </blockquote> 
<p></p> 
<blockquote>  Keywords: Acute, Adult, Agrochemicals, Air Pollutants, Aircraft, 
Algorithms, Amino Acid, Amino Acid Sequence, Animals, Anion Exchange Resins, 
Apoferritin, Aromatic, Artificial Intelligence, Automated, Bacterial, Base 
Composition, Base Sequence, Bayes Theorem, Binding, Binding Sites, Biophysics, 
Bone Marrow Cells, Brain, Breast Neoplasms, Caenorhabditis elegans Proteins, 
Carbonic Anhydrase Inhibitors, Cell Cycle, Chemical, Chromatography, Chromosome 
Aberrations, Cluster Analysis, Cohort Studies, Colonic Neoplasms, Comparative 
Study, Competitive, Computational Biology, Computer Simulation, Computer 
Systems, Computer-Assisted, Computing Methodologies, Confidence Intervals, 
Cytosine, DNA, Data Interpretation, Databases, Diabetes Mellitus, Diabetic 
Neuropathies, Diagnosis, Drug Design, Electrostatics, Emission-Computed, 
Enhancer Elements (Genetics), Environmental Monitoring, Enzyme Inhibitors, 
Escherichia coli, Ethanol, Exons, Female, Fluorescence, Forecasting, Fourier 
Transform Infrared, Fusion Proteins, Gene Expression Profiling, Gene Expression 
Regulation, Gene Rearrangement, Genetic, Genetic Screening, Glucan 
1,4-alpha-Glucosidase, Glucuronosyltransferase, Greece, Guanine, Health 
Services Research, High Pressure Liquid, Humans, Hydrocarbons, Hydrogen 
Bonding, Hydrophobicity, Image Interpretation, Internet, Ion Exchange, 
Isoenzymes, Least-Squares Analysis, Leukemia, Leukemic, Ligands, Linear Models, 
Lymphocytic, Lymphoma, Male, Mice, Middle Aged, Models, Molecular, Molecular 
Conformation, Molecular Sequence Data, Molecular Weight, Natural Disasters, 
Neoplasm Proteins, Neoplasms, Neoplastic, Neural Networks (Computer), 
Neurological, Non-P.H.S., Non-U.S. Gov't, Nonlinear Dynamics, Oligonucleotide 
Array Sequence Analysis, Online Systems, P.H.S., Pattern Recognition, 
Pharmaceutical Preparations, Phenotype, Photography, Physician's Practice 
Patterns, Polymorphism, Predictive Value of Tests, Principal Component 
Analysis, Probability, Prognosis, Protein, Protein Binding, Protein 
Conformation, Proteins, Pyrimidines, Quantitative Structure-Activity 
Relationship, RNA Precursors, RNA Splice Sites, RNA Splicing, Radiation, 
Radiographic Image Interpretation, Regression Analysis, Reproducibility of 
Results, Research Support, Sensitivity and Specificity, Sequence Alignment, 
Sequence Analysis, Sequence Homology, Signal Processing, Single Nucleotide, 
Single-Photon, Sodium Chloride, Software, Species Specificity, Spectrometry, 
Spectrophotometry, Spectroscopy, Statistical, Structure-Activity Relationship, 
Subtraction Technique, Sulfates, Surface Properties, Terminology, 
Thermodynamics, Time Factors, Tomography, Transcription Factors, Translocation, 
Type 2, U.S. Gov't, Ultraviolet, Untranslated Regions, Video Recording, 
Walking, Yeasts, bcr-abl, 14577857 </blockquote> <br>
 [Chenna2003Multiple]  
R.&nbsp;Chenna, H.&nbsp;Sugawara, T.&nbsp;Koike, R.&nbsp;Lopez, T.&nbsp;J. 
Gibson, D.&nbsp;G. Higgins, and J.&nbsp;D. Thompson. Multiple sequence 
alignment with the Clustal series of programs.<em>Nucleic Acids Res.</em>, 
31(13):3497-3500, Jul 2003. [&nbsp;bib&nbsp;] 
<blockquote>  The Clustal series of programs are widely used in molecular 
biology for the multiple alignment of both nucleic acid and protein sequences 
and for preparing phylogenetic trees. The popularity of the programs depends on 
a number of factors, including not only the accuracy of the results, but also 
the robustness, portability and user-friendliness of the programs. New features 
include NEXUS and FASTA format output, printing range numbers and faster tree 
calculation. Although, Clustal was originally developed to run on a local 
computer, numerous Web servers have been set up, notably at the EBI (European 
Bioinformatics Institute) (http://www.ebi.ac.uk/clustalw/). </blockquote> 
<p></p> 
<blockquote>  Keywords: Algorithms; Amino Acid Sequence; Internet; Nucleic 
Acids; Phylogeny; Sequence Alignment; Sequence Analysis; Sequence Analysis, 
Protein; Software </blockquote> <br>
 [Chanda2003Fulfilling]  S.&nbsp;K. Chanda 
and J.&nbsp;S. Caldwell. Fulfilling the promise: drug discovery in the 
post-genomic era.<em>Drug Discov Today</em>, 8(4):168-174, Feb 2003. [&nbsp;bib
&nbsp;]
<blockquote>  The genomic era has brought with it a basic change in 
experimentation, enabling researchers to look more comprehensively at 
biological systems. The sequencing of the human genome coupled with advances in 
automation and parallelization technologies have afforded a fundamental 
transformation in the drug target discovery paradigm, towards systematic whole 
genome and proteome analyses. In conjunction with novel proteomic techniques, 
genome-wide annotation of function in cellular models is possible. Overlaying 
data derived from whole genome sequence, expression and functional analysis 
will facilitate the identification of causal genes in disease and significantly 
streamline the target validation process. Moreover, several parallel 
technological advances in small molecule screening have resulted in the 
development of expeditious and powerful platforms for elucidating inhibitors of 
protein or pathway function. Conversely, high-throughput and automated systems 
are currently being used to identify targets of orphan small molecules. The 
consolidation of these emerging functional genomics and drug discovery 
technologies promises to reap the fruits of the genomic revolution. 
</blockquote> 
<p></p> 
<blockquote>  Keywords: Adenovirus E3 Proteins, Algorithms, Amino Acid, 
Animal, Animals, Antigens, Apoptosis, Base Sequence, Basic Helix-Loop-Helix 
Transcription Factors, Blotting, Body Patterning, Bone Neoplasms, Carrier 
Proteins, Cell Aggregation, Cell Aging, Cell Cycle, Cell Line, Cell 
Proliferation, Cell Transformation, Cells, Chick Embryo, Chickens, 
Complementary, Computer-Assisted, Cultured, Cyclin-Dependent Kinase Inhibitor 
p16, Cytomegalovirus, DNA, Depsipeptides, Developmental, Diploidy, 
Down-Regulation, Drug Design, Drug Evaluation, Drug Resistance, Endothelial 
Cells, Estrogen, Extramural, Fibroblast Growth Factors, Fibroblasts, Flow 
Cytometry, Forecasting, Fusion, G1 Phase, Gene Expression Profiling, Gene 
Expression Regulation, Gene Library, Gene Targeting, Genes, Genetic, Genome, 
Genomics, Genotype, Green Fluorescent Proteins, Growth Substances, 
Hematopoietic Stem Cells, Homeobox, Homeodomain Proteins, Human, Humans, Image 
Processing, Immediate-Early Proteins, In Vitro, Intercellular Signaling 
Peptides and Proteins, Ion Channels, Knockout, Leukemia, Mammals, Messenger, 
Mice, Models, Molecular Conformation, Molecular Sequence Data, Multigene 
Family, Myeloid-Lymphoid Leukemia Protein, N.I.H., Neoplasm, Neoplasm Proteins, 
Neoplastic, Non-U.S. Gov't, Nuclear Proteins, Oligonucleotide Array Sequence 
Analysis, Oncogene Proteins, Osteosarcoma, P.H.S., Peptides, Phenotype, 
Phosphoproteins, Phosphorylation, Post-Translational, Preclinical, Promoter 
Regions (Genetics), Protease Inhibitors, Protein Binding, Protein Isoforms, 
Protein Kinase Inhibitors, Protein Processing, Proteomics, Proto-Oncogene 
Proteins, RNA, Receptors, Repetitive Sequences, Reporter, Repressor Proteins, 
Research Support, Retroviridae, STAT3 Transcription Factor, Signal 
Transduction, Small Interfering, Structure-Activity Relationship, Tamoxifen, 
Time Factors, Trans-Activation (Genetics), Trans-Activators, Transcription, 
Transcription Factor AP-1, Transcription Factors, Transfection, Tumor, Tumor 
Cells, Tumor Suppressor Protein p53, U.S. Gov't, Ubiquitin-Protein Ligases, 
Up-Regulation, Viral, Western, Wnt Proteins, Xenopus laevis, Zebrafish, p53, 
12581711 </blockquote> <br>
 [Chan2003Detection]  Ian Chan, William Wells, 
Robert&nbsp;V Mulkern, Steven Haker, Jianqing Zhang, Kelly&nbsp;H Zou, 
Stephan&nbsp;E Maier, and Clare M&nbsp;C Tempany. Detection of prostate cancer 
by integration of line-scan diffusion, T2-mapping and T2-weighted magnetic 
resonance imaging; a multichannel statistical classifier.<em>Med Phys</em>, 
30(9):2390-8, Sep 2003. [&nbsp;bib&nbsp;| .pdf&nbsp;] 
<blockquote>  A multichannel statistical classifier for detecting prostate 
cancer was developed and validated by combining information from three 
different magnetic resonance (MR) methodologies: T2-weighted, T2-mapping, and 
line scan diffusion imaging (LSDI). From these MR sequences, four different 
sets of image intensities were obtained: T2-weighted (T2W) from T2-weighted 
imaging, Apparent Diffusion Coefficient (ADC) from LSDI, and proton density 
(PD) and T2 (T2 Map) from T2-mapping imaging. Manually segmented tumor labels 
from a radiologist, which were validated by biopsy results, served as tumor 
&quot;ground truth.&quot; Textural features were extracted from the images 
using co-occurrence matrix (CM) and discrete cosine transform (DCT). Anatomical 
location of voxels was described by a cylindrical coordinate system. A 
statistical jack-knife approach was used to evaluate our classifiers. 
Single-channel maximum likelihood (ML) classifiers were based on 1 of the 4 
basic image intensities. Our multichannel classifiers: support vector machine 
(SVM) and Fisher linear discriminant (FLD), utilized five different sets of 
derived features. Each classifier generated a summary statistical map that 
indicated tumor likelihood in the peripheral zone (PZ) of the prostate gland. 
To assess classifier accuracy, the average areas under the receiver operator 
characteristic (ROC) curves over all subjects were compared. Our best FLD 
classifier achieved an average ROC area of 0.839(+/-0.064), and our best SVM 
classifier achieved an average ROC area of 0.761(+/-0.043). The T2W ML 
classifier, our best single-channel classifier, only achieved an average ROC 
area of 0.599(+/-0.146). Compared to the best single-channel ML classifier, our 
best multichannel FLD and SVM classifiers have statistically superior ROC 
performance (P=0.0003 and 0.0017, respectively) from pairwise two-sided t-test. 
By integrating the information from multiple images and capturing the textural 
and anatomical features in tumor areas, summary statistical maps can 
potentially aid in image-guided prostate biopsy and assist in guiding and 
controlling delivery of localized therapy under image guidance. </blockquote> 
<p></p> 
<blockquote>  Keywords: Algorithms, Anion Exchange Resins, Antigen-Antibody 
Complex, Artificial Intelligence, Automated, Automatic Data Processing, 
Biological, Blood Cells, Chemical, Chromatography, Cluster Analysis, 
Comparative Study, Computational Biology, Computer Simulation, 
Computer-Assisted, Data Interpretation, Databases, Decision Making, Decision 
Trees, Diffusion Magnetic Resonance Imaging, English Abstract, Epitopes, Expert 
Systems, Factual, Fuzzy Logic, Gene Expression Profiling, Gene Expression 
Regulation, Gene Targeting, Genome, Histocompatibility Antigens Class I, 
Humans, Image Interpretation, Image Processing, In Vitro, Indicators and 
Reagents, Information Storage and Retrieval, Ion Exchange, Least-Squares 
Analysis, Liver Cirrhosis, Magnetic Resonance Imaging, Male, Models, Neural 
Networks (Computer), Non-P.H.S., Non-U.S. Gov't, Nonl, Nucleic Acid 
Conformation, P.H.S., Pattern Recognition, Pro, Prostatic Neoplasms, Protein, 
Protein Binding, Protein Interaction Mapping, Proteins, Quantitative 
Structure-Activity Relationship, RNA, ROC Curve, Reproducibility of Results, 
Research Support, Sensitivity and Specificity, Sequence Analysis, Severity of 
Illness Index, Statistical, Structure-Activity Relationship, Subtraction 
Technique, T-Lymphocyte, Transcription Factors, Transfer, Treatment Outcome, 
U.S. Gov't, User-Computer Interface, inear Dynamics, teome, 14528961 
</blockquote> <br>
 [Buus2003Sensitive]  S.&nbsp;Buus, S.&nbsp;L. 
Lauem&oslash;ller, P.&nbsp;Worning, C.&nbsp;Kesmir, T.&nbsp;Frimurer, 
S.&nbsp;Corbet, A.&nbsp;Fomsgaard, J.&nbsp;Hilden, A.&nbsp;Holm, and 
S.&nbsp;Brunak. Sensitive quantitative predictions of peptide-MHC binding by a 
'query by committee' artificial neural network approach.<em>Tissue Antigens</em>
, 62(5):378-384, Nov 2003. [&nbsp;bib&nbsp;] 
<blockquote>  We have generated Artificial Neural Networks (ANN) capable of 
performing sensitive, quantitative predictions of peptide binding to the MHC 
class I molecule, HLA-A*0204. We have shown that such quantitative ANN are 
superior to conventional classification ANN, that have been trained to predict 
binding vs non-binding peptides. Furthermore, quantitative ANN allowed a 
straightforward application of a 'Query by Committee' (QBC) principle whereby 
particularly information-rich peptides could be identified and subsequently 
tested experimentally. Iterative training based on QBC-selected peptides 
considerably increased the sensitivity without compromising the efficiency of 
the prediction. This suggests a general, rational and unbiased approach to the 
development of high quality predictions of epitopes restricted to this and 
other HLA molecules. Due to their quantitative nature, such predictions will 
cover a wide range of MHC-binding affinities of immunological interest, and 
they can be readily integrated with predictions of other events involved in 
generating immunogenic epitopes. These predictions have the capacity to perform 
rapid proteome-wide searches for epitopes. Finally, it is an example of an 
iterative feedback loop whereby advanced, computational bioinformatics optimize 
experimental strategy, and vice versa. </blockquote> 
<p></p> 
<blockquote>  Keywords: HLA-A Antigens; Humans; Neural Networks (Computer); 
Peptides; Protein Binding; Proteome; Research Support, Non-U.S. Gov't; Research 
Support, U.S. Gov't, P.H.S. </blockquote> <br>
 [Bhasin2003MHCBN]  Manoj 
Bhasin, Harpreet Singh, and G.&nbsp;P&nbsp;S Raghava. MHCBN: a comprehensive 
database of MHC binding and non-binding peptides.<em>Bioinformatics</em>, 
19(5):665-666, Mar 2003. [&nbsp;bib&nbsp;] 
<blockquote>  MHCBN is a comprehensive database of Major Histocompatibility 
Complex (MHC) binding and non-binding peptides compiled from published 
literature and existing databases. The latest version of the database has 19 
777 entries including 17 129 MHC binders and 2648 MHC non-binders for more than 
400 MHC molecules. The database has sequence and structure data of (a) source 
proteins of peptides and (b) MHC molecules. MHCBN has a number of web tools 
that include: (i) mapping of peptide on query sequence; (ii) search on any 
field; (iii) creation of data sets; and (iv) online data submission. The 
database also provides hypertext links to major databases like SWISS-PROT, PDB, 
IMGT/HLA-DB, GenBank and PUBMED. </blockquote> 
<p></p> 
<blockquote>  Keywords: Amino Acid Sequence; Binding Sites; Database 
Management Systems; Databases, Protein; Histocompatibility Antigens; 
Information Storage and Retrieval; Macromolecular Substances; Major 
Histocompatibility Complex; Molecular Sequence Data; Peptide Fragments; 
Peptides; Protein Binding; Protein Conformation; Sequence Alignment; Sequence 
Analysis, Protein; Structure-Activity Relationship; User-Computer Interface 
</blockquote> <br>
 [Bagirov2003New]  A.&nbsp;M. Bagirov, B.&nbsp;Ferguson, 
S.&nbsp;Ivkovic, G.&nbsp;Saunders, and J.&nbsp;Yearwood. New algorithms for 
multi-class cancer diagnosis using tumor gene expression signatures.<em>
Bioinformatics</em>, 19(14):1800-7, Sep 2003. [&nbsp;bib&nbsp;| http&nbsp;| .pdf
&nbsp;]
<blockquote>  MOTIVATION: The increasing use of DNA microarray-based tumor 
gene expression profiles for cancer diagnosis requires mathematical methods 
with high accuracy for solving clustering, feature selection and classification 
problems of gene expression data. RESULTS: New algorithms are developed for 
solving clustering, feature selection and classification problems of gene 
expression data. The clustering algorithm is based on optimization techniques 
and allows the calculation of clusters step-by-step. This approach allows us to 
find as many clusters as a data set contains with respect to some tolerance. 
Feature selection is crucial for a gene expression database. Our feature 
selection algorithm is based on calculating overlaps of different genes. The 
database used, contains over 16 000 genes and this number is considerably 
reduced by feature selection. We propose a classification algorithm where each 
tissue sample is considered as the center of a cluster which is a ball. The 
results of numerical experiments confirm that the classification algorithm in 
combination with the feature selection algorithm perform slightly better than 
the published results for multi-class classifiers based on support vector 
machines for this data set. AVAILABILITY: Available on request from the authors.
</blockquote> 
<p></p> 
<blockquote>  Keywords: Algorithms, Amino Acid Sequence, Anion Exchange 
Resins, Antigen-Antibody Complex, Artificial Intelligence, Automated, Automatic 
Data Processing, Biological, Blood Cells, Chemical, Chromatography, Cluster 
Analysis, Comparative Study, Computational Biology, Computer Simulation, 
Computer-Assisted, DNA, Data Interpretation, Databases, Decision Making, 
Decision Trees, Diffusion Magnetic Resonance Imaging, English Abstract, 
Epitopes, Expert Systems, Factual, Fuzzy Logic, Gene Expression Profiling, Gene 
Expression Regulation, Gene Targeting, Genetic, Genome, Histocompatibility 
Antigens Class I, Humans, Image Interpretation, Image Processing, In Vitro, 
Indicators and Reagents, Information Storage and Retrieval, Ion Exchange, 
Least-Squares Analysis, Liver Cirrhosis, Magnetic Resonance Imaging, Male, 
Models, Molecular Sequence Data, Neoplasms, Neoplastic, Neural Networks 
(Computer), Non-P.H.S., Non-U.S. Gov't, Nonl, Nucleic Acid Conformation, 
Oligonucleotide Array Sequence Analysis, P.H.S., Pattern Recognition, Pro, 
Prostatic Neoplasms, Protein, Protein Binding, Protein Interaction Mapping, 
Proteins, Quantitative Structure-Activity Relationship, RNA, ROC Curve, 
Reproducibility of Results, Research Support, Sensitivity and Specificity, 
Sequence Alignment, Sequence Analysis, Severity of Illness Index, Statistical, 
Structure-Activity Relationship, Subtraction Technique, T-Lymphocyte, 
Transcription Factors, Transfer, Treatment Outcome, Tumor Markers, U.S. Gov't, 
User-Computer Interface, inear Dynamics, teome, 14512351 </blockquote> <br>
 [
Attwood2003PRINTS]  T.&nbsp;K. Attwood, P.&nbsp;Bradley, D.&nbsp;R. Flower, 
A.&nbsp;Gaulton, N.&nbsp;Maudling, A.&nbsp;L. Mitchell, G.&nbsp;Moulton, 
A.&nbsp;Nordle, K.&nbsp;Paine, P.&nbsp;Taylor, A.&nbsp;Uddin, and 
C.&nbsp;Zygouri. Prints and its automatic supplement, preprints.<em>Nucleic 
Acids Res.</em>, 31(1):400-402, Jan 2003. [&nbsp;bib&nbsp;] 
<blockquote>  The PRINTS database houses a collection of protein fingerprints. 
These may be used to assign uncharacterised sequences to known families and 
hence to infer tentative functions. The September 2002 release (version 36.0) 
includes 1800 fingerprints, encoding approximately 11 000 motifs, covering a 
range of globular and membrane proteins, modular polypeptides and so on. In 
addition to its continued steady growth, we report here the development of an 
automatic supplement, prePRINTS, designed to increase the coverage of the 
resource and reduce some of the manual burdens inherent in its maintenance. The 
databases are accessible for interrogation and searching at 
http://www.bioinf.man.ac.uk/dbbrowser/PRINTS/. </blockquote> 
<p></p> 
<blockquote>  Keywords: Amino Acid Motifs; Animals; Automation; Conserved 
Sequence; Databases, Protein; Proteins; Software </blockquote> <br>
 [
Arakawa2003Application]  M.&nbsp;Arakawa, K.&nbsp;Hasegawa, and 
K.&nbsp;Funatsu. Application of the novel molecular alignment method using the 
Hopfield Neural Network to 3D-QSAR.<em>J Chem Inf Comput Sci</em>, 
43(5):1396-1402, 2003. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;] 
<blockquote>  Recently, we investigated and proposed the novel molecular 
alignment method with the Hopfield Neural Network (HNN). Molecules are 
represented by four kinds of chemical properties (hydrophobic group, 
hydrogen-bonding acceptor, hydrogen-bonding donor, and hydrogen-bonding 
donor/acceptor), and then those properties between two molecules correspond to 
each other using HNN. The 12 pairs of enzyme-inhibitors were used for 
validation, and our method could successfully reproduce the real molecular 
alignments obtained from X-ray crystallography. In this paper, we apply the 
molecular alignment method to three-dimensional quantitative structure-activity 
relationship (3D-QSAR) analysis. The two data sets (human epidermal growth 
factor receptor-2 inhibitors and cyclooxygenase-2 inhibitors) were investigated 
to validate our method. As a result, the robust and predictive 3D-QSAR models 
were successfully obtained in both data sets. </blockquote> 
<p></p> 
<blockquote>  Keywords: Chemical, Cyclooxygenase 2, Cyclooxygenase 2 
Inhibitors, Cyclooxygenase Inhibitors, Databases, Enzyme Inhibitors, Epidermal 
Growth Factor, Factual, Humans, Isoenzymes, Membrane Proteins, Models, 
Molecular, Neural Networks (Computer), Prostaglandin-Endoperoxide Synthases, 
Quantitative Structure-Activity Relationship, Receptor, 14502472 </blockquote> 
<br>  [Anderson2003new]  D.C. Anderson, W.&nbsp;Li, D.G. Payan, and W.S. Noble. 
A new algorithm for the evaluation of shotgun peptide sequencing in proteomics: 
support vector machine classification of peptide MS/MS spectra and SEQUEST 
scores.<em>J Proteome Res</em>, 2(2):137-146, 2003. [&nbsp;bib&nbsp;| .pdf
&nbsp;]
<blockquote>  Shotgun tandem mass spectrometry-based peptide sequencing using 
programs such as SEQUEST allows high-throughput identification of peptides, 
which in turn allows the identification of corresponding proteins. We have 
applied a machine learning algorithm, called the support vector machine, to 
discriminate between correctly and incorrectly identified peptides using 
SEQUEST output. Each peptide was characterized by SEQUEST-calculated features 
such as delta Cn and Xcorr, measurements such as precursor ion current and 
mass, and additional calculated parameters such as the fraction of matched 
MS/MS peaks. The trained SVM classifier performed significantly better than 
previous cutoff-based methods at separating positive from negative peptides. 
Positive and negative peptides were more readily distinguished in training set 
data acquired on a QTOF, compared to an ion trap mass spectrometer. The use of 
13 features, including four new parameters, significantly improved the 
separation between positive and negative peptides. Use of the support vector 
machine and these additional parameters resulted in a more accurate 
interpretation of peptide MS/MS spectra and is an important step toward 
automated interpretation of peptide tandem mass spectrometry data in proteomics.
</blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm proteomics </blockquote> <br>
 [Bleicher2003Hit]
 K.&nbsp;H. Bleicher, H.-J. B&ouml;hm, K.&nbsp;M&uuml;ller, and A.&nbsp;I. 
Alanine. Hit and lead generation: beyond high-throughput screening.<em>Nat Rev 
Drug Discov</em>, 2(5):369-378, May 2003. [&nbsp;bib&nbsp;| DOI&nbsp;| http
&nbsp;]
<blockquote>  The identification of small-molecule modulators of protein 
function, and the process of transforming these into high-content lead series, 
are key activities in modern drug discovery. The decisions taken during this 
process have far-reaching consequences for success later in lead optimization 
and even more crucially in clinical development. Recently, there has been an 
increased focus on these activities due to escalating downstream costs 
resulting from high clinical failure rates. In addition, the vast emerging 
opportunities from efforts in functional genomics and proteomics demands a 
departure from the linear process of identification, evaluation and refinement 
activities towards a more integrated parallel process. This calls for flexible, 
fast and cost-effective strategies to meet the demands of producing 
high-content lead series with improved prospects for clinical success. 
</blockquote> 
<p></p> 
<blockquote>  Keywords: Amino Acid Motifs, Combinatorial Chemistry Techniques, 
Drug Design, Drug Evaluation, Genomics, Preclinical, Proteomics, 12750740 
</blockquote> <br>
 [Yang2004Biological]  Zheng&nbsp;Rong Yang. Biological 
applications of support vector machines.<em>Brief Bioinform</em>, 5(4):328-38, 
Dec 2004. [&nbsp;bib&nbsp;] 
<blockquote>  One of the major tasks in bioinformatics is the classification 
and prediction of biological data. With the rapid increase in size of the 
biological databanks, it is essential to use computer programs to automate the 
classification process. At present, the computer programs that give the best 
prediction performance are support vector machines (SVMs). This is because SVMs 
are designed to maximise the margin to separate two classes so that the trained 
model generalises well on unseen data. Most other computer programs implement a 
classifier through the minimisation of error occurred in training, which leads 
to poorer generalisation. Because of this, SVMs have been widely applied to 
many areas of bioinformatics including protein function prediction, protease 
functional site recognition, transcription initiation site prediction and gene 
expression data classification. This paper will discuss the principles of SVMs 
and the applications of SVMs to the analysis of biological data, mainly protein 
and DNA sequences. </blockquote> 
<p></p> 
<blockquote>  Keywords: Acute, Adenocarcinoma, Algorithms, Amino Acid, Amino 
Acid Motifs, Amino Acid Sequence, Animals, Artificial Intelligence, Automated, 
B-Cell, Binding Sites, Cell Nucleus, Chemical, Child, Cluster Analysis, 
Comparative Study, Computational Biology, Computer Graphics, Computer 
Simulation, Computer-Assisted, Computing Methodologies, Cytoplasm, DNA, 
Databases, Decision Making, Decision Trees, Diagnosis, Fuzzy Logic, Gene 
Expression, Gene Expression Profiling, Gene Expression Regulation, Genetic, 
Genetic Vectors, Humans, Image Enhancement, Image Interpretation, Imaging, 
Information Storage and Retrieval, Internet, Leukemia, Leukemic, Lung 
Neoplasms, Lymphocytic, Lymphoma, Mitochondria, Models, Molecular, Molecular 
Sequence Data, Monte Carlo Method, Movement, Neoplasm Proteins, Neoplasms, 
Neural Networks (Computer), Non-P.H.S., Non-U.S. Gov't, Numerical Analysis, 
Oligonucleotide Array Sequence Analysis, P.H.S., Pattern Recognition, Peptides, 
Protein, Protein Binding, Protein Conformation, Protein Folding, Protein 
Interaction Mapping, Protein Structure, Proteins, Proteomics, ROC Curve, 
Reproducibility of Results, Research Support, Secondary, Sensitivity and 
Specificity, Sequence Alignment, Sequence Analysis, Sequence Homology, Signal 
Processing, Software, Statistical, Structural Homology, Structure-Activity 
Relationship, Subtraction Technique, Systems Biology, Tertiary, Theoretical, 
Three-Dimensional, U.S. Gov't, Unknown Primary, User-Computer Interface, Video 
Recording, 15606969 </blockquote> <br>
 [Xia2004RNAi]  H.&nbsp;Xia, 
Q.&nbsp;Mao, S.&nbsp;L. Eliason, S.&nbsp;Q. Harper, I.&nbsp;H. Martins, 
H.&nbsp;T. Orr, H.&nbsp;L. Paulson, L.&nbsp;Yang, R.&nbsp;M. Kotin, and 
B.&nbsp;L. Davidson. RNAi suppresses polyglutamine-induced neurodegeneration in 
a model of spinocerebellar ataxia.<em>Nat. Med.</em>, 10(8):816-820, Aug 2004. 
[&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;] 
<blockquote>  The dominant polyglutamine expansion diseases, which include 
spinocerebellar ataxia type 1 (SCA1) and Huntington disease, are progressive, 
untreatable, neurodegenerative disorders. In inducible mouse models of SCA1 and 
Huntington disease, repression of mutant allele expression improves disease 
phenotypes. Thus, therapies designed to inhibit expression of the mutant gene 
would be beneficial. Here we evaluate the ability of RNA interference (RNAi) to 
inhibit polyglutamine-induced neurodegeneration caused by mutant ataxin-1 in a 
mouse model of SCA1. Upon intracerebellar injection, recombinant 
adeno-associated virus (AAV) vectors expressing short hairpin RNAs profoundly 
improved motor coordination, restored cerebellar morphology and resolved 
characteristic ataxin-1 inclusions in Purkinje cells of SCA1 mice. Our data 
demonstrate in vivo the potential use of RNAi as therapy for dominant 
neurodegenerative disease. </blockquote> 
<p></p> 
<blockquote>  Keywords: Adenoviridae, Animal, Animals, Blotting, Brain, Cells, 
Comparative Study, Cultured, Disease Models, Gene Expression, Genetic, 
Glutamine, Immunohistochemistry, Messenger, Mice, Nerve Degeneration, Nerve 
Tissue Proteins, Non-U.S. Gov't, Northern, Nuclear Proteins, P.H.S., Plasmids, 
Psychomotor Performance, Purkinje Cells, RNA, RNA Interference, Research 
Support, Reverse Transcriptase Polymerase Chain Reaction, Small Interfering, 
Spinocerebellar Ataxias, Transduction, Transgenic, U.S. Gov't, 15286770 
</blockquote> <br>
 [Wang2004Simple]  Kai Wang, Ekachai Jenwitheesuk, Ram 
Samudrala, and John&nbsp;E Mittler. Simple linear model provides highly 
accurate genotypic predictions of HIV-1 drug resistance.<em>Antivir Ther</em>, 
9(3):343-52, Jun 2004. [&nbsp;bib&nbsp;] 
<blockquote>  Drug resistance is a major obstacle to the successful treatment 
of HIV-1 infection. Genotypic assays are used widely to provide indirect 
evidence of drug resistance, but the performance of these assays has been 
mixed. We used standard stepwise linear regression to construct drug resistance 
models for seven protease inhibitors and 10 reverse transcriptase inhibitors 
using data obtained from the Stanford HIV drug resistance database. We 
evaluated these models by hold-one-out experiments and by tests on an 
independent dataset. Our linear model outperformed other publicly available 
genotypic interpretation algorithms, including decision tree, support vector 
machine and four rules-based algorithms (HIVdb, VGI, ANRS and Rega) under both 
tests. Interestingly, our model did well despite the absence of any terms for 
interactions between different residues in protease or reverse transcriptase. 
The resulting linear models are easy to understand and can potentially assist 
in choosing combination therapy regimens. </blockquote> 
<p></p> 
<blockquote>  Keywords: Algorithms, Computational Biology, Databases, Drug 
Resistance, Forecasting, Genetic, Genotype, HIV Protease Inhibitors, HIV-1, 
Humans, Information Management, Information Storage and Retrieval, Kinetics, 
Linear Models, Microbial Sensitivity Tests, Models, Non-U.S. Gov't, P.H.S., 
Periodicals, Point Mutation, Pyrimidinones, Research Support, Reverse 
Transcriptase Inhibitors, Theoretical, U.S. Gov't, Viral, 15259897 </blockquote>
<br>  [Vallabhaneni2004Motor]  Anirudh Vallabhaneni and Bin He. Motor imagery 
task classification for brain computer interface applications using 
spatiotemporal principle component analysis.<em>Neurol Res</em>, 26(3):282-7, 
Apr 2004. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;] 
<blockquote>  Classification of single-trial imagined left- and right-hand 
movements recorded through scalp EEG are explored in this study. Classical 
event-related desynchronization/synchronization (ERD/ERS) calculation approach 
was utilized to extract ERD features from the raw scalp EEG signal. Principle 
Component Analysis (PCA) was used for feature extraction and applied on 
spatial, as well as temporal dimensions in two consecutive steps. A Support 
Vector Machine (SVM) classifier using a linear decision function was used to 
classify each trial as either left or right. The present approach has yielded 
good classification results and promises to have potential for further 
refinement for increased accuracy as well as application in online brain 
computer interface (BCI). </blockquote> 
<p></p> 
<blockquote>  Keywords: Amino Acids, Antibodies, Artificial Intelligence, 
Biological, Brain, Brain Mapping, Calibration, Comparative Study, Computational 
Biology, Cysteine, Cystine, Electrodes, Electroencephalography, Evoked 
Potentials, Female, Horseradish Peroxidase, Humans, Imagery (Psychotherapy), 
Imagination, Laterality, Male, Monoclonal, Movement, Neoplasms, Non-P.H.S., 
Non-U.S. Gov't, P.H.S., Perception, Principal Component Analysis, Protein, 
Protein Array Analysis, Proteins, Research Support, Sensitivity and 
Specificity, Sequence Analysis, Tumor Markers, U.S. Gov't, User-Computer 
Interface, 15142321 </blockquote> <br>
 [Tzeng2004Predicting]  Huey-Ming Tzeng, 
Jer-Guang Hsieh, and Yih-Lon Lin. Predicting nurses' intention to quit with a 
support vector machine: a new approach to set up an early warning mechanism in 
human resource management.<em>Comput Inform Nurs</em>, 22(4):232-42, 2004. 
[&nbsp;bib&nbsp;] 
<blockquote>  This project developed a Support Vector Machine for predicting 
nurses' intention to quit, using working motivation, job satisfaction, and 
stress levels as predictors. This study was conducted in three hospitals 
located in southern Taiwan. The target population was all nurses (389 valid 
cases). For cross-validation, we randomly split cases into four groups of 
approximately equal sizes, and performed four training runs. After the 
training, the average percentage of misclassification on the training data was 
0.86, while that on the testing data was 10.8, resulting in predictions with 
89.2% accuracy. This Support Vector Machine can predict nurses' intention to 
quit, without asking these nurses whether they have an intention to quit. 
</blockquote> 
<p></p> 
<blockquote>  Keywords: Adolescent, Adult, Algorithms, Amino Acid Sequence, 
Amino Acids, Anatomic, Attitude of Health Personnel, Bacterial Proteins, Bias 
(Epidemiology), Brain, Brain Mapping, Burnout, Comparative Study, Computer 
Simulation, Computer-Assisted, Data Interpretation, Diffusion Magnetic 
Resonance Imaging, Facial Asymmetry, Facial Expression, Facial Paralysis, 
Female, Gene Expression Profiling, Gram-Negative Bacteria, Gram-Positive 
Bacteria, Hospital, Humans, Image Interpretation, Intention, Job Satisfaction, 
Logistic Models, Magnetoencephalography, Male, Middle Aged, Models, Motion, 
Neural Networks (Computer), Neural Pathways, Non-U.S. Gov't, Nonlinear 
Dynamics, Nursing Administration Research, Nursing Staff, Personnel Management, 
Personnel Turnover, Photography, Predictive Value of Tests, Professional, 
Protein, Proteins, Proteome, Psychological, Questionnaires, Regression 
Analysis, Reproducibility of Results, Research Support, Retina, Risk Factors, 
Sequence Alignment, Sequence Analysis, Severity of Illness Index, Software, 
Statistical, Subcellular Fractions, Taiwan, Theoretical, Workplace, 15494654 
</blockquote> <br>
 [Tian2004novel]  Liang Tian and Afzel Noore. A novel 
approach for short-term load forecasting using support vector machines.<em>Int 
J Neural Syst</em>, 14(5):329-35, Oct 2004. [&nbsp;bib&nbsp;] 
<blockquote>  A support vector machine (SVM) modeling approach for short-term 
load forecasting is proposed. The SVM learning scheme is applied to the power 
load data, forcing the network to learn the inherent internal temporal property 
of power load sequence. We also study the performance when other related input 
variables such as temperature and humidity are considered. The performance of 
our proposed SVM modeling approach has been tested and compared with 
feed-forward neural network and cosine radial basis function neural network 
approaches. Numerical results show that the SVM approach yields better 
generalization capability and lower prediction error compared to those neural 
network approaches. </blockquote> 
<p></p> 
<blockquote>  Keywords: Acute, Adenocarcinoma, Algorithms, Amino Acid, Amino 
Acid Motifs, Amino Acid Sequence, Animals, Artificial Intelligence, Automated, 
B-Cell, Binding Sites, Biological, Cell Nucleus, Chemical, Child, Cluster 
Analysis, Comparative Study, Computational Biology, Computer Graphics, Computer 
Simulation, Computer-Assisted, Computing Methodologies, Cytoplasm, DNA, 
Databases, Decision Making, Decision Trees, Diagnosis, Discriminant Analysis, 
Fuzzy Logic, Gene Expression, Gene Expression Profiling, Gene Expression 
Regulation, Genetic, Genetic Vectors, Humans, Image Enhancement, Image 
Interpretation, Imaging, Information Storage and Retrieval, Internet, Leukemia, 
Leukemic, Lung Neoplasms, Lymphocytic, Lymphoma, Mitochondria, Models, 
Molecular, Molecular Sequence Data, Monte Carlo Method, Movement, Neoplasm 
Proteins, Neoplasms, Neoplastic, Neural Networks (Computer), Non-P.H.S., 
Non-U.S. Gov't, Numerical Analysis, Oligonucleotide Array Sequence Analysis, 
P.H.S., Pattern Recognition, Peptides, Predictive Value of Tests, Protein, 
Protein Binding, Protein Conformation, Protein Folding, Protein Interaction 
Mapping, Protein Structure, Proteins, Proteomics, ROC Curve, Reproducibility of 
Results, Research Support, Secondary, Sensitivity and Specificity, Sequence 
Alignment, Sequence Analysis, Sequence Homology, Signal Processing, Software, 
Statistical, Structural Homology, Structure-Activity Relationship, Subcellular 
Fractions, Subtraction Technique, Systems Biology, Tertiary, Theoretical, 
Three-Dimensional, Time Factors, Tumor Markers, U.S. Gov't, Unknown Primary, 
User-Computer Interface, Video Recording, 15593381 </blockquote> <br>
 [
Sun2004protein]  Zhenghong Sun, Xiaoli Fu, Lu&nbsp;Zhang, Xiaoli Yang, Feizhou 
Liu, and Gengxi Hu. A protein chip system for parallel analysis of multi-tumor 
markers and its application in cancer detection.<em>Anticancer Res</em>, 
24(2C):1159-65, 2004. [&nbsp;bib&nbsp;] 
<blockquote>  BACKGROUND: Tumor markers are routinely measured in clinical 
oncology. However, their value in cancer detection has been controversial 
largely because no single tumor marker is sensitive and specific enough to meet 
strict diagnostic criteria. One strategy to overcome the shortcomings of single 
tumor markers is to measure a combination of tumor markers to increase 
sensitivity and look for distinct patterns to increase specificity. This study 
aimed to develop a system for parallel detection of tumor markers as a tool for 
tumor detection in both cancer patients and asymptomatic populations at high 
risk. MATERIALS AND METHODS: A protein chip was fabricated with twelve 
monoclonal antibodies against the following tumor markers respectively: CA125, 
CA15-3, CA19-9, CA242, CEA, AFP, PSA, free-PSA, HGH, beta-HCG, NSE and 
ferritin. Tumor markers were captured after the protein chip was incubated with 
serum samples. A secondary antibody conjugated with HRP was used to detect the 
captured tumor markers using chemiluminescence technique. Quantification of the 
tumor markers was obtained after calibration with standard curves. RESULTS: The 
chip system showed an overall sensitivity of 68.18% after testing 1147 cancer 
patients, with high sensitivities for liver, pancreas and ovarian tumors and 
low sensitivities for gastrointestinal tumors, and a specificity of 97.1% after 
testing 793 healthy individuals. Application of the chip system in physical 
checkups of 15,867 individuals resulted in 16 cases that were subsequently 
confirmed as having cancers. Analysis of the detection results with a Support 
Vector Machine algorithm considerably increased the specificity of the system 
as reflected in healthy individuals and hepatitis/cirrhosis patients, but only 
modestly decreased the sensitivity for cancer patients. CONCLUSION: This 
protein chip system is a potential tool for assisting cancer diagnosis and for 
screening cancer in high-risk populations. </blockquote> 
<p></p> 
<blockquote>  Keywords: Antibodies, Artificial Intelligence, Biological, 
Calibration, Female, Horseradish Peroxidase, Humans, Male, Monoclonal, 
Neoplasms, Protein Array Analysis, Sensitivity and Specificity, Tumor Markers, 
15154641 </blockquote> <br>
 [Stahura2004Virtual]  Florence&nbsp;L Stahura and 
J&uuml;rgen Bajorath. Virtual screening methods that complement HTS.<em>Comb 
Chem High Throughput Screen</em>, 7(4):259-69, Jun 2004. [&nbsp;bib&nbsp;] 
<blockquote>  In this review, we discuss a number of computational methods 
that have been developed or adapted for molecule classification and virtual 
screening (VS) of compound databases. In particular, we focus on approaches 
that are complementary to high-throughput screening (HTS). The discussion is 
limited to VS methods that operate at the small molecular level, which is often 
called ligand-based VS (LBVS), and does not take into account docking 
algorithms or other structure-based screening tools. We describe areas that 
greatly benefit from combining virtual and biological screening and discuss 
computational methods that are most suitable to contribute to the integration 
of screening technologies. Relevant approaches range from established methods 
such as clustering or similarity searching to techniques that have only 
recently been introduced for LBVS applications such as statistical methods or 
support vector machines. Finally, we discuss a number of representative 
applications at the interface between VS and HTS. </blockquote> 
<p></p> 
<blockquote>  Keywords: Algorithms, Animals, Antisense, Artificial 
Intelligence, Cell Line, Cluster Analysis, Comparative Study, Computational 
Biology, Computer Simulation, DNA Fingerprinting, Drug Evaluation, 
Fluorescence, Fuzzy Logic, Gene Silencing, Gene Targeting, Genetic, Hela Cells, 
Humans, Imaging, Intracellular Space, Microscopy, Models, Neoplasms, Neural 
Networks (Computer), Non-U.S. Gov't, Oligonucleotides, P.H.S., Preclinical, 
Prognosis, Proteomics, Quantitative Structure-Activity Relationship, RNA, RNA 
Interference, Research Support, Sensitivity and Specificity, Small Interfering, 
Thionucleotides, Three-Dimensional, Tumor, U.S. Gov't, 15200375 </blockquote> 
<br>  [Song2004Comparison]  Xiaowei Song, Arnold Mitnitski, Jafna Cox, and 
Kenneth Rockwood. Comparison of machine learning techniques with classical 
statistical models in predicting health outcomes.<em>Medinfo</em>, 11(Pt 
1):736-40, 2004. [&nbsp;bib&nbsp;] 
<blockquote>  Several machine learning techniques (multilayer and single layer 
perceptron, logistic regression, least square linear separation and support 
vector machines) are applied to calculate the risk of death from two biomedical 
data sets, one from patient care records, and another from a population survey. 
Each dataset contained multiple sources of information: history of related 
symptoms and other illnesses, physical examination findings, laboratory tests, 
medications (patient records dataset), health attitudes, and disabilities in 
activities of daily living (survey dataset). Each technique showed very good 
mortality prediction in the acute patients data sample (AUC up to 0.89) and 
fair prediction accuracy for six year mortality (AUC from 0.70 to 0.76) in 
individuals from epidemiological database surveys. The results suggest that the 
nature of data is of primary importance rather than the learning technique. 
However, the consistently superior performance of the artificial neural network 
(multi-layer perceptron) indicates that nonlinear relationships (which cannot 
be discerned by linear separation techniques) can provide additional 
improvement in correctly predicting health outcomes. </blockquote> 
<p></p> 
<blockquote>  Keywords: Aged, Air, Algorithms, Amino Acids, Animals, Area 
Under Curve, Artifacts, Artificial Intelligence, Atrial, Automated, Canada, 
Carotid Stenosis, Cerebrovascular Accident, Cerebrovascular Circulation, 
Comparative Study, Computer-Assisted, Cysteine, Decision Trees, Dementia, 
Diagnosis, Disulfides, Doppler, Embolism, Expert Systems, Extramural, Factor 
Analysis, Female, Gene Expression, Gene Expression Profiling, Health Status, 
Heart Septal Defects, Humans, Intracranial Embolism, Male, Models, Molecular, 
Myocardial Infarction, N.I.H., Neoplasms, Neural Networks (Computer), Non-U.S. 
Gov't, Oligonucleotide Array Sequence Analysis, Oxidation-Reduction, P.H.S., 
Pattern Recognition, Prognosis, Protein Binding, Protein Folding, Proteins, ROC 
Curve, Research Support, Sensitivity and Specificity, Software, Statistical, 
Transcranial, Treatment Outcome, U.S. Gov't, Ultrasonography, 15360910 
</blockquote> <br>
 [Smith2004Towards]  P.&nbsp;A. Smith, M.&nbsp;J. Sorich, 
L.&nbsp;S&nbsp;C Low, R.&nbsp;A. McKinnon, and J.&nbsp;O. Miners. Towards 
integrated ADME prediction: past, present and future directions for modelling 
metabolism by UDP-glucuronosyltransferases.<em>J Mol Graph Model</em>, 
22(6):507-17, Jul 2004. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  Undesirable absorption, distribution, metabolism, excretion 
(ADME) properties are the cause of many drug development failures and this has 
led to the need to identify such problems earlier in the development process. 
This review highlights computational (in silico) approaches that have been used 
to identify the characteristics of ligands influencing molecular recognition 
and/or metabolism by the drug-metabolising enzyme UDP-gucuronosyltransferase 
(UGT). Current studies applying pharmacophore elucidation, 2D-quantitative 
structure metabolism relationships (2D-QSMR), 3D-quantitative structure 
metabolism relationships (3D-QSMR), and non-linear pattern recognition 
techniques such as artificial neural networks and support vector machines for 
modelling metabolism by UGT are reported. An assessment of the utility of in 
silico approaches for the qualitative and quantitative prediction of drug 
glucuronidation parameters highlights the benefit of using multiple 
pharmacophores and also non-linear techniques for classification. Some of the 
challenges facing the development of generalisable models for predicting 
metabolism by UGT, including the need for screening of more diverse structures, 
are also outlined. </blockquote> 
<p></p> 
<blockquote>  Keywords: Algorithms, Animals, Antisense, Artificial 
Intelligence, Astrocytoma, Automated, Autonomic Nervous System, Brain, Brain 
Neoplasms, Cell Line, Cerebral Cortex, Child, Cluster Analysis, Cognition, 
Comparative Study, Computational Biology, Computer Simulation, 
Computer-Assisted, DNA Fingerprinting, Databases, Diagnosis, Discriminant 
Analysis, Drug Design, Drug Evaluation, Electroencephalography, Emotions, 
Event-Related Potentials, Evoked Potentials, Factual, Fluorescence, Fuzzy 
Logic, Gene Silencing, Gene Targeting, Genetic, Glucuronosyltransferase, Hand, 
Hela Cells, Humans, Imaging, Intracellular Space, Magnetic Resonance 
Spectroscopy, Male, Meningeal Neoplasms, Meningioma, Microscopy, Models, 
Molecular Structure, Monitoring, Motor, Neoplasm Metastasis, Neoplasms, Neural 
Networks (Computer), Non-U.S. Gov't, Oligonucleotides, P.H.S., P300, Pattern 
Recognition, Peptides, Pharmaceutical Preparations, Physiologic, Preclinical, 
Predictive Value of Tests, Preschool, Prognosis, Protein Interaction Mapping, 
Protein Structure, Proteins, Proteomics, Quantitative Structure-Activity 
Relationship, Quaternary, RNA, RNA Interference, Recognition (Psychology), 
Reproducibility of Results, Research Support, Sensitivity and Specificity, 
Signal Processing, Small Interfering, Software, Thionucleotides, 
Three-Dimensional, Tumor, U.S. Gov't, User-Computer Interface, Word Processing, 
15182810 </blockquote> <br>
 [Shoeb2004Patient-specific]  Ali Shoeb, Herman 
Edwards, Jack Connolly, Blaise Bourgeois, S.&nbsp;Ted Treves, and John Guttag. 
Patient-specific seizure onset detection.<em>Epilepsy Behav</em>, 5(4):483-98, 
Aug 2004. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  This article presents an automated, patient-specific method for 
the detection of epileptic seizure onset from noninvasive 
electroencephalography. We adopt a patient-specific approach to exploit the 
consistency of an individual patient's seizure and nonseizure 
electroencephalograms. Our method uses a wavelet decomposition to construct a 
feature vector that captures the morphology and spatial distribution of an 
electroencephalographic epoch, and then determines whether that vector is 
representative of a patient's seizure or nonseizure electroencephalogram using 
the support vector machine classification algorithm. Our completely automated 
method was tested on noninvasive electroencephalograms from 36 pediatric 
subjects suffering from a variety of seizure types. It detected 131 of 139 
seizure events within 8.0+/-3.2 seconds of electrographic onset, and declared 
15 false detections in 60 hours of clinical electroencephalography. Our 
patient-specific method can be used to initiate delay-sensitive clinical 
procedures following seizure onset, for example, the injection of a functional 
imaging radiotracer. </blockquote> 
<p></p> 
<blockquote>  Keywords: Algorithms, Comparative Study, Computational Biology, 
Computer-Assisted, Databases, Diagnosis, Drug Resistance, 
Electroencephalography, Epilepsy, Forecasting, Genetic, Genotype, HIV Protease 
Inhibitors, HIV-1, Humans, Information Management, Information Storage and 
Retrieval, Kinetics, Linear Models, Microbial Sensitivity Tests, Models, 
Monitoring, Non-U.S. Gov't, P.H.S., Periodicals, Physiologic, Point Mutation, 
Pyrimidinones, Reaction Time, Research Support, Reverse Transcriptase 
Inhibitors, Signal Processing, Theoretical, Time Factors, U.S. Gov't, Viral, 
15256184 </blockquote> <br>
 [Seeger2004Gaussian]  Matthias Seeger. Gaussian 
processes for machine learning.<em>Int J Neural Syst</em>, 14(2):69-106, Apr 
2004. [&nbsp;bib&nbsp;] 
<blockquote>  Gaussian processes (GPs) are natural generalisations of 
multivariate Gaussian random variables to infinite (countably or continuous) 
index sets. GPs have been applied in a large number of fields to a diverse 
range of ends, and very many deep theoretical analyses of various properties 
are available. This paper gives an introduction to Gaussian processes on a 
fairly elementary level with special emphasis on characteristics relevant in 
machine learning. It draws explicit connections to branches such as spline 
smoothing models and support vector machines in which similar ideas have been 
investigated. Gaussian process models are routinely used to solve hard machine 
learning problems. They are attractive because of their flexible non-parametric 
nature and computational simplicity. Treated within a Bayesian framework, very 
powerful statistical methods can be implemented which offer valid estimates of 
uncertainties in our predictions and generic model selection procedures cast as 
nonlinear optimization problems. Their main drawback of heavy computational 
scaling has recently been alleviated by the introduction of generic sparse 
approximations.13,78,31 The mathematical literature on GPs is large and often 
uses deep concepts which are not required to fully understand most machine 
learning applications. In this tutorial paper, we aim to present 
characteristics of GPs relevant to machine learning and to show up precise 
connections to other &quot;kernel machines&quot; popular in the community. Our 
focus is on a simple presentation, but references to more detailed sources are 
provided. </blockquote> 
<p></p> 
<blockquote>  Keywords: Algorithms, Amino Acids, Antibodies, Artificial 
Intelligence, Astrocytoma, Automated, Bayes Theorem, Biological, Biopsy, Brain, 
Brain Mapping, Brain Neoplasms, Calibration, Comparative Study, Computational 
Biology, Computer-Assisted, Computing Methodologies, Cysteine, Cystine, 
Dysplastic Nevus Syndrome, Electrodes, Electroencephalography, Entropy, Eosine 
Yellowish-(YS), Evoked Potentials, Female, Gene Expression Profiling, 
Hematoxylin, Horseradish Peroxidase, Humans, Image Interpretation, Image 
Processing, Imagery (Psychotherapy), Imagination, Laterality, Linear Models, 
Male, Melanoma, Models, Monoclonal, Movement, Neoplasms, Neural Networks 
(Computer), Neuropeptides, Non-P.H.S., Non-U.S. Gov't, Nonparametric, Normal 
Distribution, P.H.S., Pattern Recognition, Perception, Principal Component 
Analysis, Protein, Protein Array Analysis, Protein Interaction Mapping, 
Proteins, Regression Analysis, Research Support, Sensitivity and Specificity, 
Sequence Alignment, Sequence Ana, Sequence Analysis, Skin Neoplasms, Software, 
Statistical, Statistics, Tumor Markers, U.S. Gov't, User-Computer Interface, 
World Health Organization, lysis, 15112367 </blockquote> <br>
 [
Qin2004[Automated]  Dong mei Qin, Zhan yi&nbsp;Hu, and Yong heng Zhao. 
Automated classification of celestial spectra based on support vector machines.
<em>Guang Pu Xue Yu Guang Pu Fen Xi</em>, 24(4):507-11, Apr 2004. [&nbsp;bib
&nbsp;]
<blockquote>  The main objective of an automatic recognition system of 
celestial objects via their spectra is to classify celestial spectra and 
estimate physical parameters automatically. This paper proposes a new automatic 
classification method based on support vector machines to separate non-active 
objects from active objects via their spectra. With low SNR and unknown 
red-shift value, it is difficult to extract true spectral lines, and as a 
result, active objects can not be determined by finding strong spectral lines 
and the spectral classification between non-active and active objects becomes 
difficult. The proposed method in this paper combines the principal component 
analysis with support vector machines, and can automatically recognize the 
spectra of active objects with unknown red-shift values from non-active 
objects. It finds its applicability in the automatic processing of voluminous 
observed data from large sky surveys in astronomy. </blockquote> 
<p></p> 
<blockquote>  Keywords: 80 and over, Adult, Aged, Algorithms, Amino Acids, 
Animals, Area Under Curve, Artifacts, Automated, Birefringence, Brain 
Chemistry, Brain Neoplasms, Comparative Study, Computer-Assisted, Cornea, 
Cross-Sectional Studies, Decision Trees, Diagnosis, Diagnostic Imaging, 
Diagnostic Techniques, Discriminant Analysis, Evolution, Face, Female, Genetic, 
Glaucoma, Humans, Intraocular Pressure, Lasers, Least-Squares Analysis, 
Magnetic Resonance Imaging, Magnetic Resonance Spectroscopy, Male, Middle Aged, 
Models, Molecular, Nerve Fibers, Non-U.S. Gov't, Numerical Analysis, 
Ophthalmological, Optic Nerve Diseases, Optical Coherence, P.H.S., Pattern 
Recognition, Photic Stimulation, Prospective Studies, Protein, ROC Curve, 
Regression Analysis, Research Support, Retinal Ganglion Cells, Sensitivity and 
Specificity, Sequence Analysis, Statistics, Tomography, U.S. Gov't, Visual 
Fields, beta-Lactamases, 15766170 </blockquote> <br>
 [Prados2004Mining]  
J.&nbsp;Prados, A.&nbsp;Kalousis, J.C. Sanchez, L.&nbsp;Allard, 
O.&nbsp;Carrette, and M.&nbsp;Hilario. Mining mass spectra for diagnosis and 
biomarker discovery of cerebral accidents.<em>Proteomics</em>, 4(8):2320-2332, 
2004. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  In this paper we try to identify potential biomarkers for early 
stroke diagnosis using surface-enhanced laser desorption/ionization mass 
spectrometry coupled with analysis tools from machine learning and data mining. 
Data consist of 42 specimen samples, i.e., mass spectra divided in two big 
categories, stroke and control specimens. Among the stroke specimens two 
further categories exist that correspond to ischemic and hemorrhagic stroke; in 
this paper we limit our data analysis to discriminating between control and 
stroke specimens. We performed two suites of experiments. In the first one we 
simply applied a number of different machine learning algorithms; in the second 
one we have chosen the best performing algorithm as it was determined from the 
first phase and coupled it with a number of different feature selection 
methods. The reason for this was 2-fold, first to establish whether feature 
selection can indeed improve performance, which in our case it did not seem to 
confirm, but more importantly to acquire a small list of potentially 
interesting biomarkers. Of the different methods explored the most promising 
one was support vector machines which gave us high levels of sensitivity and 
specificity. Finally, by analyzing the models constructed by support vector 
machines we produced a small set of 13 features that could be used as potential 
biomarkers, and which exhibited good performance both in terms of sensitivity, 
specificity and model stability. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm proteomics </blockquote> <br>
 [
Pavlidis2004Support]  Paul Pavlidis, Ilan Wapinski, and William&nbsp;Stafford 
Noble. Support vector machine classification on the web.<em>Bioinformatics</em>
, 20(4):586-7, Mar 2004. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  The support vector machine (SVM) learning algorithm has been 
widely applied in bioinformatics. We have developed a simple web interface to 
our implementation of the SVM algorithm, called Gist. This interface allows 
novice or occasional users to apply a sophisticated machine learning algorithm 
easily to their data. More advanced users can download the software and source 
code for local installation. The availability of these tools will permit more 
widespread application of this powerful learning algorithm in bioinformatics. 
</blockquote> 
<p></p> 
<blockquote>  Keywords: Adaptation, Algorithms, Ambergris, Amino Acid 
Sequence, Animals, Artifacts, Artificial Intelligence, Automated, Cadmium, 
Candida, Candida albicans, Capillary, Clinical, Cluster Analysis, Combinatorial 
Chemistry Techniques, Comparative Study, Computational Biology, Computer 
Simulation, Computer-Assisted, Computing Methodologies, Databases, Decision 
Support Systems, Electrophoresis, Enzymes, Europe, Eye Enucleation, Humans, 
Image Interpretation, Image Processing, Information Storage and Retrieval, 
Internet, Magnetic Resonance Imaging, Magnetic Resonance Spectroscopy, Markov 
Chains, Melanoma, Models, Molecular, Molecular Conformation, Molecular Sequence 
Data, Molecular Structure, Neural Networks (Computer), Non-P.H.S., Non-U.S. 
Gov't, Nonlinear Dynamics, Odors, P.H.S., Pattern Recognition, Perfume, 
Physiological, Predictive Value of Tests, Prognosis, Prospective Studies, 
Protein, Protein Structure, Proteins, Proteomics, Quantitative 
Structure-Activity Relationship, Rats, Reproducibility of Results, Research 
Support, Saccharomyces cerevisiae, Saccharomyces cerevisiae Proteins, 
Secondary, Sensitivity and Specificity, Signal Processing, Single-Blind Method, 
Soft Tissue Neoplasms, Software, Statistical, U.S. Gov't, Uveal Neoplasms, 
Visual, 14990457 </blockquote> <br>
 [Lin2004Classification]  WuMei Lin, Xin 
Yuan, Powing Yuen, William&nbsp;I Wei, Jonathan Sham, PengCheng Shi, and Jianan 
Qu. Classification of in vivo autofluorescence spectra using support vector 
machines.<em>J Biomed Opt</em>, 9(1):180-6, 2004. [&nbsp;bib&nbsp;| DOI&nbsp;| 
http&nbsp;| .pdf&nbsp;] 
<blockquote>  An algorithm based on support vector machines (SVM), the most 
recent advance in pattern recognition, is presented for use in classifying 
light-induced autofluorescence collected from cancerous and normal tissues. The 
in vivo autofluorescence spectra used for development and evaluation of SVM 
diagnostic algorithms were measured from 85 nasopharyngeal carcinoma (NPC) 
lesions and 131 normal tissue sites from 59 subjects during routine nasal 
endoscopy. Leave-one-out cross-validation was used to evaluate the performance 
of the algorithms. An overall diagnostic accuracy of 96%, a sensitivity of 94%, 
and a specificity of 97% for discriminating nasopharyngeal carcinomas from 
normal tissues were achieved using a linear SVM algorithm. A diagnostic 
accuracy of 98%, a sensitivity of 95%, and a specificity of 99% for detecting 
NPC were achieved with a nonlinear SVM algorithm. In a comparison with 
previously developed algorithms using the same dataset and the principal 
component analysis (PCA) technique, the SVM algorithms produced better 
diagnostic accuracy in all instances. In addition, we investigated a method 
combining PCA and SVM techniques for reducing the complexity of the SVM 
algorithms. </blockquote> 
<p></p> 
<blockquote>  Keywords: 80 and over, Abstracting and Indexing, Aged, 
Algorithms, Amino Acid Sequence, Amino Acids, Animals, Antibody, Antigens, 
Aromatic, Artificial Intelligence, Atrophy, Automated, BRCA1, Bibliographic, 
Binding, Binding Sites, Brain, Brain Mapping, Carrier Proteins, Cell Cycle 
Proteins, Chemical, Cluster Analysis, Comparative Study, Competitive, 
Complementary, Computational Biology, Computer Simulation, Computer-Assisted, 
Computers, Computing Methodologies, Controlled, DNA, DNA Damage, DNA-Binding 
Proteins, Data Interpretation, Database Management Systems, Databases, 
Diagnosis, Drug Design, Electrostatics, Endoscopy, Epitopes, Evolution, Female, 
Fibroblasts, Fluorescence, Fourier Analysis, Gene Expression Profiling, Gene 
Expression Regulation, Genes, HLA-DR Antigens, Heterozygote, Histocompatibility 
Antigens Class I, Humans, Hydrophobicity, Image Processing, Imaging, 
Information Storage and Retrieval, Isoelectric Point, Linear Models, 
Linguistics, Longitudinal Studies, MEDLINE, Magnetic Resonance Imaging, Male, 
Membrane Proteins, Middle Aged, Models, Molecular, Molecular Sequence Data, 
Multivariate Analysis, Mutation, Nasopharyngeal Neoplasms, Natural Language 
Processing, Neoplastic, Neural Networks (Computer), Non-P.H.S., Non-U.S. Gov't, 
Nonlinear Dynamics, Numerical Analysis, Oligonucleotide Array Sequence 
Analysis, Oligopeptides, P.H.S., Pattern Recognition, Peptide Fragments, 
Periodicals, Protein, Protein Binding, Protein Conformation, Protein Sorting 
Signals, Protein Structure, Protein-Serine-Threonine Kinases, Proteins, 
Proteome, Proteomics, RNA-Binding Proteins, ROC Curve, Randomized Controlled 
Trials, Reproducibility of Results, Research Support, Saccharomyces cerevisiae 
Proteins, Sensitivity and Specificity, Sequence Alignment, Sequence Analysis, 
Software, Solvents, Spectrometry, Statistical, Structural Homology, 
Structure-Activity Relationship, T-Lymphocyte, Terminology, Tertiary, 
Three-Dimensional, Tumor Suppressor Proteins, U.S. Gov't, Vocabulary, 14715071 
</blockquote> <br>
 [Li2004Fusing]  Shutao Li, James Tin-Yau Kwok, Ivor 
Wai-Hung Tsang, and Yaonan Wang. Fusing images with different focuses using 
support vector machines.<em>IEEE Trans Neural Netw</em>, 15(6):1555-61, Nov 
2004. [&nbsp;bib&nbsp;] 
<blockquote>  Many vision-related processing tasks, such as edge detection, 
image segmentation and stereo matching, can be performed more easily when all 
objects in the scene are in good focus. However, in practice, this may not be 
always feasible as optical lenses, especially those with long focal lengths, 
only have a limited depth of field. One common approach to recover an 
everywhere-in-focus image is to use wavelet-based image fusion. First, several 
source images with different focuses of the same scene are taken and processed 
with the discrete wavelet transform (DWT). Among these wavelet decompositions, 
the wavelet coefficient with the largest magnitude is selected at each pixel 
location. Finally, the fused image can be recovered by performing the inverse 
DWT. In this paper, we improve this fusion procedure by applying the discrete 
wavelet frame transform (DWFT) and the support vector machines (SVM). Unlike 
DWT, DWFT yields a translation-invariant signal representation. Using features 
extracted from the DWFT coefficients, a SVM is trained to select the source 
image that has the best focus at each pixel location, and the corresponding 
DWFT coefficients are then incorporated into the composite wavelet 
representation. Experimental results show that the proposed method outperforms 
the traditional approach both visually and quantitatively. </blockquote> 
<p></p> 
<blockquote>  Keywords: Algorithms, Amino Acid, Amino Acids, Artificial 
Intelligence, Ascomycota, Automated, Base Sequence, Chromosome Mapping, Codon, 
Colonic Neoplasms, Comparative Study, Computer Simulation, Computer-Assisted, 
Computing Methodologies, Crystallography, DNA, DNA Primers, Databases, 
Diagnostic Imaging, Enzymes, Fixation, Gene Expression Profiling, Genetic, 
Hordeum, Host-Parasite Relations, Humans, Image Enhancement, Image 
Interpretation, Informatics, Information Storage and Retrieval, Kinetics, 
Magnetic Resonance Spectroscopy, Models, Nanotechnology, Neural Networks 
(Computer), Non-P.H.S., Non-U.S. Gov't, Ocular, Oligonucleotide Array Sequence 
Analysis, P.H.S., Pattern Recognition, Plant, Plants, Predictive Value of 
Tests, Protein, Protein Conformation, Research Support, Sample Size, Selection 
(Genetics), Sequence Alignment, Sequence Analysis, Sequence Homology, Signal 
Processing, Skin, Software, Statistical, Subtraction Technique, Theoretical, 
Thermodynamics, U.S. Gov't, Viral Proteins, X-Ray, 15565781 </blockquote> <br>
 
[Lao2004Morphological]  Zhiqiang Lao, Dinggang Shen, Zhong Xue, Bilge Karacali, 
Susan&nbsp;M Resnick, and Christos Davatzikos. Morphological classification of 
brains via high-dimensional shape transformations and machine learning methods.
<em>Neuroimage</em>, 21(1):46-57, Jan 2004. [&nbsp;bib&nbsp;] 
<blockquote>  A high-dimensional shape transformation posed in a 
mass-preserving framework is used as a morphological signature of a brain 
image. Population differences with complex spatial patterns are then determined 
by applying a nonlinear support vector machine (SVM) pattern classification 
method to the morphological signatures. Significant reduction of the 
dimensionality of the morphological signatures is achieved via wavelet 
decomposition and feature reduction methods. Applying the method to MR images 
with simulated atrophy shows that the method can correctly detect subtle and 
spatially complex atrophy, even when the simulated atrophy represents only a 5% 
variation from the original image. Applying this method to actual MR images 
shows that brains can be correctly determined to be male or female with a 
successful classification rate of 97%, using the leave-one-out method. This 
proposed method also shows a high classification rate for old adults' age 
classification, even under difficult test scenarios. The main characteristic of 
the proposed methodology is that, by applying multivariate pattern 
classification methods, it can detect subtle and spatially complex patterns of 
morphological group differences which are often not detectable by voxel-based 
morphometric methods, because these methods analyze morphological measurements 
voxel-by-voxel and do not consider the entirety of the data simultaneously. 
</blockquote> 
<p></p> 
<blockquote>  Keywords: 80 and over, Abstracting and Indexing, Aged, 
Algorithms, Amino Acid Sequence, Amino Acids, Animals, Antibody, Antigens, 
Aromatic, Artificial Intelligence, Atrophy, Automated, BRCA1, Bibliographic, 
Binding, Binding Sites, Brain, Brain Mapping, Carrier Proteins, Cell Cycle 
Proteins, Chemical, Cluster Analysis, Comparative Study, Competitive, 
Complementary, Computational Biology, Computer Simulation, Computer-Assisted, 
Computing Methodologies, Controlled, DNA, DNA Damage, DNA-Binding Proteins, 
Data Interpretation, Database Management Systems, Databases, Drug Design, 
Electrostatics, Epitopes, Evolution, Female, Fibroblasts, Fourier Analysis, 
Gene Expression Profiling, Gene Expression Regulation, Genes, HLA-DR Antigens, 
Heterozygote, Histocompatibility Antigens Class I, Humans, Hydrophobicity, 
Image Processing, Imaging, Information Storage and Retrieval, Longitudinal 
Studies, Magnetic Resonance Imaging, Male, Membrane Proteins, Middle Aged, 
Models, Molecular, Molecular Sequence Data, Multivariate Analysis, Mutation, 
Natural Language Processing, Neoplastic, Neural Networks (Computer), 
Non-P.H.S., Non-U.S. Gov't, Nonlinear Dynamics, Numerical Analysis, 
Oligonucleotide Array Sequence Analysis, Oligopeptides, P.H.S., Pattern 
Recognition, Peptide Fragments, Periodicals, Protein, Protein Binding, Protein 
Conformation, Protein Sorting Signals, Protein Structure, 
Protein-Serine-Threonine Kinases, Proteins, Proteome, Proteomics, RNA-Binding 
Proteins, Reproducibility of Results, Research Support, Saccharomyces 
cerevisiae Proteins, Sensitivity and Specificity, Sequence Alignment, Sequence 
Analysis, Software, Solvents, Statistical, Structural Homology, 
Structure-Activity Relationship, T-Lymphocyte, Terminology, Tertiary, 
Three-Dimensional, Tumor Suppressor Proteins, U.S. Gov't, Vocabulary, 14741641 
</blockquote> <br>
 [Lal2004Support]  Thomas&nbsp;Navin Lal, Michael 
Schr&ouml;der, Thilo Hinterberger, Jason Weston, Martin Bogdan, Niels 
Birbaumer, and Bernhard Sch&ouml;lkopf. Support vector channel selection in BCI.
<em>IEEE Trans Biomed Eng</em>, 51(6):1003-10, Jun 2004. [&nbsp;bib&nbsp;] 
<blockquote>  Designing a brain computer interface (BCI) system one can choose 
from a variety of features that may be useful for classifying brain activity 
during a mental task. For the special case of classifying electroencephalogram 
(EEG) signals we propose the usage of the state of the art feature selection 
algorithms Recursive Feature Elimination and Zero-Norm Optimization which are 
based on the training of support vector machines (SVM). These algorithms can 
provide more accurate solutions than standard filter methods for feature 
selection. We adapt the methods for the purpose of selecting EEG channels. For 
a motor imagery paradigm we show that the number of used channels can be 
reduced significantly without increasing the classification error. The 
resulting best channels agree well with the expected underlying cortical 
activity patterns during the mental tasks. Furthermore we show how time 
dependent task specific information can be visualized. </blockquote> 
<p></p> 
<blockquote>  Keywords: Algorithms, Animals, Antisense, Artificial 
Intelligence, Automated, Autonomic Nervous System, Brain, Cell Line, Cerebral 
Cortex, Child, Cluster Analysis, Cognition, Comparative Study, Computational 
Biology, Computer Simulation, Computer-Assisted, DNA Fingerprinting, Databases, 
Drug Evaluation, Electroencephalography, Emotions, Event-Related Potentials, 
Evoked Potentials, Factual, Fluorescence, Fuzzy Logic, Gene Silencing, Gene 
Targeting, Genetic, Hand, Hela Cells, Humans, Imaging, Intracellular Space, 
Male, Microscopy, Models, Monitoring, Motor, Neoplasms, Neural Networks 
(Computer), Non-U.S. Gov't, Oligonucleotides, P.H.S., P300, Pattern 
Recognition, Peptides, Physiologic, Preclinical, Predictive Value of Tests, 
Preschool, Prognosis, Protein Interaction Mapping, Protein Structure, Proteins, 
Proteomics, Quantitative Structure-Activity Relationship, Quaternary, RNA, RNA 
Interference, Recognition (Psychology), Reproducibility of Results, Research 
Support, Sensitivity and Specificity, Signal Processing, Small Interfering, 
Software, Thionucleotides, Three-Dimensional, Tumor, U.S. Gov't, User-Computer 
Interface, Word Processing, 15188871 </blockquote> <br>
 [Kharchenko2004Filling]
 P.&nbsp;Kharchenko, D.&nbsp;Vitkup, and G.&nbsp;M. Church. Filling gaps in a 
metabolic network using expression information.<em>Bioinformatics</em>, 20 
Suppl 1:I178-I185, Aug 2004. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;] 
<blockquote>  MOTIVATION: The metabolic models of both newly sequenced and 
well-studied organisms contain reactions for which the enzymes have not been 
identified yet. We present a computational approach for identifying genes 
encoding such missing metabolic enzymes in a partially reconstructed metabolic 
network. RESULTS: The metabolic expression placement (MEP) method relies on the 
coexpression properties of the metabolic network and is complementary to the 
sequence homology and genome context methods that are currently being used to 
identify missing metabolic genes. The MEP algorithm predicts over 20% of all 
known Saccharomyces cerevisiae metabolic enzyme-encoding genes within the top 
50 out of 5594 candidates for their enzymatic function, and 70% of metabolic 
genes whose expression level has been significantly perturbed across the 
conditions of the expression dataset used. AVAILABILITY: Freely available (in 
Supplementary information). SUPPLEMENTARY INFORMATION: Available at the 
following URL http://arep.med.harvard.edu/kharchenko/mep/supplements.html 
</blockquote> 
<p></p> 
<blockquote>  Keywords: Bacterial, Binding Sites, Biological, Comparative 
Study, DNA, Energy Metabolism, Enzyme Induction, Enzymes, Escherichia coli 
Proteins, Fungal, Gene Expression Regulation, Genes, Genetic, Genome, Models, 
Non-P.H.S., Non-U.S. Gov't, Phylogeny, Promoter Regions (Genetics), Protein, 
Research Support, Saccharomyces cerevisiae, Saccharomyces cerevisiae Proteins, 
Sequence Analysis, Systems Biology, Transcription Factors, U.S. Gov't, 15262797 
</blockquote> <br>
 [Kellenberger2004Comparative]  E.&nbsp;Kellenberger, 
J.&nbsp;Rodrigo, P.&nbsp;Muller, and D.&nbsp;Rognan. Comparative evaluation of 
eight docking tools for docking and virtual screening accuracy.<em>Proteins</em>
, 57(2):225-242, Nov 2004. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;] 
<blockquote>  Eight docking programs (DOCK, FLEXX, FRED, GLIDE, GOLD, SLIDE, 
SURFLEX, and QXP) that can be used for either single-ligand docking or database 
screening have been compared for their propensity to recover the X-ray pose of 
100 small-molecular-weight ligands, and for their capacity to discriminate 
known inhibitors of an enzyme (thymidine kinase) from randomly chosen 
&quot;drug-like&quot; molecules. Interestingly, both properties are found to be 
correlated, since the tools showing the best docking accuracy (GLIDE, GOLD, and 
SURFLEX) are also the most successful in ranking known inhibitors in a virtual 
screening experiment. Moreover, the current study pinpoints some 
physicochemical descriptors of either the ligand or its cognate protein-binding 
site that generally lead to docking/scoring inaccuracies. </blockquote> 
<p></p> 
<blockquote>  Keywords: Algorithms, Binding Sites, Biomimetic Materials, 
Chemical, Cluster Analysis, Comparative Study, Computer Graphics, Computer 
Simulation, Crystallography, Databases, Digital, Drug Design, HLA-DR4 Antigen, 
Herpesvirus 1, Human, Libraries, Ligands, Models, Molecular, Molecular Mimicry, 
Molecular Structure, Non-U.S. Gov't, Protein, Protein Binding, Protein 
Conformation, Protein Interaction Mapping, Protein Structure, Proteins, 
Quantitative Structure-Activity Relationship, Quaternary, Reproducibility of 
Results, Research Support, Sequence Analysis, Software, Structure-Activity 
Relationship, Thymidine Kinase, User-Computer Interface, Viral Proteins, X-Ray, 
15340911 </blockquote> <br>
 [Kaper2004BCI]  Matthias Kaper, Peter Meinicke, 
Ulf Grossekathoefer, Thomas Lingner, and Helge Ritter. BCI Competition 
2003-Data set IIb: support vector machines for the P300 speller paradigm.<em>
IEEE Trans Biomed Eng</em>, 51(6):1073-6, Jun 2004. [&nbsp;bib&nbsp;] 
<blockquote>  We propose an approach to analyze data from the P300 speller 
paradigm using the machine-learning technique support vector machines. In a 
conservative classification scheme, we found the correct solution after five 
repetitions. While the classification within the competition is designed for 
offline analysis, our approach is also well-suited for a real-world online 
solution: It is fast, requires only 10 electrode positions and demands only a 
small amount of preprocessing. </blockquote> 
<p></p> 
<blockquote>  Keywords: Algorithms, Animals, Antisense, Artificial 
Intelligence, Automated, Autonomic Nervous System, Brain, Cell Line, Child, 
Cluster Analysis, Cognition, Comparative Study, Computational Biology, Computer 
Simulation, Computer-Assisted, DNA Fingerprinting, Databases, Drug Evaluation, 
Electroencephalography, Emotions, Event-Related Potentials, Factual, 
Fluorescence, Fuzzy Logic, Gene Silencing, Gene Targeting, Genetic, Hela Cells, 
Humans, Imaging, Intracellular Space, Microscopy, Models, Monitoring, 
Neoplasms, Neural Networks (Computer), Non-U.S. Gov't, Oligonucleotides, 
P.H.S., P300, Pattern Recognition, Peptides, Physiologic, Preclinical, 
Predictive Value of Tests, Preschool, Prognosis, Protein Interaction Mapping, 
Protein Structure, Proteins, Proteomics, Quantitative Structure-Activity 
Relationship, Quaternary, RNA, RNA Interference, Recognition (Psychology), 
Reproducibility of Results, Research Support, Sensitivity and Specificity, 
Signal Processing, Small Interfering, Software, Thionucleotides, 
Three-Dimensional, Tumor, U.S. Gov't, User-Computer Interface, Word Processing, 
15188881 </blockquote> <br>
 [Hizukuri2004Extraction]  Yoshiyuki Hizukuri, 
Yoshihiro Yamanishi, Kosuke Hashimoto, and Minoru Kanehisa. Extraction of 
species-specific glycan substructures.<em>Genome Inform Ser Workshop Genome 
Inform</em>, 15(1):69-81, 2004. [&nbsp;bib&nbsp;| .html&nbsp;| .pdf&nbsp;] 
<blockquote>  Glycans, which are carbohydrate sugar chains attached to some 
lipids or proteins, have a huge variety of structures and play a key role in 
cell communication, protein interaction and immunity. The availability of a 
number of glycan structures stored in the KEGG/GLYCAN database makes it 
possible for us to conduct a large-scale comparative research of glycans. In 
this paper, we present a novel approach to compare glycan structures and 
extract characteristic glycan substructures of certain organisms. In the 
algorithm we developed a new similarity measure of glycan structures taking 
into account of several biological aspects of glycan synthesis and 
glycosyltransferases, and we confirmed the validity of our similarity measure 
by conducting experiments on its ability to classify glycans between organisms 
in the framework of a support vector machine. Finally, our method successfully 
extracted a set of candidates of substructrues which are characteristic to 
human, rat, mouse, bovine, pig, chicken, yeast, wheat and sycamore, 
respectively. We confirmed that the characteristic substructures extracted by 
our method correspond to the substructures which are known as the 
species-specific sugar chain of gamma-glutamyltranspeptidases in the kidney. 
</blockquote> 
<p></p> 
<blockquote>  Keywords: Amino Acid Sequence, Animals, Carbohydrate 
Conformation, Carbohydrate Sequence, Cattle, Computer Simulation, Databases, 
Genes, Histocompatibility Antigens Class I, Humans, Least-Squares Analysis, MHC 
Class I, Major Histocompatibility Complex, Mice, Monosaccharides, Non-U.S. 
Gov't, Peptides, Phylogeny, Plants, Polysaccharides, Protein, Rats, Research 
Support, Saccharomyces cerevisiae, Species Specificity, 15712111 </blockquote> 
<br>  [Graumann2004Applicability]  Johannes Graumann, Leslie&nbsp;A Dunipace, 
Jae&nbsp;Hong Seol, W.&nbsp;Hayes McDonald, John&nbsp;R Yates, Barbara&nbsp;J 
Wold, and Raymond&nbsp;J Deshaies. Applicability of tandem affinity 
purification MudPIT to pathway proteomics in yeast.<em>Mol Cell Proteomics</em>
, 3(3):226-37, Mar 2004. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  A combined multidimensional chromatography-mass spectrometry 
approach known as &quot;MudPIT&quot; enables rapid identification of proteins 
that interact with a tagged bait while bypassing some of the problems 
associated with analysis of polypeptides excised from SDS-polyacrylamide gels. 
However, the reproducibility, success rate, and applicability of MudPIT to the 
rapid characterization of dozens of proteins have not been reported. We show 
here that MudPIT reproducibly identified bona fide partners for budding yeast 
Gcn5p. Additionally, we successfully applied MudPIT to rapidly screen through a 
collection of tagged polypeptides to identify new protein interactions. 
Twenty-five proteins involved in transcription and progression through mitosis 
were modified with a new tandem affinity purification (TAP) tag. TAP-MudPIT 
analysis of 22 yeast strains that expressed these tagged proteins uncovered 
known or likely interacting partners for 21 of the baits, a figure that 
compares favorably with traditional approaches. The proteins identified here 
comprised 102 previously known and 279 potential physical interactions. Even 
for the intensively studied Swi2p/Snf2p, the catalytic subunit of the Swi/Snf 
chromatin remodeling complex, our analysis uncovered a new interacting protein, 
Rtt102p. Reciprocal tagging and TAP-MudPIT analysis of Rtt102p revealed 
subunits of both the Swi/Snf and RSC complexes, identifying Rtt102p as a common 
interactor with, and possible integral component of, these chromatin remodeling 
machines. Our experience indicates it is feasible for an investigator working 
with a single ion trap instrument in a conventional molecular/cellular biology 
laboratory to carry out proteomic characterization of a pathway, organelle, or 
process (i.e. &quot;pathway proteomics&quot;) by systematic application of 
TAP-MudPIT. </blockquote> 
<p></p> 
<blockquote>  Keywords: Affinity Labels, Comparative Study, Electrospray 
Ionization, Genetic, Mass, Mitosis, Non-P.H.S., Non-U.S. Gov't, P.H.S., Protein 
Interaction Mapping, Proteome, Proteomics, Research Support, Saccharomyces 
cerevisiae, Saccharomyces cerevisiae Proteins, Signal Transduction, 
Spectrometry, Transcription, U.S. Gov't, 14660704 </blockquote> <br>
 [
Glotsos2004Computer-based]  Dimitris Glotsos, Panagiota Spyridonos, Panagiotis 
Petalas, Dionisis Cavouras, Panagiota Ravazoula, Petroula-Arampatoni Dadioti, 
Ioanna Lekka, and George Nikiforidis. Computer-based malignancy grading of 
astrocytomas employing a support vector machine classifier, the WHO grading 
system and the regular hematoxylin-eosin diagnostic staining procedure.<em>Anal 
Quant Cytol Histol</em>, 26(2):77-83, Apr 2004. [&nbsp;bib&nbsp;] 
<blockquote>  OBJECTIVE: To investigate and develop an automated technique for 
astrocytoma malignancy grading compatible with the clinical routine. STUDY 
DESIGN: One hundred forty biopsies of astrocytomas were collected from 2 
hospitals. The degree of tumor malignancy was defined as low or high according 
to the World Health Organization grading system. From each biopsy, images were 
digitized and segmented to isolate nuclei from background tissue. Morphologic 
and textural nuclear features were quantified to encode tumor malignancy. Each 
case was represented by a 40-dimensional feature vector. An exhaustive search 
procedure in feature space was utilized to determine the best feature 
combination that resulted in the smallest classification error. Low and high 
grade tumors were discriminated using support vector machines (SVMs). To 
evaluate the system performance, all available data were split randomly into 
training and test sets. RESULTS: The best vector combination consisted of 3 
textural and 2 morphologic features. Low and high grade cases were 
discriminated with an accuracy of 90.7% and 88.9%, respectively, using an SVM 
classifier with polynomial kernel of degree 2. CONCLUSION: The proposed 
methodology was based on standards that are common in daily clinical practice 
and might be used in parallel with conventional grading as a second-opinion 
tool to reduce subjectivity in the classification of astrocytomas. </blockquote>
<p></p> 
<blockquote>  Keywords: Amino Acids, Antibodies, Artificial Intelligence, 
Astrocytoma, Biological, Biopsy, Brain, Brain Mapping, Brain Neoplasms, 
Calibration, Comparative Study, Computational Biology, Computer-Assisted, 
Cysteine, Cystine, Electrodes, Electroencephalography, Eosine Yellowish-(YS), 
Evoked Potentials, Female, Hematoxylin, Horseradish Peroxidase, Humans, Image 
Processing, Imagery (Psychotherapy), Imagination, Laterality, Male, Monoclonal, 
Movement, Neoplasms, Non-P.H.S., Non-U.S. Gov't, P.H.S., Perception, Principal 
Component Analysis, Protein, Protein Array Analysis, Proteins, Research 
Support, Sensitivity and Specificity, Sequence Analysis, Software, Tumor 
Markers, U.S. Gov't, User-Computer Interface, World Health Organization, 
15131894 </blockquote> <br>
 [Glotsos2004Automated]  Dimitris Glotsos, 
Panagiota Spyridonos, Dionisis Cavouras, Panagiota Ravazoula, 
Petroula-Arampantoni Dadioti, and George Nikiforidis. Automated segmentation of 
routinely hematoxylin-eosin-stained microscopic images by combining support 
vector machine clustering and active contour models.<em>Anal Quant Cytol Histol
</em>, 26(6):331-40, Dec 2004. [&nbsp;bib&nbsp;] 
<blockquote>  OBJECTIVE: To develop a method for the automated segmentation of 
images of routinely hematoxylin-eosin (H-E)-stained microscopic sections to 
guarantee correct results in computer-assisted microscopy. STUDY DESIGN: 
Clinical material was composed 50 H-E-stained biopsies of astrocytomas and 50 
H-E-stained biopsies of urinary bladder cancer. The basic idea was to use a 
support vector machine clustering (SVMC) algorithm to provide gross 
segmentation of regions holding nuclei and subsequently to refine nuclear 
boundary detection with active contours. The initialization coordinates of the 
active contour model were defined using a SVMC pixel-based classification 
algorithm that discriminated nuclear regions from the surrounding tissue. 
Starting from the boundaries of these regions, the snake fired and propagated 
until converging to nuclear boundaries. RESULTS: The method was validated for 2 
different types of H-E-stained images. Results were evaluated by 2 
histopathologists. On average, 94% of nuclei were correctly delineated. 
CONCLUSION: The proposed algorithm could be of value in computer-based systems 
for automated interpretation of microscopic images. </blockquote> 
<p></p> 
<blockquote>  Keywords: Adenosinetriphosphatase, Adolescent, Adult, 
Algorithms, Amino Acid Sequence, Amino Acids, Animals, Astrocytoma, Automated, 
Automation, Base Sequence, Bayes Theorem, Biological, Biopsy, Bladder 
Neoplasms, Breast Neoplasms, Carbohydrate Conformation, Carbohydrate Sequence, 
Cattle, Cell Cycle Proteins, Cell Nucleus, Computational Biology, Computer 
Simulation, Computer-Assisted, Crystallography, DNA, Databases, Diagnosis, 
Differential, Eosine Yellowish-(YS), Exoribonucleases, Factual, False Negative 
Reactions, False Positive Reactions, Female, Gene Expression, Gene Expression 
Profiling, Genes, Genetic, Genetic Techniques, Genetic Vectors, Genome, 
Hematoxylin, Histocompatibility Antigens Class I, Human, Humans, Image 
Interpretation, Image Processing, Introns, Least-Squares Analysis, MHC Class I, 
Major Histocompatibility Complex, Markov Chains, Messenger, Mice, Middle Aged, 
Models, Molecular Structure, Monosaccharides, Multigene Family, Mutation, 
Neoplasms, Neural Networks (Computer), Non-P.H.S., Non-U.S. Gov't, 
Nonparametric, Nucleotidyltransferases, Observer Variation, Oligonucleotide 
Array Sequence Analysis, P.H.S., Pattern Recognition, Peptides, Phenotype, 
Phylogeny, Plants, Poly A, Polysaccharides, Predictive Value of Tests, Protein, 
Protein Biosynthesis, Protein Kinase Inhibitors, Protein Structure, Proteins, 
RNA, RNA Helicases, RNA Splicing, Rats, Reproducibility of Results, Research 
Support, Retrospective Studies, Saccharomyces cerevisiae, Saccharomyces 
cerevisiae Proteins, Secondary, Sensitivity and Specificity, Sequence 
Alignment, Software, Species Specificity, Staining and Labeling, Statistics, 
Theoretical, Transcription, U.S. Gov't, Ultrasonography, X-Ray, 15678615 
</blockquote> <br>
 [Faugeras2004Variational]  Olivier Faugeras, Geoffray Adde, 
Guillaume Charpiat, Christophe Chefd'hotel, Maureen Clerc, Thomas Deneux, 
Rachid Deriche, Gerardo Hermosillo, Renaud Keriven, Pierre Kornprobst, Jan 
Kybic, Christophe Lenglet, Lucero Lopez-Perez, Th&eacute;o Papadopoulo, 
Jean-Philippe Pons, Florent Segonne, Bertrand Thirion, David 
Tschumperl&eacute;, Thierry Vi&eacute;ville, and Nicolas Wotawa. Variational, 
geometric, and statistical methods for modeling brain anatomy and function.<em>
Neuroimage</em>, 23 Suppl 1:S46-55, 2004. [&nbsp;bib&nbsp;| DOI&nbsp;| http
&nbsp;|.pdf&nbsp;] 
<blockquote>  We survey the recent activities of the Odyss&eacute;e Laboratory 
in the area of the application of mathematics to the design of models for 
studying brain anatomy and function. We start with the problem of 
reconstructing sources in MEG and EEG, and discuss the variational approach we 
have developed for solving these inverse problems. This motivates the need for 
geometric models of the head. We present a method for automatically and 
accurately extracting surface meshes of several tissues of the head from 
anatomical magnetic resonance (MR) images. Anatomical connectivity can be 
extracted from diffusion tensor magnetic resonance images but, in the current 
state of the technology, it must be preceded by a robust estimation and 
regularization stage. We discuss our work based on variational principles and 
show how the results can be used to track fibers in the white matter (WM) as 
geodesics in some Riemannian space. We then go to the statistical modeling of 
functional magnetic resonance imaging (fMRI) signals from the viewpoint of 
their decomposition in a pseudo-deterministic and stochastic part that we then 
use to perform clustering of voxels in a way that is inspired by the theory of 
support vector machines and in a way that is grounded in information theory. 
Multimodal image matching is discussed next in the framework of image 
statistics and partial differential equations (PDEs) with an eye on registering 
fMRI to the anatomy. The paper ends with a discussion of a new theory of random 
shapes that may prove useful in building anatomical and functional atlases. 
</blockquote> 
<p></p> 
<blockquote>  Keywords: Adolescent, Adult, Algorithms, Anatomic, Bacterial 
Proteins, Brain, Brain Mapping, Comparative Study, Computer Simulation, 
Computer-Assisted, Diffusion Magnetic Resonance Imaging, Facial Asymmetry, 
Facial Expression, Facial Paralysis, Female, Gene Expression Profiling, 
Gram-Negative Bacteria, Gram-Positive Bacteria, Humans, Image Interpretation, 
Magnetoencephalography, Male, Middle Aged, Models, Motion, Neural Pathways, 
Non-U.S. Gov't, Photography, Protein, Proteome, Research Support, Retina, 
Sequence Alignment, Sequence Analysis, Severity of Illness Index, Software, 
Statistical, Subcellular Fractions, 15501100 </blockquote> <br>
 [
El-Naqa2004similarity]  I.&nbsp;El-Naqa, Y.&nbsp;Yang, N.&nbsp;P. Galatsanos, 
R.&nbsp;M. Nishikawa, and M.&nbsp;N. Wernick. A similarity learning approach to 
content-based image retrieval: application to digital mammography.<em>IEEE 
Trans Med Imaging</em>, 23(10):1233-44, Oct 2004. [&nbsp;bib&nbsp;] 
<blockquote>  In this paper, we describe an approach to content-based 
retrieval of medical images from a database, and provide a preliminary 
demonstration of our approach as applied to retrieval of digital mammograms. 
Content-based image retrieval (CBIR) refers to the retrieval of images from a 
database using information derived from the images themselves, rather than 
solely from accompanying text indices. In the medical-imaging context, the 
ultimate aim of CBIR is to provide radiologists with a diagnostic aid in the 
form of a display of relevant past cases, along with proven pathology and other 
suitable information. CBIR may also be useful as a training tool for medical 
students and residents. The goal of information retrieval is to recall from a 
database information that is relevant to the user's query. The most challenging 
aspect of CBIR is the definition of relevance (similarity), which is used to 
guide the retrieval machine. In this paper, we pursue a new approach, in which 
similarity is learned from training examples provided by human observers. 
Specifically, we explore the use of neural networks and support vector machines 
to predict the user's notion of similarity. Within this framework we propose 
using a hierarchal learning approach, which consists of a cascade of a binary 
classifier and a regression module to optimize retrieval effectiveness and 
efficiency. We also explore how to incorporate online human interaction to 
achieve relevance feedback in this learning framework. Our experiments are 
based on a database consisting of 76 mammograms, all of which contain clustered 
microcalcifications (MCs). Our goal is to retrieve mammogram images containing 
similar MC clusters to that in a query. The performance of the retrieval system 
is evaluated using precision-recall curves computed using a cross-validation 
procedure. Our experimental results demonstrate that: 1) the learning framework 
can accurately predict the perceptual similarity reported by human observers, 
thereby serving as a basis for CBIR; 2) the learning-based framework can 
significantly outperform a simple distance-based similarity metric; 3) the use 
of the hierarchical two-stage network can improve retrieval performance; and 4) 
relevance feedback can be effectively incorporated into this learning framework 
to achieve improvement in retrieval precision based on online interaction with 
users; and 5) the retrieved images by the network can have predicting value for 
the disease condition of the query. </blockquote> 
<p></p> 
<blockquote>  Keywords: Adolescent, Adult, Algorithms, Amino Acid Sequence, 
Amino Acids, Anatomic, Artificial Intelligence, Attitude of Health Personnel, 
Automated, Bacterial Proteins, Bias (Epidemiology), Brain, Brain Mapping, 
Breast Diseases, Burnout, Calcinosis, Clinical, Cluster Analysis, Comparative 
Study, Computer Graphics, Computer Simulation, Computer-Assisted, Data 
Interpretation, Database Management Systems, Decision Support Systems, 
Diffusion Magnetic Resonance Imaging, Facial Asymmetry, Facial Expression, 
Facial Paralysis, Female, Gene Expression Profiling, Gram-Negative Bacteria, 
Gram-Positive Bacteria, Hospital, Humans, Image Interpretation, Information 
Storage and Retrieval, Intention, Job Satisfaction, Logistic Models, 
Magnetoencephalography, Male, Mammography, Middle Aged, Models, Motion, Neural 
Networks (Computer), Neural Pathways, Non-U.S. Gov't, Nonlinear Dynamics, 
Numerical Analysis, Nursing Administration Research, Nursing Staff, P.H.S., 
Pattern Recognition, Personnel Management, Personnel Turnover, Photography, 
Precancerous Conditions, Predictive Value of Tests, Professional, Protein, 
Proteins, Proteome, Psychological, Questionnaires, Radiographic Image 
Enhancement, Radiographic Image Interpretation, Regression Analysis, 
Reproducibility of Results, Research Support, Retina, Risk Factors, Sensitivity 
and Specificity, Sequence Alignment, Sequence Analysis, Severity of Illness 
Index, Signal Processing, Software, Statistical, Subcellular Fractions, 
Subtraction Technique, Taiwan, Theoretical, U.S. Gov't, User-Computer 
Interface, Workplace, 15493691 </blockquote> <br>
 [Doytchinova2004Identifying] 
 Irini&nbsp;A Doytchinova, Pingping Guan, and Darren&nbsp;R Flower. Identifying 
human MHC supertypes using bioinformatic methods.<em>J. Immunol.</em>, 
172(7):4314-4323, Apr 2004. [&nbsp;bib&nbsp;] 
<blockquote>  Classification of MHC molecules into supertypes in terms of 
peptide-binding specificities is an important issue, with direct implications 
for the development of epitope-based vaccines with wide population coverage. In 
view of extremely high MHC polymorphism (948 class I and 633 class II HLA 
alleles) the experimental solution of this task is presently impossible. In 
this study, we describe a bioinformatics strategy for classifying MHC molecules 
into supertypes using information drawn solely from three-dimensional protein 
structure. Two chemometric techniques-hierarchical clustering and principal 
component analysis-were used independently on a set of 783 HLA class I 
molecules to identify supertypes based on structural similarities and molecular 
interaction fields calculated for the peptide binding site. Eight supertypes 
were defined: A2, A3, A24, B7, B27, B44, C1, and C4. The two techniques gave 
77% consensus, i.e., 605 HLA class I alleles were classified in the same 
supertype by both methods. The proposed strategy allowed &quot;supertype 
fingerprints&quot; to be identified. Thus, the A2 supertype fingerprint is 
Tyr(9)/Phe(9), Arg(97), and His(114) or Tyr(116); the A3-Tyr(9)/Phe(9)/Ser(9), 
Ile(97)/Met(97) and Glu(114) or Asp(116); the A24-Ser(9) and Met(97); the 
B7-Asn(63) and Leu(81); the B27-Glu(63) and Leu(81); for B44-Ala(81); the 
C1-Ser(77); and the C4-Asn(77). </blockquote> 
<p></p> 
<blockquote>  Keywords: Alleles; Amino Acid Motifs; Binding Sites; 
Computational Biology; DNA Fingerprinting; HLA Antigens; HLA-A Antigens; HLA-B 
Antigens; HLA-C Antigens; Histocompatibility Antigens Class I; 
Histocompatibility Testing; Humans; Multigene Family; Protein Interaction 
Mapping </blockquote> <br>
 [Darbellay2004Solid]  Georges&nbsp;A Darbellay, 
Rebecca Duff, Jean-Marc Vesin, Paul-Andr&eacute; Despland, Dirk&nbsp;W Droste, 
Carlos Molina, Joachim Serena, Roman Sztajzel, Patrick Ruchat, Theodoros 
Karapanayiotides, Afksendyios Kalangos, Julien Bogousslavsky, Erich&nbsp;B 
Ringelstein, and G&eacute;rald Devuyst. Solid or gaseous circulating brain 
emboli: are they separable by transcranial ultrasound?<em>J Cereb Blood Flow 
Metab</em>, 24(8):860-8, Aug 2004. [&nbsp;bib&nbsp;] 
<blockquote>  High-intensity transient signals (HITS) detected by transcranial 
Doppler (TCD) ultrasound may correspond to artifacts or to microembolic 
signals, the latter being either solid or gaseous emboli. The goal of this 
study was to assess what can be achieved with an automatic signal processing 
system for artifact/microembolic signals and solid/gas differentiation in 
different clinical situations. The authors studied 3,428 HITS in vivo in a 
multicenter study, i.e., 1,608 artifacts in healthy subjects, 649 solid emboli 
in stroke patients with a carotid stenosis, and 1,171 gaseous emboli in stroke 
patients with patent foramen ovale. They worked with the dual-gate TCD combined 
to three types of statistical classifiers: binary decision trees (BDT), 
artificial neural networks (ANN), and support vector machines (SVM). The 
sensitivity and specificity to separate artifacts from microembolic signals by 
BDT reached was 94% and 97%, respectively. For the discrimination between solid 
and gaseous emboli, the classifier achieved a sensitivity and specificity of 
81% and 81% for BDT, 84% and 84% for ANN, and 86% and 86% for SVM, 
respectively. The current results for artifact elimination and solid/gas 
differentiation are already useful to extract data for future prospective 
clinical studies. </blockquote> 
<p></p> 
<blockquote>  Keywords: Air, Algorithms, Amino Acids, Animals, Artifacts, 
Atrial, Carotid Stenosis, Cerebrovascular Accident, Cerebrovascular 
Circulation, Comparative Study, Cysteine, Decision Trees, Disulfides, Doppler, 
Embolism, Heart Septal Defects, Humans, Intracranial Embolism, Models, 
Molecular, Neural Networks (Computer), Non-U.S. Gov't, Oxidation-Reduction, 
Protein Binding, Protein Folding, Proteins, Research Support, Sensitivity and 
Specificity, Transcranial, Ultrasonography, 15362716 </blockquote> <br>
 [
Cohen2004application]  Gilles Cohen, M&eacute;lanie Hilario, Hugo Sax, 
St&Atilde;&Acirc;&#131;&Acirc;&copy;phane Hugonnet, Christian Pellegrini, and 
Antoine Geissbuhler. An application of one-class support vector machine to 
nosocomial infection detection.<em>Medinfo</em>, 11(Pt 1):716-20, 2004. [&nbsp;
bib&nbsp;] 
<blockquote>  Nosocomial infections (NIs)-those acquired in health care 
settings-are among the major causes of increased mortality among hospitalized 
patients. They are a significant burden for patients and health authorities 
alike; it is thus important to monitor and detect them through an effective 
surveillance system. This paper describes a retrospective analysis of a 
prevalence survey of NIs done in the Geneva University Hospital. Our goal is to 
identify patients with one or more NIs on the basis of clinical and other data 
collected during the survey. In this two-class classification task, the main 
difficulty lies in the significant imbalance between positive or infected (11%) 
and negative (89%) cases. To cope with class imbalance, we investigate 
one-class SVMs which can be trained to distinguish two classes on the basis of 
examples from a single class (in this case, only &quot;normal&quot; or non 
infected patients). The infected ones are then identified as 
&quot;abnormal&quot; cases or outliers that deviate significantly from the 
normal profile. Experimental results are encouraging: whereas standard 2-class 
SVMs scored a baseline sensitivity of 50.6% on this problem, the one-class 
approach increased sensitivity to as much as 92.6%. These results are 
comparable to those obtained by the authors in a previous study on asymmetrical 
soft margin SVMs; they suggest that one-class SVMs can provide an effective and 
efficient way of overcoming data imbalance in classification problems. 
</blockquote> 
<p></p> 
<blockquote>  Keywords: Aged, Air, Algorithms, Amino Acids, Animals, Area 
Under Curve, Artifacts, Artificial Intelligence, Atrial, Automated, Canada, 
Carotid Stenosis, Cerebrovascular Accident, Cerebrovascular Circulation, 
Comparative Study, Computer-Assisted, Cross Infection, Cysteine, Data 
Collection, Decision Trees, Dementia, Diagnosis, Disulfides, Doppler, Embolism, 
Expert Systems, Extramural, Factor Analysis, Female, Gene Expression, Gene 
Expression Profiling, Health Status, Heart Septal Defects, Hospitals, Humans, 
Infection Control, Intracranial Embolism, Male, Models, Molecular, Myocardial 
Infarction, N.I.H., Neoplasms, Neural Networks (Computer), Non-U.S. Gov't, 
Oligonucleotide Array Sequence Analysis, Oxidation-Reduction, P.H.S., Pattern 
Recognition, Population Surveillance, Prevalence, Prognosis, Protein Binding, 
Protein Folding, Proteins, ROC Curve, Research Support, Retrospective Studies, 
Sensitivity and Specificity, Software, Statistical, Switzerland, Transcranial, 
Treatment Outcome, U.S. Gov't, Ultrasonography, University, 15360906 
</blockquote> <br>
 [Cherkassky2004Practical]  Vladimir Cherkassky and Yunqian 
Ma. Practical selection of SVM parameters and noise estimation for SVM 
regression.<em>Neural Netw</em>, 17(1):113-26, Jan 2004. [&nbsp;bib&nbsp;| DOI
&nbsp;|http&nbsp;| .pdf&nbsp;] 
<blockquote>  We investigate practical selection of hyper-parameters for 
support vector machines (SVM) regression (that is, epsilon-insensitive zone and 
regularization parameter C). The proposed methodology advocates analytic 
parameter selection directly from the training data, rather than re-sampling 
approaches commonly used in SVM applications. In particular, we describe a new 
analytical prescription for setting the value of insensitive zone epsilon, as a 
function of training sample size. Good generalization performance of the 
proposed parameter selection is demonstrated empirically using several low- and 
high-dimensional regression problems. Further, we point out the importance of 
Vapnik's epsilon-insensitive loss for regression problems with finite samples. 
To this end, we compare generalization performance of SVM regression (using 
proposed selection of epsilon-values) with regression using 'least-modulus' 
loss (epsilon=0) and standard squared loss. These comparisons indicate superior 
generalization performance of SVM regression under sparse sample settings, for 
various types of additive noise. </blockquote> 
<p></p> 
<blockquote>  Keywords: 80 and over, Abstracting and Indexing, Aged, 
Algorithms, Amino Acid Sequence, Amino Acids, Animals, Antibody, Antigens, 
Aromatic, Artificial Intelligence, Atrophy, Automated, BRCA1, Bayes Theorem, 
Bibliographic, Binding, Binding Sites, Brain, Brain Mapping, Breast, Breast 
Neoplasms, Carcinoma, Carrier Proteins, Cell Cycle Proteins, Chemical, Cluster 
Analysis, Comparative Study, Competitive, Complementary, Computational Biology, 
Computer Simulation, Computer-Assisted, Computers, Computing Methodologies, 
Controlled, DNA, DNA Damage, DNA-Binding Proteins, Data Interpretation, 
Database Management Systems, Databases, Diagnosis, Discriminant Analysis, Drug 
Design, Ductal, Electrostatics, Endoscopy, Epitopes, Evolution, Female, 
Fibroblasts, Fluorescence, Fourier Analysis, Gene Expression Profiling, Gene 
Expression Regulation, Genes, Genetic, HLA-DR Antigens, Heterozygote, 
Histocompatibility Antigens Class I, Humans, Hydrophobicity, Image Processing, 
Imaging, Information Storage and Retrieval, Isoelectric Point, Linear Models, 
Linguistics, Longitudinal Studies, MEDLINE, Magnetic Resonance Imaging, Male, 
Membrane Proteins, Middle Aged, Models, Molecular, Molecular Sequence Data, 
Multivariate Analysis, Mutation, Nasopharyngeal Neoplasms, Natural Language 
Processing, Neoplastic, Neural Networks (Computer), Non-P.H.S., Non-U.S. Gov't, 
Nonlinear Dynamics, Normal Distribution, Numerical Analysis, Oligonucleotide 
Array Sequence Analysis, Oligopeptides, P.H.S., Pattern Recognition, Peptide 
Fragments, Periodicals, Principal Component Analysis, Protein, Protein Binding, 
Protein Conformation, Protein Sorting Signals, Protein Structure, 
Protein-Serine-Threonine Kinases, Proteins, Proteome, Proteomics, Quality 
Control, RNA-Binding Proteins, ROC Curve, Randomized Controlled Trials, 
Regression Analysis, Reproducibility of Results, Research Support, 
Saccharomyces cerevisiae Proteins, Selection Bias, Sensitivity and Specificity, 
Sequence Alignment, Sequence Analysis, Software, Solvents, Spectrometry, 
Statistical, Structural Homology, Structure-Activity Relationship, 
T-Lymphocyte, Terminology, Tertiary, Theoretical, Three-Dimensional, Tumor 
Suppressor Proteins, U.S. Gov't, Variation (Genetics), Vocabulary, 14690712 
</blockquote> <br>
 [Campanini2004novel]  Renato Campanini, Danilo Dongiovanni, 
Emiro Iampieri, Nico Lanconelli, Matteo Masotti, Giuseppe Palermo, Alessandro 
Riccardi, and Matteo Roffilli. A novel featureless approach to mass detection 
in digital mammograms based on support vector machines.<em>Phys Med Biol</em>, 
49(6):961-75, Mar 2004. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  In this work, we present a novel approach to mass detection in 
digital mammograms. The great variability of the appearance of masses is the 
main obstacle to building a mass detection method. It is indeed demanding to 
characterize all the varieties of masses with a reduced set of features. Hence, 
in our approach we have chosen not to extract any feature, for the detection of 
the region of interest; in contrast, we exploit all the information available 
on the image. A multiresolution overcomplete wavelet representation is 
performed, in order to codify the image with redundancy of information. The 
vectors of the very-large space obtained are then provided to a first support 
vector machine (SVM) classifier. The detection task is considered here as a 
two-class pattern recognition problem: crops are classified as suspect or not, 
by using this SVM classifier. False candidates are eliminated with a second 
cascaded SVM. To further reduce the number of false positives, an ensemble of 
experts is applied: the final suspect regions are achieved by using a voting 
strategy. The sensitivity of the presented system is nearly 80% with a 
false-positive rate of 1.1 marks per image, estimated on images coming from the 
USF DDSM database. </blockquote> 
<p></p> 
<blockquote>  Keywords: Algorithms, Amino Acids, Antibodies, Artificial 
Intelligence, Astrocytoma, Automated, Bayes Theorem, Biological, Biopsy, Brain, 
Brain Mapping, Brain Neoplasms, Breast Neoplasms, Calibration, Cluster 
Analysis, Comparative Study, Computational Biology, Computer-Assisted, 
Computing Methodologies, Cysteine, Cystine, Dysplastic Nevus Syndrome, 
Electrodes, Electroencephalography, Entropy, Eosine Yellowish-(YS), Evoked 
Potentials, Expert Systems, False Positive Reactions, Female, Gene Expression 
Profiling, Hematoxylin, Horseradish Peroxidase, Humans, Image Interpretation, 
Image Processing, Imagery (Psychotherapy), Imagination, Information Storage and 
Retrieval, Laterality, Linear Models, Male, Mammography, Melanoma, Models, 
Monoclonal, Movement, Neoplasms, Neural Networks (Computer), Neuropeptides, 
Non-P.H.S., Non-U.S. Gov't, Nonparametric, Normal Distribution, P.H.S., Pattern 
Recognition, Perception, Principal Component Analysis, Protein, Protein Array 
Analysis, Protein Interaction Mapping, Proteins, Radiographic Image 
Enhancement, Radiographic Image Interpretation, Regression Analysis, 
Reproducibility of Results, Research Support, Sensitivity and Specificity, 
Sequence Alignment, Sequence Ana, Sequence Analysis, Signal Processing, Skin 
Neoplasms, Software, Statistical, Statistics, Tumor Markers, U.S. Gov't, 
User-Computer Interface, World Health Organization, lysis, 15104319 
</blockquote> <br>
 [Bowd2004Confocal]  Christopher Bowd, Linda&nbsp;M 
Zangwill, Felipe&nbsp;A Medeiros, Jiucang Hao, Kwokleung Chan, Te-Won Lee, 
Terrence&nbsp;J Sejnowski, Michael&nbsp;H Goldbaum, Pamela&nbsp;A Sample, 
Jonathan&nbsp;G Crowston, and Robert&nbsp;N Weinreb. Confocal scanning laser 
ophthalmoscopy classifiers and stereophotograph evaluation for prediction of 
visual field abnormalities in glaucoma-suspect eyes.<em>Invest Ophthalmol Vis 
Sci</em>, 45(7):2255-62, Jul 2004. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf
&nbsp;]
<blockquote>  PURPOSE: To determine whether Heidelberg Retina Tomograph (HRT; 
Heidelberg Engineering, Dossenheim, Germany) classification techniques and 
investigational support vector machine (SVM) analyses can detect optic disc 
abnormalities in glaucoma-suspect eyes before the development of visual field 
abnormalities. METHODS: Glaucoma-suspect eyes (n = 226) were classified as 
converts or nonconverts based on the development of repeatable (either two or 
three consecutive) standard automated perimetry (SAP)-detected abnormalities 
over the course of the study (mean follow-up, approximately 4.5 years). Hazard 
ratios for development of SAP abnormalities were calculated based on baseline 
classification results, follow-up time, and end point status (convert, 
nonconvert). Classification techniques applied were HRT classification (HRTC), 
Moorfields Regression Analysis, forward-selection optimized SVM (SVM fwd) and 
backward elimination-optimized SVM (SVM back) analysis of HRT data, and 
stereophotograph assessment. RESULTS: Univariate analyses indicated that all 
classification techniques were predictors of the development of two repeatable 
abnormal SAP results, with hazards ratios (95% confidence interval [CI]) 
ranging from 1.32 (1.00-1.75) for HRTC to 2.0 (1.48-2.76) for stereophotograph 
assessment (all P &lt; or = 0.05). Only SVM (SVM fwd and SVM back) analysis of 
HRT data and stereophotograph assessment were univariate predictors of the 
development of three repeatable abnormal SAP results, with hazard ratios (95% 
CI) ranging from 1.73 (1.16-2.82) for SVM fwd to 1.82 (1.19-3.12) for SVM back 
(both P &lt; 0.007). Multivariate analyses including each classification 
technique individually in a model with age, baseline SAP pattern standard 
deviation [PSD], and baseline IOP indicated that all classification techniques 
except HRTC (P = 0.06) were predictors of the development of two repeatable 
abnormal SAP results with hazards ratios ranging from 1.30 (0.99, 1.73) for 
HRTC to 1.90 (1.37, 2.69) for stereophotograph assessment. Only SVM (SVM fwd 
and SVM back) analysis of HRT data and stereophotograph assessment were 
significant predictors of the development of three repeatable abnormal SAP 
results in multivariate analyses; hazard ratios of 1.57 (1.03, 2.59) and 1.70 
(1.18, 2.51), respectively. SAP PSD was a significant predictor of two 
repeatable abnormal SAP results in multivariate models with all classification 
techniques, with hazard ratios ranging from 3.31 (1.39, 7.89) to 4.70 (2.02, 
10.93) per 1-dB increase. CONCLUSIONS: HRT classifications techniques and 
stereophotograph assessment can detect optic disc topography abnormalities in 
glaucoma-suspect eyes before the development of SAP abnormalities. These data 
support strongly the importance of optic disc examination for early glaucoma 
diagnosis. </blockquote> 
<p></p> 
<blockquote>  Keywords: 80 and over, Adolescent, Adult, Aged, Algorithms, 
Artificial Intelligence, Auditory, Benchmarking, Binding Sites, Brain Stem, 
Breast Diseases, Chemical, Child, Chromosomes, Comparative Study, Computational 
Biology, Computer Simulation, Computer-Assisted, Data Interpretation, 
Databases, Diagnosis, Diagnostic Errors, Differential, Drug Resistance, 
Electroencephalography, Epilepsy, Evoked Potentials, Female, Forecasting, Gene 
Expression, Gene Expression Profiling, Genetic, Genotype, Glaucoma, Greece, HIV 
Protease Inhibitors, HIV-1, Human, Humans, Infant, Information Management, 
Information Storage and Retrieval, Intraocular Pressure, Kinetics, Language 
Development Disorders, Lasers, Least-Squares Analysis, Linear Models, Male, 
Microbial Sensitivity Tests, Middle Aged, Models, Molecular, Monitoring, 
Nephroblastoma, Non-U.S. Gov't, Nonlinear Dynamics, Ocular Hypertension, 
Oligonucleotide Array Sequence Analysis, Ophthalmoscopy, Optic Disk, Optic 
Nerve Diseases, P.H.S., Pair 1, Perimetry, Periodicals, Phosphorylation, 
Phosphotransferases, Photography, Physiologic, Point Mutation, Preschool, 
Prognosis, Protein, Proteins, Pyrimidinones, Reaction Time, Recurrence, 
Reproducibility of Results, Research Support, Reverse Transcriptase Inhibitors, 
Sensitivity and Specificity, Sequence Alignment, Sequence Analysis, Signal 
Processing, Software, Sound Localization, Statistical, Stochastic Processes, 
Structure-Activity Relationship, Theoretical, Time Factors, U.S. Gov't, Viral, 
Vision Disorders, Visual Fields, 15223803 </blockquote> <br>
 [Bern2004Automatic
]  M.&nbsp;Bern, D.&nbsp;Goldberg, W.&nbsp;H. McDonald, and III Yates, 
J.&nbsp;R. Automatic Quality Assessment of Peptide Tandem Mass Spectra.<em>
Bioinformatics</em>, 20(Suppl. 1):i49-i54, 2004. [&nbsp;bib&nbsp;| http&nbsp;| 
.pdf&nbsp;] 
<blockquote>  Motivation: A powerful proteomics methodology couples 
high-performance liquid chromatography (HPLC) with tandem mass spectrometry and 
database-search software, such as SEQUEST. Such a set-up, however, produces a 
large number of spectra, many of which are of too poor quality to be useful. 
Hence a filter that eliminates poor spectra before the database search can 
significantly improve throughput and robustness. Moreover, spectra judged to be 
of high quality, but that cannot be identified by database search, are prime 
candidates for still more computationally intensive methods, such as de novo 
sequencing or wider database searches including post-translational 
modifications. Results: We report on two different approaches to assessing 
spectral quality prior to identification: binary classification, which predicts 
whether or not SEQUEST will be able to make an identification, and statistical 
regression, which predicts a more universal quality metric involving the number 
of b- and y-ion peaks. The best of our binary classifiers can eliminate over 75 
spectra while losing only 10 regression can pick out spectra of modified 
peptides that can be identified by a de novo program but not by SEQUEST. In a 
section of independent interest, we discuss intensity normalization of mass 
spectra. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm proteomics </blockquote> <br>
 [
Benito2004Adjustment]  Monica Benito, Joel Parker, Quan Du, Junyuan Wu, Dong 
Xiang, Charles&nbsp;M Perou, and J.&nbsp;S. Marron. Adjustment of systematic 
microarray data biases.<em>Bioinformatics</em>, 20(1):105-14, Jan 2004. [&nbsp;
bib&nbsp;] 
<blockquote>  MOTIVATION: Systematic differences due to experimental features 
of microarray experiments are present in most large microarray data sets. Many 
different experimental features can cause biases including different sources of 
RNA, different production lots of microarrays or different microarray 
platforms. These systematic effects present a substantial hurdle to the 
analysis of microarray data. RESULTS: We present here a new method for the 
identification and adjustment of systematic biases that are present within 
microarray data sets. Our approach is based on modern statistical 
discrimination methods and is shown to be very effective in removing systematic 
biases present in a previously published breast tumor cDNA microarray data set. 
The new method of 'Distance Weighted Discrimination (DWD)' is shown to be 
better than Support Vector Machines and Singular Value Decomposition for the 
adjustment of systematic microarray effects. In addition, it is shown to be of 
general use as a tool for the discrimination of systematic problems present in 
microarray data sets, including the merging of two breast tumor data sets 
completed on different microarray platforms. AVAILABILITY: Matlab software to 
perform DWD can be retrieved from https://genome.unc.edu/pubsup/dwd/ 
</blockquote> 
<p></p> 
<blockquote>  Keywords: 80 and over, Abstracting and Indexing, Aged, 
Algorithms, Amino Acid Sequence, Amino Acids, Animals, Antibody, Antigens, 
Aromatic, Artificial Intelligence, Atrophy, Automated, BRCA1, Bibliographic, 
Binding, Binding Sites, Brain, Brain Mapping, Breast, Breast Neoplasms, 
Carcinoma, Carrier Proteins, Cell Cycle Proteins, Chemical, Cluster Analysis, 
Comparative Study, Competitive, Complementary, Computational Biology, Computer 
Simulation, Computer-Assisted, Computers, Computing Methodologies, Controlled, 
DNA, DNA Damage, DNA-Binding Proteins, Data Interpretation, Database Management 
Systems, Databases, Diagnosis, Discriminant Analysis, Drug Design, Ductal, 
Electrostatics, Endoscopy, Epitopes, Evolution, Female, Fibroblasts, 
Fluorescence, Fourier Analysis, Gene Expression Profiling, Gene Expression 
Regulation, Genes, Genetic, HLA-DR Antigens, Heterozygote, Histocompatibility 
Antigens Class I, Humans, Hydrophobicity, Image Processing, Imaging, 
Information Storage and Retrieval, Isoelectric Point, Linear Models, 
Linguistics, Longitudinal Studies, MEDLINE, Magnetic Resonance Imaging, Male, 
Membrane Proteins, Middle Aged, Models, Molecular, Molecular Sequence Data, 
Multivariate Analysis, Mutation, Nasopharyngeal Neoplasms, Natural Language 
Processing, Neoplastic, Neural Networks (Computer), Non-P.H.S., Non-U.S. Gov't, 
Nonlinear Dynamics, Numerical Analysis, Oligonucleotide Array Sequence 
Analysis, Oligopeptides, P.H.S., Pattern Recognition, Peptide Fragments, 
Periodicals, Principal Component Analysis, Protein, Protein Binding, Protein 
Conformation, Protein Sorting Signals, Protein Structure, 
Protein-Serine-Threonine Kinases, Proteins, Proteome, Proteomics, Quality 
Control, RNA-Binding Proteins, ROC Curve, Randomized Controlled Trials, 
Reproducibility of Results, Research Support, Saccharomyces cerevisiae 
Proteins, Selection Bias, Sensitivity and Specificity, Sequence Alignment, 
Sequence Analysis, Software, Solvents, Spectrometry, Statistical, Structural 
Homology, Structure-Activity Relationship, T-Lymphocyte, Terminology, Tertiary, 
Three-Dimensional, Tumor Suppressor Proteins, U.S. Gov't, Variation (Genetics), 
Vocabulary, 14693816 </blockquote> <br>
 [Baumgartner2004Supervised]  
C.&nbsp;Baumgartner, C.&nbsp;Bohm, D.&nbsp;Baumgartner, G.&nbsp;Marini, 
K.&nbsp;Weinberger, B.&nbsp;Olgemoller, B.&nbsp;Liebl, and A.&nbsp;A. Roscher. 
Supervised machine learning techniques for the classification of metabolic 
disorders in newborns.<em>Bioinformatics</em>, 20(17):2985-2996, 2004. [&nbsp;
bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  Motivation: During the Bavarian newborn screening programme all 
newborns have been tested for about 20 inherited metabolic disorders. Owing to 
the amount and complexity of the generated experimental data, machine learning 
techniques provide a promising approach to investigate novel patterns in 
high-dimensional metabolic data which form the source for constructing 
classification rules with high discriminatory power. Results: Six machine 
learning techniques have been investigated for their classification accuracy 
focusing on two metabolic disorders, phenylketo nuria (PKU) and medium-chain 
acyl-CoA dehydrogenase deficiency (MCADD). Logistic regression analysis led to 
superior classification rules (sensitivity &gt;96.8 to all investigated 
algorithms. Including novel constellations of metabolites into the models, the 
positive predictive value could be strongly increased (PKU 71.9 54.6 clearly 
prove that the mined data confirm the known and indicate some novel metabolic 
patterns which may contribute to a better understanding of newborn metabolism. 
Availability: WEKA machine learning package: www.cs.waikato.ac.nz/&nbsp;ml/weka 
and statistical software package ADE-4: http://pbil.univ-lyon1.fr/ADE-4 
</blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm proteomics </blockquote> <br>
 [
Avidan2004Support]  Shai Avidan. Support vector tracking. <em>IEEE Trans 
Pattern Anal Mach Intell</em>, 26(8):1064-72, Aug 2004. [&nbsp;bib&nbsp;| DOI
&nbsp;|http&nbsp;| .pdf&nbsp;] 
<blockquote>  Support Vector Tracking (SVT) integrates the Support Vector 
Machine (SVM) classifier into an optic-flow-based tracker. Instead of 
minimizing an intensity difference function between successive frames, SVT 
maximizes the SVM classification score. To account for large motions between 
successive frames, we build pyramids from the support vectors and use a 
coarse-to-fine approach in the classification stage. We show results of using 
SVT for vehicle tracking in image sequences. </blockquote> 
<p></p> 
<blockquote>  Keywords: Algorithms, Artificial Intelligence, Automated, Cell 
Nucleus, Comparative Study, Computer Graphics, Computer-Assisted, Cytoplasm, 
Databases, Genetic Vectors, Humans, Image Enhancement, Image Interpretation, 
Imaging, Information Storage and Retrieval, Internet, Mitochondria, Models, 
Movement, Non-U.S. Gov't, Numerical Analysis, Pattern Recognition, Peptides, 
Protein, Proteins, Proteomics, Reproducibility of Results, Research Support, 
Sensitivity and Specificity, Signal Processing, Software, Subtraction 
Technique, Theoretical, Three-Dimensional, Video Recording, 15641735 
</blockquote> <br>
 [Zhou2004Recognizing]  GuoDong Zhou, Jie Zhang, Jian Su, 
Dan Shen, and ChewLim Tan. Recognizing names in biomedical texts: a machine 
learning approach.<em>Bioinformatics</em>, 20(7):1178-90, May 2004. [&nbsp;bib
&nbsp;|DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  MOTIVATION: With an overwhelming amount of textual information 
in molecular biology and biomedicine, there is a need for effective and 
efficient literature mining and knowledge discovery that can help biologists to 
gather and make use of the knowledge encoded in text documents. In order to 
make organized and structured information available, automatically recognizing 
biomedical entity names becomes critical and is important for information 
retrieval, information extraction and automated knowledge acquisition. RESULTS: 
In this paper, we present a named entity recognition system in the biomedical 
domain, called PowerBioNE. In order to deal with the special phenomena of 
naming conventions in the biomedical domain, we propose various evidential 
features: (1) word formation pattern; (2) morphological pattern, such as prefix 
and suffix; (3) part-of-speech; (4) head noun trigger; (5) special verb trigger 
and (6) name alias feature. All the features are integrated effectively and 
efficiently through a hidden Markov model (HMM) and a HMM-based named entity 
recognizer. In addition, a k-Nearest Neighbor (k-NN) algorithm is proposed to 
resolve the data sparseness problem in our system. Finally, we present a 
pattern-based post-processing to automatically extract rules from the training 
data to deal with the cascaded entity name phenomenon. From our best knowledge, 
PowerBioNE is the first system which deals with the cascaded entity name 
phenomenon. Evaluation shows that our system achieves the F-measure of 66.6 and 
62.2 on the 23 classes of GENIA V3.0 and V1.1, respectively. In particular, our 
system achieves the F-measure of 75.8 on the &quot;protein&quot; class of GENIA 
V3.0. For comparison, our system outperforms the best published result by 7.8 
on GENIA V1.1, without help of any dictionaries. It also shows that our HMM and 
the k-NN algorithm outperform other models, such as back-off HMM, linear 
interpolated HMM, support vector machines, C4.5, C4.5 rules and RIPPER, by 
effectively capturing the local context dependency and resolving the data 
sparseness problem. Moreover, evaluation on GENIA V3.0 shows that the 
post-processing for the cascaded entity name phenomenon improves the F-measure 
by 3.9. Finally, error analysis shows that about half of the errors are caused 
by the strict annotation scheme and the annotation inconsistency in the GENIA 
corpus. This suggests that our system achieves an acceptable F-measure of 83.6 
on the 23 classes of GENIA V3.0 and in particular 86.2 on the 
&quot;protein&quot; class, without help of any dictionaries. We think that a 
F-measure of 90 on the 23 classes of GENIA V3.0 and in particular 92 on the 
&quot;protein&quot; class, can be achieved through refining of the annotation 
scheme in the GENIA corpus, such as flexible annotation scheme and annotation 
consistency, and inclusion of a reasonable biomedical dictionary. AVAILABILITY: 
A demo system is available at http://textmining.i2r.a-star.edu.sg/NLS/demo.htm. 
Technology license is available upon the bilateral agreement. </blockquote> 
<p></p> 
<blockquote>  Keywords: Abstracting and Indexing, Algorithms, Amino Acid 
Sequence, Amino Acids, Animals, Antibody, Antigens, Aromatic, Artificial 
Intelligence, Automated, BRCA1, Bibliographic, Binding, Binding Sites, Carrier 
Proteins, Cell Cycle Proteins, Chemical, Cluster Analysis, Comparative Study, 
Competitive, Complementary, Computational Biology, Computing Methodologies, 
Controlled, DNA, DNA Damage, DNA-Binding Proteins, Data Interpretation, 
Database Management Systems, Databases, Drug Design, Electrostatics, Epitopes, 
Evolution, Female, Fibroblasts, Gene Expression Profiling, Gene Expression 
Regulation, Genes, HLA-DR Antigens, Heterozygote, Histocompatibility Antigens 
Class I, Humans, Hydrophobicity, Information Storage and Retrieval, Membrane 
Proteins, Models, Molecular, Molecular Sequence Data, Mutation, Natural 
Language Processing, Neoplastic, Neural Networks (Computer), Non-P.H.S., 
Non-U.S. Gov't, Oligonucleotide Array Sequence Analysis, Oligopeptides, P.H.S., 
Pattern Recognition, Peptide Fragments, Periodicals, Protein, Protein Binding, 
Protein Conformation, Protein Sorting Signals, Protein Structure, 
Protein-Serine-Threonine Kinases, Proteins, RNA-Binding Proteins, 
Reproducibility of Results, Research Support, Saccharomyces cerevisiae 
Proteins, Sensitivity and Specificity, Sequence Alignment, Sequence Analysis, 
Software, Statistical, Structural Homology, Structure-Activity Relationship, 
T-Lymphocyte, Terminology, Tertiary, Tumor Suppressor Proteins, U.S. Gov't, 
Vocabulary, 14871877 </blockquote> <br>
 [Perola2004Conformational]  Emanuele 
Perola and Paul&nbsp;S Charifson. Conformational analysis of drug-like 
molecules bound to proteins: an extensive study of ligand reorganization upon 
binding.<em>J. Med. Chem.</em>, 47(10):2499-2510, May 2004. [&nbsp;bib&nbsp;| 
DOI&nbsp;| http&nbsp;] 
<blockquote>  This paper describes a large-scale study on the nature and the 
energetics of the conformational changes drug-like molecules experience upon 
binding. Ligand strain energies and conformational reorganization were analyzed 
with different computational methods on 150 crystal structures of 
pharmaceutically relevant protein-ligand complexes. The common knowledge that 
ligands rarely bind in their lowest calculated energy conformation was 
confirmed. Additionally, we found that over 60% of the ligands do not bind in a 
local minimum conformation. While approximately 60% of the ligands were 
calculated to bind with strain energies lower than 5 kcal/mol, strain energies 
over 9 kcal/mol were calculated in at least 10% of the cases regardless of the 
method used. A clear correlation was found between acceptable strain energy and 
ligand flexibility, while there was no correlation between strain energy and 
binding affinity, thus indicating that expensive conformational rearrangements 
can be tolerated in some cases without overly penalizing the tightness of 
binding. On the basis of the trends observed, thresholds for the acceptable 
strain energies of bioactive conformations were defined with consideration of 
the impact of ligand flexibility. An analysis of the degree of folding of the 
bound ligands confirmed the general tendency of small molecules to bind in an 
extended conformation. The results suggest that the unfolding of hydrophobic 
ligands during binding, which exposes hydrophobic surfaces to contact with 
protein residues, could be one of the factors accounting for high 
reorganization energies. Finally, different methods for conformational analysis 
were evaluated, and guidelines were defined to maximize the prevalence of 
bioactive conformations in computationally generated ensembles. </blockquote> 
<p></p> 
<blockquote>  Keywords: Drug Design; Endopeptidases; Ligands; Molecular 
Conformation; Pharmaceutical Preparations; Phosphotransferases; Protein 
Binding; Protein Folding; Proteins; Thermodynamics </blockquote> <br>
 [
Mestres2004Computational]  Jordi Mestres. Computational chemogenomics 
approaches to systematic knowledge-based drug discovery.<em>Curr Opin Drug 
Discov Devel</em>, 7(3):304-313, May 2004. [&nbsp;bib&nbsp;] 
<blockquote>  Chemogenomics, the identification of all possible drugs for all 
possible targets, has recently emerged as a new paradigm in drug discovery in 
which efficiency in the compound design and optimization process is achieved 
through the gain and reuse of targeted knowledge. As targeted knowledge resides 
at the interface between chemistry and biology, computational tools aimed at 
integrating the chemical and biological spaces play a central role in 
chemogenomics. This review covers the recent progress made in integrative 
computational approaches to data annotation and knowledge generation for the 
systematic knowledge-based design and screening of chemical libraries. 
</blockquote> 
<p></p> 
<blockquote>  Keywords: Chemistry, Pharmaceutical; Combinatorial Chemistry 
Techniques; Computational Biology; Drug Design; Genomics; Ligands; Proteins; 
Receptors, G-Protein-Coupled </blockquote> <br>
 [Luo2004gene-silencing]  
K.&nbsp;Q. Luo and D.&nbsp;C. Chang. The gene-silencing efficiency of siRNA is 
strongly dependent on the local structure of mRNA at the targeted region.<em>
Biochem. Biophys. Res. Commun.</em>, 318(1):303-10, May 2004. [&nbsp;bib&nbsp;| 
DOI&nbsp;| http&nbsp;] 
<blockquote>  The gene-silencing effect of short interfering RNA (siRNA) is 
known to vary strongly with the targeted position of the mRNA. A number of 
hypotheses have been suggested to explain this phenomenon. We would like to 
test if this positional effect is mainly due to the secondary structure of the 
mRNA at the target site. We proposed that this structural factor can be 
characterized by a single parameter called &quot;the hydrogen bond (H-b) 
index,&quot; which represents the average number of hydrogen bonds formed 
between nucleotides in the target region and the rest of the mRNA. This index 
can be determined using a computational approach. We tested the correlation 
between the H-b index and the gene-silencing effects on three genes (Bcl-2, 
hTF, and cyclin B1) using a variety of siRNAs. We found that the gene-silencing 
effect is inversely dependent on the H-b index, indicating that the local mRNA 
structure at the targeted site is the main cause of the positional effect. 
Based on this finding, we suggest that the H-b index can be a useful guideline 
for future siRNA design. </blockquote> 
<p></p> 
<blockquote>  Keywords: Animals, Apoptosis, Base Composition, Base Pairing, 
Base Sequence, Binding Sites, Cell Cycle, Cell Proliferation, Comparative 
Study, Cultured, Cyclin B, Cyclin D1, DNA-Binding Proteins, Down-Regulation, 
Extramural, Fluorescence, Gene Silencing, Gene Targeting, Genetic Vectors, 
Green Fluorescent Proteins, Hela Cells, Humans, Hydrogen Bonding, Luminescent 
Proteins, Male, Messenger, Mice, Microscopy, Models, Molecular, Molecular 
Sequence Data, N.I.H., Non-U.S. Gov't, Nucleic Acid Conformation, Nude, P.H.S., 
Prostatic Neoplasms, Proto-Oncogene Proteins c-bcl-2, Proto-Oncogene Proteins 
c-myc, RNA, Regression Analysis, Research Support, STAT3 Transcription Factor, 
Small Interfering, Thromboplastin, Trans-Activators, Tumor Cells, U.S. Gov't, 
15110788 </blockquote> <br>
 [Kim2004Emotion]  K.&nbsp;H. Kim, S.&nbsp;W. Bang, 
and S.&nbsp;R. Kim. Emotion recognition system using short-term monitoring of 
physiological signals.<em>Med Biol Eng Comput</em>, 42(3):419-27, May 2004. 
[&nbsp;bib&nbsp;] 
<blockquote>  A physiological signal-based emotion recognition system is 
reported. The system was developed to operate as a user-independent system, 
based on physiological signal databases obtained from multiple subjects. The 
input signals were electrocardiogram, skin temperature variation and 
electrodermal activity, all of which were acquired without much discomfort from 
the body surface, and can reflect the influence of emotion on the autonomic 
nervous system. The system consisted of preprocessing, feature extraction and 
pattern classification stages. Preprocessing and feature extraction methods 
were devised so that emotion-specific characteristics could be extracted from 
short-segment signals. Although the features were carefully extracted, their 
distribution formed a classification problem, with large overlap among clusters 
and large variance within clusters. A support vector machine was adopted as a 
pattern classifier to resolve this difficulty. Correct-classification ratios 
for 50 subjects were 78.4% and 61.8%, for the recognition of three and four 
categories, respectively. </blockquote> 
<p></p> 
<blockquote>  Keywords: Algorithms, Animals, Antisense, Artificial 
Intelligence, Autonomic Nervous System, Cell Line, Child, Cluster Analysis, 
Comparative Study, Computational Biology, Computer Simulation, 
Computer-Assisted, DNA Fingerprinting, Drug Evaluation, Emotions, Fluorescence, 
Fuzzy Logic, Gene Silencing, Gene Targeting, Genetic, Hela Cells, Humans, 
Imaging, Intracellular Space, Microscopy, Models, Monitoring, Neoplasms, Neural 
Networks (Computer), Non-U.S. Gov't, Oligonucleotides, P.H.S., Physiologic, 
Preclinical, Preschool, Prognosis, Proteomics, Quantitative Structure-Activity 
Relationship, RNA, RNA Interference, Recognition (Psychology), Research 
Support, Sensitivity and Specificity, Signal Processing, Small Interfering, 
Thionucleotides, Three-Dimensional, Tumor, U.S. Gov't, User-Computer Interface, 
15191089 </blockquote> <br>
 [Galan2004Odor-driven]  Roberto&nbsp;Fdez 
Gal&Atilde;&Acirc;&iexcl;n, Silke Sachse, C.&nbsp;Giovanni Galizia, and Andreas 
V&nbsp;M Herz. Odor-driven attractor dynamics in the antennal lobe allow for 
simple and rapid olfactory pattern classification.<em>Neural Comput</em>, 
16(5):999-1012, May 2004. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;] 
<blockquote>  The antennal lobe plays a central role for odor processing in 
insects, as demonstrated by electrophysiological and imaging experiments. Here 
we analyze the detailed temporal evolution of glomerular activity patterns in 
the antennal lobe of honeybees. We represent these spatiotemporal patterns as 
trajectories in a multidimensional space, where each dimension accounts for the 
activity of one glomerulus. Our data show that the trajectories reach 
odor-specific steady states (attractors) that correspond to stable activity 
patterns at about 1 second after stimulus onset. As revealed by a detailed 
mathematical investigation, the trajectories are characterized by different 
phases: response onset, steady-state plateau, response offset, and periods of 
spontaneous activity. An analysis based on support-vector machines quantifies 
the odor specificity of the attractors and the optimal time needed for odor 
discrimination. The results support the hypothesis of a spatial olfactory code 
in the antennal lobe and suggest a perceptron-like readout mechanism that is 
biologically implemented in a downstream network, such as the mushroom body. 
</blockquote> 
<p></p> 
<blockquote>  Keywords: Abnormalities, Adolescent, Adult, Aldosterone 
Synthase, Algorithms, Amino Acids, Anatomic, Animals, Antibodies, Arrhythmia, 
Artificial Intelligence, Aryl Hydrocarbon Hydroxylases, Astrocytoma, Automated, 
Bacterial Proteins, Bayes Theorem, Bees, Biological, Biopsy, Brain, Brain 
Mapping, Brain Neoplasms, Breast Neoplasms, Calibration, Cation Exchange 
Resins, Child, Chromatography, Cluster Analysis, Comparative Study, 
Computational Biology, Computer-Assisted, Computing Methodologies, Congenital, 
Craniofacial Abnormalities, Cysteine, Cystine, DNA-Binding Proteins, Data 
Interpretation, Databases, Diagnosis, Disease Susceptibility, Dysplastic Nevus 
Syndrome, Electrocardiography, Electrodes, Electroencephalography, Entropy, 
Eosine Yellowish-(YS), Evoked Potentials, Expert Systems, Face, Factual, False 
Positive Reactions, Female, Gene Expression Profiling, Genetic Predisposition 
to Disease, Genome, Gram-Negative Bacteria, Heart Defects, Heart Rate, 
Hematoxylin, High Pressure Liquid, Horseradish Peroxidase, Humans, Image 
Interpretation, Image Processing, Imagery (Psychotherapy), Imagination, 
Imaging, Infant, Information Storage and Retrieval, Intracellular Space, Ion 
Exchange, Laterality, Linear Models, Male, Mammography, Melanoma, Middle Aged, 
Models, Monoclonal, Movement, Multiple, Neoplasms, Nerve Net, Neural Networks 
(Computer), Neuropeptides, Newborn, Non-P.H.S., Non-U.S. Gov't, Nonparametric, 
Noonan Syndrome, Normal Distribution, Odds Ratio, Odors, Olfactory Bulb, 
P.H.S., Pattern Recognition, Peptides, Perception, Polymorphism, Preschool, 
Principal Component Analysis, Prognosis, Protein, Protein Array Analysis, 
Protein Interaction Mapping, Proteins, Proto-Oncogene Proteins, Radiographic 
Image Enhancement, Radiographic Image Interpretation, Regression Analysis, 
Reproducibility of Results, Research Support, Risk, Sensitivity and 
Specificity, Sequence Alignment, Sequence Ana, Sequence Analysis, Signal 
Processing, Single Nucleotide, Skin Neoplasms, Smell, Software, Statistical, 
Statistics, Syndrome, Theoretical, Three-Dimensional, Time Factors, 
Transcription Factors, Tumor Markers, U.S. Gov't, User-Computer Interface, 
Velopharyngeal Insufficiency, World Health Organization, lysis, 15070507 
</blockquote> <br>
 [Zhang2005MULTIPRED]  G.&nbsp;L. Zhang, A.&nbsp;M. Khan, 
K.&nbsp;N. Srinivasan, J.&nbsp;T. August, and V.&nbsp;Brusic. MULTIPRED: a 
computational system for prediction of promiscuous HLA binding peptides.<em>
Nucleic Acids Res/</em>, 33(Web Server issue):W172-W179, Jul 2005. [&nbsp;bib
&nbsp;|DOI&nbsp;| http&nbsp;] 
<blockquote>  MULTIPRED is a web-based computational system for the prediction 
of peptide binding to multiple molecules (proteins) belonging to human 
leukocyte antigens (HLA) class I A2, A3 and class II DR supertypes. It uses 
hidden Markov models and artificial neural network methods as predictive 
engines. A novel data representation method enables MULTIPRED to predict 
peptides that promiscuously bind multiple HLA alleles within one HLA supertype. 
Extensive testing was performed for validation of the prediction models. 
Testing results show that MULTIPRED is both sensitive and specific and it has 
good predictive ability (area under the receiver operating characteristic curve 
A(ROC) &gt; 0.80). MULTIPRED can be used for the mapping of promiscuous T-cell 
epitopes as well as the regions of high concentration of these targets-termed 
T-cell epitope hotspots. MULTIPRED is available at 
http://antigen.i2r.a-star.edu.sg/multipred/. </blockquote> 
<p></p> 
<blockquote>  Keywords: Algorithms, Amino Acid Sequence, Antigen-Antibody 
Complex, Automated, Binding Sites, Computational Biology, Drug Delivery 
Systems, Drug Design, Epitopes, HLA Antigens, HLA-A Antigens, HLA-DR Antigens, 
Humans, Internet, Markov Chains, Molecular Sequence Data, Neural Networks 
(Computer), Pattern Recognition, Peptides, Protein, Protein Binding, Protein 
Interaction Mapping, Sequence Analysis, Software, T-Lymphocyte, User-Computer 
Interface, Viral Vaccines, 15980449 </blockquote> <br>
 [Tung2005GenSo-FDSS]  
W.&nbsp;L. Tung and C.&nbsp;Quek. GenSo-FDSS: a neural-fuzzy decision support 
system for pediatric ALL cancer subtype identification using gene expression 
data.<em>Artif. Intell. Med.</em>, 33(1):61-88, Jan 2005. [&nbsp;bib&nbsp;| DOI
&nbsp;|http&nbsp;| .pdf&nbsp;] 
<blockquote>  OBJECTIVE: Acute lymphoblastic leukemia (ALL) is the most common 
malignancy of childhood, representing nearly one third of all pediatric 
cancers. Currently, the treatment of pediatric ALL is centered on tailoring the 
intensity of the therapy applied to a patient's risk of relapse, which is 
linked to the type of leukemia the patient has. Hence, accurate and correct 
diagnosis of the various leukemia subtypes becomes an important first step in 
the treatment process. Recently, gene expression profiling using DNA 
microarrays has been shown to be a viable and accurate diagnostic tool to 
identify the known prognostically important ALL subtypes. Thus, there is 
currently a huge interest in developing autonomous classification systems for 
cancer diagnosis using gene expression data. This is to achieve an unbiased 
analysis of the data and also partly to handle the large amount of genetic 
information extracted from the DNA microarrays. METHODOLOGY: Generally, 
existing medical decision support systems (DSS) for cancer classification and 
diagnosis are based on traditional statistical methods such as Bayesian 
decision theory and machine learning models such as neural networks (NN) and 
support vector machine (SVM). Though high accuracies have been reported for 
these systems, they fall short on certain critical areas. These included (a) 
being able to present the extracted knowledge and explain the computed 
solutions to the users; (b) having a logical deduction process that is similar 
and intuitive to the human reasoning process; and (c) flexible enough to 
incorporate new knowledge without running the risk of eroding old but valid 
information. On the other hand, a neural fuzzy system, which is synthesized to 
emulate the human ability to learn and reason in the presence of imprecise and 
incomplete information, has the ability to overcome the above-mentioned 
shortcomings. However, existing neural fuzzy systems have their own limitations 
when used in the design and implementation of DSS. Hence, this paper proposed 
the use of a novel neural fuzzy system: the generic self-organising fuzzy 
neural network (GenSoFNN) with truth-value restriction (TVR) fuzzy inference, 
as a fuzzy DSS (denoted as GenSo-FDSS) for the classification of ALL subtypes 
using gene expression data. RESULTS AND CONCLUSION: The performance of the 
GenSo-FDSS system is encouraging when benchmarked against those of NN, SVM and 
the K-nearest neighbor (K-NN) classifier. On average, a classification rate of 
above 90% has been achieved using the GenSo-FDSS system. </blockquote> 
<p></p> 
<blockquote>  Keywords: Acute, Adenocarcinoma, Algorithms, Amino Acid, Amino 
Acid Motifs, Amino Acid Sequence, Artificial Intelligence, Automated, Cell 
Nucleus, Chemical, Child, Comparative Study, Computational Biology, Computer 
Graphics, Computer-Assisted, Cytoplasm, Databases, Decision Making, Decision 
Trees, Fuzzy Logic, Gene Expression, Gene Expression Profiling, Genetic 
Vectors, Humans, Image Enhancement, Image Interpretation, Imaging, Information 
Storage and Retrieval, Internet, Leukemia, Lymphocytic, Mitochondria, Models, 
Molecular, Molecular Sequence Data, Monte Carlo Method, Movement, Neoplasms, 
Neural Networks (Computer), Non-P.H.S., Non-U.S. Gov't, Numerical Analysis, 
Oligonucleotide Array Sequence Analysis, P.H.S., Pattern Recognition, Peptides, 
Protein, Protein Conformation, Protein Folding, Protein Structure, Proteins, 
Proteomics, ROC Curve, Reproducibility of Results, Research Support, Secondary, 
Sensitivity and Specificity, Sequence Alignment, Sequence Analysis, Sequence 
Homology, Signal Processing, Software, Statistical, Structural Homology, 
Subtraction Technique, Tertiary, Theoretical, Three-Dimensional, U.S. Gov't, 
Unknown Primary, Video Recording, 15617982 </blockquote> <br>
 [
Thukral2005Prediction]  Sushil&nbsp;K Thukral, Paul&nbsp;J Nordone, Rong Hu, 
Leah Sullivan, Eric Galambos, Vincent&nbsp;D Fitzpatrick, Laura Healy, 
Michael&nbsp;B Bass, Mary&nbsp;E Cosenza, and Cynthia&nbsp;A Afshari. 
Prediction of nephrotoxicant action and identification of candidate 
toxicity-related biomarkers.<em>Toxicol Pathol</em>, 33(3):343-55, 2005. [&nbsp;
bib&nbsp;| DOI&nbsp;| http&nbsp;] 
<blockquote>  A vast majority of pharmacological compounds and their 
metabolites are excreted via the urine, and within the complex structure of the 
kidney,the proximal tubules are a main target site of nephrotoxic compounds. We 
used the model nephrotoxicants mercuric chloride, 2-bromoethylamine 
hydrobromide, hexachlorobutadiene, mitomycin, amphotericin, and puromycin to 
elucidate time- and dose-dependent global gene expression changes associated 
with proximal tubular toxicity. Male Sprague-Dawley rats were dosed via 
intraperitoneal injection once daily for mercuric chloride and amphotericin (up 
to 7 doses), while a single dose was given for all other compounds. Animals 
were exposed to 2 different doses of these compounds and kidney tissues were 
collected on day 1, 3, and 7 postdosing. Gene expression profiles were 
generated from kidney RNA using 17K rat cDNA dual dye microarray and analyzed 
in conjunction with histopathology. Analysis of gene expression profiles showed 
that the profiles clustered based on similarities in the severity and type of 
pathology of individual animals. Further, the expression changes were 
indicative of tubular toxicity showing hallmarks of tubular 
degeneration/regeneration and necrosis. Use of gene expression data in 
predicting the type of nephrotoxicity was then tested with a support vector 
machine (SVM)-based approach. A SVM prediction module was trained using 120 
profiles of total profiles divided into four classes based on the severity of 
pathology and clustering. Although mitomycin C and amphotericin B treatments 
did not cause toxicity, their expression profiles were included in the SVM 
prediction module to increase the sample size. Using this classifier, the SVM 
predicted the type of pathology of 28 test profiles with 100% selectivity and 
82% sensitivity. These data indicate that valid predictions could be made based 
on gene expression changes from a small set of expression profiles. A set of 
potential biomarkers showing a time- and dose-response with respect to the 
progression of proximal tubular toxicity were identified. These include several 
transporters (Slc21a2, Slc15, Slc34a2), Kim 1, IGFbp-1, osteopontin, 
alpha-fibrinogen, and Gstalpha. </blockquote> 
<p></p> 
<blockquote>  Keywords: Algorithms, Animals, Antibiotics, Antineoplastic, 
Artificial Intelligence, Butadienes, Chloroplasts, Comparative Study, Computer 
Simulation, Computer-Assisted, Diagnosis, Disinfectants, Dose-Response 
Relationship, Drug, Drug Toxicity, Electrodes, Electroencephalography, 
Ethylamines, Expert Systems, Feedback, Fungicides, Gene Expression Profiling, 
Genes, Genetic Markers, Humans, Implanted, Industrial, Information Storage and 
Retrieval, Kidney, Kidney Tubules, MEDLINE, Male, Mercuric Chloride, Microarray 
Analysis, Molecular Biology, Motor Cortex, Movement, Natural Language 
Processing, Neural Networks (Computer), Non-P.H.S., Non-U.S. Gov't, Plant 
Proteins, Predictive Value of Tests, Proteins, Proteome, Proximal, Puromycin 
Aminonucleoside, Rats, Reproducibility of Results, Research Support, 
Sprague-Dawley, Subcellular Fractions, Terminology, Therapy, Time Factors, 
Toxicogenetics, U.S. Gov't, User-Computer Interface, 15805072 </blockquote> <br>
 [Shen2005[Detection]  Li&nbsp;Shen, Jie Yang, and Yue Zhou. Detection of PVCs 
with support vector machine.<em>Sheng Wu Yi Xue Gong Cheng Xue Za Zhi</em>, 
22(1):78-81, Feb 2005. [&nbsp;bib&nbsp;] 
<blockquote>  The classifiction of heart beats is the foundation for automated 
arrhythmia monitoring devices. Support vector machnies (SVMs) have meant a 
great advance in solving classification or pattern recognition. This study 
describes SVM for the identification of premature ventricular contractions 
(PVCs) in surface ECGs. Features for the classification task are extracted by 
analyzing the heart rate, morphology and wavelet energy of the heart beats from 
a single lead. The performance of different SVMs is evaluated on the MIT-BIH 
arrhythmia database following the association for the advancement of medical 
instrumentation (AAMI) recommendations. </blockquote> 
<p></p> 
<blockquote>  Keywords: 80 and over, Adult, Aged, Algorithms, Amino Acids, 
Animals, Area Under Curve, Artifacts, Automated, Birefringence, Brain 
Chemistry, Brain Neoplasms, Comparative Study, Computer-Assisted, Cornea, 
Cross-Sectional Studies, Decision Trees, Diagnosis, Diagnostic Imaging, 
Diagnostic Techniques, Discriminant Analysis, Evolution, Face, Female, Genetic, 
Glaucoma, Humans, Intraocular Pressure, Lasers, Least-Squares Analysis, 
Magnetic Resonance Imaging, Magnetic Resonance Spectroscopy, Male, Middle Aged, 
Models, Molecular, Nerve Fibers, Non-U.S. Gov't, Numerical Analysis, 
Ophthalmological, Optic Nerve Diseases, Optical Coherence, P.H.S., Pattern 
Recognition, Photic Stimulation, Prospective Studies, Protein, ROC Curve, 
Regression Analysis, Research Support, Retinal Ganglion Cells, Sensitivity and 
Specificity, Sequence Analysis, Statistics, Tomography, U.S. Gov't, Visual 
Fields, beta-Lactamases, 15762121 </blockquote> <br>
 [Sheinerman2005High]  
Felix&nbsp;B Sheinerman, Elie Giraud, and Abdelazize Laoui. High affinity 
targets of protein kinase inhibitors have similar residues at the positions 
energetically important for binding.<em>J. Mol. Biol.</em>, 352(5):1134-1156, 
Oct 2005. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;] 
<blockquote>  Inhibition of protein kinase activity is a focus of intense drug 
discovery efforts in several therapeutic areas. Major challenges facing the 
field include understanding of the factors determining the selectivity of 
kinase inhibitors and the development of compounds with the desired selectivity 
profile. Here, we report the analysis of sequence variability among high and 
low affinity targets of eight different small molecule kinase inhibitors 
(BIRB796, Tarceva, NU6102, Gleevec, SB203580, balanol, H89, PP1). It is 
observed that all high affinity targets of each inhibitor are found among a 
relatively small number of kinases, which have similar residues at the specific 
positions important for binding. The findings are highly statistically 
significant, and allow one to exclude the majority of kinases in a genome from 
a list of likely targets for an inhibitor. The findings have implications for 
the design of novel inhibitors with a desired selectivity profile (e.g. 
targeted at multiple kinases), the discovery of new targets for kinase 
inhibitor drugs, comparative analysis of different in vivo models, and the 
design of &quot;a-la-carte&quot; chemical libraries tailored for individual 
kinases. </blockquote> 
<p></p> 
<blockquote>  Keywords: Amino Acid Sequence; Amino Acids; Binding Sites; 
Electrostatics; Humans; Ligands; Molecular Sequence Data; Piperazines; Protein 
Binding; Protein Kinase Inhibitors; Protein Kinases; Pyrazoles; Pyrimidines; 
Sequence Alignment; Thermodynamics </blockquote> <br>
 [Seike2005Proteomic]  
M.&nbsp;Seike, T.&nbsp;Kondo, K.&nbsp;Fujii, T.&nbsp;Okano, T.&nbsp;Yamada, 
Y.&nbsp;Matsuno, A.&nbsp;Gemma, S.&nbsp;Kudoh, and S.&nbsp;Hirohashi. Proteomic 
signatures for histological types of lung cancer.<em>Proteomics</em>, Jul 2005. 
[&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  We performed proteomic studies on lung cancer cells to elucidate 
the mechanisms that determine histological phenotype. Thirty lung cancer cell 
lines with three different histological backgrounds (squamous cell carcinoma, 
small cell lung carcinoma and adenocarcinoma) were subjected to two-dimensional 
difference gel electrophoresis (2-D DIGE) and grouped by multivariate analyses 
on the basis of their protein expression profiles. 2-D DIGE achieves more 
accurate quantification of protein expression by using highly sensitive 
fluorescence dyes to label the cysteine residues of proteins prior to 
two-dimensional polyacrylamide gel electrophoresis. We found that hierarchical 
clustering analysis and principal component analysis divided the cell lines 
according to their original histology. Spot ranking analysis using a support 
vector machine algorithm and unsupervised classification methods identified 32 
protein spots essential for the classification. The proteins corresponding to 
the spots were identified by mass spectrometry. Next, lung cancer cells 
isolated from tumor tissue by laser microdissection were classified on the 
basis of the expression pattern of these 32 protein spots. Based on the 
expression profile of the 32 spots, the isolated cancer cells were categorized 
into three histological groups: the squamous cell carcinoma group, the 
adenocarcinoma group, and a group of carcinomas with other histological types. 
In conclusion, our results demonstrate the utility of quantitative proteomic 
analysis for molecular diagnosis and classification of lung cancer cells. 
</blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm proteomics </blockquote> <br>
 [
Sassi2005automated]  Alexander&nbsp;P Sassi, Frank Andel, Hans-Marcus&nbsp;L 
Bitter, Michael P&nbsp;S Brown, Robert&nbsp;G Chapman, Jeraldine Espiritu, 
Alfred&nbsp;C Greenquist, Isabelle Guyon, Mariana Horchi-Alegre, Kathy&nbsp;L 
Stults, Ann Wainright, Jonathan&nbsp;C Heller, and John&nbsp;T Stults. An 
automated, sheathless capillary electrophoresis-mass spectrometry platform for 
discovery of biomarkers in human serum.<em>Electrophoresis</em>, 
26(7-8):1500-12, Apr 2005. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  A capillary electrophoresis-mass spectrometry (CE-MS) method has 
been developed to perform routine, automated analysis of low-molecular-weight 
peptides in human serum. The method incorporates transient isotachophoresis for 
in-line preconcentration and a sheathless electrospray interface. To evaluate 
the performance of the method and demonstrate the utility of the approach, an 
experiment was designed in which peptides were added to sera from individuals 
at each of two different concentrations, artificially creating two groups of 
samples. The CE-MS data from the serum samples were divided into separate 
training and test sets. A pattern-recognition/feature-selection algorithm based 
on support vector machines was used to select the mass-to-charge (m/z) values 
from the training set data that distinguished the two groups of samples from 
each other. The added peptides were identified correctly as the distinguishing 
features, and pattern recognition based on these peptides was used to assign 
each sample in the independent test set to its respective group. A twofold 
difference in peptide concentration could be detected with statistical 
significance (p-value &lt; 0.0001). The accuracy of the assignment was 95%, 
demonstrating the utility of this technique for the discovery of patterns of 
biomarkers in serum. </blockquote> 
<p></p> 
<blockquote>  Keywords: 80 and over, Adult, Aged, Algorithms, Amino Acids, 
Animals, Area Under Curve, Artifacts, Automated, Birefringence, Brain 
Chemistry, Brain Neoplasms, Comparative Study, Computer-Assisted, Cornea, 
Cross-Sectional Studies, Decision Trees, Diagnosis, Diagnostic Imaging, 
Diagnostic Techniques, Discriminant Analysis, Evolution, Face, Female, Genetic, 
Glaucoma, Humans, Intraocular Pressure, Lasers, Least-Squares Analysis, 
Magnetic Resonance Imaging, Magnetic Resonance Spectroscopy, Male, Middle Aged, 
Models, Molecular, Nerve Fibers, Non-U.S. Gov't, Numerical Analysis, 
Ophthalmological, Optic Nerve Diseases, Optical Coherence, P.H.S., Pattern 
Recognition, Photic Stimulation, Prospective Studies, Protein, ROC Curve, 
Regression Analysis, Research Support, Retinal Ganglion Cells, Sensitivity and 
Specificity, Sequence Analysis, Statistics, Tomography, U.S. Gov't, Visual 
Fields, beta-Lactamases, 15765480 </blockquote> <br>
 [Perez-Cruz2005Convergence
]  Fernando P&eacute;rez-Cruz, Carlos Bouso&ntilde;o-Calz&oacute;n, and Antonio 
Art&eacute;s-Rodr&Atilde;&Acirc;&shy;guez. Convergence of the IRWLS Procedure 
to the Support Vector Machine Solution.<em>Neural Comput</em>, 17(1):7-18, Jan 
2005. [&nbsp;bib&nbsp;] 
<blockquote>  An iterative reweighted least squares (IRWLS) procedure recently 
proposed is shown to converge to the support vector machine solution. The 
convergence to a stationary point is ensured by modifying the original IRWLS 
procedure. </blockquote> 
<p></p> 
<blockquote>  Keywords: 80 and over, Aged, Algorithms, Amino Acids, Animals, 
Area Under Curve, Automated, Brain Chemistry, Brain Neoplasms, Comparative 
Study, Computer-Assisted, Cross-Sectional Studies, Decision Trees, Diagnosis, 
Diagnostic Imaging, Diagnostic Techniques, Discriminant Analysis, Evolution, 
Face, Genetic, Glaucoma, Humans, Lasers, Least-Squares Analysis, Magnetic 
Resonance Imaging, Magnetic Resonance Spectroscopy, Middle Aged, Models, 
Molecular, Nerve Fibers, Non-U.S. Gov't, Numerical Analysis, Ophthalmological, 
Optic Nerve Diseases, P.H.S., Pattern Recognition, Photic Stimulation, Protein, 
ROC Curve, Regression Analysis, Research Support, Retinal Ganglion Cells, 
Sensitivity and Specificity, Sequence Analysis, Statistics, U.S. Gov't, 
beta-Lactamases, 15779160 </blockquote> <br>
 [Peters2005Generating]  Bjoern 
Peters and Alessandro Sette. Generating quantitative models describing the 
sequence specificity of biological processes with the stabilized matrix method.
<em>BMC Bioinformatics</em>, 6:132, 2005. [&nbsp;bib&nbsp;| DOI&nbsp;| http
&nbsp;]
<blockquote>  BACKGROUND: Many processes in molecular biology involve the 
recognition of short sequences of nucleic-or amino acids, such as the binding 
of immunogenic peptides to major histocompatibility complex (MHC) molecules. 
From experimental data, a model of the sequence specificity of these processes 
can be constructed, such as a sequence motif, a scoring matrix or an artificial 
neural network. The purpose of these models is two-fold. First, they can 
provide a summary of experimental results, allowing for a deeper understanding 
of the mechanisms involved in sequence recognition. Second, such models can be 
used to predict the experimental outcome for yet untested sequences. In the 
past we reported the development of a method to generate such models called the 
Stabilized Matrix Method (SMM). This method has been successfully applied to 
predicting peptide binding to MHC molecules, peptide transport by the 
transporter associated with antigen presentation (TAP) and proteasomal cleavage 
of protein sequences. RESULTS: Herein we report the implementation of the SMM 
algorithm as a publicly available software package. Specific features 
determining the type of problems the method is most appropriate for are 
discussed. Advantageous features of the package are: (1) the output generated 
is easy to interpret, (2) input and output are both quantitative, (3) specific 
computational strategies to handle experimental noise are built in, (4) the 
algorithm is designed to effectively handle bounded experimental data, (5) 
experimental data from randomized peptide libraries and conventional peptides 
can easily be combined, and (6) it is possible to incorporate pair interactions 
between positions of a sequence. CONCLUSION: Making the SMM method publicly 
available enables bioinformaticians and experimental biologists to easily 
access it, to compare its performance to other prediction methods, and to 
extend it to other applications. </blockquote> 
<p></p> 
<blockquote>  Keywords: Algorithms; Amino Acid Sequence; Biology; 
Computational Biology; Computer Simulation; Data Interpretation, Statistical; 
Databases, Protein; Models, Biological; Models, Statistical; Neural Networks 
(Computer); Peptide Library; Peptides; Programming Languages; Prote; 
Sensitivity and Specificity; Software; in Binding </blockquote> <br>
 [
Papadopoulos2005Characterization]  A.&nbsp;Papadopoulos, D.&nbsp;I. Fotiadis, 
and A.&nbsp;Likas. Characterization of clustered microcalcifications in 
digitized mammograms using neural networks and support vector machines.<em>
Artif. Intell. Med.</em>, 34(2):141-50, Jun 2005. [&nbsp;bib&nbsp;| DOI&nbsp;| 
http&nbsp;| .pdf&nbsp;] 
<blockquote>  OBJECTIVE: Detection and characterization of microcalcification 
clusters in mammograms is vital in daily clinical practice. The scope of this 
work is to present a novel computer-based automated method for the 
characterization of microcalcification clusters in digitized mammograms. 
METHODS AND MATERIAL: The proposed method has been implemented in three stages: 
(a) the cluster detection stage to identify clusters of microcalcifications, 
(b) the feature extraction stage to compute the important features of each 
cluster and (c) the classification stage, which provides with the final 
characterization. In the classification stage, a rule-based system, an 
artificial neural network (ANN) and a support vector machine (SVM) have been 
implemented and evaluated using receiver operating characteristic (ROC) 
analysis. The proposed method was evaluated using the Nijmegen and Mammographic 
Image Analysis Society (MIAS) mammographic databases. The original feature set 
was enhanced by the addition of four rule-based features. RESULTS AND 
CONCLUSIONS: In the case of Nijmegen dataset, the performance of the SVM was 
Az=0.79 and 0.77 for the original and enhanced feature set, respectively, while 
for the MIAS dataset the corresponding characterization scores were Az=0.81 and 
0.80. Utilizing neural network classification methodology, the corresponding 
performance for the Nijmegen dataset was Az=0.70 and 0.76 while for the MIAS 
dataset it was Az=0.73 and 0.78. Although the obtained high classification 
performance can be successfully applied to microcalcification clusters 
characterization, further studies must be carried out for the clinical 
evaluation of the system using larger datasets. The use of additional features 
originating either from the image itself (such as cluster location and 
orientation) or from the patient data may further improve the diagnostic value 
of the system. </blockquote> 
<p></p> 
<blockquote>  Keywords: Apoptosis, Gene Expression Profiling, Humans, 
Neoplasms, Non-U.S. Gov't, Oligonucleotide Array Sequence Analysis, Polymerase 
Chain Reaction, Proteins, Research Support, Subcellular Fractions, Unknown 
Primary, 15894178 </blockquote> <br>
 [Miteva2005Fast]  M.&nbsp;A. Miteva, 
W.&nbsp;H. Lee, M.&nbsp;O. Montes, and B.&nbsp;O. Villoutreix. Fast 
structure-based virtual ligand screening combining FRED, DOCK, and Surflex.<em>
J. Med. Chem.</em>, 48(19):6012-6022, Sep 2005. [&nbsp;bib&nbsp;| DOI&nbsp;| 
http&nbsp;] 
<blockquote>  A protocol was devised in which FRED, DOCK, and Surflex were 
combined in a multistep virtual ligand screening (VLS) procedure to screen the 
pocket of four different proteins. One goal was to evaluate the impact of 
chaining &quot;freely available packages to academic users&quot; on 
docking/scoring accuracy and CPU time consumption. A bank of 65 660 compounds 
including 49 known actives was generated. Our procedure is successful because 
docking/scoring parameters are tuned according to the nature of the binding 
pocket and because a shape-based filtering tool is applied prior to flexible 
docking. The obtained enrichment factors are in line with those reported in 
recent studies. We suggest that consensus docking/scoring could be valuable to 
some drug discovery projects. The present protocol could process the entire 
bank for one receptor in less than a week on one processor, suggesting that VLS 
experiments could be performed even without large computer resources. 
</blockquote> 
<p></p> 
<blockquote>  Keywords: Binding Sites, Databases, Estrogen, Factor VIIa, 
Factual, Ligands, Molecular Structure, Neuraminidase, Non-U.S. Gov't, Protein 
Binding, Quantitative Structure-Activity Relationship, Receptors, Research 
Support, Thymidine Kinase, 16162004 </blockquote> <br>
 [Micchelli2005On]  
Charles&nbsp;A Micchelli and Massimiliano Pontil. On learning vector-valued 
functions.<em>Neural Comput</em>, 17(1):177-204, Jan 2005. [&nbsp;bib&nbsp;| DOI
&nbsp;|http&nbsp;] 
<blockquote>  In this letter, we provide a study of learning in a Hilbert 
space of vectorvalued functions. We motivate the need for extending learning 
theory of scalar-valued functions by practical considerations and establish 
some basic results for learning vector-valued functions that should prove 
useful in applications. Specifically, we allow an output space Y to be a 
Hilbert space, and we consider a reproducing kernel Hilbert space of functions 
whose values lie in Y. In this setting, we derive the form of the minimal norm 
interpolant to a finite set of data and apply it to study some regularization 
functionals that are important in learning theory. We consider specific 
examples of such functionals corresponding to multiple-output regularization 
networks and support vector machines, for both regression and classification. 
Finally, we provide classes of operator-valued kernels of the dot product and 
translation-invariant type. </blockquote> 
<p></p> 
<blockquote>  Keywords: Algorithms, Amino Acid, Amino Acids, Artificial 
Intelligence, Ascomycota, Automated, Base Sequence, Chromosome Mapping, Codon, 
Colonic Neoplasms, Comparative Study, Computer Simulation, Computer-Assisted, 
Computing Methodologies, Crystallography, DNA, DNA Primers, Databases, Decision 
Support Techniques, Diagnostic Imaging, Enzymes, Feedback, Fixation, Gene 
Expression Profiling, Genetic, Hordeum, Host-Parasite Relations, Humans, Image 
Enhancement, Image Interpretation, Informatics, Information Storage and 
Retrieval, Kinetics, Logistic Models, Magnetic Resonance Spectroscopy, 
Mathematical Computing, Models, Nanotechnology, Neural Networks (Computer), 
Non-P.H.S., Non-U.S. Gov't, Nonlinear Dynamics, Ocular, Oligonucleotide Array 
Sequence Analysis, P.H.S., Pattern Recognition, Plant, Plants, Predictive Value 
of Tests, Protein, Protein Conformation, Regression Analysis, Research Support, 
Sample Size, Selection (Genetics), Sequence Alignment, Sequence Analysis, 
Sequence Homology, Signal Processing, Skin, Software, Statistical, Subtraction 
Technique, Theoretical, Thermodynamics, U.S. Gov't, Viral Proteins, X-Ray, 
15563752 </blockquote> <br>
 [Mavroforakis2005Significance]  Michael 
Mavroforakis, Harris Georgiou, Nikos Dimitropoulos, Dionisis Cavouras, and 
Sergios Theodoridis. Significance analysis of qualitative mammographic 
features, using linear classifiers, neural networks and support vector machines.
<em>Eur J Radiol</em>, 54(1):80-9, Apr 2005. [&nbsp;bib&nbsp;| DOI&nbsp;| http
&nbsp;|.pdf&nbsp;] 
<blockquote>  Advances in modern technologies and computers have enabled 
digital image processing to become a vital tool in conventional clinical 
practice, including mammography. However, the core problem of the clinical 
evaluation of mammographic tumors remains a highly demanding cognitive task. In 
order for these automated diagnostic systems to perform in levels of 
sensitivity and specificity similar to that of human experts, it is essential 
that a robust framework on problem-specific design parameters is formulated. 
This study is focused on identifying a robust set of clinical features that can 
be used as the base for designing the input of any computer-aided diagnosis 
system for automatic mammographic tumor evaluation. A thorough list of clinical 
features was constructed and the diagnostic value of each feature was verified 
against current clinical practices by an expert physician. These features were 
directly or indirectly related to the overall morphological properties of the 
mammographic tumor or the texture of the fine-scale tissue structures as they 
appear in the digitized image, while others contained external clinical data of 
outmost importance, like the patient's age. The entire feature set was used as 
an annotation list for describing the clinical properties of mammographic tumor 
cases in a quantitative way, such that subsequent objective analyses were 
possible. For the purposes of this study, a mammographic image database was 
created, with complete clinical evaluation descriptions and positive 
histological verification for each case. All tumors contained in the database 
were characterized according to the identified clinical features' set and the 
resulting dataset was used as input for discrimination and diagnostic value 
analysis for each one of these features. Specifically, several standard 
methodologies of statistical significance analysis were employed to create 
feature rankings according to their discriminating power. Moreover, three 
different classification models, namely linear classifiers, neural networks and 
support vector machines, were employed to investigate the true efficiency of 
each one of them, as well as the overall complexity of the diagnostic task of 
mammographic tumor characterization. Both the statistical and the 
classification results have proven the explicit correlation of all the selected 
features with the final diagnosis, qualifying them as an adequate input base 
for any type of similar automated diagnosis system. The underlying complexity 
of the diagnostic task has justified the high value of sophisticated pattern 
recognition architectures. </blockquote> 
<p></p> 
<blockquote>  Keywords: Algorithms, Animals, Antibiotics, Antineoplastic, 
Artificial Intelligence, Butadienes, Chloroplasts, Comparative Study, Computer 
Simulation, Computer-Assisted, Diagnosis, Disinfectants, Dose-Response 
Relationship, Drug, Drug Toxicity, Electrodes, Electroencephalography, 
Ethylamines, Expert Systems, Feedback, Fungicides, Gene Expression Profiling, 
Genes, Genetic Markers, Humans, Implanted, Industrial, Information Storage and 
Retrieval, Kidney, Kidney Tubules, MEDLINE, Male, Mercuric Chloride, Microarray 
Analysis, Molecular Biology, Motor Cortex, Movement, Natural Language 
Processing, Neural Networks (Computer), Non-P.H.S., Non-U.S. Gov't, Plant 
Proteins, Predictive Value of Tests, Proteins, Proteome, Proximal, Puromycin 
Aminonucleoside, Rats, Reproducibility of Results, Research Support, 
Sprague-Dawley, Subcellular Fractions, Terminology, Therapy, Time Factors, 
Toxicogenetics, U.S. Gov't, User-Computer Interface, 15797296 </blockquote> <br>
 [Lu2005Potential]  Wei-Zhen Lu and Wen-Jian Wang. Potential assessment of the 
&quot;support vector machine&quot; method in forecasting ambient air pollutant 
trends.<em>Chemosphere</em>, 59(5):693-701, Apr 2005. [&nbsp;bib&nbsp;| DOI
&nbsp;|http&nbsp;| .pdf&nbsp;] 
<blockquote>  Monitoring and forecasting of air quality parameters are popular 
and important topics of atmospheric and environmental research today due to the 
health impact caused by exposing to air pollutants existing in urban air. The 
accurate models for air pollutant prediction are needed because such models 
would allow forecasting and diagnosing potential compliance or non-compliance 
in both short- and long-term aspects. Artificial neural networks (ANN) are 
regarded as reliable and cost-effective method to achieve such tasks and have 
produced some promising results to date. Although ANN has addressed more 
attentions to environmental researchers, its inherent drawbacks, e.g., local 
minima, over-fitting training, poor generalization performance, determination 
of the appropriate network architecture, etc., impede the practical application 
of ANN. Support vector machine (SVM), a novel type of learning machine based on 
statistical learning theory, can be used for regression and time series 
prediction and have been reported to perform well by some promising results. 
The work presented in this paper aims to examine the feasibility of applying 
SVM to predict air pollutant levels in advancing time series based on the 
monitored air pollutant database in Hong Kong downtown area. At the same time, 
the functional characteristics of SVM are investigated in the study. The 
experimental comparisons between the SVM model and the classical radial basis 
function (RBF) network demonstrate that the SVM is superior to the conventional 
RBF network in predicting air quality parameters with different time series and 
of better generalization performance than the RBF model. </blockquote> 
<p></p> 
<blockquote>  Keywords: Air Pollutants, Algorithms, Animals, Antibiotics, 
Antineoplastic, Artificial Intelligence, Automated, Automatic Data Processing, 
Butadienes, Chloroplasts, Cluster Analysis, Comparative Study, Computer 
Simulation, Computer-Assisted, Computing Methodologies, Database Management 
Systems, Databases, Diagnosis, Disinfectants, Dose-Response Relationship, Drug, 
Drug Toxicity, Electrodes, Electroencephalography, Environmental, Ethylamines, 
Expert Systems, Factual, Feedback, Forecasting, Fungicides, Gene Expression 
Profiling, Genes, Genetic Markers, Hong Kong, Humans, Image Enhancement, Image 
Interpretation, Implanted, Industrial, Information Storage and Retrieval, 
Kidney, Kidney Tubules, MEDLINE, Male, Mercuric Chloride, Microarray Analysis, 
Models, Molecular Biology, Motor Cortex, Movement, Natural Language Processing, 
Neural Networks (Computer), Non-P.H.S., Non-U.S. Gov't, Numerical Analysis, 
Pattern Recognition, Plant Proteins, Predictive Value of Tests, Proteins, 
Proteome, Proximal, Puromycin Aminonucleoside, Rats, Reproducibility of 
Results, Research Support, Sensitivity and Specificity, Signal Processing, 
Sprague-Dawley, Subcellular Fractions, Terminology, Theoretical, Therapy, Time 
Factors, Toxicogenetics, U.S. Gov't, User-Computer Interface, Vehicle 
Emissions, 15792667 </blockquote> <br>
 [Lasso2005Vessel]  
Andr&Atilde;&Acirc;&iexcl;s Lass&oacute; and Emanuele Trucco. Vessel 
enhancement in digital X-ray angiographic sequences by temporal statistical 
learning.<em>Comput Med Imaging Graph</em>, 29(5):343-55, Jul 2005. [&nbsp;bib
&nbsp;|DOI&nbsp;| http&nbsp;] 
<blockquote>  Keywords: Apoptosis, Gene Expression Profiling, Humans, 
Neoplasms, Non-U.S. Gov't, Oligonucleotide Array Sequence Analysis, Polymerase 
Chain Reaction, Proteins, Research Support, Subcellular Fractions, Unknown 
Primary, 15893453 </blockquote> <br>
 [Larsen2005integrative]  
Mette&nbsp;Voldby Larsen, Claus Lundegaard, Kasper Lamberth, S&oslash;ren Buus, 
S&oslash;ren Brunak, Ole Lund, and Morten Nielsen. An integrative approach to 
CTL epitope prediction: a combined algorithm integrating MHC class I binding, 
TAP transport efficiency, and proteasomal cleavage predictions.<em>Eur. J. 
Immunol.</em>, 35(8):2295-2303, Aug 2005. [&nbsp;bib&nbsp;| DOI&nbsp;| http
&nbsp;]
<blockquote>  Reverse immunogenetic approaches attempt to optimize the 
selection of candidate epitopes, and thus minimize the experimental effort 
needed to identify new epitopes. When predicting cytotoxic T cell epitopes, the 
main focus has been on the highly specific MHC class I binding event. Methods 
have also been developed for predicting the antigen-processing steps preceding 
MHC class I binding, including proteasomal cleavage and transporter associated 
with antigen processing (TAP) transport efficiency. Here, we use a dataset 
obtained from the SYFPEITHI database to show that a method integrating 
predictions of MHC class I binding affinity, TAP transport efficiency, and 
C-terminal proteasomal cleavage outperforms any of the individual methods. 
Using an independent evaluation dataset of HIV epitopes from the Los Alamos 
database, the validity of the integrated method is confirmed. The performance 
of the integrated method is found to be significantly higher than that of the 
two publicly available prediction methods BIMAS and SYFPEITHI. To identify 85% 
of the epitopes in the HIV dataset, 9% and 10% of all possible nonamers in the 
HIV proteins must be tested when using the BIMAS and SYFPEITHI methods, 
respectively, for the selection of candidate epitopes. This number is reduced 
to 7% when using the integrated method. In practical terms, this means that the 
experimental effort needed to identify an epitope in a hypothetical protein 
with 85% probability is reduced by 20-30% when using the integrated method.The 
method is available at http://www.cbs.dtu.dk/services/NetCTL. Supplementary 
material is available at http://www.cbs.dtu.dk/suppl/immunology/CTL.php. 
</blockquote> 
<p></p> 
<blockquote>  Keywords: Algorithms; Data Interpretation, Statistical; 
Epitopes, T-Lymphocyte; Histocompatibility Antigens Class I; Humans; 
Hydrolysis; Predictive Value of Tests; Proteasome Endopeptidase Complex; 
Protein Binding; Research Support, N.I.H., Extramural; Research Support, 
Non-U.S. Gov't; Research Support, U.S. Gov't, P.H.S.; T-Lymphocytes, Cytotoxic 
</blockquote> <br>
 [Ikeda2005asymptotic]  Kazushi Ikeda and Tsutomu Aoishi. An 
asymptotic statistical analysis of support vector machines with soft margins.
<em>Neural Netw</em>, 18(3):251-9, Apr 2005. [&nbsp;bib&nbsp;| DOI&nbsp;| http
&nbsp;|.pdf&nbsp;] 
<blockquote>  The generalization properties of support vector machines (SVMs) 
are examined. From a geometrical point of view, the estimated parameter of an 
SVM is the one nearest the origin in the convex hull formed with given 
examples. Since introducing soft margins is equivalent to reducing the convex 
hull of the examples, an SVM with soft margins has a different learning curve 
from the original. In this paper we derive the asymptotic average 
generalization error of SVMs with soft margins in simple cases, that is, only 
when the dimension of inputs is one, and quantitatively show that soft margins 
increase the generalization error. </blockquote> 
<p></p> 
<blockquote>  Keywords: Apoptosis, Gene Expression Profiling, Humans, 
Neoplasms, Non-U.S. Gov't, Oligonucleotide Array Sequence Analysis, Polymerase 
Chain Reaction, Proteins, Research Support, Subcellular Fractions, Unknown 
Primary, 15896573 </blockquote> <br>
 [Haasdonk2005Feature]  Bernard Haasdonk. 
Feature space interpretation of SVMs with indefinite kernels.<em>IEEE Trans 
Pattern Anal Mach Intell</em>, 27(4):482-92, Apr 2005. [&nbsp;bib&nbsp;| DOI
&nbsp;|http&nbsp;| .pdf&nbsp;] 
<blockquote>  Kernel methods are becoming increasingly popular for various 
kinds of machine learning tasks, the most famous being the support vector 
machine (SVM) for classification. The SVM is well understood when using 
conditionally positive definite (cpd) kernel functions. However, in practice, 
non-cpd kernels arise and demand application in SVMs. The procedure of 
&quot;plugging&quot; these indefinite kernels in SVMs often yields good 
empirical classification results. However, they are hard to interpret due to 
missing geometrical and theoretical understanding. In this paper, we provide a 
step toward the comprehension of SVM classifiers in these situations. We give a 
geometric interpretation of SVMs with indefinite kernel functions. We show that 
such SVMs are optimal hyperplane classifiers not by margin maximization, but by 
minimization of distances between convex hulls in pseudo-Euclidean spaces. By 
this, we obtain a sound framework and motivation for indefinite SVMs. This 
interpretation is the basis for further theoretical analysis, e.g., 
investigating uniqueness, and for the derivation of practical guidelines like 
characterizing the suitability of indefinite SVMs. </blockquote> 
<p></p> 
<blockquote>  Keywords: Algorithms, Animals, Antibiotics, Antineoplastic, 
Artificial Intelligence, Automated, Automatic Data Processing, Butadienes, 
Chloroplasts, Cluster Analysis, Comparative Study, Computer Simulation, 
Computer-Assisted, Computing Methodologies, Database Management Systems, 
Databases, Diagnosis, Disinfectants, Dose-Response Relationship, Drug, Drug 
Toxicity, Electrodes, Electroencephalography, Ethylamines, Expert Systems, 
Factual, Feedback, Fungicides, Gene Expression Profiling, Genes, Genetic 
Markers, Humans, Image Enhancement, Image Interpretation, Implanted, 
Industrial, Information Storage and Retrieval, Kidney, Kidney Tubules, MEDLINE, 
Male, Mercuric Chloride, Microarray Analysis, Molecular Biology, Motor Cortex, 
Movement, Natural Language Processing, Neural Networks (Computer), Non-P.H.S., 
Non-U.S. Gov't, Numerical Analysis, Pattern Recognition, Plant Proteins, 
Predictive Value of Tests, Proteins, Proteome, Proximal, Puromycin 
Aminonucleoside, Rats, Reproducibility of Results, Research Support, 
Sensitivity and Specificity, Signal Processing, Sprague-Dawley, Subcellular 
Fractions, Terminology, Therapy, Time Factors, Toxicogenetics, U.S. Gov't, 
User-Computer Interface, 15794155 </blockquote> <br>
 [Guo2005Feature]  Hong 
Guo, Lindsay&nbsp;B Jack, and Asoke&nbsp;K Nandi. Feature generation using 
genetic programming with application to fault classification.<em>IEEE Trans 
Syst Man Cybern B Cybern</em>, 35(1):89-99, Feb 2005. [&nbsp;bib&nbsp;] 
<blockquote>  One of the major challenges in pattern recognition problems is 
the feature extraction process which derives new features from existing 
features, or directly from raw data in order to reduce the cost of computation 
during the classification process, while improving classifier efficiency. Most 
current feature extraction techniques transform the original pattern vector 
into a new vector with increased discrimination capability but lower 
dimensionality. This is conducted within a predefined feature space, and thus, 
has limited searching power. Genetic programming (GP) can generate new features 
from the original dataset without prior knowledge of the probabilistic 
distribution. In this paper, a GP-based approach is developed for feature 
extraction from raw vibration data recorded from a rotating machine with six 
different conditions. The created features are then used as the inputs to a 
neural classifier for the identification of six bearing conditions. 
Experimental results demonstrate the ability of GP to discover autimatically 
the different bearing conditions using features expressed in the form of 
nonlinear functions. Furthermore, four sets of results-using GP extracted 
features with artificial neural networks (ANN) and support vector machines 
(SVM), as well as traditional features with ANN and SVM-have been obtained. 
This GP-based approach is used for bearing fault classification for the first 
time and exhibits superior searching power over other techniques. Additionaly, 
it significantly reduces the time for computation compared with genetic 
algorithm (GA), therefore, makes a more practical realization of the solution. 
</blockquote> 
<p></p> 
<blockquote>  Keywords: 80 and over, Acute, Adolescent, Adult, Aged, 
Algorithms, Amino Acids, Animals, Artificial Intelligence, Automated, Bacillus, 
Bacteria, Bacterial, Bacterial Typing Techniques, Base Sequence, Bayes Theorem, 
Binding Sites, Biological, Biometry, Blood Coagulation, Breast Neoplasms, 
Breath Tests, Calcinosis, Carcinoma, Child, Chromatography, Chromosomes, 
Cluster Analysis, Colonic Neoplasms, Comparative Study, Complementary, 
Computational Biology, Computer Simulation, Computer-Assisted, Computing 
Methodologies, Controlled, Cross-Sectional Studies, DNA, Data Interpretation, 
Databases, Diagnosis, Discriminant Analysis, Environment, Environmental 
Microbiology, Equipment Failure, Equipment Failure Analysis, Escherichia coli, 
Extramural, Face, Factual, False Negative Reactions, Female, Gas, Gene 
Expression Profiling, Genetic, Human, Humans, Image Enhancement, Image 
Interpretation, Information Storage and Retrieval, Lasers, Leukemia, Linear 
Models, Lung Neoplasms, Male, Mass, Messenger, Micrococcus, Middle Aged, 
Models, Molecular Sequence Data, N.I.H., Neural Networks (Computer), 
Non-P.H.S., Non-U.S. Gov't, Numerical Analysis, Oligonucleotide Array Sequence 
Analysis, Ovarian Neoplasms, P.H.S., Pair 15, Pair 17, Pair 22, Papillary, 
Pattern Recognition, Peptide Mapping, Phenotype, Photobleaching, Photography, 
Precancerous Conditions, Predictive Value of Tests, Preschool, Principal 
Component Analysis, Promoter Regions (Genetics), Promyelocytic, Protein 
Biosynthesis, Proteomics, RNA, RNA Polymerase II, Radiographic Image 
Enhancement, Radiographic Image Interpretation, Raman, Reproducibility of 
Results, Research Support, Reverse Transcriptase Polymerase Chain Reaction, 
Sensitivity and Specificit, Sensitivity and Specificity, Sequence Analysis, 
Severity of Illness Index, Signal Processing, Spectrum Analysis, Spores, 
Staphylococcus, Statistical, Statistical Distributions, Theoretical, Thyroid 
Neoplasms, Time Factors, Translocation, Tumor Markers, U.S. Gov't, y, 15719936 
</blockquote> <br>
 [Golland2005Detection]  Polina Golland, W.&nbsp;Eric&nbsp;L 
Grimson, Martha&nbsp;E Shenton, and Ron Kikinis. Detection and analysis of 
statistical differences in anatomical shape.<em>Med Image Anal</em>, 
9(1):69-86, Feb 2005. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;] 
<blockquote>  We present a computational framework for image-based analysis 
and interpretation of statistical differences in anatomical shape between 
populations. Applications of such analysis include understanding developmental 
and anatomical aspects of disorders when comparing patients versus normal 
controls, studying morphological changes caused by aging, or even differences 
in normal anatomy, for example, differences between genders. Once a 
quantitative description of organ shape is extracted from input images, the 
problem of identifying differences between the two groups can be reduced to one 
of the classical questions in machine learning of constructing a classifier 
function for assigning new examples to one of the two groups while making as 
few misclassifications as possible. The resulting classifier must be 
interpreted in terms of shape differences between the two groups back in the 
image domain. We demonstrate a novel approach to such interpretation that 
allows us to argue about the identified shape differences in anatomically 
meaningful terms of organ deformation. Given a classifier function in the 
feature space, we derive a deformation that corresponds to the differences 
between the two classes while ignoring shape variability within each class. 
Based on this approach, we present a system for statistical shape analysis 
using distance transforms for shape representation and the support vector 
machines learning algorithm for the optimal classifier estimation and 
demonstrate it on artificially generated data sets, as well as real medical 
studies. </blockquote> 
<p></p> 
<blockquote>  Keywords: Algorithms, Amino Acid, Artificial Intelligence, 
Ascomycota, Automated, Base Sequence, Chromosome Mapping, Codon, Colonic 
Neoplasms, Comparative Study, Computer-Assisted, Crystallography, DNA, DNA 
Primers, Databases, Diagnostic Imaging, Gene Expression Profiling, Hordeum, 
Host-Parasite Relations, Humans, Image Interpretation, Informatics, Kinetics, 
Magnetic Resonance Spectroscopy, Models, Nanotechnology, Non-P.H.S., Non-U.S. 
Gov't, Oligonucleotide Array Sequence Analysis, P.H.S., Pattern Recognition, 
Plant, Plants, Predictive Value of Tests, Protein, Research Support, Selection 
(Genetics), Sequence Alignment, Sequence Analysis, Sequence Homology, Skin, 
Software, Statistical, Theoretical, Thermodynamics, U.S. Gov't, Viral Proteins, 
X-Ray, 15581813 </blockquote> <br>
 [Ehlers2005NBS1]  Justis&nbsp;P Ehlers and 
J.&nbsp;William Harbour. NBS1 expression as a prognostic marker in uveal 
melanoma.<em>Clin. Cancer Res.</em>, 11(5):1849-53, Mar 2005. [&nbsp;bib&nbsp;| 
DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  PURPOSE: Up to half of uveal melanoma patients die of metastatic 
disease. Treatment of the primary eye tumor does not improve survival in 
high-risk patients due to occult micrometastatic disease, which is present at 
the time of eye tumor diagnosis but is not detected and treated until months to 
years later. Here, we use microarray gene expression data to identify a new 
prognostic marker. EXPERIMENTAL DESIGN: Microarray gene expression profiles 
were analyzed in 25 primary uveal melanomas. Tumors were ranked by support 
vector machine (SVM) and by cytologic severity. Nbs1 protein expression was 
assessed by quantitative immunohistochemistry in 49 primary uveal melanomas. 
Survival was assessed using Kaplan-Meier life-table analysis. RESULTS: 
Expression of the Nijmegen breakage syndrome (NBS1) gene correlated strongly 
with SVM and cytologic tumor rankings (P &lt; 0.0001). Further, 
immunohistochemistry expression of the Nbs1 protein correlated strongly with 
both SVM and cytologic rankings (P &lt; 0.0001). The 6-year actuarial survival 
was 100% in patients with low immunohistochemistry expression of Nbs1 and 22% 
in those with high Nbs1 expression (P = 0.01). CONCLUSIONS: NBS1 is a strong 
predictor of uveal melanoma survival and potentially could be used as a 
clinical marker for guiding clinical management. </blockquote> 
<p></p> 
<blockquote>  Keywords: 80 and over, Adult, Aged, Algorithms, Amino Acid 
Sequence, Amino Acids, Analysis of Variance, Animals, Area Under Curve, 
Artifacts, Automated, Bacteriophage T4, Base Sequence, Biological, 
Birefringence, Brain Chemistry, Brain Neoplasms, Cell Cycle Proteins, 
Comparative Study, Computational Biology, Computer-Assisted, Cornea, 
Cross-Sectional Studies, Databases, Decision Trees, Diagnosis, Diagnostic 
Imaging, Diagnostic Techniques, Discriminant Analysis, Evolution, Extramural, 
Face, Female, Gene Expression Profiling, Genetic, Glaucoma, Humans, 
Immunohistochemistry, Intraocular Pressure, Lasers, Least-Squares Analysis, 
Likelihood Functions, Magnetic Resonance Imaging, Magnetic Resonance 
Spectroscopy, Male, Markov Chains, Melanoma, Middle Aged, Models, Molecular, 
Mutation, N.I.H., Nerve Fibers, Non-P.H.S., Non-U.S. Gov't, Nuclear Proteins, 
Nucleic Acid, Nucleic Acid Conformation, Numerical Analysis, Oligonucleotide 
Array Sequence Analysis, Ophthalmological, Optic Nerve Diseases, Optical 
Coherence, P.H.S., Pattern Recognition, Photic Stimulation, Polymorphism, 
Prognosis, Prospective Studies, Protein, Protein Structure, Proteins, RNA, ROC 
Curve, Regression Analysis, Reproducibility of Results, Research Support, 
Retinal Ganglion Cells, Secondary, Sensitivity and Specificity, Sequence 
Analysis, Single Nucleotide, Single-Stranded Conformational, Software, 
Statistics, Survival Analysis, Tertiary, Tomography, Tumor Markers, U.S. Gov't, 
Untranslated, Uveal Neoplasms, Visual Fields, beta-Lactamases, 15756009 
</blockquote> <br>
 [Dong2005Fast]  Jian xiong Dong, Adam Krzyzak, and 
Ching&nbsp;Y Suen. Fast SVM training algorithm with decomposition on very large 
data sets.<em>IEEE Trans Pattern Anal Mach Intell</em>, 27(4):603-18, Apr 2005. 
[&nbsp;bib&nbsp;] 
<blockquote>  Training a support vector machine on a data set of huge size 
with thousands of classes is a challenging problem. This paper proposes an 
efficient algorithm to solve this problem. The key idea is to introduce a 
parallel optimization step to quickly remove most of the nonsupport vectors, 
where block diagonal matrices are used to approximate the original kernel 
matrix so that the original problem can be split into hundreds of subproblems 
which can be solved more efficiently. In addition, some effective strategies 
such as kernel caching and efficient computation of kernel matrix are 
integrated to speed up the training process. Our analysis of the proposed 
algorithm shows that its time complexity grows linearly with the number of 
classes and size of the data set. In the experiments, many appealing properties 
of the proposed algorithm have been investigated and the results show that the 
proposed algorithm has a much better scaling capability than Libsvm, SVMlight, 
and SVMTorch. Moreover, the good generalization performances on several large 
databases have also been achieved. </blockquote> 
<p></p> 
<blockquote>  Keywords: Algorithms, Animals, Antibiotics, Antineoplastic, 
Artificial Intelligence, Automated, Automatic Data Processing, Butadienes, 
Chloroplasts, Comparative Study, Computer Simulation, Computer-Assisted, 
Database Management Systems, Databases, Diagnosis, Disinfectants, Dose-Response 
Relationship, Drug, Drug Toxicity, Electrodes, Electroencephalography, 
Ethylamines, Expert Systems, Factual, Feedback, Fungicides, Gene Expression 
Profiling, Genes, Genetic Markers, Humans, Image Enhancement, Image 
Interpretation, Implanted, Industrial, Information Storage and Retrieval, 
Kidney, Kidney Tubules, MEDLINE, Male, Mercuric Chloride, Microarray Analysis, 
Molecular Biology, Motor Cortex, Movement, Natural Language Processing, Neural 
Networks (Computer), Non-P.H.S., Non-U.S. Gov't, Numerical Analysis, Pattern 
Recognition, Plant Proteins, Predictive Value of Tests, Proteins, Proteome, 
Proximal, Puromycin Aminonucleoside, Rats, Reproducibility of Results, Research 
Support, Sensitivity and Specificity, Signal Processing, Sprague-Dawley, 
Subcellular Fractions, Terminology, Therapy, Time Factors, Toxicogenetics, U.S. 
Gov't, User-Computer Interface, 15794164 </blockquote> <br>
 [Dong2005Prediction
]  Hai-Long Dong and Yan-Fang Sui. Prediction of HLA-A2-restricted CTL epitope 
specific to HCC by SYFPEITHI combined with polynomial method.<em>World J 
Gastroenterol</em>, 11(2):208-211, Jan 2005. [&nbsp;bib&nbsp;] 
<blockquote>  AIM: To predict the HLA-A2-restricted CTL epitopes of tumor 
antigens associated with hepatocellular carcinoma (HCC). METHODS: MAGE-1, 
MAGE-3, MAGE-8, P53 and AFP were selected as objective antigens in this study 
for the close association with HCC. The HLA-A*0201 restricted CTL epitopes of 
objective tumor antigens were predicted by SYFPEITHI prediction method combined 
with the polynomial quantitative motifs method. The threshold of polynomial 
scores was set to -24. RESULTS: The SYFPEITHI prediction values of all possible 
nonamers of a given protein sequence were added together and the ten 
high-scoring peptides of each protein were chosen for further analysis in 
primary prediction. Thirty-five candidates of CTL epitopes (nonamers) derived 
from the primary prediction results were selected by analyzing with the 
polynomial method and compared with reported CTL epitopes. CONCLUSION: The 
combination of SYFPEITHI prediction method and polynomial method can improve 
the prediction efficiency and accuracy. These nonamers may be useful in the 
design of therapeutic peptide vaccine for HCC and as immunotherapeutic 
strategies against HCC after identified by immunology experiment. </blockquote> 
<p></p> 
<blockquote>  Keywords: Amino Acid Sequence; Carcinoma, Hepatocellular; 
Databases, Protein; Epitopes; HLA-A2 Antigen; Humans; Liver Neoplasms; Major 
Histocompatibility Complex; Research Support, Non-U.S. Gov't; T-Lymphocytes, 
Cytotoxic </blockquote> <br>
 [Ding2005Minimum]  Chris Ding and Hanchuan Peng. 
Minimum redundancy feature selection from microarray gene expression data.<em>J 
Bioinform Comput Biol</em>, 3(2):185-205, Apr 2005. [&nbsp;bib&nbsp;] 
<blockquote>  How to selecting a small subset out of the thousands of genes in 
microarray data is important for accurate classification of phenotypes. Widely 
used methods typically rank genes according to their differential expressions 
among phenotypes and pick the top-ranked genes. We observe that feature sets so 
obtained have certain redundancy and study methods to minimize it. We propose a 
minimum redundancy - maximum relevance (MRMR) feature selection framework. 
Genes selected via MRMR provide a more balanced coverage of the space and 
capture broader characteristics of phenotypes. They lead to significantly 
improved class predictions in extensive experiments on 6 gene expression data 
sets: NCI, Lymphoma, Lung, Child Leukemia, Leukemia, and Colon. Improvements 
are observed consistently among 4 classification methods: Naive Bayes, Linear 
discriminant analysis, Logistic regression, and Support vector machines. 
SUPPLIMENTARY: The top 60 MRMR genes for each of the datasets are listed in 
http://crd.lbl.gov/&nbsp;cding/MRMR/. More information related to MRMR methods 
can be found at http://www.hpeng.net/. </blockquote> 
<p></p> 
<blockquote>  Keywords: Adult, Aged, Aging, Algorithms, Animals, Apoptosis, 
Artificial Intelligence, Automated, Biological, Bone Marrow, Breast Neoplasms, 
Classification, Cluster Analysis, Comparative Study, Computer Simulation, 
Computer-Assisted, Diagnosis, Dose-Response Relationship, Drug, Female, Foot, 
Gait, Gene Expression Profiling, Gene Expression Regulation, Gene Silencing, 
Genetic Vectors, Humans, Image Interpretation, Information Storage and 
Retrieval, Kidney, Liver, Logistic Models, Male, Messenger, Models, Myocardium, 
Neoplasms, Non-U.S. Gov't, Oligonucleotide Array Sequence Analysis, Pattern 
Recognition, Pharmaceutical Preparations, Polymerase Chain Reaction, Principal 
Component Analysis, Proteins, RNA, Rats, Reproducibility of Results, Research 
Support, Sensitivity and Specificity, Small Interfering, Sprague-Dawley, 
Statistical, Subcellular Fractions, Unknown Primary, 15852500 </blockquote> <br>
 [Dhingra2005Substantial]  Vikas Dhingra, Mukta Gupta, Tracy Andacht, and 
Zhen&nbsp;F Fu. New frontiers in proteomics research: a perspective.<em>Int. J. 
Pharm.</em>, 299(1-2):1-18, Aug 2005. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;] 
<blockquote>  Substantial advances have been made in the fundamental 
understanding of human biology, ranging from DNA structure to identification of 
diseases associated with genetic abnormalities. Genome sequence information is 
becoming available in unprecedented amounts. The absence of a direct functional 
correlation between gene transcripts and their corresponding proteins, however, 
represents a significant roadblock for improving the efficiency of biological 
discoveries. The success of proteomics depends on the ability to identify and 
analyze protein products in a cell or tissue and, this is reliant on the 
application of several key technologies. Proteomics is in its exponential 
growth phase. Two-dimensional electrophoresis complemented with mass 
spectrometry provides a global view of the state of the proteins from the 
sample. Proteins identification is a requirement to understand their functional 
diversity. Subtle difference in protein structure and function can contribute 
to complexity and diversity of life. This review focuses on the progress and 
the applications of proteomics science with special reference to integration of 
the evolving technologies involved to address biological questions. 
</blockquote> 
<p></p> 
<blockquote>  Keywords: Computational Biology; Electrophoresis, Gel, 
Two-Dimensional; Humans; Peptide Mapping; Protein Interaction Mapping; 
Proteomics; Spectrometry, Mass, Matrix-Assisted Laser Desorption-Ionization 
</blockquote> <br>
 [Dennis2005Hunting]  Jayne&nbsp;L Dennis and Karin&nbsp;A 
Oien. Hunting the primary: novel strategies for defining the origin of tumours.
<em>J Pathol</em>, 205(2):236-47, Jan 2005. [&nbsp;bib&nbsp;| DOI&nbsp;| http
&nbsp;|.pdf&nbsp;] 
<blockquote>  In 1995, two methods of genome-wide expression profiling were 
first described: expression microarrays and serial analysis of gene expression 
(SAGE). In the subsequent 10 years, many hundreds of papers have been published 
describing the application of these technologies to a wide spectrum of 
biological and clinical questions. Common to all of this research is a basic 
process of data gathering and analysis. The techniques and statistical and 
bio-informatic tools involved in this process are reviewed. The processes of 
class discovery (using clustering and self-organizing maps), class prediction 
(weighted voting, k nearest neighbour, support vector machines, and artificial 
neural networks), target identification (fold change, discriminant analysis, 
and principal component analysis), and target validation (RT-PCR and tissue 
microarrays) are described. Finally, the diagnostic problem of adenocarcinomas 
that present as metastases of unknown origin is reviewed, and it is 
demonstrated how integration of expression profiling techniques promises to 
throw new light on this important clinical challenge. </blockquote> 
<p></p> 
<blockquote>  Keywords: Adenocarcinoma, Algorithms, Artificial Intelligence, 
Automated, Cell Nucleus, Comparative Study, Computational Biology, Computer 
Graphics, Computer-Assisted, Cytoplasm, Databases, Gene Expression Profiling, 
Genetic Vectors, Humans, Image Enhancement, Image Interpretation, Imaging, 
Information Storage and Retrieval, Internet, Mitochondria, Models, Movement, 
Neoplasms, Non-U.S. Gov't, Numerical Analysis, Oligonucleotide Array Sequence 
Analysis, Pattern Recognition, Peptides, Protein, Proteins, Proteomics, 
Reproducibility of Results, Research Support, Sensitivity and Specificity, 
Signal Processing, Software, Subtraction Technique, Theoretical, 
Three-Dimensional, Unknown Primary, Video Recording, 15641019 </blockquote> <br>
 [Cole2005Comparing]  Jason&nbsp;C Cole, Christopher&nbsp;W Murray, 
J.&nbsp;Willem&nbsp;M Nissink, Richard&nbsp;D Taylor, and Robin Taylor. 
Comparing protein-ligand docking programs is difficult.<em>Proteins</em>, 
60(3):325-332, Aug 2005. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;] 
<blockquote>  There is currently great interest in comparing protein-ligand 
docking programs. A review of recent comparisons shows that it is difficult to 
draw conclusions of general applicability. Statistical hypothesis testing is 
required to ensure that differences in pose-prediction success rates and 
enrichment rates are significant. Numerical measures such as root-mean-square 
deviation need careful interpretation and may profitably be supplemented by 
interaction-based measures and visual inspection of dockings. Test sets must be 
of appropriate diversity and of good experimental reliability. The effects of 
crystal-packing interactions may be important. The method used for generating 
starting ligand geometries and positions may have an appreciable effect on 
docking results. For fair comparison, programs must be given search problems of 
equal complexity (e.g. binding-site regions of the same size) and approximately 
equal time in which to solve them. Comparisons based on rescoring require local 
optimization of the ligand in the space of the new objective function. 
Re-implementations of published scoring functions may give significantly 
different results from the originals. Ostensibly minor details in methodology 
may have a profound influence on headline success rates. </blockquote> 
<p></p> 
<blockquote>  Keywords: Algorithms; Artificial Intelligence; Binding Sites; 
Computational Biology, methods; Computer Simulation; Crystallization; 
Crystallography, X-Ray; Databases, Protein; Ligands; Models, Molecular; 
Molecular Structure; Programming Languages; Protein Binding; Proteins, 
chemistry; Proteomics, methods; Reproducibility of Results; Software 
</blockquote> <br>
 [Bui2005Automated]  Huynh-Hoa Bui, John Sidney, Bjoern 
Peters, Muthuraman Sathiamurthy, Asabe Sinichi, Kelly-Anne Purton, 
Bianca&nbsp;R Moth&eacute;, Francis&nbsp;V Chisari, David&nbsp;I Watkins, and 
Alessandro Sette. Automated generation and evaluation of specific mhc binding 
predictive tools: Arb matrix applications.<em>Immunogenetics</em>, 
57(5):304-314, Jun 2005. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;] 
<blockquote>  Prediction of which peptides can bind major histocompatibility 
complex (MHC) molecules is commonly used to assist in the identification of T 
cell epitopes. However, because of the large numbers of different MHC molecules 
of interest, each associated with different predictive tools, tool generation 
and evaluation can be a very resource intensive task. A methodology commonly 
used to predict MHC binding affinity is the matrix or linear coefficients 
method. Herein, we described Average Relative Binding (ARB) matrix methods that 
directly predict IC(50) values allowing combination of searches involving 
different peptide sizes and alleles into a single global prediction. A computer 
program was developed to automate the generation and evaluation of ARB 
predictive tools. Using an in-house MHC binding database, we generated a total 
of 85 and 13 MHC class I and class II matrices, respectively. Results from the 
automated evaluation of tool efficiency are presented. We anticipate that this 
automation framework will be generally applicable to the generation and 
evaluation of large numbers of MHC predictive methods and tools, and will be of 
value to centralize and rationalize the process of evaluation of MHC 
predictions. MHC binding predictions based on ARB matrices were made available 
at http://epitope.liai.org:8080/matrix web server. </blockquote> 
<p></p> 
<blockquote>  Keywords: Animals; Binding Sites; Computer Simulation; 
Databases, Protein; Epitopes; Histocompatibility Antigens; Humans; Major 
Histocompatibility Complex; Models, Biological; Protein Binding </blockquote> 
<br>  [Bagos2005Evaluation]  Pantelis&nbsp;G Bagos, Theodore&nbsp;D 
Liakopoulos, and Stavros&nbsp;J Hamodrakas. Evaluation of methods for 
predicting the topology of beta-barrel outer membrane proteins and a consensus 
prediction method.<em>BMC Bioinformatics</em>, 6(1):7, Jan 2005. [&nbsp;bib
&nbsp;|DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  BACKGROUND: Prediction of the transmembrane strands and topology 
of beta-barrel outer membrane proteins is of interest in current bioinformatics 
research. Several methods have been applied so far for this task, utilizing 
different algorithmic techniques and a number of freely available predictors 
exist. The methods can be grossly divided to those based on Hidden Markov 
Models (HMMs), on Neural Networks (NNs) and on Support Vector Machines (SVMs). 
In this work, we compare the different available methods for topology 
prediction of beta-barrel outer membrane proteins. We evaluate their 
performance on a non-redundant dataset of 20 beta-barrel outer membrane 
proteins of gram-negative bacteria, with structures known at atomic resolution. 
Also, we describe, for the first time, an effective way to combine the 
individual predictors, at will, to a single consensus prediction method. 
RESULTS: We assess the statistical significance of the performance of each 
prediction scheme and conclude that Hidden Markov Model based methods, 
HMM-B2TMR, ProfTMB and PRED-TMBB, are currently the best predictors, according 
to either the per-residue accuracy, the segments overlap measure (SOV) or the 
total number of proteins with correctly predicted topologies in the test set. 
Furthermore, we show that the available predictors perform better when only 
transmembrane beta-barrel domains are used for prediction, rather than the 
precursor full-length sequences, even though the HMM-based predictors are not 
influenced significantly. The consensus prediction method performs 
significantly better than each individual available predictor, since it 
increases the accuracy up to 4% regarding SOV and up to 15% in correctly 
predicted topologies. CONCLUSIONS: The consensus prediction method described in 
this work, optimizes the predicted topology with a dynamic programming 
algorithm and is implemented in a web-based application freely available to 
non-commercial users at http://bioinformatics.biol.uoa.gr/ConBBPRED. 
</blockquote> 
<p></p> 
<blockquote>  Keywords: Algorithms, Cell Nucleus, Cytoplasm, Databases, 
Genetic Vectors, Humans, Internet, Mitochondria, Models, Non-U.S. Gov't, 
Peptides, Protein, Proteins, Proteomics, Reproducibility of Results, Research 
Support, Software, Theoretical, 15647112 </blockquote> <br>
 [
Bagga2005Quantitative]  Harmohina Bagga, David&nbsp;S Greenfield, and 
William&nbsp;J Feuer. Quantitative assessment of atypical birefringence images 
using scanning laser polarimetry with variable corneal compensation.<em>Am J 
Ophthalmol</em>, 139(3):437-46, Mar 2005. [&nbsp;bib&nbsp;| DOI&nbsp;| http
&nbsp;|.pdf&nbsp;] 
<blockquote>  PURPOSE: To define the clinical characteristics of atypical 
birefringence images and to describe a quantitative method for their 
identification. DESIGN: Prospective, comparative, clinical observational study. 
METHODS: Normal and glaucomatous eyes underwent complete examination, standard 
automated perimetry, scanning laser polarimetry with variable corneal 
compensation (GDx-VCC), and optical coherence tomography (OCT) of the macula, 
peripapillary retinal nerve fiber layer (RNFL), and optic disk. Eyes were 
classified into two groups: normal birefringence pattern (NBP) and atypical 
birefringence pattern (ABP). Clinical, functional, and structural 
characteristics were assessed separately. A multiple logistic regression model 
was used to predict eyes with ABP on the basis of a quantitative scan score 
generated by a support vector machine (SVM) with GDx-VCC. RESULTS: Sixty-five 
eyes of 65 patients were enrolled. ABP images were observed in 5 of 20 (25%) 
normal eyes and 23 of 45 (51%) glaucomatous eyes. Compared with eyes with NBP, 
glaucomatous eyes with ABP demonstrated significantly lower SVM scores (P &lt; 
.0001, &lt; 0.0001, 0.008, 0.03, and 0.03, respectively) and greater temporal, 
mean, inferior, and nasal RNFL thickness using GDx-VCC; and a weaker 
correlation with OCT generated RNFL thickness (R(2) = .75 vs .27). ABP images 
were significantly correlated with older age (R(2) = .16, P = .001). The SVM 
score was the only significant (P &lt; .0001) predictor of ABP images and 
provided high discriminating power between eyes with NBP and ABP (area under 
the receiver operator characteristic curve = 0.98). CONCLUSIONS: ABP images 
exist in a subset of normal and glaucomatous eyes, are associated with older 
patient age, and produce an artifactual increase in RNFL thickness using 
GDx-VCC. The SVM score is highly predictive of ABP images. </blockquote> 
<p></p> 
<blockquote>  Keywords: 80 and over, Adult, Aged, Algorithms, Amino Acids, 
Animals, Area Under Curve, Artifacts, Automated, Birefringence, Brain 
Chemistry, Brain Neoplasms, Comparative Study, Computer-Assisted, Cornea, 
Cross-Sectional Studies, Decision Trees, Diagnosis, Diagnostic Imaging, 
Diagnostic Techniques, Discriminant Analysis, Evolution, Face, Female, Genetic, 
Glaucoma, Humans, Intraocular Pressure, Lasers, Least-Squares Analysis, 
Magnetic Resonance Imaging, Magnetic Resonance Spectroscopy, Male, Middle Aged, 
Models, Molecular, Nerve Fibers, Non-U.S. Gov't, Numerical Analysis, 
Ophthalmological, Optic Nerve Diseases, Optical Coherence, P.H.S., Pattern 
Recognition, Photic Stimulation, Prospective Studies, Protein, ROC Curve, 
Regression Analysis, Research Support, Retinal Ganglion Cells, Sensitivity and 
Specificity, Sequence Analysis, Statistics, Tomography, U.S. Gov't, Visual 
Fields, beta-Lactamases, 15767051 </blockquote> <br>
 [Asefa2005Support]  
Tirusew Asefa, Mariush Kemblowski, Gilberto Urroz, and Mac McKee. Support 
vector machines (SVMs) for monitoring network design.<em>Ground Water</em>, 
43(3):413-22, 2005. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  In this paper we present a hydrologic application of a new 
statistical learning methodology called support vector machines (SVMs). SVMs 
are based on minimization of a bound on the generalized error (risk) model, 
rather than just the mean square error over a training set. Due to Mercer's 
conditions on the kernels, the corresponding optimization problems are convex 
and hence have no local minima. In this paper, SVMs are illustratively used to 
reproduce the behavior of Monte Carlo-based flow and transport models that are 
in turn used in the design of a ground water contamination detection monitoring 
system. The traditional approach, which is based on solving transient transport 
equations for each new configuration of a conductivity field, is too time 
consuming in practical applications. Thus, there is a need to capture the 
behavior of the transport phenomenon in random media in a relatively simple 
manner. The objective of the exercise is to maximize the probability of 
detecting contaminants that exceed some regulatory standard before they reach a 
compliance boundary, while minimizing cost (i.e., number of monitoring wells). 
Application of the method at a generic site showed a rather promising 
performance, which leads us to believe that SVMs could be successfully employed 
in other areas of hydrology. The SVM was trained using 510 monitoring 
configuration samples generated from 200 Monte Carlo flow and transport 
realizations. The best configurations of well networks selected by the SVM were 
identical with the ones obtained from the physical model, but the reliabilities 
provided by the respective networks differ slightly. </blockquote> 
<p></p> 
<blockquote>  Keywords: Adult, Aged, Aging, Algorithms, Apoptosis, Artificial 
Intelligence, Automated, Computer-Assisted, Female, Foot, Gait, Gene Expression 
Profiling, Humans, Image Interpretation, Male, Neoplasms, Non-U.S. Gov't, 
Oligonucleotide Array Sequence Analysis, Pattern Recognition, Polymerase Chain 
Reaction, Proteins, Reproducibility of Results, Research Support, Sensitivity 
and Specificity, Subcellular Fractions, Unknown Primary, 15882333 </blockquote> 
<br>  [Yu2005Ovarian]  J.&nbsp;S. Yu, S.&nbsp;Ongarello, R.&nbsp;Fiedler, 
X.&nbsp;W. Chen, G.&nbsp;Toffolo, C.&nbsp;Cobelli, and Z.&nbsp;Trajanoski. 
Ovarian cancer identification based on dimensionality reduction for 
high-throughput mass spectrometry data.<em>Bioinformatics</em>, 21(10):2200-9, 
May 2005. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  MOTIVATION: High-throughput and high-resolution mass 
spectrometry instruments are increasingly used for disease classification and 
therapeutic guidance. However, the analysis of immense amount of data poses 
considerable challenges. We have therefore developed a novel method for 
dimensionality reduction and tested on a published ovarian high-resolution 
SELDI-TOF dataset. RESULTS: We have developed a four-step strategy for data 
preprocessing based on: (1) binning, (2) Kolmogorov-Smirnov test, (3) 
restriction of coefficient of variation and (4) wavelet analysis. Subsequently, 
support vector machines were used for classification. The developed method 
achieves an average sensitivity of 97.38% (sd = 0.0125) and an average 
specificity of 93.30% (sd = 0.0174) in 1000 independent k-fold 
cross-validations, where k = 2, ..., 10. AVAILABILITY: The software is 
available for academic and non-commercial institutions. </blockquote> 
<p></p> 
<blockquote>  Keywords: biosvm proteomics </blockquote> <br>
 [Begg2005Support]
 Rezaul&nbsp;K Begg, Marimuthu Palaniswami, and Brendan Owen. Support vector 
machines for automated gait classification.<em>IEEE Trans Biomed Eng</em>, 
52(5):828-38, May 2005. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;| .pdf&nbsp;] 
<blockquote>  Ageing influences gait patterns causing constant threats to 
control of locomotor balance. Automated recognition of gait changes has many 
advantages including, early identification of at-risk gait and monitoring the 
progress of treatment outcomes. In this paper, we apply an artificial 
intelligence technique [support vector machines (SVM)] for the automatic 
recognition of young-old gait types from their respective gait-patterns. 
Minimum foot clearance (MFC) data of 30 young and 28 elderly participants were 
analyzed using a PEAK-2D motion analysis system during a 20-min continuous walk 
on a treadmill at self-selected walking speed. Gait features extracted from 
individual MFC histogram-plot and Poincar&eacute;-plot images were used to 
train the SVM. Cross-validation test results indicate that the generalization 
performance of the SVM was on average 83.3% (+/-2.9) to recognize young and 
elderly gait patterns, compared to a neural network's accuracy of 75.0+/-5.0%. 
A &quot;hill-climbing&quot; feature selection algorithm demonstrated that a 
small subset (3-5) of gait features extracted from MFC plots could 
differentiate the gait patterns with 90% accuracy. Performance of the gait 
classifier was evaluated using areas under the receiver operating 
characteristic plots. Improved performance of the classifier was evident when 
trained with reduced number of selected good features and with radial basis 
function kernel. These results suggest that SVMs can function as an efficient 
gait classifier for recognition of young and elderly gait patterns, and has the 
potential for wider applications in gait identification for falls-risk 
minimization in the elderly. </blockquote> 
<p></p> 
<blockquote>  Keywords: Adult, Aged, Aging, Algorithms, Apoptosis, Artificial 
Intelligence, Automated, Computer-Assisted, Female, Foot, Gait, Gene Expression 
Profiling, Humans, Image Interpretation, Male, Neoplasms, Non-U.S. Gov't, 
Oligonucleotide Array Sequence Analysis, Pattern Recognition, Polymerase Chain 
Reaction, Proteins, Reproducibility of Results, Research Support, Sensitivity 
and Specificity, Subcellular Fractions, Unknown Primary, 15887532 </blockquote> 
<br>  [Surgand2006chemogenomic]  Jean-Sebastien Surgand, Jordi Rodrigo, Esther 
Kellenberger, and Didier Rognan. A chemogenomic analysis of the transmembrane 
binding cavity of human g-protein-coupled receptors.<em>Proteins</em>, 
62(2):509-538, Feb 2006. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;] 
<blockquote>  The amino acid sequences of 369 human nonolfactory 
G-protein-coupled receptors (GPCRs) have been aligned at the seven 
transmembrane domain (TM) and used to extract the nature of 30 critical 
residues supposed-from the X-ray structure of bovine rhodopsin bound to 
retinal-to line the TM binding cavity of ground-state receptors. Interestingly, 
the clustering of human GPCRs from these 30 residues mirrors the recently 
described phylogenetic tree of full-sequence human GPCRs (Fredriksson et al., 
Mol Pharmacol 2003;63:1256-1272) with few exceptions. A TM cavity could be 
found for all investigated GPCRs with physicochemical properties matching that 
of their cognate ligands. The current approach allows a very fast comparison of 
most human GPCRs from the focused perspective of the predicted TM cavity and 
permits to easily detect key residues that drive ligand selectivity or 
promiscuity. </blockquote> 
<p></p> 
<blockquote>  Keywords: Amino Acid Sequence; Binding Sites; Genomics; Humans; 
Ligands; Models, Molecular; Phylogeny; Receptors, G-Protein-Coupled 
</blockquote> <br>
 [Salomon2006Predicting]  J.&nbsp;Salomon and D.&nbsp;R. 
Flower. Predicting Class II MHC-Peptide binding: a kernel based approach using 
similarity scores.<em>BMC Bioinformatics</em>, 7:501, 2006. [&nbsp;bib&nbsp;| 
DOI&nbsp;| http&nbsp;] 
<blockquote>  BACKGROUND: Modelling the interaction between potentially 
antigenic peptides and Major Histocompatibility Complex (MHC) molecules is a 
key step in identifying potential T-cell epitopes. For Class II MHC alleles, 
the binding groove is open at both ends, causing ambiguity in the positional 
alignment between the groove and peptide, as well as creating uncertainty as to 
what parts of the peptide interact with the MHC. Moreover, the antigenic 
peptides have variable lengths, making naive modelling methods difficult to 
apply. This paper introduces a kernel method that can handle variable length 
peptides effectively by quantifying similarities between peptide sequences and 
integrating these into the kernel. RESULTS: The kernel approach presented here 
shows increased prediction accuracy with a significantly higher number of true 
positives and negatives on multiple MHC class II alleles, when testing data 
sets from MHCPEP 1, MCHBN 2, and MHCBench 3. Evaluation by cross validation, 
when segregating binders and non-binders, produced an average of 0.824 AROC for 
the MHCBench data sets (up from 0.756), and an average of 0.96 AROC for 
multiple alleles of the MHCPEP database. CONCLUSION: The method improves 
performance over existing state-of-the-art methods of MHC class II peptide 
binding predictions by using a custom, knowledge-based representation of 
peptides. Similarity scores, in contrast to a fixed-length, pocket-specific 
representation of amino acids, provide a flexible and powerful way of modelling 
MHC binding, and can easily be applied to other dynamic sequence problems. 
</blockquote> 
<p></p> 
<blockquote>  Keywords: Amino Acid, Binding Sites, Computational Biology, 
Databases, Epitope Mapping, Genetic, HLA-A Antigens, HLA-DR Antigens, 
Histocompatibility Antigens Class II, Humans, Peptides, Protein, Protein 
Binding, Protein Conformation, ROC Curve, Reproducibility of Results, Sequence 
Alignment, Sequence Analysis, Sequence Homology, 17105666 </blockquote> <br>
 [
Leach2006Prediction]  Andrew&nbsp;R Leach, Brian&nbsp;K Shoichet, and 
Catherine&nbsp;E Peishoff. Prediction of protein-ligand interactions. docking 
and scoring: successes and gaps.<em>J. Med. Chem.</em>, 49(20):5851-5855, Oct 
2006. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;] 
<blockquote>  Keywords: Binding Sites; Drug Design; Ligands; Models, 
Molecular; Protein Binding; Proteins, chemistry; Quantitative 
Structure-Activity Relationship </blockquote> <br>
 [Kubinyi2006Chemogenomics]  
H.&nbsp;Kubinyi. Chemogenomics in drug discovery.<em>Ernst Schering Res Found 
Workshop</em>, 58:1-19, 2006. [&nbsp;bib&nbsp;] 
<blockquote>  Chemogenomics is a new strategy in drug discovery which, in 
principle, searches for all molecules that are capable of interacting with any 
biological target. Because of the almost infinite number of drug-like organic 
molecules, this is an impossible task. Therefore chemogenomics has been defined 
as the investigation of classes of compounds (libraries) against families of 
functionally related proteins. In this definition, chemogenomics deals with the 
systematic analysis of chemical-biological interactions. Congeneric series of 
chemical analogs are probes to investigate their action on specific target 
classes, e.g., GPCRs, kinases, phosphodiesterases, ion channels, serine 
proteases, and others. Whereas such a strategy developed in pharmaceutical 
industry almost 20 years ago, it is now more systematically applied in the 
search for target- and subtype-specific ligands. The term &quot;privileged 
structures&quot; has been defined for scaffolds, such as the benzodiazepines, 
which very often produce biologically active analogs in a target family, in 
this case in the class of G-protein-coupled receptors. The SOSA approach is a 
strategy to modify the selectivity of biologically active compounds, generating 
new drug candidates from the side activities of therapeutically used drugs. 
</blockquote> 
<p></p> 
<blockquote>  Keywords: Animals; Chemistry, Pharmaceutical; Combinatorial 
Chemistry Techniques; Drug Design; Drug Industry; Genomics; Humans; Models, 
Chemical; Molecular Structure; Mutation; Pharmacogenetics; Protein Binding 
</blockquote> <br>
 [Huang2006Ligsite]  Bingding Huang and Michael Schroeder. 
Ligsitecsc: predicting ligand binding sites using the connolly surface and 
degree of conservation.<em>BMC Struct Biol</em>, 6:19, 2006. [&nbsp;bib&nbsp;| 
DOI&nbsp;| http&nbsp;] 
<blockquote>  BACKGROUND: Identifying pockets on protein surfaces is of great 
importance for many structure-based drug design applications and protein-ligand 
docking algorithms. Over the last ten years, many geometric methods for the 
prediction of ligand-binding sites have been developed. RESULTS: We present 
LIGSITEcsc, an extension and implementation of the LIGSITE algorithm. 
LIGSITEcsc is based on the notion of surface-solvent-surface events and the 
degree of conservation of the involved surface residues. We compare our 
algorithm to four other approaches, LIGSITE, CAST, PASS, and SURFNET, and 
evaluate all on a dataset of 48 unbound/bound structures and 210 
bound-structures. LIGSITEcsc performs slightly better than the other tools and 
achieves a success rate of 71% and 75%, respectively. CONCLUSION: The use of 
the Connolly surface leads to slight improvements, the prediction re-ranking by 
conservation to significant improvements of the binding site predictions. A web 
server for LIGSITEcsc and its source code is available at 
scoppi.biotec.tu-dresden.de/pocket </blockquote> 
<p></p> 
<blockquote>  Keywords: Algorithms; Binding Sites; Databases, Protein; 
Ligands; Models, Molecular; Proteins, chemistry </blockquote> <br>
 [
Theres2006Structural]  Theres Fagerberg, Jean-Charles Cerottini, and Olivier 
Michielin. Structural prediction of peptides bound to MHC class I.<em>J. Mol. 
Biol.</em>, 356(2):521-546, Feb 2006. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;] 
<blockquote>  An ab initio structure prediction approach adapted to the 
peptide-major histocompatibility complex (MHC) class I system is presented. 
Based on structure comparisons of a large set of peptide-MHC class I complexes, 
a molecular dynamics protocol is proposed using simulated annealing (SA) cycles 
to sample the conformational space of the peptide in its fixed MHC environment. 
A set of 14 peptide-human leukocyte antigen (HLA) A0201 and 27 peptide-non-HLA 
A0201 complexes for which X-ray structures are available is used to test the 
accuracy of the prediction method. For each complex, 1000 peptide conformers 
are obtained from the SA sampling. A graph theory clustering algorithm based on 
heavy atom root-mean-square deviation (RMSD) values is applied to the sampled 
conformers. The clusters are ranked using cluster size, mean effective or 
conformational free energies, with solvation free energies computed using 
Generalized Born MV 2 (GB-MV2) and Poisson-Boltzmann (PB) continuum models. The 
final conformation is chosen as the center of the best-ranked cluster. With 
conformational free energies, the overall prediction success is 83% using a 
1.00 Angstroms crystal RMSD criterion for main-chain atoms, and 76% using a 
1.50 Angstroms RMSD criterion for heavy atoms. The prediction success is even 
higher for the set of 14 peptide-HLA A0201 complexes: 100% of the peptides have 
main-chain RMSD values &lt; or =1.00 Angstroms and 93% of the peptides have 
heavy atom RMSD values &lt; or =1.50 Angstroms. This structure prediction 
method can be applied to complexes of natural or modified antigenic peptides in 
their MHC environment with the aim to perform rational structure-based 
optimizations of tumor vaccines. </blockquote> 
<p></p> 
<blockquote>  Keywords: , Algorithms, Amino Acid Sequence, Antibodies, 
Artificial Intelligence, Automated, Binding Sites, Chemical, Computer 
Simulation, Databases, Epitope Mapping, Genes, HLA-A Antigens, HLA-DQ Antigens, 
Histocompatibility Antigens Class I, Humans, Immunoassay, Immunological, MHC 
Class I, Models, Molecular, Molecular Sequence Data, Pattern Recognition, 
Peptides, Protein, Protein Binding, Protein Conformation, Protein Interaction 
Mapping, Protein Structure, Sequence Alignment, Sequence Analysis, Software, 
Tertiary, Water, 16368108 </blockquote> <br>
 [Bui2006Structural]  H.-H. Bui, 
A.&nbsp;J. Schiewe, H.&nbsp;von Grafenstein, and I.&nbsp;S. Haworth. Structural 
prediction of peptides binding to MHC class I molecules.<em>Proteins</em>, 
63(1):43-52, Apr 2006. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;] 
<blockquote>  Peptide binding to class I major histocompatibility complex 
(MHCI) molecules is a key step in the immune response and the structural 
details of this interaction are of importance in the design of peptide 
vaccines. Algorithms based on primary sequence have had success in predicting 
potential antigenic peptides for MHCI, but such algorithms have limited 
accuracy and provide no structural information. Here, we present an algorithm, 
PePSSI (peptide-MHC prediction of structure through solvated interfaces), for 
the prediction of peptide structure when bound to the MHCI molecule, HLA-A2. 
The algorithm combines sampling of peptide backbone conformations and flexible 
movement of MHC side chains and is unique among other prediction algorithms in 
its incorporation of explicit water molecules at the peptide-MHC interface. In 
an initial test of the algorithm, PePSSI was used to predict the conformation 
of eight peptides bound to HLA-A2, for which X-ray data are available. 
Comparison of the predicted and X-ray conformations of these peptides gave RMSD 
values between 1.301 and 2.475 A. Binding conformations of 266 peptides with 
known binding affinities for HLA-A2 were then predicted using PePSSI. 
Structural analyses of these peptide-HLA-A2 conformations showed that peptide 
binding affinity is positively correlated with the number of peptide-MHC 
contacts and negatively correlated with the number of interfacial water 
molecules. These results are consistent with the relatively hydrophobic binding 
nature of the HLA-A2 peptide binding interface. In summary, PePSSI is capable 
of rapid and accurate prediction of peptide-MHC binding conformations, which 
may in turn allow estimation of MHCI-peptide binding affinity. </blockquote> 
<p></p> 
<blockquote>  Keywords: Algorithms, Amino Acid Sequence, Antigens, Artificial 
Intelligence, Automated, Binding Sites, Chemical, Computational Biology, 
Computer Simulation, Crystallog, Crystallography, Electrostatics, Genes, 
Genetic, HLA Antigens, Histocompatibility Antigens Class I, Humans, Hydrogen 
Bonding, Ligands, MHC Class I, Major Histocompatibility Complex, Models, 
Molecular, Molecular Conformation, Molecular Sequence Data, Pattern 
Recognition, Peptides, Protein, Protein Binding, Protein Conformation, 
Proteomics, Quantitative Structure-Activity Relationship, Sequence Alignment, 
Sequence Analysis, Software, Structural Homology, Structure-Activity 
Relationship, Thermodynamics, Water, X-Ray, X-Rays, raphy, 16447245 
</blockquote> <br>
 [Bhavani2006Substructure-based]  S.&nbsp;Bhavani, 
A.&nbsp;Nagargadde, A.&nbsp;Thawani, V.&nbsp;Sridhar, and N.&nbsp;Chandra. 
Substructure-based support vector machine classifiers for prediction of adverse 
effects in diverse classes of drugs.<em>J. Chem. Inf. Model.</em>, 
46(6):2478-2486, 2006. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;] 
<blockquote>  Unforeseen adverse effects exhibited by drugs contribute heavily 
to late-phase failure and even withdrawal of marketed drugs. Torsade de pointes 
(TdP) is one such important adverse effect, which causes cardiac arrhythmia 
and, in some cases, sudden death, making it crucial for potential drugs to be 
screened for torsadogenicity. The need to tap the power of computational 
approaches for the prediction of adverse effects such as TdP is increasingly 
becoming evident. The availability of screening data including those in 
organized databases greatly facilitates exploration of newer computational 
approaches. In this paper, we report the development of a prediction method 
based on a support machine vector algorithm. The method uses a combination of 
descriptors, encoding both the type of toxicophore as well as the position of 
the toxicophore in the drug molecule, thus considering both the pharmacophore 
and the three-dimensional shape information of the molecule. For delineating 
toxicophores, a novel pattern-recognition method that utilizes substructures 
within a molecule has been developed. The results obtained using the hybrid 
approach have been compared with those available in the literature for the same 
data set. An improvement in prediction accuracy is clearly seen, with the 
accuracy reaching up to 97% in predicting compounds that can cause TdP and 90% 
for predicting compounds that do not cause TdP. The generic nature of the 
method has been demonstrated with four data sets available for carcinogenicity, 
where prediction accuracies were significantly higher, with a best receiver 
operating characteristics (ROC) value of 0.81 as against a best ROC value of 
0.7 reported in the literature for the same data set. Thus, the method holds 
promise for wide applicability in toxicity prediction. </blockquote> 
<p></p> 
<blockquote>  Keywords: Algorithms; Carcinogens; Chemistry, Pharmaceutical; 
Computational Biology; Drug Evaluation, Preclinical; Drug Industry; Humans; 
Models, Chemical; Models, Statistical; Neural Networks (Computer); Pattern 
Recognition, Automated; ROC Curve; Sequence Analysis, Protein; Software; 
Torsades de Pointes </blockquote> <br>
 [Glaser2006Method]  F.&nbsp;Glaser, 
R.&nbsp;J. Morris, R.&nbsp;J. Najmanovich, R.&nbsp;A. Laskowski, and J.&nbsp;M. 
Thornton. A method for localizing ligand binding pockets in protein structures.
<em>Proteins</em>, 62(2):479-488, February 2006. [&nbsp;bib&nbsp;| DOI&nbsp;| 
http&nbsp;] 
<blockquote>  The accurate identification of ligand binding sites in protein 
structures can be valuable in determining protein function. Once the binding 
site is known, it becomes easier to perform in silico and experimental 
procedures that may allow the ligand type and the protein function to be 
determined. For example, binding pocket shape analysis relies heavily on the 
correct localization of the ligand binding site. We have developed 
SURFNET-ConSurf, a modular, two-stage method for identifying the location and 
shape of potential ligand binding pockets in protein structures. In the first 
stage, the SURFNET program identifies clefts in the protein surface that are 
potential binding sites. In the second stage, these clefts are trimmed in size 
by cutting away regions distant from highly conserved residues, as defined by 
the ConSurf-HSSP database. The largest clefts that remain tend to be those 
where ligands bind. To test the approach, we analyzed a nonredundant set of 244 
protein structures from the PDB and found that SURFNET-ConSurf identifies a 
ligand binding pocket in 75% of them. The trimming procedure reduces the 
original cleft volumes by 30% on average, while still encompassing an average 
87% of the ligand volume. From the analysis of the results we conclude that for 
those cases in which the ligands are found in large, highly conserved clefts, 
the combined SURFNET-ConSurf method gives pockets that are a better match to 
the ligand shape and location. We also show that this approach works better for 
enzymes than for nonenzyme proteins. </blockquote> 
<p></p> 
<blockquote>  Keywords: ligand-volume, protein-ligand, surface </blockquote> 
<br>  [Tung2007POPI:]  Chun-Wei Tung and Shinn-Ying Ho. Popi: predicting 
immunogenicity of mhc class i binding peptides by mining informative 
physicochemical properties.<em>Bioinformatics</em>, 23(8):942-949, Apr 2007. 
[&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;] 
<blockquote>  MOTIVATION: Both modeling of antigen-processing pathway 
including major histocompatibility complex (MHC) binding and immunogenicity 
prediction of those MHC-binding peptides are essential to develop a 
computer-aided system of peptide-based vaccine design that is one goal of 
immunoinformatics. Numerous studies have dealt with modeling the immunogenic 
pathway but not the intractable problem of immunogenicity prediction due to 
complex effects of many intrinsic and extrinsic factors. Moderate affinity of 
the MHC-peptide complex is essential to induce immune responses, but the 
relationship between the affinity and peptide immunogenicity is too weak to use 
for predicting immunogenicity. This study focuses on mining informative 
physicochemical properties from known experimental immunogenicity data to 
understand immune responses and predict immunogenicity of MHC-binding peptides 
accurately. RESULTS: This study proposes a computational method to mine a 
feature set of informative physicochemical properties from MHC class I binding 
peptides to design a support vector machine (SVM) based system (named POPI) for 
the prediction of peptide immunogenicity. High performance of POPI arises 
mainly from an inheritable bi-objective genetic algorithm, which aims to 
automatically determine the best number m out of 531 physicochemical 
properties, identify these m properties and tune SVM parameters simultaneously. 
The dataset consisting of 428 human MHC class I binding peptides belonging to 
four classes of immunogenicity was established from MHCPEP, a database of 
MHC-binding peptides (Brusic et al., 1998). POPI, utilizing the m = 23 selected 
properties, performs well with the accuracy of 64.72% using leave-one-out 
cross-validation, compared with two sequence alignment-based prediction methods 
ALIGN (54.91%) and PSI-BLAST (53.23%). POPI is the first computational system 
for prediction of peptide immunogenicity based on physicochemical properties. 
AVAILABILITY: A web server for prediction of peptide immunogenicity (POPI) and 
the used dataset of MHC class I binding peptides (PEPMHCI) are available at 
http://iclab.life.nctu.edu.tw/POPI </blockquote> 
<p></p> 
<blockquote>  Keywords: Algorithms; Artificial Intelligence; Binding Sites; 
Epitope Mapping; Histocompatibility Antigens Class I; Oligopeptides; Pattern 
Recognition, Automated; Protein Binding; Software; Structure-Activity 
Relationship </blockquote> <br>
 [Kroemer2007Structure]  Romano&nbsp;T Kroemer. 
Structure-based drug design: docking and scoring.<em>Curr. Protein Pept. Sci.
</em>, 8(4):312-328, Aug 2007. [&nbsp;bib&nbsp;] 
<blockquote>  This review gives an introduction into ligand - receptor docking 
and illustrates the basic underlying concepts. An overview of different 
approaches and algorithms is provided. Although the application of docking and 
scoring has led to some remarkable successes, there are still some major 
challenges ahead, which are outlined here as well. Approaches to address some 
of these challenges and the latest developments in the area are presented. Some 
aspects of the assessment of docking program performance are discussed. A 
number of successful applications of structure-based virtual screening are 
described. </blockquote> 
<p></p> 
<blockquote>  Keywords: Algorithms; Artificial Intelligence; Computational 
Biology; Computer Simulation; Computer-Aided Design; Drug Design; Imaging, 
Three-Dimensional; Ligands; Models, Molecular; Protein Binding; Protein 
Conformation; Software; Structure-Activity Relationship </blockquote> <br>
 [
Kahraman2007Shape]  Abdullah Kahraman, Richard&nbsp;J Morris, Roman&nbsp;A 
Laskowski, and Janet&nbsp;M Thornton. Shape variation in protein binding 
pockets and their ligands.<em>J. Mol. Biol.</em>, 368(1):283-301, Apr 2007. 
[&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;] 
<blockquote>  A common assumption about the shape of protein binding pockets 
is that they are related to the shape of the small ligand molecules that can 
bind there. But to what extent is that assumption true? Here we use a recently 
developed shape matching method to compare the shapes of protein binding 
pockets to the shapes of their ligands. We find that pockets binding the same 
ligand show greater variation in their shapes than can be accounted for by the 
conformational variability of the ligand. This suggests that geometrical 
complementarity in general is not sufficient to drive molecular recognition. 
Nevertheless, we show when considering only shape and size that a significant 
proportion of the recognition power of a binding pocket for its ligand resides 
in its shape. Additionally, we observe a &quot;buffer zone&quot; or a region of 
free space between the ligand and protein, which results in binding pockets 
being on average three times larger than the ligand that they bind. 
</blockquote> 
<p></p> 
<blockquote>  Keywords: Binding Sites; Computer Simulation; Ligands; Models, 
Molecular; Models, Statistical; Protein Binding; Protein Conformation; Protein 
Folding </blockquote> <br>
 [Dalton2007Evaluation]  James A&nbsp;R Dalton and 
Richard&nbsp;M Jackson. An evaluation of automated homology modelling methods 
at low target template sequence similarity.<em>Bioinformatics</em>, 
23(15):1901-1908, Aug 2007. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;] 
<blockquote>  MOTIVATION: There are two main areas of difficulty in homology 
modelling that are particularly important when sequence identity between target 
and template falls below 50%: sequence alignment and loop building. These 
problems become magnified with automatic modelling processes, as there is no 
human input to correct mistakes. As such we have benchmarked several 
stand-alone strategies that could be implemented in a workflow for automated 
high-throughput homology modelling. These include three new sequence-structure 
alignment programs: 3D-Coffee, Staccato and SAlign, plus five homology 
modelling programs and their respective loop building methods: Builder, Nest, 
Modeller, SegMod/ENCAD and Swiss-Model. The SABmark database provided 123 
targets with at least five templates from the same SCOP family and sequence 
identities &lt;/=50%. RESULTS: When using Modeller as the common modelling 
program, 3D-Coffee outperforms Staccato and SAlign using both multiple 
templates and the best single template, and across the sequence identity range 
20-50%. The mean model RMSD generated from 3D-Coffee using multiple templates 
is 15 and 28% (or using single templates, 3 and 13%) better than those 
generated by Staccato and Salign, respectively. 3D-Coffee gives equivalent 
modelling accuracy from multiple and single templates, but Staccato and SAlign 
are more successful with single templates, their quality deteriorating as 
additional lower sequence identity templates are added. Evaluating the 
different homology modelling programs, on average Modeller performs marginally 
better in overall modelling than the others tested. However, on average Nest 
produces the best loops with an 8% improvement by mean RMSD compared to the 
loops generated by Builder. </blockquote> 
<p></p> 
<blockquote>  Keywords: Algorithms; Amino Acid Sequence; Computer Simulation; 
Models, Chemical; Models, Molecular; Molecular Sequence Data; Proteins, 
chemistry; Reproducibility of Results; Sensitivity and Specificity; Sequence 
Alignment, methods; Sequence Analysis, Protein, methods; Software; Software 
Validation </blockquote> <br>
 [Bock2007Effective]  Mary&nbsp;Ellen Bock, 
Claudio Garutti, and Conettina Guerra. Effective labeling of molecular surface 
points for cavity detection and location of putative binding sites.<em>Comput 
Syst Bioinformatics Conf</em>, 6:263-274, 2007. [&nbsp;bib&nbsp;] 
<blockquote>  We present a method for detecting and comparing cavities on 
protein surfaces that is useful for protein binding site recognition. The 
method is based on a representation of the protein structures by a collection 
of spin-images and their associated spin-image profiles. Results of the cavity 
detection procedure are presented for a large set of non-redundant proteins and 
compared with SURFNET-ConSurf. Our comparison method is used to find a surface 
region in one cavity of a protein that is geometrically similar to a surface 
region in the cavity of another protein. Such a finding would be an indication 
that the two regions likely bind to the same ligand. Our overall approach for 
cavity detection and comparison is benchmarked on several pairs of known 
complexes, obtaining a good coverage of the atoms of the binding sites. 
</blockquote> 
<p></p> 
<blockquote>  Keywords: Binding Sites; Computer Simulation; Models, Chemical; 
Models, Molecular; Protein Binding; Protein Conformation; Protein Folding; 
Proteins, chemistry/ultrastructure; Sequence Analysis, Protein, methods; 
Surface Properties </blockquote> <br>
 [Qiu2007structural]  J.&nbsp;Qiu, 
J.&nbsp;Hue, A.&nbsp;Ben-Hur, J.-P. Vert, and W.&nbsp;S. Noble. A structural 
alignment kernel for protein structures.<em>Bioinformatics</em>, 
23(9):1090-1098, May 2007. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;] 
<blockquote>  MOTIVATION: This work aims to develop computational methods to 
annotate protein structures in an automated fashion. We employ a support vector 
machine (SVM) classifier to map from a given class of structures to their 
corresponding structural (SCOP) or functional (Gene Ontology) annotation. In 
particular, we build upon recent work describing various kernels for protein 
structures, where a kernel is a similarity function that the classifier uses to 
compare pairs of structures. RESULTS: We describe a kernel that is derived in a 
straightforward fashion from an existing structural alignment program, MAMMOTH. 
We find in our benchmark experiments that this kernel significantly 
out-performs a variety of other kernels, including several previously described 
kernels. Furthermore, in both benchmarks, classifying structures using MAMMOTH 
alone does not work as well as using an SVM with the MAMMOTH kernel. 
AVAILABILITY: http://noble.gs.washington.edu/proj/3dkernel </blockquote> 
<p></p> 
<blockquote>  Keywords: Algorithms; Amino Acid Sequence; Artificial 
Intelligence; Molecular Sequence Data; Pattern Recognition, Automated; 
Proteins; Sequence Alignment; Sequence Analysis, Protein; Sequence Homology, 
Amino Acid </blockquote> <br>
 [Schalon2008Simple]  Claire Schalon, 
Jean-S&eacute;bastien Surgand, Esther Kellenberger, and Didier Rognan. A simple 
and fuzzy method to align and compare druggable ligand-binding sites.<em>
Proteins</em>, 71(4):1755-1778, Jun 2008. [&nbsp;bib&nbsp;| DOI&nbsp;| http
&nbsp;]
<blockquote>  A novel method to measure distances between druggable protein 
cavities is presented. Starting from user-defined ligand binding sites, eight 
topological and physicochemical properties are projected from cavity-lining 
protein residues to an 80 triangle-discretised sphere placed at the centre of 
the binding site, thus defining a cavity fingerprint. Representing binding site 
properties onto a discretised sphere presents many advantages: (i) a normalised 
distance between binding sites of different sizes may be easily derived by 
summing up the normalised differences between the 8 computed descriptors; (ii) 
a structural alignment of two proteins is simply done by systematically 
rotating/translating one mobile sphere around one immobile reference; (iii) a 
certain degree of fuzziness in the comparison is reached by projecting global 
amino acid properties (e.g., charge, size, functional groups count, distance to 
the site centre) independently of local rotameric/tautomeric states of 
cavity-lining residues. The method was implemented in a new program (SiteAlign) 
and tested in a number of various scenarios: measuring the distance between 376 
related active site pairs, computing the cross-similarity of members of a 
protein family, predicting the targets of ligands with various promiscuity 
levels. The proposed method is robust enough to detect local similarity among 
active sites of different sizes, to discriminate between protein subfamilies 
and to recover the known targets of promiscuous ligands by virtual screening. 
</blockquote> 
<p></p> 
<blockquote>  Keywords: Algorithms; Amino Acid Sequence; Binding Sites, drug 
effects; Drug Design; Hydrogen Bonding; Ligands; Protein Binding; Sequence 
Alignment; Structure-Activity Relationship </blockquote> <br>
 [
Moitessier2008Towards]  N.&nbsp;Moitessier, P.&nbsp;Englebienne, D.&nbsp;Lee, 
J.&nbsp;Lawandi, and C.&nbsp;R. Corbeil. Towards the development of universal, 
fast and highly accurate docking/scoring methods: a long way to go.<em>Br. J. 
Pharmacol.</em>, 153 Suppl 1:S7-26, Mar 2008. [&nbsp;bib&nbsp;| DOI&nbsp;| http
&nbsp;]
<blockquote>  Accelerating the drug discovery process requires predictive 
computational protocols capable of reducing or simplifying the synthetic and/or 
combinatorial challenge. Docking-based virtual screening methods have been 
developed and successfully applied to a number of pharmaceutical targets. In 
this review, we first present the current status of docking and scoring 
methods, with exhaustive lists of these. We next discuss reported comparative 
studies, outlining criteria for their interpretation. In the final section, we 
describe some of the remaining developments that would potentially lead to a 
universally applicable docking/scoring method. </blockquote> 
<p></p> 
<blockquote>  Keywords: Algorithms; Animals; Artificial Intelligence; Computer 
Simulation; Drug Evaluation, Preclinical, methods; Humans; Metals, chemistry; 
Models, Molecular; Molecular Conformation; Nucleic Acids, chemistry/drug 
effects; Proteins, chemistry/drug effects; Reproducibility of Results; 
Stochastic Processes </blockquote> <br>
 [Launay2008Homology]  Guillaume Launay 
and Thomas Simonson. Homology modelling of protein-protein complexes: a simple 
method and its possibilities and limitations.<em>BMC Bioinformatics</em>, 
9:427, 2008. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;] 
<blockquote>  BACKGROUND: Structure-based computational methods are needed to 
help identify and characterize protein-protein complexes and their function. 
For individual proteins, the most successful technique is homology modelling. 
We investigate a simple extension of this technique to protein-protein 
complexes. We consider a large set of complexes of known structures, involving 
pairs of single-domain proteins. The complexes are compared with each other to 
establish their sequence and structural similarities and the relation between 
the two. Compared to earlier studies, a simpler dataset, a simpler structural 
alignment procedure, and an additional energy criterion are used. Next, we 
compare the Xray structures to models obtained by threading the native sequence 
onto other, homologous complexes. An elementary requirement for a successful 
energy function is to rank the native structure above any threaded structure. 
We use the DFIREbeta energy function, whose quality and complexity are typical 
of the models used today. Finally, we compare near-native models to distinctly 
non-native models. RESULTS: If weakly stable complexes are excluded (defined by 
a binding energy cutoff), as well as a few unusual complexes, a simple homology 
principle holds: complexes that share more than 35% sequence identity share 
similar structures and interaction modes; this principle was less clearcut in 
earlier studies. The energy function was then tested for its ability to 
identify experimental structures among sets of decoys, produced by a simple 
threading procedure. On average, the experimental structure is ranked above 92% 
of the alternate structures. Thus, discrimination of the native structure is 
good but not perfect. The discrimination of near-native structures is fair. 
Typically, a single, alternate, non-native binding mode exists that has a 
native-like energy. Some of the associated failures may correspond to genuine, 
alternate binding modes and/or native complexes that are artefacts of the 
crystal environment. In other cases, additional model filtering with more 
sophisticated tools is needed. CONCLUSION: The results suggest that the simple 
modelling procedure applied here could help identify and characterize 
protein-protein complexes. The next step is to apply it on a genomic scale. 
</blockquote> 
<p></p> 
<blockquote>  Keywords: Algorithms; Protein Binding; Protein Conformation; 
Protein Interaction Domains and Motifs; Proteins, chemistry/metabolism; 
Structural Homology, Protein </blockquote> <br>
 [Xie2009Unified]  Lei Xie, 
Li&nbsp;Xie, and Philip&nbsp;E Bourne. A unified statistical model to support 
local sequence order independent similarity searching for ligand-binding sites 
and its application to genome-based drug discovery.<em>Bioinformatics</em>, 
25(12):i305-i312, Jun 2009. [&nbsp;bib&nbsp;| DOI&nbsp;| http&nbsp;] 
<blockquote>  Functional relationships between proteins that do not share 
global structure similarity can be established by detecting their 
ligand-binding-site similarity. For a large-scale comparison, it is critical to 
accurately and efficiently assess the statistical significance of this 
similarity. Here, we report an efficient statistical model that supports local 
sequence order independent ligand-binding-site similarity searching. Most 
existing statistical models only take into account the matching vertices 
between two sites that are defined by a fixed number of points. In reality, the 
boundary of the binding site is not known or is dependent on the bound ligand 
making these approaches limited. To address these shortcomings and to perform 
binding-site mapping on a genome-wide scale, we developed a sequence-order 
independent profile-profile alignment (SOIPPA) algorithm that is able to detect 
local similarity between unknown binding sites a priori. The SOIPPA scoring 
integrates geometric, evolutionary and physical information into a unified 
framework. However, this imposes a significant challenge in assessing the 
statistical significance of the similarity because the conventional probability 
model that is based on fixed-point matching cannot be applied. Here we find 
that scores for binding-site matching by SOIPPA follow an extreme value 
distribution (EVD). Benchmark studies show that the EVD model performs at least 
two-orders faster and is more accurate than the non-parametric statistical 
method in the previous SOIPPA version. Efficient statistical analysis makes it 
possible to apply SOIPPA to genome-based drug discovery. Consequently, we have 
applied the approach to the structural genome of Mycobacterium tuberculosis to 
construct a protein-ligand interaction network. The network reveals highly 
connected proteins, which represent suitable targets for promiscuous drugs. 
</blockquote> 
<p></p> 
<blockquote>  Keywords: Binding Sites; Computational Biology, methods; Drug 
Discovery, methods; Genome; Ligands; Models, Statistical; Mycobacterium 
tuberculosis, genetics/metabolism; Proteins, chemistry </blockquote> <br>
<br>

<p><em>This file was generated by bibtex2html 1.94.</em></p> 
</body>