<!doctype html>
<meta charset="utf-8">
<title>R FAQ</title>
<body>

<h1>R FAQ</h1> 
<p></p>  Next:&nbsp; Introduction, Previous:&nbsp; (dir), Up:&nbsp; (dir) <br>

<h2>R FAQ</h2> 
<p></p> 
<h2>Frequently Asked Questions on R</h2> 
<h2>Version 2.7.2008-09-17</h2> 
<h2>ISBN 3-900051-08-9</h2> Kurt Hornik 
<p></p> 
<p></p> 
<p> </p> 
<ul> 
<li>Introduction </li> 
<li>R Basics </li> 
<li>R and S </li> 
<li>R Web Interfaces </li> 
<li>R Add-On Packages </li> 
<li>R and Emacs </li> 
<li>R Miscellanea </li> 
<li>R Programming </li> 
<li>R Bugs </li> 
<li>Acknowledgments </li> </ul> 
<p></p>  Next:&nbsp; R Basics, Previous:&nbsp; Top, Up:&nbsp; Top <br>

<h2>1 Introduction</h2> 
<p>This document contains answers to some of the most frequently asked 
questions about R.</p> 
<ul> 
<li>Legalese </li> 
<li>Obtaining this document </li> 
<li>Citing this document </li> 
<li>Notation </li> 
<li>Feedback </li> </ul> 
<p></p>  Next:&nbsp; Obtaining this document, Previous:&nbsp; Introduction, 
Up:&nbsp;Introduction <br>

<h3>1.1 Legalese</h3> 
<p>This document is copyright &copy; 1998&ndash;2008 by Kurt Hornik. </p> 
<p>This document is free software; you can redistribute it and/or modify it 
under the terms of theGNU General Public License as published by the Free 
Software Foundation; either version 2, or (at your option) any later version.
</p> 
<p>This document is distributed in the hope that it will be useful, but 
WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or 
FITNESS FOR A PARTICULAR PURPOSE. See theGNU General Public License for more 
details.</p> 
<p>A copy of the GNU General Public License is available via WWW at </p> 
<pre> http://www.gnu.org/copyleft/gpl.html. </pre> 
<p>You can also obtain it by writing to the Free Software Foundation, Inc., 51 
Franklin Street, Fifth Floor, Boston, MA 02110-1301, U.S.A.</p> 
<p></p>  Next:&nbsp; Citing this document, Previous:&nbsp; Legalese, Up:&nbsp; 
Introduction <br>

<h3>1.2 Obtaining this document</h3> 
<p>The latest version of this document is always available from </p> 
<pre> http://CRAN.R-project.org/doc/FAQ/ </pre> 
<p>From there, you can obtain versions converted to plain ASCII text, DVI, GNU 
info, HTML, PDF, PostScript as well as the Texinfo source used for creating all 
these formats using theGNU Texinfo system. </p> 
<p>You can also obtain the R FAQ from the doc/FAQ subdirectory of a CRAN site 
(seeWhat is CRAN?). </p> 
<p></p>  Next:&nbsp; Notation, Previous:&nbsp; Obtaining this document, 
Up:&nbsp;Introduction <br>

<h3>1.3 Citing this document</h3> 
<p>In publications, please refer to this FAQ as Hornik (2008), &ldquo;The R FAQ
&rdquo;, and give the above,<em>official</em> URL and the ISBN 3-900051-08-9: 
</p> 
<pre> @Misc{, author = {Kurt Hornik}, title = {The {R} {FAQ}}, year = {2008}, 
note = {{ISBN} 3-900051-08-9}, url = 
{http://CRAN.R-project.org/doc/FAQ/R-FAQ.html} }</pre> 
<p></p>  Next:&nbsp; Feedback, Previous:&nbsp; Citing this document, Up:&nbsp; 
Introduction <br>

<h3>1.4 Notation</h3> 
<p>Everything should be pretty standard. `R&gt;' is used for the R prompt, and 
a `$' for the shell prompt (where applicable). </p> 
<p></p>  Previous:&nbsp; Notation, Up:&nbsp; Introduction <br>

<h3>1.5 Feedback</h3> 
<p>Feedback via email to Kurt.Hornik@R-project.org is of course most welcome. 
</p> 
<p>In particular, note that I do not have access to Windows or Macintosh 
systems. Features specific to the Windows and Mac OS X ports of R are described 
in the&ldquo;R for Windows FAQ&rdquo; and the &ldquo;R for Mac OS X FAQ. If you 
have information on Macintosh or Windows systems that you think should be added 
to this document, please let me know. </p> 
<p></p>  Next:&nbsp; R and S, Previous:&nbsp; Introduction, Up:&nbsp; Top <br>

<h2>2 R Basics</h2> 
<ul> 
<li>What is R? </li> 
<li>What machines does R run on? </li> 
<li>What is the current version of R? </li> 
<li>How can R be obtained? </li> 
<li>How can R be installed? </li> 
<li>Are there Unix binaries for R? </li> 
<li>What documentation exists for R? </li> 
<li>Citing R </li> 
<li>What mailing lists exist for R? </li> 
<li>What is CRAN? </li> 
<li>Can I use R for commercial purposes? </li> 
<li>Why is R named R? </li> 
<li>What is the R Foundation? </li> </ul> 
<p></p>  Next:&nbsp; What machines does R run on?, Previous:&nbsp; R Basics, 
Up:&nbsp;R Basics <br>

<h3>2.1 What is R?</h3> 
<p>R is a system for statistical computation and graphics. It consists of a 
language plus a run-time environment with graphics, a debugger, access to 
certain system functions, and the ability to run programs stored in script 
files.</p> 
<p>The design of R has been heavily influenced by two existing languages: 
Becker, Chambers &amp; Wilks' S (seeWhat is S?) and Sussman's Scheme. Whereas 
the resulting language is very similar in appearance to S, the underlying 
implementation and semantics are derived from Scheme. SeeWhat are the 
differences between R and S?, for further details. </p> 
<p>The core of R is an interpreted computer language which allows branching 
and looping as well as modular programming using functions. Most of the 
user-visible functions in R are written in R. It is possible for the user to 
interface to procedures written in the C, C++, or FORTRAN languages for 
efficiency. The R distribution contains functionality for a large number of 
statistical procedures. Among these are: linear and generalized linear models, 
nonlinear regression models, time series analysis, classical parametric and 
nonparametric tests, clustering and smoothing. There is also a large set of 
functions which provide a flexible graphical environment for creating various 
kinds of data presentations. Additional modules (&ldquo;add-on packages&rdquo;) 
are available for a variety of specific purposes (seeR Add-On Packages). </p> 
<p>R was initially written by Ross Ihaka and Robert Gentleman at the 
Department of Statistics of the University of Auckland in Auckland, New 
Zealand. In addition, a large group of individuals has contributed to R by 
sending code and bug reports.</p> 
<p>Since mid-1997 there has been a core group (the &ldquo;R Core Team&rdquo;) 
who can modify the R source code archive. The group currently consists of Doug 
Bates, John Chambers, Peter Dalgaard, Robert Gentleman, Kurt Hornik, Stefano 
Iacus, Ross Ihaka, Friedrich Leisch, Thomas Lumley, Martin Maechler, Duncan 
Murdoch, Paul Murrell, Martyn Plummer, Brian Ripley, Duncan Temple Lang, Luke 
Tierney, and Simon Urbanek.</p> 
<p>R has a home page at http://www.R-project.org/. It is free software 
distributed under aGNU-style copyleft, and an official part of the GNU project 
(&ldquo;GNU S&rdquo;). </p> 
<p></p>  Next:&nbsp; What is the current version of R?, Previous:&nbsp; What 
is R?, Up:&nbsp; R Basics <br>

<h3>2.2 What machines does R run on?</h3> 
<p>R is being developed for the Unix, Windows and Mac families of operating 
systems. Support for Mac OS Classic ended with R 1.7.1.</p> 
<p>The current version of R will configure and build under a number of common 
Unix platforms includingcpu-linux-gnu for the i386, alpha, arm, hppa, ia64, 
m68k, mips/mipsel, powerpc, s390, sparc (e.g.,
http://buildd.debian.org/build.php?&amp;pkg=r-base), and x86_64 CPUs, 
powerpc-apple-darwin, mips-sgi-irix,  rs6000-ibm-aix, and sparc-sun-solaris. 
</p> 
<p>If you know about other platforms, please drop us a note. </p> 
<p></p>  Next:&nbsp; How can R be obtained?, Previous:&nbsp; What machines 
does R run on?, Up:&nbsp; R Basics <br>

<h3>2.3 What is the current version of R?</h3> 
<p>The current released version is 2.7.2. Based on this 
`major.minor.patchlevel' numbering scheme, there are two development versions 
of R, a patched version of the current release (`r-patched') and one working 
towards the next minor or eventually major (`r-devel') releases of R, 
respectively. Version r-patched is for bug fixes mostly. New features are 
typically introduced in r-devel.</p> 
<p></p>  Next:&nbsp; How can R be installed?, Previous:&nbsp; What is the 
current version of R?, Up:&nbsp; R Basics <br>

<h3>2.4 How can R be obtained?</h3> 
<p>Sources, binaries and documentation for R can be obtained via CRAN, the 
&ldquo;Comprehensive R Archive Network&rdquo; (seeWhat is CRAN?). </p> 
<p>Sources are also available via https://svn.R-project.org/R/, the R 
Subversion repository, but currently not via anonymous rsync (nor CVS).</p> 
<p>Tarballs with daily snapshots of the r-devel and r-patched development 
versions of R can be found atftp://ftp.stat.math.ethz.ch/Software/R. </p> 
<p></p>  Next:&nbsp; Are there Unix binaries for R?, Previous:&nbsp; How can R 
be obtained?, Up:&nbsp; R Basics <br>

<h3>2.5 How can R be installed?</h3> 
<ul> 
<li>How can R be installed (Unix) </li> 
<li>How can R be installed (Windows) </li> 
<li>How can R be installed (Macintosh) </li> </ul> 
<p></p>  Next:&nbsp; How can R be installed (Windows), Previous:&nbsp; How can 
R be installed?, Up:&nbsp; How can R be installed? <br>

<h4>2.5.1 How can R be installed (Unix)</h4> 
<p>If R is already installed, it can be started by typing R at the shell 
prompt (of course, provided that the executable is in your path).</p> 
<p>If binaries are available for your platform (see Are there Unix binaries 
for R?), you can use these, following the instructions that come with them. </p>
<p>Otherwise, you can compile and install R yourself, which can be done very 
easily under a number of common Unix platforms (seeWhat machines does R run on?
). The fileINSTALL that comes with the R distribution contains a brief 
introduction, and the &ldquo;R Installation and Administration&rdquo; guide (see
What documentation exists for R?) has full details. </p> 
<p>Note that you need a FORTRAN compiler or perhaps f2c in addition to a C 
compiler to build R. Also, you need Perl version 5 to build the R object 
documentations. (If this is not available on your system, you can obtain a PDF 
version of the object reference manual viaCRAN.) </p> 
<p>In the simplest case, untar the R source code, change to the directory thus 
created, and issue the following commands (at the shell prompt):</p> 
<pre> $ ./configure $ make </pre> 
<p>If these commands execute successfully, the R binary and a shell script 
front-end calledR are created and copied to the bin directory. You can copy the 
script to a place where users can invoke it, for example to/usr/local/bin. In 
addition, plain text help pages as well asHTML and LaTeX versions of the 
documentation are built.</p> 
<p>Use make dvi to create DVI versions of the R manuals, such as refman.dvi 
(an R object reference index) andR-exts.dvi, the &ldquo;R Extension Writers 
Guide&rdquo;, in thedoc/manual subdirectory. These files can be previewed and 
printed using standard programs such asxdvi and dvips. You can also use make pdf
 to build PDF (Portable Document Format) version of the manuals, and view these 
using e.g. Acrobat. Manuals written in theGNU Texinfo system can also be 
converted to info files suitable for reading online with Emacs or stand-aloneGNU
 Info; usemake info to create these versions (note that this requires Makeinfo 
version 4.5).</p> 
<p>Finally, use make check to find out whether your R system works correctly. 
</p> 
<p>You can also perform a &ldquo;system-wide&rdquo; installation using make 
install. By default, this will install to the following directories: </p> 
<dl> 
<dt> ${prefix}/bin </dt> 
<dd> the front-end shell script <br>
</dd> 
<dt> ${prefix}/man/man1 </dt> 
<dd> the man page <br>
</dd> 
<dt> ${prefix}/lib/R </dt> 
<dd> all the rest (libraries, on-line help system, ...). This is the &ldquo;R 
Home Directory&rdquo; (R_HOME) of the installed system. </dd> </dl> 
<p>In the above, <code>prefix</code> is determined during configuration 
(typically/usr/local) and can be set by running configure with the option </p> 
<pre> $ ./configure --prefix=/where/you/want/R/to/go </pre> 
<p>(E.g., the R executable will then be installed into 
/where/you/want/R/to/go/bin.) </p> 
<p>To install DVI, info and PDF versions of the manuals, use make install-dvi, 
make install-info and make install-pdf, respectively. </p> 
<p></p>  Next:&nbsp; How can R be installed (Macintosh), Previous:&nbsp; How 
can R be installed (Unix), Up:&nbsp; How can R be installed? <br>

<h4>2.5.2 How can R be installed (Windows)</h4> 
<p>The bin/windows directory of a CRAN site contains binaries for a base 
distribution and a large number of add-on packages fromCRAN to run on Windows 
2000 and later (including 64-bit versions of Windows) on ix86 and x86_64 chips. 
The Windows version of R was created by Robert Gentleman and Guido Masarotto, 
and is now being developed and maintained byDuncan Murdoch and Brian D. Ripley. 
</p> 
<p>For most installations the Windows installer program will be the easiest 
tool to use.</p> 
<p>See the &ldquo;R for Windows FAQ&rdquo; for more details. </p> 
<p></p>  Previous:&nbsp; How can R be installed (Windows), Up:&nbsp; How can R 
be installed? <br>

<h4>2.5.3 How can R be installed (Macintosh)</h4> 
<p>The bin/macosx directory of a CRAN site contains a standard Apple installer 
package inside a disk image namedR.dmg. Once downloaded and executed, the 
installer will install the current non-developer release of R. RAqua is a 
native Mac OS X Darwin version of R with a R.app Mac OS XGUI. Inside 
bin/macosx/powerpc/contrib/x.y there are prebuilt binary packages (for powerpc 
version of Mac OS X) to be used with RAqua corresponding to the &ldquo;x.y
&rdquo; release of R. The installation of these packages is available through 
the &ldquo;Package&rdquo; menu of the R.appGUI. This port of R for Mac OS X is 
maintained byStefano Iacus. The &ldquo;R for Mac OS X FAQ has more details. </p>
<p>The bin/macos directory of a CRAN site contains bin-hexed (hqx) and stuffit 
(sit) archives for a base distribution and a large number of add-on packages of 
R 1.7.1 to run under Mac OS 8.6 to Mac OS 9.2.2. This port of R for Macintosh 
is no longer supported.</p> 
<p></p>  Next:&nbsp; What documentation exists for R?, Previous:&nbsp; How can 
R be installed?, Up:&nbsp; R Basics <br>

<h3>2.6 Are there Unix binaries for R?</h3> 
<p>The bin/linux directory of a CRAN site contains the following packages. </p>
<blockquote> 
<p></p> CPU Versions Provider <br>
<br>
Debian i386/amd64 stable/oldstable 
Johannes Ranke <br>
<br>
Red Hat i386/x86_64 fedora7/fedora8 Martyn Plummer <br>
<br> i386/x86_64 el4/el5 Bob Kinney <br>
<br>
SuSE i586/x86_64 
10.0/10.1/10.2/10.3 Detlef Steuer <br>
<br>
Ubuntu i386 
dapper/feisty/gutsy/hardy Vincent Goulet <br>
<br>
amd64 
dapper/feisty/gutsy/hardy Michael Rutter <br>
<br>
<br>
</blockquote> 
<p>Debian packages, maintained by Dirk Eddelbuettel and Doug Bates, have long 
been part of the Debian distribution, and can be accessed through APT, the 
Debian package maintenance tool. Use e.g.<code>apt-get install r-base 
r-recommended</code> to install the R environment and recommended packages. If 
you also want to build R packages from source, also run<code>apt-get install 
r-base-dev</code> to obtain the additional tools required for this. So-called 
&ldquo;backports&rdquo; of the current R packages for thestable distribution of 
Debian are provided by Johannes Ranke, and available from CRAN. Simply add the 
line</p> 
<pre> deb http://CRAN.R-project.org/bin/linux/debian stable/ </pre> 
<p>(feel free to use a CRAN mirror instead of the master) to the file 
/etc/apt/sources.list, and install as usual. More details on installing and 
administering R on Debian Linux can be found at
http://CRAN.R-project.org/bin/linux/debian/README. These backports should also 
be suitable for other Debian derivatives. Native backports for Ubuntu are 
provided by Vincent Goulet and Michael Rutter.</p> 
<p>On SUSE, you can set up an installation source for R within Yast by setting 
(e.g.)</p> 
<pre> Protocol: HTTP Server name: software.openSUSE.org Directory: 
/download/home:/dsteuer/openSUSE_10.2/</pre> 
<p>With this setting, online updates will check for new versions of R. </p> 
<p>The bin/solaris directory of a CRAN site contains binary packages for 
Solaris on the SPARC and x64 platforms, provided by Mithun Sridharan.</p> 
<p>No other binary distributions are currently publically available via CRAN. 
</p> 
<p>A &ldquo;live&rdquo; Linux distribution with a particular focus on R is 
Quantian, which provides a directly bootable and self-configuring &ldquo;Live 
DVD&rdquo; containing numerous applications of interests to scientists and 
researchers, including several hundredCRAN and Bioconductor packages, the 
&ldquo;ESS&rdquo; extensions for Emacs, the &ldquo;JGR&rdquo; JavaGUI for R, 
the Ggobi visualization tool as well as several other R interfaces. TheQuantian 
website athttp://dirk.eddelbuettel.com/quantian/ contains more details as well 
download information.</p> 
<p></p>  Next:&nbsp; Citing R, Previous:&nbsp; Are there Unix binaries for R?, 
Up:&nbsp;R Basics <br>

<h3>2.7 What documentation exists for R?</h3> 
<p>Online documentation for most of the functions and variables in R exists, 
and can be printed on-screen by typinghelp(name) (or ?name) at the R prompt, 
wherename is the name of the topic help is sought for. (In the case of unary 
and binary operators and control-flow special forms, the name may need to be be 
quoted.)</p> 
<p>This documentation can also be made available as one reference manual for 
on-line reading inHTML and PDF formats, and as hardcopy via LaTeX, see How can 
R be installed?. An up-to-date HTML version is always available for web 
browsing athttp://stat.ethz.ch/R-manual/. </p> 
<p>Printed copies of the R reference manual for some version(s) are available 
from Network Theory Ltd, athttp://www.network-theory.co.uk/R/base/. For each 
set of manuals sold, the publisher donates USD 10 to the R Foundation (seeWhat 
is the R Foundation?). </p> 
<p>The R distribution also comes with the following manuals. </p> 
<ul> 
<li>&ldquo;An Introduction to R&rdquo; (R-intro) includes information on data 
types, programming elements, statistical modeling and graphics. This document 
is based on the &ldquo;Notes onS-Plus&rdquo; by Bill Venables and David Smith. 
</li> 
<li>&ldquo;Writing R Extensions&rdquo; (R-exts) currently describes the 
process of creating R add-on packages, writing R documentation, R's system and 
foreign language interfaces, and the RAPI. </li> 
<li>&ldquo;R Data Import/Export&rdquo; (R-data) is a guide to importing and 
exporting data to and from R.</li> 
<li>&ldquo;The R Language Definition&rdquo; (R-lang), a first version of the 
&ldquo;Kernighan &amp; Ritchie of R&rdquo;, explains evaluation, parsing, 
object oriented programming, computing on the language, and so forth.</li> 
<li>&ldquo;R Installation and Administration&rdquo; (R-admin). </li> 
<li>&ldquo;R Internals&rdquo; (R-ints) is a guide to R's internal structures. 
(Added in R 2.4.0.)</li> </ul> 
<p>Books on R include </p> 
<blockquote> P. Dalgaard (2002), &ldquo;Introductory Statistics with R&rdquo;, 
Springer: New York, ISBN 0-387-95475-9,http://www.biostat.ku.dk/~pd/ISwR.html. 
<p>J. Fox (2002), &ldquo;An R and S-Plus Companion to Applied 
Regression&rdquo;, Sage Publications, ISBN 0-761-92280-6 (softcover) or 
0-761-92279-2 (hardcover),
http://socserv.socsci.mcmaster.ca/jfox/Books/Companion/. </p> 
<p>J. Maindonald and J. Braun (2003), &ldquo;Data Analysis and Graphics Using 
R: An Example-Based Approach&rdquo;, Cambridge University Press, ISBN 
0-521-81336-0,http://wwwmaths.anu.edu.au/~johnm/. </p> 
<p>S. M. Iacus and G. Masarotto (2002), &ldquo;Laboratorio di statistica con 
R&rdquo;, McGraw-Hill, ISBN 88-386-6084-0 (in Italian),<br>

http://www.ateneonline.it/LibroAteneo.asp?item_id=1436. </p> 
<p>P. Murrell (2005), &ldquo;R Graphics&rdquo;, Chapman &amp; Hall/CRC, ISBN: 
1-584-88486-X,http://www.stat.auckland.ac.nz/~paul/RGraphics/rgraphics.html. 
</p> </blockquote> 
<p>The book </p> 
<blockquote> W. N. Venables and B. D. Ripley (2002), &ldquo;Modern Applied 
Statistics with S&rdquo; (4th edition). Springer, ISBN 0-387-95457-0
</blockquote> 
<p>has a home page at http://www.stats.ox.ac.uk/pub/MASS4/ providing 
additional material. Its companion is</p> 
<blockquote> W. N. Venables and B. D. Ripley (2000), &ldquo;S 
Programming&rdquo;. Springer, ISBN 0-387-98966-8</blockquote> 
<p>and provides an in-depth guide to writing software in the S language which 
forms the basis of both the commercialS-Plus and the Open Source R data 
analysis software systems. Seehttp://www.stats.ox.ac.uk/pub/MASS3/Sprog/ for 
more information.</p> 
<p>In addition to material written specifically or explicitly for R, 
documentation for S/S-Plus (see R and S) can be used in combination with this 
FAQ (see What are the differences between R and S?). Introductory books include 
</p> 
<blockquote> P. Spector (1994), &ldquo;An introduction to S and S-Plus&rdquo;, 
Duxbury Press.
<p>A. Krause and M. Olsen (2005), &ldquo;The Basics of S-Plus&rdquo; (4th 
edition). Springer, ISBN 0-387-26109-5.</p> </blockquote> 
<p>The book </p> 
<blockquote> J. C. Pinheiro and D. M. Bates (2000), &ldquo;Mixed-Effects 
Models in S andS-Plus&rdquo;, Springer, ISBN 0-387-98957-0 </blockquote> 
<p>provides a comprehensive guide to the use of the <strong>nlme</strong> 
package for linear and nonlinear mixed-effects models. </p> 
<p>As an example of how R can be used in teaching an advanced introductory 
statistics course, see</p> 
<blockquote> D. Nolan and T. Speed (2000), &ldquo;Stat Labs: Mathematical 
Statistics Through Applications&rdquo;, Springer Texts in Statistics, ISBN 
0-387-98974-9</blockquote> 
<p>This integrates theory of statistics with the practice of statistics 
through a collection of case studies (&ldquo;labs&rdquo;), and uses R to 
analyze the data. More information can be found at
http://www.stat.Berkeley.EDU/users/statlabs/. </p> 
<p>Last, but not least, Ross' and Robert's experience in designing and 
implementing R is described in Ihaka &amp; Gentleman (1996), &ldquo;R: A 
Language for Data Analysis and Graphics&rdquo;,<em>Journal of Computational and 
Graphical Statistics</em>, <strong>5</strong>, 299&ndash;314. </p> 
<p>An annotated bibliography (BibTeX format) of R-related publications which 
includes most of the above references can be found at</p> 
<pre> http://www.R-project.org/doc/bib/R.bib </pre> 
<p></p>  Next:&nbsp; What mailing lists exist for R?, Previous:&nbsp; What 
documentation exists for R?, Up:&nbsp; R Basics <br>

<h3>2.8 Citing R</h3> 
<p>To cite R in publications, use </p> 
<pre> @Manual{, title = {R: A Language and Environment for Statistical 
Computing}, author = {{R Development Core Team}}, organization = {R Foundation 
for Statistical Computing}, address = {Vienna, Austria}, year = 2008, note = 
{{ISBN} 3-900051-07-0}, url = {http://www.R-project.org} }</pre> 
<p>Citation strings (or BibTeX entries) for R and R packages can also be 
obtained by<code>citation()</code>. </p> 
<p></p>  Next:&nbsp; What is CRAN?, Previous:&nbsp; Citing R, Up:&nbsp; R 
Basics <br>

<h3>2.9 What mailing lists exist for R?</h3> 
<p>Thanks to Martin Maechler, there are four mailing lists devoted to R. </p> 
<dl> 
<dt> <code>R-announce</code> </dt> 
<dd> A moderated list for major announcements about the development of R and 
the availability of new code.<br>
</dd> 
<dt> <code>R-packages</code> </dt> 
<dd> A moderated list for announcements on the availability of new or enhanced 
contributed packages.<br>
</dd> 
<dt> <code>R-help</code> </dt> 
<dd> The `main' R mailing list, for discussion about problems and solutions 
using R, announcements (not covered by `R-announce' and `R-packages') about the 
development of R and the availability of new code. <br>
</dd> 
<dt> <code>R-devel</code> </dt> 
<dd> This list is for questions and discussion about code development in R. 
</dd> </dl> 
<p>Please read the posting guide <em>before</em> sending anything to any 
mailing list.</p> 
<p>Note in particular that R-help is intended to be comprehensible to people 
who want to use R to solve problems but who are not necessarily interested in 
or knowledgeable about programming. Questions likely to prompt discussion 
unintelligible to non-programmers (e.g., questions involving C or C++) should 
go to R-devel.</p> 
<p>Convenient access to information on these lists, subscription, and archives 
is provided by the web interface athttp://stat.ethz.ch/mailman/listinfo/. One 
can also subscribe (or unsubscribe) via email, e.g. to R-help by sending `
subscribe' (or `unsubscribe') in the <em>body</em> of the message (not in the 
subject!) toR-help-request@lists.R-project.org. </p> 
<p>Send email to R-help@lists.R-project.org to send a message to everyone on 
the R-help mailing list. Subscription and posting to the other lists is done 
analogously, with `R-help' replaced by `R-announce', `R-packages', and `R-devel
', respectively. Note that the R-announce and R-packages lists are gatewayed 
into R-help. Hence, you should subscribe to either of them only in case you are 
not subscribed to R-help.</p> 
<p>It is recommended that you send mail to R-help rather than only to the R 
Core developers (who are also subscribed to the list, of course). This may save 
them precious time they can use for constantly improving R, and will typically 
also result in much quicker feedback for yourself.</p> 
<p>Of course, in the case of bug reports it would be very helpful to have code 
which reliably reproduces the problem. Also, make sure that you include 
information on the system and version of R being used. SeeR Bugs for more 
details.</p> 
<p>See http://www.R-project.org/mail.html for more information on the R 
mailing lists.</p> 
<p>The R Core Team can be reached at R-core@lists.R-project.org for comments 
and reports.</p> 
<p>Many of the R project's mailing lists are also available via Gmane, from 
which they can be read with a web browser, using an NNTP news reader, or via 
RSS feeds. Seehttp://dir.gmane.org/index.php?prefix=gmane.comp.lang.r. for the 
available mailing lists, andhttp://www.gmane.org/rss.php for details on RSS 
feeds.</p> 
<p></p>  Next:&nbsp; Can I use R for commercial purposes?, Previous:&nbsp; 
What mailing lists exist for R?, Up:&nbsp; R Basics <br>

<h3>2.10 What is CRAN?</h3> 
<p>The &ldquo;Comprehensive R Archive Network&rdquo; (CRAN) is a collection of 
sites which carry identical material, consisting of the R distribution(s), the 
contributed extensions, documentation for R, and binaries.</p> 
<p>The CRAN master site at Wirtschaftsuniversit&auml;t Wien, Austria, can be 
found at theURL </p> 
<blockquote> http://CRAN.R-project.org/ </blockquote> 
<p>  Daily mirrors are available at URLs including </p> 
<blockquote> 
<p></p> http://cran.at.R-project.org/ (WU Wien, Austria) <br>
<br>

http://cran.au.R-project.org/ (PlanetMirror, Australia) <br>
<br>

http://cran.br.R-project.org/ (Universidade Federal do Paran&aacute;, Brazil) 
<br> <br>
http://cran.ch.R-project.org/ (ETH Z&uuml;rich, Switzerland) <br>
<br>
http://cran.dk.R-project.org/ (SunSITE, Denmark) <br>
<br>

http://cran.es.R-project.org/ (Spanish National Research Network, Madrid, Spain)
<br> <br>
http://cran.fr.R-project.org/ (INRA, Toulouse, France) <br>
<br>

http://cran.pt.R-project.org/ (Universidade do Porto, Portugal) <br>
<br>

http://cran.uk.R-project.org/ (U of Bristol, United Kingdom) <br>
<br>

http://cran.za.R-project.org/ (Rhodes U, South Africa) <br>
<br>
<br>

</blockquote> 
<p>See http://CRAN.R-project.org/mirrors.html for a complete list of mirrors. 
Please use theCRAN site closest to you to reduce network load. </p> 
<p>From CRAN, you can obtain the latest official release of R, daily snapshots 
of R (copies of the current source trees), as gzipped and bzipped tar files, a 
wealth of additional contributed code, as well as prebuilt binaries for various 
operating systems (Linux, Mac OS Classic, Mac OS X, and MS Windows).CRAN also 
provides access to documentation on R, existing mailing lists and the R Bug 
Tracking system.</p> 
<p>To &ldquo;submit&rdquo; to CRAN, simply upload to 
ftp://CRAN.R-project.org/incoming/ and send an email to CRAN@R-project.org. 
Note thatCRAN generally does not accept submissions of precompiled binaries due 
to security reasons. In particular, binary packages for Windows and Mac OS X 
are provided by the respective binary package maintainers.</p> 
<blockquote> <b>Note:</b> It is very important that you indicate the copyright 
(license) information (GPL-2, GPL-3, BSD, Artistic, ...) in your submission. 
</blockquote> 
<p>Please always use the URL of the master site when referring to CRAN. </p> 
<p></p>  Next:&nbsp; Why is R named R?, Previous:&nbsp; What is CRAN?, 
Up:&nbsp;R Basics <br>

<h3>2.11 Can I use R for commercial purposes?</h3> 
<p>R is released under the GNU General Public License (GPL). If you have any 
questions regarding the legality of using R in any particular situation you 
should bring it up with your legal counsel. We are in no position to offer 
legal advice.</p> 
<p>It is the opinion of the R Core Team that one can use R for commercial 
purposes (e.g., in business or in consulting). The GPL, like all Open Source 
licenses, permits all and any use of the package. It only restricts 
distribution of R or of other programs containing code from R. This is made 
clear in clause 6 (&ldquo;No Discrimination Against Fields of Endeavor&rdquo;) 
of theOpen Source Definition: </p> 
<blockquote> The license must not restrict anyone from making use of the 
program in a specific field of endeavor. For example, it may not restrict the 
program from being used in a business, or from being used for genetic research.
</blockquote> 
<p>It is also explicitly stated in clause 0 of the GPL, which says in part </p>
<blockquote> Activities other than copying, distribution and modification are 
not covered by this License; they are outside its scope. The act of running the 
Program is not restricted, and the output from the Program is covered only if 
its contents constitute a work based on the Program.</blockquote> 
<p>Most add-on packages, including all recommended ones, also explicitly allow 
commercial use in this way. A few packages are restricted to 
&ldquo;non-commercial use&rdquo;; you should contact the author to clarify 
whether these may be used or seek the advice of your legal counsel.</p> 
<p>None of the discussion in this section constitutes legal advice. The R Core 
Team does not provide legal advice under any circumstances.</p> 
<p></p>  Next:&nbsp; What is the R Foundation?, Previous:&nbsp; Can I use R 
for commercial purposes?, Up:&nbsp; R Basics <br>

<h3>2.12 Why is R named R?</h3> 
<p>The name is partly based on the (first) names of the first two R authors 
(Robert Gentleman and Ross Ihaka), and partly a play on the name of the Bell 
Labs language `S' (seeWhat is S?). </p> 
<p></p>  Previous:&nbsp; Why is R named R?, Up:&nbsp; R Basics <br>

<h3>2.13 What is the R Foundation?</h3> 
<p>The R Foundation is a not for profit organization working in the public 
interest. It was founded by the members of the R Core Team in order to provide 
support for the R project and other innovations in statistical computing, 
provide a reference point for individuals, institutions or commercial 
enterprises that want to support or interact with the R development community, 
and to hold and administer the copyright of R software and documentation. See
http://www.R-project.org/foundation/ for more information. </p> 
<p></p>  Next:&nbsp; R Web Interfaces, Previous:&nbsp; R Basics, Up:&nbsp; Top 
<br> 
<h2>3 R and S</h2> 
<ul> 
<li>What is S? </li> 
<li>What is S-PLUS? </li> 
<li>What are the differences between R and S? </li> 
<li>Is there anything R can do that S-PLUS cannot? </li> 
<li>What is R-plus? </li> </ul> 
<p></p>  Next:&nbsp; What is S-PLUS?, Previous:&nbsp; R and S, Up:&nbsp; R and 
S <br>

<h3>3.1 What is S?</h3> 
<p>S is a very high level language and an environment for data analysis and 
graphics. In 1998, the Association for Computing Machinery (ACM) presented its 
Software System Award to John M. Chambers, the principal designer of S, for</p> 
<blockquote> the S system, which has forever altered the way people analyze, 
visualize, and manipulate data... 
<p>S is an elegant, widely accepted, and enduring software system, with 
conceptual integrity, thanks to the insight, taste, and effort of John Chambers.
</p> </blockquote> 
<p>The evolution of the S language is characterized by four books by John 
Chambers and coauthors, which are also the primary references for S.</p> 
<ul> 
<li>Richard A. Becker and John M. Chambers (1984), &ldquo;S. An Interactive 
Environment for Data Analysis and Graphics,&rdquo; Monterey: Wadsworth and 
Brooks/Cole.
<p>This is also referred to as the &ldquo;<em>Brown Book</em>&rdquo;, and of 
historical interest only.</p></li> 
<li>Richard A. Becker, John M. Chambers and Allan R. Wilks (1988), &ldquo;The 
New S Language,&rdquo; London: Chapman &amp; Hall.
<p>This book is often called the &ldquo;<em>Blue Book</em>&rdquo;, and 
introduced what is now known as S version 2.</p></li> 
<li>John M. Chambers and Trevor J. Hastie (1992), &ldquo;Statistical Models in 
S,&rdquo; London: Chapman &amp; Hall.
<p>This is also called the &ldquo;<em>White Book</em>&rdquo;, and introduced S 
version 3, which added structures to facilitate statistical modeling in S.</p>
</li> 
<li>John M. Chambers (1998), &ldquo;Programming with Data,&rdquo; New York: 
Springer, ISBN 0-387-98503-4 (
http://cm.bell-labs.com/cm/ms/departments/sia/Sbook/). 
<p>This &ldquo;<em>Green Book</em>&rdquo; describes version 4 of S, a major 
revision of S designed by John Chambers to improve its usefulness at every 
stage of the programming process.</p></li> </ul> 
<p>See http://cm.bell-labs.com/cm/ms/departments/sia/S/history.html for 
further information on &ldquo;Stages in the Evolution of S&rdquo;.</p> 
<p>There is a huge amount of user-contributed code for S, available at the S 
Repository at CMU. </p> 
<p></p>  Next:&nbsp; What are the differences between R and S?, Previous:&nbsp;
What is S?, Up:&nbsp; R and S <br>

<h3>3.2 What is S-Plus?</h3> 
<p>S-Plus is a value-added version of S sold by Insightful Corporation. Based 
on the S language,S-Plus provides functionality in a wide variety of areas, 
including robust regression, modern non-parametric regression, time series, 
survival analysis, multivariate analysis, classical statistical tests, quality 
control, and graphics drivers. Add-on modules add additional capabilities.</p> 
<p>See the Insightful S-Plus page for further information. </p> 
<p></p>  Next:&nbsp; Is there anything R can do that S-PLUS cannot?, 
Previous:&nbsp;What is S-PLUS?, Up:&nbsp; R and S <br>

<h3>3.3 What are the differences between R and S?</h3> 
<p>We can regard S as a language with three current implementations or 
&ldquo;engines&rdquo;, the &ldquo;old S engine&rdquo; (S version 3;S-Plus 3.x 
and 4.x), the &ldquo;new S engine&rdquo; (S version 4;S-Plus 5.x and above), 
and R. Given this understanding, asking for &ldquo;the differences between R 
and S&rdquo; really amounts to asking for the specifics of the R implementation 
of the S language, i.e., the difference between the R and S<em>engines</em>. 
</p> 
<p>For the remainder of this section, &ldquo;S&rdquo; refers to the S engines 
and not the S language.</p> 
<ul> 
<li>Lexical scoping </li> 
<li>Models </li> 
<li>Others </li> </ul> 
<p></p>  Next:&nbsp; Models, Previous:&nbsp; What are the differences between 
R and S?, Up:&nbsp; What are the differences between R and S? <br>

<h4>3.3.1 Lexical scoping</h4> 
<p>Contrary to other implementations of the S language, R has adopted an 
evaluation model in which nested function definitions are lexically scoped. 
This is analogous to the evalutation model in Scheme.</p> 
<p>This difference becomes manifest when <em>free</em> variables occur in a 
function. Free variables are those which are neither formal parameters 
(occurring in the argument list of the function) nor local variables (created 
by assigning to them in the body of the function). In S, the values of free 
variables are determined by a set of global variables (similar to C, there is 
only local and global scope). In R, they are determined by the environment in 
which the function was created.</p> 
<p>Consider the following function: </p> 
<pre> cube &lt;- function(n) { sq &lt;- function() n * n n * sq() } </pre> 
<p>Under S, <code>sq()</code> does not &ldquo;know&rdquo; about the variable 
<code>n</code> unless it is defined globally: </p> 
<pre> S&gt; cube(2) Error in sq(): Object &quot;n&quot; not found Dumped S&gt; 
n &lt;- 3 S&gt; cube(2) [1] 18</pre> 
<p>In R, the &ldquo;environment&rdquo; created when <code>cube()</code> was 
invoked is also looked in:</p> 
<pre> R&gt; cube(2) [1] 8 </pre> 
<p>As a more &ldquo;interesting&rdquo; real-world problem, suppose you want to 
write a function which returns the density function of the r-th order statistic 
from a sample of size n from a (continuous) distribution. For simplicity, we 
shall use both the cdf and pdf of the distribution as explicit arguments. 
(Example compiled from various postings by Luke Tierney.)</p> 
<p>The S-Plus documentation for <code>call()</code> basically suggests the 
following:</p> 
<pre> dorder &lt;- function(n, r, pfun, dfun) { f &lt;- function(x) NULL con 
&lt;- round(exp(lgamma(n + 1) - lgamma(r) - lgamma(n - r + 1))) PF &lt;- 
call(substitute(pfun), as.name(&quot;x&quot;)) DF &lt;- call(substitute(dfun), 
as.name(&quot;x&quot;)) f[[length(f)]] &lt;- call(&quot;*&quot;, con, 
call(&quot;*&quot;, call(&quot;^&quot;, PF, r - 1), call(&quot;*&quot;, 
call(&quot;^&quot;, call(&quot;-&quot;, 1, PF), n - r), DF))) f }</pre> 
<p>Rather tricky, isn't it? The code uses the fact that in S, functions are 
just lists of special mode with the function body as the last argument, and 
hence does not work in R (one could make the idea work, though).</p> 
<p>A version which makes heavy use of <code>substitute()</code> and seems to 
work under both S and R is</p> 
<pre> dorder &lt;- function(n, r, pfun, dfun) { con &lt;- round(exp(lgamma(n + 
1) - lgamma(r) - lgamma(n - r + 1))) eval(substitute(function(x) K * PF(x)^a * 
(1 - PF(x))^b * DF(x), list(PF = substitute(pfun), DF = substitute(dfun), a = r 
- 1, b = n - r, K = con))) }</pre> 
<p>(the <code>eval()</code> is not needed in S). </p> 
<p>However, in R there is a much easier solution: </p> 
<pre> dorder &lt;- function(n, r, pfun, dfun) { con &lt;- round(exp(lgamma(n + 
1) - lgamma(r) - lgamma(n - r + 1))) function(x) { con * pfun(x)^(r - 1) * (1 - 
pfun(x))^(n - r) * dfun(x) } }</pre> 
<p>This seems to be the &ldquo;natural&rdquo; implementation, and it works 
because the free variables in the returned function can be looked up in the 
defining environment (this is lexical scope).</p> 
<p>Note that what you really need is the function <em>closure</em>, i.e., the 
body along with all variable bindings needed for evaluating it. Since in the 
above version, the free variables in the value function are not modified, you 
can actually use it in S as well if you abstract out the closure operation into 
a function<code>MC()</code> (for &ldquo;make closure&rdquo;): </p> 
<pre> dorder &lt;- function(n, r, pfun, dfun) { con &lt;- round(exp(lgamma(n + 
1) - lgamma(r) - lgamma(n - r + 1))) MC(function(x) { con * pfun(x)^(r - 1) * 
(1 - pfun(x))^(n - r) * dfun(x) }, list(con = con, pfun = pfun, dfun = dfun, r 
= r, n = n)) }</pre> 
<p>Given the appropriate definitions of the closure operator, this works in 
both R and S, and is much &ldquo;cleaner&rdquo; than a substitute/eval solution 
(or one which overrules the default scoping rules by using explicit access to 
evaluation frames, as is of course possible in both R and S).</p> 
<p>For R, <code>MC()</code> simply is </p> 
<pre> MC &lt;- function(f, env) f </pre> 
<p>(lexical scope!), a version for S is </p> 
<pre> MC &lt;- function(f, env = NULL) { env &lt;- as.list(env) if (mode(f) != 
&quot;function&quot;) stop(paste(&quot;not a function:&quot;, f)) if 
(length(env) &gt; 0 &amp;&amp; any(names(env) == &quot;&quot;)) 
stop(paste(&quot;not all arguments are named:&quot;, env)) fargs &lt;- 
if(length(f) &gt; 1) f[1:(length(f) - 1)] else NULL fargs &lt;- c(fargs, env) 
if (any(duplicated(names(fargs)))) stop(paste(&quot;duplicated 
arguments:&quot;, paste(names(fargs)), collapse = &quot;, &quot;)) fbody &lt;- 
f[length(f)] cf &lt;- c(fargs, fbody) mode(cf) &lt;- &quot;function&quot; 
return(cf) }</pre> 
<p>Similarly, most optimization (or zero-finding) routines need some arguments 
to be optimized over and have other parameters that depend on the data but are 
fixed with respect to optimization. With R scoping rules, this is a trivial 
problem; simply make up the function with the required definitions in the same 
environment and scoping takes care of it. With S, one solution is to add an 
extra parameter to the function and to the optimizer to pass in these extras, 
which however can only work if the optimizer supports this.</p> 
<p>Nested lexically scoped functions allow using function closures and 
maintaining local state. A simple example (taken from Abelson and Sussman) is 
obtained by typingdemo(&quot;scoping&quot;) at the R prompt. Further 
information is provided in the standard R reference &ldquo;R: A Language for 
Data Analysis and Graphics&rdquo; (seeWhat documentation exists for R?) and in 
Robert Gentleman and Ross Ihaka (2000), &ldquo;Lexical Scope and Statistical 
Computing&rdquo;,<em>Journal of Computational and Graphical Statistics</em>, 
<strong>9</strong>, 491&ndash;508. </p> 
<p>Nested lexically scoped functions also imply a further major difference. 
Whereas S stores all objects as separate files in a directory somewhere (usually
.Data under the current directory), R does not. All objects in R are stored 
internally. When R is started up it grabs a piece of memory and uses it to 
store the objects. R performs its own memory management of this piece of 
memory, growing and shrinking its size as needed. Having everything in memory 
is necessary because it is not really possible to externally maintain all 
relevant &ldquo;environments&rdquo; of symbol/value pairs. This difference also 
seems to make R<em>faster</em> than S. </p> 
<p>The down side is that if R crashes you will lose all the work for the 
current session. Saving and restoring the memory &ldquo;images&rdquo; (the 
functions and data stored in R's internal memory at any time) can be a bit 
slow, especially if they are big. In S this does not happen, because everything 
is saved in disk files and if you crash nothing is likely to happen to them. 
(In fact, one might conjecture that the S developers felt that the price of 
changing their approach to persistent storage just to accommodate lexical scope 
was far too expensive.) Hence, when doing important work, you might consider 
saving often (seeHow can I save my workspace?) to safeguard against possible 
crashes. Other possibilities are logging your sessions, or have your R commands 
stored in text files which can be read in using<code>source()</code>. </p> 
<blockquote> <b>Note:</b> If you run R from within Emacs (see R and Emacs), 
you can save the contents of the interaction buffer to a file and conveniently 
manipulate it using<code>ess-transcript-mode</code>, as well as save source 
copies of all functions and data used.</blockquote> 
<p></p>  Next:&nbsp; Others, Previous:&nbsp; Lexical scoping, Up:&nbsp; What 
are the differences between R and S? <br>

<h4>3.3.2 Models</h4> 
<p>There are some differences in the modeling code, such as </p> 
<ul> 
<li>Whereas in S, you would use <code>lm(y ~ x^3)</code> to regress <code>y
</code> on <code>x^3</code>, in R, you have to insulate powers of numeric 
vectors (using<code>I()</code>), i.e., you have to use <code>lm(y ~ I(x^3))
</code>. </li> 
<li>The glm family objects are implemented differently in R and S. The same 
functionality is available but the components have different names.</li> 
<li>Option <code>na.action</code> is set to <code>&quot;na.omit&quot;</code> 
by default in R, but not set in S.</li> 
<li>Terms objects are stored differently. In S a terms object is an expression 
with attributes, in R it is a formula with attributes. The attributes have the 
same names but are mostly stored differently.</li> 
<li>Finally, in R <code>y ~ x + 0</code> is an alternative to <code>y ~ x - 1
</code> for specifying a model with no intercept. Models with no parameters at 
all can be specified by<code>y ~ 0</code>. </li> </ul> 
<p></p>  Previous:&nbsp; Models, Up:&nbsp; What are the differences between R 
and S? <br>

<h4>3.3.3 Others</h4> 
<p>Apart from lexical scoping and its implications, R follows the S language 
definition in the Blue and White Books as much as possible, and hence really is 
an &ldquo;implementation&rdquo; of S. There are some intentional differences 
where the behavior of S is considered &ldquo;not clean&rdquo;. In general, the 
rationale is that R should help you detect programming errors, while at the 
same time being as compatible as possible with S.</p> 
<p>Some known differences are the following. </p> 
<ul> 
<li>In R, if <code>x</code> is a list, then <code>x[i] &lt;- NULL</code> and 
<code>x[[i]] &lt;- NULL</code> remove the specified elements from <code>x</code>
. The first of these is incompatible with S, where it is a no-op. (Note that 
you can set elements to<code>NULL</code> using <code>x[i] &lt;- list(NULL)
</code>.) </li> 
<li>In S, the functions named <code>.First</code> and <code>.Last</code> in the
.Data directory can be used for customizing, as they are executed at the very 
beginning and end of a session, respectively.
<p>In R, the startup mechanism is as follows. Unless --no-environ was given on 
the command line, R searches for site and user files to process for setting 
environment variables. Then, R searches for a site-wide startup profile unless 
the command line option--no-site-file was given. This code is loaded in package 
<strong>base</strong>. Then, unless --no-init-file was given, R searches for a 
user profile file, and sources it into the user workspace. It then loads a 
saved image of the user workspace from.RData in case there is one (unless 
--no-restore-data or --no-restore were specified). Next, a function <code>
.First()</code> is run if found on the search path. Finally, function <code>
.First.sys</code> in the <strong>base</strong> package is run. When terminating 
an R session, by default a function<code>.Last</code> is run if found on the 
search path, followed by<code>.Last.sys</code>. If needed, the functions <code>
.First()</code> and <code>.Last()</code> should be defined in the appropriate 
startup profiles. See the help pages for<code>.First</code> and <code>.Last
</code> for more details. </p></li> 
<li>In R, <code>T</code> and <code>F</code> are just variables being set to 
<code>TRUE</code> and <code>FALSE</code>, respectively, but are not reserved 
words as in S and hence can be overwritten by the user. (This helps e.g. when 
you have factors with levels<code>&quot;T&quot;</code> or <code>&quot;F&quot;
</code>.) Hence, when writing code you should always use <code>TRUE</code> and 
<code>FALSE</code>. </li> 
<li>In R, <code>dyn.load()</code> can only load <em>shared objects</em>, as 
created for example byR CMD SHLIB. </li> 
<li>In R, <code>attach()</code> currently only works for lists and data 
frames, but not for directories. (In fact,<code>attach()</code> also works for 
R data files created with<code>save()</code>, which is analogous to attaching 
directories in S.) Also, you cannot attach at position 1.</li> 
<li>Categories do not exist in R, and never will as they are deprecated now in 
S. Use factors instead.</li> 
<li>In R, <code>For()</code> loops are not necessary and hence not supported. 
</li> 
<li>In R, <code>assign()</code> uses the argument envir= rather than where= as 
in S.</li> 
<li>The random number generators are different, and the seeds have different 
length.</li> 
<li>R passes integer objects to C as <code>int *</code> rather than <code>long 
*</code> as in S. </li> 
<li>R has no single precision storage mode. However, as of version 0.65.1, 
there is a single precision interface to C/FORTRAN subroutines.</li> 
<li>By default, <code>ls()</code> returns the names of the objects in the 
current (under R) and global (under S) environment, respectively. For example, 
given
<pre> x &lt;- 1; fun &lt;- function() {y &lt;- 1; ls()} </pre> 
<p>then <code>fun()</code> returns <code>&quot;y&quot;</code> in R and <code>
&quot;x&quot;</code> (together with the rest of the global environment) in S. 
</p></li> 
<li>R allows for zero-extent matrices (and arrays, i.e., some elements of the 
<code>dim</code> attribute vector can be 0). This has been determined a useful 
feature as it helps reducing the need for special-case tests for empty subsets. 
For example, if<code>x</code> is a matrix, <code>x[, FALSE]</code> is not <code>
NULL</code> but a &ldquo;matrix&rdquo; with 0 columns. Hence, such objects need 
to be tested for by checking whether their<code>length()</code> is zero (which 
works in both R and S), and not using<code>is.null()</code>. </li> 
<li>Named vectors are considered vectors in R but not in S (e.g., <code>
is.vector(c(a = 1:3))</code> returns <code>FALSE</code> in S and <code>TRUE
</code> in R). </li> 
<li>Data frames are not considered as matrices in R (i.e., if <code>DF</code> 
is a data frame, then<code>is.matrix(DF)</code> returns <code>FALSE</code> in R 
and<code>TRUE</code> in S). </li> 
<li>R by default uses treatment contrasts in the unordered case, whereas S 
uses the Helmert ones. This is a deliberate difference reflecting the opinion 
that treatment contrasts are more natural.</li> 
<li>In R, the argument of a replacement function which corresponds to the 
right hand side must be named `value'. E.g., <code>f(a) &lt;- b</code> is 
evaluated as<code>a &lt;- &quot;f&lt;-&quot;(a, value = b)</code>. S always 
takes the last argument, irrespective of its name.</li> 
<li>In S, <code>substitute()</code> searches for names for substitution in the 
given expression in three places: the actual and the default arguments of the 
matching call, and the local frame (in that order). R looks in the local frame 
only, with the special rule to use a &ldquo;promise&rdquo; if a variable is not 
evaluated. Since the local frame is initialized with the actual arguments or 
the default expressions, this is usually equivalent to S, until assignment 
takes place.</li> 
<li>In S, the index variable in a <code>for()</code> loop is local to the 
inside of the loop. In R it is local to the environment where the<code>for()
</code> statement is executed. </li> 
<li>In S, <code>tapply(simplify=TRUE)</code> returns a vector where R returns 
a one-dimensional array (which can have named dimnames).</li> 
<li>In S(-Plus) the C locale is used, whereas in R the current operating 
system locale is used for determining which characters are alphanumeric and how 
they are sorted. This affects the set of valid names for R objects (for example 
accented chars may be allowed in R) and ordering in sorts and comparisons (such 
as whether<code>&quot;aA&quot; &lt; &quot;Bb&quot;</code> is true or false). 
From version 1.2.0 the locale can be (re-)set in R by the<code>Sys.setlocale()
</code> function. </li> 
<li>In S, <code>missing(</code>arg<code>)</code> remains <code>TRUE</code> if 
arg is subsequently modified; in R it doesn't. </li> 
<li>From R version 1.3.0, <code>data.frame</code> strips <code>I()</code> when 
creating (column) names.</li> 
<li>In R, the string <code>&quot;NA&quot;</code> is not treated as a missing 
value in a character variable. Use<code>as.character(NA)</code> to create a 
missing character value.</li> 
<li>R disallows repeated formal arguments in function calls. </li> 
<li>In S, <code>dump()</code>, <code>dput()</code> and <code>deparse()</code> 
are essentially different interfaces to the same code. In R from version 2.0.0, 
this is only true if the same<code>control</code> argument is used, but by 
default it is not. By default<code>dump()</code> tries to write code that will 
evaluate to reproduce the object, whereas<code>dput()</code> and <code>deparse()
</code> default to options for producing deparsed code that is readable. </li> 
<li>In R, indexing a vector, matrix, array or data frame with <code>[</code> 
using a character vector index looks only for exact matches (whereas<code>[[
</code> and <code>$</code> allow partial matches). In S, <code>[</code> allows 
partial matches.</li> 
<li>S has a two-argument version of <code>atan</code> and no <code>atan2</code>
. A call in S such as<code>atan(x1, x2)</code> is equivalent to R's <code>
atan2(x1, x2)</code>. However, beware of named arguments since S's <code>atan(x 
= a, y = b)</code> is equivalent to R's <code>atan2(y = a, x = b)</code> with 
the meanings of<code>x</code> and <code>y</code> interchanged. (R used to have 
undocumented support for a two-argument<code>atan</code> with positional 
arguments, but this has been withdrawn to avoid further confusion.)</li> 
<li>Numeric constants with no fractional and exponent (i.e., only integer) 
part are taken as integer inS-Plus 6.x or later, but as double in R. </li> </ul>
<p>There are also differences which are not intentional, and result from 
missing or incorrect code in R. The developers would appreciate hearing about 
any deficiencies you may find (in a written report fully documenting the 
difference as you see it). Of course, it would be useful if you were to 
implement the change yourself and make sure it works.</p> 
<p></p>  Next:&nbsp; What is R-plus?, Previous:&nbsp; What are the differences 
between R and S?, Up:&nbsp; R and S <br>

<h3>3.4 Is there anything R can do that S-Plus cannot?</h3> 
<p>Since almost anything you can do in R has source code that you could port to
S-Plus with little effort there will never be much you can do in R that you 
couldn't do inS-Plus if you wanted to. (Note that using lexical scoping may 
simplify matters considerably, though.)</p> 
<p>R offers several graphics features that S-Plus does not, such as finer 
handling of line types, more convenient color handling (via palettes), gamma 
correction for color, and, most importantly, mathematical annotation in plot 
texts, via input expressions reminiscent of TeX constructs. See the help page 
for<code>plotmath</code>, which features an impressive on-line example. More 
details can be found in Paul Murrell and Ross Ihaka (2000), &ldquo;An Approach 
to Providing Mathematical Annotation in Plots&rdquo;,<em>Journal of 
Computational and Graphical Statistics</em>, <strong>9</strong>, 582&ndash;599. 
</p> 
<p></p>  Previous:&nbsp; Is there anything R can do that S-PLUS cannot?, 
Up:&nbsp;R and S <br>

<h3>3.5 What is R-plus?</h3> 
<p>For a very long time, there was no such thing. </p> 
<p>XLSolutions Corporation is currently beta testing a commercially supported 
version of R named R+ (read R plus).</p> 
<p>In addition, REvolution Computing has released RPro, an enterprise-class 
statistical analysis system based on R, suitable for deployment in 
professional, commercial and regulated environments.</p> 
<p>Random Technologies offers RStat, an enterprise-strength statistical 
computing environment which combines R with enterprise-level validation, 
documentation, software support, and consulting services, as well as related 
R-based products.</p> 
<p></p>  Next:&nbsp; R Add-On Packages, Previous:&nbsp; R and S, Up:&nbsp; Top 
<br> 
<h2>4 R Web Interfaces</h2> 
<p><strong>Rweb</strong> is developed and maintained by Jeff Banfield. The 
Rweb Home Page provides access to all three versions of Rweb&mdash;a simple 
text entry form that returns output and graphs, a more sophisticated Javascript 
version that provides a multiple window environment, and a set of point and 
click modules that are useful for introductory statistics courses and require 
no knowledge of the R language. All of the Rweb versions can analyze Web 
accessible datasets if aURL is provided. </p> 
<p>The paper &ldquo;Rweb: Web-based Statistical Analysis&rdquo;, providing a 
detailed explanation of the different versions of Rweb and an overview of how 
Rweb works, was published in the Journal of Statistical Software (
http://www.jstatsoft.org/v04/i01/). </p> 
<p>Ulf Bartel has developed <strong>R-Online</strong>, a simple on-line 
programming environment for R which intends to make the first steps in 
statistical programming with R (especially with time series) as easy as 
possible. There is no need for a local installation since the only requirement 
for the user is a JavaScript capable browser. Seehttp://osvisions.com/r-online/ 
for more information.</p> 
<p><strong>Rcgi</strong> is a CGI WWW interface to R by MJ Ray. It had the 
ability to use &ldquo;embedded code&rdquo;: you could mix user input and code, 
allowing theHTML author to do anything from load in data sets to enter most of 
the commands for users without writing CGI scripts. Graphical output was 
possible in PostScript or GIF formats and the executed code was presented to 
the user for revision. However, it is not clear if the project is still active. 
 Currently, a modified version of<strong>Rcgi</strong> by Mai Zhou (actually, 
two versions: one with (bitmap) graphics and one without) as well as the 
original code are available fromhttp://www.ms.uky.edu/~statweb/. </p> 
<p>CGI-based web access to R is also provided at 
http://hermes.sdu.dk/cgi-bin/go/. There are many additional examples of web 
interfaces to R which basically allow to submit R code to a remote server, see 
for example the collection of links available from
http://biostat.mc.vanderbilt.edu/twiki/bin/view/Main/StatCompCourse. </p> 
<p>David Firth has written <strong>CGIwithR</strong>, an R add-on package 
available fromCRAN. It provides some simple extensions to R to facilitate 
running R scripts through the CGI interface to a web server, and allows 
submission of data using both GET and POST methods. It is easily installed 
using Apache under Linux and in principle should run on any platform that 
supports R and a web server provided that the installer has the necessary 
security permissions. David's paper &ldquo;CGIwithR: Facilities for Processing 
Web Forms Using R&rdquo; was published in the Journal of Statistical Software (
http://www.jstatsoft.org/v08/i10/). The package is now maintained by Duncan 
Temple Lang and has a web page at http://www.omegahat.org/CGIwithR/. </p> 
<p>Rpad, developed and actively maintained by Tom Short, provides a 
sophisticated environment which combines some of the features of the previous 
approaches with quite a bit of Javascript, allowing for aGUI-like behavior 
(with sortable tables, clickable graphics, editable output), etc.</p> 
<p>Jeff Horner is working on the R/Apache Integration Project which embeds the 
R interpreter inside Apache 2 (and beyond). A tutorial and presentation are 
available from the project web page at
http://biostat.mc.vanderbilt.edu/twiki/bin/view/Main/RApacheProject. </p> 
<p>Rserve is a project actively developed by Simon Urbanek. It implements a 
TCP/IP server which allows other programs to use facilities of R. Clients are 
available from the web site for Java and C++ (and could be written for other 
languages that support TCP/IP sockets).</p> 
<p>OpenStatServer is being developed by a team lead by Greg Warnes; it aims 
&ldquo;to provide clean access to computational modules defined in a variety of 
computational environments (R, SAS, Matlab, etc) via a single well-defined 
client interface&rdquo; and to turn computational services into web services.
</p> 
<p>Two projects use PHP to provide a web interface to R. R_PHP_Online by Steve 
Chen (though it is unclear if this project is still active) is somewhat similar 
to the above Rcgi and Rweb.R-php is actively developed by Alfredo Pontillo and 
Angelo Mineo and provides both a web interface to R and a set of pre-specified 
analyses that need no R code input.</p> 
<p>webbioc is &ldquo;an integrated web interface for doing microarray analysis 
using several of the Bioconductor packages&rdquo; and is designed to be 
installed at local sites as a shared computing resource.</p> 
<p>Finally, Rwui is a web application to to create user-friendly web 
interfaces for R scripts. All code for the web interface is created 
automatically. There is no need for the user to do any extra scripting or learn 
any new scripting techniques.</p> 
<p></p>  Next:&nbsp; R and Emacs, Previous:&nbsp; R Web Interfaces, Up:&nbsp; 
Top <br>

<h2>5 R Add-On Packages</h2> 
<ul> 
<li>Which add-on packages exist for R? </li> 
<li>How can add-on packages be installed? </li> 
<li>How can add-on packages be used? </li> 
<li>How can add-on packages be removed? </li> 
<li>How can I create an R package? </li> 
<li>How can I contribute to R? </li> </ul> 
<p></p>  Next:&nbsp; How can add-on packages be installed?, Previous:&nbsp; R 
Add-On Packages, Up:&nbsp; R Add-On Packages <br>

<h3>5.1 Which add-on packages exist for R?</h3> 
<ul> 
<li>Add-on packages in R </li> 
<li>Add-on packages from CRAN </li> 
<li>Add-on packages from Omegahat </li> 
<li>Add-on packages from Bioconductor </li> 
<li>Other add-on packages </li> </ul> 
<p></p>  Next:&nbsp; Add-on packages from CRAN, Previous:&nbsp; Which add-on 
packages exist for R?, Up:&nbsp; Which add-on packages exist for R? <br>

<h4>5.1.1 Add-on packages in R</h4> 
<p>The R distribution comes with the following packages: </p> 
<dl> 
<dt> <strong>base</strong> </dt> 
<dd> Base R functions (and datasets before R 2.0.0). <br>
</dd> 
<dt> <strong>datasets</strong> </dt> 
<dd> Base R datasets (added in R 2.0.0). <br>
</dd> 
<dt> <strong>grDevices</strong> </dt> 
<dd> Graphics devices for base and grid graphics (added in R 2.0.0). <br>
</dd>
<dt> <strong>graphics</strong> </dt> 
<dd> R functions for base graphics. <br>
</dd> 
<dt> <strong>grid</strong> </dt> 
<dd> A rewrite of the graphics layout capabilities, plus some support for 
interaction.<br>
</dd> 
<dt> <strong>methods</strong> </dt> 
<dd> Formally defined methods and classes for R objects, plus other 
programming tools, as described in the Green Book.<br>
</dd> 
<dt> <strong>splines</strong> </dt> 
<dd> Regression spline functions and classes. <br>
</dd> 
<dt> <strong>stats</strong> </dt> 
<dd> R statistical functions. <br>
</dd> 
<dt> <strong>stats4</strong> </dt> 
<dd> Statistical functions using S4 classes. <br>
</dd> 
<dt> <strong>tcltk</strong> </dt> 
<dd> Interface and language bindings to Tcl/Tk GUI elements. <br>
</dd> 
<dt> <strong>tools</strong> </dt> 
<dd> Tools for package development and administration. <br>
</dd> 
<dt> <strong>utils</strong> </dt> 
<dd> R utility functions. </dd> </dl> These &ldquo;base packages&rdquo; were 
substantially reorganized in R 1.9.0. The former<strong>base</strong> was split 
into the four packages<strong>base</strong>, <strong>graphics</strong>, <strong>
stats</strong>, and <strong>utils</strong>. Packages <strong>ctest</strong>, 
<strong>eda</strong>, <strong>modreg</strong>, <strong>mva</strong>, <strong>nls
</strong>, <strong>stepfun</strong> and <strong>ts</strong> were merged into 
<strong>stats</strong>, package <strong>lqs</strong> returned to the 
recommended package<strong>MASS</strong>, and package <strong>mle</strong> 
moved to<strong>stats4</strong>. 
<p></p>  Next:&nbsp; Add-on packages from Omegahat, Previous:&nbsp; Add-on 
packages in R, Up:&nbsp; Which add-on packages exist for R? <br>

<h4>5.1.2 Add-on packages from CRAN</h4> 
<p>The following packages are available from the CRAN src/contrib area. 
(Packages denoted as<em>Recommended</em> are to be included in all binary 
distributions of R.)</p> 
<dl> 
<dt> <strong>ADaCGH</strong> </dt> 
<dd> Analysis of data from aCGH experiments. <br>
</dd> 
<dt> <strong>AER</strong> </dt> 
<dd> Functions, data sets, examples and vignettes for the book &ldquo;Applied 
Econometrics with R&rdquo; by Christian Kleiber and Achim Zeileis, 2008, 
Springer-Verlag, New York.<br>
</dd> 
<dt> <strong>AIS</strong> </dt> 
<dd> Tools to look at the data (&ldquo;Ad Inidicia Spectata&rdquo;). <br>
</dd>
<dt> <strong>ALS</strong> </dt> 
<dd> Multivariate curve resolution alternating least squares (MCR-ALS). <br>

</dd> 
<dt> <strong>AMORE</strong> </dt> 
<dd> A MORE flexible neural network package, providing the TAO robust neural 
network algorithm.<br>
</dd> 
<dt> <strong>ARES</strong> </dt> 
<dd> Allelic richness estimation, with extrapolation beyond the sample size. 
<br> </dd> 
<dt> <strong>AcceptanceSampling</strong> </dt> 
<dd> Creation and evaluation of acceptance sampling plans, <br>
</dd> 
<dt> <strong>AdMit</strong> </dt> 
<dd> Adaptive mixture of Student t distributions. <br>
</dd> 
<dt> <strong>AdaptFit</strong> </dt> 
<dd> Adaptive semiparametic regression. <br>
</dd> 
<dt> <strong>AlgDesign</strong> </dt> 
<dd> Algorithmic experimental designs. Calculates exact and approximate theory 
experimental designs for D, A, and I criteria.<br>
</dd> 
<dt> <strong>Amelia</strong> </dt> 
<dd> Amelia II: a program for missing data. <br>
</dd> 
<dt> <strong>AnalyzeFMRI</strong> </dt> 
<dd> Functions for I/O, visualisation and analysis of functional Magnetic 
Resonance Imaging (fMRI) datasets stored in the ANALYZE format.<br>
</dd> 
<dt> <strong>ArDec</strong> </dt> 
<dd> Time series autoregressive decomposition. <br>
</dd> 
<dt> <strong>BACCO</strong> </dt> 
<dd> Bayesian Analysis of Computer Code Output. Contains <strong>approximator
</strong>, <strong>calibrator</strong>, and <strong>emulator</strong>, for 
Bayesian prediction of complex computer codes, calibration of computer models, 
and emulation of computer programs, respectively.<br>
</dd> 
<dt> <strong>BARD</strong> </dt> 
<dd> Better Automated ReDistricting. <br>
</dd> 
<dt> <strong>BAYSTAR</strong> </dt> 
<dd> Bayesian analysis of threshold autoregressive models. <br>
</dd> 
<dt> <strong>BB</strong> </dt> 
<dd> Barzilai-Borwein spectral methods for solving nonlinear system of 
equations, and for optimizing nonlinear objective functions subject to simple 
constraints.<br>
</dd> 
<dt> <strong>BCE</strong> </dt> 
<dd> Bayesian Composition Estimator for sample (taxonomic) composition from 
biomarker data.<br>
</dd> 
<dt> <strong>BHH2</strong> </dt> 
<dd> Functions and data sets reproducing some examples in &ldquo;Statistics 
for Experimenters II&rdquo; by G. E. P. Box, J. S. Hunter, and W. C. Hunter, 
2005, John Wiley and Sons.<br>
</dd> 
<dt> <strong>BLCOP</strong> </dt> 
<dd> Black-Litterman and copula-opinion pooling frameworks. <br>
</dd> 
<dt> <strong>BMA</strong> </dt> 
<dd> Bayesian Model Averaging for linear models, generalizable linear models 
and survival models (Cox regression).<br>
</dd> 
<dt> <strong>BPHO</strong> </dt> 
<dd> Bayesian Prediction with High-order Interactions. <br>
</dd> 
<dt> <strong>BaM</strong> </dt> 
<dd> Functions and datasets for &ldquo;Bayesian Methods: A Social and 
Behavioral Sciences Approach&rdquo; (2nd edition) by Jeff Gill, 2007, CRC Press.
<br> </dd> 
<dt> <strong>BayHaz</strong> </dt> 
<dd> Functions for Bayesian Hazard rate estimation. <br>
</dd> 
<dt> <strong>BayesTree</strong> </dt> 
<dd> Bayesian methods for tree based models. <br>
</dd> 
<dt> <strong>BayesValidate</strong> </dt> 
<dd> Bayesian software validation using posterior quantiles. <br>
</dd> 
<dt> <strong>Bchron</strong> </dt> 
<dd> Create chronologies based on radiocarbon and non-radiocarbon dated depths.
<br> </dd> 
<dt> <strong>Bhat</strong> </dt> 
<dd> Functions for general likelihood exploration (MLE, MCMC, CIs). <br>
</dd> 
<dt> <strong>BiasedUrn</strong> </dt> 
<dd> Biased urn model distributions. <br>
</dd> 
<dt> <strong>BioIDMapper</strong> </dt> 
<dd> Mapping between BioIDs. <br>
</dd> 
<dt> <strong>Biodem</strong> </dt> 
<dd> A number of functions for biodemographycal analysis. <br>
</dd> 
<dt> <strong>BiodiversityR</strong> </dt> 
<dd> GUI for biodiversity and community ecology analysis. <br>
</dd> 
<dt> <strong>BiplotGUI</strong> </dt> 
<dd> Interactive biplots in R. <br>
</dd> 
<dt> <strong>Bolstad</strong> </dt> 
<dd> Functions and data sets for the book &ldquo;Introduction to Bayesian 
Statistics&rdquo; by W. M. Bolstad, 2004, John Wiley and Sons.<br>
</dd> 
<dt> <strong>BootCL</strong> </dt> 
<dd> Bootstrapping test for chromosomal localization. <br>
</dd> 
<dt> <strong>BootPR</strong> </dt> 
<dd> Bootstrap prediction intervals and bias-corrected forecasting. <br>
</dd> 
<dt> <strong>BradleyTerry</strong> </dt> 
<dd> Specify and fit the Bradley-Terry model and structured versions. <br>

</dd> 
<dt> <strong>Brobdingnag</strong> </dt> 
<dd> Very large numbers in R. <br>
</dd> 
<dt> <strong>BSDA</strong> </dt> 
<dd> Data sets for the book &ldquo;Basic Statistics and Data Analysis&rdquo; 
by L. J. Kitchens, 2003, Duxbury.<br>
</dd> 
<dt> <strong>BsMD</strong> </dt> 
<dd> Bayes screening and model discrimination follow-up designs. <br>
</dd> 
<dt> <strong>CADStat</strong> </dt> 
<dd> A GUI to several statistical methods for biological inferences. <br>
</dd>
<dt> <strong>CCA</strong> </dt> 
<dd> Canonical correlation analysis. <br>
</dd> 
<dt> <strong>CDNmoney</strong> </dt> 
<dd> Components of Canadian monetary aggregates. <br>
</dd> 
<dt> <strong>CGIwithR</strong> </dt> 
<dd> Facilities for the use of R to write CGI scripts. <br>
</dd> 
<dt> <strong>CHsharp</strong> </dt> 
<dd> Choi and Hall clustering in 3d. <br>
</dd> 
<dt> <strong>CORREP</strong> </dt> 
<dd> Multivariate correlation estimation. <br>
</dd> 
<dt> <strong>COZIGAM</strong> </dt> 
<dd> Constrained Zero-Inflated Generalized Additive Model. <br>
</dd> 
<dt> <strong>CPE</strong> </dt> 
<dd> Concordance probability estimates in survival analysis. <br>
</dd> 
<dt> <strong>CTFS</strong> </dt> 
<dd> The CTFS large plot forest dynamics analyses. <br>
</dd> 
<dt> <strong>CTT</strong> </dt> 
<dd> Classical Test Theory functions. <br>
</dd> 
<dt> <strong>CVThresh</strong> </dt> 
<dd> Level-dependent Cross-Validation Thresholding. <br>
</dd> 
<dt> <strong>Cairo</strong> </dt> 
<dd> Graphics device using cairographics library for creating high-quality 
PNG, PDF, SVG, PostScript output and interactive display devices such as X11.
<br> </dd> 
<dt> <strong>CarbonEL</strong> </dt> 
<dd> Carbon Event Loop. <br>
</dd> 
<dt> <strong>CellularAutomaton</strong> </dt> 
<dd> One-dimensional cellular automata. <br>
</dd> 
<dt> <strong>ChainLadder</strong> </dt> 
<dd> Mack- and Munich-chain-ladder methods for insurance claims reserving. <br>
</dd> 
<dt> <strong>CircStats</strong> </dt> 
<dd> Circular Statistics, from &ldquo;Topics in Circular Statistics&rdquo; by 
S. Rao Jammalamadaka and A. SenGupta, 2001, World Scientific.<br>
</dd> 
<dt> <strong>CoCo</strong> </dt> 
<dd> Graphical modeling for contingency tables using CoCo. <br>
</dd> 
<dt> <strong>ComPairWise</strong> </dt> 
<dd> Compare phylogenetic or population genetic data alignments. <br>
</dd> 
<dt> <strong>CombMSC</strong> </dt> 
<dd> Combined Model Selection Criteria. <br>
</dd> 
<dt> <strong>CompetingRiskFrailty</strong> </dt> 
<dd> Competing risk model with frailties for right censored survival data. <br>
</dd> 
<dt> <strong>Containers</strong> </dt> 
<dd> Object-oriented data structures including stack, queue, and binary search 
tree.<br>
</dd> 
<dt> <strong>ConvergenceConcepts</strong> </dt> 
<dd> Seeing convergence concepts in action. <br>
</dd> 
<dt> <strong>CoxBoost</strong> </dt> 
<dd> Cox survival models by likelihood based boosting. <br>
</dd> 
<dt> <strong>CreditMetrics</strong> </dt> 
<dd> Functions for calculating the CreditMetrics risk model. <br>
</dd> 
<dt> <strong>CvM2SL2Test</strong> </dt> 
<dd> Cramer-von Mises two sample tests. <br>
</dd> 
<dt> <strong>DAAG</strong> </dt> 
<dd> Various data sets used in examples and exercises in &ldquo;Data Analysis 
and Graphics Using R&rdquo; by John H. Maindonald and W. John Brown, 2003.<br>

</dd> 
<dt> <strong>DAAGbio</strong> </dt> 
<dd> Data sets and functions, for demonstrations with expression arrays. <br>

</dd> 
<dt> <strong>DAAGxtras</strong> </dt> 
<dd> Data sets and functions additional to <strong>DAAG</strong>. <br>
</dd> 
<dt> <strong>DBI</strong> </dt> 
<dd> A common database interface (DBI) class and method definitions. All 
classes in this package are virtual and need to be extended by the various DBMS 
implementations.<br>
</dd> 
<dt> <strong>DCluster</strong> </dt> 
<dd> A set of functions for the detection of spatial clusters of diseases 
using count data.<br>
</dd> 
<dt> <strong>DDHFm</strong> </dt> 
<dd> Variance stabilization by Data-Driven Haar-Fisz (for microarrays). <br>

</dd> 
<dt> <strong>DEA</strong> </dt> 
<dd> Data Envelopment Analysis. <br>
</dd> 
<dt> <strong>DEoptim</strong> </dt> 
<dd> Differential Evolution Optimization. <br>
</dd> 
<dt> <strong>DICOM</strong> </dt> 
<dd> Import and manipulate medical imaging data using the Digital Imaging and 
Communications in Medicine (DICOM) Standard.<br>
</dd> 
<dt> <strong>DPpackage</strong> </dt> 
<dd> Semiparametric Bayesian analysis using Dirichlet process priors. <br>

</dd> 
<dt> <strong>Davies</strong> </dt> 
<dd> Functions for the Davies quantile function and the Generalized Lambda 
distribution.<br>
</dd> 
<dt> <strong>Defaults</strong> </dt> 
<dd> Create global function defaults. <br>
</dd> 
<dt> <strong>DescribeDisplay</strong> </dt> 
<dd> R interface to the DescribeDisplay GGobi plugin. <br>
</dd> 
<dt> <strong>Design</strong> </dt> 
<dd> Regression modeling, testing, estimation, validation, graphics, 
prediction, and typesetting by storing enhanced model design attributes in the 
fit. Design is a collection of about 180 functions that assist and streamline 
modeling, especially for biostatistical and epidemiologic applications. It also 
contains new functions for binary and ordinal logistic regression models and 
the Buckley-James multiple regression model for right-censored responses, and 
implements penalized maximum likelihood estimation for logistic and ordinary 
linear models. Design works with almost any regression model, but it was 
especially written to work with logistic regression, Cox regression, 
accelerated failure time models, ordinary linear models, and the Buckley-James 
model.<br>
</dd> 
<dt> <strong>Devore5</strong> </dt> 
<dd> Data sets and sample analyses from &ldquo;Probability and Statistics for 
Engineering and the Sciences (5th ed)&rdquo; by Jay L. Devore, 2000, Duxbury.
<br> </dd> 
<dt> <strong>Devore6</strong> </dt> 
<dd> Data sets and sample analyses from &ldquo;Probability and Statistics for 
Engineering and the Sciences (6th ed)&rdquo; by Jay L. Devore, 2003, Duxbury.
<br> </dd> 
<dt> <strong>Devore7</strong> </dt> 
<dd> Data sets and sample analyses from &ldquo;Probability and Statistics for 
Engineering and the Sciences (7th ed)&rdquo; by Jay L. Devore, 2008, Thomson.
<br> </dd> 
<dt> <strong>DiagnosisMed</strong> </dt> 
<dd> Diagnostic test accuracy evaluation for medical professionals. <br>
</dd> 
<dt> <strong>DierckxSpline</strong> </dt> 
<dd> R companion to &ldquo;Curve and Surface Fitting with Splines&rdquo; by 
Paul Dierckx, 1993, Oxford University Press.<br>
</dd> 
<dt> <strong>EDR</strong> </dt> 
<dd> Estimation of the effective dimension reduction (EDR) space. <br>
</dd> 
<dt> <strong>EMC</strong> </dt> 
<dd> Evolutionary Monte Carlo (EMC) algorithm. <br>
</dd> 
<dt> <strong>EMCC</strong> </dt> 
<dd> Evolutionary Monte Carlo (EMC) methods for clustering. <br>
</dd> 
<dt> <strong>EMD</strong> </dt> 
<dd> Empirical mode decomposition and Hilbert spectral analysis. <br>
</dd> 
<dt> <strong>EMV</strong> </dt> 
<dd> Estimation of missing values in a matrix by a k-th nearest neighboors 
algorithm.<br>
</dd> 
<dt> <strong>ETC</strong> </dt> 
<dd> Tests and simultaneous confidence intervals for equivalence to control. 
<br> </dd> 
<dt> <strong>EVER</strong> </dt> 
<dd> Estimation of Variance by Efficient Replication. <br>
</dd> 
<dt> <strong>EbayesThresh</strong> </dt> 
<dd> Empirical Bayes thresholding and related methods. <br>
</dd> 
<dt> <strong>Ecdat</strong> </dt> 
<dd> Data sets from econometrics textbooks. <br>
</dd> 
<dt> <strong>EffectiveDose</strong> </dt> 
<dd> Estimate the effective dose level for quantal bioassay data by 
nonparametric techniques.<br>
</dd> 
<dt> <strong>ElemStatLearn</strong> </dt> 
<dd> Data sets, functions and examples from the book &ldquo;The Elements of 
Statistical Learning: Data Mining, Inference, and Prediction&rdquo; by Trevor 
Hastie, Robert Tibshirani and Jerome Friedman (2001), Springer.<br>
</dd> 
<dt> <strong>Epi</strong> </dt> 
<dd> Statistical analysis in epidemiology, with functions for demographic and 
epidemiological analysis in the Lexis diagram.<br>
</dd> 
<dt> <strong>FAiR</strong> </dt> 
<dd> Factor Analysis in R, using genetic algorithms. <br>
</dd> 
<dt> <strong>FGN</strong> </dt> 
<dd> Fractional Gaussian Noise model fitting. <br>
</dd> 
<dt> <strong>FITSio</strong> </dt> 
<dd> FITS (Flexible Image Transport System) utilities. <br>
</dd> 
<dt> <strong>FKBL</strong> </dt> 
<dd> Fuzzy Knowledge Base Learning. <br>
</dd> 
<dt> <strong>FTICRMS</strong> </dt> 
<dd> Analysis of Fourier Transform-Ion Cyclotron Resonance Mass Spectrometry 
data.<br>
</dd> 
<dt> <strong>FactoMineR</strong> </dt> 
<dd> Factor analysis and data mining with R. <br>
</dd> 
<dt> <strong>Fahrmeir</strong> </dt> 
<dd> Data from the book &ldquo;Multivariate Statistical Modelling Based on 
Generalized Linear Models&rdquo; by Ludwig Fahrmeir and Gerhard Tutz (1994), 
Springer.<br>
</dd> 
<dt> <strong>FieldSim</strong> </dt> 
<dd> Random fields simulations. <br>
</dd> 
<dt> <strong>FinTS</strong> </dt> 
<dd> Companion to the book &ldquo;Analysis of Financial Time Series&rdquo; 
(2nd edition) by Ruey Tsay (2005), Wiley.<br>
</dd> 
<dt> <strong>FitAR</strong> </dt> 
<dd> Subset AR model fitting. <br>
</dd> 
<dt> <strong>Flury</strong> </dt> 
<dd> Data sets from from &ldquo;A First Course in Multivariate 
Statistics&rdquo; by Bernard Flury (1997), Springer. <br>
</dd> 
<dt> <strong>FrF2</strong> </dt> 
<dd> Analysis of fractional factorial designs with 2-level factors. <br>
</dd> 
<dt> <strong>FracSim</strong> </dt> 
<dd> Simulation of one- and two-dimensional fractional and multifractional 
Levy motions.<br>
</dd> 
<dt> <strong>FunCluster</strong> </dt> 
<dd> Functional profiling of cDNA microarray expression data. <br>
</dd> 
<dt> <strong>FunNet</strong> </dt> 
<dd> Functional analysis of gene co-expression networks. <br>
</dd> 
<dt> <strong>G1DBN</strong> </dt> 
<dd> Dynamic Bayesian Network inference using 1st order conditional 
dependencies.<br>
</dd> 
<dt> <strong>GAMBoost</strong> </dt> 
<dd> Generalized additive models by likelihood based boosting. <br>
</dd> 
<dt> <strong>GDD</strong> </dt> 
<dd> Platform and X11 independent device for creating bitmaps (png, gif and 
jpeg) using the GD graphics library.<br>
</dd> 
<dt> <strong>GEOmap</strong> </dt> 
<dd> Topographic and geologic mapping. <br>
</dd> 
<dt> <strong>GExMap</strong> </dt> 
<dd> Functions for the analysis of genomic distribution of genes lists 
produced by transcriptomic studies.<br>
</dd> 
<dt> <strong>GLDEX</strong> </dt> 
<dd> Fit RS and FMKL generalised lambda distributions using discretized and 
maximum likelihood methods.<br>
</dd> 
<dt> <strong>GOSim</strong> </dt> 
<dd> Computation of functional similarities between GO terms and gene products.
<br> </dd> 
<dt> <strong>GPArotation</strong> </dt> 
<dd> Gradient Projection Algorithm rotation for factor analysis. <br>
</dd> 
<dt> <strong>GRASS</strong> </dt> 
<dd> An interface between the GRASS geographical information system and R, 
based on starting R from within the GRASS environment and chosen LOCATION_NAME 
and MAPSET. Wrapper and helper functions are provided for a range of R 
functions to match the interface metadata structures.<br>
</dd> 
<dt> <strong>GSA</strong> </dt> 
<dd> Gene set analysis. <br>
</dd> 
<dt> <strong>GSM</strong> </dt> 
<dd> Gamma Shape Mixture. <br>
</dd> 
<dt> <strong>GenABEL</strong> </dt> 
<dd> Genome-wide SNP association analysis. <br>
</dd> 
<dt> <strong>GenKern</strong> </dt> 
<dd> Functions for generating and manipulating generalised binned kernel 
density estimates.<br>
</dd> 
<dt> <strong>GeneCycle</strong> </dt> 
<dd> Identification of periodically expressed genes. <br>
</dd> 
<dt> <strong>GeneF</strong> </dt> 
<dd> Generalized F-statistics. <br>
</dd> 
<dt> <strong>GeneNT</strong> </dt> 
<dd> Relevance or Dependency network and signaling pathway discovery. <br>

</dd> 
<dt> <strong>GeneNet</strong> </dt> 
<dd> Modeling and inferring gene networks. <br>
</dd> 
<dt> <strong>GeneTS</strong> </dt> 
<dd> A package for analysing multiple gene expression time series data. 
Currently, implements methods for cell cycle analysis and for inferring large 
sparse graphical Gaussian models.<br>
</dd> 
<dt> <strong>Geneland</strong> </dt> 
<dd> MCMC inference from individual genetic data based on a spatial 
statistical model.<br>
</dd> 
<dt> <strong>GeoXp</strong> </dt> 
<dd> Interactive exploratory spatial data analysis. <br>
</dd> 
<dt> <strong>GillespieSSA</strong> </dt> 
<dd> Gillespie's Stochastic Simulation Algorithm (SSA). <br>
</dd> 
<dt> <strong>GroupSeq</strong> </dt> 
<dd> Computations related to group-seqential boundaries. <br>
</dd> 
<dt> <strong>HAPim</strong> </dt> 
<dd> Methods for QTL detection and fine mapping. <br>
</dd> 
<dt> <strong>HH</strong> </dt> 
<dd> Support software for &ldquo;Statistical Analysis and Data Display&rdquo; 
by Richard M. Heiberger and Burt Holland, Springer, 2005.<br>
</dd> 
<dt> <strong>HI</strong> </dt> 
<dd> Simulation from distributions supported by nested hyperplanes. <br>
</dd> 
<dt> <strong>HSAUR</strong> </dt> 
<dd> Functions, data sets, analyses and examples from the book &ldquo;A 
Handbook of Statistical Analyses Using R&rdquo; by Brian S. Everitt and Torsten 
Hothorn (2006), Chapman &amp; Hall/CRC.<br>
</dd> 
<dt> <strong>HTMLapplets</strong> </dt> 
<dd> Functions inserting dynamic scatterplots and grids in documents generated 
by<strong>R2HTML</strong>. <br>
</dd> 
<dt> <strong>HFWutils</strong> </dt> 
<dd> Utilities by H. Felix Wittmann: Excel connections, string matching, and 
passing by reference.<br>
</dd> 
<dt> <strong>HardyWeinberg</strong> </dt> 
<dd> Graphical tests for Hardy-Weinberg equilibrium. <br>
</dd> 
<dt> <strong>HiddenMarkov</strong> </dt> 
<dd> Hidden Markov Models. <br>
</dd> 
<dt> <strong>Hmisc</strong> </dt> 
<dd> Functions useful for data analysis, high-level graphics, utility 
operations, functions for computing sample size and power, importing datasets, 
imputing missing values, advanced table making, variable clustering, character 
string manipulation, conversion of S objects to LaTeX code, recoding variables, 
and bootstrap repeated measures analysis.<br>
</dd> 
<dt> <strong>HydroMe</strong> </dt> 
<dd> Estimation of soil hydraulic parameters from experimental data. <br>
</dd>
<dt> <strong>HyperbolicDist</strong> </dt> 
<dd> Basic functions for the hyperbolic distribution: probability density 
function, distribution function, quantile function, a routine for generating 
observations from the hyperbolic, and a function for fitting the hyperbolic 
distribution to data.<br>
</dd> 
<dt> <strong>IBrokers</strong> </dt> 
<dd> R API to Interactive Brokers Trader Workstation. <br>
</dd> 
<dt> <strong>ICE</strong> </dt> 
<dd> Iterated Conditional Expectation: kernel estimators for interval-censored 
data.<br>
</dd> 
<dt> <strong>ICEinfer</strong> </dt> 
<dd> Incremental Cost-Effectiveness (ICE) statistical inference (from two 
unbiased samples).<br>
</dd> 
<dt> <strong>ICS</strong> </dt> 
<dd> ICS/ICA computation based on two scatter matrices. <br>
</dd> 
<dt> <strong>ICSNP</strong> </dt> 
<dd> Tools for multivariate nonparametrics. <br>
</dd> 
<dt> <strong>IDPmisc</strong> </dt> 
<dd> Utilities from the Institute of Data Analyses and Process Design, IDP/ZHW.
<br> </dd> 
<dt> <strong>ISA</strong> </dt> 
<dd> Functions to support &ldquo;Introduzione alla Statistica Applicata con 
esempi in R&rdquo; by Federico M. Stefanini, Pearson Education Milano, 2007.<br>
</dd> 
<dt> <strong>ISOcodes</strong> </dt> 
<dd> ISO language, territory, currency, script and character codes. <br>
</dd> 
<dt> <strong>ISwR</strong> </dt> 
<dd> Data sets for &ldquo;Introductory Statistics with R&rdquo; by Peter 
Dalgaard, 2002, Springer.<br>
</dd> 
<dt> <strong>Icens</strong> </dt> 
<dd> Functions for computing the NPMLE for censored and truncated data. <br>

</dd> 
<dt> <strong>Iso</strong> </dt> 
<dd> Functions to perform isotonic regression. <br>
</dd> 
<dt> <strong>JADE</strong> </dt> 
<dd> JADE and ICA performance criteria. <br>
</dd> 
<dt> <strong>JGR</strong> </dt> 
<dd> Java Gui for R. <br>
</dd> 
<dt> <strong>JM</strong> </dt> 
<dd> Joint Modeling of longitudinal and survival data. <br>
</dd> 
<dt> <strong>JavaGD</strong> </dt> 
<dd> Java Graphics Device. <br>
</dd> 
<dt> <strong>JointGLM</strong> </dt> 
<dd> Joint modeling of mean and dispersion through two interlinked GLM's. <em>
Defunct</em> in favor of <strong>JointModeling</strong>. <br>
</dd> 
<dt> <strong>JointModeling</strong> </dt> 
<dd> Joint modeling of mean and dispersion. <br>
</dd> 
<dt> <strong>JudgeIt</strong> </dt> 
<dd> Calculates bias, responsiveness, and other characteristics of two-party 
electoral systems, with district-level electoral and other data.<br>
</dd> 
<dt> <strong>KMsurv</strong> </dt> 
<dd> Data sets and functions for &ldquo;Survival Analysis, Techniques for 
Censored and Truncated Data&rdquo; by Klein and Moeschberger, 1997, Springer.
<br> </dd> 
<dt> <strong>Kendall</strong> </dt> 
<dd> Kendall rank correlation and Mann-Kendall trend test. <br>
</dd> 
<dt> <strong>KernSmooth</strong> </dt> 
<dd> Functions for kernel smoothing (and density estimation) corresponding to 
the book &ldquo;Kernel Smoothing&rdquo; by M. P. Wand and M. C. Jones, 1995.<em>
Recommended</em>. <br>
</dd> 
<dt> <strong>LDheatmap</strong> </dt> 
<dd> Heat maps of linkage disequilibrium measures. <br>
</dd> 
<dt> <strong>LDtests</strong> </dt> 
<dd> Exact tests for linkage disequilibrium and Hardy-Weinberg equilibrium. 
<br> </dd> 
<dt> <strong>LIStest</strong> </dt> 
<dd> Longest increasing subsequence independence test. <br>
</dd> 
<dt> <strong>LLAhclust</strong> </dt> 
<dd> Hierarchical clustering of variables or objects based on the likelihood 
linkage analysis method.<br>
</dd> 
<dt> <strong>LLN</strong> </dt> 
<dd> Learning with latent networks. <br>
</dd> 
<dt> <strong>LMGene</strong> </dt> 
<dd> Date transformation and identification of differentially expressed genes 
in gene expression arrays.<br>
</dd> 
<dt> <strong>LearnBayes</strong> </dt> 
<dd> Functions for Learning Bayesian Inference. <br>
</dd> 
<dt> <strong>Lmoments</strong> </dt> 
<dd> Estimation of L-moments and the parameters of normal and Cauchy 
polynomial quantile mixtures.<br>
</dd> 
<dt> <strong>LogConcDEAD</strong> </dt> 
<dd> Maximum likelihood estimation of a log-concave density. <br>
</dd> 
<dt> <strong>LogicReg</strong> </dt> 
<dd> Routines for Logic Regression. <br>
</dd> 
<dt> <strong>LoopAnalyst</strong> </dt> 
<dd> A collection of tools to conduct Levins' Loop Analysis. <br>
</dd> 
<dt> <strong>LowRankQP</strong> </dt> 
<dd> Low Rank Quadratic Programming: QP problems where the hessian is 
represented as the product of two matrices.<br>
</dd> 
<dt> <strong>MASS</strong> </dt> 
<dd> Functions and datasets from the main package of Venables and Ripley, 
&ldquo;Modern Applied Statistics with S&rdquo;. Contained in theVR bundle. <em>
Recommended</em>. <br>
</dd> 
<dt> <strong>MBA</strong> </dt> 
<dd> Multilevel B-spline Approximation. <br>
</dd> 
<dt> <strong>MBESS</strong> </dt> 
<dd> Methods for the Behavioral, Educational, and Social Sciences. <br>
</dd> 
<dt> <strong>MCAPS</strong> </dt> 
<dd> Weather and air pollution data, risk estimates, and other information 
from the Medicare Air Pollution Study (MCAPS) of 204 U.S. counties, 
1999&ndash;2002.<br>
</dd> 
<dt> <strong>MCMCpack</strong> </dt> 
<dd> Markov chain Monte Carlo (MCMC) package: functions for posterior 
simulation for a number of statistical models.<br>
</dd> 
<dt> <strong>MCPAN</strong> </dt> 
<dd> Multiple comparisons using normal approximation. <br>
</dd> 
<dt> <strong>MCPMod</strong> </dt> 
<dd> Design and analysis of dose-finding studies. <br>
</dd> 
<dt> <strong>MChtest</strong> </dt> 
<dd> Monte Carlo hypothesis tests. <br>
</dd> 
<dt> <strong>MDD</strong> </dt> 
<dd> Minimum Detectable Difference calculations for several continuous and 
binary endpoints.<br>
</dd> 
<dt> <strong>MEMSS</strong> </dt> 
<dd> Data sets and sample analyses from &ldquo;Mixed-effects Models in S and 
S-PLUS&rdquo; by J. Pinheiro and D. Bates, 2000, Springer.<br>
</dd> 
<dt> <strong>MFDA</strong> </dt> 
<dd> Model Based Functional Data Analysis. <br>
</dd> 
<dt> <strong>MIfuns</strong> </dt> 
<dd> Pharmacometric tools for data preparation, analysis, simulation, and 
reporting.<br>
</dd> 
<dt> <strong>MKLE</strong> </dt> 
<dd> Maximum kernel likelihood estimation. <br>
</dd> 
<dt> <strong>MKmisc</strong> </dt> 
<dd> Miscellaneous Functions from M. Kohl. <br>
</dd> 
<dt> <strong>MLDA</strong> </dt> 
<dd> Methylation Linear Discriminant Analysis (MLDA). <br>
</dd> 
<dt> <strong>MLDS</strong> </dt> 
<dd> Maximum Likelihood Difference Scaling. <br>
</dd> 
<dt> <strong>MLEcens</strong> </dt> 
<dd> Computation of the MLE for bivariate (interval) censored data. <br>
</dd> 
<dt> <strong>MMG</strong> </dt> 
<dd> Mixture Model on Graphs. <br>
</dd> 
<dt> <strong>MNP</strong> </dt> 
<dd> Fitting Bayesian Multinomial Probit models via Markov chain Monte Carlo. 
Along with the standard Multinomial Probit model, it can also fit models with 
different choice sets for each observation and complete or partial ordering of 
all the available alternatives.<br>
</dd> 
<dt> <strong>MPV</strong> </dt> 
<dd> Data sets from the book &ldquo;Introduction to Linear Regression 
Analysis&rdquo; by D. C. Montgomery, E. A. Peck, and C. G. Vining, 2001, John 
Wiley and Sons.<br>
</dd> 
<dt> <strong>MSBVAR</strong> </dt> 
<dd> Bayesian vector autoregression models, impulse responses and forecasting. 
<br> </dd> 
<dt> <strong>MSVAR</strong> </dt> 
<dd> Markov Switching VAR. <br>
</dd> 
<dt> <strong>MarkedPointProcess</strong> </dt> 
<dd> Non-parametric analysis of the marks of marked point processes. <br>
</dd>
<dt> <strong>MasterBayes</strong> </dt> 
<dd> Maximum likelihood and Markov chain Monte Carlo methods for pedigree 
reconstruction, analysis and simulation.<br>
</dd> 
<dt> <strong>MatchIt</strong> </dt> 
<dd> Select matched samples of the original treated and control groups with 
similar covariate distributions.<br>
</dd> 
<dt> <strong>Matching</strong> </dt> 
<dd> Multivariate and propensity score matching with formal tests of balance. 
<br> </dd> 
<dt> <strong>Matrix</strong> </dt> 
<dd> A Matrix package. <br>
</dd> 
<dt> <strong>Metabonomic</strong> </dt> 
<dd> GUI for metabonomic analysis. <br>
</dd> 
<dt> <strong>MiscPsycho</strong> </dt> 
<dd> Miscellaneous Psychometrics. <br>
</dd> 
<dt> <strong>MultEq</strong> </dt> 
<dd> Equivalence tests and simultaneous confidence intervals for multiple 
endpoints.<br>
</dd> 
<dt> <strong>NADA</strong> </dt> 
<dd> Methods described in &ldquo;Nondetects And Data Analysis: Statistics for 
Censored Environmental Data&rdquo; by Dennis R. Helsel, 2004, John Wiley and 
Sons.<br>
</dd> 
<dt> <strong>NISTnls</strong> </dt> 
<dd> A set of test nonlinear least squares examples from NIST, the U.S. 
National Institute for Standards and Technology.<br>
</dd> 
<dt> <strong>NMMAPSlite</strong> </dt> 
<dd> U.S. National Morbidity, Mortality, and Air Pollution Study data lite. 
<br> </dd> 
<dt> <strong>NORMT3</strong> </dt> 
<dd> Evaluates complex erf, erfc and density of sum of Gaussian and Student's 
t.<br>
</dd> 
<dt> <strong>NRAIA</strong> </dt> 
<dd> Data sets with sample code from &ldquo;Nonlinear Regression Analysis and 
Its Applications&rdquo; by Doug Bates and Donald Watts, 1988, Wiley.<br>
</dd> 
<dt> <strong>NestedCohort</strong> </dt> 
<dd> Survival analysis for cohorts with missing covariate information. <br>

</dd> 
<dt> <strong>NetIndices</strong> </dt> 
<dd> Estimates network indices, including trophic structure of foodwebs. <br>

</dd> 
<dt> <strong>OPE</strong> </dt> 
<dd> Fit an outer-product emlator to the multivariate evaluations of a 
computer model.<br>
</dd> 
<dt> <strong>ORMDR</strong> </dt> 
<dd> Odds ratio based multivactor-dimensionality reduction method for 
detecting gene-gene interactions.<br>
</dd> 
<dt> <strong>Oarray</strong> </dt> 
<dd> Arrays with arbitrary offsets. <br>
</dd> 
<dt> <strong>PARccs</strong> </dt> 
<dd> Estimation of partial attributable risks (PAR) from case-control data. 
<br> </dd> 
<dt> <strong>PASWR</strong> </dt> 
<dd> Data and functions for the book &ldquo;Probability and Statistics with 
R&rdquo; by M. D. Ugarte, A. F. Militino and A. T. Arnholt, 2008, Chapman &amp; 
Hall/CRC.<br>
</dd> 
<dt> <strong>PBSmapping</strong> </dt> 
<dd> Software evolved from fisheries research conducted at the Pacific 
Biological Station (PBS) in Nanaimo, British Columbia, Canada. Draws maps and 
implements other GIS procedures.<br>
</dd> 
<dt> <strong>PBSmodelling</strong> </dt> 
<dd> Software to facilitate the design, testing, and operation of computer 
models.<br>
</dd> 
<dt> <strong>PET</strong> </dt> 
<dd> Simulation and reconstruction of PET images. <br>
</dd> 
<dt> <strong>PHYLOGR</strong> </dt> 
<dd> Manipulation and analysis of phylogenetically simulated data sets (as 
obtained from PDSIMUL in package PDAP) and phylogenetically-based analyses 
using GLS.<br>
</dd> 
<dt> <strong>PK</strong> </dt> 
<dd> Estimation of pharmacokinetic parameters. <br>
</dd> 
<dt> <strong>PKfit</strong> </dt> 
<dd> A nonlinear regression (including a genetic algorithm) program designed 
to deal with curve fitting for pharmacokinetics.<br>
</dd> 
<dt></dt> 
<dd> Unified computational interfaces for pop PK. <br>
</dd> 
<dt> <strong>POT</strong> </dt> 
<dd> Generalized Pareto distribution and Peaks Over Threshold. <br>
</dd> 
<dt> <strong>PSAgraphics</strong> </dt> 
<dd> Propensity Score Analysis Graphics. <br>
</dd> 
<dt> <strong>PSM</strong> </dt> 
<dd> Non-linear mixed-effects modeling using stochastic differential equations.
<br> </dd> 
<dt> <strong>PTAk</strong> </dt> 
<dd> A multiway method to decompose a tensor (array) of any order, as a 
generalisation of SVD also supporting non-identity metrics and penalisations. 
Also includes some other multiway methods.<br>
</dd> 
<dt> <strong>Peaks</strong> </dt> 
<dd> Spectrum manipulation: background estimation, Markov smoothing, 
deconvolution and peaks search functions.<br>
</dd> 
<dt> <strong>PearsonICA</strong> </dt> 
<dd> Independent component analysis using score functions from the Pearson 
system.<br>
</dd> 
<dt> <strong>PerformanceAnalytics</strong> </dt> 
<dd> Econometric tools for performance and risk analysis. <br>
</dd> 
<dt> <strong>PhySim</strong> </dt> 
<dd> Phylogenetic tree simulation. <br>
</dd> 
<dt> <strong>PolynomF</strong> </dt> 
<dd> Univariate polynomials. <br>
</dd> 
<dt> <strong>Pomic</strong> </dt> 
<dd> Pattern oriented modeling information criterion. <br>
</dd> 
<dt> <strong>PredictiveRegression</strong> </dt> 
<dd> Prediction intervals for three basic statistical models. <br>
</dd> 
<dt> <strong>PresenceAbsence</strong> </dt> 
<dd> Presence-absence model evaluation. <br>
</dd> 
<dt> <strong>ProbForecastGOP</strong> </dt> 
<dd> Probabilistic weather field forecasts using the Geostatistical Output 
Perturbation method introduced by Gel, Raftery and Gneiting (2004).<br>
</dd> 
<dt> <strong>ProbeR</strong> </dt> 
<dd> Reliability for gene expression from Affymetrix chip. <br>
</dd> 
<dt> <strong>ProfessR</strong> </dt> 
<dd> Programs to determine student grades and create examinations from 
question banks.<br>
</dd> 
<dt> <strong>PtProcess</strong> </dt> 
<dd> Time dependent point process modeling. <br>
</dd> 
<dt> <strong>PwrGSD</strong> </dt> 
<dd> Power in a Group Sequential Design. <br>
</dd> 
<dt> <strong>QCA</strong> </dt> 
<dd> Qualitative Comparative Analysis for crisp sets. <br>
</dd> 
<dt> <strong>QCAGUI</strong> </dt> 
<dd> QCA Graphical User Interface. <br>
</dd> 
<dt> <strong>QRMlib</strong> </dt> 
<dd> Code to examine Quantitative Risk Management concepts. <br>
</dd> 
<dt> <strong>QuantPsyc</strong> </dt> 
<dd> Quantitative Psychology tools. <br>
</dd> 
<dt> <strong>R.cache</strong> </dt> 
<dd> Fast and light-weight caching of objects. <br>
</dd> 
<dt> <strong>R.huge</strong> </dt> 
<dd> Methods for accessing huge amounts of data. <br>
</dd> 
<dt> <strong>R.matlab</strong> </dt> 
<dd> Read and write of MAT files together with R-to-Matlab connectivity. <br>

</dd> 
<dt> <strong>R.methodsS3</strong> </dt> 
<dd> Utility functions for defining S3 methods. <br>
</dd> 
<dt> <strong>R.oo</strong> </dt> 
<dd> R object-oriented programming with or without references. <br>
</dd> 
<dt> <strong>R.rsp</strong> </dt> 
<dd> R server pages. <br>
</dd> 
<dt> <strong>R.utils</strong> </dt> 
<dd> Utility classes and methods useful when programming in R and developing R 
packages.<br>
</dd> 
<dt> <strong>R2HTML</strong> </dt> 
<dd> Functions for exporting R objects &amp; graphics in an HTML document. <br>
</dd> 
<dt> <strong>R2WinBUGS</strong> </dt> 
<dd> Running WinBUGS from R: call a BUGS model, summarize inferences and 
convergence in a table and graph, and save the simulations in arrays for easy 
access in R.<br>
</dd> 
<dt> <strong>RArcInfo</strong> </dt> 
<dd> Functions to import Arc/Info V7.x coverages and data. <br>
</dd> 
<dt> <strong>RBGL</strong> </dt> 
<dd> Interface to the boost C++ graph library. <br>
</dd> 
<dt> <strong>RBloomberg</strong> </dt> 
<dd> Fetch data from a Bloomberg API using COM. <br>
</dd> 
<dt> <strong>RColorBrewer</strong> </dt> 
<dd> ColorBrewer palettes for drawing nice maps shaded according to a variable.
<br> </dd> 
<dt> <strong>RDieHarder</strong> </dt> 
<dd> R interface to the dieharder random number generator test suite. <br>

</dd> 
<dt> <strong>RExcelInstaller</strong> </dt> 
<dd> Integration of R and Excel under MS Windows. <br>
</dd> 
<dt> <strong>RFA</strong> </dt> 
<dd> Regional Frequency Analysis. <br>
</dd> 
<dt> <strong>RFOC</strong> </dt> 
<dd> Graphics for spherical distributions and earthquake focal mechanisms. <br>
</dd> 
<dt> <strong>RFreak</strong> </dt> 
<dd> An R interface to a modified version of the Free Evolutionary Algorithm 
Kit FrEAK.g<br>
</dd> 
<dt> <strong>RGtk2</strong> </dt> 
<dd> Facilities for programming graphical interfaces using Gtk (the Gimp Tool 
Kit) version 2. <br>
</dd> 
<dt> <strong>RGrace</strong> </dt> 
<dd> Mouse/menu driven interactive plotting application. <br>
</dd> 
<dt> <strong>RGraphics</strong> </dt> 
<dd> Data and functions from the book &ldquo;R Graphics&rdquo; by Paul 
Murrell, 2005, Chapman &amp; Hall/CRC.<br>
</dd> 
<dt> <strong>RHmm</strong> </dt> 
<dd> Hidden Markov Model simulations and estimations. <br>
</dd> 
<dt> <strong>RII</strong> </dt> 
<dd> Estimation of the relative index of inequality for interval-censored data 
using natural cubic splines.<br>
</dd> 
<dt> <strong>RJDBC</strong> </dt> 
<dd> Access to databases through the JDBC interface. <br>
</dd> 
<dt> <strong>RJaCGH</strong> </dt> 
<dd> Reversible Jump MCMC for the analysis of CGH arrays. <br>
</dd> 
<dt> <strong>RKEA</strong> </dt> 
<dd> R/KEA interface for extracting keyphrases from text documents. <br>
</dd> 
<dt> <strong>RLMM</strong> </dt> 
<dd> A genotype calling algorithm for Affymetrix SNP arrays. <br>
</dd> 
<dt> <strong>RLRsim</strong> </dt> 
<dd> Exact (Restricted) Likelihood Ratio tests for mixed and additive models. 
<br> </dd> 
<dt> <strong>RLadyBug</strong> </dt> 
<dd> Analysis of infectious diseases using stochastic epidemic models. <br>

</dd> 
<dt> <strong>RM2</strong> </dt> 
<dd> Revenue management and pricing. <br>
</dd> 
<dt> <strong>RMySQL</strong> </dt> 
<dd> An interface between R and the MySQL database system. <br>
</dd> 
<dt> <strong>RNetCDF</strong> </dt> 
<dd> An interface to Unidata's NetCDF library functions (version 3) and 
furthermore access to Unidata's udunits calendar conversions.<br>
</dd> 
<dt> <strong>ROCR</strong> </dt> 
<dd> Visualizing the performance of scoring classifiers. <br>
</dd> 
<dt> <strong>RODBC</strong> </dt> 
<dd> An ODBC database interface. <br>
</dd> 
<dt> <strong>ROptEst</strong> </dt> 
<dd> Optimally robust estimation. <br>
</dd> 
<dt> <strong>ROptEstTS</strong> </dt> 
<dd> Optimally robust estimation for regression-type models. <br>
</dd> 
<dt> <strong>ROracle</strong> </dt> 
<dd> Oracle Database Interface driver for R. Uses the ProC/C++ embedded SQL. 
<br> </dd> 
<dt> <strong>RPMG</strong> </dt> 
<dd> Poor Man's Gui: create interactive R analysis sessions. <br>
</dd> 
<dt> <strong>RPyGeo</strong> </dt> 
<dd> ArcGIS Geoprocessing in R via Python. <br>
</dd> 
<dt> <strong>RQuantLib</strong> </dt> 
<dd> Provides access to (some) of the QuantLib functions from within R; 
currently limited to some Option pricing and analysis functions. The QuantLib 
project aims to provide a comprehensive software framework for quantitative 
finance.<br>
</dd> 
<dt> <strong>RSAGA</strong> </dt> 
<dd> SAGA geoprocessing and terrain analysis in R. <br>
</dd> 
<dt> <strong>RSEIS</strong> </dt> 
<dd> Seismic time series analysis tools. <br>
</dd> 
<dt> <strong>RSQLite</strong> </dt> 
<dd> Database Interface R driver for SQLite. Embeds the SQLite database engine 
in R.<br>
</dd> 
<dt> <strong>RScaLAPACK</strong> </dt> 
<dd> An interface to ScaLAPACK functions from R. <br>
</dd> 
<dt> <strong>RSVGTipsDevice</strong> </dt> 
<dd> An R SVG graphics device with dynamic tips and hyperlinks. <br>
</dd> 
<dt> <strong>RSeqMeth</strong> </dt> 
<dd> analysis of Sequenom EpiTYPER data. <br>
</dd> 
<dt> <strong>RSvgDevice</strong> </dt> 
<dd> A graphics device for R that uses the new w3.org XML standard for 
Scalable Vector Graphics.<br>
</dd> 
<dt> <strong>RTOMO</strong> </dt> 
<dd> Visualization for seismic tomography. <br>
</dd> 
<dt> <strong>RTisean</strong> </dt> 
<dd> R interface to Tisean algorithms. <br>
</dd> 
<dt> <strong>RUnit</strong> </dt> 
<dd> Functions implementing a standard Unit Testing framework, with additional 
code inspection and report generation tools.<br>
</dd> 
<dt> <strong>RWeka</strong> </dt> 
<dd> An R interface to Weka, a rich collection of machine learning algorithms 
for data mining tasks.<br>
</dd> 
<dt> <strong>RWinEdt</strong> </dt> 
<dd> A plug in for using WinEdt as an editor for R. <br>
</dd> 
<dt> <strong>RXshrink</strong> </dt> 
<dd> Maximum Likelihood Shrinkage via Ridge or Least Angle Regression. <br>

</dd> 
<dt> <strong>RadioSonde</strong> </dt> 
<dd> A collection of programs for reading and plotting SKEW-T,log p diagrams 
and wind profiles for data collected by radiosondes (the typical weather 
balloon-borne instrument).<br>
</dd> 
<dt> <strong>RandVar</strong> </dt> 
<dd> Implementation of random variables by means of S4 classes and methods. 
<br> </dd> 
<dt> <strong>RandomFields</strong> </dt> 
<dd> Creating random fields using various methods. <br>
</dd> 
<dt> <strong>RankAggreg</strong> </dt> 
<dd> Weighted rank aggregation. <br>
</dd> 
<dt> <strong>RaschSampler</strong> </dt> 
<dd> Sampling binary matrices with fixed margins. <br>
</dd> 
<dt> <strong>Ratings</strong> </dt> 
<dd> Model-based ratings figures. <br>
</dd> 
<dt> <strong>Rcapture</strong> </dt> 
<dd> Loglinear models in capture-recapture experiments. <br>
</dd> 
<dt> <strong>Rcmdr</strong> </dt> 
<dd> A platform-independent basic-statistics GUI (graphical user interface) 
for R, based on the<strong>tcltk</strong> package. <br>
</dd> 
<dt> <strong>RcmdrPlugin.Export</strong> </dt> 
<dd> Graphically export objects to LaTeX or HTML. <br>
</dd> 
<dt> <strong>RcmdrPlugin.FactoMineR</strong> </dt> 
<dd> Rcmdr plug-in for the <strong>FactoMineR</strong> package. <br>
</dd> 
<dt> <strong>RcmdrPlugin.HH</strong> </dt> 
<dd> Rcmdr support for the introductory course at Temple University. <br>
</dd>
<dt> <strong>RcmdrPlugin.IPSUR</strong> </dt> 
<dd> Rcmdr plugin for &ldquo;Introduction to Probability and Statistics Using 
R&rdquo;.<br>
</dd> 
<dt> <strong>RcmdrPlugin.TeachingDemos</strong> </dt> 
<dd> Rcmdr Teaching Demos plug-in. <br>
</dd> 
<dt> <strong>RcmdrPlugin.epack</strong> </dt> 
<dd> Rcmdr epack demos plug-in. <br>
</dd> 
<dt> <strong>Rcplex</strong> </dt> 
<dd> R interface to CPLEX solvers for linear, quadratic, and (linear and 
quadratic) mixed integer programs. <br>
</dd> 
<dt> <strong>Read.isi</strong> </dt> 
<dd> Access old data saved in fixed-width format based on ISI-formatted 
codebooks.<br>
</dd> 
<dt> <strong>Reliability</strong> </dt> 
<dd> Functions for estimating parameters in software reliability models. <br>

</dd> 
<dt> <strong>ResearchMethods</strong> </dt> 
<dd> Using GUIs to help teach statistics to non-statistics students. <br>
</dd>
<dt> <strong>ResistorArray</strong> </dt> 
<dd> Electrical properties of resistor networks. <br>
</dd> 
<dt> <strong>Rfwdmv</strong> </dt> 
<dd> Forward Search for Multivariate Data. <br>
</dd> 
<dt> <strong>Rglpk</strong> </dt> 
<dd> R/GNU Linear Programming Kit interface. <br>
</dd> 
<dt> <strong>RiboSort</strong> </dt> 
<dd> Classification and analysis of microbial community profiles. <br>
</dd> 
<dt> <strong>Rigroup</strong> </dt> 
<dd> Provides small integer group functions. <br>
</dd> 
<dt> <strong>Rlab</strong> </dt> 
<dd> Functions and data sets for the NCSU ST370 class. <br>
</dd> 
<dt> <strong>Rlsf</strong> </dt> 
<dd> Interface to the LSF queuing system. <br>
</dd> 
<dt> <strong>Rmdr</strong> </dt> 
<dd> R-Multifactor Dimensionality Reduction. <br>
</dd> 
<dt> <strong>Rmetrics</strong> </dt> 
<dd> Financial engineering and computational finance. <br>
</dd> 
<dt> <strong>Rmpi</strong> </dt> 
<dd> An interface (wrapper) to MPI (Message-Passing Interface) APIs. It also 
provides an interactive R slave environment in which distributed statistical 
computing can be carried out. <br>
</dd> 
<dt> <strong>RobAStBase</strong> </dt> 
<dd> Base classes and functions for robust asymptotic statistics. <br>
</dd> 
<dt> <strong>RobLox</strong> </dt> 
<dd> Optimally robust influence curves for location and scale. <br>
</dd> 
<dt> <strong>RobRex</strong> </dt> 
<dd> Optimally robust influence curves for regression and scale. <br>
</dd> 
<dt> <strong>Rpad</strong> </dt> 
<dd> Utility functions for the Rpad workbook-style interface. <br>
</dd> 
<dt> <strong>Rsac</strong> </dt> 
<dd> Seismic tools for R. <br>
</dd> 
<dt> <strong>Rserve</strong> </dt> 
<dd> A socket server (TCP/IP or local sockets) which allows binary requests to 
be sent to R.<br>
</dd> 
<dt> <strong>Rsge</strong> </dt> 
<dd> Interface to the SGE cluster/grid queuing system. <br>
</dd> 
<dt> <strong>Rsundials</strong> </dt> 
<dd> SUite of Nonlinear DIfferential ALgebraic equations Solvers in R. <br>

</dd> 
<dt> <strong>Rsymphony</strong> </dt> 
<dd> An R interface to the SYMPHONY mixed integer linear program (MILP) solver.
<br> </dd> 
<dt> <strong>Runuran</strong> </dt> 
<dd> Interface to the UNU.RAN library for Universal Non-Uniform RANdom variate 
generators.<br>
</dd> 
<dt> <strong>Rvelslant</strong> </dt> 
<dd> Downhole seismic analysis. <br>
</dd> 
<dt> <strong>Rwave</strong> </dt> 
<dd> An environment for the time-frequency analysis of 1-D signals (and 
especially for the wavelet and Gabor transforms of noisy signals), based on the 
book &ldquo;Practical Time-Frequency Analysis: Gabor and Wavelet Transforms 
with an Implementation in S&rdquo; by Rene Carmona, Wen L. Hwang and Bruno 
Torresani, 1998, Academic Press.<br>
</dd> 
<dt> <strong>Ryacas</strong> </dt> 
<dd> An R interfaces to the yacas computer algebra system. <br>
</dd> 
<dt> <strong>SASPECT</strong> </dt> 
<dd> Significant AnalysiS of PEptide CounTs. <br>
</dd> 
<dt> <strong>SASmixed</strong> </dt> 
<dd> Data sets and sample linear mixed effects analyses corresponding to the 
examples in &ldquo;SAS System for Mixed Models&rdquo; by R. C. Littell, G. A. 
Milliken, W. W. Stroup and R. D. Wolfinger, 1996, SAS Institute.<br>
</dd> 
<dt> <strong>SASxport</strong> </dt> 
<dd> Read and write SAS XPORT files. <br>
</dd> 
<dt> <strong>SDDA</strong> </dt> 
<dd> Stepwise Diagonal Discriminant Analysis. <br>
</dd> 
<dt> <strong>SIN</strong> </dt> 
<dd> A SINful approach to selection of Gaussian Graphical Markov Models. <br>

</dd> 
<dt> <strong>SLmisc</strong> </dt> 
<dd> Miscellaneous Functions for analysis of gene expression data at SIRS-Lab 
GmbH.<br>
</dd> 
<dt> <strong>SMC</strong> </dt> 
<dd> Sequential Monte Carlo (SMC) Algorithm. <br>
</dd> 
<dt> <strong>SMPracticals</strong> </dt> 
<dd> Data sets and a few functions for use with the practicals outlined in 
Appendix A of the book &ldquo;Statistical Models&rdquo; by Anthony Davison, 
2003, Cambridge University Press.<br>
</dd> 
<dt> <strong>SNPMaP</strong> </dt> 
<dd> SNP Microarrays and Pooling in R. <br>
</dd> 
<dt> <strong>SNPMaP.cdm</strong> </dt> 
<dd> Annotation for SNP microarrays and pooling in R. <br>
</dd> 
<dt> <strong>SNPassoc</strong> </dt> 
<dd> SNP-based whole genome association studies. <br>
</dd> 
<dt> <strong>SNPmaxsel</strong> </dt> 
<dd> Maximally selected statistics for SNP data. <br>
</dd> 
<dt> <strong>SQLiteMap</strong> </dt> 
<dd> Manage vector graphical maps using SQLite. <br>
</dd> 
<dt> <strong>SQLiteDF</strong> </dt> 
<dd> Stores data frames and matrices in SQLite tables. <br>
</dd> 
<dt> <strong>SRPM</strong> </dt> 
<dd> Shared Reproducibility Package Management. <br>
</dd> 
<dt> <strong>STAR</strong> </dt> 
<dd> Spike Train Analysis with R. <br>
</dd> 
<dt> <strong>SciViews</strong> </dt> 
<dd> A bundle of packages to implement a full reusable GUI API for R. Contains 
<strong>svGUI</strong> with the main GUI features, <strong>svDialogs</strong> 
for the dialog boxes,<strong>svIO</strong> for data import/export, <strong>
svMisc</strong> with miscellaneous supporting functions, and <strong>svViews
</strong> providing views and report features (views are HTML presentations of 
the content of R objects, combining text, tables and graphs in the same 
document).<br>
</dd> 
<dt> <strong>SemiPar</strong> </dt> 
<dd> Functions for semiparametric regression analysis, to complement the book 
&ldquo;Semiparametric Regression&rdquo; by R. Ruppert, M. P. Wand, and R. J. 
Carroll, 2003, Cambridge University Press.<br>
</dd> 
<dt> <strong>SenSrivastava</strong> </dt> 
<dd> Collection of datasets from &ldquo;Regression Analysis, Theory, Methods 
and Applications&rdquo; by A. Sen and M. Srivastava, 1990, Springer.<br>
</dd> 
<dt> <strong>SensoMineR</strong> </dt> 
<dd> Sensory data analysis. <br>
</dd> 
<dt> <strong>SeqKnn</strong> </dt> 
<dd> Sequential KNN imputation. <br>
</dd> 
<dt> <strong>SharedHT2</strong> </dt> 
<dd> Shared Hotelling T^2 test for small sample microarray experiments. <br>

</dd> 
<dt> <strong>SiZer</strong> </dt> 
<dd> Significant Zero crossings. <br>
</dd> 
<dt> <strong>SimHap</strong> </dt> 
<dd> A comprehensive modeling framework for epidemiological outcomes and a 
multiple-imputation approach to haplotypic analysis of population-based data.
<br> </dd> 
<dt> <strong>SimpleTable</strong> </dt> 
<dd> Bayesian inference and sensitivity analysis for causal effects from 2 x 2 
and 2 x 2 x K tables in the presence of unmeasured confounding.<br>
</dd> 
<dt> <strong>Snowball</strong> </dt> 
<dd> Snowball stemmers. <br>
</dd> 
<dt> <strong>SoDA</strong> </dt> 
<dd> Utilities and examples from the book &ldquo;Software for Data Analysis: 
Programming with R&rdquo; by John Chambers, Springer, 2008.<br>
</dd> 
<dt> <strong>SoPhy</strong> </dt> 
<dd> Soil Physics Tools: simulation of water flux and solute transport in soil.
<br> </dd> 
<dt> <strong>SparseLogReg</strong> </dt> 
<dd> Sparse logistic regression. <br>
</dd> 
<dt> <strong>SparseM</strong> </dt> 
<dd> Basic linear algebra for sparse matrices. <br>
</dd> 
<dt> <strong>SpatialNP</strong> </dt> 
<dd> Multivariate nonparametric methods based on spatial signs and ranks. <br>

</dd> 
<dt> <strong>SpherWave</strong> </dt> 
<dd> Spherical Wavelets and SW-based spatially adaptive methods. <br>
</dd> 
<dt> <strong>StatDA</strong> </dt> 
<dd> Statistical analysis for environmental data, a companion to the book 
&ldquo;Statistical Data Analysis Explained: Applied Environmental Statistics 
with R&rdquo; by C. Reimann, P. Filzmoser, R. G. Garrett, and R. Dutter, 2008, 
John Wiley and Sons.<br>
</dd> 
<dt> <strong>StatDataML</strong> </dt> 
<dd> Read and write StatDataML. <br>
</dd> 
<dt> <strong>StreamMetabolism</strong> </dt> 
<dd> Calculation of single station metabolism from diurnal oxygen curves. <br>

</dd> 
<dt> <strong>SuppDists</strong> </dt> 
<dd> Ten distributions supplementing those built into R (Inverse Gauss, 
Kruskal-Wallis, Kendall's Tau, Friedman's chi squared, Spearman's rho, maximum 
F ratio, the Pearson product moment correlation coefficiant, Johnson 
distributions, normal scores and generalized hypergeometric distributions).<br>

</dd> 
<dt> <strong>SwissAir</strong> </dt> 
<dd> Air quality data of Switzerland for one year in 30 min resolution. <br>

</dd> 
<dt> <strong>SyNet</strong> </dt> 
<dd> Inference and analysis of sympatry networks. <br>
</dd> 
<dt> <strong>Synth</strong> </dt> 
<dd> Causal inference using the synthetic control group method. <br>
</dd> 
<dt> <strong>TIMP</strong> </dt> 
<dd> A problem solving environment for fitting superposition models. <br>
</dd>
<dt> <strong>TRAMPR</strong> </dt> 
<dd> Terminal Restriction Fragment Length Polymorphism (TRFLP) Analysis and 
Matching Package for R.<br>
</dd> 
<dt> <strong>TSA</strong> </dt> 
<dd> Functions and datasets detailed in the book &ldquo;Time Series Analysis 
With Applications in R&rdquo; (3rd edition) by Jonathan Cryer and Kung-Sik 
Chan, 2008, Springer.<br>
</dd> 
<dt> <strong>TSHRC</strong> </dt> 
<dd> Two Stage Hazard Rate Comparison. <br>
</dd> 
<dt> <strong>TSMySQL</strong> </dt> 
<dd> Time Series Database Interface extensions for MySQL. <br>
</dd> 
<dt> <strong>TSP</strong> </dt> 
<dd> Traveling Salesperson Problem (TSP). <br>
</dd> 
<dt> <strong>TSSQLite</strong> </dt> 
<dd> Time Series Database Interface extensions for SQLite. <br>
</dd> 
<dt> <strong>TSdbi</strong> </dt> 
<dd> Time Series Database Interface. <br>
</dd> 
<dt> <strong>TSpadi</strong> </dt> 
<dd> Connect to a time series database (e.g., Fame) via PADI (Protocol for 
Application Database Interface), using the<strong>TSdbi</strong> infrastructure.
<br> </dd> 
<dt> <strong>TTR</strong> </dt> 
<dd> Technical Trading Rules. <br>
</dd> 
<dt> <strong>TWIX</strong> </dt> 
<dd> Trees WIth eXtra splits. <br>
</dd> 
<dt> <strong>TeachingDemos</strong> </dt> 
<dd> A set of demonstration functions that can be used in a classroom to 
demonstrate statistical concepts, or on your own to better understand the 
concepts or the programming.<br>
</dd> 
<dt> <strong>TinnR</strong> </dt> 
<dd> Resources for the Tinn-R GUI/Editor for R. <br>
</dd> 
<dt> <strong>TraMineR</strong> </dt> 
<dd> Sequences and trajectories mining for social scientists. <br>
</dd> 
<dt> <strong>TwoWaySurvival</strong> </dt> 
<dd> Additive two-way hazards modeling of right censored survival data. <br>

</dd> 
<dt> <strong>TwslmSpikeWeight</strong> </dt> 
<dd> Normalization of cDNA microarray data with the two-way semilinear model 
(TW-SLM).<br>
</dd> 
<dt> <strong>UNF</strong> </dt> 
<dd> Tools for creating universal numeric fingerprints for data. <br>
</dd> 
<dt> <strong>USPS</strong> </dt> 
<dd> Unsupervised and Supervised methods of Propensity Score adjustment for 
bias.<br>
</dd> 
<dt> <strong>Umacs</strong> </dt> 
<dd> Universal MArkov Chain Sampler. <br>
</dd> 
<dt> <strong>UsingR</strong> </dt> 
<dd> Data sets to accompany the textbook &ldquo;Using R for Introductory 
Statistics&rdquo; by J. Verzani, 2005, Chapman &amp; Hall/CRC.<br>
</dd> 
<dt> <strong>VDCutil</strong> </dt> 
<dd> Utilities supporting VDC, an open source digital library system for 
quantitative data.<br>
</dd> 
<dt> <strong>VGAM</strong> </dt> 
<dd> Vector Generalized Linear and Additive Models. <br>
</dd> 
<dt> <strong>VIM</strong> </dt> 
<dd> Visualization and Imputation of Missing Values. <br>
</dd> 
<dt> <strong>VLMC</strong> </dt> 
<dd> Functions, classes &amp; methods for estimation, prediction, and 
simulation (bootstrap) of VLMC (Variable Length Markov Chain) models.<br>
</dd> 
<dt> <strong>VaR</strong> </dt> 
<dd> Methods for calculation of Value at Risk (VaR). <br>
</dd> 
<dt> <strong>VhayuR</strong> </dt> 
<dd> R interface to the Vhayu Velocity high volume fast financial market data 
archival and analysis products.<br>
</dd> 
<dt> <strong>WINRPACK</strong> </dt> 
<dd> Reads in WIN pickfile and waveform files. <br>
</dd> 
<dt> <strong>WWGbook</strong> </dt> 
<dd> Functions and datasets for the book &ldquo;Linear Mixed Models: A 
Practical Guide Using Statistical Software&rdquo; by B. West, K. Welch, and A. 
Galecki, 2006, Chapman &amp; Hall/CRC.<br>
</dd> 
<dt> <strong>WeedMap</strong> </dt> 
<dd> Map of weed intensity. <br>
</dd> 
<dt> <strong>WhatIf</strong> </dt> 
<dd> Software for evaluating counterfactuals. <br>
</dd> 
<dt> <strong>XML</strong> </dt> 
<dd> Tools for reading XML documents and DTDs. <br>
</dd> 
<dt> <strong>XReg</strong> </dt> 
<dd> Extreme regression. <br>
</dd> 
<dt> <strong>YaleToolkit</strong> </dt> 
<dd> Data exploration tools from Yale University. <br>
</dd> 
<dt> <strong>ZIGP</strong> </dt> 
<dd> Zero Inflated Generalized Poisson (ZIGP) regression models. <br>
</dd> 
<dt> <strong>Zelig</strong> </dt> 
<dd> Everyone's statistical software: an easy-to-use program that can 
estimate, and help interpret the results of, an enormous range of statistical 
models.<br>
</dd> 
<dt> <strong>aaMI</strong> </dt> 
<dd> Mutual information for protein sequence alignments. <br>
</dd> 
<dt> <strong>abind</strong> </dt> 
<dd> Combine multi-dimensional arrays. <br>
</dd> 
<dt> <strong>accuracy</strong> </dt> 
<dd> A suite of tools designed to test and improve the accuracy of statistical 
computation.<br>
</dd> 
<dt> <strong>acepack</strong> </dt> 
<dd> ACE (Alternating Conditional Expectations) and AVAS (Additivity and 
VAriance Stabilization for regression) methods for selecting regression 
transformations.<br>
</dd> 
<dt> <strong>actuar</strong> </dt> 
<dd> Functions related to actuarial science applications. <br>
</dd> 
<dt> <strong>ada</strong> </dt> 
<dd> Performs boosting algorithms for a binary response. <br>
</dd> 
<dt> <strong>adabag</strong> </dt> 
<dd> Adaboost.M1 and Bagging. <br>
</dd> 
<dt> <strong>adapt</strong> </dt> 
<dd> Adaptive quadrature in up to 20 dimensions. <br>
</dd> 
<dt> <strong>ade4</strong> </dt> 
<dd> Multivariate data analysis and graphical display. <br>
</dd> 
<dt> <strong>ade4TkGUI</strong> </dt> 
<dd> Tcl/Tk Graphical User Interface for <strong>ade4</strong>. <br>
</dd> 
<dt> <strong>adegenet</strong> </dt> 
<dd> Genetic data handling for multivariate analysis using <strong>ade4
</strong>. <br>
</dd> 
<dt> <strong>adehabitat</strong> </dt> 
<dd> A collection of tools for the analysis of habitat selection by animals. 
<br> </dd> 
<dt> <strong>adimpro</strong> </dt> 
<dd> Adaptive smoothing of digital images. <br>
</dd> 
<dt> <strong>adk</strong> </dt> 
<dd> Anderson-Darling K-sample test and combinations of such tests. <br>
</dd> 
<dt> <strong>adlift</strong> </dt> 
<dd> Adaptive Wavelet transforms for signal denoising. <br>
</dd> 
<dt> <strong>ads</strong> </dt> 
<dd> Spatial point patterns analysis. <br>
</dd> 
<dt> <strong>agce</strong> </dt> 
<dd> Analysis of growth curve experiments. <br>
</dd> 
<dt> <strong>agricolae</strong> </dt> 
<dd> Statistical procedures for agricultural research. <br>
</dd> 
<dt> <strong>agsemisc</strong> </dt> 
<dd> Miscellaneous plotting and utility functions. <br>
</dd> 
<dt> <strong>akima</strong> </dt> 
<dd> Linear or cubic spline interpolation for irregularly gridded data. <br>

</dd> 
<dt> <strong>allelic</strong> </dt> 
<dd> A fast, unbiased and exact allelic exact test. <br>
</dd> 
<dt> <strong>alr3</strong> </dt> 
<dd> Methods and data to accompany the textbook &ldquo;Applied Linear 
Regression&rdquo; by S. Weisberg, 2005, Wiley.<br>
</dd> 
<dt> <strong>amap</strong> </dt> 
<dd> Another Multidimensional Analysis Package. <br>
</dd> 
<dt> <strong>anacor</strong> </dt> 
<dd> Simple and Canonical Correspondence Analysis. <br>
</dd> 
<dt> <strong>analogue</strong> </dt> 
<dd> Analogue methods for palaeoecology. <br>
</dd> 
<dt> <strong>anapuce</strong> </dt> 
<dd> Tools for microarray data analysis. <br>
</dd> 
<dt> <strong>animation</strong> </dt> 
<dd> Demonstrate animations in statistics. <br>
</dd> 
<dt> <strong>anm</strong> </dt> 
<dd> Analog model for statistical/empirical downscaling. <br>
</dd> 
<dt> <strong>aod</strong> </dt> 
<dd> Analysis of Overdispersed Data. <br>
</dd> 
<dt> <strong>apTreeshape</strong> </dt> 
<dd> Analyses of phylogenetic treeshape. <br>
</dd> 
<dt> <strong>ape</strong> </dt> 
<dd> Analyses of Phylogenetics and Evolution, providing functions for reading 
and plotting phylogenetic trees in parenthetic format (standard Newick format), 
analyses of comparative data in a phylogenetic framework, analyses of 
diversification and macroevolution, computing distances from allelic and 
nucleotide data, reading nucleotide sequences from GenBank via internet, and 
several tools such as Mantel's test, computation of minimum spanning tree, or 
the population parameter theta based on various approaches.<br>
</dd> 
<dt> <strong>aplpack</strong> </dt> 
<dd> Another PLot PACKage: stem.leaf, bagplot, faces, spin3R, .... <br>
</dd> 
<dt> <strong>apsrtable</strong> </dt> 
<dd> American Political Science Review style table formatting. <br>
</dd> 
<dt> <strong>argosfilter</strong> </dt> 
<dd> Argos locations filter. <br>
</dd> 
<dt> <strong>arm</strong> </dt> 
<dd> Data Analysis using Regression and Multilevel/hierarchical models. <br>

</dd> 
<dt> <strong>arrayImpute</strong> </dt> 
<dd> Missing imputation for microarray data. <br>
</dd> 
<dt> <strong>arrayMissPattern</strong> </dt> 
<dd> Exploratory analysis of missing patterns for microarray data. <br>
</dd> 
<dt> <strong>ars</strong> </dt> 
<dd> Adaptive Rejection Sampling. <br>
</dd> 
<dt> <strong>arules</strong> </dt> 
<dd> Mining association rules and frequent itemsets with R. <br>
</dd> 
<dt> <strong>arulesSequences</strong> </dt> 
<dd> Mining frequent sequences. <br>
</dd> 
<dt> <strong>ash</strong> </dt> 
<dd> David Scott's ASH routines for 1D and 2D density estimation. <br>
</dd> 
<dt> <strong>aspace</strong> </dt> 
<dd> Estimating centrographic statistics and computational geometries from 
spatial point patterns.<br>
</dd> 
<dt> <strong>assist</strong> </dt> 
<dd> A suite of functions implementing smoothing splines. <br>
</dd> 
<dt> <strong>aster</strong> </dt> 
<dd> Functions and datasets for Aster modeling (forest graph exponential 
family conditional or unconditional canonical statistic models for life history 
trait modeling).<br>
</dd> 
<dt> <strong>asypow</strong> </dt> 
<dd> A set of routines that calculate power and related quantities utilizing 
asymptotic likelihood ratio methods.<br>
</dd> 
<dt> <strong>asuR</strong> </dt> 
<dd> Functions and data sets for a lecture in &ldquo;Advanced Statistics using 
R&rdquo;.<br>
</dd> 
<dt> <strong>aws</strong> </dt> 
<dd> Functions to perform adaptive weights smoothing. <br>
</dd> 
<dt> <strong>backfitRichards</strong> </dt> 
<dd> Backfitted independent values of Richards curves. <br>
</dd> 
<dt> <strong>backtest</strong> </dt> 
<dd> Exploring portfolio-based hypotheses about financial instruments. <br>

</dd> 
<dt> <strong>bark</strong> </dt> 
<dd> Bayesian Additive Regression Kernels. <br>
</dd> 
<dt> <strong>bayesGARCH</strong> </dt> 
<dd> Bayesian estimation of the GARCH(1,1) model with Student's t innovations. 
<br> </dd> 
<dt> <strong>bayesSurv</strong> </dt> 
<dd> Bayesian survival regression with flexible error and (later on also 
random effects) distributions.<br>
</dd> 
<dt> <strong>bayescount</strong> </dt> 
<dd> Bayesian analysis of count distributions with JAGS. <br>
</dd> 
<dt> <strong>bayesm</strong> </dt> 
<dd> Bayes Inference for Marketing/Micro-econometrics. <br>
</dd> 
<dt> <strong>bayesmix</strong> </dt> 
<dd> Bayesian mixture models of univariate Gaussian distributions using JAGS. 
<br> </dd> 
<dt> <strong>bbmle</strong> </dt> 
<dd> Modifications and extensions of <strong>stats4</strong> MLE code. <br>

</dd> 
<dt> <strong>bcp</strong> </dt> 
<dd> Bayesian Change Point based on the Barry and Hartigan product partition 
model.<br>
</dd> 
<dt> <strong>bear</strong> </dt> 
<dd> Bioavability and bioequivalence data analysis with crossover design. <br>

</dd> 
<dt> <strong>benchden</strong> </dt> 
<dd> 28 benchmark densities from Berlinet/Devroye (1994). <br>
</dd> 
<dt> <strong>bentcableAR</strong> </dt> 
<dd> Bent-cable regression for independent data or autoregressive time series. 
<br> </dd> 
<dt> <strong>betaper</strong> </dt> 
<dd> Distance decay of similarity among biological inventories in the face of 
taxonomic uncertainty.<br>
</dd> 
<dt> <strong>betareg</strong> </dt> 
<dd> Beta regression for modeling rates and proportions. <br>
</dd> 
<dt> <strong>bs</strong> </dt> 
<dd> Utilities for the Birnbaum-Saunders distribution. <br>
</dd> 
<dt> <strong>biOps</strong> </dt> 
<dd> Basic image operations and image processing. <br>
</dd> 
<dt> <strong>biOpsGUI</strong> </dt> 
<dd> GUI for Basic image operations. <br>
</dd> 
<dt> <strong>biclust</strong> </dt> 
<dd> BiCluster algorithms. <br>
</dd> 
<dt> <strong>bicreduc</strong> </dt> 
<dd> Reduction algorithm for the NPMLE for the distribution function of 
bivariate interval-censored data.<br>
</dd> 
<dt> <strong>bifactorial</strong> </dt> 
<dd> Inferences for bi- and trifactorial trial designs. <br>
</dd> 
<dt> <strong>biglm</strong> </dt> 
<dd> Linear regression for data too large to fit in memory. <br>
</dd> 
<dt> <strong>bigmemory</strong> </dt> 
<dd> Manage massive matrices in R using C++, with UNIX support for shared 
memory.<br>
</dd> 
<dt> <strong>bim</strong> </dt> 
<dd> Bayesian interval mapping diagnostics: functions to interpret QTLCart and 
Bmapqtl samples.<br>
</dd> 
<dt> <strong>binGroup</strong> </dt> 
<dd> Evaluation and experimental design for binomial group testing. <br>
</dd> 
<dt> <strong>binMto</strong> </dt> 
<dd> Asymptotic simultaneous confidence intervals for many-to-one comparisons 
of proportions.<br>
</dd> 
<dt> <strong>bindata</strong> </dt> 
<dd> Generation of correlated artificial binary data. <br>
</dd> 
<dt> <strong>binom</strong> </dt> 
<dd> Binomial confidence intervals for several parameterizations. <br>
</dd> 
<dt> <strong>bio.infer</strong> </dt> 
<dd> Compute biological inferences. <br>
</dd> 
<dt> <strong>biopara</strong> </dt> 
<dd> Self-contained parallel system for R. <br>
</dd> 
<dt> <strong>bipartite</strong> </dt> 
<dd> Visualises bipartite networks and calculates some ecological indices. <br>
</dd> 
<dt> <strong>birch</strong> </dt> 
<dd> Dealing with very large datasets using BIRCH. <br>
</dd> 
<dt> <strong>bise</strong> </dt> 
<dd> Auxiliary functions for phenological data analysis. <br>
</dd> 
<dt> <strong>bitops</strong> </dt> 
<dd> Functions for Bitwise operations on integer vectors. <br>
</dd> 
<dt> <strong>bivpois</strong> </dt> 
<dd> Bivariate Poisson models using the EM algorithm. <br>
</dd> 
<dt> <strong>blighty</strong> </dt> 
<dd> Function for drawing the coastline of the United Kingdom. <br>
</dd> 
<dt> <strong>blockTools</strong> </dt> 
<dd> Block, randomly assign, and diagnose potential problems between units in 
randomized experiments.<br>
</dd> 
<dt> <strong>blockrand</strong> </dt> 
<dd> Randomization for block random clinical trials. <br>
</dd> 
<dt> <strong>bmd</strong> </dt> 
<dd> Benchmark dose analysis for dose-response data. <br>
</dd> 
<dt> <strong>bnlearn</strong> </dt> 
<dd> Bayesian network structure learning. <br>
</dd> 
<dt> <strong>boa</strong> </dt> 
<dd> Bayesian Output Analysis Program for MCMC. <br>
</dd> 
<dt> <strong>boolean</strong> </dt> 
<dd> Boolean logit and probit: a procedure for testing Boolean hypotheses. <br>
</dd> 
<dt> <strong>boost</strong> </dt> 
<dd> Boosting methods for real and simulated data, featuring `BagBoost', 
`LogitBoost', `AdaBoost', and `L2Boost'.<br>
</dd> 
<dt> <strong>boot</strong> </dt> 
<dd> Functions and datasets for bootstrapping from the book &ldquo;Bootstrap 
Methods and Their Applications&rdquo; by A. C. Davison and D. V. Hinkley, 1997, 
Cambridge University Press.<em>Recommended</em>. <br>
</dd> 
<dt> <strong>bootStepAIC</strong> </dt> 
<dd> Model selection by bootstrapping the <code>stepAIC()</code> procedure. 
<br> </dd> 
<dt> <strong>bootstrap</strong> </dt> 
<dd> Software (bootstrap, cross-validation, jackknife), data and errata for 
the book &ldquo;An Introduction to the Bootstrap&rdquo; by B. Efron and R. 
Tibshirani, 1993, Chapman and Hall.<br>
</dd> 
<dt> <strong>bpca</strong> </dt> 
<dd> Biplot of multivariate data based on Principal Components Analysis. <br>

</dd> 
<dt> <strong>bqtl</strong> </dt> 
<dd> QTL mapping toolkit for inbred crosses and recombinant inbred lines. 
Includes maximum likelihood and Bayesian tools.<br>
</dd> 
<dt> <strong>brainwaver</strong> </dt> 
<dd> Basic wavelet analysis of multivariate time series with a vizualisation 
and parametrization using graph theory.<br>
</dd> 
<dt> <strong>brew</strong> </dt> 
<dd> Templating framework for report generation. <br>
</dd> 
<dt> <strong>brglm</strong> </dt> 
<dd> Bias-reduction in binomial-response GLMs. <br>
</dd> 
<dt> <strong>brlr</strong> </dt> 
<dd> Bias-reduced logistic regression: fits logistic regression models by 
maximum penalized likelihood.<br>
</dd> 
<dt> <strong>bspec</strong> </dt> 
<dd> Bayesian inference on the (discrete) power spectrum of time series. <br>

</dd> 
<dt> <strong>bvls</strong> </dt> 
<dd> The Stark-Parker algorithm for bounded-variable least squares. <br>
</dd> 
<dt> <strong>ca</strong> </dt> 
<dd> Simple, multiple and joint Correspondence Analysis. <br>
</dd> 
<dt> <strong>caMassClass</strong> </dt> 
<dd> Processing and Classification of protein mass spectra (SELDI) data. <br>

</dd> 
<dt> <strong>caTools</strong> </dt> 
<dd> Miscellaneous utility functions, including reading/writing ENVI binary 
files, a LogitBoost classifier, and a base64 encoder/decoder.<br>
</dd> 
<dt> <strong>cacheSweave</strong> </dt> 
<dd> Tools for caching Sweave computations. <br>
</dd> 
<dt> <strong>cacher</strong> </dt> 
<dd> Tools for caching and distributing statistical analyses. <br>
</dd> 
<dt> <strong>cairoDevice</strong> </dt> 
<dd> Loadable CAIRO/GTK device driver. <br>
</dd> 
<dt> <strong>calib</strong> </dt> 
<dd> Statistical tool for calibration of plate based bioassays. <br>
</dd> 
<dt> <strong>calibrate</strong> </dt> 
<dd> Calibration of biplot axes. <br>
</dd> 
<dt> <strong>candisc</strong> </dt> 
<dd> Generalized canonical discriminant analysis. <br>
</dd> 
<dt> <strong>car</strong> </dt> 
<dd> Companion to Applied Regression, containing functions for applied 
regession, linear models, and generalized linear models, with an emphasis on 
regression diagnostics, particularly graphical diagnostic methods.<br>
</dd> 
<dt> <strong>caret</strong> </dt> 
<dd> Classification and REgression Training. <br>
</dd> 
<dt> <strong>caretLSF</strong> </dt> 
<dd> Classification and REgression Training, LSF style. <br>
</dd> 
<dt> <strong>caretNWS</strong> </dt> 
<dd> Classification and REgression Training in parallel using NetworkSpaces. 
<br> </dd> 
<dt> <strong>cat</strong> </dt> 
<dd> Analysis of categorical-variable datasets with missing values. <br>
</dd> 
<dt> <strong>catmap</strong> </dt> 
<dd> Case-control and TDT meta-analysis package. <br>
</dd> 
<dt> <strong>catspec</strong> </dt> 
<dd> Special models for categorical variables. <br>
</dd> 
<dt> <strong>cba</strong> </dt> 
<dd> Clustering for Business Analytics, including implementations of Proximus 
and Rock.<br>
</dd> 
<dt> <strong>ccgarch</strong> </dt> 
<dd> Conditional Correlation GARCH models. <br>
</dd> 
<dt> <strong>cclust</strong> </dt> 
<dd> Convex clustering methods, including k-means algorithm, on-line update 
algorithm (Hard Competitive Learning) and Neural Gas algorithm (Soft 
Competitive Learning) and calculation of several indexes for finding the number 
of clusters in a data set.<br>
</dd> 
<dt> <strong>celsius</strong> </dt> 
<dd> Retrieve Affymetrix microarray measurements and metadata from Celsius. 
<br> </dd> 
<dt> <strong>cem</strong> </dt> 
<dd> The coarsened exact matching algorithm (and many extensions). <br>
</dd> 
<dt> <strong>cfa</strong> </dt> 
<dd> Analysis of configuration frequencies. <br>
</dd> 
<dt> <strong>cggd</strong> </dt> 
<dd> Continuous Generalized Gradient Descent. <br>
</dd> 
<dt> <strong>cgh</strong> </dt> 
<dd> Analysis of microarray comparative genome hybridisation data using the 
Smith-Waterman algorithm.<br>
</dd> 
<dt> <strong>cghFLasso</strong> </dt> 
<dd> Hot spot detecting for CGH array data with fused lasso regression. <br>

</dd> 
<dt> <strong>chplot</strong> </dt> 
<dd> Augmented convex hull plots: informative and nice plots for grouped 
bivariate data.<br>
</dd> 
<dt> <strong>changeLOS</strong> </dt> 
<dd> Change in length of hospital stay (LOS). <br>
</dd> 
<dt> <strong>cheb</strong> </dt> 
<dd> Discrete linear Chebyshev approximation. <br>
</dd> 
<dt> <strong>chemCal</strong> </dt> 
<dd> Calibration functions for analytical chemistry. <br>
</dd> 
<dt> <strong>chemometrics</strong> </dt> 
<dd> Companion to the book &ldquo;Introduction to Multivariate Statistical 
Analysis in Chemometrics&rdquo; by K. Varmuza and P. Filzmoser, CRC Press, to 
appear.<br>
</dd> 
<dt> <strong>choplump</strong> </dt> 
<dd> Choplump tests (permutation tests for comparing two groups with some 
positive but many zero responses).<br>
</dd> 
<dt> <strong>chron</strong> </dt> 
<dd> A package for working with chronological objects (times and dates). <br>

</dd> 
<dt> <strong>cir</strong> </dt> 
<dd> Nonparametric estimation of monotone functions via isotonic regression 
and centered isotonic regression.<br>
</dd> 
<dt> <strong>circular</strong> </dt> 
<dd> Circular statistics, from &ldquo;Topics in Circular Statistics&rdquo; by 
Rao Jammalamadaka and A. SenGupta, 2001, World Scientific.<br>
</dd> 
<dt> <strong>clValid</strong> </dt> 
<dd> Statistical and biological validation of clustering results. <br>
</dd> 
<dt> <strong>clac</strong> </dt> 
<dd> Clust Along Chromosomes, a method to call gains/losses in CGH array data. 
<br> </dd> 
<dt> <strong>class</strong> </dt> 
<dd> Functions for classification (k-nearest neighbor and LVQ). Contained in 
theVR bundle. <em>Recommended</em>. <br>
</dd> 
<dt> <strong>classGraph</strong> </dt> 
<dd> Construct graph of S4 class hierarchies. <br>
</dd> 
<dt> <strong>classInt</strong> </dt> 
<dd> Choose univariate class intervals for mapping or other graphics purposes. 
<br> </dd> 
<dt> <strong>classPP</strong> </dt> 
<dd> Projection Pursuit for supervised classification. <br>
</dd> 
<dt> <strong>classifly</strong> </dt> 
<dd> Explore classification models in high dimensions. <br>
</dd> 
<dt> <strong>clim.pact</strong> </dt> 
<dd> Climate analysis and downscaling for monthly and daily data. <br>
</dd> 
<dt> <strong>climatol</strong> </dt> 
<dd> Functions to fill missing data in climatological (monthly) series and to 
test their homogeneity, plus functions to draw wind-rose and Walter&amp;Lieth 
diagrams. <br>
</dd> 
<dt> <strong>clinfun</strong> </dt> 
<dd> Utilities for clinical study design and data analyses. <br>
</dd> 
<dt> <strong>clue</strong> </dt> 
<dd> CLUster Ensembles. <br>
</dd> 
<dt> <strong>clustTool</strong> </dt> 
<dd> GUI for clustering data with spatial information. <br>
</dd> 
<dt> <strong>cluster</strong> </dt> 
<dd> Functions for cluster analysis. <em>Recommended</em>. <br>
</dd> 
<dt> <strong>clusterGeneration</strong> </dt> 
<dd> Random cluster generation (with specified degree of separation). <br>

</dd> 
<dt> <strong>clusterRepro</strong> </dt> 
<dd> Reproducibility of gene expression clusters. <br>
</dd> 
<dt> <strong>clusterSim</strong> </dt> 
<dd> Searching for optimal clustering procedure for a data set. <br>
</dd> 
<dt> <strong>clusterfly</strong> </dt> 
<dd> Explore clustering interactively using R and GGobi. <br>
</dd> 
<dt> <strong>clustvarsel</strong> </dt> 
<dd> Variable selection for model-based clustering. <br>
</dd> 
<dt> <strong>clv</strong> </dt> 
<dd> Cluster validation techniques. <br>
</dd> 
<dt> <strong>cmprsk</strong> </dt> 
<dd> Estimation, testing and regression modeling of subdistribution functions 
in competing risks.<br>
</dd> 
<dt> <strong>cobs99</strong> </dt> 
<dd> Constrained B-splines: outdated 1999 version. <br>
</dd> 
<dt> <strong>cobs</strong> </dt> 
<dd> Constrained B-splines: qualitatively constrained (regression) smoothing 
via linear programming and sparse matrices.<br>
</dd> 
<dt> <strong>cocorresp</strong> </dt> 
<dd> Co-correspondence analysis ordination methods for community ecology. <br>

</dd> 
<dt> <strong>coda</strong> </dt> 
<dd> Output analysis and diagnostics for Markov Chain Monte Carlo (MCMC) 
simulations.<br>
</dd> 
<dt> <strong>codetools</strong> </dt> 
<dd> Code analysis tools. <em>Recommended</em> for R 2.5.0 or later. <br>
</dd>
<dt> <strong>coin</strong> </dt> 
<dd> COnditional INference procedures for the general independence problem 
including two-sample, K-sample, correlation, censored, ordered and multivariate 
problems.<br>
</dd> 
<dt> <strong>colorRamp</strong> </dt> 
<dd> Builds single and double gradient color maps. <br>
</dd> 
<dt> <strong>colorspace</strong> </dt> 
<dd> Mapping between assorted color spaces. <br>
</dd> 
<dt> <strong>combinat</strong> </dt> 
<dd> Combinatorics utilities. <br>
</dd> 
<dt> <strong>compHclust</strong> </dt> 
<dd> Complementary hierarchical clustering. <br>
</dd> 
<dt> <strong>compOverlapCorr</strong> </dt> 
<dd> Comparing overlapping correlation coefficients. <br>
</dd> 
<dt> <strong>compare</strong> </dt> 
<dd> Comparing objects for differences. <br>
</dd> 
<dt> <strong>compoisson</strong> </dt> 
<dd> Conway-Maxwell-Poisson distribution. <br>
</dd> 
<dt> <strong>compositions</strong> </dt> 
<dd> Functions for the consistent analysis of compositional data (e.g., 
portions of substances) and positive numbers (e.g., concentrations).<br>
</dd> 
<dt> <strong>concor</strong> </dt> 
<dd> Concordance, providing &ldquo;SVD by blocks&rdquo;. <br>
</dd> 
<dt> <strong>concord</strong> </dt> 
<dd> Measures of concordance and reliability. <br>
</dd> 
<dt> <strong>conf.design</strong> </dt> 
<dd> A series of simple tools for constructing and manipulating confounded and 
fractional factorial designs.<br>
</dd> 
<dt> <strong>connectedness</strong> </dt> 
<dd> Find disconnected sets for two-way classification. <br>
</dd> 
<dt> <strong>contfrac</strong> </dt> 
<dd> Continued fractions. <br>
</dd> 
<dt> <strong>contrast</strong> </dt> 
<dd> A collection of contrast methods. <br>
</dd> 
<dt> <strong>convexHaz</strong> </dt> 
<dd> Nonparametric MLE/LSE of convex hazard. <br>
</dd> 
<dt> <strong>copas</strong> </dt> 
<dd> Statistical methods to model and adjust for bias in meta-analysis. <br>

</dd> 
<dt> <strong>copula</strong> </dt> 
<dd> Classes of commonly used copulas (including elliptical and Archimedian), 
and methods for density, distribution, random number generators, and plotting.
<br> </dd> 
<dt> <strong>corpora</strong> </dt> 
<dd> Utility functions for the statistical analysis of corpus frequency data. 
<br> </dd> 
<dt> <strong>corpcor</strong> </dt> 
<dd> Efficient estimation of covariance and (partial) correlation. <br>
</dd> 
<dt> <strong>corrgram</strong> </dt> 
<dd> Plot a correlogram. <br>
</dd> 
<dt> <strong>corrperm</strong> </dt> 
<dd> Permutation tests of correlation with repeated measurements. <br>
</dd> 
<dt> <strong>covRobust</strong> </dt> 
<dd> Robust covariance estimation via nearest neighbor cleaning. <br>
</dd> 
<dt> <strong>coxphf</strong> </dt> 
<dd> Cox regression with Firth's penalized likelihood. <br>
</dd> 
<dt> <strong>coxphw</strong> </dt> 
<dd> Weighted estimation for Cox regression. <br>
</dd> 
<dt> <strong>coxrobust</strong> </dt> 
<dd> Robust Estimation in the Cox proportional hazards regression model. <br>

</dd> 
<dt> <strong>cramer</strong> </dt> 
<dd> Routine for the multivariate nonparametric Cramer test. <br>
</dd> 
<dt> <strong>crank</strong> </dt> 
<dd> Functions for completing and recalculating rankings. <br>
</dd> 
<dt> <strong>crossdes</strong> </dt> 
<dd> Functions for the construction and randomization of balanced carryover 
balanced designs, to check given designs for balance, and for simulation 
studies on the validity of two randomization procedures.<br>
</dd> 
<dt> <strong>crosshybDetector</strong> </dt> 
<dd> Detection of cross-hybridization events in microarray experiments. <br>

</dd> 
<dt> <strong>crq</strong> </dt> 
<dd> Quantile regression for randomly censored data. <br>
</dd> 
<dt> <strong>cslogistic</strong> </dt> 
<dd> Likelihood and posterior analysis of conditionally specified logistic 
regression models.<br>
</dd> 
<dt> <strong>cts</strong> </dt> 
<dd> Continuous time autoregressive models and the Kalman filter. <br>
</dd> 
<dt> <strong>ctv</strong> </dt> 
<dd> Server-side and client-side tools for CRAN task views. <br>
</dd> 
<dt> <strong>cwhmisc</strong> </dt> 
<dd> Miscellaneous functions by Christian W. Hoffmann. <br>
</dd> 
<dt> <strong>cyclones</strong> </dt> 
<dd> Cyclone identification. <br>
</dd> 
<dt> <strong>data.table</strong> </dt> 
<dd> Extension of data frames to allow subscripting by expressions evaluated 
within the frame.<br>
</dd> 
<dt> <strong>dataframes2xls</strong> </dt> 
<dd> Write data frames to .xls files. <br>
</dd> 
<dt> <strong>date</strong> </dt> 
<dd> Functions for dealing with dates. The most useful of them accepts a 
vector of input dates in any of the forms `8/30/53', ` 30Aug53', ` 30 August 
1953', ..., ` August 30 53', or any mixture of these. <br>
</dd> 
<dt> <strong>dblcens</strong> </dt> 
<dd> Calculates the NPMLE of the survival distribution for doubly censored 
data.<br>
</dd> 
<dt> <strong>ddesolve</strong> </dt> 
<dd> Solver for Delay Differential Equations. <br>
</dd> 
<dt> <strong>ddst</strong> </dt> 
<dd> Data driven smooth Neyman test. <br>
</dd> 
<dt> <strong>deSolve</strong> </dt> 
<dd> General solvers for ordinary differential equations (ODE) and for 
differential algebraic equations (DAE).<br>
</dd> 
<dt> <strong>deal</strong> </dt> 
<dd> Bayesian networks with continuous and/or discrete variables can be 
learned and compared from data.<br>
</dd> 
<dt> <strong>debug</strong> </dt> 
<dd> Debugger for R functions, with code display, graceful error recovery, 
line-numbered conditional breakpoints, access to exit code, flow control, and 
full keyboard input.<br>
</dd> 
<dt> <strong>degreenet</strong> </dt> 
<dd> Models for skewed count distributions relevant to networks. <br>
</dd> 
<dt> <strong>deldir</strong> </dt> 
<dd> Calculates the Delaunay triangulation and the Dirichlet or Voronoi 
tesselation (with respect to the entire plane) of a planar point set.<br>
</dd> 
<dt> <strong>delt</strong> </dt> 
<dd> Estimation of multivariate densities with adaptive histograms. <br>
</dd> 
<dt> <strong>demogR</strong> </dt> 
<dd> Analysis of age-structured demographic models. <br>
</dd> 
<dt> <strong>denpro</strong> </dt> 
<dd> Visualization of multivariate density functions and estimates with level 
set trees and shape trees, and visualization of multivariate data with tail 
trees.<br>
</dd> 
<dt> <strong>denstrip</strong> </dt> 
<dd> Density strips and other methods for compactly illustrating distributions.
<br> </dd> 
<dt> <strong>depmix</strong> </dt> 
<dd> Dependent Mixture Models: fit (multi-group) mixtures of latent Markov 
models on mixed categorical and continuous (time series) data.<br>
</dd> 
<dt> <strong>depmixS4</strong> </dt> 
<dd> Dependent Mixture Models: fit latent (hidden) Markov models on mixed 
categorical and continuous (time series) data.<br>
</dd> 
<dt> <strong>desirability</strong> </dt> 
<dd> Desirabiliy function optimization and ranking. <br>
</dd> 
<dt> <strong>dfcrm</strong> </dt> 
<dd> Dose-finding by the continual reassessment method. <br>
</dd> 
<dt> <strong>dglm</strong> </dt> 
<dd> Double generalized linear models. <br>
</dd> 
<dt> <strong>diagram</strong> </dt> 
<dd> Functions for visualising simple graphs (networks) and plotting flow 
diagrams.<br>
</dd> 
<dt> <strong>diamonds</strong> </dt> 
<dd> Functions for illustrating aperture-4 diamond partitions in the plane, or 
on the surface of an octahedron or icosahedron, for use as analysis or sampling 
grids.<br>
</dd> 
<dt> <strong>dice</strong> </dt> 
<dd> Calculate probabilities of various dice-rolling events. <br>
</dd> 
<dt> <strong>dichromat</strong> </dt> 
<dd> Color schemes for dichromats: collapse red-green distinctions to simulate 
the effects of colour-blindness.<br>
</dd> 
<dt> <strong>diffractometry</strong> </dt> 
<dd> Baseline identification and peak decomposition for x-ray diffractograms. 
<br> </dd> 
<dt> <strong>digest</strong> </dt> 
<dd> Two functions for the creation of &ldquo;hash&rdquo; digests of arbitrary 
R objects using the md5 and sha-1 algorithms permitting easy comparison of R 
language objects.<br>
</dd> 
<dt> <strong>diptest</strong> </dt> 
<dd> Compute Hartigan's dip test statistic for unimodality. <br>
</dd> 
<dt> <strong>dirichlet</strong> </dt> 
<dd> Dirichlet model of consumer buying behavior for marketing research. <br>

</dd> 
<dt> <strong>dispmod</strong> </dt> 
<dd> Functions for modelling dispersion in GLMs. <br>
</dd> 
<dt> <strong>distr</strong> </dt> 
<dd> An object orientated implementation of distributions and some additional 
functionality.<br>
</dd> 
<dt> <strong>distrDoc</strong> </dt> 
<dd> Documentation for packages <strong>distr</strong>, <strong>distrEx
</strong>, <strong>distrSim</strong>, and <strong>distrTEst</strong>. <br>
</dd>
<dt> <strong>distrEx</strong> </dt> 
<dd> Extensions of package <strong>distr</strong>. <br>
</dd> 
<dt> <strong>distrMod</strong> </dt> 
<dd> Object orientated implementation of probability models based on <strong>
distr</strong> and <strong>distrEx</strong>. <br>
</dd> 
<dt> <strong>distrSim</strong> </dt> 
<dd> Simulation classes based on package <strong>distr</strong>. <br>
</dd> 
<dt> <strong>distrTEst</strong> </dt> 
<dd> Estimation and Testing classes based on package <strong>distr</strong>. 
<br> </dd> 
<dt> <strong>distrTeach</strong> </dt> 
<dd> Extensions of <strong>distr</strong> for teaching stochastics/statistics 
in secondary school.<br>
</dd> 
<dt> <strong>distributions</strong> </dt> 
<dd> Probability distributions based on TI-83 Plus. <br>
</dd> 
<dt> <strong>divagis</strong> </dt> 
<dd> Tools for quality checks of georeferenced plant species accessions. <br>

</dd> 
<dt> <strong>diveMove</strong> </dt> 
<dd> Dive analysis and calibration. <br>
</dd> 
<dt> <strong>dlm</strong> </dt> 
<dd> Maximum likelihood and Bayesian analysis of Dynamic Linear Models. <br>

</dd> 
<dt> <strong>doBy</strong> </dt> 
<dd> Facilities for groupwise computations. <br>
</dd> 
<dt> <strong>dplR</strong> </dt> 
<dd> Dendrochronology Program Library in R. <br>
</dd> 
<dt> <strong>dprep</strong> </dt> 
<dd> Data preprocessing and visualization functions for classification. <br>

</dd> 
<dt> <strong>dr</strong> </dt> 
<dd> Functions, methods, and datasets for fitting dimension reduction 
regression, including pHd and inverse regression methods SIR and SAVE.<br>
</dd>
<dt> <strong>drc</strong> </dt> 
<dd> Non-linear regression analysis for multiple curves with focus on 
concentration-response, dose-response and time-response curves.<br>
</dd> 
<dt> <strong>drm</strong> </dt> 
<dd> Regression and association models for clustered categorical responses. 
<br> </dd> 
<dt> <strong>drfit</strong> </dt> 
<dd> Dose-response data evaluation. <br>
</dd> 
<dt> <strong>dse</strong> </dt> 
<dd> Dynamic System Estimation, a multivariate time series package bundle. 
Contains<strong>dse1</strong> (the base system, including multivariate ARMA and 
state space models) and<strong>dse2</strong> (extensions for evaluating 
estimation techniques, forecasting, and for evaluating forecasting model).<br>

</dd> 
<dt> <strong>dti</strong> </dt> 
<dd> DTI (Diffusion Tensor Image) analysis. <br>
</dd> 
<dt> <strong>dtt</strong> </dt> 
<dd> Discrete Trigonometric Transforms. <br>
</dd> 
<dt> <strong>dtw</strong> </dt> 
<dd> Dynamic Time Warping algorithms. <br>
</dd> 
<dt> <strong>dyn</strong> </dt> 
<dd> Time series regression. <br>
</dd> 
<dt> <strong>dynamicGraph</strong> </dt> 
<dd> Interactive graphical tool for manipulating graphs. <br>
</dd> 
<dt> <strong>dynamicTreeCut</strong> </dt> 
<dd> Methods for detection of clusters in hierarchical clustering dendrograms. 
<br> </dd> 
<dt> <strong>dynamo</strong> </dt> 
<dd> Estimation, simulation, regularization and prediction of univariate 
dynamic models including ARMA, ARMA-GARCH, ACD, and MEM.<br>
</dd> 
<dt> <strong>dynlm</strong> </dt> 
<dd> Dynamic linear models and time series regression. <br>
</dd> 
<dt> <strong>e1071</strong> </dt> 
<dd> Miscellaneous functions used at the Department of Statistics at TU Wien 
(E1071), including moments, short-time Fourier transforms, Independent 
Component Analysis, Latent Class Analysis, support vector machines, and fuzzy 
clustering, shortest path computation, bagged clustering, and some more.<br>

</dd> 
<dt> <strong>eRm</strong> </dt> 
<dd> Estimating extended Rasch models. <br>
</dd> 
<dt> <strong>earth</strong> </dt> 
<dd> Earth: multivariate adaptive regression spline models. <br>
</dd> 
<dt> <strong>eba</strong> </dt> 
<dd> Fitting and testing probabilistic choice models, especially the BTL, 
elimination-by-aspects (EBA), and preference tree (Pretree) models.<br>
</dd> 
<dt> <strong>ecespa</strong> </dt> 
<dd> Functions and data for spatial point pattern analysis. <br>
</dd> 
<dt> <strong>eco</strong> </dt> 
<dd> Fitting Bayesian models of ecological inference in 2 by 2 tables. <br>

</dd> 
<dt> <strong>ecodist</strong> </dt> 
<dd> Dissimilarity-based functions for ecological analysis. <br>
</dd> 
<dt> <strong>ecolMod</strong> </dt> 
<dd> Figures, data sets and examples from the book &ldquo;A Practical Guide to 
Ecological Modelling &mdash; Using R as a Simulation Platform&rdquo; by Karline 
Soetaert and Peter M. J. Herman, 2008, Springer. <br>
</dd> 
<dt> <strong>effects</strong> </dt> 
<dd> Graphical and tabular effect displays, e.g., of interactions, for linear 
and generalised linear models.<br>
</dd> 
<dt> <strong>eha</strong> </dt> 
<dd> A package for survival and event history analysis. <br>
</dd> 
<dt> <strong>eiPack</strong> </dt> 
<dd> Ecological inference and higher-dimension data management. <br>
</dd> 
<dt> <strong>eigenmodel</strong> </dt> 
<dd> Semiparametric factor and regression models for symmetric relational data.
<br> </dd> 
<dt> <strong>elasticnet</strong> </dt> 
<dd> Elastic net regularization and variable selection. <br>
</dd> 
<dt> <strong>ellipse</strong> </dt> 
<dd> Package for drawing ellipses and ellipse-like confidence regions. <br>

</dd> 
<dt> <strong>elliptic</strong> </dt> 
<dd> A suite of elliptic and related functions including Weierstrass and 
Jacobi forms.<br>
</dd> 
<dt> <strong>elrm</strong> </dt> 
<dd> Exact Logistic Regression via MCMC. <br>
</dd> 
<dt> <strong>emdbook</strong> </dt> 
<dd> Data sets and auxiliary functions for &ldquo;Ecological Models and 
Data&rdquo; by Ben Bolker (work in progress).<br>
</dd> 
<dt> <strong>emme2</strong> </dt> 
<dd> Functions to read from and write to an EMME/2 databank. <br>
</dd> 
<dt> <strong>empiricalBayes</strong> </dt> 
<dd> A bundle for dealing with extreme multiple testing problems by estimating 
local false discovery rates. Contains packages<strong>localFDR</strong> and 
<strong>HighProbability</strong>. <br>
</dd> 
<dt> <strong>emplik</strong> </dt> 
<dd> Empirical likelihood ratio for means/quantiles/hazards from possibly 
right censored data.<br>
</dd> 
<dt> <strong>emu</strong> </dt> 
<dd> Interface to the Emu speech database system. <br>
</dd> 
<dt> <strong>energy</strong> </dt> 
<dd> E-statistics (energy) tests for comparing distributions: multivariate 
normality, Poisson test, multivariate k-sample test for equal distributions, 
hierarchical clustering by e-distances.<br>
</dd> 
<dt> <strong>ensembleBMA</strong> </dt> 
<dd> Probabilistic forecasting using Bayesian Model Averaging of ensembles 
using a mixture of normal distributions.<br>
</dd> 
<dt> <strong>entropy</strong> </dt> 
<dd> Entropy estimation. <br>
</dd> 
<dt> <strong>epiR</strong> </dt> 
<dd> Functions for analyzing epidemiological data. <br>
</dd> 
<dt> <strong>epibasix</strong> </dt> 
<dd> Elementary functions for epidemiological analysis. <br>
</dd> 
<dt> <strong>epicalc</strong> </dt> 
<dd> Epidemiological calculator. <br>
</dd> 
<dt> <strong>epitools</strong> </dt> 
<dd> Basic tools for applied epidemiology. <br>
</dd> 
<dt> <strong>eqtl</strong> </dt> 
<dd> Tools for analyzing eQTL experiments. <br>
</dd> 
<dt> <strong>equivalence</strong> </dt> 
<dd> Tests and graphics for assessing tests of equivalence. <br>
</dd> 
<dt> <strong>ergm</strong> </dt> 
<dd> An integrated set of tools to analyze and simulate networks based on 
exponential-family random graph models (ERGM).<br>
</dd> 
<dt> <strong>etm</strong> </dt> 
<dd> Empirical Transition Matrix. <br>
</dd> 
<dt> <strong>evd</strong> </dt> 
<dd> Functions for extreme value distributions. Extends simulation, 
distribution, quantile and density functions to univariate, bivariate and (for 
simulation) multivariate parametric extreme value distributions, and provides 
fitting functions which calculate maximum likelihood estimates for univariate 
and bivariate models.<br>
</dd> 
<dt> <strong>evdbayes</strong> </dt> 
<dd> Functions for the bayesian analysis of extreme value models, using MCMC 
methods.<br>
</dd> 
<dt> <strong>evir</strong> </dt> 
<dd> Extreme Values in R: Functions for extreme value theory, which may be 
divided into the following groups; exploratory data analysis, block maxima, 
peaks over thresholds (univariate and bivariate), point processes, gev/gpd 
distributions.<br>
</dd> 
<dt> <strong>exactLoglinTest</strong> </dt> 
<dd> Monte Carlo exact tests for log-linear models. <br>
</dd> 
<dt> <strong>exactRankTests</strong> </dt> 
<dd> Computes exact p-values and quantiles using an implementation of the 
Streitberg/Roehmel shift algorithm.<br>
</dd> 
<dt> <strong>exactmaxsel</strong> </dt> 
<dd> Exact methods for maximally selected statistics for binary response 
variables.<br>
</dd> 
<dt> <strong>experiment</strong> </dt> 
<dd> Designing and analyzing randomized experiments. <br>
</dd> 
<dt> <strong>expert</strong> </dt> 
<dd> Modeling of data using expert opinion. <br>
</dd> 
<dt> <strong>extRemes</strong> </dt> 
<dd> Extreme value toolkit. <br>
</dd> 
<dt> <strong>fArma</strong> </dt> 
<dd> The Rmetrics module for &ldquo;ARMA Time Series Modelling&rdquo;. <br>

</dd> 
<dt> <strong>fAsianOptions</strong> </dt> 
<dd> The Rmetrics module for &ldquo;Option Valuation&rdquo;. <br>
</dd> 
<dt> <strong>fAssets</strong> </dt> 
<dd> The Rmetrics module for &ldquo;Assets Selection and Modelling&rdquo;. <br>
</dd> 
<dt> <strong>fBasics</strong> </dt> 
<dd> The Rmetrics module for &ldquo;Markets and Basic Statistics&rdquo;. <br>

</dd> 
<dt> <strong>fBonds</strong> </dt> 
<dd> The Rmetrics module for &ldquo;Bonds and Interest Rate Models&rdquo;. <br>
</dd> 
<dt> <strong>fCalendar</strong> </dt> 
<dd> The Rmetrics module for &ldquo;Chronological and Calendarical 
Objects&rdquo;.<br>
</dd> 
<dt> <strong>fCopulae</strong> </dt> 
<dd> The Rmetrics module for &ldquo;Dependence Structures with Copulas&rdquo;. 
<br> </dd> 
<dt> <strong>fEcofin</strong> </dt> 
<dd> The Rmetrics module for &ldquo;Economic and Financial Data Sets&rdquo;. 
<br> </dd> 
<dt> <strong>fExoticOptions</strong> </dt> 
<dd> The Rmetrics module for &ldquo;Option Valuation&rdquo;. <br>
</dd> 
<dt> <strong>fExtremes</strong> </dt> 
<dd> The Rmetrics module for &ldquo;Extreme Financial Market Data&rdquo;. <br>

</dd> 
<dt> <strong>fGarch</strong> </dt> 
<dd> The Rmetrics module for &ldquo;Autoregressive Conditional Heteroskedastic 
Modelling&rdquo;.<br>
</dd> 
<dt> <strong>fImport</strong> </dt> 
<dd> The Rmetrics module for &ldquo;Chronological and Calendarical 
Objects&rdquo;.<br>
</dd> 
<dt> <strong>fMultivar</strong> </dt> 
<dd> The Rmetrics module for &ldquo;Multivariate Market Analysis&rdquo;. <br>

</dd> 
<dt> <strong>fNonlinear</strong> </dt> 
<dd> The Rmetrics module for &ldquo;Nonlinear and Chaotic Time Series 
Modelling&rdquo;.<br>
</dd> 
<dt> <strong>fOptions</strong> </dt> 
<dd> The Rmetrics module for &ldquo;Basics of Option Valuation&rdquo;. <br>

</dd> 
<dt> <strong>fPortfolio</strong> </dt> 
<dd> The Rmetrics module for &ldquo;Portfolio Selection and 
Optimization&rdquo;.<br>
</dd> 
<dt> <strong>fRegression</strong> </dt> 
<dd> The Rmetrics module for &ldquo;Regression Based Decision and 
Prediction&rdquo;.<br>
</dd> 
<dt> <strong>fSeries</strong> </dt> 
<dd> The Rmetrics module for &ldquo;Financial Time Series Objects&rdquo;. <br>

</dd> 
<dt> <strong>fTrading</strong> </dt> 
<dd> The Rmetrics module for &ldquo;Technical Trading Analysis&rdquo;. <br>

</dd> 
<dt> <strong>fUnitRoots</strong> </dt> 
<dd> The Rmetrics module for &ldquo;The Dynamical Process Behind 
Markets&rdquo;.<br>
</dd> 
<dt> <strong>fUtilities</strong> </dt> 
<dd> The Rmetrics module for &ldquo;Rmetrics Function Utilities&rdquo;. <br>

</dd> 
<dt> <strong>fame</strong> </dt> 
<dd> Interface for FAME time series database. <br>
</dd> 
<dt> <strong>far</strong> </dt> 
<dd> Modelization for Functional AutoRegressive processes. <br>
</dd> 
<dt> <strong>faraway</strong> </dt> 
<dd> Functions and datasets for books by Julian Faraway. <br>
</dd> 
<dt> <strong>fastICA</strong> </dt> 
<dd> Implementation of FastICA algorithm to perform Independent Component 
Analysis (ICA) and Projection Pursuit.<br>
</dd> 
<dt> <strong>fbati</strong> </dt> 
<dd> Gene by environment interaction tests. <br>
</dd> 
<dt> <strong>fda</strong> </dt> 
<dd> Functional Data Analysis: analysis of data where the basic observation is 
a function of some sort.<br>
</dd> 
<dt> <strong>fdim</strong> </dt> 
<dd> Functions for calculating fractal dimension. <br>
</dd> 
<dt> <strong>fdrtool</strong> </dt> 
<dd> Estimation and control of (local) False Discovery Rates. <br>
</dd> 
<dt> <strong>feature</strong> </dt> 
<dd> Feature significance for multivariate kernel density estimation. <br>

</dd> 
<dt> <strong>femmeR</strong> </dt> 
<dd> Output analysis of FEMME model results. <br>
</dd> 
<dt> <strong>ff</strong> </dt> 
<dd> Flat file database designed for large vectors and multi-dimensional 
arrays.<br>
</dd> 
<dt> <strong>ffmanova</strong> </dt> 
<dd> Fifty-fifty MANOVA. <br>
</dd> 
<dt> <strong>fgac</strong> </dt> 
<dd> Families of Generalized Archimedean Copulas. <br>
</dd> 
<dt> <strong>fgui</strong> </dt> 
<dd> Function GUI. <br>
</dd> 
<dt> <strong>fields</strong> </dt> 
<dd> A collection of programs for curve and function fitting with an emphasis 
on spatial data. The major methods implemented include cubic and thin plate 
splines, universal Kriging and Kriging for large data sets. The main feature is 
that any covariance function implemented in R can be used for spatial 
prediction.<br>
</dd> 
<dt> <strong>filehash</strong> </dt> 
<dd> Simple file-based hash table. <br>
</dd> 
<dt> <strong>filehashSQLite</strong> </dt> 
<dd> Simple key-value database using SQLite as the backend. <br>
</dd> 
<dt> <strong>financial</strong> </dt> 
<dd> Solving financial problems in R. <br>
</dd> 
<dt> <strong>fingerprint</strong> </dt> 
<dd> Functions to operate on binary fingerprint data. <br>
</dd> 
<dt> <strong>flexclust</strong> </dt> 
<dd> Flexible cluster algorithms. <br>
</dd> 
<dt> <strong>flexmix</strong> </dt> 
<dd> Flexible Mixture Modeling: a general framework for finite mixtures of 
regression models using the EM algorithm.<br>
</dd> 
<dt> <strong>fmri</strong> </dt> 
<dd> Functions for the analysis of fMRI experiments. <br>
</dd> 
<dt> <strong>forecasting</strong> </dt> 
<dd> A bundle with functions and datasets for forecasting. Contains <strong>
forecast</strong> (time series forecasting), <strong>fma</strong> (data sets 
from the book &ldquo;Forecasting: Methods and Applications&rdquo; by 
Makridakis, Wheelwright &amp; Hyndman, 1998), and<strong>Mcomp</strong> (data 
from the M-competitions).<br>
</dd> 
<dt> <strong>foreign</strong> </dt> 
<dd> Functions for reading and writing data stored by statistical software 
like Minitab, S, SAS, SPSS, Stata, Systat, etc.<em>Recommended</em>. <br>
</dd> 
<dt> <strong>forensic</strong> </dt> 
<dd> Statistical methods in forensic genetics. <br>
</dd> 
<dt> <strong>fork</strong> </dt> 
<dd> Functions for handling multiple processes: simple wrappers around the 
Unix process management API calls.<br>
</dd> 
<dt> <strong>fortunes</strong> </dt> 
<dd> R fortunes. <br>
</dd> 
<dt> <strong>forward</strong> </dt> 
<dd> Forward search approach to robust analysis in linear and generalized 
linear regression models.<br>
</dd> 
<dt> <strong>fpc</strong> </dt> 
<dd> Fixed point clusters, clusterwise regression and discriminant plots. <br>

</dd> 
<dt> <strong>fpca</strong> </dt> 
<dd> Restricted MLE for Functional Principal Components Analysis. <br>
</dd> 
<dt> <strong>fracdiff</strong> </dt> 
<dd> Maximum likelihood estimation of the parameters of a fractionally 
differenced ARIMA(p,d,q) model (Haslett and Raftery, Applied Statistics, 1989).
<br> </dd> 
<dt> <strong>fractal</strong> </dt> 
<dd> Insightful fractal time series modeling and analysis. <br>
</dd> 
<dt> <strong>frailtypack</strong> </dt> 
<dd> Fit a shared gamma frailty model and Cox proportional hazards model using 
a Penalized Likelihood on the hazard function.<br>
</dd> 
<dt> <strong>fso</strong> </dt> 
<dd> Fuzzy set ordination. <br>
</dd> 
<dt> <strong>ftnonpar</strong> </dt> 
<dd> Features and strings for nonparametric regression. <br>
</dd> 
<dt> <strong>fts</strong> </dt> 
<dd> Fast operations for time series objects via an interface to tslib (a C++ 
time series library).<br>
</dd> 
<dt> <strong>fuzzyFDR</strong> </dt> 
<dd> Exact calculation of fuzzy decision rules for multiple testing. <br>
</dd>
<dt> <strong>fuzzyOP</strong> </dt> 
<dd> Fuzzy numbers and the main mathematical operations on these. <br>
</dd> 
<dt> <strong>fuzzyRankTests</strong> </dt> 
<dd> Fuzzy rank tests and confidence intervals. <br>
</dd> 
<dt> <strong>fxregime</strong> </dt> 
<dd> Frankel-Wei regression and structural change tools for estimating, 
testing, dating and monitoring (de facto) exchange rate regimes.<br>
</dd> 
<dt> <strong>g.data</strong> </dt> 
<dd> Create and maintain delayed-data packages (DDP's). <br>
</dd> 
<dt> <strong>gRain</strong> </dt> 
<dd> Probability propagation in graphical independence networks. <br>
</dd> 
<dt> <strong>gRbase</strong> </dt> 
<dd> A package for graphical modelling in R. Defines S4 classes for graphical 
meta data and graphical models, and illustrates how hierarchical log-linear 
models may be implemented and combined with<strong>dynamicGraph</strong>. <br>

</dd> 
<dt> <strong>gRc</strong> </dt> 
<dd> Inference in graphical Gaussian models with edge and vertex symmetries. 
<br> </dd> 
<dt> <strong>gWidgets</strong> </dt> 
<dd> gWidgets API for building toolkit-independent, interactive GUIs. <br>

</dd> 
<dt> <strong>gWidgetsRGtk2</strong> </dt> 
<dd> Toolkit implementation of <strong>gWidgets</strong> for <strong>RGtk2
</strong>. <br>
</dd> 
<dt> <strong>gWidgetsWWW</strong> </dt> 
<dd> Toolkit implementation of gWidgets for www. <br>
</dd> 
<dt> <strong>gWidgetsrJava</strong> </dt> 
<dd> Toolkit implementation of <strong>gWidgets</strong> for <strong>rJava
</strong>. <br>
</dd> 
<dt> <strong>gWidgetstcltk</strong> </dt> 
<dd> Toolkit implementation of <strong>gWidgets</strong> for <strong>tcltk
</strong>. <br>
</dd> 
<dt> <strong>gafit</strong> </dt> 
<dd> Genetic algorithm for curve fitting. <br>
</dd> 
<dt> <strong>gam</strong> </dt> 
<dd> Functions for fitting and working with Generalized Additive Models, as 
described in chapter 7 of the White Book, and in &ldquo;Generalized Additive 
Models&rdquo; by T. Hastie and R. Tibshirani (1990).<br>
</dd> 
<dt> <strong>gamair</strong> </dt> 
<dd> Data sets used in the book &ldquo;Generalized Additive Models: An 
Introduction with R&rdquo; by S. Wood (2006).<br>
</dd> 
<dt> <strong>gamlss</strong> </dt> 
<dd> Functions to fit Generalized Additive Models for Location Scale and Shape.
<br> </dd> 
<dt> <strong>gamlss.cens</strong> </dt> 
<dd> A GAMLSS add on package for censored data. <br>
</dd> 
<dt> <strong>gamlss.dist</strong> </dt> 
<dd> Extra distributions for GAMLSS modeling. <br>
</dd> 
<dt> <strong>gamlss.mx</strong> </dt> 
<dd> A GAMLSS add on package for fitting mixture distributions. <br>
</dd> 
<dt> <strong>gamlss.nl</strong> </dt> 
<dd> A GAMLSS add on package for fitting non linear parametric models. <br>

</dd> 
<dt> <strong>gamlss.tr</strong> </dt> 
<dd> A GAMLSS add on for generating and fitting truncated (gamlss.family) 
distributions.<br>
</dd> 
<dt> <strong>gap</strong> </dt> 
<dd> Genetic analysis package for both population and family data. <br>
</dd> 
<dt> <strong>gbev</strong> </dt> 
<dd> Gradient Boosted regression trees with Errors-in-Variables. <br>
</dd> 
<dt> <strong>gbm</strong> </dt> 
<dd> Generalized Boosted Regression Models: implements extensions to Freund 
and Schapire's AdaBoost algorithm and J. Friedman's gradient boosting machine. 
Includes regression methods for least squares, absolute loss, logistic, 
Poisson, Cox proportional hazards partial likelihood, and AdaBoost exponential 
loss.<br>
</dd> 
<dt> <strong>gbs</strong> </dt> 
<dd> Generalized Birnbaum-Saunders distributions. <br>
</dd> 
<dt> <strong>gcl</strong> </dt> 
<dd> Compute a fuzzy rules or tree classifier from data. <br>
</dd> 
<dt> <strong>gclus</strong> </dt> 
<dd> Clustering Graphics. Orders panels in scatterplot matrices and parallel 
coordinate displays by some merit index.<br>
</dd> 
<dt> <strong>gcmrec</strong> </dt> 
<dd> Parameters estimation of the general semiparametric model for recurrent 
event data proposed by Pe&ntilde;a and Hollander.<br>
</dd> 
<dt> <strong>gdata</strong> </dt> 
<dd> Various functions to manipulate data. <br>
</dd> 
<dt> <strong>gee</strong> </dt> 
<dd> An implementation of the Liang/Zeger generalized estimating equation 
approach to GLMs for dependent data.<br>
</dd> 
<dt> <strong>geepack</strong> </dt> 
<dd> Generalized estimating equations solver for parameters in mean, scale, 
and correlation structures, through mean link, scale link, and correlation 
link. Can also handle clustered categorical responses.<br>
</dd> 
<dt> <strong>geiger</strong> </dt> 
<dd> Analysis of evolutionary diversification. <br>
</dd> 
<dt> <strong>genalg</strong> </dt> 
<dd> R based genetic algorithm for binary and floating point chromosomes. <br>

</dd> 
<dt> <strong>gene2pathway</strong> </dt> 
<dd> Prediction of KEGG pathway membership for individual genes based on 
InterPro domain signatures.<br>
</dd> 
<dt> <strong>genetics</strong> </dt> 
<dd> Classes and methods for handling genetic data. Includes classes to 
represent genotypes and haplotypes at single markers up to multiple markers on 
multiple chromosomes, and functions for allele frequencies, flagging 
homo/heterozygotes, flagging carriers of certain alleles, computing 
disequlibrium, testing Hardy-Weinberg equilibrium,... <br>
</dd> 
<dt> <strong>geoR</strong> </dt> 
<dd> Functions to perform geostatistical data analysis including model-based 
methods.<br>
</dd> 
<dt> <strong>geoRglm</strong> </dt> 
<dd> Functions for inference in generalised linear spatial models. <br>
</dd> 
<dt> <strong>geomapdata</strong> </dt> 
<dd> Data for topographic and geologic mapping. <br>
</dd> 
<dt> <strong>geometry</strong> </dt> 
<dd> Mesh generation and surface tesselation, based on the Qhull library. <br>

</dd> 
<dt> <strong>geozoo</strong> </dt> 
<dd> Definition of geometric objects and display via <strong>rggobi</strong>. 
<br> </dd> 
<dt> <strong>getopt</strong> </dt> 
<dd> C-like getopt behavior for R scripts. <br>
</dd> 
<dt> <strong>ggm</strong> </dt> 
<dd> Functions for defining directed acyclic graphs and undirected graphs, 
finding induced graphs and fitting Gaussian Markov models.<br>
</dd> 
<dt> <strong>ggplot</strong> </dt> 
<dd> Grammar of graphics based plots for R. <br>
</dd> 
<dt> <strong>ggplot2</strong> </dt> 
<dd> An implementation of the Grammar of Graphics in R. <br>
</dd> 
<dt> <strong>ghyp</strong> </dt> 
<dd> Univariate and multivariate generalized hyperbolic distributions. <br>

</dd> 
<dt> <strong>giRaph</strong> </dt> 
<dd> Data structures and algorithms for computations on graphs. <br>
</dd> 
<dt> <strong>gibbs.met</strong> </dt> 
<dd> Naive Gibbs sampling with Metropolis steps. <br>
</dd> 
<dt> <strong>glasso</strong> </dt> 
<dd> Graphical lasso. <br>
</dd> 
<dt> <strong>gld</strong> </dt> 
<dd> Basic functions for the generalised (Tukey) lambda distribution. <br>

</dd> 
<dt> <strong>gllm</strong> </dt> 
<dd> Routines for log-linear models of incomplete contingency tables, 
including some latent class models via EM and Fisher scoring approaches.<br>

</dd> 
<dt> <strong>glmc</strong> </dt> 
<dd> Fitting Generalized Linear Models subject to Constraints. <br>
</dd> 
<dt> <strong>glmmAK</strong> </dt> 
<dd> Generalized Linear Mixed Models. <br>
</dd> 
<dt> <strong>glmmBUGS</strong> </dt> 
<dd> Generalised Linear Mixed Models with WinBUGS. <br>
</dd> 
<dt> <strong>glmmML</strong> </dt> 
<dd> A Maximum Likelihood approach to generalized linear models with random 
intercept.<br>
</dd> 
<dt> <strong>glmnet</strong> </dt> 
<dd> Lasso and elastic-net regularized generalized linear models. <br>
</dd> 
<dt> <strong>glmpath</strong> </dt> 
<dd> L1 regularization path for Generalized Linear Models. <br>
</dd> 
<dt> <strong>glpk</strong> </dt> 
<dd> Interface to the GNU Linear Programming Kit (GLPK). <br>
</dd> 
<dt> <strong>gmaps</strong> </dt> 
<dd> Wrapper and auxiliary functions for the <strong>maps</strong> package to 
work with the grid graphics system.<br>
</dd> 
<dt> <strong>gmodels</strong> </dt> 
<dd> Various functions to manipulate models. <br>
</dd> 
<dt> <strong>gmp</strong> </dt> 
<dd> Arithmetic &ldquo;without limitations&rdquo; using the GNU Multiple 
Precision library.<br>
</dd> 
<dt> <strong>gmt</strong> </dt> 
<dd> Interface between the GMT 4.0 map-making software and R. <br>
</dd> 
<dt> <strong>gnm</strong> </dt> 
<dd> Functions to specify and fit generalized nonlinear models, including 
models with multiplicative interaction terms such as the UNIDIFF model from 
sociology and the AMMI model from crop science.<br>
</dd> 
<dt> <strong>goalprog</strong> </dt> 
<dd> Weighted and lexicographical goal programming and optimization. <br>
</dd>
<dt> <strong>gpclib</strong> </dt> 
<dd> General polygon clipping routines for R based on Alan Murta's C library. 
<br> </dd> 
<dt> <strong>gpls</strong> </dt> 
<dd> Classification using generalized partial least squares for two-group and 
multi-group (more than 2 group) classification.<br>
</dd> 
<dt> <strong>gplots</strong> </dt> 
<dd> Various functions to draw plots. <br>
</dd> 
<dt> <strong>grImport</strong> </dt> 
<dd> Importing vector graphics. <br>
</dd> 
<dt> <strong>grade</strong> </dt> 
<dd> Binary grading functions. <br>
</dd> 
<dt> <strong>granova</strong> </dt> 
<dd> Graphical Analysis of Variance. <br>
</dd> 
<dt> <strong>graph</strong> </dt> 
<dd> Handling of graph data structures. <br>
</dd> 
<dt> <strong>grasp</strong> </dt> 
<dd> Generalized Regression Analysis and Spatial Predictions for R. <br>
</dd> 
<dt> <strong>gregmisc</strong> </dt> 
<dd> Miscellaneous functions written/maintained by Gregory R. Warnes. <br>

</dd> 
<dt> <strong>gridBase</strong> </dt> 
<dd> Integration of base and grid graphics. <br>
</dd> 
<dt> <strong>grnnR</strong> </dt> 
<dd> A Generalized Regression Neural Network. <br>
</dd> 
<dt> <strong>grouped</strong> </dt> 
<dd> Regression models for grouped and coarse data, under the Coarsened At 
Random assumption.<br>
</dd> 
<dt> <strong>grplasso</strong> </dt> 
<dd> Fit user specified models with group lasso penalty. <br>
</dd> 
<dt> <strong>gsarima</strong> </dt> 
<dd> functions for Generalized SARIMA time series simulation. <br>
</dd> 
<dt> <strong>gsl</strong> </dt> 
<dd> Wrapper for special functions of the Gnu Scientific Library (GSL). <br>

</dd> 
<dt> <strong>gss</strong> </dt> 
<dd> A comprehensive package for structural multivariate function estimation 
using smoothing splines.<br>
</dd> 
<dt> <strong>gstat</strong> </dt> 
<dd> multivariable geostatistical modelling, prediction and simulation. 
Includes code for variogram modelling; simple, ordinary and universal point or 
block (co)kriging, sequential Gaussian or indicator (co)simulation, and map 
plotting functions.<br>
</dd> 
<dt> <strong>gsubfn</strong> </dt> 
<dd> Miscellaneous string utilities. <br>
</dd> 
<dt> <strong>gtools</strong> </dt> 
<dd> Various functions to help manipulate data. <br>
</dd> 
<dt> <strong>gumbel</strong> </dt> 
<dd> Functions for the Gumbel-Hougaard copula. <br>
</dd> 
<dt> <strong>gvlma</strong> </dt> 
<dd> Global Validation of Linear Models Assumptions. <br>
</dd> 
<dt> <strong>hapassoc</strong> </dt> 
<dd> Likelihood inference of trait associations with SNP haplotypes and other 
attributes using the EM Algorithm.<br>
</dd> 
<dt> <strong>haplo.ccs</strong> </dt> 
<dd> Estimate haplotype relative risks in case-control data. <br>
</dd> 
<dt> <strong>haplo.stats</strong> </dt> 
<dd> Statistical analysis of haplotypes with traits and covariates when 
linkage phase is ambiguous.<br>
</dd> 
<dt> <strong>hapsim</strong> </dt> 
<dd> Haplotype data simulation. <br>
</dd> 
<dt> <strong>hbim</strong> </dt> 
<dd> Hill/Bliss Independence Model for combination vaccines. <br>
</dd> 
<dt> <strong>hddplot</strong> </dt> 
<dd> Use known groups in high-dimensional data to derive scores for plots. <br>
</dd> 
<dt> <strong>hdeco</strong> </dt> 
<dd> Hierarchical DECOmposition of entropy for categorical map comparisons. 
<br> </dd> 
<dt> <strong>hdf5</strong> </dt> 
<dd> Interface to the NCSA HDF5 library. <br>
</dd> 
<dt> <strong>hdrcde</strong> </dt> 
<dd> Highest Density Regions and Conditional Density Estimation. <br>
</dd> 
<dt> <strong>heatmap.plus</strong> </dt> 
<dd> Heatmap with sensible behavior. <br>
</dd> 
<dt> <strong>helloJavaWorld</strong> </dt> 
<dd> A demonstration how to interface to a jar file that resides inside an R 
package.<br>
</dd> 
<dt> <strong>heplots</strong> </dt> 
<dd> Visualizing tests in multivariate linear models. <br>
</dd> 
<dt> <strong>hett</strong> </dt> 
<dd> Functions for the fitting and summarizing of heteroscedastic t-regression.
<br> </dd> 
<dt> <strong>hexView</strong> </dt> 
<dd> Viewing binary files. <br>
</dd> 
<dt> <strong>hier.part</strong> </dt> 
<dd> Hierarchical Partitioning: variance partition of a multivariate data set. 
<br> </dd> 
<dt> <strong>hierfstat</strong> </dt> 
<dd> Estimation of hierarchical F-statistics from haploid or diploid genetic 
data with any numbers of levels in the hierarchy, and tests for the 
significance of each F and variance components.<br>
</dd> 
<dt> <strong>hints</strong> </dt> 
<dd> Provide hints on what to do next. <br>
</dd> 
<dt> <strong>hlr</strong> </dt> 
<dd> Hidden logistic regression. <br>
</dd> 
<dt> <strong>hmm.discnp</strong> </dt> 
<dd> Hidden Markov models with discrete non-parametric observation 
distributions.<br>
</dd> 
<dt> <strong>hoa</strong> </dt> 
<dd> A bundle of packages for higher order likelihood-based inference. Contains
<strong>cond</strong> for approximate conditional inference for logistic and 
loglinear models,<strong>csampling</strong> for conditional simulation in 
regression-scale models,<strong>marg</strong> for approximate marginal 
inference for regression-scale models, and<strong>nlreg</strong> for higher 
order inference for nonlinear heteroscedastic models.<br>
</dd> 
<dt> <strong>homals</strong> </dt> 
<dd> Homogeneity Analysis (HOMALS) package with optional Tcl/Tk interface. <br>
</dd> 
<dt> <strong>homtest</strong> </dt> 
<dd> Homogeneity tests for regional frequency analysis. <br>
</dd> 
<dt> <strong>hopach</strong> </dt> 
<dd> Hierarchical Ordered Partitioning and Collapsing Hybrid (HOPACH). <br>

</dd> 
<dt> <strong>hot</strong> </dt> 
<dd> Computation on micro-arrays. <br>
</dd> 
<dt> <strong>howmany</strong> </dt> 
<dd> A lower bound for the number of correct rejections. <br>
</dd> 
<dt> <strong>hsmm</strong> </dt> 
<dd> Hidden Semi Markov Models. <br>
</dd> 
<dt> <strong>httpRequest</strong> </dt> 
<dd> Implements HTTP Request protocols (GET, POST, and multipart POST 
requests).<br>
</dd> 
<dt> <strong>hwde</strong> </dt> 
<dd> Models and tests for departure from Hardy-Weinberg equilibrium and 
independence between loci.<br>
</dd> 
<dt> <strong>hwriter</strong> </dt> 
<dd> Easy-to-use and versatile functions to output R objects in HTML format. 
<br> </dd> 
<dt> <strong>hybridHclust</strong> </dt> 
<dd> Hybrid hierarchical clustering via mutual clusters. <br>
</dd> 
<dt> <strong>hydrogeo</strong> </dt> 
<dd> Groundwater data presentation and interpretation. <br>
</dd> 
<dt> <strong>hydrosanity</strong> </dt> 
<dd> Graphical user interface for exploring hydrological time series. <br>

</dd> 
<dt> <strong>hypergeo</strong> </dt> 
<dd> The hypergeometric function over the whole complex plane. <br>
</dd> 
<dt> <strong>ibdreg</strong> </dt> 
<dd> Regression methods for IBD linkage with covariates. <br>
</dd> 
<dt> <strong>ic.infer</strong> </dt> 
<dd> Inequality constrained inference in linear normal situations. <br>
</dd> 
<dt> <strong>ic50</strong> </dt> 
<dd> Evaluation of compound screens. <br>
</dd> 
<dt> <strong>icomp</strong> </dt> 
<dd> Calculates the ICOMP criterion and its variations. <br>
</dd> 
<dt> <strong>identity</strong> </dt> 
<dd> Jacquard condensed coefficients of identity. <br>
</dd> 
<dt> <strong>ifa</strong> </dt> 
<dd> Independent Factor Analysis. <br>
</dd> 
<dt> <strong>ifs</strong> </dt> 
<dd> Iterated Function Systems distribution function estimator. <br>
</dd> 
<dt> <strong>ifultools</strong> </dt> 
<dd> Insightful research tools. <br>
</dd> 
<dt> <strong>ig</strong> </dt> 
<dd> Robust and classical versions of the inverse Gaussian distribution. <br>

</dd> 
<dt> <strong>igraph</strong> </dt> 
<dd> Routines for simple graphs. <br>
</dd> 
<dt> <strong>iid.test</strong> </dt> 
<dd> Testing whether data is independent and identically distributed. <br>

</dd> 
<dt> <strong>impute</strong> </dt> 
<dd> Imputation for microarray data (currently KNN only). <br>
</dd> 
<dt> <strong>ineq</strong> </dt> 
<dd> Inequality, concentration and poverty measures, and Lorenz curves 
(empirical and theoretic).<br>
</dd> 
<dt> <strong>inetwork</strong> </dt> 
<dd> Network analysis and plotting. <br>
</dd> 
<dt> <strong>inline</strong> </dt> 
<dd> Inline C/C++ function calls from R. <br>
</dd> 
<dt> <strong>intcox</strong> </dt> 
<dd> Implementation of the Iterated Convex Minorant Algorithm for the Cox 
proportional hazard model for interval censored event data. <br>
</dd> 
<dt> <strong>iplots</strong> </dt> 
<dd> Interactive graphics for R. <br>
</dd> 
<dt> <strong>ipred</strong> </dt> 
<dd> Improved predictive models by direct and indirect bootstrap aggregation 
in classification and regression as well as resampling based estimators of 
prediction error.<br>
</dd> 
<dt> <strong>irr</strong> </dt> 
<dd> Coefficients of Interrater Reliability and Agreement for quantitative, 
ordinal and nominal data.<br>
</dd> 
<dt> <strong>irtoys</strong> </dt> 
<dd> Simple interface to the estimation and plotting of IRT models. <br>
</dd> 
<dt> <strong>ismev</strong> </dt> 
<dd> Functions to support the computations carried out in &ldquo;An 
Introduction to Statistical Modeling of Extreme Values;' by S. Coles, 2001, 
Springer. The functions may be divided into the following groups; 
maxima/minima, order statistics, peaks over thresholds and point processes.<br>

</dd> 
<dt> <strong>its</strong> </dt> 
<dd> An S4 class for handling irregular time series. <br>
</dd> 
<dt> <strong>ivivc</strong> </dt> 
<dd> In vitro in vivo correlation (IVIVC) modeling. <br>
</dd> 
<dt> <strong>jit</strong> </dt> 
<dd> Just-in-time compiler. <br>
</dd> 
<dt> <strong>kappalab</strong> </dt> 
<dd> The &ldquo;laboratory for capacities&rdquo;, an S4 tool box for capacity 
(or non-additive measure, fuzzy measure) and integral manipulation on a finite 
setting.<br>
</dd> 
<dt> <strong>kerfdr</strong> </dt> 
<dd> Semi-parametric kernel-based approach to local fdr estimations. <br>
</dd>
<dt> <strong>kernelPop</strong> </dt> 
<dd> Spatially explicit population genetic simulations. <br>
</dd> 
<dt> <strong>kernlab</strong> </dt> 
<dd> Kernel-based machine learning methods including support vector machines. 
<br> </dd> 
<dt> <strong>kin.cohort</strong> </dt> 
<dd> Analysis of kin-cohort studies. <br>
</dd> 
<dt> <strong>kinship</strong> </dt> 
<dd> Mixed-effects Cox models, sparse matrices, and modeling data from large 
pedigrees.<br>
</dd> 
<dt> <strong>kknn</strong> </dt> 
<dd> Weighted k-nearest neighbors classification and regression. <br>
</dd> 
<dt> <strong>klaR</strong> </dt> 
<dd> Miscellaneous functions for classification and visualization developed at 
the Department of Statistics, University of Dortmund.<br>
</dd> 
<dt> <strong>klin</strong> </dt> 
<dd> Linear equations with Kronecker structure. <br>
</dd> 
<dt> <strong>kml</strong> </dt> 
<dd> K-Means for Longitudinal data. <br>
</dd> 
<dt> <strong>knnFinder</strong> </dt> 
<dd> Fast nearest neighbor search. <br>
</dd> 
<dt> <strong>knnTree</strong> </dt> 
<dd> Construct or predict with k-nearest-neighbor classifiers, using 
cross-validation to select k, choose variables (by forward or backwards 
selection), and choose scaling (from among no scaling, scaling each column by 
its SD, or scaling each column by its MAD). The finished classifier will 
consist of a classification tree with one such k-nn classifier in each leaf.<br>
</dd> 
<dt> <strong>knncat</strong> </dt> 
<dd> Nearest-neighbor classification with categorical variables. <br>
</dd> 
<dt> <strong>knnflex</strong> </dt> 
<dd> A more flexible k-NN. <br>
</dd> 
<dt> <strong>knorm</strong> </dt> 
<dd> Microarray data from multiple biologically interrelated experiments. <br>

</dd> 
<dt> <strong>kohonen</strong> </dt> 
<dd> Supervised and unsupervised self-organising maps. <br>
</dd> 
<dt> <strong>ks</strong> </dt> 
<dd> Kernel smoothing: bandwidth matrices for kernel density estimators and 
kernel discriminant analysis for bivariate data.<br>
</dd> 
<dt> <strong>kza</strong> </dt> 
<dd> Kolmogorov-Zurbenko Adpative filter for locating change points in a time 
series.<br>
</dd> 
<dt> <strong>kzft</strong> </dt> 
<dd> Kolmogorov-Zurbenko Fourier Transform and application. <br>
</dd> 
<dt> <strong>kzs</strong> </dt> 
<dd> Kolmogorov-Zurbenko Spline. <br>
</dd> 
<dt> <strong>labdsv</strong> </dt> 
<dd> Laboratory for Dynamic Synthetic Vegephenomenology. <br>
</dd> 
<dt> <strong>labstatR</strong> </dt> 
<dd> Functions for the book &ldquo;Laboratorio di statistica con R&rdquo; by 
S. M. Iacus and G. Masarotto, 2002, McGraw-Hill. Function names and 
documentation in Italian.<br>
</dd> 
<dt> <strong>laercio</strong> </dt> 
<dd> Tests to compare means. <br>
</dd> 
<dt> <strong>lago</strong> </dt> 
<dd> LAGO for rare target detection. <br>
</dd> 
<dt> <strong>lancet.iraqmortality</strong> </dt> 
<dd> Surveys of Iraq mortality published in The Lancet. <br>
</dd> 
<dt> <strong>languageR</strong> </dt> 
<dd> Data sets and functions for the book &ldquo;Analyzing Linguistic Data: A 
practical introduction to statistics&rdquo; by R. H. Baayen, 2007, Cambridge: 
Cambridge University Press.<br>
</dd> 
<dt> <strong>lars</strong> </dt> 
<dd> Least Angle Regression, Lasso and Forward Stagewise: efficient procedures 
for fitting an entire lasso sequence with the cost of a single least squares 
fit.<br>
</dd> 
<dt> <strong>laser</strong> </dt> 
<dd> Likelihood Analysis of Speciation/Extinction Rates from phylogenies. <br>

</dd> 
<dt> <strong>lasso2</strong> </dt> 
<dd> Routines and documentation for solving regression problems while imposing 
an L1 constraint on the estimates, based on the algorithm of Osborne et al. 
(1998).<br>
</dd> 
<dt> <strong>latentnet</strong> </dt> 
<dd> Latent position and cluster models for statistical networks. <br>
</dd> 
<dt> <strong>latentnetHRT</strong> </dt> 
<dd> Latent position and cluster models for statistical networks, implementing 
the original specification by Handcock, Raftery and Tantrum.<br>
</dd> 
<dt> <strong>lattice</strong> </dt> 
<dd> Lattice graphics, an implementation of Trellis Graphics functions. <em>
Recommended</em>. <br>
</dd> 
<dt> <strong>latticeExtra</strong> </dt> 
<dd> Generic functions and standard methods for Trellis-based displays. <br>

</dd> 
<dt> <strong>lawstat</strong> </dt> 
<dd> Statistical tests widely utilized in biostatistics, public policy and law.
<br> </dd> 
<dt> <strong>lazy</strong> </dt> 
<dd> Lazy learning for local regression. <br>
</dd> 
<dt> <strong>ldDesign</strong> </dt> 
<dd> Design of experiments for detection of linkage disequilibrium, <br>
</dd> 
<dt> <strong>lda.cv</strong> </dt> 
<dd> Cross-validation for linear discriminant analysis. <br>
</dd> 
<dt> <strong>ldbounds</strong> </dt> 
<dd> Lan-DeMets method for group sequential boundaries. <br>
</dd> 
<dt> <strong>leaps</strong> </dt> 
<dd> A package which performs an exhaustive search for the best subsets of a 
given set of potential regressors, using a branch-and-bound algorithm, and also 
performs searches using a number of less time-consuming techniques.<br>
</dd> 
<dt> <strong>lga</strong> </dt> 
<dd> Tools for Linear Grouping Analysis (LGA). <br>
</dd> 
<dt> <strong>lgtdl</strong> </dt> 
<dd> A set of methods for longitudinal data objects. <br>
</dd> 
<dt> <strong>lhs</strong> </dt> 
<dd> Latin Hypercube Samples. <br>
</dd> 
<dt> <strong>limSolve</strong> </dt> 
<dd> Solving linear inverse models. <br>
</dd> 
<dt> <strong>linprog</strong> </dt> 
<dd> Solve linear programming/linear optimization problems by using the 
simplex algorithm.<br>
</dd> 
<dt> <strong>ljr</strong> </dt> 
<dd> Logistic Joinpoint Regression. <br>
</dd> 
<dt> <strong>lme4</strong> </dt> 
<dd> Fit linear and generalized linear mixed-effects models. <br>
</dd> 
<dt> <strong>lmeSplines</strong> </dt> 
<dd> Fit smoothing spline terms in Gaussian linear and nonlinear mixed-effects 
models.<br>
</dd> 
<dt> <strong>lmm</strong> </dt> 
<dd> Linear mixed models. <br>
</dd> 
<dt> <strong>lmom</strong> </dt> 
<dd> L-moments. <br>
</dd> 
<dt> <strong>lmomco</strong> </dt> 
<dd> L-moments and L-comoments. <br>
</dd> 
<dt> <strong>lmtest</strong> </dt> 
<dd> A collection of tests on the assumptions of linear regression models from 
the book &ldquo;The linear regression model under test&rdquo; by W. Kraemer and 
H. Sonnberger, 1986, Physica.<br>
</dd> 
<dt> <strong>lnMLE</strong> </dt> 
<dd> Marginally specified logistic normal models for longitudinal binary data. 
<br> </dd> 
<dt> <strong>locfdr</strong> </dt> 
<dd> Computation of local false discovery rates. <br>
</dd> 
<dt> <strong>locfit</strong> </dt> 
<dd> Local Regression, likelihood and density estimation. <br>
</dd> 
<dt> <strong>locpol</strong> </dt> 
<dd> Kernel local polynomial regression. <br>
</dd> 
<dt> <strong>lodplot</strong> </dt> 
<dd> Assorted plots of location score versus genetic map position. <br>
</dd> 
<dt> <strong>logcondens</strong> </dt> 
<dd> Estimate a log-concave probability density from i.i.d. observations. <br>

</dd> 
<dt> <strong>logilasso</strong> </dt> 
<dd> Analysis of sparse contingency tables with penalization approaches. <br>

</dd> 
<dt> <strong>logistf</strong> </dt> 
<dd> Firth's bias reduced logistic regression approach with penalized profile 
likelihood based confidence intervals for parameter estimates.<br>
</dd> 
<dt> <strong>loglognorm</strong> </dt> 
<dd> Double log normal distribution functions. <br>
</dd> 
<dt> <strong>logregperm</strong> </dt> 
<dd> Inference in Logistic Regression using permutation tests. <br>
</dd> 
<dt> <strong>logspline</strong> </dt> 
<dd> Logspline density estimation. <br>
</dd> 
<dt> <strong>lokern</strong> </dt> 
<dd> Kernel regression smoothing with adaptive local or global plug-in 
bandwidth selection.<br>
</dd> 
<dt> <strong>longmemo</strong> </dt> 
<dd> Datasets and Functionality from the textbook &ldquo;Statistics for 
Long-Memory Processes&rdquo; by J. Beran, 1994, Chapman &amp; Hall.<br>
</dd> 
<dt> <strong>lpSolve</strong> </dt> 
<dd> Functions that solve general linear/integer problems, assignment 
problems, and transportation problems via interfacing Lp_solve.<br>
</dd> 
<dt> <strong>lpSolveAPI</strong> </dt> 
<dd> An R interface to the lp_solve library API. <br>
</dd> 
<dt> <strong>lpc</strong> </dt> 
<dd> Lassoed principal components for testing significance of features. <br>

</dd> 
<dt> <strong>lpridge</strong> </dt> 
<dd> Local polynomial (ridge) regression. <br>
</dd> 
<dt> <strong>lsa</strong> </dt> 
<dd> Latent Semantic Analysis. <br>
</dd> 
<dt> <strong>lspls</strong> </dt> 
<dd> LS-PLS (least squares &mdash; partial least squares) models. <br>
</dd> 
<dt> <strong>lss</strong> </dt> 
<dd> Accelerated failure time model to right censored data based on 
least-squares principle.<br>
</dd> 
<dt> <strong>ltm</strong> </dt> 
<dd> Analysis of multivariate Bernoulli data using latent trait models 
(including the Rasch model) under the Item Response Theory approach.<br>
</dd> 
<dt> <strong>ltsa</strong> </dt> 
<dd> Linear Time Series Analysis. <br>
</dd> 
<dt> <strong>luca</strong> </dt> 
<dd> Likelihood Under Covariate Assumptions (LUCA). <br>
</dd> 
<dt> <strong>lvplot</strong> </dt> 
<dd> Letter-value box plots. <br>
</dd> 
<dt> <strong>mAr</strong> </dt> 
<dd> Estimation of multivariate AR models through a computationally efficient 
stepwise least-squares algorithm.<br>
</dd> 
<dt> <strong>mFilter</strong> </dt> 
<dd> Miscellenous time series filters. <br>
</dd> 
<dt> <strong>maanova</strong> </dt> 
<dd> Analysis of N-dye Micro Array experiments using mixed model effect. 
Contains anlysis of variance, permutation and bootstrap, cluster and consensus 
tree.<br>
</dd> 
<dt> <strong>magic</strong> </dt> 
<dd> A variety of methods for creating magic squares of any order greater than 
2, and various magic hypercubes.<br>
</dd> 
<dt> <strong>mapLD</strong> </dt> 
<dd> Linkage Disequilibrium mapping. <br>
</dd> 
<dt> <strong>mapdata</strong> </dt> 
<dd> Supplement to package <strong>maps</strong>, providing the larger and/or 
higher-resolution databases.<br>
</dd> 
<dt> <strong>mapproj</strong> </dt> 
<dd> Map Projections: converts latitude/longitude into projected coordinates. 
<br> </dd> 
<dt> <strong>maps</strong> </dt> 
<dd> Draw geographical maps. Projection code and larger maps are in separate 
packages.<br>
</dd> 
<dt> <strong>maptools</strong> </dt> 
<dd> Set of tools for manipulating and reading geographic data, in particular 
ESRI shapefiles.<br>
</dd> 
<dt> <strong>maptree</strong> </dt> 
<dd> Functions with example data for graphing and mapping models from 
hierarchical clustering and classification and regression trees.<br>
</dd> 
<dt> <strong>marginTree</strong> </dt> 
<dd> Margin trees for high-dimensional classification. <br>
</dd> 
<dt> <strong>marginalmodelplots</strong> </dt> 
<dd> Marginal model plots for linear and generalized linear models. <br>
</dd> 
<dt> <strong>mathgraph</strong> </dt> 
<dd> Tools for constructing and manipulating objects from a class of directed 
and undirected graphs.<br>
</dd> 
<dt> <strong>matlab</strong> </dt> 
<dd> Emulate MATLAB code using R. <br>
</dd> 
<dt> <strong>matrixcalc</strong> </dt> 
<dd> Collection of functions for matrix differential calculus. <br>
</dd> 
<dt> <strong>maxLik</strong> </dt> 
<dd> Tools for Maximum Likelihood Estimation. <br>
</dd> 
<dt> <strong>maxstat</strong> </dt> 
<dd> Maximally selected rank and Gauss statistics with several p-value 
approximations.<br>
</dd> 
<dt> <strong>mblm</strong> </dt> 
<dd> Median-based Linear models, using Theil-Sen single or Siegel repeated 
medians.<br>
</dd> 
<dt> <strong>mboost</strong> </dt> 
<dd> Gradient boosting for fitting generalized linear, additive and 
interaction models.<br>
</dd> 
<dt> <strong>mcgibbsit</strong> </dt> 
<dd> Warnes and Raftery's MCGibbsit MCMC diagnostic. <br>
</dd> 
<dt> <strong>mclust</strong> </dt> 
<dd> Model-based clustering and normal mixture modeling including Bayesian 
regularization. <br>
</dd> 
<dt> <strong>mclust02</strong> </dt> 
<dd> Model-based cluster analysis: the 2002 version of MCLUST. <br>
</dd> 
<dt> <strong>mcmc</strong> </dt> 
<dd> Functions for Markov Chain Monte Carlo (MCMC). <br>
</dd> 
<dt> <strong>mco</strong> </dt> 
<dd> Multi criteria optimization algorithms. <br>
</dd> 
<dt> <strong>mda</strong> </dt> 
<dd> Code for mixture discriminant analysis (MDA), flexible discriminant 
analysis (FDA), penalized discriminant analysis (PDA), multivariate additive 
regression splines (MARS), adaptive back-fitting splines (BRUTO), and penalized 
regression. <br>
</dd> 
<dt> <strong>meboot</strong> </dt> 
<dd> Maximum entropy bootstrap for time series. <br>
</dd> 
<dt> <strong>mefa</strong> </dt> 
<dd> Faunistic count data handling and reporting. <br>
</dd> 
<dt> <strong>meifly</strong> </dt> 
<dd> Interactive model exploration using GGobi. <br>
</dd> 
<dt> <strong>memisc</strong> </dt> 
<dd> Miscellaneous Tools for data management, simulation, and presentation of 
estimates.<br>
</dd> 
<dt> <strong>merror</strong> </dt> 
<dd> Accuracy and precision of measurements. <br>
</dd> 
<dt> <strong>meta</strong> </dt> 
<dd> Fixed and random effects meta-analysis, with functions for tests of bias, 
forest and funnel plot.<br>
</dd> 
<dt> <strong>mfp</strong> </dt> 
<dd> Multiple Fractional Polynomials. <br>
</dd> 
<dt> <strong>mgcv</strong> </dt> 
<dd> Routines for GAMs and other genralized ridge regression problems with 
multiple smoothing parameter selection by GCV or UBRE.<em>Recommended</em>. <br>
</dd> 
<dt> <strong>micEcon</strong> </dt> 
<dd> Tools for microeconomic analysis and microeconomic modelling. <br>
</dd> 
<dt> <strong>mice</strong> </dt> 
<dd> Multivariate Imputation by Chained Equations. <br>
</dd> 
<dt> <strong>mimR</strong> </dt> 
<dd> An R interface to MIM for graphical modeling in R. <br>
</dd> 
<dt> <strong>minet</strong> </dt> 
<dd> Mutual Information NETwork. <br>
</dd> 
<dt> <strong>minpack.lm</strong> </dt> 
<dd> R interface for two functions from the MINPACK least squares optimization 
library, solving the nonlinear least squares problem by a modification of the 
Levenberg-Marquardt algorithm.<br>
</dd> 
<dt> <strong>misc3d</strong> </dt> 
<dd> A collection of miscellaneous 3d plots, including rgl-based isosurfaces. 
<br> </dd> 
<dt> <strong>mirf</strong> </dt> 
<dd> Multiple Imputation and Random Forests for unobservable phase, 
high-dimensional data.<br>
</dd> 
<dt> <strong>mitools</strong> </dt> 
<dd> Tools to perform analyses and combine results from multiple-imputation 
datasets.<br>
</dd> 
<dt> <strong>mix</strong> </dt> 
<dd> Estimation/multiple imputation programs for mixed categorical and 
continuous data.<br>
</dd> 
<dt> <strong>mixPHM</strong> </dt> 
<dd> Mixtures of proportional hazard models. <br>
</dd> 
<dt> <strong>mixdist</strong> </dt> 
<dd> Finite mixture distribution models. <br>
</dd> 
<dt> <strong>mixnet</strong> </dt> 
<dd> Random graph clustering via estimation of Erdo''s-R&eacute;nyi mixtures. 
<br> </dd> 
<dt> <strong>mixreg</strong> </dt> 
<dd> Functions to fit mixtures of regressions. <br>
</dd> 
<dt> <strong>mixstock</strong> </dt> 
<dd> Mixed stock analysis functions. <br>
</dd> 
<dt> <strong>mixtools</strong> </dt> 
<dd> Tools for mixture models. <br>
</dd> 
<dt> <strong>mlCopulaSelection</strong> </dt> 
<dd> Copula selection and fitting using maximum likelihood. <br>
</dd> 
<dt> <strong>mlbench</strong> </dt> 
<dd> A collection of artificial and real-world machine learning benchmark 
problems, including the Boston housing data.<br>
</dd> 
<dt> <strong>mlegp</strong> </dt> 
<dd> Maximum Likelihood Estimates of Gaussian Processes. <br>
</dd> 
<dt> <strong>mlica</strong> </dt> 
<dd> Independent Component Analysis using Maximum Likelihood. <br>
</dd> 
<dt> <strong>mlmRev</strong> </dt> 
<dd> Examples from Multilevel Modelling Software Review. <br>
</dd> 
<dt> <strong>mlmmm</strong> </dt> 
<dd> Maximum likelihood estimation under multivariate linear mixed models with 
missing values.<br>
</dd> 
<dt> <strong>mlogit</strong> </dt> 
<dd> Estimation of the multinomial logit model with choice specific variables. 
<br> </dd> 
<dt> <strong>mmlcr</strong> </dt> 
<dd> Mixed-mode latent class regression (also known as mixed-mode mixture 
model regression or mixed-mode mixture regression models) which can handle both 
longitudinal and one-time responses.<br>
</dd> 
<dt> <strong>mnormt</strong> </dt> 
<dd> The multivariate normal and t distributions. <br>
</dd> 
<dt> <strong>moc</strong> </dt> 
<dd> Fits a variety of mixtures models for multivariate observations with 
user-difined distributions and curves.<br>
</dd> 
<dt> <strong>modeest</strong> </dt> 
<dd> Mode estimation and Chernoff distribution. <br>
</dd> 
<dt> <strong>modehunt</strong> </dt> 
<dd> Multiscale analysis for density functions. <br>
</dd> 
<dt> <strong>modeltools</strong> </dt> 
<dd> A collection of tools to deal with statistical models. <br>
</dd> 
<dt> <strong>moduleColor</strong> </dt> 
<dd> Methods for color labeling, calculation of eigengenes, and merging of 
closely related modules.<br>
</dd> 
<dt> <strong>mokken</strong> </dt> 
<dd> Mokken Scale Analysis for test and questionnaire data. <br>
</dd> 
<dt> <strong>mombf</strong> </dt> 
<dd> Moment and inverse moment Bayes factors. <br>
</dd> 
<dt> <strong>moments</strong> </dt> 
<dd> Moments, skewness, kurtosis and related tests. <br>
</dd> 
<dt> <strong>monomvn</strong> </dt> 
<dd> Estimation for multivariate normal data with monotone missingness. <br>

</dd> 
<dt> <strong>monreg</strong> </dt> 
<dd> Estimation of monotone regression and variance functions in nonparametric 
models.<br>
</dd> 
<dt> <strong>monoProc</strong> </dt> 
<dd> Strictly monotone smoothing procedure. <br>
</dd> 
<dt> <strong>moonsun</strong> </dt> 
<dd> Basic astronomical calculations. <br>
</dd> 
<dt> <strong>mota</strong> </dt> 
<dd> Mean Optimal Transformation Approach. <br>
</dd> 
<dt> <strong>mpm</strong> </dt> 
<dd> Spectral map analysis. <br>
</dd> 
<dt> <strong>mprobit</strong> </dt> 
<dd> Multivariate probit model for binary/ordinal response. <br>
</dd> 
<dt> <strong>mra</strong> </dt> 
<dd> Analysis of capture-recapture data. <br>
</dd> 
<dt> <strong>mratios</strong> </dt> 
<dd> Inferences for ratios of coefficients in the general linear model. <br>

</dd> 
<dt> <strong>mrdrc</strong> </dt> 
<dd> Model-robust concentration-response analysis. <br>
</dd> 
<dt> <strong>msBreast</strong> </dt> 
<dd> Protein mass spectra dataset from a breast cancer study. <br>
</dd> 
<dt> <strong>msDilution</strong> </dt> 
<dd> Protein mass spectra dataset from a dilution experiment. <br>
</dd> 
<dt> <strong>msProcess</strong> </dt> 
<dd> Tools for protein mass spectra processing including data preparation, 
denoising, noise estimation, baseline correction, intensity normalization, peak 
detection, peak alignment, peak quantification, and various functionalities for 
data ingestion/conversion, mass calibration, data quality assessment, and 
protein mass spectra simulation.<br>
</dd> 
<dt> <strong>msProstate</strong> </dt> 
<dd> Protein mass spectra dataset from a prostate cancer study. <br>
</dd> 
<dt> <strong>msm</strong> </dt> 
<dd> Functions for fitting continuous-time Markov multi-state models to 
categorical processes observed at arbitrary times, optionally with 
misclassified responses, and covariates on transition or misclassification 
rates.<br>
</dd> 
<dt> <strong>muS2RC</strong> </dt> 
<dd> S-plus to R Compatibility for package <strong>muStat</strong>. <br>
</dd> 
<dt> <strong>muStat</strong> </dt> 
<dd> Prentice rank sum test and McNemar test. <br>
</dd> 
<dt> <strong>muUtil</strong> </dt> 
<dd> Utility functions for package <strong>muStat</strong>. <br>
</dd> 
<dt> <strong>muhaz</strong> </dt> 
<dd> Hazard function estimation in survival analysis. <br>
</dd> 
<dt> <strong>multcomp</strong> </dt> 
<dd> Multiple comparison procedures for the one-way layout. <br>
</dd> 
<dt> <strong>multcompView</strong> </dt> 
<dd> Visualizations of paired comparisons. <br>
</dd> 
<dt> <strong>multic</strong> </dt> 
<dd> Quantitative linkage analysis tools using the variance components 
approach.<br>
</dd> 
<dt> <strong>multilevel</strong> </dt> 
<dd> Analysis of multilevel data by organizational and social psychologists. 
<br> </dd> 
<dt> <strong>multinomRob</strong> </dt> 
<dd> Overdispersed multinomial regression using robust (LQD and tanh) 
estimation.<br>
</dd> 
<dt> <strong>multipol</strong> </dt> 
<dd> Utilities to manipulate multivariate polynomials. <br>
</dd> 
<dt> <strong>multtest</strong> </dt> 
<dd> Resampling-based multiple hypothesis testing. <br>
</dd> 
<dt> <strong>mvbutils</strong> </dt> 
<dd> Utilities by Mark V. Bravington for project organization, editing and 
backup, sourcing, documentation (formal and informal), package preparation, 
macro functions, and more.<br>
</dd> 
<dt> <strong>mvna</strong> </dt> 
<dd> Nelson-Aalen estimator of the cumulative hazard in multistate models. <br>
</dd> 
<dt> <strong>mvnmle</strong> </dt> 
<dd> ML estimation for multivariate normal data with missing values. <br>
</dd>
<dt> <strong>mvnormtest</strong> </dt> 
<dd> Generalization of the Shapiro-Wilk test for multivariate variables. <br>

</dd> 
<dt> <strong>mvtnormpcs</strong> </dt> 
<dd> Multivariate Normal and T Distribution functions of Dunnett (1989). <br>

</dd> 
<dt> <strong>mvoutlier</strong> </dt> 
<dd> Multivariate outlier detection based on robust estimates of location and 
covariance structure.<br>
</dd> 
<dt> <strong>mvpart</strong> </dt> 
<dd> Multivariate partitioning. <br>
</dd> 
<dt> <strong>mvtnorm</strong> </dt> 
<dd> Multivariate normal and t distributions. <br>
</dd> 
<dt> <strong>nFDR</strong> </dt> 
<dd> Nonparametric Estimate of FDR Based on Bernstein polynomials. <br>
</dd> 
<dt> <strong>nFactors</strong> </dt> 
<dd> Non-graphical solution to the Cattell Scree Test. <br>
</dd> 
<dt> <strong>ncdf</strong> </dt> 
<dd> Interface to Unidata netCDF data files. <br>
</dd> 
<dt> <strong>ncf</strong> </dt> 
<dd> Spatial nonparametric covariance functions. <br>
</dd> 
<dt> <strong>ncomplete</strong> </dt> 
<dd> Functions to perform the regression depth method (RDM) to binary 
regression to approximate the minimum number of observations that can be 
removed such that the reduced data set has complete separation. <br>
</dd> 
<dt> <strong>negenes</strong> </dt> 
<dd> Estimating the number of essential genes in a genome on the basis of data 
from a random transposon mutagenesis experiment, through the use of a Gibbs 
sampler. <br>
</dd> 
<dt> <strong>netmodels</strong> </dt> 
<dd> Tools for the study of scale free and small world networks. <br>
</dd> 
<dt> <strong>network</strong> </dt> 
<dd> Tools to create and modify network objects, which can represent a range 
of relational data types.<br>
</dd> 
<dt> <strong>networksis</strong> </dt> 
<dd> Simulate bipartite graphs with fixed marginals through sequential 
importance sampling.<br>
</dd> 
<dt> <strong>neural</strong> </dt> 
<dd> RBF and MLP neural networks with graphical user interface. <br>
</dd> 
<dt> <strong>neuralnet</strong> </dt> 
<dd> Training of neural networks. <br>
</dd> 
<dt> <strong>nice</strong> </dt> 
<dd> Get or set UNIX priority (niceness) of running R process. <br>
</dd> 
<dt> <strong>nlme</strong> </dt> 
<dd> Fit and compare Gaussian linear and nonlinear mixed-effects models. <em>
Recommended</em>. <br>
</dd> 
<dt> <strong>nlmeODE</strong> </dt> 
<dd> Combine the <strong>nlme</strong> and <strong>odesolve</strong> packages 
for mixed-effects modelling using differential equations. <br>
</dd> 
<dt> <strong>nlrwr</strong> </dt> 
<dd> Non-linear regression with R. <br>
</dd> 
<dt> <strong>nls2</strong> </dt> 
<dd> Non-linear regression with brute force. <br>
</dd> 
<dt> <strong>nlstools</strong> </dt> 
<dd> Tools for nonlinear regression diagnostics. <br>
</dd> 
<dt> <strong>nlt</strong> </dt> 
<dd> A nondecimated lifting transform for signal denoising. <br>
</dd> 
<dt> <strong>nlts</strong> </dt> 
<dd> (Non)linear time series analysis. <br>
</dd> 
<dt> <strong>nltm</strong> </dt> 
<dd> NonLinear Transformation Models for survival analysis. <br>
</dd> 
<dt> <strong>nnet</strong> </dt> 
<dd> Software for single hidden layer perceptrons (&ldquo;feed-forward neural 
networks&rdquo;), and for multinomial log-linear models. Contained in theVR 
bundle.<em>Recommended</em>. <br>
</dd> 
<dt> <strong>nnls</strong> </dt> 
<dd> The Lawson-Hanson NNLS algorithm for non-negative least squares. <br>

</dd> 
<dt> <strong>noia</strong> </dt> 
<dd> Implementation of the Natural and Orthogonal InterAction (NOIA) model. 
<br> </dd> 
<dt> <strong>nonbinROC</strong> </dt> 
<dd> ROC-type analysis for non-binary gold standards. <br>
</dd> 
<dt> <strong>nor1mix</strong> </dt> 
<dd> One-dimensional normal mixture models classes, for, e.g., density 
estimation or clustering algorithms research and teaching; providing the widely 
used Marron-Wand densities.<br>
</dd> 
<dt> <strong>norm</strong> </dt> 
<dd> Analysis of multivariate normal datasets with missing values. <br>
</dd> 
<dt> <strong>normwm.test</strong> </dt> 
<dd> Normality and white noise testing. <br>
</dd> 
<dt> <strong>normalp</strong> </dt> 
<dd> A collection of utilities for normal of order p distributions (General 
Error Distributions).<br>
</dd> 
<dt> <strong>nortest</strong> </dt> 
<dd> Five omnibus tests for the composite hypothesis of normality. <br>
</dd> 
<dt> <strong>noverlap</strong> </dt> 
<dd> Functions to perform the regression depth method (RDM) to binary 
regression to approximate the amount of overlap, i.e., the minimal number of 
observations that need to be removed such that the reduced data set has no 
longer overlap.<br>
</dd> 
<dt> <strong>np</strong> </dt> 
<dd> Nonparametric kernel smoothing methods for mixed datatypes. <br>
</dd> 
<dt> <strong>nparcomp</strong> </dt> 
<dd> Nonparametric relative contrast effects. <br>
</dd> 
<dt> <strong>npde</strong> </dt> 
<dd> Normalized prediction distribution errors for nonlinear mixed-effect 
models.<br>
</dd> 
<dt> <strong>nplplot</strong> </dt> 
<dd> Plotting non-parametric LOD scores from multiple input files. <br>
</dd> 
<dt> <strong>npmc</strong> </dt> 
<dd> Nonparametric Multiple Comparisons: provides simultaneous rank test 
procedures for the one-way layout without presuming a certain distribution. <br>
</dd> 
<dt> <strong>nsRFA</strong> </dt> 
<dd> Non-supervised Regional Frequency Analysis. <br>
</dd> 
<dt> <strong>numDeriv</strong> </dt> 
<dd> Accurate numerical derivatives. <br>
</dd> 
<dt> <strong>nws</strong> </dt> 
<dd> Functions for NetWorkSpaces and Sleigh. <br>
</dd> 
<dt> <strong>obsSens</strong> </dt> 
<dd> Sensitivity analysis for observational studies. <br>
</dd> 
<dt> <strong>oc</strong> </dt> 
<dd> Optimal Classification roll call analysis. <br>
</dd> 
<dt> <strong>oce</strong> </dt> 
<dd> Analysis of oceanographic data. <br>
</dd> 
<dt> <strong>odesolve</strong> </dt> 
<dd> An interface for the Ordinary Differential Equation (ODE) solver lsoda. 
ODEs are expressed as R functions.<br>
</dd> 
<dt> <strong>odfWeave</strong> </dt> 
<dd> Sweave processing of Open Document Format (ODF) files. <br>
</dd> 
<dt> <strong>ofw</strong> </dt> 
<dd> Optimal Feature Weighting algorithm. <br>
</dd> 
<dt> <strong>onemap</strong> </dt> 
<dd> Analysis of molecular marker data from non-model systems to 
simultaneously estimate linkage and linkage phases (genetic map construction).
<br> </dd> 
<dt> <strong>onion</strong> </dt> 
<dd> A collection of routines to manipulate and visualize quaternions and 
octonions.<br>
</dd> 
<dt> <strong>openNLP</strong> </dt> 
<dd> An interface to openNLP, a collection of natural language processing 
tools including a sentence detector, tokenizer, pos-tagger, shallow and full 
syntactic parser, and named-entity detector, using theMaxent Java package for 
training and using maximum entropy models.<br>
</dd> 
<dt> <strong>openNLPmodels</strong> </dt> 
<dd> English and spanish models for <strong>openNLP</strong>. <br>
</dd> 
<dt> <strong>optmatch</strong> </dt> 
<dd> Functions to perform optimal matching, particularly full matching. <br>

</dd> 
<dt> <strong>orientlib</strong> </dt> 
<dd> Representations, conversions and display of orientation SO(3) data. <br>

</dd> 
<dt> <strong>orloca</strong> </dt> 
<dd> Operations Research LOCational Analysis models. <br>
</dd> 
<dt> <strong>orloca.es</strong> </dt> 
<dd> Spanish version of <strong>orloca</strong> package. <br>
</dd> 
<dt> <strong>orth</strong> </dt> 
<dd> Multivariate logistic regressions using orthogonalized residuals. <br>

</dd> 
<dt> <strong>orthopolynom</strong> </dt> 
<dd> Functions for orthogonal and orthonormal polynomials. <br>
</dd> 
<dt> <strong>ouch</strong> </dt> 
<dd> Ornstein-Uhlenbeck models for phylogenetic comparative hypotheses. <br>

</dd> 
<dt> <strong>outliers</strong> </dt> 
<dd> A collection of some tests commonly used for identifying outliers. <br>

</dd> 
<dt> <strong>oz</strong> </dt> 
<dd> Functions for plotting Australia's coastline and state boundaries. <br>

</dd> 
<dt> <strong>pARtial</strong> </dt> 
<dd> (Partial) attributable risk estimates, corresponding variance estimates 
and confidence intervals.<br>
</dd> 
<dt> <strong>pack</strong> </dt> 
<dd> Create and manipulate raw vectors. <br>
</dd> 
<dt> <strong>pairwiseCI</strong> </dt> 
<dd> Calculate and plot unadjusted confidence intervals for two sample 
comparisons.<br>
</dd> 
<dt> <strong>paleoTS</strong> </dt> 
<dd> Modeling evolution in paleontological time-series. <br>
</dd> 
<dt> <strong>paltran</strong> </dt> 
<dd> Functions for paleolimnology. <br>
</dd> 
<dt> <strong>pamr</strong> </dt> 
<dd> Pam: Prediction Analysis for Microarrays. <br>
</dd> 
<dt> <strong>pan</strong> </dt> 
<dd> Multiple imputation for multivariate panel or clustered data. <br>
</dd> 
<dt> <strong>panel</strong> </dt> 
<dd> Functions and datasets for fitting models to Panel data. <br>
</dd> 
<dt> <strong>papply</strong> </dt> 
<dd> Parallel apply function using MPI. <br>
</dd> 
<dt> <strong>paran</strong> </dt> 
<dd> Horn's test of principal components/factors. <br>
</dd> 
<dt> <strong>partitions</strong> </dt> 
<dd> Additive partitions of integers. <br>
</dd> 
<dt> <strong>partsm</strong> </dt> 
<dd> Periodic AutoRegressive Time Series Models. <br>
</dd> 
<dt> <strong>party</strong> </dt> 
<dd> Unbiased recursive partitioning in a conditional inference framework. <br>
</dd> 
<dt> <strong>pastecs</strong> </dt> 
<dd> Package for Analysis of Space-Time Ecological Series. <br>
</dd> 
<dt> <strong>pbatR</strong> </dt> 
<dd> Frontend to PBAT to run within R. <br>
</dd> 
<dt> <strong>pcaPP</strong> </dt> 
<dd> Robust PCA by Projection Pursuit. <br>
</dd> 
<dt> <strong>pcalg</strong> </dt> 
<dd> Standard and robust estimation of the skeleton (ugraph) of a Directed 
Acyclic Graph (DAG) via the PC algorithm.<br>
</dd> 
<dt> <strong>pcse</strong> </dt> 
<dd> Panel-Corrected Standard Error estimation. <br>
</dd> 
<dt> <strong>pcurve</strong> </dt> 
<dd> Fits a principal curve to a numeric multivariate dataset in arbitrary 
dimensions. Produces diagnostic plots. Also calculates Bray-Curtis and other 
distance matrices and performs multi-dimensional scaling and principal 
component analyses.<br>
</dd> 
<dt> <strong>pear</strong> </dt> 
<dd> Periodic Autoregression Analysis. <br>
</dd> 
<dt> <strong>pec</strong> </dt> 
<dd> Prediction Error Curves for survival models. <br>
</dd> 
<dt> <strong>penalized</strong> </dt> 
<dd> Penalized estimation in generalized linear models. <br>
</dd> 
<dt> <strong>peperr</strong> </dt> 
<dd> Parallelised Estimation of Prediction ERRor. <br>
</dd> 
<dt> <strong>permax</strong> </dt> 
<dd> Functions intended to facilitate certain basic analyses of DNA array 
data, especially with regard to comparing expression levels between two types 
of tissue.<br>
</dd> 
<dt> <strong>permtest</strong> </dt> 
<dd> Permutation test to compare variability within and distance between two 
groups.<br>
</dd> 
<dt> <strong>perturb</strong> </dt> 
<dd> Perturbation analysis for evaluating collinearity. <br>
</dd> 
<dt> <strong>pga</strong> </dt> 
<dd> An ensemble method for variable selection by carrying out Darwinian 
evolution in parallel universes.<br>
</dd> 
<dt> <strong>pgam</strong> </dt> 
<dd> Poisson-Gamma Additive Models. <br>
</dd> 
<dt> <strong>pgirmess</strong> </dt> 
<dd> Functions for analysis and display of ecological and spatial data. <br>

</dd> 
<dt> <strong>phangorn</strong> </dt> 
<dd> Phylogenetic analysis in R. <br>
</dd> 
<dt> <strong>pheno</strong> </dt> 
<dd> Some easy-to-use functions for time series analyses of (plant-) 
phenological data sets.<br>
</dd> 
<dt> <strong>phpSerialize</strong> </dt> 
<dd> Serialize R to PHP associative array. <br>
</dd> 
<dt> <strong>picante</strong> </dt> 
<dd> Tools for integrating phylogenies and ecology. <br>
</dd> 
<dt> <strong>pinktoe</strong> </dt> 
<dd> Converts S trees to HTML/Perl files for interactive tree traversal. <br>

</dd> 
<dt> <strong>pixmap</strong> </dt> 
<dd> Functions for import, export, plotting and other manipulations of 
bitmapped images.<br>
</dd> 
<dt> <strong>plRasch</strong> </dt> 
<dd> Log linear by linear asscociation models. <br>
</dd> 
<dt> <strong>playwith</strong> </dt> 
<dd> A GUI for interactive plots using GTK+. <br>
</dd> 
<dt> <strong>plink</strong> </dt> 
<dd> Separate calibration linking methods. <br>
</dd> 
<dt> <strong>plm</strong> </dt> 
<dd> Linear models for panel data. <br>
</dd> 
<dt> <strong>plotSEMM</strong> </dt> 
<dd> Graphing nonlinear latent variable interactions in SEMM. <br>
</dd> 
<dt> <strong>plotrix</strong> </dt> 
<dd> Various useful functions for enhancing plots. <br>
</dd> 
<dt> <strong>plugdensity</strong> </dt> 
<dd> Kernel density estimation with global bandwidth selection via 
&ldquo;plug-in&rdquo;.<br>
</dd> 
<dt> <strong>pls</strong> </dt> 
<dd> Partial Least Squares Regression (PLSR) and Principal Component 
Regression (PCR). <br>
</dd> 
<dt> <strong>plsgenomics</strong> </dt> 
<dd> PLS analyses for genomics. <br>
</dd> 
<dt> <strong>pmg</strong> </dt> 
<dd> Poor Man's GUI. <br>
</dd> 
<dt> <strong>pmml</strong> </dt> 
<dd> Generate Predictive Modelling Markup Language (PMML) for various models. 
<br> </dd> 
<dt> <strong>poLCA</strong> </dt> 
<dd> POlytomous variable Latent Class Analysis. <br>
</dd> 
<dt> <strong>poilog</strong> </dt> 
<dd> Poisson lognormal and bivariate Poisson lognormal distribution. <br>
</dd>
<dt> <strong>polspline</strong> </dt> 
<dd> Routines for the polynomial spline fitting routines hazard regression, 
hazard estimation with flexible tails, logspline, lspec, polyclass, and 
polymars, by C. Kooperberg and co-authors.<br>
</dd> 
<dt> <strong>polyapost</strong> </dt> 
<dd> Simulating from the Polya posterior. <br>
</dd> 
<dt> <strong>polycor</strong> </dt> 
<dd> Polychoric and polyserial correlations. <br>
</dd> 
<dt> <strong>polynom</strong> </dt> 
<dd> A collection of functions to implement a class for univariate polynomial 
manipulations.<br>
</dd> 
<dt> <strong>pomp</strong> </dt> 
<dd> Partially-observed Markov processes. <br>
</dd> 
<dt> <strong>popbio</strong> </dt> 
<dd> Construction and analysis of matrix population models. <br>
</dd> 
<dt> <strong>popgen</strong> </dt> 
<dd> Statistical and POPulation GENetics. <br>
</dd> 
<dt> <strong>poplab</strong> </dt> 
<dd> Population Lab, a tool for constructing a virtual electronic population 
evolving over time.<br>
</dd> 
<dt> <strong>portfolio</strong> </dt> 
<dd> Classes for analyzing and implementing portfolios. <br>
</dd> 
<dt> <strong>portfolioSim</strong> </dt> 
<dd> Framework for simulating equity portfolio strategies. <br>
</dd> 
<dt> <strong>powell</strong> </dt> 
<dd> Optimizes a function using Powell's UObyQA algorithm. <br>
</dd> 
<dt> <strong>powerpkg</strong> </dt> 
<dd> Power analyses for the affected sib pair and the TDT design. <br>
</dd> 
<dt> <strong>ppc</strong> </dt> 
<dd> Sample classification of protein mass spectra by peak probabilty 
contrasts.<br>
</dd> 
<dt> <strong>ppls</strong> </dt> 
<dd> Penalized Partial Least Squares. <br>
</dd> 
<dt> <strong>pps</strong> </dt> 
<dd> Functions to select samples using PPS (probability proportional to size) 
sampling, for stratified simple random sampling, and to compute joint inclusion 
probabilities for Sampford's method of PPS sampling.<br>
</dd> 
<dt> <strong>prabclus</strong> </dt> 
<dd> Distance based parametric bootstrap tests for clustering, mainly thought 
for presence-absence data (clustering of species distribution maps). Jaccard 
and Kulczynski distance measures, clustering of MDS scores, and nearest 
neighbor based noise detection.<br>
</dd> 
<dt> <strong>predbayescor</strong> </dt> 
<dd> Classification rule based on Bayesian naive Bayes models with feature 
selection bias corrected.<br>
</dd> 
<dt> <strong>predmixcor</strong> </dt> 
<dd> Classification rule based on Bayesian mixture models with feature 
selection bias corrected.<br>
</dd> 
<dt> <strong>prettyR</strong> </dt> 
<dd> Pretty descriptive stats. <br>
</dd> 
<dt> <strong>prim</strong> </dt> 
<dd> Patient Rule Induction Method (PRIM). <br>
</dd> 
<dt> <strong>princurve</strong> </dt> 
<dd> Fits a principal curve to a matrix of points in arbitrary dimension. <br>

</dd> 
<dt> <strong>prob</strong> </dt> 
<dd> Elementary probability on finite sample spaces. <br>
</dd> 
<dt> <strong>prodlim</strong> </dt> 
<dd> Product limit estimation. <br>
</dd> 
<dt> <strong>profileModel</strong> </dt> 
<dd> Tools for profiling inference functions for various model classes. <br>

</dd> 
<dt> <strong>profr</strong> </dt> 
<dd> Alternative display for profiling information. <br>
</dd> 
<dt> <strong>proftools</strong> </dt> 
<dd> Profile output processing tools for R. <br>
</dd> 
<dt> <strong>proj4</strong> </dt> 
<dd> A simple interface to the PROJ.4 cartographic projections library. <br>

</dd> 
<dt> <strong>proptest</strong> </dt> 
<dd> Tests of the proportional hazards assumption in the Cox model. <br>
</dd> 
<dt> <strong>proto</strong> </dt> 
<dd> An object oriented system using prototype or object-based (rather than 
class-based) object oriented ideas.<br>
</dd> 
<dt> <strong>proxy</strong> </dt> 
<dd> Distance and similarity measures. <br>
</dd> 
<dt> <strong>pscl</strong> </dt> 
<dd> R in the Political Science Computational Laboratory, Stanford University. 
<br> </dd> 
<dt> <strong>pseudo</strong> </dt> 
<dd> Pseudo-observations. <br>
</dd> 
<dt> <strong>pspline</strong> </dt> 
<dd> Smoothing splines with penalties on order m derivatives. <br>
</dd> 
<dt> <strong>psy</strong> </dt> 
<dd> Various procedures used in psychometry: Kappa, ICC, Cronbach alpha, 
screeplot, PCA and related methods.<br>
</dd> 
<dt> <strong>psych</strong> </dt> 
<dd> Procedures for personality and psychological research. <br>
</dd> 
<dt> <strong>psychometric</strong> </dt> 
<dd> Applied psychometric theory: functions useful for correlation theory, 
meta-analysis (validity-generalization), reliability, item analysis, 
inter-rater reliability, and classical utility.<br>
</dd> 
<dt> <strong>psyphy</strong> </dt> 
<dd> Functions for analyzing psychophysical data in R. <br>
</dd> 
<dt> <strong>pwr</strong> </dt> 
<dd> Basic functions for power analysis. <br>
</dd> 
<dt> <strong>pwt</strong> </dt> 
<dd> The Penn World Table providing purchasing power parity and national 
income accounts converted to international prices for 168 countries for some or 
all of the years 1950&ndash;2000.<br>
</dd> 
<dt> <strong>pvclust</strong> </dt> 
<dd> Hierarchical clustering with p-value. <br>
</dd> 
<dt> <strong>qAnalyst</strong> </dt> 
<dd> Variables and attributes control charts. <br>
</dd> 
<dt> <strong>qcc</strong> </dt> 
<dd> Quality Control Charts. Shewhart quality control charts for continuous, 
attribute and count data. Cusum and EWMA charts. Operating characteristic 
curves. Process capability analysis. Pareto chart and cause-and-effect chart.
<br> </dd> 
<dt> <strong>qdg</strong> </dt> 
<dd> Infer QTL-directed Dependency Graphs for phenotype networks. <br>
</dd> 
<dt> <strong>qgen</strong> </dt> 
<dd> Quantitative Genetics using R. <br>
</dd> 
<dt> <strong>qlspack</strong> </dt> 
<dd> Quasi least squares package. <br>
</dd> 
<dt> <strong>qp</strong> </dt> 
<dd> q-order partial correlation graph search algorithm. <br>
</dd> 
<dt> <strong>qpcR</strong> </dt> 
<dd> Modelling and analysis of real-time PCR data. <br>
</dd> 
<dt> <strong>qtl</strong> </dt> 
<dd> Analysis of experimental crosses to identify QTLs. <br>
</dd> 
<dt> <strong>qtlDesign</strong> </dt> 
<dd> Tools for the design of QTL experiments. <br>
</dd> 
<dt> <strong>qtlbim</strong> </dt> 
<dd> QTL Bayesian Interval Mapping. <br>
</dd> 
<dt> <strong>quadprog</strong> </dt> 
<dd> For solving quadratic programming problems. <br>
</dd> 
<dt> <strong>qualV</strong> </dt> 
<dd> Qualitative methods for the validation of models. <br>
</dd> 
<dt> <strong>quantchem</strong> </dt> 
<dd> Quantitative chemical analysis: calibration and evaluation of results. 
<br> </dd> 
<dt> <strong>quantmod</strong> </dt> 
<dd> Quantitative financial modelling framework. <br>
</dd> 
<dt> <strong>quantreg</strong> </dt> 
<dd> Quantile regression and related methods. <br>
</dd> 
<dt> <strong>quantregForest</strong> </dt> 
<dd> Quantile Regression Forests, a tree-based ensemble method for estimation 
of conditional quantiles.<br>
</dd> 
<dt> <strong>qvalue</strong> </dt> 
<dd> Q-value estimation for false discovery rate control. <br>
</dd> 
<dt> <strong>qvcalc</strong> </dt> 
<dd> Functions to compute quasi-variances and associated measures of 
approximation error.<br>
</dd> 
<dt> <strong>r2lUniv</strong> </dt> 
<dd> R to LaTeX Univariate: perform basic analysis and generate corresponding 
LaTeX code.<br>
</dd> 
<dt> <strong>rJava</strong> </dt> 
<dd> Low-level R to Java interface. Allows creation of objects, calling 
methods and accessing fields.<br>
</dd> 
<dt> <strong>rPorta</strong> </dt> 
<dd> An R interface to PORTA, a collection of routines for analyzing polytopes 
and polyhedra.<br>
</dd> 
<dt> <strong>race</strong> </dt> 
<dd> Implementation of some racing methods for the empirical selection of the 
best.<br>
</dd> 
<dt> <strong>rake</strong> </dt> 
<dd> Raking survey datasets by re-weighting. <br>
</dd> 
<dt> <strong>ramps</strong> </dt> 
<dd> Bayesian geostatistical modeling of Gaussian processes using a 
reparameterized and marginalized posterior sampling (RAMPS) algorithm.<br>
</dd>
<dt> <strong>randaes</strong> </dt> 
<dd> Random number generator based on AES cipher. <br>
</dd> 
<dt> <strong>random</strong> </dt> 
<dd> True random numbers using random.org. <br>
</dd> 
<dt> <strong>randomLCA</strong> </dt> 
<dd> Random effects Latent Class Analysis. <br>
</dd> 
<dt> <strong>randomSurvivalForest</strong> </dt> 
<dd> Ishwaran and Kogalur's random survival forest. <br>
</dd> 
<dt> <strong>randomForest</strong> </dt> 
<dd> Breiman's random forest classifier. <br>
</dd> 
<dt> <strong>randtoolbox</strong> </dt> 
<dd> Toolbox for pseudo and quasi random number generation. <br>
</dd> 
<dt> <strong>rankreg</strong> </dt> 
<dd> Rank regression estimator for the AFT model with right censored data. <br>
</dd> 
<dt> <strong>rateratio.test</strong> </dt> 
<dd> Exact rate ratio test. <br>
</dd> 
<dt> <strong>rattle</strong> </dt> 
<dd> A graphical user interface for data mining in R using GTK. <br>
</dd> 
<dt> <strong>rbugs</strong> </dt> 
<dd> Functions to prepare files needed for running BUGS in batch mode, and 
running BUGS from R. Support for Linux systems with Wine is emphasized.<br>

</dd> 
<dt> <strong>rcdd</strong> </dt> 
<dd> C Double Description for R, an interface to the CDD computational 
geometry library.<br>
</dd> 
<dt> <strong>rcdk</strong> </dt> 
<dd> Interface to the CDK libraries, a Java framework for cheminformatics. <br>
</dd> 
<dt> <strong>rcom</strong> </dt> 
<dd> R COM Client Interface and internal COM Server. <br>
</dd> 
<dt> <strong>rcompgen</strong> </dt> 
<dd> Completion generator for R. <em>Recommended</em> for R 2.5.0 or 2.6.0. 
<br> </dd> 
<dt> <strong>rda</strong> </dt> 
<dd> Shrunken Centroids Regularized Discriminant Analysis. <br>
</dd> 
<dt> <strong>rdetools</strong> </dt> 
<dd> Relevant Dimension Estimation (RDE) in feature spaces. <br>
</dd> 
<dt> <strong>realized</strong> </dt> 
<dd> Realized variance toolkit. <br>
</dd> 
<dt> <strong>ref</strong> </dt> 
<dd> Functions for creating references, reading from and writing ro references 
and a memory efficient refdata type that transparently encapsulates matrices 
and data frames.<br>
</dd> 
<dt> <strong>regress</strong> </dt> 
<dd> Fitting Gaussian linear models where the covariance structure is a linear 
combination of known matrices by maximising the residual log likelihood. Can be 
used for multivariate models and random effects models.<br>
</dd> 
<dt> <strong>regsubseq</strong> </dt> 
<dd> Detect and test regular sequences and subsequences. <br>
</dd> 
<dt> <strong>regtest</strong> </dt> 
<dd> Regression testing. <br>
</dd> 
<dt> <strong>relaimpo</strong> </dt> 
<dd> RELAtive IMPOrtance of regressors in linear models. <br>
</dd> 
<dt> <strong>relations</strong> </dt> 
<dd> Data structures for k-ary relations with arbitrary domains, predicate 
functions, and fitters for consensus relations.<br>
</dd> 
<dt> <strong>relax</strong> </dt> 
<dd> Functions for report writing, presentation, and programming. <br>
</dd> 
<dt> <strong>relaxo</strong> </dt> 
<dd> Relaxed Lasso. <br>
</dd> 
<dt> <strong>reldist</strong> </dt> 
<dd> Functions for the comparison of distributions, including nonparametric 
estimation of the relative distribution PDF and CDF and numerical summaries as 
described in &ldquo;Relative Distribution Methods in the Social Sciences&rdquo; 
by Mark S. Handcock and Martina Morris, 1999, Springer.<br>
</dd> 
<dt> <strong>relimp</strong> </dt> 
<dd> Functions to facilitate inference on the relative importance of 
predictors in a linear or generalized linear model.<br>
</dd> 
<dt> <strong>relsurv</strong> </dt> 
<dd> Various functions for regression in relative survival. <br>
</dd> 
<dt> <strong>repolr</strong> </dt> 
<dd> Repeated measures proportional odds logistic regression. <br>
</dd> 
<dt> <strong>reshape</strong> </dt> 
<dd> Flexibly reshape data. <br>
</dd> 
<dt> <strong>resper</strong> </dt> 
<dd> Sampling from restricted permutations. <br>
</dd> 
<dt> <strong>reweight</strong> </dt> 
<dd> Adjustment of survey respondent weights. <br>
</dd> 
<dt> <strong>rgcvpack</strong> </dt> 
<dd> R interface for GCVPACK Fortran package. <br>
</dd> 
<dt> <strong>rgdal</strong> </dt> 
<dd> Provides bindings to Frank Warmerdam's Geospatial Data Abstraction 
Library (GDAL).<br>
</dd> 
<dt> <strong>rgenoud</strong> </dt> 
<dd> R version of GENetic Optimization Using Derivatives. <br>
</dd> 
<dt> <strong>rggm</strong> </dt> 
<dd> Robustified methods for Gaussian Graphical Models. <br>
</dd> 
<dt> <strong>rggobi</strong> </dt> 
<dd> Interface between R and GGobi. <br>
</dd> 
<dt> <strong>rgl</strong> </dt> 
<dd> 3D visualization device system (OpenGL). <br>
</dd> 
<dt> <strong>rgr</strong> </dt> 
<dd> The GSC (Geological Survey of Canada) applied geochemistry EDA package. 
<br> </dd> 
<dt> <strong>rhosp</strong> </dt> 
<dd> Side effect risks in hospital: simulation and estimation. <br>
</dd> 
<dt> <strong>richards</strong> </dt> 
<dd> Richards curves. <br>
</dd> 
<dt> <strong>rimage</strong> </dt> 
<dd> Functions for image processing, including Sobel filter, rank filters, 
fft, histogram equalization, and readingJPEG files. <br>
</dd> 
<dt> <strong>rindex</strong> </dt> 
<dd> Indexing for R. <br>
</dd> 
<dt> <strong>risksetROC</strong> </dt> 
<dd> Riskset ROC curve estimation from censored survival data. <br>
</dd> 
<dt> <strong>rjacobi</strong> </dt> 
<dd> Jacobi polynomials and Gauss-Jacobi quadrature related operations. <br>

</dd> 
<dt> <strong>rjags</strong> </dt> 
<dd> Bayesian graphical models via an interface to the JAGS MCMC library. <br>

</dd> 
<dt> <strong>rjson</strong> </dt> 
<dd> JSON (JavaScript Object Notation) for R. <br>
</dd> 
<dt> <strong>rlecuyer</strong> </dt> 
<dd> R interface to RNG with multiple streams. <br>
</dd> 
<dt> <strong>rmeta</strong> </dt> 
<dd> Functions for simple fixed and random effects meta-analysis for 
two-sample comparison of binary outcomes.<br>
</dd> 
<dt> <strong>rmetasim</strong> </dt> 
<dd> An interface between R and the metasim simulation engine. Facilitates the 
use of the metasim engine to build and run individual based population genetics 
simulations.<br>
</dd> 
<dt> <strong>robfilter</strong> </dt> 
<dd> Robust time series filters. <br>
</dd> 
<dt> <strong>robust</strong> </dt> 
<dd> Insightful robust package. <br>
</dd> 
<dt> <strong>robustbase</strong> </dt> 
<dd> Basic Robust Statistics. <br>
</dd> 
<dt> <strong>rootSolve</strong> </dt> 
<dd> Nonlinear root finding, equilibrium and steady-state analysis of ordinary 
differential equations.<br>
</dd> 
<dt> <strong>roxygen</strong> </dt> 
<dd> A Doxygen-like in-source documentation system for Rd, collation, 
namespace and callgraphs.<br>
</dd> 
<dt> <strong>rpanel</strong> </dt> 
<dd> Simple interactive controls for R using the tcltk package. <br>
</dd> 
<dt> <strong>rpart</strong> </dt> 
<dd> Recursive PARTitioning and regression trees. <em>Recommended</em>. <br>

</dd> 
<dt> <strong>rpubchem</strong> </dt> 
<dd> R interface to the PubChem collection. <br>
</dd> 
<dt> <strong>rpvm</strong> </dt> 
<dd> R interface to PVM (Parallel Virtual Machine). Provides interface to PVM 
APIs, and examples and documentation for its use.<br>
</dd> 
<dt> <strong>rqmcmb2</strong> </dt> 
<dd> Markov chain marginal bootstrap for quantile regression. <br>
</dd> 
<dt> <strong>rrcov</strong> </dt> 
<dd> Functions for robust location and scatter estimation and robust 
regression with high breakdown point.<br>
</dd> 
<dt> <strong>rrp</strong> </dt> 
<dd> Random Recursive Partitioning. <br>
</dd> 
<dt> <strong>rscproxy</strong> </dt> 
<dd> A portable C-style interface to R (StatConnector). <br>
</dd> 
<dt> <strong>rsprng</strong> </dt> 
<dd> Provides interface to SPRNG (Scalable Parallel Random Number Generators) 
APIs, and examples and documentation for its use.<br>
</dd> 
<dt> <strong>rstream</strong> </dt> 
<dd> Unified object oriented interface for multiple independent streams of 
random numbers from different sources.<br>
</dd> 
<dt> <strong>rtiff</strong> </dt> 
<dd> Read TIFF format images and return them as pixmap objects. <br>
</dd> 
<dt> <strong>runjags</strong> </dt> 
<dd> Run Bayesian MCMC models in the BUGS syntax using JAGS. <br>
</dd> 
<dt> <strong>rv</strong> </dt> 
<dd> Simulation-based random variable object class. <br>
</dd> 
<dt> <strong>rwm</strong> </dt> 
<dd> R Workspace Manager. <br>
</dd> 
<dt> <strong>rwt</strong> </dt> 
<dd> Rice Wavelet Toolbox wrapper, providing a set of functions for performing 
digital signal processing.<br>
</dd> 
<dt> <strong>s20x</strong> </dt> 
<dd> Stats 20x functions. <br>
</dd> 
<dt> <strong>sac</strong> </dt> 
<dd> Semiparametric empirical likelihood ratio based test of changepoint with 
one-change or epidemic alternatives with data-based model diagnostic.<br>
</dd> 
<dt> <strong>sampfling</strong> </dt> 
<dd> Implements a modified version of the Sampford sampling algorithm. Given a 
quantity assigned to each unit in the population, samples are drawn with 
probability proportional to te product of the quantities of the units included 
in the sample.<br>
</dd> 
<dt> <strong>sampleSelection</strong> </dt> 
<dd> Estimation of sample selection models. <br>
</dd> 
<dt> <strong>sampling</strong> </dt> 
<dd> A set of tools to select and to calibrate samples. <br>
</dd> 
<dt> <strong>samr</strong> </dt> 
<dd> Significance Analysis of Microarrays. <br>
</dd> 
<dt> <strong>sandwich</strong> </dt> 
<dd> Model-robust standard error estimators for time series and longitudinal 
data.<br>
</dd> 
<dt> <strong>sapa</strong> </dt> 
<dd> Insightful Spectral Analysis for Physical Applications. <br>
</dd> 
<dt> <strong>sbgcop</strong> </dt> 
<dd> Semiparametric Bayesian Gaussian copula estimation. <br>
</dd> 
<dt> <strong>sca</strong> </dt> 
<dd> Simple Component Analysis. <br>
</dd> 
<dt> <strong>scagnostics</strong> </dt> 
<dd> Compute scagnostics. <br>
</dd> 
<dt> <strong>scaleboot</strong> </dt> 
<dd> Approximately unbiased p-values via multiscale bootstrap. <br>
</dd> 
<dt> <strong>scape</strong> </dt> 
<dd> functions to import and plot results from statistical catch-at-age 
models, used in fisheries stock assessments.<br>
</dd> 
<dt> <strong>scapeMCMC</strong> </dt> 
<dd> Markov-chain Monte Carlo diagnostic plots, accompanying the <strong>scape
</strong> package. <br>
</dd> 
<dt> <strong>scatterplot3d</strong> </dt> 
<dd> Plots a three dimensional (3D) point cloud perspectively. <br>
</dd> 
<dt> <strong>schoolmath</strong> </dt> 
<dd> Functions and datasets for math used in school. <br>
</dd> 
<dt> <strong>sciplot</strong> </dt> 
<dd> Scientific graphing functions for factorial designs. <br>
</dd> 
<dt> <strong>scope</strong> </dt> 
<dd> Data manipulation using arbitrary row and column criteria. <br>
</dd> 
<dt> <strong>scrime</strong> </dt> 
<dd> Tools for the analysis of high-dimensional data developed/implemented at 
the group &ldquo;Statistical Complexity Reduction In Molecular 
Epidemiology&rdquo; (SCRIME), with main focus on SNP data.<br>
</dd> 
<dt> <strong>scuba</strong> </dt> 
<dd> Scuba diving calculations and decompression models. <br>
</dd> 
<dt> <strong>sdcMicro</strong> </dt> 
<dd> Statistical Disclosure Control methods for the generation of public and 
scientific use files.<br>
</dd> 
<dt> <strong>sde</strong> </dt> 
<dd> Simulation and inference for Stochastic Differential Equations. <br>
</dd>
<dt> <strong>sdtalt</strong> </dt> 
<dd> Signal Detection Theory measures and ALTernatives. <br>
</dd> 
<dt> <strong>seacarb</strong> </dt> 
<dd> Calculates parameters of the seawater carbonate system. <br>
</dd> 
<dt> <strong>seas</strong> </dt> 
<dd> Detailed seasonal plots of temperature and precipitation data. <br>
</dd> 
<dt> <strong>seewave</strong> </dt> 
<dd> Time wave analysis and graphical representation. <br>
</dd> 
<dt> <strong>segclust</strong> </dt> 
<dd> Segmentation and segmentation/clustering. <br>
</dd> 
<dt> <strong>segmented</strong> </dt> 
<dd> Functions to estimate break-points of segmented relationships in 
regression models (GLMs).<br>
</dd> 
<dt> <strong>sem</strong> </dt> 
<dd> Functions for fitting general linear Structural Equation Models (with 
observed and unobserved variables) by the method of maximum likelihood using 
the RAM approach.<br>
</dd> 
<dt> <strong>sendplot</strong> </dt> 
<dd> Tool for sending interactive plots. <br>
</dd> 
<dt> <strong>sensR</strong> </dt> 
<dd> Thurstonian models for sensory discrimination. <br>
</dd> 
<dt> <strong>sensitivity</strong> </dt> 
<dd> Sensitivity analysis. <br>
</dd> 
<dt> <strong>seqinr</strong> </dt> 
<dd> Exploratory data analysis and data visualization for biological sequence 
(DNA and protein) data.<br>
</dd> 
<dt> <strong>seqmon</strong> </dt> 
<dd> Sequential monitoring of clinical trials. <br>
</dd> 
<dt> <strong>seriation</strong> </dt> 
<dd> Infrastructure for seriation. <br>
</dd> 
<dt> <strong>session</strong> </dt> 
<dd> Functions for interacting with, saving and restoring R sessions. <br>

</dd> 
<dt> <strong>setRNG</strong> </dt> 
<dd> Set (normal) random number generator and seed. <br>
</dd> 
<dt> <strong>sets</strong> </dt> 
<dd> Data structures and basic operations for ordinary sets, and 
generalizations such as fuzzy sets, multisets, and fuzzy multisets.<br>
</dd> 
<dt> <strong>sfsmisc</strong> </dt> 
<dd> Utilities from Seminar fuer Statistik ETH Zurich. <br>
</dd> 
<dt> <strong>sgeostat</strong> </dt> 
<dd> An object-oriented framework for geostatistical modeling. <br>
</dd> 
<dt> <strong>shape</strong> </dt> 
<dd> Functions for plotting graphical shapes. <br>
</dd> 
<dt> <strong>shapefiles</strong> </dt> 
<dd> Functions to read and write ESRI shapefiles. <br>
</dd> 
<dt> <strong>shapes</strong> </dt> 
<dd> Routines for the statistical analysis of shapes, including procrustes 
analysis, displaying shapes and principal components, testing for mean shape 
difference, thin-plate spline transformation grids and edge superimposition 
methods.<br>
</dd> 
<dt> <strong>siar</strong> </dt> 
<dd> Stable Isotope Analysis in R. <br>
</dd> 
<dt> <strong>sigma2tools</strong> </dt> 
<dd> Test of hypothesis about sigma2. <br>
</dd> 
<dt> <strong>signal</strong> </dt> 
<dd> A set of generally Matlab/Octave-compatible signal processing functions. 
<br> </dd> 
<dt> <strong>signalextraction</strong> </dt> 
<dd> Real-time signal extraction (Direct Filter Approach). <br>
</dd> 
<dt> <strong>simba</strong> </dt> 
<dd> Functions for similarity calculation of binary data. <br>
</dd> 
<dt> <strong>simco</strong> </dt> 
<dd> Import Structure files and deduce similarity coefficients from them. <br>

</dd> 
<dt> <strong>simecol</strong> </dt> 
<dd> SIMulation of ECOLogical (and other) dynamic systems. <br>
</dd> 
<dt> <strong>simex</strong> </dt> 
<dd> SIMEX and MCSIMEX algorithms for measurement error models. <br>
</dd> 
<dt> <strong>similarityRichards</strong> </dt> 
<dd> Similarity of Richards curves. <br>
</dd> 
<dt> <strong>simpleboot</strong> </dt> 
<dd> Simple bootstrap routines. <br>
</dd> 
<dt> <strong>singlecase</strong> </dt> 
<dd> Tests for single case studies in neuropsychology. <br>
</dd> 
<dt> <strong>skewt</strong> </dt> 
<dd> Density, distribution function, quantile function and random generation 
for the skewed t distribution of Fernandez and Steel.<br>
</dd> 
<dt> <strong>sm</strong> </dt> 
<dd> Software linked to the book &ldquo;Applied Smoothing Techniques for Data 
Analysis: The Kernel Approach withS-Plus Illustrations&rdquo; by A. W. Bowman 
and A. Azzalini, 1997, Oxford University Press.<br>
</dd> 
<dt> <strong>sma</strong> </dt> 
<dd> Functions for exploratory (statistical) microarray analysis. <br>
</dd> 
<dt> <strong>smacof</strong> </dt> 
<dd> Multidimensional scaling based on stress minimization by means of 
majorization (smacof).<br>
</dd> 
<dt> <strong>smatr</strong> </dt> 
<dd> (Standardized) Major Axis estimation and Testing Routines. <br>
</dd> 
<dt> <strong>smoothSurv</strong> </dt> 
<dd> Survival regression with smoothed error distribution. <br>
</dd> 
<dt> <strong>smoothtail</strong> </dt> 
<dd> Smooth estimation of generalized Pareto distribution shape parameter. <br>
</dd> 
<dt> <strong>sn</strong> </dt> 
<dd> Functions for manipulating skew-normal probability distributions and for 
fitting them to data, in the scalar and the multivariate case.<br>
</dd> 
<dt> <strong>sna</strong> </dt> 
<dd> A range of tools for social network analysis, including node and 
graph-level indices, structural distance and covariance methods, structural 
equivalence detection, p* modeling, and network visualization. <br>
</dd> 
<dt> <strong>snow</strong> </dt> 
<dd> Simple Network of Workstations: support for simple parallel computing in 
R.<br>
</dd> 
<dt> <strong>snowFT</strong> </dt> 
<dd> Fault Tolerant Simple Network of Workstations. <br>
</dd> 
<dt> <strong>snowfall</strong> </dt> 
<dd> Wrapper around <strong>snow</strong> for easier development of parallel R 
programs.<br>
</dd> 
<dt> <strong>snp.plotter</strong> </dt> 
<dd> Plots of p-values using single SNP and/or haplotype data. <br>
</dd> 
<dt> <strong>snpXpert</strong> </dt> 
<dd> Tools to analyze SNP data. <br>
</dd> 
<dt> <strong>som</strong> </dt> 
<dd> Self-Organizing Maps (with application in gene clustering). <br>
</dd> 
<dt> <strong>sound</strong> </dt> 
<dd> A sound interface for R: Basic functions for dealing with .wav files and 
sound samples.<br>
</dd> 
<dt> <strong>sp</strong> </dt> 
<dd> A package that provides classes and methods for spatial data, including 
utility functions for plotting data as maps, spatial selection, amd much more.
<br> </dd> 
<dt> <strong>spBayes</strong> </dt> 
<dd> Fit Gaussian models with potentially complex hierarchical error 
structures by Markov chain Monte Carlo (MCMC).<br>
</dd> 
<dt> <strong>space</strong> </dt> 
<dd> Sparse PArtial Correlation Estimation. <br>
</dd> 
<dt> <strong>spam</strong> </dt> 
<dd> SPArse Matrix algebra. <br>
</dd> 
<dt> <strong>spatclus</strong> </dt> 
<dd> Arbitrarily shaped multiple spatial cluster detection for case event data.
<br> </dd> 
<dt> <strong>spatgraphs</strong> </dt> 
<dd> Graphs for 2-d point patterns. <br>
</dd> 
<dt> <strong>spatial</strong> </dt> 
<dd> Functions for kriging and point pattern analysis from &ldquo;Modern 
Applied Statistics with S&rdquo; by W. Venables and B. Ripley. Contained in the
VR bundle. <em>Recommended</em>. <br>
</dd> 
<dt> <strong>spatialCovariance</strong> </dt> 
<dd> Computation of spatial covariance matrices for data on rectangles using 
one dimensional numerical integration and analytic results.<br>
</dd> 
<dt> <strong>spatialkernel</strong> </dt> 
<dd> Nonparameteric estimation of spatial segregation in a multivariate point 
process.<br>
</dd> 
<dt> <strong>spatstat</strong> </dt> 
<dd> Data analysis and modelling of two-dimensional point patterns, including 
multitype points and spatial covariates.<br>
</dd> 
<dt> <strong>spc</strong> </dt> 
<dd> Statistical Process Control: evaluation of control charts by means of the 
zero-state, steady-state ARL (Average Run Length), setting up control charts 
for given in-control ARL, and plotting of the related figures.<br>
</dd> 
<dt> <strong>spdep</strong> </dt> 
<dd> A collection of functions to create spatial weights matrix objects from 
polygon contiguities, from point patterns by distance and tesselations, for 
summarising these objects, and for permitting their use in spatial data 
analysis; a collection of tests for spatial autocorrelation, including global 
Moran's I and Geary's C, local Moran's I, saddlepoint approximations for global 
and local Moran's I; and functions for estimating spatial simultaneous 
autoregressive (SAR) models. (Was formerly the three packages:<strong>spweights
</strong>, <strong>sptests</strong>, and <strong>spsarlm</strong>.) <br>
</dd> 
<dt> <strong>spe</strong> </dt> 
<dd> Stochastic Proximity Embedding. <br>
</dd> 
<dt> <strong>spectralGP</strong> </dt> 
<dd> Approximate Gaussian processes using the Fourier basis. <br>
</dd> 
<dt> <strong>spectrino</strong> </dt> 
<dd> Spectra organizer, visualization and data extraction from within R. <br>

</dd> 
<dt> <strong>spgrass6</strong> </dt> 
<dd> Interface between the GRASS 6.0 geographical information system and R. 
<br> </dd> 
<dt> <strong>spgwr</strong> </dt> 
<dd> Geographically weighted regression. <br>
</dd> 
<dt> <strong>splancs</strong> </dt> 
<dd> Spatial and space-time point pattern analysis functions. <br>
</dd> 
<dt> <strong>splus2R</strong> </dt> 
<dd> Insightful package providing missing S-PLUS functionality in R. <br>
</dd>
<dt> <strong>spssDDI</strong> </dt> 
<dd> Read SPSS system files and produce valid DDI version 3.0 documents. <br>

</dd> 
<dt> <strong>spsurvey</strong> </dt> 
<dd> Spatial survey design and analysis. <br>
</dd> 
<dt> <strong>sqldf</strong> </dt> 
<dd> Perform SQL selects on R data frames. <br>
</dd> 
<dt> <strong>ssanv</strong> </dt> 
<dd> Sample Size Adjusted for Nonadherence or Variability of input parameters. 
<br> </dd> 
<dt> <strong>sspir</strong> </dt> 
<dd> State SPace models In R. <br>
</dd> 
<dt> <strong>sspline</strong> </dt> 
<dd> Smoothing splines on the sphere. <br>
</dd> 
<dt> <strong>st</strong> </dt> 
<dd> Shrinkage t statistic. <br>
</dd> 
<dt> <strong>staRt</strong> </dt> 
<dd> Inferenza classica con TI-83 Plus. <br>
</dd> 
<dt> <strong>stab</strong> </dt> 
<dd> Data analysis of drug stability. <br>
</dd> 
<dt> <strong>startupmsg</strong> </dt> 
<dd> Utilities for start-up messages. <br>
</dd> 
<dt> <strong>stashR</strong> </dt> 
<dd> A Set of Tools for Administering SHared Repositories. <br>
</dd> 
<dt> <strong>statmod</strong> </dt> 
<dd> Miscellaneous biostatistical modelling functions. <br>
</dd> 
<dt> <strong>statnet</strong> </dt> 
<dd> Software tools for the statistical modeling of network data. <br>
</dd> 
<dt> <strong>stepPlr</strong> </dt> 
<dd> L2 penalized logistic regression with a stepwise variable selection. <br>

</dd> 
<dt> <strong>stepwise</strong> </dt> 
<dd> A stepwise approach to identifying recombination breakpoints in a 
sequence alignment.<br>
</dd> 
<dt> <strong>stinepack</strong> </dt> 
<dd> Stineman interpolation package. <br>
</dd> 
<dt> <strong>stochasticGEM</strong> </dt> 
<dd> Fitting Stochastic General Epidemic Models. <br>
</dd> 
<dt> <strong>stochmod</strong> </dt> 
<dd> Learning and inference algorithms for a variety of probabilistic models. 
<br> </dd> 
<dt> <strong>stream.net</strong> </dt> 
<dd> Building and analyzing binary stream networks. <br>
</dd> 
<dt> <strong>strucchange</strong> </dt> 
<dd> Various tests on structural change in linear regression models. <br>
</dd>
<dt> <strong>subplex</strong> </dt> 
<dd> The subplex algorithm for unconstrained optimization. <br>
</dd> 
<dt> <strong>subselect</strong> </dt> 
<dd> A collection of functions which assess the quality of variable subsets as 
surrogates for a full data set, and search for subsets which are optimal under 
various criteria.<br>
</dd> 
<dt> <strong>sudoku</strong> </dt> 
<dd> Sudoku puzzle solver. <br>
</dd> 
<dt> <strong>supclust</strong> </dt> 
<dd> Methodology for supervised grouping of predictor variables. <br>
</dd> 
<dt> <strong>superpc</strong> </dt> 
<dd> Supervised principal components. <br>
</dd> 
<dt> <strong>surv2sample</strong> </dt> 
<dd> Two-sample tests for survival analysis. <br>
</dd> 
<dt> <strong>survBayes</strong> </dt> 
<dd> Fits a proportional hazards model to time to event data by a Bayesian 
approach.<br>
</dd> 
<dt> <strong>surveillance</strong> </dt> 
<dd> Outbreak detection algorithms for surveillance data. <br>
</dd> 
<dt> <strong>survey</strong> </dt> 
<dd> Summary statistics, generalized linear models, and general maximum 
likelihood estimation for stratified, cluster-sampled, unequally weighted 
survey samples.<br>
</dd> 
<dt> <strong>surveyNG</strong> </dt> 
<dd> Complex survey samples: database interface, sparse matrices. <br>
</dd> 
<dt> <strong>survival</strong> </dt> 
<dd> Functions for survival analysis, including penalised likelihood. <em>
Recommended</em>. <br>
</dd> 
<dt> <strong>survivalROC</strong> </dt> 
<dd> Time-dependent ROC curve estimation from censored survival data. <br>

</dd> 
<dt> <strong>survrec</strong> </dt> 
<dd> Survival analysis for recurrent event data. <br>
</dd> 
<dt> <strong>svcR</strong> </dt> 
<dd> A support vector machine technique for clustering. <br>
</dd> 
<dt> <strong>svcm</strong> </dt> 
<dd> 2d and 3d Space-Varying Coefficient Models. <br>
</dd> 
<dt> <strong>svmpath</strong> </dt> 
<dd> Computes the entire regularization path for the two-class svm classifier 
with essentialy the same cost as a single SVM fit.<br>
</dd> 
<dt> <strong>systemfit</strong> </dt> 
<dd> Contains functions for fitting simultaneous systems of equations using 
Ordinary Least Sqaures (OLS), Two-Stage Least Squares (2SLS), and Three-Stage 
Least Squares (3SLS). <br>
</dd> 
<dt> <strong>taskPR</strong> </dt> 
<dd> Task-Parallel R package. <br>
</dd> 
<dt> <strong>tcltk2</strong> </dt> 
<dd> A series of widgets and functions to supplement <strong>tcltk</strong>. 
<br> </dd> 
<dt> <strong>tdist</strong> </dt> 
<dd> Computes the distribution of a linear combination of independent 
Student's t variables.<br>
</dd> 
<dt> <strong>tdm</strong> </dt> 
<dd> A tool for Therapeutic Drug Monitoring. <br>
</dd> 
<dt> <strong>tdthap</strong> </dt> 
<dd> Transmission/disequilibrium tests for extended marker haplotypes. <br>

</dd> 
<dt> <strong>tensor</strong> </dt> 
<dd> Tensor product of arrays. <br>
</dd> 
<dt> <strong>tensorA</strong> </dt> 
<dd> Advanced tensors arithmetic with named indices. <br>
</dd> 
<dt> <strong>termstrc</strong> </dt> 
<dd> Term structure and credit spread estimation. <br>
</dd> 
<dt> <strong>tframe</strong> </dt> 
<dd> Time Frame coding kernel: functions for writing code that is independent 
of the way time is represented.<br>
</dd> 
<dt> <strong>tframePlus</strong> </dt> 
<dd> Time Frame coding kernel extensions. <br>
</dd> 
<dt> <strong>tgp</strong> </dt> 
<dd> Bayesian regression and adaptive sampling with Treed Gaussian Process 
models.<br>
</dd> 
<dt> <strong>tileHMM</strong> </dt> 
<dd> Hidden Markov Models for ChIP-on-Chip analysis. <br>
</dd> 
<dt> <strong>time</strong> </dt> 
<dd> Time tracking for developers. <br>
</dd> 
<dt> <strong>timereg</strong> </dt> 
<dd> Code and data sets for the book &ldquo;Dynamic Regression Models for 
Survival Data&rdquo; by T. Martinussen and T. Scheike, 2006, Springer Verlag, 
plus more recent developments.<br>
</dd> 
<dt> <strong>timsac</strong> </dt> 
<dd> TIMe Series Analysis and Control package. <br>
</dd> 
<dt> <strong>tis</strong> </dt> 
<dd> Time indexes and time indexed series. <br>
</dd> 
<dt> <strong>titan</strong> </dt> 
<dd> Titration analysis for mass spectrometry data. <br>
</dd> 
<dt> <strong>titecrm</strong> </dt> 
<dd> TIme-To-Event Continual Reassessment Method and calibration tools. <br>

</dd> 
<dt> <strong>tkrgl</strong> </dt> 
<dd> TK widget tools for <strong>rgl</strong> package. <br>
</dd> 
<dt> <strong>tkrplot</strong> </dt> 
<dd> Simple mechanism for placing R graphics in a Tk widget. <br>
</dd> 
<dt> <strong>tlnise</strong> </dt> 
<dd> Two-level normal independent sampling estimation. <br>
</dd> 
<dt> <strong>tm</strong> </dt> 
<dd> A framework for text mining applications within R. <br>
</dd> 
<dt> <strong>topmodel</strong> </dt> 
<dd> An R implementation of TOPMODEL. <br>
</dd> 
<dt> <strong>tossm</strong> </dt> 
<dd> Testing Of Spatial Structure Methods. <br>
</dd> 
<dt> <strong>tpr</strong> </dt> 
<dd> Temporal Process Regression. <br>
</dd> 
<dt> <strong>trackObjs</strong> </dt> 
<dd> Track objects. <br>
</dd> 
<dt> <strong>tradeCosts</strong> </dt> 
<dd> Post-trade analysis of transaction costs. <br>
</dd> 
<dt> <strong>tree</strong> </dt> 
<dd> Classification and regression trees. <br>
</dd> 
<dt> <strong>triangle</strong> </dt> 
<dd> Standard distribution functions for the triangle distribution. <br>
</dd> 
<dt> <strong>trimcluster</strong> </dt> 
<dd> Cluster analysis with trimming. <br>
</dd> 
<dt> <strong>trip</strong> </dt> 
<dd> Spatial analysis of animal track data. <br>
</dd> 
<dt> <strong>tripEstimation</strong> </dt> 
<dd> Metropolis sampler and supporting functions for estimating animal 
movement from archival tags and satellite fixes.<br>
</dd> 
<dt> <strong>tripack</strong> </dt> 
<dd> A constrained two-dimensional Delaunay triangulation package. <br>
</dd> 
<dt> <strong>truncgof</strong> </dt> 
<dd> Goodness-of-fit tests allowing for left truncated data. <br>
</dd> 
<dt> <strong>trust</strong> </dt> 
<dd> Local optimization using two derivatives and trust regions. <br>
</dd> 
<dt> <strong>tsDyn</strong> </dt> 
<dd> Time series analysis based on dynamical systems theory. <br>
</dd> 
<dt> <strong>tsModel</strong> </dt> 
<dd> Time series modeling for air pollution and health. <br>
</dd> 
<dt> <strong>tseries</strong> </dt> 
<dd> Package for time series analysis with emphasis on non-linear modelling. 
<br> </dd> 
<dt> <strong>tseriesChaos</strong> </dt> 
<dd> Routines for the analysis of non-linear time series. <br>
</dd> 
<dt> <strong>tsfa</strong> </dt> 
<dd> Time Series Factor Analysis. <br>
</dd> 
<dt> <strong>tuneR</strong> </dt> 
<dd> Collection of tools to analyze music, handle wave files, transcription, 
etc.<br>
</dd> 
<dt> <strong>tutoR</strong> </dt> 
<dd> Student-friendly package to mask common functions. <br>
</dd> 
<dt> <strong>twang</strong> </dt> 
<dd> Toolkit for Weighting and Analysis of Nonequivalent Groups. <br>
</dd> 
<dt> <strong>tweedie</strong> </dt> 
<dd> Maximum likelihood computations for Tweedie exponential family models. 
<br> </dd> 
<dt> <strong>twslm</strong> </dt> 
<dd> A two-way semilinear model for normalization and analysis of cDNA 
microarray data.<br>
</dd> 
<dt> <strong>udunits</strong> </dt> 
<dd> Interface to Unidata's routines to convert units. <br>
</dd> 
<dt> <strong>ump</strong> </dt> 
<dd> Uniformly Most Powerful tests. <br>
</dd> 
<dt> <strong>unbalhaar</strong> </dt> 
<dd> Function estimation via Unbalanced Haar wavelets. <br>
</dd> 
<dt> <strong>uncompress</strong> </dt> 
<dd> For uncompressing .Z files. <br>
</dd> 
<dt> <strong>untb</strong> </dt> 
<dd> Ecological drift under the UNTB (Unified Neutral Theory of Biodiversity). 
<br> </dd> 
<dt> <strong>urca</strong> </dt> 
<dd> Unit root and cointegration tests for time series data. <br>
</dd> 
<dt> <strong>urn</strong> </dt> 
<dd> Functions for sampling without replacement (simulated urns). <br>
</dd> 
<dt> <strong>uroot</strong> </dt> 
<dd> Unit root tests and graphics for seasonal time series. <br>
</dd> 
<dt> <strong>vabayelMix</strong> </dt> 
<dd> Variational Bayesian mixture model. <br>
</dd> 
<dt> <strong>varSelRF</strong> </dt> 
<dd> Variable selection using random forests. <br>
</dd> 
<dt> <strong>varmixt</strong> </dt> 
<dd> Mixture model on the variance for the analysis of gene expression data. 
<br> </dd> 
<dt> <strong>vars</strong> </dt> 
<dd> VAR modeling. <br>
</dd> 
<dt> <strong>vbmp</strong> </dt> 
<dd> Variational Bayesian Multinomial Probit Regression. <br>
</dd> 
<dt> <strong>vcd</strong> </dt> 
<dd> Functions and data sets based on the book &ldquo;Visualizing Categorical 
Data&rdquo; by Michael Friendly.<br>
</dd> 
<dt> <strong>vegan</strong> </dt> 
<dd> Various help functions for vegetation scientists and community ecologists.
<br> </dd> 
<dt> <strong>verification</strong> </dt> 
<dd> Utilities for verification of discrete and probabilistic forecasts. <br>

</dd> 
<dt> <strong>verify</strong> </dt> 
<dd> Construction of test suites using verify objects. <br>
</dd> 
<dt> <strong>vioplot</strong> </dt> 
<dd> Violin plots, which are a combination of a box plot and a kernel density 
plot.<br>
</dd> 
<dt> <strong>vrtest</strong> </dt> 
<dd> Variance ratio tests for weak-form market efficiency. <br>
</dd> 
<dt> <strong>waveclock</strong> </dt> 
<dd> Time-frequency analysis of cycling cell luminescence data. <br>
</dd> 
<dt> <strong>waved</strong> </dt> 
<dd> WaveD transform in R. <br>
</dd> 
<dt> <strong>wavelets</strong> </dt> 
<dd> Functions for computing wavelet filters, wavelet transforms and 
multiresolution analyses.<br>
</dd> 
<dt> <strong>waveslim</strong> </dt> 
<dd> Basic wavelet routines for time series analysis. <br>
</dd> 
<dt> <strong>wavethresh</strong> </dt> 
<dd> Software to perform 1-d and 2-d wavelet statistics and transforms. <br>

</dd> 
<dt> <strong>wccsom</strong> </dt> 
<dd> SOM networks for comparing patterns with peak shifts. <br>
</dd> 
<dt> <strong>wgaim</strong> </dt> 
<dd> Whole Genome Average Interval Mapping for QTL detection using mixed 
models.<br>
</dd> 
<dt> <strong>wikibooks</strong> </dt> 
<dd> Functions and datasets for the German WikiBook &ldquo;GNU R&rdquo;. <br>

</dd> 
<dt> <strong>wle</strong> </dt> 
<dd> Robust statistical inference via a weighted likelihood approach. <br>

</dd> 
<dt> <strong>wmtsa</strong> </dt> 
<dd> Insightful Wavelet Methods for Time Series Analysis. <br>
</dd> 
<dt> <strong>wnominate</strong> </dt> 
<dd> WNOMINATE roll call analysis software. <br>
</dd> 
<dt> <strong>wombsoft</strong> </dt> 
<dd> Wombling computation. <br>
</dd> 
<dt> <strong>wordnet</strong> </dt> 
<dd> WordNet interface. <br>
</dd> 
<dt> <strong>write.snns</strong> </dt> 
<dd> Function for exporting data to SNNS (Stuttgart Neural Network Simulator) 
pattern files.<br>
</dd> 
<dt> <strong>xgobi</strong> </dt> 
<dd> Interface to the XGobi and XGvis programs for graphical data analysis. 
<br> </dd> 
<dt> <strong>xlsReadWrite</strong> </dt> 
<dd> Natively read and write Excel files. <br>
</dd> 
<dt> <strong>xtable</strong> </dt> 
<dd> Export data to LaTeX and HTML tables. <br>
</dd> 
<dt> <strong>xts</strong> </dt> 
<dd> Extensible time series. <br>
</dd> 
<dt> <strong>yaImpute</strong> </dt> 
<dd> Performs popular nearest neighbor routines. <br>
</dd> 
<dt> <strong>yaml</strong> </dt> 
<dd> Methods to convert R to YAML and back. <br>
</dd> 
<dt> <strong>yest</strong> </dt> 
<dd> Gaussian independence models. <br>
</dd> 
<dt> <strong>zipfR</strong> </dt> 
<dd> Statistical models for word frequency distributions. <br>
</dd> 
<dt> <strong>zoeppritz</strong> </dt> 
<dd> Zoeppritz equations: calculate and plot scattering coefficients of 
seismic waves when they interact at an interface between two layers.<br>
</dd> 
<dt> <strong>zoo</strong> </dt> 
<dd> A class with methods for totally ordered indexed observations such as 
irregular time series.</dd> </dl> 
<p>See CRAN src/contrib/PACKAGES for more information. </p> 
<p>Some CRAN packages that do not build out of the box on Windows, require 
additional software, or are shipping third party libraries for Windows cannot 
be made available onCRAN in form of a Windows binary packages. Nevertheless, 
some of these packages are available at the &ldquo;CRAN extras&rdquo; 
repository athttp://www.stats.ox.ac.uk/pub/RWin/ kindly provided by Brian D. 
Ripley. Note that this repository is a default repository for recent versions 
of R for Windows.</p> 
<p>There used to be a CRAN src/contrib/Devel directory with packages still 
&ldquo;under development&rdquo; or depending on features only present in the 
current development versions of R. This area is no longer provided, with 
packages formerly in this area either in the regular package area or the archive
src/contrib/Archive. </p> 
<p></p>  Next:&nbsp; Add-on packages from Bioconductor, Previous:&nbsp; Add-on 
packages from CRAN, Up:&nbsp; Which add-on packages exist for R? <br>

<h4>5.1.3 Add-on packages from Omegahat</h4> 
<p>The Omegahat Project for Statistical Computing provides a variety of 
open-source software for statistical applications, with special emphasis on 
web-based software, Java, the Java virtual machine, and distributed computing. A
CRAN style R package repository is available via http://www.omegahat.org/R/. 
</p> 
<p>Currently, there are the following packages. </p> 
<dl> 
<dt> <strong>Aspell</strong> </dt> 
<dd> An interface to facilities in the aspell library. <br>
</dd> 
<dt> <strong>CGIwithR</strong> </dt> 
<dd> Facilities for the use of R to write CGI scripts. <br>
</dd> 
<dt> <strong>CORBA</strong> </dt> 
<dd> Dynamic CORBA client/server facilities for R. Connects to other 
CORBA-aware applications developed in arbitrary languages, on different 
machines and allows R functionality to be exported in the same way to other 
applications.<br>
</dd> 
<dt> <strong>Combinations</strong> </dt> 
<dd> Compute the combinations of choosing r items from n elements. <br>
</dd> 
<dt> <strong>IDocs</strong> </dt> 
<dd> Infrastructure for interactive documents. <br>
</dd> 
<dt> <strong>OOP</strong> </dt> 
<dd> OOP style classes and methods for R and S-Plus. Object references and 
class-based method definition are supported in the style of languages such as 
Java and C++.<br>
</dd> 
<dt> <strong>RCurl</strong> </dt> 
<dd> Allows one to compose HTTP requests to fetch URIs, post forms, etc., and 
process the results returned by the Web server.<br>
</dd> 
<dt> <strong>RDCOMClient</strong> </dt> 
<dd> Provides dynamic client-side access to (D)COM applications from within R. 
<br> </dd> 
<dt> <strong>RDCOMEvents</strong> </dt> 
<dd> Provides facilities to use R functions and objects as handlers for DCOM 
events.<br>
</dd> 
<dt> <strong>RDCOMServer</strong> </dt> 
<dd> Facilities for exporting S objects and functions as COM objects. <br>

</dd> 
<dt> <strong>REmbeddedPostgres</strong> </dt> 
<dd> Allows R functions and objects to be used to implement SQL functions 
&mdash; per-record, aggregate and trigger functions.<br>
</dd> 
<dt> <strong>REventLoop</strong> </dt> 
<dd> An abstract event loop mechanism that is toolkit independent and can be 
used to to replace the R event loop.<br>
</dd> 
<dt> <strong>Rexif</strong> </dt> 
<dd> Extract meta-information from JPEG files. <br>
</dd> 
<dt> <strong>RGdkPixbuf</strong> </dt> 
<dd> S language functions to access the facilities in the GdkPixbuf library 
for manipulating images.<br>
</dd> 
<dt> <strong>RGnumeric</strong> </dt> 
<dd> A plugin for the Gnumeric spreadsheet that allows R functions to be 
called from cells within the sheet, automatic recalculation, etc.<br>
</dd> 
<dt> <strong>RGtk</strong> </dt> 
<dd> Facilities in the S language for programming graphical interfaces using 
Gtk, the GnomeGUI toolkit. <br>
</dd> 
<dt> <strong>RGtkBindingGenerator</strong> </dt> 
<dd> A meta-package which generates C and R code to provide bindings to a 
Gtk-based library.<br>
</dd> 
<dt> <strong>RGtkExtra</strong> </dt> 
<dd> A collection of S functions that provide an interface to the widgets in 
the gtk+extra library such as the GtkSheet data-grid display, icon list, file 
list and directory tree.<br>
</dd> 
<dt> <strong>RGtkGlade</strong> </dt> 
<dd> S language bindings providing an interface to Glade, the interactive Gnome
GUI creator. <br>
</dd> 
<dt> <strong>RGtkHTML</strong> </dt> 
<dd> A collection of S functions that provide an interface to creating and 
controlling anHTML widget which can be used to display HTML documents from 
files or content generated dynamically in S.<br>
</dd> 
<dt> <strong>RGtkIPrimitives</strong> </dt> 
<dd> A collection of low-level primitives for interactive use with R graphics 
and the gtkDevice using<strong>RGtk</strong>. <br>
</dd> 
<dt> <strong>RGtkViewers</strong> </dt> 
<dd> A collection of tools for viewing different S objects, databases, class 
and widget hierarchies, S source file contents, etc.<br>
</dd> 
<dt> <strong>RJavaDevice</strong> </dt> 
<dd> A graphics device for R that uses Java components and graphics. APIs. <br>
</dd> 
<dt> <strong>RMatlab</strong> </dt> 
<dd> A bi-directional interface between R and Matlab. <br>
</dd> 
<dt> <strong>RObjectTables</strong> </dt> 
<dd> The C and S code allows one to define R objects to be used as elements of 
the search path with their own semantics and facilities for reading and writing 
variables. The objects implement a simple interface via R functions (either 
methods or closures) and can access external data, e.g., in other applications, 
languages, formats,... <br>
</dd> 
<dt> <strong>RSMethods</strong> </dt> 
<dd> An implementation of S version 4 methods and classes for R, consistent 
with the basic material in &ldquo;Programming with Data&rdquo; by John M. 
Chambers, 1998, Springer NY.<br>
</dd> 
<dt> <strong>RSPerl</strong> </dt> 
<dd> An interface from R to an embedded, persistent Perl interpreter, allowing 
one to call arbitrary Perl subroutines, classes and methods.<br>
</dd> 
<dt> <strong>RSPython</strong> </dt> 
<dd> Allows Python programs to invoke S functions, methods, etc., and S code 
to call Python functionality.<br>
</dd> 
<dt> <strong>RXLisp</strong> </dt> 
<dd> An interface to call XLisp-Stat functions from within R. <br>
</dd> 
<dt> <strong>Rcompression</strong> </dt> 
<dd> In-memory decompression for GNU zip and bzip2 formats. <br>
</dd> 
<dt> <strong>Rlibstree</strong> </dt> 
<dd> Suffix Trees in R via the libstree C library. <br>
</dd> 
<dt> <strong>Rstem</strong> </dt> 
<dd> Interface to Snowball implementation of Porter's word stemming algorithm. 
<br> </dd> 
<dt> <strong>RwxWidgets</strong> </dt> 
<dd> Facilities to program GUIs using wxWidgets in R. <br>
</dd> 
<dt> <strong>Ryacas</strong> </dt> 
<dd> R interface to yacas. <br>
</dd> 
<dt> <strong>SASXML</strong> </dt> 
<dd> Example for reading XML files in SAS 8.2 manner. <br>
</dd> 
<dt> <strong>SJava</strong> </dt> 
<dd> An interface from R to Java to create and call Java objects and methods. 
<br> </dd> 
<dt> <strong>SLanguage</strong> </dt> 
<dd> Functions and C support utilities to support S language programming that 
can work in both R andS-Plus. <br>
</dd> 
<dt> <strong>SNetscape</strong> </dt> 
<dd> Plugin for Netscape and JavaScript. <br>
</dd> 
<dt> <strong>SSOAP</strong> </dt> 
<dd> A client interface to SOAP (Simple Object Access Protocol) servers from 
within S.<br>
</dd> 
<dt> <strong>SWinRegistry</strong> </dt> 
<dd> Provides access from within R to read and write the Windows registry. <br>
</dd> 
<dt> <strong>SWinTypeLibs</strong> </dt> 
<dd> Provides ways to extract type information from type libraries and/or DCOM 
objects that describes the methods, properties, etc., of an interface.<br>
</dd>
<dt> <strong>SXalan</strong> </dt> 
<dd> Process XML documents using XSL functions implemented in R and 
dynamically substituting output from R.<br>
</dd> 
<dt> <strong>Slcc</strong> </dt> 
<dd> Parses C source code, allowing one to analyze and automatically generate 
interfaces from S to that code, including the table of S-accessible native 
symbols, parameter count and type information, S constructors from C objects, 
call graphs, etc.<br>
</dd> 
<dt> <strong>Sxslt</strong> </dt> 
<dd> An extension module for libxslt, the XML- XSL document translator, that 
allowsXSL functions to be implemented via R functions. <br>
</dd> 
<dt> <strong>XML</strong> </dt> 
<dd> Tools for reading XML documents and DTDs. </dd> </dl> 
<p></p>  Next:&nbsp; Other add-on packages, Previous:&nbsp; Add-on packages 
from Omegahat, Up:&nbsp; Which add-on packages exist for R? <br>

<h4>5.1.4 Add-on packages from Bioconductor</h4> 
<p>The Bioconductor Project produces an open source software framework that 
will assist biologists and statisticians working in bioinformatics, with 
primary emphasis on inference using DNA microarrays. ACRAN style R package 
repository is available viahttp://www.bioconductor.org/. </p> 
<p>The following R packages are contained in the current release of 
Bioconductor, with more packages under development.</p> 
<dl> 
<dt> <strong>ABarray</strong> </dt> 
<dd> Microarray QA and statistical data analysis for Applied Biosystems Genome 
Survey Micorarray (AB1700) gene expression data.<br>
</dd> 
<dt> <strong>ACME</strong> </dt> 
<dd> Algorithms for Calculating Microarray Enrichment (ACME). <br>
</dd> 
<dt> <strong>AffyCompatible</strong> </dt> 
<dd> Affymetrix GeneChip software compatibility <br>
</dd> 
<dt> <strong>AffyExpress</strong> </dt> 
<dd> Affymetrix quality assessment and analysis tool. <br>
</dd> 
<dt> <strong>AnnBuilder</strong> </dt> 
<dd> Processing annotation date from public data repositories and building 
annoation data packages or XML data documents using the source data.<br>
</dd> 
<dt> <strong>AnnotationDbi</strong> </dt> 
<dd> Annotation DataBase Interface. <br>
</dd> 
<dt> <strong>BAC</strong> </dt> 
<dd> Bayesian Analysis of Chip-chip experiment. <br>
</dd> 
<dt> <strong>BCRANK</strong> </dt> 
<dd> Predicting binding site consensus from ranked DNA sequences. <br>
</dd> 
<dt> <strong>BGmix</strong> </dt> 
<dd> Bayesian models for differential gene expression. <br>
</dd> 
<dt> <strong>BSgenome</strong> </dt> 
<dd> Infrastructure for Biostrings-based genome data packages. <br>
</dd> 
<dt> <strong>BioMVCClass</strong> </dt> 
<dd> Model-View-Controller (MVC) classes that use <strong>Biobase</strong>. 
<br> </dd> 
<dt> <strong>Biobase</strong> </dt> 
<dd> Object-oriented representation and manipulation of genomic data (S4 class 
structure).<br>
</dd> 
<dt> <strong>BiocCaseStudies</strong> </dt> 
<dd> Support for the Bioconductor Case Studies monograph. <br>
</dd> 
<dt> <strong>Biostrings</strong> </dt> 
<dd> Class definitions and generics for biological sequences along with 
pattern matching algorithms.<br>
</dd> 
<dt> <strong>BufferedMatrix</strong> </dt> 
<dd> Microarray analysis methods that use BufferedMatrix objects. <br>
</dd> 
<dt> <strong>BufferedMatrixMethods</strong> </dt> 
<dd> A matrix data storage object held in temporary files. <br>
</dd> 
<dt> <strong>CALIB</strong> </dt> 
<dd> Calibration model for estimating absolute expression levels from 
microarray data.<br>
</dd> 
<dt> <strong>CGHcall</strong> </dt> 
<dd> Calling aberrations for array CGH tumor profiles. <br>
</dd> 
<dt> <strong>CORREP</strong> </dt> 
<dd> Multivariate correlation estimation and statistical inference. <br>
</dd> 
<dt> <strong>Category</strong> </dt> 
<dd> A collection of tools for performing category analysis. <br>
</dd> 
<dt> <strong>CoCiteStats</strong> </dt> 
<dd> A collection of software tools for dealing with co-citation data. <br>

</dd> 
<dt> <strong>DEDS</strong> </dt> 
<dd> Differential Expression via Distance Summary for microarray data. <br>

</dd> 
<dt> <strong>DNAcopy</strong> </dt> 
<dd> Segments DNA copy number data using circular binary segmentation to 
detect regions with abnormal copy number.<br>
</dd> 
<dt> <strong>DynDoc</strong> </dt> 
<dd> Functionality to create and interact with dynamic documents, vignettes, 
and other navigable documents.<br>
</dd> 
<dt> <strong>EBImage</strong> </dt> 
<dd> Image processing and image analysis toolkit. <br>
</dd> 
<dt> <strong>EBarrays</strong> </dt> 
<dd> Empirical Bayes tools for the analysis of replicated microarray data 
across multiple conditions.<br>
</dd> 
<dt> <strong>GEOmetadb</strong> </dt> 
<dd> A compilation of metadata from NCBI GEO. <br>
</dd> 
<dt> <strong>GEOquery</strong> </dt> 
<dd> Get data from NCBI Gene Expression Omnibus (GEO). <br>
</dd> 
<dt> <strong>GGBase</strong> </dt> 
<dd> Infrastructure for genetics of gene expression. <br>
</dd> 
<dt> <strong>GGtools</strong> </dt> 
<dd> Software and data for genetical genomics. <br>
</dd> 
<dt> <strong>GLAD</strong> </dt> 
<dd> Gain and Loss Analysis of DNA. <br>
</dd> 
<dt> <strong>GOstats</strong> </dt> 
<dd> Tools for manipulating GO and microarrays. <br>
</dd> 
<dt> <strong>GSEABase</strong> </dt> 
<dd> Gene set enrichment data structures and methods. <br>
</dd> 
<dt> <strong>GSEAlm</strong> </dt> 
<dd> Linear model toolset for Gene Set Enrichment Analysis. <br>
</dd> 
<dt> <strong>GeneMeta</strong> </dt> 
<dd> A collection of meta-analysis tools for analyzing high throughput 
experimental data.<br>
</dd> 
<dt> <strong>GeneR</strong> </dt> 
<dd> Package manipulating nucleotidic sequences (Embl, Fasta, GenBank). <br>

</dd> 
<dt> <strong>GeneRfold</strong> </dt> 
<dd> R for genes and sequences, using viennaRNA package (fold). <br>
</dd> 
<dt> <strong>GeneSelector</strong> </dt> 
<dd> GeneSelector. <br>
</dd> 
<dt> <strong>GeneSpring</strong> </dt> 
<dd> Functions and class definitions to be able to read and write GeneSpring 
specific data objects and convert them to Bioconductor objects. <br>
</dd> 
<dt> <strong>GeneTraffic</strong> </dt> 
<dd> GeneTraffic R integration functions. <br>
</dd> 
<dt> <strong>GeneticsBase</strong> </dt> 
<dd> Classes and functions for handling genetic data. <br>
</dd> 
<dt> <strong>GeneticsDesign</strong> </dt> 
<dd> Functions for designing genetics studies. <br>
</dd> 
<dt> <strong>GeneticsPed</strong> </dt> 
<dd> Pedigree and genetic relationship functions. <br>
</dd> 
<dt> <strong>GenomeGraphs</strong> </dt> 
<dd> Plotting genomic information from Ensembl. <br>
</dd> 
<dt> <strong>GlobalAncova</strong> </dt> 
<dd> Calculates a global test for differential gene expression between groups. 
<br> </dd> 
<dt> <strong>GraphAT</strong> </dt> 
<dd> Graph theoretic Association Tests. <br>
</dd> 
<dt> <strong>GraphAlignment</strong> </dt> 
<dd> GraphAlignment. <br>
</dd> 
<dt> <strong>HEM</strong> </dt> 
<dd> Heterogeneous Error Model for analysis of microarray data. <br>
</dd> 
<dt> <strong>Harshlight</strong> </dt> 
<dd> A &ldquo;corrective make-up&rdquo; program for microarray chips. <br>

</dd> 
<dt> <strong>Heatplus</strong> </dt> 
<dd> A heat map displaying covariates and coloring clusters. <br>
</dd> 
<dt> <strong>Icens</strong> </dt> 
<dd> Functions for computing the NPMLE for censored and truncated data. <br>

</dd> 
<dt> <strong>KEGGSOAP</strong> </dt> 
<dd> Client-side SOAP access KEGG. <br>
</dd> 
<dt> <strong>LBE</strong> </dt> 
<dd> Estimation of the false discovery rate. <br>
</dd> 
<dt> <strong>LMGene</strong> </dt> 
<dd> Analysis of microarray data using a linear model and glog data 
transformation.<br>
</dd> 
<dt> <strong>LPE</strong> </dt> 
<dd> Significance analysis of microarray data with small number of replicates 
using the Local Pooled Error (LPE) method.<br>
</dd> 
<dt> <strong>MANOR</strong> </dt> 
<dd> Micro-Array NORmalization. <br>
</dd> 
<dt> <strong>MLInterfaces</strong> </dt> 
<dd> Uniform interfaces to machine learning code for the exprSet class from 
Bioconductor.<br>
</dd> 
<dt> <strong>MVCClass</strong> </dt> 
<dd> Model-View-Controller (MVC) classes. <br>
</dd> 
<dt> <strong>MantelCorr</strong> </dt> 
<dd> Compute Mantel Cluster Correlations. <br>
</dd> 
<dt> <strong>MassSpecWavelet</strong> </dt> 
<dd> Mass spectrum processing by wavelet-based algorithms. <br>
</dd> 
<dt> <strong>MeasurementError.cor</strong> </dt> 
<dd> Two-stage measurement error model for correlation estimation with smaller 
bias than the usual sample correlation.<br>
</dd> 
<dt> <strong>MergeMaid</strong> </dt> 
<dd> Cross-study comparison of gene expression array data. <br>
</dd> 
<dt> <strong>Mfuzz</strong> </dt> 
<dd> Soft clustering of time series gene expression data. <br>
</dd> 
<dt> <strong>MiPP</strong> </dt> 
<dd> Misclassification Penalized Posterior Classification. <br>
</dd> 
<dt> <strong>OCplus</strong> </dt> 
<dd> Operating characteristics plus sample size and local fdr for microarray 
experiments.<br>
</dd> 
<dt> <strong>OLIN</strong> </dt> 
<dd> Optimized Local Intensity-dependent Normalisation of two-color 
microarrays.<br>
</dd> 
<dt> <strong>OLINgui</strong> </dt> 
<dd> Graphical user interface for <strong>OLIN</strong>. <br>
</dd> 
<dt> <strong>OrderedList</strong> </dt> 
<dd> Similarities of ordered gene lists. <br>
</dd> 
<dt> <strong>OutlierD</strong> </dt> 
<dd> Outlier detection using quantile regression on the M-A scatterplots of 
high-throughput data.<br>
</dd> 
<dt> <strong>PCpheno</strong> </dt> 
<dd> Phenotypes and cellular organizational units. <br>
</dd> 
<dt> <strong>PGSEA</strong> </dt> 
<dd> Parametric Gene Set Enrichment Analysis. <br>
</dd> 
<dt> <strong>PROcess</strong> </dt> 
<dd> Ciphergen SELDI-TOF processing. <br>
</dd> 
<dt> <strong>RBGL</strong> </dt> 
<dd> An interface between the graph package and the Boost graph libraries, 
allowing for fast manipulation of graph objects in R.<br>
</dd> 
<dt> <strong>RBioinf</strong> </dt> 
<dd> Support for R for Bioinformatics monograph. <br>
</dd> 
<dt> <strong>RLMM</strong> </dt> 
<dd> A genotype calling algorithm for Affymetrix SNP arrays. <br>
</dd> 
<dt> <strong>RMAGEML</strong> </dt> 
<dd> Functionality to handle MAGEML documents. <br>
</dd> 
<dt> <strong>ROC</strong> </dt> 
<dd> Receiver Operating Characteristic (ROC) approach for identifying genes 
that are differentially expressed in two types of samples.<br>
</dd> 
<dt> <strong>RSNPper</strong> </dt> 
<dd> Interface to chip.org::SNPper for SNP-related data. <br>
</dd> 
<dt> <strong>RWebServices</strong> </dt> 
<dd> Expose R functions as web services through Java/Axis/Apache. <br>
</dd> 
<dt> <strong>RankProd</strong> </dt> 
<dd> Rank Product method for identifying differentially expressed genes. <br>

</dd> 
<dt> <strong>RbcBook1</strong> </dt> 
<dd> Support for Springer monograph on Bioconductor. <br>
</dd> 
<dt> <strong>Rdbi</strong> </dt> 
<dd> Generic framework for database access in R. <br>
</dd> 
<dt> <strong>RdbiPgSQL</strong> </dt> 
<dd> Methods for accessing data stored in PostgreSQL tables. <br>
</dd> 
<dt> <strong>Rdisop</strong> </dt> 
<dd> Decomposition of isotopic patterns. <br>
</dd> 
<dt> <strong>RefPlus</strong> </dt> 
<dd> Functions for pre-processing Affymetrix data using the RMA+ and the RMA++ 
methods.<br>
</dd> 
<dt> <strong>Resourcerer</strong> </dt> 
<dd> Read annotation data from TIGR Resourcerer or convert the annotation data 
into Bioconductor data package.<br>
</dd> 
<dt> <strong>Rgraphviz</strong> </dt> 
<dd> An interface with Graphviz for plotting graph objects in R. <br>
</dd> 
<dt> <strong>Ringo</strong> </dt> 
<dd> R Investigation of NimbleGen Oligoarrays.. <br>
</dd> 
<dt> <strong>Rintact</strong> </dt> 
<dd> Interface to EBI Intact protein interaction data base. <br>
</dd> 
<dt> <strong>Rredland</strong> </dt> 
<dd> Interface to redland RDF utilities. <br>
</dd> 
<dt> <strong>Rtreemix</strong> </dt> 
<dd> Mutagenetic trees mixture models. <br>
</dd> 
<dt> <strong>Ruuid</strong> </dt> 
<dd> Creates Universally Unique ID values (UUIDs) in R. <br>
</dd> 
<dt> <strong>SAGx</strong> </dt> 
<dd> Statistical Analysis of the GeneChip. <br>
</dd> 
<dt> <strong>SBMLR</strong> </dt> 
<dd> Systems Biology Markup Language (SBML) interface and biochemical system 
analysis tools.<br>
</dd> 
<dt> <strong>SLGI</strong> </dt> 
<dd> Synthetic Lethal Genetic Interaction. <br>
</dd> 
<dt> <strong>SLqPCR</strong> </dt> 
<dd> Functions for analysis of real-time quantitative PCR data at SIRS-Lab 
GmbH.<br>
</dd> 
<dt> <strong>SMAP</strong> </dt> 
<dd> A Segmental Maximum A Posteriori approach to array-CGH copy number 
profiling.<br>
</dd> 
<dt> <strong>SNPchip</strong> </dt> 
<dd> Classes and methods for high throughput SNP chip data. <br>
</dd> 
<dt> <strong>ScISI</strong> </dt> 
<dd> In Silico Interactome. <br>
</dd> 
<dt> <strong>SemSim</strong> </dt> 
<dd> Gene ontology-based semantic similarity measures. <br>
</dd> 
<dt> <strong>TypeInfo</strong> </dt> 
<dd> Optional type specification prototype. <br>
</dd> 
<dt> <strong>VanillaICE</strong> </dt> 
<dd> Methods for fitting Hidden Markov Models to SNP chip data. <br>
</dd> 
<dt> <strong>XDE</strong> </dt> 
<dd> A Bayesian hierarchical model for cross-study analysis of differential 
gene expression.<br>
</dd> 
<dt> <strong>aCGH</strong> </dt> 
<dd> Classes and functions for Array Comparative Genomic Hybridization data. 
<br> </dd> 
<dt> <strong>adSplit</strong> </dt> 
<dd> Annotation-driven clustering. <br>
</dd> 
<dt> <strong>affxparser</strong> </dt> 
<dd> Package for parsing Affymetrix files (CDF, CEL, CHP, BPMAP, BAR). <br>

</dd> 
<dt> <strong>affy</strong> </dt> 
<dd> Methods for Affymetrix Oligonucleotide Arrays. <br>
</dd> 
<dt> <strong>affyPLM</strong> </dt> 
<dd> For fitting Probe Level Models. <br>
</dd> 
<dt> <strong>affyPara</strong> </dt> 
<dd> Parallelized preprocessing methods for Affymetrix Oligonucleotide Arrays. 
<br> </dd> 
<dt> <strong>affyQCReport</strong> </dt> 
<dd> QC Report Generation for affyBatch objects. <br>
</dd> 
<dt> <strong>affycomp</strong> </dt> 
<dd> Graphics toolbox for assessment of Affymetrix expression measures. <br>

</dd> 
<dt> <strong>affycoretools</strong> </dt> 
<dd> Functions useful for those doing repetitive analyses. <br>
</dd> 
<dt> <strong>affyio</strong> </dt> 
<dd> Tools for parsing Affymetrix data files. <br>
</dd> 
<dt> <strong>affylmGUI</strong> </dt> 
<dd> Graphical User Interface for affy analysis using package <strong>limma
</strong>. <br>
</dd> 
<dt> <strong>affypdnn</strong> </dt> 
<dd> Probe Dependent Nearest Neighbors (PDNN) for the <strong>affy</strong> 
package.<br>
</dd> 
<dt> <strong>altcdfenvs</strong> </dt> 
<dd> Utilities to handle cdfenvs. <br>
</dd> 
<dt> <strong>annaffy</strong> </dt> 
<dd> Functions for handling data from Bioconductor Affymetrix annotation data 
packages.<br>
</dd> 
<dt> <strong>annotate</strong> </dt> 
<dd> Associate experimental data in real time to biological metadata from web 
databases such as GenBank, LocusLink and PubMed. Process and store query 
results. GenerateHTML reports of analyses. <br>
</dd> 
<dt> <strong>annotationTools</strong> </dt> 
<dd> Annotate microarrays and perform cross-species gene expression analyses 
using flat file databases.<br>
</dd> 
<dt> <strong>apComplex</strong> </dt> 
<dd> Estimate protein complex membership using AP-MS protein data. <br>
</dd> 
<dt> <strong>aroma.light</strong> </dt> 
<dd> Light-weight methods for normalization and visualization of microarray 
data using only basic R data types. <br>
</dd> 
<dt> <strong>arrayQuality</strong> </dt> 
<dd> Performing print-run and array level quality assessment. <br>
</dd> 
<dt> <strong>arrayQualityMetrics</strong> </dt> 
<dd> Quality metrics on ExpressionSets. <br>
</dd> 
<dt> <strong>beadarray</strong> </dt> 
<dd> Quality control and low-level analysis of BeadArrays. <br>
</dd> 
<dt> <strong>beadarraySNP</strong> </dt> 
<dd> Normalization and reporting of Illumina SNP bead arrays. <br>
</dd> 
<dt> <strong>bgafun</strong> </dt> 
<dd> A method to identify specifity determining residues in protein families. 
<br> </dd> 
<dt> <strong>bgx</strong> </dt> 
<dd> Bayesian Gene eXpression. <br>
</dd> 
<dt> <strong>bioDist</strong> </dt> 
<dd> A collection of software tools for calculating distance measures. <br>

</dd> 
<dt> <strong>biocGraph</strong> </dt> 
<dd> Graph examples and use cases in Bioinformatics. <br>
</dd> 
<dt> <strong>biocViews</strong> </dt> 
<dd> Categorized views of R package repositories. <br>
</dd> 
<dt> <strong>biomaRt</strong> </dt> 
<dd> Interface to BioMart databases (e.g., Ensembl) <br>
</dd> 
<dt> <strong>bridge</strong> </dt> 
<dd> Bayesian Robust Inference for Differential Gene Expression. <br>
</dd> 
<dt> <strong>cellHTS</strong> </dt> 
<dd> Analysis of cell-based screens. <br>
</dd> 
<dt> <strong>cellHTS2</strong> </dt> 
<dd> Analysis of cell-based screens &mdash; revised version of <strong>cellHTS
</strong>. <br>
</dd> 
<dt> <strong>cghMCR</strong> </dt> 
<dd> Find chromosome regions showing common gains/losses. <br>
</dd> 
<dt> <strong>clusterStab</strong> </dt> 
<dd> Compute cluster stability scores for microarray data. <br>
</dd> 
<dt> <strong>codelink</strong> </dt> 
<dd> Manipulation of Codelink Bioarrays data. <br>
</dd> 
<dt> <strong>convert</strong> </dt> 
<dd> Convert Microarray Data Objects. <br>
</dd> 
<dt> <strong>copa</strong> </dt> 
<dd> Functions to perform cancer outlier profile analysis. <br>
</dd> 
<dt> <strong>cosmo</strong> </dt> 
<dd> Supervised detection of conserved motifs in DNA sequences. <br>
</dd> 
<dt> <strong>cosmoGUI</strong> </dt> 
<dd> GUI for constructing constraint sets used by the <strong>cosmo</strong> 
package.<br>
</dd> 
<dt> <strong>ctc</strong> </dt> 
<dd> Tools to export and import Tree and Cluster to other programs. <br>
</dd> 
<dt> <strong>daMA</strong> </dt> 
<dd> Functions for the efficient design of factorial two-color microarray 
experiments and for the statistical analysis of factorial microarray data.<br>

</dd> 
<dt> <strong>diffGeneAnalysis</strong> </dt> 
<dd> Performs differential Gene expression Analysis. <br>
</dd> 
<dt> <strong>ecolitk</strong> </dt> 
<dd> Metadata and tools to work with E. coli. <br>
</dd> 
<dt> <strong>edd</strong> </dt> 
<dd> Expression density diagnostics: graphical methods and pattern recognition 
algorithms for distribution shape classification.<br>
</dd> 
<dt> <strong>exonmap</strong> </dt> 
<dd> High level analysis of Affymetrix exon array data. <br>
</dd> 
<dt> <strong>explorase</strong> </dt> 
<dd> GUI for exploratory data analysis of systems biology data. <br>
</dd> 
<dt> <strong>externalVector</strong> </dt> 
<dd> Vector objects for R with external storage. <br>
</dd> 
<dt> <strong>factDesign</strong> </dt> 
<dd> A set of tools for analyzing data from factorial designed microarray 
experiments. The functions can be used to evaluate appropriate tests of 
contrast and perform single outlier detection.<br>
</dd> 
<dt> <strong>fbat</strong> </dt> 
<dd> Family Based Association Tests for genetic data. <br>
</dd> 
<dt> <strong>fdrame</strong> </dt> 
<dd> FDR Adjustments of Microarray Experiments (FDR-AME). <br>
</dd> 
<dt> <strong>flowClust</strong> </dt> 
<dd> Clustering for flow cytometry. <br>
</dd> 
<dt> <strong>flowCore</strong> </dt> 
<dd> Basic structures for flow cytometry data. <br>
</dd> 
<dt> <strong>flowQ</strong> </dt> 
<dd> Qualitiy control for flow cytometry. <br>
</dd> 
<dt> <strong>flowUtils</strong> </dt> 
<dd> Utilities for flow cytometry. <br>
</dd> 
<dt> <strong>flowViz</strong> </dt> 
<dd> Visualization for flow cytometry. <br>
</dd> 
<dt> <strong>gaga</strong> </dt> 
<dd> GaGa hierarchical model for microarray data analysis. <br>
</dd> 
<dt> <strong>gaggle</strong> </dt> 
<dd> Broadcast data between R and Java bioinformatics programs. <br>
</dd> 
<dt> <strong>gcrma</strong> </dt> 
<dd> Background adjustment using sequence information. <br>
</dd> 
<dt> <strong>genArise</strong> </dt> 
<dd> A tool for dual color microarray data. <br>
</dd> 
<dt> <strong>geneRecommender</strong> </dt> 
<dd> A gene recommender algorithm to identify genes coexpressed with a query 
set of genes.<br>
</dd> 
<dt> <strong>genefilter</strong> </dt> 
<dd> Tools for sequentially filtering genes using a wide variety of filtering 
functions. Example of filters include: number of missing value, coefficient of 
variation of expression measures, ANOVA p-value, Cox model p-values. Sequential 
application of filtering functions to genes.<br>
</dd> 
<dt> <strong>geneplotter</strong> </dt> 
<dd> Graphical tools for genomic data, for example for plotting expression 
data along a chromosome or producing color images of expression data matrices.
<br> </dd> 
<dt> <strong>globaltest</strong> </dt> 
<dd> Testing globally whether a group of genes is significantly related to 
some clinical variable of interest.<br>
</dd> 
<dt> <strong>goProfiles</strong> </dt> 
<dd> Statistical analysis of functional profiles. <br>
</dd> 
<dt> <strong>goTools</strong> </dt> 
<dd> Functions for description/comparison of oligo ID list using the Gene 
Ontology database.<br>
</dd> 
<dt> <strong>gpls</strong> </dt> 
<dd> Classification using generalized partial least squares for two-group and 
multi-group classification.<br>
</dd> 
<dt> <strong>graph</strong> </dt> 
<dd> Classes and tools for creating and manipulating graphs within R. <br>

</dd> 
<dt> <strong>hexbin</strong> </dt> 
<dd> Binning functions, in particular hexagonal bins for graphing. <br>
</dd> 
<dt> <strong>hopach</strong> </dt> 
<dd> Hierarchical Ordered Partitioning and Collapsing Hybrid (HOPACH). <br>

</dd> 
<dt> <strong>hypergraph</strong> </dt> 
<dd> Capabilities for representing and manipulating hypergraphs. <br>
</dd> 
<dt> <strong>idiogram</strong> </dt> 
<dd> Plotting genomic data by chromosomal location. <br>
</dd> 
<dt> <strong>impute</strong> </dt> 
<dd> Imputation for microarray data (currently KNN only). <br>
</dd> 
<dt> <strong>keggorth</strong> </dt> 
<dd> Graph support for KO, KEGG Orthology. <br>
</dd> 
<dt> <strong>lapmix</strong> </dt> 
<dd> Laplace mixture model in microarray experiments. <br>
</dd> 
<dt> <strong>limma</strong> </dt> 
<dd> Linear models for microarray data. <br>
</dd> 
<dt> <strong>limmaGUI</strong> </dt> 
<dd> Graphical User Interface for package <strong>limma</strong>. <br>
</dd> 
<dt> <strong>logicFS</strong> </dt> 
<dd> Identification of SNP interactions. <br>
</dd> 
<dt> <strong>lumi</strong> </dt> 
<dd> BeadArray specific methods for Illumina microarrays. <br>
</dd> 
<dt> <strong>maCorrPlot</strong> </dt> 
<dd> Visualize artificial correlation in microarray data. <br>
</dd> 
<dt> <strong>maDB</strong> </dt> 
<dd> Microarray database and utility functions for microarray analysis. <br>

</dd> 
<dt> <strong>maSigPro</strong> </dt> 
<dd> Significant gene expression profile differeneces in time course 
microarray data.<br>
</dd> 
<dt> <strong>maanova</strong> </dt> 
<dd> Tools for analyzing micro array experiments. <br>
</dd> 
<dt> <strong>macat</strong> </dt> 
<dd> MicroArray Chromosome Analysis Tool. <br>
</dd> 
<dt> <strong>made4</strong> </dt> 
<dd> Multivariate analysis of microarray data using ADE4. <br>
</dd> 
<dt> <strong>maigesPack</strong> </dt> 
<dd> Functions to handle cDNA microarray data, including several methods of 
data analysis.<br>
</dd> 
<dt> <strong>makePlatformDesign</strong> </dt> 
<dd> Creates the Platform Design environments (PDenvs) required by <strong>
oligo</strong>. <br>
</dd> 
<dt> <strong>makecdfenv</strong> </dt> 
<dd> Two functions. One reads a Affymetrix chip description file (CDF) and 
creates a hash table environment containing the location/probe set membership 
mapping. The other creates a package that automatically loads that environment.
<br> </dd> 
<dt> <strong>marray</strong> </dt> 
<dd> Exploratory analysis for two-color spotted microarray data. <br>
</dd> 
<dt> <strong>matchprobes</strong> </dt> 
<dd> Tools for sequence matching of probes on arrays. <br>
</dd> 
<dt> <strong>mdqc</strong> </dt> 
<dd> Mahalanobis Distance Quality Control for microarrays. <br>
</dd> 
<dt> <strong>metaArray</strong> </dt> 
<dd> Integration of microarray data for meta-analysis. <br>
</dd> 
<dt> <strong>multtest</strong> </dt> 
<dd> Multiple testing procedures for controlling the family-wise error rate 
(FWER) and the false discovery rate (FDR). Tests can be based on t- or 
F-statistics for one- and two-factor designs, and permutation procedures are 
available to estimate adjusted p-values.<br>
</dd> 
<dt> <strong>nem</strong> </dt> 
<dd> Nested Effects Models to reconstruct phenotypic hierarchies. <br>
</dd> 
<dt> <strong>nnNorm</strong> </dt> 
<dd> Spatial and intensity based normalization of cDNA microarray data based 
on robust neural nets.<br>
</dd> 
<dt> <strong>nudge</strong> </dt> 
<dd> Normal Uniform Differential Gene Expression detection. <br>
</dd> 
<dt> <strong>occugene</strong> </dt> 
<dd> Functions for multinomial occupancy distribution. <br>
</dd> 
<dt> <strong>oligo</strong> </dt> 
<dd> Oligonucleotide arrays. <br>
</dd> 
<dt> <strong>oligoClasses</strong> </dt> 
<dd> Classes for high-throughput SNP arrays. <br>
</dd> 
<dt> <strong>oneChannelGUI</strong> </dt> 
<dd> Extend the capabilities of <strong>affylmGUI</strong>. <br>
</dd> 
<dt> <strong>ontoTools</strong> </dt> 
<dd> Graphs and sparse matrices for working with ontologies; formal objects 
for nomenclatures with provenance management.<br>
</dd> 
<dt> <strong>pamr</strong> </dt> 
<dd>  Pam: Prediction Analysis for Microarrays. <br>
</dd> 
<dt> <strong>panp</strong> </dt> 
<dd> Presence-Absence calls from Negative strand matching Probesets. <br>
</dd>
<dt> <strong>pathRender</strong> </dt> 
<dd> Render molecular pathways. <br>
</dd> 
<dt> <strong>pcaMethods</strong> </dt> 
<dd> A collection of PCA methods. <br>
</dd> 
<dt> <strong>pcot2</strong> </dt> 
<dd> Principal coordinates and Hotelling's T-square method. <br>
</dd> 
<dt> <strong>pdInfoBuilder</strong> </dt> 
<dd> Platform design information package builder. <br>
</dd> 
<dt> <strong>pdmclass</strong> </dt> 
<dd> CLASSification of microarray samples using Penalized Discriminant Methods.
<br> </dd> 
<dt> <strong>pgUtils</strong> </dt> 
<dd> Utility functions for PostgreSQL databases. <br>
</dd> 
<dt> <strong>pickgene</strong> </dt> 
<dd> Adaptive gene picking for microarray expression data analysis. <br>
</dd> 
<dt> <strong>pkgDepTools</strong> </dt> 
<dd> Package dependency tools. <br>
</dd> 
<dt> <strong>plgem</strong> </dt> 
<dd> Power Law Global Error Model. <br>
</dd> 
<dt> <strong>plier</strong> </dt> 
<dd> Implements the Affymetrix PLIER (Probe Logarithmic Error Intensity 
Estimate) algorithm.<br>
</dd> 
<dt> <strong>plw</strong> </dt> 
<dd> Probe level Locally moderated Weighted t-tests. <br>
</dd> 
<dt> <strong>ppiStats</strong> </dt> 
<dd> Protein-Protein Interaction Statistical package. <br>
</dd> 
<dt> <strong>prada</strong> </dt> 
<dd> Tools for analyzing and navigating data from high-throughput phenotyping 
experiments based on cellular assays and fluorescent detection.<br>
</dd> 
<dt> <strong>preprocessCore</strong> </dt> 
<dd> A collection of pre-processing functions. <br>
</dd> 
<dt> <strong>puma</strong> </dt> 
<dd> Propagating Uncertainty in Microarray Analysis. <br>
</dd> 
<dt> <strong>quantsmooth</strong> </dt> 
<dd> Quantile smoothing and genomic visualization of array data. <br>
</dd> 
<dt> <strong>qvalue</strong> </dt> 
<dd> Q-value estimation for false discovery rate control. <br>
</dd> 
<dt> <strong>rHVDM</strong> </dt> 
<dd> Hidden Variable Dynamic Modeling. <br>
</dd> 
<dt> <strong>rama</strong> </dt> 
<dd> Robust Analysis of MicroArrays: robust estimation of cDNA microarray 
intensities with replicates using a Bayesian hierarchical model.<br>
</dd> 
<dt> <strong>rbsurv</strong> </dt> 
<dd> Robust likelihood-based survival modeling with microarray data. <br>
</dd>
<dt> <strong>reb</strong> </dt> 
<dd> Regional Expression Biases. <br>
</dd> 
<dt> <strong>rflowcyt</strong> </dt> 
<dd> Statistical tools and data structures for analytic flow cytometry. <br>

</dd> 
<dt> <strong>rsbml</strong> </dt> 
<dd> R support for SBML, using libsbml. <br>
</dd> 
<dt> <strong>rtracklayer</strong> </dt> 
<dd> R interface to genome browsers and their annotation tracks. <br>
</dd> 
<dt> <strong>safe</strong> </dt> 
<dd> Significance Analysis of Function and Expression. <br>
</dd> 
<dt> <strong>sagenhaft</strong> </dt> 
<dd> Functions for reading and comparing SAGE (Serial Analysis of Gene 
Expression) libraries.<br>
</dd> 
<dt> <strong>seqLogo</strong> </dt> 
<dd> Sequence logos for DNA sequence alignments. <br>
</dd> 
<dt> <strong>sigPathway</strong> </dt> 
<dd> Pathway analysis. <br>
</dd> 
<dt> <strong>siggenes</strong> </dt> 
<dd> Identifying differentially expressed genes and estimating the False 
Discovery Rate (FDR) with both the Significance Analysis of Microarrays (SAM) 
and the Empirical Bayes Analyses of Microarrays (EBAM).<br>
</dd> 
<dt> <strong>simpleaffy</strong> </dt> 
<dd> Very simple high level analysis of Affymetrix data. <br>
</dd> 
<dt> <strong>simulatorAPMS</strong> </dt> 
<dd> Computationally simulates the AP-MS technology. <br>
</dd> 
<dt> <strong>sizepower</strong> </dt> 
<dd> Sample size and power calculation in microrarray studies. <br>
</dd> 
<dt> <strong>snapCGH</strong> </dt> 
<dd> Segmentation, normalization and processing of aCGH data. <br>
</dd> 
<dt> <strong>snpMatrix</strong> </dt> 
<dd> The snp.matrix and X.snp.matrix classes. <br>
</dd> 
<dt> <strong>spikeLI</strong> </dt> 
<dd> Affymetrix Spike-in Langmuir Isotherm data analysis tool. <br>
</dd> 
<dt> <strong>splicegear</strong> </dt> 
<dd> A set of tools to work with alternative splicing. <br>
</dd> 
<dt> <strong>splots</strong> </dt> 
<dd> Visualization routines for high throughput screens. <br>
</dd> 
<dt> <strong>spotSegmentation</strong> </dt> 
<dd> Microarray spot segmentation and gridding for blocks of microarray spots. 
<br> </dd> 
<dt> <strong>sscore</strong> </dt> 
<dd> S-score algorithm for Affymetrix oligonucleotide microarrays. <br>
</dd> 
<dt> <strong>ssize</strong> </dt> 
<dd> Estimate microarry sample size. <br>
</dd> 
<dt> <strong>stam</strong> </dt> 
<dd> STructured Analysis of Microarray data. <br>
</dd> 
<dt> <strong>stepNorm</strong> </dt> 
<dd> Stepwise normalization functions for cDNA microarrays. <br>
</dd> 
<dt> <strong>tilingArray</strong> </dt> 
<dd> Analysis of tiling arrays. <br>
</dd> 
<dt> <strong>timecourse</strong> </dt> 
<dd> Statistical analysis for developmental microarray time course data. <br>

</dd> 
<dt> <strong>tkWidgets</strong> </dt> 
<dd> Widgets in Tcl/Tk that provide functionality for Bioconductor packages. 
<br> </dd> 
<dt> <strong>topGO</strong> </dt> 
<dd> Enrichment analysis for Gene Ontology. <br>
</dd> 
<dt> <strong>twilight</strong> </dt> 
<dd> Estimation of local false discovery rate. <br>
</dd> 
<dt> <strong>vbmp</strong> </dt> 
<dd> Variational Bayesian Multinomial Probit regression. <br>
</dd> 
<dt> <strong>vsn</strong> </dt> 
<dd> Calibration and variance stabilizing transformations for both Affymetrix 
and cDNA array data.<br>
</dd> 
<dt> <strong>weaver</strong> </dt> 
<dd> Tools and extensions for processing Sweave documents. <br>
</dd> 
<dt> <strong>webbioc</strong> </dt> 
<dd> Integrated web interface for doing microarray analysis using several of 
the Bioconductor packages.<br>
</dd> 
<dt> <strong>widgetInvoke</strong> </dt> 
<dd> Evaluation widgets for functions. <br>
</dd> 
<dt> <strong>widgetTools</strong> </dt> 
<dd> Tools for creating Tcl/Tk widgets, i.e., small-scale graphical user 
interfaces.<br>
</dd> 
<dt> <strong>xcms</strong> </dt> 
<dd> LC/MS and GC/MS data analysis: framework for processing and visualization 
of chromatographically separated mass spectral data.<br>
</dd> 
<dt> <strong>xps</strong> </dt> 
<dd> Methods for processing and analysis of Affymetrix Oligonucleotide Arrays. 
<br> </dd> 
<dt> <strong>yaqcaffy</strong> </dt> 
<dd> Affymetrix expression data quality control and reproducibility analysis. 
</dd> </dl> 
<p></p>  Previous:&nbsp; Add-on packages from Bioconductor, Up:&nbsp; Which 
add-on packages exist for R? <br>

<h4>5.1.5 Other add-on packages</h4> 
<p>Jim Lindsey has written a collection of R packages for nonlinear regression 
and repeated measurements, consisting of<strong>event</strong> (event history 
procedures and models),<strong>gnlm</strong> (generalized nonlinear regression 
models),<strong>growth</strong> (multivariate normal and elliptically-contoured 
repeated measurements models),<strong>repeated</strong> (non-normal repeated 
measurements models),<strong>rmutil</strong> (utilities for nonlinear 
regression and repeated measurements), and<strong>stable</strong> (probability 
functions and generalized regression models for stable distributions). All 
analyses in the new edition of his book &ldquo;Models for Repeated 
Measurements&rdquo; (1999, Oxford University Press) were carried out using 
these packages. Jim has also started<strong>dna</strong>, a package with 
procedures for the analysis of DNA sequences. Jim's packages can be obtained 
fromhttp://popgen.unimaas.nl/~jlindsey/rcode.html. </p> 
<p>More code has been posted to the R-help mailing list, and can be obtained 
from the mailing list archive.</p> 
<p></p>  Next:&nbsp; How can add-on packages be used?, Previous:&nbsp; Which 
add-on packages exist for R?, Up:&nbsp; R Add-On Packages <br>

<h3>5.2 How can add-on packages be installed?</h3> 
<p>(Unix only.) The add-on packages on CRAN come as gzipped tar files named pkg
<code>_</code>version<code>.tar.gz</code>, which may in fact be 
&ldquo;bundles&rdquo; containing more than one package. Provided thattar and 
gzip are available on your system, type </p> 
<pre> $ R CMD INSTALL /path/to/pkg_version.tar.gz </pre> 
<p>at the shell prompt to install to the library tree rooted at the first 
directory in your library search path (see the help page for<code>.libPaths()
</code> for details on how the search path is determined). </p> 
<p>To install to another tree (e.g., your private one), use </p> 
<pre> $ R CMD INSTALL -l lib /path/to/pkg_version.tar.gz </pre> 
<p>where lib gives the path to the library tree to install to. </p> 
<p>Even more conveniently, you can install and automatically update packages 
from within R if you have access to repositories such asCRAN. See the help page 
for<code>available.packages()</code> for more information. </p> 
<p></p>  Next:&nbsp; How can add-on packages be removed?, Previous:&nbsp; How 
can add-on packages be installed?, Up:&nbsp; R Add-On Packages <br>

<h3>5.3 How can add-on packages be used?</h3> 
<p>To find out which additional packages are available on your system, type 
</p> 
<pre> library() </pre> 
<p>at the R prompt. </p> 
<p>This produces something like </p> 
<blockquote> 
<p></p> 
<pre> Packages in `/home/me/lib/R': mystuff My own R functions, nicely 
packaged but not documented Packages in `/usr/local/lib/R/library': KernSmooth 
Functions for kernel smoothing for Wand &amp; Jones (1995) MASS Main Package of 
Venables and Ripley's MASS base The R Base package boot Bootstrap R (S-Plus) 
Functions (Canty) class Functions for Classification cluster Functions for 
clustering (by Rousseeuw et al.) codetools Code Analysis Tools for R datasets 
The R Datasets Package foreign Read Data Stored by Minitab, S, SAS, SPSS, 
Stata, Systat, dBase, ... grDevices The R Graphics Devices and Support for 
Colours and Fonts graphics The R Graphics Package grid The Grid Graphics 
Package lattice Lattice Graphics methods Formal Methods and Classes mgcv GAMs 
with GCV smoothness estimation and GAMMs by REML/PQ nlme Linear and Nonlinear 
Mixed Effects Models nnet Feed-forward Neural Networks and Multinomial 
Log-Linear Models rpart Recursive Partitioning spatial Functions for Kriging 
and Point Pattern Analysis splines Regression Spline Functions and Classes 
stats The R Stats Package stats4 Statistical functions using S4 Classes 
survival Survival analysis, including penalised likelihood tcltk Tcl/Tk 
Interface tools Tools for Package Development utils The R Utils Package</pre> 
<br> <br>
</blockquote> 
<p>You can &ldquo;load&rdquo; the installed package pkg by </p> 
<pre> library(pkg) </pre> 
<p>You can then find out which functions it provides by typing one of </p> 
<pre> library(help = pkg) help(package = pkg) </pre> 
<p>You can unload the loaded package pkg by </p> 
<pre> detach(&quot;package:pkg&quot;) </pre> 
<p></p>  Next:&nbsp; How can I create an R package?, Previous:&nbsp; How can 
add-on packages be used?, Up:&nbsp; R Add-On Packages <br>

<h3>5.4 How can add-on packages be removed?</h3> 
<p>Use </p> 
<pre> $ R CMD REMOVE pkg_1 ... pkg_n </pre> 
<p>to remove the packages pkg_1, ..., pkg_n from the library tree rooted at 
the first directory given inR_LIBS if this is set and non-null, and from the 
default library otherwise. (Versions of R prior to 1.3.0 removed from the 
default library by default.)</p> 
<p>To remove from library lib, do </p> 
<pre> $ R CMD REMOVE -l lib pkg_1 ... pkg_n </pre> 
<p></p>  Next:&nbsp; How can I contribute to R?, Previous:&nbsp; How can 
add-on packages be removed?, Up:&nbsp; R Add-On Packages <br>

<h3>5.5 How can I create an R package?</h3> 
<p>A package consists of a subdirectory containing the files DESCRIPTION and 
INDEX, and the subdirectories R, data, demo, exec, inst, man, src, and tests 
(some of which can be missing). Optionally the package can also contain script 
filesconfigure and cleanup which are executed before and after installation. 
</p> 
<p>See Creating R packages, for details. </p> 
<p>R version 1.3.0 has added the function <code>package.skeleton()</code> 
which will set up directories, save data and code, and create skeleton help 
files for a set of R functions and datasets.</p> 
<p>See What is CRAN?, for information on uploading a package to CRAN. </p> 
<p></p>  Previous:&nbsp; How can I create an R package?, Up:&nbsp; R Add-On 
Packages <br>

<h3>5.6 How can I contribute to R?</h3> 
<p>R is in active development and there is always a risk of bugs creeping in. 
Also, the developers do not have access to all possible machines capable of 
running R. So, simply using it and communicating problems is certainly of great 
value.</p> 
<p>One place where functionality is still missing is the modeling software as 
described in &ldquo;Statistical Models in S&rdquo; (seeWhat is S?); some of the 
nonlinear modeling code is not there yet.</p> 
<p>The R Developer Page acts as an intermediate repository for more or less 
finalized ideas and plans for the R statistical system. It contains (pointers 
to) TODO lists, RFCs, various other writeups, ideas lists, and CVS miscellanea.
</p> 
<p>Many (more) of the packages available at the Statlib S Repository might be 
worth porting to R.</p> 
<p>If you are interested in working on any of these projects, please notify 
Kurt Hornik. </p> 
<p></p>  Next:&nbsp; R Miscellanea, Previous:&nbsp; R Add-On Packages, 
Up:&nbsp;Top <br>

<h2>6 R and Emacs</h2> 
<ul> 
<li>Is there Emacs support for R? </li> 
<li>Should I run R from within Emacs? </li> 
<li>Debugging R from within Emacs </li> </ul> 
<p></p>  Next:&nbsp; Should I run R from within Emacs?, Previous:&nbsp; R and 
Emacs, Up:&nbsp; R and Emacs <br>

<h3>6.1 Is there Emacs support for R?</h3> 
<p>There is an Emacs package called ESS (&ldquo;Emacs Speaks 
Statistics&rdquo;) which provides a standard interface between statistical 
programs and statistical processes. It is intended to provide assistance for 
interactive statistical programming and data analysis. Languages supported 
include: S dialects (R, S 3/4, andS-Plus 3.x/4.x/5.x/6.x/7.x), LispStat 
dialects (XLispStat, ViSta), SAS, Stata, and BUGS.</p> 
<p>ESS grew out of the need for bug fixes and extensions to S-mode 4.8 (which 
was aGNU Emacs interface to S/S-Plus version 3 only). The current set of 
developers desired support for XEmacs, R, S4, and MS Windows. In addition, with 
new modes being developed for R, Stata, and SAS, it was felt that a unifying 
interface and framework for the user interface would benefit both the user and 
the developer, by helping both groups conform to standard Emacs usage. The end 
result is an increase in efficiency for statistical programming and data 
analysis, over the usual tools.</p> 
<p>R support contains code for editing R source code (syntactic indentation 
and highlighting of source code, partial evaluations of code, loading and 
error-checking of code, and source code revision maintenance) and documentation 
(syntactic indentation and highlighting of source code, sending examples to 
runningESS process, and previewing), interacting with an inferior R process 
from within Emacs (command-line editing, searchable command history, 
command-line completion of R object and file names, quick access to object and 
search lists, transcript recording, and an interface to the help system), and 
transcript manipulation (recording and saving transcript files, manipulating 
and editing saved transcripts, and re-evaluating commands from transcript 
files).</p> 
<p>The latest stable version of ESS are available via CRAN or the ESS web page
. TheHTML version of the documentation can be found at http://stat.ethz.ch/ESS/.
</p> 
<p>ESS comes with detailed installation instructions. </p> 
<p>For help with ESS, send email to ESS-help@stat.math.ethz.ch. </p> 
<p>Please send bug reports and suggestions on ESS to ESS-bugs@stat.math.ethz.ch
. The easiest way to do this from is within Emacs by typingM-x 
ess-submit-bug-report or using the [ESS] or [iESS] pulldown menus. </p> 
<p></p>  Next:&nbsp; Debugging R from within Emacs, Previous:&nbsp; Is there 
Emacs support for R?, Up:&nbsp; R and Emacs <br>

<h3>6.2 Should I run R from within Emacs?</h3> 
<p>Yes, <em>definitely</em>. Inferior R mode provides a readline/history 
mechanism, object name completion, and syntax-based highlighting of the 
interaction buffer using Font Lock mode, as well as a very convenient interface 
to the R help system.</p> 
<p>Of course, it also integrates nicely with the mechanisms for editing R 
source using Emacs. One can write code in one Emacs buffer and send whole or 
parts of it for execution to R; this is helpful for both data analysis and 
programming. One can also seamlessly integrate with a revision control system, 
in order to maintain a log of changes in your programs and data, as well as to 
allow for the retrieval of past versions of the code.</p> 
<p>In addition, it allows you to keep a record of your session, which can also 
be used for error recovery through the use of the transcript mode.</p> 
<p>To specify command line arguments for the inferior R process, use C-u M-x R 
for starting R. </p> 
<p></p>  Previous:&nbsp; Should I run R from within Emacs?, Up:&nbsp; R and 
Emacs <br>

<h3>6.3 Debugging R from within Emacs</h3> 
<p>To debug R &ldquo;from within Emacs&rdquo;, there are several 
possibilities. To use the Emacs GUD (Grand Unified Debugger) library with the 
recommended debugger GDB, typeM-x gdb and give the path to the R <em>binary</em>
 as argument. At thegdb prompt, set R_HOME and other environment variables as 
needed (using e.g.set env R_HOME /path/to/R/, but see also below), and start 
the binary with the desired arguments (e.g.,run --quiet). </p> 
<p>If you have ESS, you can do C-u M-x R &lt;RET&gt; - d &lt;SPC&gt; g d b 
&lt;RET&gt; to start an inferior R process with arguments -d gdb. </p> 
<p>A third option is to start an inferior R process via ESS (M-x R) and then 
start GUD (M-x gdb) giving the R binary (using its full path name) as the 
program to debug. Use the programps to find the process number of the currently 
running R process then use the<code>attach</code> command in gdb to attach it 
to that process. One advantage of this method is that you have separate<code>*R*
</code> and <code>*gud-gdb*</code> windows. Within the <code>*R*</code> window 
you have all theESS facilities, such as object-name completion, that we know 
and love.</p> 
<p>When using GUD mode for debugging from within Emacs, you may find it most 
convenient to use the directory with your code in it as the current working 
directory and then make a symbolic link from that directory to the R binary. 
That way.gdbinit can stay in the directory with the code and be used to set up 
the environment and the search paths for the source, e.g. as follows:</p> 
<pre> set env R_HOME /opt/R set env R_PAPERSIZE letter set env R_PRINTCMD lpr 
dir /opt/R/src/appl dir /opt/R/src/main dir /opt/R/src/nmath dir /opt/R/src/unix
</pre> 
<p></p>  Next:&nbsp; R Programming, Previous:&nbsp; R and Emacs, Up:&nbsp; Top 
<br> 
<h2>7 R Miscellanea</h2> 
<ul> 
<li>How can I set components of a list to NULL? </li> 
<li>How can I save my workspace? </li> 
<li>How can I clean up my workspace? </li> 
<li>How can I get eval() and D() to work? </li> 
<li>Why do my matrices lose dimensions? </li> 
<li>How does autoloading work? </li> 
<li>How should I set options? </li> 
<li>How do file names work in Windows? </li> 
<li>Why does plotting give a color allocation error? </li> 
<li>How do I convert factors to numeric? </li> 
<li>Are Trellis displays implemented in R? </li> 
<li>What are the enclosing and parent environments? </li> 
<li>How can I substitute into a plot label? </li> 
<li>What are valid names? </li> 
<li>Are GAMs implemented in R? </li> 
<li>Why is the output not printed when I source() a file? </li> 
<li>Why does outer() behave strangely with my function? </li> 
<li>Why does the output from anova() depend on the order of factors in the 
model? </li> 
<li>How do I produce PNG graphics in batch mode? </li> 
<li>How can I get command line editing to work? </li> 
<li>How can I turn a string into a variable? </li> 
<li>Why do lattice/trellis graphics not work? </li> 
<li>How can I sort the rows of a data frame? </li> 
<li>Why does the help.start() search engine not work? </li> 
<li>Why did my .Rprofile stop working when I updated R? </li> 
<li>Where have all the methods gone? </li> 
<li>How can I create rotated axis labels? </li> 
<li>Why is read.table() so inefficient? </li> 
<li>What is the difference between package and library? </li> 
<li>I installed a package but the functions are not there </li> 
<li>Why doesn't R think these numbers are equal? </li> 
<li>How can I capture or ignore errors in a long simulation? </li> 
<li>Why are powers of negative numbers wrong? </li> 
<li>How can I save the result of each iteration in a loop into a separate file?
</li> 
<li>Why are p-values not displayed when using lmer()? </li> 
<li>Why are there unwanted lines between polygons in PDF output viewed in 
Adobe Reader? </li> 
<li>Why does backslash behave strangely inside strings? </li> </ul> 
<p></p>  Next:&nbsp; How can I save my workspace?, Previous:&nbsp; R 
Miscellanea, Up:&nbsp; R Miscellanea <br>

<h3>7.1 How can I set components of a list to NULL?</h3> 
<p>You can use </p> 
<pre> x[i] &lt;- list(NULL) </pre> 
<p>to set component <code>i</code> of the list <code>x</code> to <code>NULL
</code>, similarly for named components. Do not set <code>x[i]</code> or <code>
x[[i]]</code> to <code>NULL</code>, because this will remove the corresponding 
component from the list.</p> 
<p>For dropping the row names of a matrix <code>x</code>, it may be easier to 
use<code>rownames(x) &lt;- NULL</code>, similarly for column names. </p> 
<p></p>  Next:&nbsp; How can I clean up my workspace?, Previous:&nbsp; How can 
I set components of a list to NULL?, Up:&nbsp; R Miscellanea <br>

<h3>7.2 How can I save my workspace?</h3> 
<p><code>save.image()</code> saves the objects in the user's <code>.GlobalEnv
</code> to the file .RData in the R startup directory. (This is also what 
happens afterq(&quot;yes&quot;).) Using <code>save.image(</code>file<code>)
</code> one can save the image under a different name. </p> 
<p></p>  Next:&nbsp; How can I get eval() and D() to work?, Previous:&nbsp; 
How can I save my workspace?, Up:&nbsp; R Miscellanea <br>

<h3>7.3 How can I clean up my workspace?</h3> 
<p>To remove all objects in the currently active environment (typically <code>
.GlobalEnv</code>), you can do </p> 
<pre> rm(list = ls(all = TRUE)) </pre> 
<p>(Without all = TRUE, only the objects with names not starting with a `.' 
are removed.)</p> 
<p></p>  Next:&nbsp; Why do my matrices lose dimensions?, Previous:&nbsp; How 
can I clean up my workspace?, Up:&nbsp; R Miscellanea <br>

<h3>7.4 How can I get eval() and D() to work?</h3> 
<p>Strange things will happen if you use <code>eval(print(x), envir = e)</code>
 or<code>D(x^2, &quot;x&quot;)</code>. The first one will either tell you that 
&quot;<code>x</code>&quot; is not found, or print the value of the wrong <code>x
</code>. The other one will likely return zero if <code>x</code> exists, and an 
error otherwise.</p> 
<p>This is because in both cases, the first argument is evaluated in the 
calling environment first. The result (which should be an object of mode<code>
&quot;expression&quot;</code> or <code>&quot;call&quot;</code>) is then 
evaluated or differentiated. What you (most likely) really want is obtained by 
&ldquo;quoting&rdquo; the first argument upon surrounding it with<code>
expression()</code>. For example, </p> 
<pre> R&gt; D(expression(x^2), &quot;x&quot;) 2 * x </pre> 
<p>Although this behavior may initially seem to be rather strange, is 
perfectly logical. The &ldquo;intuitive&rdquo; behavior could easily be 
implemented, but problems would arise whenever the expression is contained in a 
variable, passed as a parameter, or is the result of a function call. Consider 
for instance the semantics in cases like</p> 
<pre> D2 &lt;- function(e, n) D(D(e, n), n) </pre> 
<p>or </p> 
<pre> g &lt;- function(y) eval(substitute(y), sys.frame(sys.parent(n = 2))) 
g(a * b)</pre> 
<p>See the help page for <code>deriv()</code> for more examples. </p> 
<p></p>  Next:&nbsp; How does autoloading work?, Previous:&nbsp; How can I get 
eval() and D() to work?, Up:&nbsp; R Miscellanea <br>

<h3>7.5 Why do my matrices lose dimensions?</h3> 
<p>When a matrix with a single row or column is created by a subscripting 
operation, e.g.,<code>row &lt;- mat[2, ]</code>, it is by default turned into a 
vector. In a similar way if an array with dimension, say, 
2&nbsp;x&nbsp;3&nbsp;x&nbsp;1&nbsp;x&nbsp;4 is created by subscripting it will 
be coerced into a 2&nbsp;x&nbsp;3&nbsp;x&nbsp;4 array, losing the unnecessary 
dimension. After much discussion this has been determined to be a<em>feature
</em>. </p> 
<p>To prevent this happening, add the option drop = FALSE to the subscripting. 
For example,</p> 
<pre> rowmatrix &lt;- mat[2, , drop = FALSE] # creates a row matrix colmatrix 
&lt;- mat[, 2, drop = FALSE] #creates a column matrix a &lt;- b[1, 1, 1, drop = 
FALSE] #creates a 1 x 1 x 1 array </pre> 
<p>The drop = FALSE option should be used defensively when programming. For 
example, the statement</p> 
<pre> somerows &lt;- mat[index, ] </pre> 
<p>will return a vector rather than a matrix if <code>index</code> happens to 
have length 1, causing errors later in the code. It should probably be 
rewritten as</p> 
<pre> somerows &lt;- mat[index, , drop = FALSE] </pre> 
<p></p>  Next:&nbsp; How should I set options?, Previous:&nbsp; Why do my 
matrices lose dimensions?, Up:&nbsp; R Miscellanea <br>

<h3>7.6 How does autoloading work?</h3> 
<p>R has a special environment called <code>.AutoloadEnv</code>. Using 
autoload(name, pkg), where name and pkg are strings giving the names of an 
object and the package containing it, stores some information in this 
environment. When R tries to evaluatename, it loads the corresponding package 
pkg and reevaluates name in the new package's environment. </p> 
<p>Using this mechanism makes R behave as if the package was loaded, but does 
not occupy memory (yet).</p> 
<p>See the help page for <code>autoload()</code> for a very nice example. </p> 
<p></p>  Next:&nbsp; How do file names work in Windows?, Previous:&nbsp; How 
does autoloading work?, Up:&nbsp; R Miscellanea <br>

<h3>7.7 How should I set options?</h3> 
<p>The function <code>options()</code> allows setting and examining a variety 
of global &ldquo;options&rdquo; which affect the way in which R computes and 
displays its results. The variable<code>.Options</code> holds the current 
values of these options, but should never directly be assigned to unless you 
want to drive yourself crazy&mdash;simply pretend that it is a 
&ldquo;read-only&rdquo; variable.</p> 
<p>For example, given </p> 
<pre> test1 &lt;- function(x = pi, dig = 3) { oo &lt;- options(digits = dig); 
on.exit(options(oo)); cat(.Options$digits, x, &quot;\n&quot;) } test2 &lt;- 
function(x = pi, dig = 3) { .Options$digits &lt;- dig cat(.Options$digits, x, 
&quot;\n&quot;) }</pre> 
<p>we obtain: </p> 
<pre> R&gt; test1() 3 3.14 R&gt; test2() 3 3.141593 </pre> 
<p>What is really used is the <em>global</em> value of <code>.Options</code>, 
and usingoptions(OPT = VAL) correctly updates it. Local copies of <code>.Options
</code>, either in <code>.GlobalEnv</code> or in a function environment 
(frame), are just silently disregarded.</p> 
<p></p>  Next:&nbsp; Why does plotting give a color allocation error?, 
Previous:&nbsp;How should I set options?, Up:&nbsp; R Miscellanea <br>

<h3>7.8 How do file names work in Windows?</h3> 
<p>As R uses C-style string handling, `\' is treated as an escape character, 
so that for example one can enter a newline as `\n'. When you really need a `\
', you have to escape it with another `\'. </p> 
<p>Thus, in filenames use something like <code>&quot;c:\\data\\money.dat&quot;
</code>. You can also replace `\' by `/' (<code>&quot;c:/data/money.dat&quot;
</code>). </p> 
<p></p>  Next:&nbsp; How do I convert factors to numeric?, Previous:&nbsp; How 
do file names work in Windows?, Up:&nbsp; R Miscellanea <br>

<h3>7.9 Why does plotting give a color allocation error?</h3> 
<p>On an X11 device, plotting sometimes, e.g., when running <code>
demo(&quot;image&quot;)</code>, results in &ldquo;Error: color allocation 
error&rdquo;. This is an X problem, and only indirectly related to R. It occurs 
when applications started prior to R have used all the available colors. (How 
many colors are available depends on the X configuration; sometimes only 256 
colors can be used.)</p> 
<p>One application which is notorious for &ldquo;eating&rdquo; colors is 
Netscape. If the problem occurs when Netscape is running, try (re)starting it 
with either the-no-install (to use the default colormap) or the -install (to 
install a private colormap) option.</p> 
<p>You could also set the <code>colortype</code> of <code>X11()</code> to 
<code>&quot;pseudo.cube&quot;</code> rather than the default <code>
&quot;pseudo&quot;</code>. See the help page for <code>X11()</code> for more 
information. </p> 
<p></p>  Next:&nbsp; Are Trellis displays implemented in R?, Previous:&nbsp; 
Why does plotting give a color allocation error?, Up:&nbsp; R Miscellanea <br>

<h3>7.10 How do I convert factors to numeric?</h3> 
<p>It may happen that when reading numeric data into R (usually, when reading 
in a file), they come in as factors. If<code>f</code> is such a factor object, 
you can use</p> 
<pre> as.numeric(as.character(f)) </pre> 
<p>to get the numbers back. More efficient, but harder to remember, is </p> 
<pre> as.numeric(levels(f))[as.integer(f)] </pre> 
<p>In any case, do not call <code>as.numeric()</code> or their likes directly 
for the task at hand (as<code>as.numeric()</code> or <code>unclass()</code> 
give the internal codes).</p> 
<p></p>  Next:&nbsp; What are the enclosing and parent environments?, 
Previous:&nbsp;How do I convert factors to numeric?, Up:&nbsp; R Miscellanea 
<br> 
<h3>7.11 Are Trellis displays implemented in R?</h3> 
<p>The recommended package <strong>lattice</strong> (which is based on another 
recommended package,<strong>grid</strong>) provides graphical functionality 
that is compatible with most Trellis commands.</p> 
<p>You could also look at <code>coplot()</code> and <code>dotchart()</code> 
which might do at least some of what you want. Note also that the R version of
<code>pairs()</code> is fairly general and provides most of the functionality of
<code>splom()</code>, and that R's default plot method has an argument <code>asp
</code> allowing to specify (and fix against device resizing) the aspect ratio 
of the plot.</p> 
<p>(Because the word &ldquo;Trellis&rdquo; has been claimed as a trademark we 
do not use it in R. The name &ldquo;lattice&rdquo; has been chosen for the R 
equivalent.)</p> 
<p></p>  Next:&nbsp; How can I substitute into a plot label?, Previous:&nbsp; 
Are Trellis displays implemented in R?, Up:&nbsp; R Miscellanea <br>

<h3>7.12 What are the enclosing and parent environments?</h3> 
<p>Inside a function you may want to access variables in two additional 
environments: the one that the function was defined in 
(&ldquo;enclosing&rdquo;), and the one it was invoked in (&ldquo;parent&rdquo;).
</p> 
<p>If you create a function at the command line or load it in a package its 
enclosing environment is the global workspace. If you define a function<code>f()
</code> inside another function <code>g()</code> its enclosing environment is 
the environment inside<code>g()</code>. The enclosing environment for a 
function is fixed when the function is created. You can find out the enclosing 
environment for a function<code>f()</code> using <code>environment(f)</code>. 
</p> 
<p>The &ldquo;parent&rdquo; environment, on the other hand, is defined when 
you invoke a function. If you invoke<code>lm()</code> at the command line its 
parent environment is the global workspace, if you invoke it inside a function
<code>f()</code> then its parent environment is the environment inside <code>f()
</code>. You can find out the parent environment for an invocation of a 
function by using<code>parent.frame()</code> or <code>sys.frame(sys.parent())
</code>. </p> 
<p>So for most user-visible functions the enclosing environment will be the 
global workspace, since that is where most functions are defined. The parent 
environment will be wherever the function happens to be called from. If a 
function<code>f()</code> is defined inside another function <code>g()</code> it 
will probably be used inside<code>g()</code> as well, so its parent environment 
and enclosing environment will probably be the same.</p> 
<p>Parent environments are important because things like model formulas need 
to be evaluated in the environment the function was called from, since that's 
where all the variables will be available. This relies on the parent 
environment being potentially different with each invocation.</p> 
<p>Enclosing environments are important because a function can use variables 
in the enclosing environment to share information with other functions or with 
other invocations of itself (see the section on lexical scoping). This relies 
on the enclosing environment being the same each time the function is invoked. 
(In C this would be done with static variables.)</p> 
<p>Scoping <em>is</em> hard. Looking at examples helps. It is particularly 
instructive to look at examples that work differently in R and S and try to see 
why they differ. One way to describe the scoping differences between R and S is 
to say that in S the enclosing environment is<em>always</em> the global 
workspace, but in R the enclosing environment is wherever the function was 
created.</p> 
<p></p>  Next:&nbsp; What are valid names?, Previous:&nbsp; What are the 
enclosing and parent environments?, Up:&nbsp; R Miscellanea <br>

<h3>7.13 How can I substitute into a plot label?</h3> 
<p>Often, it is desired to use the value of an R object in a plot label, e.g., 
a title. This is easily accomplished using<code>paste()</code> if the label is 
a simple character string, but not always obvious in case the label is an 
expression (for refined mathematical annotation). In such a case, either use
<code>parse()</code> on your pasted character string or use <code>substitute()
</code> on an expression. For example, if <code>ahat</code> is an estimator of 
your parameter a of interest, use</p> 
<pre> title(substitute(hat(a) == ahat, list(ahat = ahat))) </pre> 
<p>(note that it is `==' and not `='). Sometimes <code>bquote()</code> gives a 
more compact form, e.g.,</p> 
<pre> title(bquote(hat(a) = .(ahat))) </pre> 
<p>where subexpressions enclosed in `.()' are replaced by their values. </p> 
<p>There are more worked examples in the mailing list achives. </p> 
<p></p>  Next:&nbsp; Are GAMs implemented in R?, Previous:&nbsp; How can I 
substitute into a plot label?, Up:&nbsp; R Miscellanea <br>

<h3>7.14 What are valid names?</h3> 
<p>When creating data frames using <code>data.frame()</code> or <code>
read.table()</code>, R by default ensures that the variable names are 
syntactically valid. (The argumentcheck.names to these functions controls 
whether variable names are checked and adjusted by<code>make.names()</code> if 
needed.)</p> 
<p>To understand what names are &ldquo;valid&rdquo;, one needs to take into 
account that the term &ldquo;name&rdquo; is used in several different (but 
related) ways in the language:</p> 
<ol> 
<li>A <em>syntactic name</em> is a string the parser interprets as this type 
of expression. It consists of letters, numbers, and the dot and (for version of 
R at least 1.9.0) underscore characters, and starts with either a letter or a 
dot not followed by a number. Reserved words are not syntactic names.</li> 
<li>An <em>object name</em> is a string associated with an object that is 
assigned in an expression either by having the object name on the left of an 
assignment operation or as an argument to the<code>assign()</code> function. It 
is usually a syntactic name as well, but can be any non-empty string if it is 
quoted (and it is always quoted in the call to<code>assign()</code>). </li> 
<li>An <em>argument name</em> is what appears to the left of the equals sign 
when supplying an argument in a function call (for example,<code>f(trim=.5)
</code>). Argument names are also usually syntactic names, but again can be 
anything if they are quoted.</li> 
<li>An <em>element name</em> is a string that identifies a piece of an object 
(a component of a list, for example.) When it is used on the right of the `$' 
operator, it must be a syntactic name, or quoted. Otherwise, element names can 
be any strings. (When an object is used as a database, as in a call to<code>
eval()</code> or <code>attach()</code>, the element names become object names.) 
</li> 
<li>Finally, a <em>file name</em> is a string identifying a file in the 
operating system for reading, writing, etc. It really has nothing much to do 
with names in the language, but it is traditional to call these strings file 
&ldquo;names&rdquo;.</li> </ol> 
<p></p>  Next:&nbsp; Why is the output not printed when I source() a file?, 
Previous:&nbsp;What are valid names?, Up:&nbsp; R Miscellanea <br>

<h3>7.15 Are GAMs implemented in R?</h3> 
<p>Package <strong>gam</strong> from CRAN implements all the Generalized 
Additive Models (GAM) functionality as described in the GAM chapter of the 
White Book. In particular, it implements backfitting with both local regression 
and smoothing splines, and is extendable. There is a<code>gam()</code> function 
for GAMs in package<strong>mgcv</strong>, but it is not an exact clone of what 
is described in the White Book (no<code>lo()</code> for example). Package 
<strong>gss</strong> can fit spline-based GAMs too. And if you can accept 
regression splines you can use<code>glm()</code>. For gaussian GAMs you can use 
<code>bruto()</code> from package <strong>mda</strong>. </p> 
<p></p>  Next:&nbsp; Why does outer() behave strangely with my function?, 
Previous:&nbsp;Are GAMs implemented in R?, Up:&nbsp; R Miscellanea <br>

<h3>7.16 Why is the output not printed when I source() a file?</h3> 
<p>Most R commands do not generate any output. The command </p> 
<pre> 1+1 </pre> 
<p>computes the value 2 and returns it; the command </p> 
<pre> summary(glm(y~x+z, family=binomial)) </pre> 
<p>fits a logistic regression model, computes some summary information and 
returns an object of class<code>&quot;summary.glm&quot;</code> (see How should 
I write summary methods?). </p> 
<p>If you type `1+1' or `summary(glm(y~x+z, family=binomial))' at the command 
line the returned value is automatically printed (unless it is<code>invisible()
</code>), but in other circumstances, such as in a <code>source()</code>d file 
or inside a function it isn't printed unless you specifically print it.</p> 
<p>To print the value use </p> 
<pre> print(1+1) </pre> 
<p>or </p> 
<pre> print(summary(glm(y~x+z, family=binomial))) </pre> 
<p>instead, or use <code>source(</code>file<code>, echo=TRUE)</code>. </p> 
<p></p>  Next:&nbsp; Why does the output from anova() depend on the order of 
factors in the model?, Previous:&nbsp; Why is the output not printed when I 
source() a file?, Up:&nbsp; R Miscellanea <br>

<h3>7.17 Why does outer() behave strangely with my function?</h3> 
<p>As the help for <code>outer()</code> indicates, it does not work on 
arbitrary functions the way the<code>apply()</code> family does. It requires 
functions that are vectorized to work elementwise on arrays. As you can see by 
looking at the code,<code>outer(x, y, FUN)</code> creates two large vectors 
containing every possible combination of elements of<code>x</code> and <code>y
</code> and then passes this to <code>FUN</code> all at once. Your function 
probably cannot handle two large vectors as parameters.</p> 
<p>If you have a function that cannot handle two vectors but can handle two 
scalars, then you can still use<code>outer()</code> but you will need to wrap 
your function up first, to simulate vectorized behavior. Suppose your function 
is</p> 
<pre> foo &lt;- function(x, y, happy) { stopifnot(length(x) == 1, length(y) == 
1) # scalars only! (x + y) * happy }</pre> 
<p>If you define the general function </p> 
<pre> wrapper &lt;- function(x, y, my.fun, ...) { sapply(seq(along = x), FUN = 
function(i) my.fun(x[i], y[i], ...)) }</pre> 
<p>then you can use <code>outer()</code> by writing, e.g., </p> 
<pre> outer(1:4, 1:2, FUN = wrapper, my.fun = foo, happy = 10) </pre> 
<p></p>  Next:&nbsp; How do I produce PNG graphics in batch mode?, 
Previous:&nbsp;Why does outer() behave strangely with my function?, Up:&nbsp; R 
Miscellanea <br>

<h3>7.18 Why does the output from anova() depend on the order of factors in 
the model?</h3> 
<p>In a model such as <code>~A+B+A:B</code>, R will report the difference in 
sums of squares between the models<code>~1</code>, <code>~A</code>, <code>~A+B
</code> and <code>~A+B+A:B</code>. If the model were <code>~B+A+A:B</code>, R 
would report differences between<code>~1</code>, <code>~B</code>, <code>~A+B
</code>, and <code>~A+B+A:B</code> . In the first case the sum of squares for 
<code>A</code> is comparing <code>~1</code> and <code>~A</code>, in the second 
case it is comparing<code>~B</code> and <code>~B+A</code>. In a non-orthogonal 
design (i.e., most unbalanced designs) these comparisons are (conceptually and 
numerically) different.</p> 
<p>Some packages report instead the sums of squares based on comparing the 
full model to the models with each factor removed one at a time (the famous 
`Type III sums of squares' from SAS, for example). These do not depend on the 
order of factors in the model. The question of which set of sums of squares is 
the Right Thing provokes low-level holy wars on R-help from time to time.</p> 
<p>There is no need to be agitated about the particular sums of squares that R 
reports. You can compute your favorite sums of squares quite easily. Any two 
models can be compared with<code>anova(</code>model1<code>, </code>model2<code>)
</code>, and <code>drop1(</code>model1<code>)</code> will show the sums of 
squares resulting from dropping single terms.</p> 
<p></p>  Next:&nbsp; How can I get command line editing to work?, 
Previous:&nbsp;Why does the output from anova() depend on the order of factors 
in the model?, Up:&nbsp; R Miscellanea <br>

<h3>7.19 How do I produce PNG graphics in batch mode?</h3> 
<p>Under a Unix-alike, if your installation supports the <code>
type=&quot;cairo&quot;</code> option to the <code>png()</code> device there 
should be no problems, and the default settings should just work. This option 
is not available for versions of R prior to 2.7.0, or without support for 
cairo. From R 2.7.0<code>png()</code> by default uses the Quartz device on Mac 
OS X, and that too works in batch mode.</p> 
<p>Earlier versions of the <code>png()</code> device uses the X11 driver, 
which is a problem in batch mode or for remote operation. If you have 
Ghostscript you can use<code>bitmap()</code>, which produces a PostScript or 
PDF file then converts it to any bitmap format supported by Ghostscript. On 
some installations this produces ugly output, on others it is perfectly 
satisfactory. Many systems now come with Xvfb fromX.Org (possibly as an 
optional install), which is an X11 server that does not require a screen; and 
there is the<strong>GDD</strong> package from CRAN, which produces PNG, JPEG 
and GIF bitmaps without X11.</p> 
<p></p>  Next:&nbsp; How can I turn a string into a variable?, Previous:&nbsp; 
How do I produce PNG graphics in batch mode?, Up:&nbsp; R Miscellanea <br>

<h3>7.20 How can I get command line editing to work?</h3> 
<p>The Unix command-line interface to R can only provide the inbuilt command 
line editor which allows recall, editing and re-submission of prior commands 
provided that theGNU readline library is available at the time R is configured 
for compilation. Note that the `development' version of readline including the 
appropriate headers is needed: users of Linux binary distributions will need to 
install packages such as<code>libreadline-dev</code> (Debian) or <code>
readline-devel</code> (Red Hat). </p> 
<p></p>  Next:&nbsp; Why do lattice/trellis graphics not work?, Previous:&nbsp;
How can I get command line editing to work?, Up:&nbsp; R Miscellanea <br>

<h3>7.21 How can I turn a string into a variable?</h3> 
<p>If you have </p> 
<pre> varname &lt;- c(&quot;a&quot;, &quot;b&quot;, &quot;d&quot;) </pre> 
<p>you can do </p> 
<pre> get(varname[1]) + 2 </pre> 
<p>for </p> 
<pre> a + 2 </pre> 
<p>or </p> 
<pre> assign(varname[1], 2 + 2) </pre> 
<p>for </p> 
<pre> a &lt;- 2 + 2 </pre> 
<p>or </p> 
<pre> eval(substitute(lm(y ~ x + variable), list(variable = 
as.name(varname[1]))</pre> 
<p>for </p> 
<pre> lm(y ~ x + a) </pre> 
<p>At least in the first two cases it is often easier to just use a list, and 
then you can easily index it by name</p> 
<pre> vars &lt;- list(a = 1:10, b = rnorm(100), d = LETTERS) 
vars[[&quot;a&quot;]]</pre> 
<p>without any of this messing about. </p> 
<p></p>  Next:&nbsp; How can I sort the rows of a data frame?, Previous:&nbsp; 
How can I turn a string into a variable?, Up:&nbsp; R Miscellanea <br>

<h3>7.22 Why do lattice/trellis graphics not work?</h3> 
<p>The most likely reason is that you forgot to tell R to display the graph. 
Lattice functions such as<code>xyplot()</code> create a graph object, but do 
not display it (the same is true of Trellis graphics inS-Plus). The <code>
print()</code> method for the graph object produces the actual display. When 
you use these functions interactively at the command line, the result is 
automatically printed, but in<code>source()</code> or inside your own functions 
you will need an explicit<code>print()</code> statement. </p> 
<p></p>  Next:&nbsp; Why does the help.start() search engine not work?, 
Previous:&nbsp;Why do lattice/trellis graphics not work?, Up:&nbsp; R 
Miscellanea <br>

<h3>7.23 How can I sort the rows of a data frame?</h3> 
<p>To sort the rows within a data frame, with respect to the values in one or 
more of the columns, simply use<code>order()</code> (e.g., <code>DF[order(DF$a, 
DF[[&quot;b&quot;]]), ]</code> to sort the data frame <code>DF</code> on 
columns named<code>a</code> and <code>b</code>). </p> 
<p></p>  Next:&nbsp; Why did my .Rprofile stop working when I updated R?, 
Previous:&nbsp;How can I sort the rows of a data frame?, Up:&nbsp; R Miscellanea
<br> 
<h3>7.24 Why does the help.start() search engine not work?</h3> 
<p>The browser-based search engine in <code>help.start()</code> utilizes a 
Java applet. In order for this to function properly, a compatible version of 
Java must installed on your system and linked to your browser, and both Java<em>
and</em> JavaScript need to be enabled in your browser. </p> 
<p>There have been a number of compatibility issues with versions of Java and 
of browsers. SeeEnabling search in HTML help, for further details. </p> 
<p></p>  Next:&nbsp; Where have all the methods gone?, Previous:&nbsp; Why 
does the help.start() search engine not work?, Up:&nbsp; R Miscellanea <br>

<h3>7.25 Why did my .Rprofile stop working when I updated R?</h3> 
<p>Did you read the NEWS file? For functions that are not in the <strong>base
</strong> package you need to specify the correct package namespace, since the 
code will be run<em>before</em> the packages are loaded. E.g., </p> 
<pre> ps.options(horizontal = FALSE) help.start() </pre> 
<p>needs to be </p> 
<pre> grDevices::ps.options(horizontal = FALSE) utils::help.start() </pre> 
<p>(<code>graphics::ps.options(horizontal = FALSE)</code> in R 1.9.x). </p> 
<p></p>  Next:&nbsp; How can I create rotated axis labels?, Previous:&nbsp; 
Why did my .Rprofile stop working when I updated R?, Up:&nbsp; R Miscellanea 
<br> 
<h3>7.26 Where have all the methods gone?</h3> 
<p>Many functions, particularly S3 methods, are now hidden in namespaces. This 
has the advantage that they cannot be called inadvertently with arguments of 
the wrong class, but it makes them harder to view.</p> 
<p>To see the code for an S3 method (e.g., <code>[.terms</code>) use </p> 
<pre> getS3method(&quot;[&quot;, &quot;terms&quot;) </pre> 
<p>To see the code for an unexported function <code>foo()</code> in the 
namespace of package<code>&quot;bar&quot;</code> use <code>bar:::foo</code>. 
Don't use these constructions to call unexported functions in your own 
code&mdash;they are probably unexported for a reason and may change without 
warning.</p> 
<p></p>  Next:&nbsp; Why is read.table() so inefficient?, Previous:&nbsp; 
Where have all the methods gone?, Up:&nbsp; R Miscellanea <br>

<h3>7.27 How can I create rotated axis labels?</h3> 
<p>To rotate axis labels (using base graphics), you need to use <code>text()
</code>, rather than <code>mtext()</code>, as the latter does not support <code>
par(&quot;srt&quot;)</code>. </p> 
<pre> ## Increase bottom margin to make room for rotated labels par(mar = c(7, 
4, 4, 2) + 0.1) ##Create plot with no x axis and no x axis label plot(1 : 8, 
xaxt = &quot;n&quot;, xlab = &quot;&quot;) ##Set up x axis with tick marks alone
 axis(1, labels = FALSE) ##Create some text labels labels &lt;- 
paste(&quot;Label&quot;, 1:8, sep = &quot; &quot;) ##Plot x axis labels at 
default tick marks text(1:8, par(&quot;usr&quot;)[3] - 0.25, srt = 45, adj = 1, 
labels = labels, xpd = TRUE) ##Plot x axis label at line 6 (of 7) mtext(1, text 
= &quot;X Axis Label&quot;, line = 6)</pre> 
<p>When plotting the x axis labels, we use <code>srt = 45</code> for text 
rotation angle,<code>adj = 1</code> to place the right end of text at the tick 
marks, and<code>xpd = TRUE</code> to allow for text outside the plot region. 
You can adjust the value of the<code>0.25</code> offset as required to move the 
axis labels up or down relative to the x axis. See<code>?par</code> for more 
information.</p> 
<p>Also see Figure 1 and associated code in Paul Murrell (2003), 
&ldquo;Integrating grid Graphics Output with Base Graphics Output&rdquo;,<em>R 
News</em>, <strong>3/2</strong>, 7&ndash;12. </p> 
<p></p>  Next:&nbsp; What is the difference between package and library?, 
Previous:&nbsp;How can I create rotated axis labels?, Up:&nbsp; R Miscellanea 
<br> 
<h3>7.28 Why is read.table() so inefficient?</h3> 
<p>By default, <code>read.table()</code> needs to read in everything as 
character data, and then try to figure out which variables to convert to 
numerics or factors. For a large data set, this takes condiderable amounts of 
time and memory. Performance can substantially be improved by using the<code>
colClasses</code> argument to specify the classes to be assumed for the columns 
of the table.</p> 
<p></p>  Next:&nbsp; I installed a package but the functions are not there, 
Previous:&nbsp;Why is read.table() so inefficient?, Up:&nbsp; R Miscellanea <br>
<h3>7.29 What is the difference between package and library?</h3> 
<p>A package is a standardized collection of material extending R, e.g. 
providing code, data, or documentation. Alibrary is a place (directory) where R 
knows to find packages it can use (i.e., which wereinstalled). R is told to use 
a package (to &ldquo;load&rdquo; it and add it to the search path) via calls to 
the function<code>library</code>. I.e., <code>library()</code> is employed to 
load a package from libraries containing packages.</p> 
<p>See R Add-On Packages, for more details. See also Uwe Ligges (2003), 
&ldquo;R Help Desk: Package Management&rdquo;,<em>R News</em>, <strong>3/3
</strong>, 37&ndash;39. </p> 
<p></p>  Next:&nbsp; Why doesn't R think these numbers are equal?, 
Previous:&nbsp;What is the difference between package and library?, Up:&nbsp; R 
Miscellanea <br>

<h3>7.30 I installed a package but the functions are not there</h3> 
<p>To actually <em>use</em> the package, it needs to be <em>loaded</em> using 
<code>library()</code>. </p> 
<p>See R Add-On Packages and What is the difference between package and 
library? for more information. </p> 
<p></p>  Next:&nbsp; How can I capture or ignore errors in a long simulation?, 
Previous:&nbsp;I installed a package but the functions are not there, Up:&nbsp; 
R Miscellanea <br>

<h3>7.31 Why doesn't R think these numbers are equal?</h3> 
<p>The only numbers that can be represented exactly in R's numeric type are 
integers and fractions whose denominator is a power of 2. Other numbers have to 
be rounded to (typically) 53 binary digits accuracy. As a result, two floating 
point numbers will not reliably be equal unless they have been computed by the 
same algorithm, and not always even then. For example</p> 
<pre> R&gt; a &lt;- sqrt(2) R&gt; a * a == 2 [1] FALSE R&gt; a * a - 2 [1] 
4.440892e-16</pre> 
<p>The function <code>all.equal()</code> compares two objects using a numeric 
tolerance of<code>.Machine$double.eps ^ 0.5</code>. If you want much greater 
accuracy than this you will need to consider error propagation carefully.</p> 
<p>For more information, see e.g. David Goldberg (1991), &ldquo;What Every 
Computer Scientist Should Know About Floating-Point Arithmetic&rdquo;,<em>ACM 
Computing Surveys</em>, <strong>23/1</strong>, 5&ndash;48, also available via 
http://docs.sun.com/source/806-3568/ncg_goldberg.html. </p> 
<p>To quote from &ldquo;The Elements of Programming Style&rdquo; by Kernighan 
and Plauger:</p> 
<blockquote> <em>10.0 times 0.1 is hardly ever 1.0</em>. </blockquote> 
<p></p>  Next:&nbsp; Why are powers of negative numbers wrong?, Previous:&nbsp;
Why doesn't R think these numbers are equal?, Up:&nbsp; R Miscellanea <br>

<h3>7.32 How can I capture or ignore errors in a long simulation?</h3> 
<p>Use <code>try()</code>, which returns an object of class <code>
&quot;try-error&quot;</code> instead of an error, or preferably <code>tryCatch()
</code>, where the return value can be configured more flexibly. For example 
</p> 
<pre> beta[i,] &lt;- tryCatch(coef(lm(formula, data)), error = function(e) 
rep(NaN, 4))</pre> 
<p>would return the coefficients if the <code>lm()</code> call succeeded and 
would return<code>c(NaN, NaN, NaN, NaN)</code> if it failed (presumably there 
are supposed to be 4 coefficients in this example).</p> 
<p></p>  Next:&nbsp; How can I save the result of each iteration in a loop 
into a separate file?, Previous:&nbsp; How can I capture or ignore errors in a 
long simulation?, Up:&nbsp; R Miscellanea <br>

<h3>7.33 Why are powers of negative numbers wrong?</h3> 
<p>You are probably seeing something like </p> 
<pre> R&gt; -2^2 [1] -4 </pre> 
<p>and misunderstanding the precedence rules for expressions in R. Write </p> 
<pre> R&gt; (-2)^2 [1] 4 </pre> 
<p>to get the square of -2. </p> 
<p>The precedence rules are documented in <code>?Syntax</code>, and to see how 
R interprets an expression you can look at the parse tree</p> 
<pre> R&gt; as.list(quote(-2^2)) [[1]] `-` [[2]] 2^2 </pre> 
<p></p>  Next:&nbsp; Why are p-values not displayed when using lmer()?, 
Previous:&nbsp;Why are powers of negative numbers wrong?, Up:&nbsp; R 
Miscellanea <br>

<h3>7.34 How can I save the result of each iteration in a loop into a separate 
file?</h3> 
<p>One way is to use <code>paste()</code> (or <code>sprintf()</code>) to 
concatenate a stem filename and the iteration number while<code>file.path()
</code> constructs the path. For example, to save results into files result1.rda
,..., result100.rda in the subdirectory Results of the current working 
directory, one can use</p> 
<pre> for(i in 1:100) { ## Calculations constructing &quot;some_object&quot; 
... fp &lt;- file.path(&quot;Results&quot;, paste(&quot;result&quot;, i, 
&quot;.rda&quot;, sep = &quot;&quot;)) save(list = &quot;some_object&quot;, 
file = fp) }</pre> 
<p></p>  Next:&nbsp; Why are there unwanted lines between polygons in PDF 
output viewed in Adobe Reader?, Previous:&nbsp; How can I save the result of 
each iteration in a loop into a separate file?, Up:&nbsp; R Miscellanea <br>

<h3>7.35 Why are p-values not displayed when using lmer()?</h3> 
<p>Doug Bates has kindly provided an extensive response in a post to the 
r-help list, which can be reviewed at
https://stat.ethz.ch/pipermail/r-help/2006-May/094765.html. </p> 
<p></p>  Next:&nbsp; Why does backslash behave strangely inside strings?, 
Previous:&nbsp;Why are p-values not displayed when using lmer()?, Up:&nbsp; R 
Miscellanea <br>

<h3>7.36 Why are there unwanted lines between polygons in PDF output viewed in 
Adobe Reader?</h3> 
<p>Output from <code>polygon()</code> (and other functions calling <code>
polygon()</code>) with the argument <code>border=NA</code> or <code>
border=&quot;transparent&quot;</code> should suppress border lines between 
polygons for all graphics devices.</p> 
<p>PDF output from R can be made in many ways, both directly, and through for 
example Postscript or Windows Metafiles converted to PDF in external software. 
In Adobe Reader, the default setting for line art, such as polygons, is to 
smooth, which produces the impression of thin borders. Adobe Reader does this 
both for PDF files written by R or through other software.</p> 
<p>This is irritating, especially when using Adobe Reader for presentation. 
The unwanted effect can be removed by turning off smoothing for line art: use 
the `Edit | Preferences | Page Display | Smooth line art' menu in Adobe Reader 
7.0.</p> 
<p></p>  Previous:&nbsp; Why are there unwanted lines between polygons in PDF 
output viewed in Adobe Reader?, Up:&nbsp; R Miscellanea <br>

<h3>7.37 Why does backslash behave strangely inside strings?</h3> 
<p>This question most often comes up in relation to file names (see How do 
file names work in Windows?) but it also happens that people complain that they 
cannot seem to put a single `\' character into a text string unless it happens 
to be followed by certain other characters.</p> 
<p>To understand this, you have to distinguish between character strings and 
<em>representations</em> of character strings. Mostly, the representation in R 
is just the string with a single or double quote at either end, but there are 
strings that cannot be represented that way, e.g., strings that themselves 
contains the quote character. So</p> 
<pre> &gt; str &lt;- &quot;This \&quot;text\&quot; is quoted&quot; &gt; str 
[1] &quot;This \&quot;text\&quot; is quoted&quot; &gt; cat(str, &quot;\n&quot;) 
This &quot;text&quot; is quoted</pre> 
<p>The <em>escape sequences</em> `\&quot;' and `\n' represent a double quote 
and the newline character respectively. Printing text strings, using<code>
print()</code> or by typing the name at the prompt will use the escape 
sequences too, but the<code>cat()</code> function will display the string 
as-is. Notice that `&quot;\n&quot;' is a one-character string, not two; the 
backslash is not actually in the string, it is just generated in the printed 
representation.</p> 
<pre> &gt; nchar(&quot;\n&quot;) [1] 1 &gt; substring(&quot;\n&quot;, 1, 1) 
[1] &quot;\n&quot;</pre> 
<p>So how do you put a backslash in a string? For this, you have to escape the 
escape character. I.e., you have to double the backslash. as in</p> 
<pre> &gt; cat(&quot;\\n&quot;, &quot;\n&quot;) \n </pre> 
<p>Some functions, particularly those involving regular expression matching, 
themselves use metacharacters, which may need to be escaped by the backslash 
mechanism. In those cases you may need a<em>quadruple</em> backslash to 
represent a single literal one.</p> 
<p>In versions of R up to 2.4.1 an unknown escape sequence like `\p' was 
quietly interpreted as just `p'. Current versions of R emit a warning. </p> 
<p></p>  Next:&nbsp; R Bugs, Previous:&nbsp; R Miscellanea, Up:&nbsp; Top <br>

<h2>8 R Programming</h2> 
<ul> 
<li>How should I write summary methods? </li> 
<li>How can I debug dynamically loaded code? </li> 
<li>How can I inspect R objects when debugging? </li> 
<li>How can I change compilation flags? </li> 
<li>How can I debug S4 methods? </li> </ul> 
<p></p>  Next:&nbsp; How can I debug dynamically loaded code?, Previous:&nbsp; 
R Programming, Up:&nbsp; R Programming <br>

<h3>8.1 How should I write summary methods?</h3> 
<p>Suppose you want to provide a summary method for class <code>&quot;foo&quot;
</code>. Then <code>summary.foo()</code> should not print anything, but return 
an object of class<code>&quot;summary.foo&quot;</code>, <em>and</em> you should 
write a method<code>print.summary.foo()</code> which nicely prints the summary 
information and invisibly returns its object. This approach is preferred over 
having<code>summary.foo()</code> print summary information and return something 
useful, as sometimes you need to grab something computed by<code>summary()
</code> inside a function or similar. In such cases you don't want anything 
printed.</p> 
<p></p>  Next:&nbsp; How can I inspect R objects when debugging?, 
Previous:&nbsp;How should I write summary methods?, Up:&nbsp; R Programming <br>
<h3>8.2 How can I debug dynamically loaded code?</h3> 
<p>Roughly speaking, you need to start R inside the debugger, load the code, 
send an interrupt, and then set the required breakpoints.</p> 
<p>See Finding entry points in dynamically loaded code. </p> 
<p></p>  Next:&nbsp; How can I change compilation flags?, Previous:&nbsp; How 
can I debug dynamically loaded code?, Up:&nbsp; R Programming <br>

<h3>8.3 How can I inspect R objects when debugging?</h3> 
<p>The most convenient way is to call <code>R_PV</code> from the symbolic 
debugger.</p> 
<p>See Inspecting R objects when debugging. </p> 
<p></p>  Next:&nbsp; How can I debug S4 methods?, Previous:&nbsp; How can I 
inspect R objects when debugging?, Up:&nbsp; R Programming <br>

<h3>8.4 How can I change compilation flags?</h3> 
<p>Suppose you have C code file for dynloading into R, but you want to use 
<code>R CMD SHLIB</code> with compilation flags other than the default ones 
(which were determined when R was built).</p> 
<p>Starting with R 2.1.0, users can provide personal Makevars configuration 
files in$HOME/.R to override the default flags. See Add-on packages. </p> 
<p>For earlier versions of R, you could change the file R_HOME/etc/Makeconf to 
reflect your preferences, or (at least for systems usingGNU Make) override them 
by the environment variableMAKEFLAGS. See Creating shared objects. </p> 
<p></p>  Previous:&nbsp; How can I change compilation flags?, Up:&nbsp; R 
Programming <br>

<h3>8.5 How can I debug S4 methods?</h3> 
<p>Use the <code>trace()</code> function with argument <code>signature=</code> 
to add calls to the browser or any other code to the method that will be 
dispatched for the corresponding signature. See<code>?trace</code> for details. 
</p> 
<p></p>  Next:&nbsp; Acknowledgments, Previous:&nbsp; R Programming, Up:&nbsp; 
Top <br>

<h2>9 R Bugs</h2> 
<ul> 
<li>What is a bug? </li> 
<li>How to report a bug </li> </ul> 
<p></p>  Next:&nbsp; How to report a bug, Previous:&nbsp; R Bugs, Up:&nbsp; R 
Bugs <br>

<h3>9.1 What is a bug?</h3> 
<p>If R executes an illegal instruction, or dies with an operating system 
error message that indicates a problem in the program (as opposed to something 
like &ldquo;disk full&rdquo;), then it is certainly a bug. If you call<code>.C()
</code>, <code>.Fortran()</code>, <code>.External()</code> or <code>.Call()
</code> (or <code>.Internal()</code>) yourself (or in a function you wrote), 
you can always crash R by using wrong argument types (modes). This is not a bug.
</p> 
<p>Taking forever to complete a command can be a bug, but you must make 
certain that it was really R's fault. Some commands simply take a long time. If 
the input was such that you<em>know</em> it should have been processed quickly, 
report a bug. If you don't know whether the command should take a long time, 
find out by looking in the manual or by asking for assistance.</p> 
<p>If a command you are familiar with causes an R error message in a case 
where its usual definition ought to be reasonable, it is probably a bug. If a 
command does the wrong thing, that is a bug. But be sure you know for certain 
what it ought to have done. If you aren't familiar with the command, or don't 
know for certain how the command is supposed to work, then it might actually be 
working right. For example, people sometimes think there is a bug in R's 
mathematics because they don't understand how finite-precision arithmetic 
works. Rather than jumping to conclusions, show the problem to someone who 
knows for certain. Unexpected results of comparison of decimal numbers, for 
example<code>0.28 * 100 != 28</code> or <code>0.1 + 0.2 != 0.3</code>, are not 
a bug. SeeWhy doesn't R think these numbers are equal?, for more details. </p> 
<p>Finally, a command's intended definition may not be best for statistical 
analysis. This is a very important sort of problem, but it is also a matter of 
judgment. Also, it is easy to come to such a conclusion out of ignorance of 
some of the existing features. It is probably best not to complain about such a 
problem until you have checked the documentation in the usual ways, feel 
confident that you understand it, and know for certain that what you want is 
not available. If you are not sure what the command is supposed to do after a 
careful reading of the manual this indicates a bug in the manual. The manual's 
job is to make everything clear. It is just as important to report 
documentation bugs as program bugs. However, we know that the introductory 
documentation is seriously inadequate, so you don't need to report this.</p> 
<p>If the online argument list of a function disagrees with the manual, one of 
them must be wrong, so report the bug.</p> 
<p></p>  Previous:&nbsp; What is a bug?, Up:&nbsp; R Bugs <br>

<h3>9.2 How to report a bug</h3> 
<p>When you decide that there is a bug, it is important to report it and to 
report it in a way which is useful. What is most useful is an exact description 
of what commands you type, starting with the shell command to run R, until the 
problem happens. Always include the version of R, machine, and operating system 
that you are using; typeversion in R to print this. </p> 
<p>The most important principle in reporting a bug is to report <em>facts</em>
, not hypotheses or categorizations. It is always easier to report the facts, 
but people seem to prefer to strain to posit explanations and report them 
instead. If the explanations are based on guesses about how R is implemented, 
they will be useless; others will have to try to figure out what the facts must 
have been to lead to such speculations. Sometimes this is impossible. But in 
any case, it is unnecessary work for the ones trying to fix the problem.</p> 
<p>For example, suppose that on a data set which you know to be quite large 
the command</p> 
<pre> R&gt; data.frame(x, y, z, monday, tuesday) </pre> 
<p>never returns. Do not report that <code>data.frame()</code> fails for large 
data sets. Perhaps it fails when a variable name is a day of the week. If this 
is so then when others got your report they would try out the<code>data.frame()
</code> command on a large data set, probably with no day of the week variable 
name, and not see any problem. There is no way in the world that others could 
guess that they should try a day of the week variable name.</p> 
<p>Or perhaps the command fails because the last command you used was a method 
for<code>&quot;[&quot;()</code> that had a bug causing R's internal data 
structures to be corrupted and making the<code>data.frame()</code> command fail 
from then on. This is why others need to know what other commands you have 
typed (or read from your startup file).</p> 
<p>It is very useful to try and find simple examples that produce apparently 
the same bug, and somewhat useful to find simple examples that might be 
expected to produce the bug but actually do not. If you want to debug the 
problem and find exactly what caused it, that is wonderful. You should still 
report the facts as well as any explanations or solutions. Please include an 
example that reproduces the problem, preferably the simplest one you have found.
</p> 
<p>Invoking R with the --vanilla option may help in isolating a bug. This 
ensures that the site profile and saved data files are not read.</p> 
<p>Before you actually submit a bug report, you should check whether the bug 
has already been reported and/or fixed. First, try the &ldquo;Search Existing 
Reports&rdquo; facility in the Bug Tracking page athttp://bugs.R-project.org/. 
Second, consulthttps://svn.R-project.org/R/trunk/NEWS, which records changes 
that will appear in the<em>next</em> release of R, including some bug fixes 
that do not appear in Bug Tracking. (Windows users should additionally consult
https://svn.R-project.org/R/trunk/src/gnuwin32/CHANGES.) Third, if possible try 
the current r-patched or r-devel version of R. If a bug has already been 
reported or fixed, please do not submit further bug reports on it. Finally, 
check carefully whether the bug is with R, or a contributed package. Bug 
reports on contributed packages should be sent first to the package maintainer, 
and only submitted to the R-bugs repository by package maintainers, mentioning 
the package in the subject line.</p> 
<p>On Unix systems a bug report can be generated using the function <code>
bug.report()</code>. This automatically includes the version information and 
sends the bug to the correct address. Alternatively the bug report can be 
emailed toR-bugs@R-project.org or submitted to the Web page at 
http://bugs.R-project.org/. Please try including results of <code>sessionInfo()
</code> in your bug report. </p> 
<p>There is a section of the bug repository for suggestions for enhancements 
for R labelled `wishlist'. Suggestions can be submitted in the same ways as 
bugs, but please ensure that the subject line makes clear that this is for the 
wishlist and not a bug report, for example by starting with `Wishlist:'. </p> 
<p>Comments on and suggestions for the Windows port of R should be sent to 
R-windows@R-project.org. </p> 
<p>Corrections to and comments on message translation should be sent to the 
last translator (listed at the top of the appropriate `.po' file) or to the 
translation team as listed at
http://developer.R-project.org/TranslationTeams.html. </p> 
<p></p>  Previous:&nbsp; R Bugs, Up:&nbsp; Top <br>

<h2>10 Acknowledgments</h2> 
<p>Of course, many many thanks to Robert and Ross for the R system, and to the 
package writers and porters for adding to it.</p> 
<p>Special thanks go to Doug Bates, Peter Dalgaard, Paul Gilbert, Stefano 
Iacus, Fritz Leisch, Jim Lindsey, Thomas Lumley, Martin Maechler, Brian D. 
Ripley, Anthony Rossini, and Andreas Weingessel for their comments which helped 
me improve thisFAQ. </p> 
<p>More to come soon ... </p> 
</body>